{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import time\n",
    "pd.set_option('display.max_rows', None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>module</th>\n",
       "      <th>usage_2019_01</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2pg_cartesian</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>a5-miseq</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>abawaca</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>abeona</td>\n",
       "      <td>1618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>abra2</td>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>abricate</td>\n",
       "      <td>1187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>abritamr</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>abundancebin</td>\n",
       "      <td>3422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>abyss</td>\n",
       "      <td>1633</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>ac</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>ac-diamond</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>adam</td>\n",
       "      <td>305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>adapterremoval</td>\n",
       "      <td>4908</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>adapterremovalfixprefix</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>addrg</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>admixtools</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>admixture</td>\n",
       "      <td>156</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>advntr</td>\n",
       "      <td>794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>afplot</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>afterqc</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>agat</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>age-metasv</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>agfusion</td>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>agg</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>airr</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>akt</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>albatradis</td>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>alder</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>ale</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>alfred</td>\n",
       "      <td>1244</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>align_it</td>\n",
       "      <td>231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>alignlib-lite</td>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>alignstats</td>\n",
       "      <td>479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>allegro</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>altamisa</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>amap</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>amas</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>amos</td>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>amptk</td>\n",
       "      <td>402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>amused</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>andi</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>angsd</td>\n",
       "      <td>516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>anise_basil</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>anndata2ri</td>\n",
       "      <td>60</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>ant</td>\n",
       "      <td>213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>antarna</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>antismash</td>\n",
       "      <td>452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>anvio</td>\n",
       "      <td>1327</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>anvio-minimal</td>\n",
       "      <td>683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>any2fasta</td>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>apoc</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>apollo</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>apt-probeset-summarize</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>aquila</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>aquila_stlfr</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>aquila_umap</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>aragorn</td>\n",
       "      <td>1500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>arb-bio</td>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>arboreto</td>\n",
       "      <td>51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>arcs</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>arem</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>aria2</td>\n",
       "      <td>324</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>ariba</td>\n",
       "      <td>1376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>array-as-vcf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>arriba</td>\n",
       "      <td>432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>art</td>\n",
       "      <td>402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>artemis</td>\n",
       "      <td>364</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>arvados-cli</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>arvados-cwl-runner</td>\n",
       "      <td>2537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>arvados-python-client</td>\n",
       "      <td>2510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>ascat</td>\n",
       "      <td>319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>asciigenome</td>\n",
       "      <td>838</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>askocli</td>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>asn2gb</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>asqcan</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>assemblerflow</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>assembly-scan</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>assembly-stats</td>\n",
       "      <td>250</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>assemblytics</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>assemblyutility</td>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>astalavista</td>\n",
       "      <td>373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>atac</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>atactk</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>ataqv</td>\n",
       "      <td>420</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>athena_meta</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>atlas-metadata-validator</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>atropos</td>\n",
       "      <td>1420</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>augur</td>\n",
       "      <td>53</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>augustus</td>\n",
       "      <td>1994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>autodock-vina</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>autolog</td>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>avro-cwl</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>avro-python2</td>\n",
       "      <td>431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>avro-python3</td>\n",
       "      <td>416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>axiome</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>azure-cli</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>backspinpy</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>bactopia</td>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>baitfisher</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>bam2fasta</td>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>bam2fastx</td>\n",
       "      <td>250</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>bambamc</td>\n",
       "      <td>163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>bamclipper</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>bamhash</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>bamm</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>bamread</td>\n",
       "      <td>334</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>bam-readcount</td>\n",
       "      <td>2597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>bamscale</td>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>bamtools</td>\n",
       "      <td>4141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>bamutil</td>\n",
       "      <td>391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>bandage</td>\n",
       "      <td>224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>banner</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>barcode_splitter</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>barriers</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>barrnap</td>\n",
       "      <td>1993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>barrnap-python</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>basic</td>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>batvi</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>bax2bam</td>\n",
       "      <td>169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>bayescan</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>bayestyper</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>bazam</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>bbknn</td>\n",
       "      <td>308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>bbmap</td>\n",
       "      <td>7568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>bcalm</td>\n",
       "      <td>226</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>bcbio-gff</td>\n",
       "      <td>166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>bcbiogff</td>\n",
       "      <td>538</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>bcbio_monitor</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>bcbio-nextgen</td>\n",
       "      <td>4567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>bcbio-nextgen-vm</td>\n",
       "      <td>835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>bcbio-prioritize</td>\n",
       "      <td>665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>bcbio-rnaseq</td>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>bcbio-variation</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>bcbio-variation-recall</td>\n",
       "      <td>592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>bcbreport</td>\n",
       "      <td>961</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>bcftools</td>\n",
       "      <td>44152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>bcftools-gtc2vcf-plugin</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>bcftools-snvphyl-plugin</td>\n",
       "      <td>382</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>bcl2fastq-nextseq</td>\n",
       "      <td>225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>bcool</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>bcov</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>bctools</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>bd2k-python-lib</td>\n",
       "      <td>709</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>beagle</td>\n",
       "      <td>348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>beagle-lib</td>\n",
       "      <td>625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>beast2</td>\n",
       "      <td>224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>beast</td>\n",
       "      <td>722</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>bedops</td>\n",
       "      <td>2257</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>bedtools</td>\n",
       "      <td>24168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>behst</td>\n",
       "      <td>1116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>bellmans-gapc</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>bel-resources</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>berokka</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>besst</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>bfc</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>bftools</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>bgreat</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>bgt</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>biasaway</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>bibliospec</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>bicseq2-norm</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>bicseq2-seg</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>bigsi</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>bindash</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>binsanity</td>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>bio_assembly_refinement</td>\n",
       "      <td>250</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>bioawk</td>\n",
       "      <td>500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>biobambam</td>\n",
       "      <td>1597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>biobasehttptools</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>biobb_adapters</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>biobb_analysis</td>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>biobb_chemistry</td>\n",
       "      <td>27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>biobb_common</td>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>biobb_io</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>biobb_md</td>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>biobb_model</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>biobb_pmx</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>biobb_structure_checking</td>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>biobb_structure_manager</td>\n",
       "      <td>60</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>biobb_structure_utils</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>biobb_wf_mutations</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>bioblend</td>\n",
       "      <td>1965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>bioconda-utils</td>\n",
       "      <td>1024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>bioconductor-a4</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>bioconductor-a4base</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>bioconductor-a4classif</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>bioconductor-a4core</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>bioconductor-a4preproc</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>bioconductor-a4reporting</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>bioconductor-abadata</td>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>bioconductor-abaenrichment</td>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>bioconductor-abarray</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>bioconductor-abseqr</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>bioconductor-absseq</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>bioconductor-acde</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>bioconductor-ace</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>bioconductor-acgh</td>\n",
       "      <td>576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>bioconductor-acme</td>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>bioconductor-adacgh2</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>bioconductor-adam</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>bioconductor-adamgui</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>bioconductor-adaptest</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>bioconductor-adductdata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>bioconductor-adductomicsr</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>bioconductor-adme16cod.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>bioconductor-adsplit</td>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>bioconductor-affixcan</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>bioconductor-affxparser</td>\n",
       "      <td>626</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>bioconductor-affy</td>\n",
       "      <td>1354</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>bioconductor-affycomp</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>bioconductor-affycompatible</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>bioconductor-affycompdata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>bioconductor-affycontam</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>bioconductor-affycoretools</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>bioconductor-affydata</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>bioconductor-affyexpress</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>bioconductor-affyhgu133a2expr</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>bioconductor-affyhgu133aexpr</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>bioconductor-affyhgu133plus2expr</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>bioconductor-affyilm</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>bioconductor-affyio</td>\n",
       "      <td>1368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>bioconductor-affylmgui</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>bioconductor-affymetrixdatatestfiles</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>bioconductor-affymoe4302expr</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>bioconductor-affypara</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>bioconductor-affyplm</td>\n",
       "      <td>499</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>bioconductor-affyqcreport</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>bioconductor-affyrnadegradation</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>bioconductor-agcdf</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>bioconductor-ag.db</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>bioconductor-agdex</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>bioconductor-agilp</td>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>bioconductor-agimicrorna</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>bioconductor-agprobe</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>bioconductor-ahcytobands</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>bioconductor-ahensdbs</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>bioconductor-aims</td>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>bioconductor-airway</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>bioconductor-aldex2</td>\n",
       "      <td>312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>bioconductor-alevinqc</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>bioconductor-all</td>\n",
       "      <td>504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>bioconductor-allelicimbalance</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>bioconductor-allmll</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>bioconductor-alphabeta</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>bioconductor-alpine</td>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>bioconductor-alpinedata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>bioconductor-alps</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>bioconductor-alsace</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>bioconductor-altcdfenvs</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>bioconductor-alternativesplicingevents.hg19</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>bioconductor-alternativesplicingevents.hg38</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>bioconductor-amaretto</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>bioconductor-amountain</td>\n",
       "      <td>573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>bioconductor-ampaffyexample</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>bioconductor-amplican</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>bioconductor-analysispageserver</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>bioconductor-anaquin</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>bioconductor-aneufinder</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>bioconductor-aneufinderdata</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>bioconductor-anf</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>bioconductor-animalcules</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>bioconductor-annaffy</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>bioconductor-annmap</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>bioconductor-annotate</td>\n",
       "      <td>3317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>bioconductor-annotationdbi</td>\n",
       "      <td>4575</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>bioconductor-annotationfilter</td>\n",
       "      <td>873</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>bioconductor-annotationforge</td>\n",
       "      <td>686</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>bioconductor-annotationfuncs</td>\n",
       "      <td>341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>bioconductor-annotationhub</td>\n",
       "      <td>904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>bioconductor-annotationhubdata</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>bioconductor-annotationtools</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>bioconductor-annotatr</td>\n",
       "      <td>175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>bioconductor-anopheles.db0</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>bioconductor-anota</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>bioconductor-anota2seq</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>bioconductor-antiprofiles</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>bioconductor-antiprofilesdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>bioconductor-apalyzer</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>bioconductor-apcomplex</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>bioconductor-apeglm</td>\n",
       "      <td>685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>bioconductor-appreci8r</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>bioconductor-arabidopsis.db0</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>bioconductor-aracne.networks</td>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>bioconductor-aroma.light</td>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>bioconductor-arrayexpress</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>bioconductor-arrayexpresshts</td>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>286</th>\n",
       "      <td>bioconductor-arraymvout</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>287</th>\n",
       "      <td>bioconductor-arrayquality</td>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>bioconductor-arrayqualitymetrics</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>bioconductor-arraytools</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>bioconductor-arraytv</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>bioconductor-arrmdata</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>bioconductor-arrmnormalization</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>bioconductor-artms</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>bioconductor-asafe</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>bioconductor-aseb</td>\n",
       "      <td>584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>bioconductor-asgsca</td>\n",
       "      <td>301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>bioconductor-ashkenazimsonchr21</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>bioconductor-asics</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>bioconductor-asicsdata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>bioconductor-aspediafi</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>bioconductor-aspli</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>bioconductor-assessorf</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>bioconductor-assessorfdata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>bioconductor-asset</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>bioconductor-assign</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>bioconductor-atacseqqc</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>bioconductor-ath1121501cdf</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>bioconductor-ath1121501.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>bioconductor-ath1121501probe</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>bioconductor-atsnp</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>bioconductor-attract</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>bioconductor-aucell</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>bioconductor-autotuner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>bioconductor-awfisher</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>bioconductor-baalchip</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>bioconductor-bac</td>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>bioconductor-bacon</td>\n",
       "      <td>676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>bioconductor-bader</td>\n",
       "      <td>660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>bioconductor-badregionfinder</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>bioconductor-bags</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>bioconductor-ballgown</td>\n",
       "      <td>432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>bioconductor-bamsignals</td>\n",
       "      <td>503</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>bioconductor-bandits</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>bioconductor-banocc</td>\n",
       "      <td>305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>bioconductor-barley1cdf</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>bioconductor-barley1probe</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>bioconductor-basecallqc</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>bioconductor-basespacer</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>bioconductor-basic4cseq</td>\n",
       "      <td>391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>bioconductor-basics</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>331</th>\n",
       "      <td>bioconductor-basicstarrseq</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>332</th>\n",
       "      <td>bioconductor-batchelor</td>\n",
       "      <td>259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>333</th>\n",
       "      <td>bioconductor-batchqc</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>334</th>\n",
       "      <td>bioconductor-bayesknockdown</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>335</th>\n",
       "      <td>bioconductor-bayespeak</td>\n",
       "      <td>675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>bioconductor-baynorm</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>bioconductor-bayseq</td>\n",
       "      <td>325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>338</th>\n",
       "      <td>bioconductor-bbcanalyzer</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>339</th>\n",
       "      <td>bioconductor-bcellviper</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>340</th>\n",
       "      <td>bioconductor-bcrank</td>\n",
       "      <td>565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>bioconductor-bcseq</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>342</th>\n",
       "      <td>bioconductor-bdmmacorrect</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>343</th>\n",
       "      <td>bioconductor-beachmat</td>\n",
       "      <td>1158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>344</th>\n",
       "      <td>bioconductor-beadarray</td>\n",
       "      <td>585</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>bioconductor-beadarrayexampledata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>346</th>\n",
       "      <td>bioconductor-beadarraysnp</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>347</th>\n",
       "      <td>bioconductor-beadarrayusecases</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>348</th>\n",
       "      <td>bioconductor-beaddatapackr</td>\n",
       "      <td>572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>349</th>\n",
       "      <td>bioconductor-bearscc</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>bioconductor-beat</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>bioconductor-beclear</td>\n",
       "      <td>488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>bioconductor-benchmarkfdrdata2019</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>bioconductor-beta7</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>bioconductor-bgafun</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>bioconductor-bgeedb</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>356</th>\n",
       "      <td>bioconductor-bgmix</td>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>bioconductor-bgx</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>358</th>\n",
       "      <td>bioconductor-bhc</td>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>bioconductor-bicare</td>\n",
       "      <td>572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>360</th>\n",
       "      <td>bioconductor-bifet</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>bioconductor-bigmelon</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>362</th>\n",
       "      <td>bioconductor-bigmemoryextras</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>363</th>\n",
       "      <td>bioconductor-bigpint</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>364</th>\n",
       "      <td>bioconductor-bioassayr</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>365</th>\n",
       "      <td>bioconductor-biobase</td>\n",
       "      <td>12941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>bioconductor-biobroom</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>bioconductor-biocancer</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>bioconductor-bioccasestudies</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>bioconductor-bioccheck</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>bioconductor-biocfilecache</td>\n",
       "      <td>1225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>371</th>\n",
       "      <td>bioconductor-biocgenerics</td>\n",
       "      <td>13832</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>372</th>\n",
       "      <td>bioconductor-biocgraph</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>bioconductor-biocneighbors</td>\n",
       "      <td>643</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>bioconductor-bioconcotk</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>bioconductor-biocor</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>bioconductor-biocparallel</td>\n",
       "      <td>12476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>bioconductor-biocpkgtools</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>bioconductor-biocsingular</td>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>bioconductor-biocsklearn</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>bioconductor-biocstyle</td>\n",
       "      <td>919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>bioconductor-biocversion</td>\n",
       "      <td>160</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>bioconductor-biocviews</td>\n",
       "      <td>601</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>bioconductor-biocworkflowtools</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>bioconductor-biodist</td>\n",
       "      <td>304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>bioconductor-biomart</td>\n",
       "      <td>2686</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>bioconductor-biomformat</td>\n",
       "      <td>655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>bioconductor-biomm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>388</th>\n",
       "      <td>bioconductor-biomvcclass</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>389</th>\n",
       "      <td>bioconductor-biomvrcns</td>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>bioconductor-bionet</td>\n",
       "      <td>391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>391</th>\n",
       "      <td>bioconductor-bionetstat</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>bioconductor-bioqc</td>\n",
       "      <td>467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>393</th>\n",
       "      <td>bioconductor-bioseqclass</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>394</th>\n",
       "      <td>bioconductor-biosigner</td>\n",
       "      <td>552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>bioconductor-biostrings</td>\n",
       "      <td>11390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>396</th>\n",
       "      <td>bioconductor-biosvd</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>bioconductor-biotip</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>398</th>\n",
       "      <td>bioconductor-biotmle</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>bioconductor-biotmledata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>400</th>\n",
       "      <td>bioconductor-biovizbase</td>\n",
       "      <td>1204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>401</th>\n",
       "      <td>bioconductor-birewire</td>\n",
       "      <td>488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>402</th>\n",
       "      <td>bioconductor-birta</td>\n",
       "      <td>557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>bioconductor-biscuiteer</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>404</th>\n",
       "      <td>bioconductor-biscuiteerdata</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>405</th>\n",
       "      <td>bioconductor-biseq</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>bioconductor-bitseq</td>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>bioconductor-blacksheepr</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>bioconductor-bladderbatch</td>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>bioconductor-blima</td>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>bioconductor-blimatestingdata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>bioconductor-blma</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>412</th>\n",
       "      <td>bioconductor-bloodcancermultiomics2017</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>413</th>\n",
       "      <td>bioconductor-bnbc</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>bioconductor-bodymaprat</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>415</th>\n",
       "      <td>bioconductor-bovinecdf</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>bioconductor-bovine.db0</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>417</th>\n",
       "      <td>bioconductor-bovine.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>bioconductor-bovineprobe</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>419</th>\n",
       "      <td>bioconductor-bprmeth</td>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>bioconductor-brain</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>bioconductor-brainimagerdata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>bioconductor-brainstars</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>bioconductor-branchpointer</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>bioconductor-breakpointr</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>425</th>\n",
       "      <td>bioconductor-breakpointrdata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>bioconductor-breastcancermainz</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>427</th>\n",
       "      <td>bioconductor-breastcancernki</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>bioconductor-breastcancertransbig</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>429</th>\n",
       "      <td>bioconductor-breastcancerunt</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>430</th>\n",
       "      <td>bioconductor-breastcancerupp</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>431</th>\n",
       "      <td>bioconductor-breastcancervdx</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>432</th>\n",
       "      <td>bioconductor-brendadb</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>bioconductor-brgedata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>434</th>\n",
       "      <td>bioconductor-bridge</td>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>bioconductor-bridgedbr</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>bioconductor-bronchialil13</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>437</th>\n",
       "      <td>bioconductor-browserviz</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>438</th>\n",
       "      <td>bioconductor-bsgenome</td>\n",
       "      <td>1622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>439</th>\n",
       "      <td>bioconductor-bsgenome.alyrata.jgi.v1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>440</th>\n",
       "      <td>bioconductor-bsgenome.amellifera.beebase.assem...</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>bioconductor-bsgenome.amellifera.ucsc.apimel2</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>442</th>\n",
       "      <td>bioconductor-bsgenome.amellifera.ucsc.apimel2....</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>bioconductor-bsgenome.aofficinalis.ncbi.v1</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>444</th>\n",
       "      <td>bioconductor-bsgenome.athaliana.tair.04232008</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>445</th>\n",
       "      <td>bioconductor-bsgenome.athaliana.tair.tair9</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>bioconductor-bsgenome.btaurus.ucsc.bostau3</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>447</th>\n",
       "      <td>bioconductor-bsgenome.btaurus.ucsc.bostau3.masked</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>bioconductor-bsgenome.btaurus.ucsc.bostau4</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>449</th>\n",
       "      <td>bioconductor-bsgenome.btaurus.ucsc.bostau4.masked</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>bioconductor-bsgenome.btaurus.ucsc.bostau6</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>451</th>\n",
       "      <td>bioconductor-bsgenome.btaurus.ucsc.bostau6.masked</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>452</th>\n",
       "      <td>bioconductor-bsgenome.btaurus.ucsc.bostau8</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>453</th>\n",
       "      <td>bioconductor-bsgenome.carietinum.ncbi.v1</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>454</th>\n",
       "      <td>bioconductor-bsgenome.celegans.ucsc.ce10</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>bioconductor-bsgenome.celegans.ucsc.ce11</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>456</th>\n",
       "      <td>bioconductor-bsgenome.celegans.ucsc.ce2</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>bioconductor-bsgenome.celegans.ucsc.ce6</td>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>bioconductor-bsgenome.cfamiliaris.ucsc.canfam2</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>bioconductor-bsgenome.cfamiliaris.ucsc.canfam2...</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>bioconductor-bsgenome.cfamiliaris.ucsc.canfam3</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>461</th>\n",
       "      <td>bioconductor-bsgenome.cfamiliaris.ucsc.canfam3...</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>bioconductor-bsgenome.cjacchus.ucsc.caljac3</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>463</th>\n",
       "      <td>bioconductor-bsgenome.dmelanogaster.ucsc.dm2</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>464</th>\n",
       "      <td>bioconductor-bsgenome.dmelanogaster.ucsc.dm2.m...</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>465</th>\n",
       "      <td>bioconductor-bsgenome.dmelanogaster.ucsc.dm3</td>\n",
       "      <td>114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>bioconductor-bsgenome.dmelanogaster.ucsc.dm3.m...</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>467</th>\n",
       "      <td>bioconductor-bsgenome.dmelanogaster.ucsc.dm6</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>bioconductor-bsgenome.drerio.ucsc.danrer10</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>469</th>\n",
       "      <td>bioconductor-bsgenome.drerio.ucsc.danrer5</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>bioconductor-bsgenome.drerio.ucsc.danrer5.masked</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>bioconductor-bsgenome.drerio.ucsc.danrer6</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>bioconductor-bsgenome.drerio.ucsc.danrer6.masked</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>bioconductor-bsgenome.drerio.ucsc.danrer7</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>bioconductor-bsgenome.drerio.ucsc.danrer7.masked</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>bioconductor-bsgenome.ecoli.ncbi.20080805</td>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>476</th>\n",
       "      <td>bioconductor-bsgenome.gaculeatus.ucsc.gasacu1</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>477</th>\n",
       "      <td>bioconductor-bsgenome.gaculeatus.ucsc.gasacu1....</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>bioconductor-bsgenome.ggallus.ucsc.galgal3</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>bioconductor-bsgenome.ggallus.ucsc.galgal3.masked</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>bioconductor-bsgenome.ggallus.ucsc.galgal4</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>bioconductor-bsgenome.ggallus.ucsc.galgal4.masked</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>bioconductor-bsgenome.ggallus.ucsc.galgal5</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>483</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.1000genomes.hs37d5</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>484</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ncbi.grch38</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ucsc.hg17</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>486</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ucsc.hg17.masked</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>487</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ucsc.hg18</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>488</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ucsc.hg18.masked</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>489</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ucsc.hg19</td>\n",
       "      <td>355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ucsc.hg19.masked</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>491</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ucsc.hg38</td>\n",
       "      <td>373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>492</th>\n",
       "      <td>bioconductor-bsgenome.hsapiens.ucsc.hg38.masked</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>493</th>\n",
       "      <td>bioconductor-bsgenome.mdomestica.ucsc.mondom5</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>bioconductor-bsgenome.mfascicularis.ncbi.5.0</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>bioconductor-bsgenome.mfuro.ucsc.musfur1</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>bioconductor-bsgenome.mmulatta.ucsc.rhemac2</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>bioconductor-bsgenome.mmulatta.ucsc.rhemac2.ma...</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>bioconductor-bsgenome.mmulatta.ucsc.rhemac3</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>bioconductor-bsgenome.mmulatta.ucsc.rhemac3.ma...</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>500</th>\n",
       "      <td>bioconductor-bsgenome.mmulatta.ucsc.rhemac8</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>bioconductor-bsgenome.mmusculus.ucsc.mm10</td>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>502</th>\n",
       "      <td>bioconductor-bsgenome.mmusculus.ucsc.mm10.masked</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>bioconductor-bsgenome.mmusculus.ucsc.mm8</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>bioconductor-bsgenome.mmusculus.ucsc.mm8.masked</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>bioconductor-bsgenome.mmusculus.ucsc.mm9</td>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>bioconductor-bsgenome.mmusculus.ucsc.mm9.masked</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>bioconductor-bsgenome.osativa.msu.msu7</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>508</th>\n",
       "      <td>bioconductor-bsgenome.ptroglodytes.ucsc.pantro2</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>bioconductor-bsgenome.ptroglodytes.ucsc.pantro...</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>bioconductor-bsgenome.ptroglodytes.ucsc.pantro3</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>bioconductor-bsgenome.ptroglodytes.ucsc.pantro...</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>bioconductor-bsgenome.ptroglodytes.ucsc.pantro5</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>bioconductor-bsgenome.ptroglodytes.ucsc.pantro6</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>514</th>\n",
       "      <td>bioconductor-bsgenome.rnorvegicus.ucsc.rn4</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>515</th>\n",
       "      <td>bioconductor-bsgenome.rnorvegicus.ucsc.rn4.masked</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>516</th>\n",
       "      <td>bioconductor-bsgenome.rnorvegicus.ucsc.rn5</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>517</th>\n",
       "      <td>bioconductor-bsgenome.rnorvegicus.ucsc.rn5.masked</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>bioconductor-bsgenome.rnorvegicus.ucsc.rn6</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>bioconductor-bsgenome.scerevisiae.ucsc.saccer1</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>520</th>\n",
       "      <td>bioconductor-bsgenome.scerevisiae.ucsc.saccer2</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>521</th>\n",
       "      <td>bioconductor-bsgenome.scerevisiae.ucsc.saccer3</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>522</th>\n",
       "      <td>bioconductor-bsgenome.sscrofa.ucsc.susscr11</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>523</th>\n",
       "      <td>bioconductor-bsgenome.sscrofa.ucsc.susscr3</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>524</th>\n",
       "      <td>bioconductor-bsgenome.sscrofa.ucsc.susscr3.masked</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>525</th>\n",
       "      <td>bioconductor-bsgenome.tgondii.toxodb.7.0</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>526</th>\n",
       "      <td>bioconductor-bsgenome.tguttata.ucsc.taegut1</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>527</th>\n",
       "      <td>bioconductor-bsgenome.tguttata.ucsc.taegut1.ma...</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>528</th>\n",
       "      <td>bioconductor-bsgenome.tguttata.ucsc.taegut2</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>529</th>\n",
       "      <td>bioconductor-bsgenome.vvinifera.urgi.iggp12xv0</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>530</th>\n",
       "      <td>bioconductor-bsgenome.vvinifera.urgi.iggp12xv2</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>531</th>\n",
       "      <td>bioconductor-bsgenome.vvinifera.urgi.iggp8x</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>532</th>\n",
       "      <td>bioconductor-bsseq</td>\n",
       "      <td>641</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>533</th>\n",
       "      <td>bioconductor-bsseqdata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>bioconductor-bsubtiliscdf</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>535</th>\n",
       "      <td>bioconductor-bsubtilisprobe</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>536</th>\n",
       "      <td>bioconductor-bubbletree</td>\n",
       "      <td>587</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>537</th>\n",
       "      <td>bioconductor-bufferedmatrix</td>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>538</th>\n",
       "      <td>bioconductor-bufferedmatrixmethods</td>\n",
       "      <td>464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>539</th>\n",
       "      <td>bioconductor-bumhmm</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>540</th>\n",
       "      <td>bioconductor-bumphunter</td>\n",
       "      <td>811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>541</th>\n",
       "      <td>bioconductor-bus</td>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>542</th>\n",
       "      <td>bioconductor-buscorrect</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>543</th>\n",
       "      <td>bioconductor-busparse</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>544</th>\n",
       "      <td>bioconductor-cafe</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>545</th>\n",
       "      <td>bioconductor-cagefightr</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>546</th>\n",
       "      <td>bioconductor-cager</td>\n",
       "      <td>383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>547</th>\n",
       "      <td>bioconductor-calm</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>548</th>\n",
       "      <td>bioconductor-camera</td>\n",
       "      <td>920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>bioconductor-camthc</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>bioconductor-cancerclass</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>bioconductor-cancerdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>552</th>\n",
       "      <td>bioconductor-cancerinsilico</td>\n",
       "      <td>558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>553</th>\n",
       "      <td>bioconductor-cancermutationanalysis</td>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>bioconductor-cancersubtypes</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>555</th>\n",
       "      <td>bioconductor-cand</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>556</th>\n",
       "      <td>bioconductor-canine2cdf</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>557</th>\n",
       "      <td>bioconductor-canine2.db</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>558</th>\n",
       "      <td>bioconductor-canine2probe</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>559</th>\n",
       "      <td>bioconductor-caninecdf</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>560</th>\n",
       "      <td>bioconductor-canine.db0</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>561</th>\n",
       "      <td>bioconductor-canine.db</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>562</th>\n",
       "      <td>bioconductor-canineprobe</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>563</th>\n",
       "      <td>bioconductor-caomicsv</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>564</th>\n",
       "      <td>bioconductor-cardinal</td>\n",
       "      <td>763</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>565</th>\n",
       "      <td>bioconductor-cardinalworkflows</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>566</th>\n",
       "      <td>bioconductor-casper</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>567</th>\n",
       "      <td>bioconductor-catalyst</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>568</th>\n",
       "      <td>bioconductor-category</td>\n",
       "      <td>594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>569</th>\n",
       "      <td>bioconductor-categorycompare</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>570</th>\n",
       "      <td>bioconductor-causalr</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>571</th>\n",
       "      <td>bioconductor-cbaf</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>572</th>\n",
       "      <td>bioconductor-ccdata</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>573</th>\n",
       "      <td>bioconductor-ccfindr</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>574</th>\n",
       "      <td>bioconductor-ccl4</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>575</th>\n",
       "      <td>bioconductor-ccmap</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>576</th>\n",
       "      <td>bioconductor-ccpromise</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>577</th>\n",
       "      <td>bioconductor-ccrepe</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>578</th>\n",
       "      <td>bioconductor-cctutorial</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>bioconductor-celaref</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>580</th>\n",
       "      <td>bioconductor-celarefdata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>581</th>\n",
       "      <td>bioconductor-celda</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>bioconductor-celeganscdf</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>583</th>\n",
       "      <td>bioconductor-celegans.db</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>584</th>\n",
       "      <td>bioconductor-celegansprobe</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>585</th>\n",
       "      <td>bioconductor-cellbaser</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>586</th>\n",
       "      <td>bioconductor-cellbench</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>587</th>\n",
       "      <td>bioconductor-cellhts2</td>\n",
       "      <td>301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>588</th>\n",
       "      <td>bioconductor-cellity</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>589</th>\n",
       "      <td>bioconductor-cellmapper</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>590</th>\n",
       "      <td>bioconductor-cellmapperdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>591</th>\n",
       "      <td>bioconductor-cellmixs</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>592</th>\n",
       "      <td>bioconductor-cellnoptr</td>\n",
       "      <td>646</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>593</th>\n",
       "      <td>bioconductor-cellscape</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>594</th>\n",
       "      <td>bioconductor-cellscore</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>bioconductor-celltrails</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>bioconductor-celltree</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>bioconductor-cemitool</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>bioconductor-ceu1kg</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>bioconductor-ceu1kgv</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>600</th>\n",
       "      <td>bioconductor-ceuhm3</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>601</th>\n",
       "      <td>bioconductor-cexor</td>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>602</th>\n",
       "      <td>bioconductor-cfassay</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>603</th>\n",
       "      <td>bioconductor-cgdv17</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>604</th>\n",
       "      <td>bioconductor-cgen</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>605</th>\n",
       "      <td>bioconductor-cghbase</td>\n",
       "      <td>422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>606</th>\n",
       "      <td>bioconductor-cghcall</td>\n",
       "      <td>511</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>607</th>\n",
       "      <td>bioconductor-cghmcr</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>608</th>\n",
       "      <td>bioconductor-cghnormaliter</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>609</th>\n",
       "      <td>bioconductor-cghregions</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>610</th>\n",
       "      <td>bioconductor-champ</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>611</th>\n",
       "      <td>bioconductor-champdata</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>612</th>\n",
       "      <td>bioconductor-charge</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>613</th>\n",
       "      <td>bioconductor-chemminedrugs</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>bioconductor-chemminer</td>\n",
       "      <td>573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>615</th>\n",
       "      <td>bioconductor-chetah</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>616</th>\n",
       "      <td>bioconductor-chic</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>617</th>\n",
       "      <td>bioconductor-chicago</td>\n",
       "      <td>389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>618</th>\n",
       "      <td>bioconductor-chic.data</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>619</th>\n",
       "      <td>bioconductor-chickencdf</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>620</th>\n",
       "      <td>bioconductor-chicken.db0</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>621</th>\n",
       "      <td>bioconductor-chicken.db</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>622</th>\n",
       "      <td>bioconductor-chickenprobe</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>623</th>\n",
       "      <td>bioconductor-chimera</td>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>624</th>\n",
       "      <td>bioconductor-chimeraviz</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>625</th>\n",
       "      <td>bioconductor-chimp.db0</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>626</th>\n",
       "      <td>bioconductor-chimphumanbraindata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>627</th>\n",
       "      <td>bioconductor-chipanalyser</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>628</th>\n",
       "      <td>bioconductor-chipcomp</td>\n",
       "      <td>652</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>629</th>\n",
       "      <td>bioconductor-chipenrich</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>630</th>\n",
       "      <td>bioconductor-chipenrich.data</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>631</th>\n",
       "      <td>bioconductor-chipexoqual</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>632</th>\n",
       "      <td>bioconductor-chipexoqualexample</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>633</th>\n",
       "      <td>bioconductor-chippeakanno</td>\n",
       "      <td>320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>634</th>\n",
       "      <td>bioconductor-chipqc</td>\n",
       "      <td>398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>635</th>\n",
       "      <td>bioconductor-chipseeker</td>\n",
       "      <td>632</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>636</th>\n",
       "      <td>bioconductor-chipseq</td>\n",
       "      <td>697</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>637</th>\n",
       "      <td>bioconductor-chipseqdbdata</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>638</th>\n",
       "      <td>bioconductor-chipseqr</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>639</th>\n",
       "      <td>bioconductor-chipseqspike</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>640</th>\n",
       "      <td>bioconductor-chipsim</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>641</th>\n",
       "      <td>bioconductor-chipxpress</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>642</th>\n",
       "      <td>bioconductor-chipxpressdata</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>643</th>\n",
       "      <td>bioconductor-chopsticks</td>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>644</th>\n",
       "      <td>bioconductor-chromdraw</td>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>645</th>\n",
       "      <td>bioconductor-chromheatmap</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>646</th>\n",
       "      <td>bioconductor-chromplot</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>647</th>\n",
       "      <td>bioconductor-chromstar</td>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>648</th>\n",
       "      <td>bioconductor-chromstardata</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>649</th>\n",
       "      <td>bioconductor-chromswitch</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>650</th>\n",
       "      <td>bioconductor-chromvar</td>\n",
       "      <td>506</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>651</th>\n",
       "      <td>bioconductor-chronos</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>652</th>\n",
       "      <td>bioconductor-cicero</td>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>653</th>\n",
       "      <td>bioconductor-cindex</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>bioconductor-circrnaprofiler</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>655</th>\n",
       "      <td>bioconductor-cispath</td>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>656</th>\n",
       "      <td>bioconductor-citruscdf</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>657</th>\n",
       "      <td>bioconductor-citrusprobe</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>658</th>\n",
       "      <td>bioconductor-clariomdhumanprobeset.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>659</th>\n",
       "      <td>bioconductor-clariomdhumantranscriptcluster.db</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>660</th>\n",
       "      <td>bioconductor-clariomshumanhttranscriptcluster.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>661</th>\n",
       "      <td>bioconductor-clariomshumantranscriptcluster.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>662</th>\n",
       "      <td>bioconductor-clariomsmousehttranscriptcluster.db</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>663</th>\n",
       "      <td>bioconductor-clariomsmousetranscriptcluster.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>664</th>\n",
       "      <td>bioconductor-clariomsrathttranscriptcluster.db</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>665</th>\n",
       "      <td>bioconductor-clariomsrattranscriptcluster.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>666</th>\n",
       "      <td>bioconductor-classifyr</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>667</th>\n",
       "      <td>bioconductor-cleanupdtseq</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>668</th>\n",
       "      <td>bioconductor-cleaver</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>669</th>\n",
       "      <td>bioconductor-clippda</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>670</th>\n",
       "      <td>bioconductor-clipper</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>671</th>\n",
       "      <td>bioconductor-cll</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>672</th>\n",
       "      <td>bioconductor-cllmethylation</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>673</th>\n",
       "      <td>bioconductor-clomial</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>674</th>\n",
       "      <td>bioconductor-clonality</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>675</th>\n",
       "      <td>bioconductor-clonotyper</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>676</th>\n",
       "      <td>bioconductor-clst</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>677</th>\n",
       "      <td>bioconductor-clstutils</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>678</th>\n",
       "      <td>bioconductor-clumsid</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>679</th>\n",
       "      <td>bioconductor-clumsiddata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>680</th>\n",
       "      <td>bioconductor-clustcomp</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>681</th>\n",
       "      <td>bioconductor-clusterexperiment</td>\n",
       "      <td>363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>682</th>\n",
       "      <td>bioconductor-clusterjudge</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>683</th>\n",
       "      <td>bioconductor-clusterprofiler</td>\n",
       "      <td>905</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>684</th>\n",
       "      <td>bioconductor-clusterseq</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>685</th>\n",
       "      <td>bioconductor-clustersignificance</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>686</th>\n",
       "      <td>bioconductor-clusterstab</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>687</th>\n",
       "      <td>bioconductor-cma</td>\n",
       "      <td>228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>688</th>\n",
       "      <td>bioconductor-cmap</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>689</th>\n",
       "      <td>bioconductor-cmap2data</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>690</th>\n",
       "      <td>bioconductor-cnanorm</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>691</th>\n",
       "      <td>bioconductor-cner</td>\n",
       "      <td>681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>692</th>\n",
       "      <td>bioconductor-cn.farms</td>\n",
       "      <td>472</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>693</th>\n",
       "      <td>bioconductor-cn.mops</td>\n",
       "      <td>862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>694</th>\n",
       "      <td>bioconductor-cnordt</td>\n",
       "      <td>369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>695</th>\n",
       "      <td>bioconductor-cnorfeeder</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>696</th>\n",
       "      <td>bioconductor-cnorfuzzy</td>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>697</th>\n",
       "      <td>bioconductor-cnorode</td>\n",
       "      <td>469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>698</th>\n",
       "      <td>bioconductor-cntools</td>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>699</th>\n",
       "      <td>bioconductor-cnvfilter</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>700</th>\n",
       "      <td>bioconductor-cnvgsa</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>701</th>\n",
       "      <td>bioconductor-cnvgsadata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>702</th>\n",
       "      <td>bioconductor-cnvpanelizer</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>703</th>\n",
       "      <td>bioconductor-cnvranger</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>704</th>\n",
       "      <td>bioconductor-cnvrd2</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>705</th>\n",
       "      <td>bioconductor-cnvtools</td>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>706</th>\n",
       "      <td>bioconductor-cobindr</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>707</th>\n",
       "      <td>bioconductor-cocitestats</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>708</th>\n",
       "      <td>bioconductor-cocoa</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>709</th>\n",
       "      <td>bioconductor-codelink</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>710</th>\n",
       "      <td>bioconductor-codex</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>711</th>\n",
       "      <td>bioconductor-coexnet</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>712</th>\n",
       "      <td>bioconductor-cogaps</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>713</th>\n",
       "      <td>bioconductor-cogena</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>714</th>\n",
       "      <td>bioconductor-cogps</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>715</th>\n",
       "      <td>bioconductor-cohcap</td>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>716</th>\n",
       "      <td>bioconductor-cohcapanno</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>717</th>\n",
       "      <td>bioconductor-cola</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>718</th>\n",
       "      <td>bioconductor-colonca</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>719</th>\n",
       "      <td>bioconductor-comet</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>720</th>\n",
       "      <td>bioconductor-compartmap</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>721</th>\n",
       "      <td>bioconductor-compass</td>\n",
       "      <td>674</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>722</th>\n",
       "      <td>bioconductor-compcoder</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>723</th>\n",
       "      <td>bioconductor-compepitools</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>724</th>\n",
       "      <td>bioconductor-compgo</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>725</th>\n",
       "      <td>bioconductor-complexheatmap</td>\n",
       "      <td>759</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>726</th>\n",
       "      <td>bioconductor-confess</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>727</th>\n",
       "      <td>bioconductor-confessdata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>728</th>\n",
       "      <td>bioconductor-connectivitymap</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>729</th>\n",
       "      <td>bioconductor-consensus</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>730</th>\n",
       "      <td>bioconductor-consensusclusterplus</td>\n",
       "      <td>488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>731</th>\n",
       "      <td>bioconductor-consensusde</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>732</th>\n",
       "      <td>bioconductor-consensusov</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>733</th>\n",
       "      <td>bioconductor-consensusseeker</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>734</th>\n",
       "      <td>bioconductor-contibait</td>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>735</th>\n",
       "      <td>bioconductor-conumee</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>736</th>\n",
       "      <td>bioconductor-convert</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>737</th>\n",
       "      <td>bioconductor-copa</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>738</th>\n",
       "      <td>bioconductor-copdsexualdimorphism.data</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>739</th>\n",
       "      <td>bioconductor-copyhelper</td>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>740</th>\n",
       "      <td>bioconductor-copyneutralima</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>741</th>\n",
       "      <td>bioconductor-copynumber</td>\n",
       "      <td>491</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>742</th>\n",
       "      <td>bioconductor-copynumberplots</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>743</th>\n",
       "      <td>bioconductor-copywriter</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>744</th>\n",
       "      <td>bioconductor-cordon</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>745</th>\n",
       "      <td>bioconductor-coregflux</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>746</th>\n",
       "      <td>bioconductor-coregnet</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>747</th>\n",
       "      <td>bioconductor-cormotif</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>748</th>\n",
       "      <td>bioconductor-cormut</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>749</th>\n",
       "      <td>bioconductor-correp</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>750</th>\n",
       "      <td>bioconductor-coseq</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>bioconductor-cosmic.67</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>752</th>\n",
       "      <td>bioconductor-cosmiq</td>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>753</th>\n",
       "      <td>bioconductor-cosnet</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>754</th>\n",
       "      <td>bioconductor-cottoncdf</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>755</th>\n",
       "      <td>bioconductor-cottonprobe</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>756</th>\n",
       "      <td>bioconductor-countclust</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>757</th>\n",
       "      <td>bioconductor-countsimqc</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>758</th>\n",
       "      <td>bioconductor-coveb</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>759</th>\n",
       "      <td>bioconductor-coverageview</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>760</th>\n",
       "      <td>bioconductor-covrna</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>761</th>\n",
       "      <td>bioconductor-cpvsnp</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>762</th>\n",
       "      <td>bioconductor-cqn</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>763</th>\n",
       "      <td>bioconductor-crcl18</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>764</th>\n",
       "      <td>bioconductor-crimage</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>765</th>\n",
       "      <td>bioconductor-crisprseek</td>\n",
       "      <td>240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>766</th>\n",
       "      <td>bioconductor-crisprseekplus</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>767</th>\n",
       "      <td>bioconductor-crisprvariants</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>768</th>\n",
       "      <td>bioconductor-crlmm</td>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>769</th>\n",
       "      <td>bioconductor-crossicc</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>770</th>\n",
       "      <td>bioconductor-crossmeta</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>771</th>\n",
       "      <td>bioconductor-csar</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>772</th>\n",
       "      <td>bioconductor-csaw</td>\n",
       "      <td>1092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>773</th>\n",
       "      <td>bioconductor-cssp</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>774</th>\n",
       "      <td>bioconductor-ctc</td>\n",
       "      <td>223</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>775</th>\n",
       "      <td>bioconductor-ctrap</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>776</th>\n",
       "      <td>bioconductor-ctsge</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>777</th>\n",
       "      <td>bioconductor-cummerbund</td>\n",
       "      <td>621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>778</th>\n",
       "      <td>bioconductor-curatedadipochip</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>779</th>\n",
       "      <td>bioconductor-curatedadiporna</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>780</th>\n",
       "      <td>bioconductor-curatedbladderdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>781</th>\n",
       "      <td>bioconductor-curatedbreastdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>782</th>\n",
       "      <td>bioconductor-curatedcrcdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>783</th>\n",
       "      <td>bioconductor-curatedmetagenomicdata</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>784</th>\n",
       "      <td>bioconductor-curatedovariandata</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>785</th>\n",
       "      <td>bioconductor-curatedtcgadata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>786</th>\n",
       "      <td>bioconductor-customprodb</td>\n",
       "      <td>119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>787</th>\n",
       "      <td>bioconductor-cycle</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>788</th>\n",
       "      <td>bioconductor-cydar</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>789</th>\n",
       "      <td>bioconductor-cyp450cdf</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>790</th>\n",
       "      <td>bioconductor-cytodx</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>791</th>\n",
       "      <td>bioconductor-cytofast</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>792</th>\n",
       "      <td>bioconductor-cytolib</td>\n",
       "      <td>239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>793</th>\n",
       "      <td>bioconductor-cytoml</td>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>794</th>\n",
       "      <td>bioconductor-dada2</td>\n",
       "      <td>3751</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>795</th>\n",
       "      <td>bioconductor-daglogo</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>796</th>\n",
       "      <td>bioconductor-dama</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>797</th>\n",
       "      <td>bioconductor-damirseq</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>798</th>\n",
       "      <td>bioconductor-dapar</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>799</th>\n",
       "      <td>bioconductor-dapardata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>800</th>\n",
       "      <td>bioconductor-dart</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>801</th>\n",
       "      <td>bioconductor-davidtiling</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>802</th>\n",
       "      <td>bioconductor-dbchip</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>803</th>\n",
       "      <td>bioconductor-dcanr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>804</th>\n",
       "      <td>bioconductor-dcgsa</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>805</th>\n",
       "      <td>bioconductor-dchiprep</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>806</th>\n",
       "      <td>bioconductor-ddct</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>807</th>\n",
       "      <td>bioconductor-ddpcrclust</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>808</th>\n",
       "      <td>bioconductor-debcam</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>809</th>\n",
       "      <td>bioconductor-debrowser</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>810</th>\n",
       "      <td>bioconductor-decipher</td>\n",
       "      <td>510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>811</th>\n",
       "      <td>bioconductor-deco</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>812</th>\n",
       "      <td>bioconductor-decomplexdisease</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>813</th>\n",
       "      <td>bioconductor-decomptumor2sig</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>814</th>\n",
       "      <td>bioconductor-deconrnaseq</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>815</th>\n",
       "      <td>bioconductor-decontam</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>816</th>\n",
       "      <td>bioconductor-deepbluer</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>817</th>\n",
       "      <td>bioconductor-deepsnv</td>\n",
       "      <td>652</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>818</th>\n",
       "      <td>bioconductor-deformats</td>\n",
       "      <td>259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>819</th>\n",
       "      <td>bioconductor-degraph</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>820</th>\n",
       "      <td>bioconductor-degreport</td>\n",
       "      <td>632</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>821</th>\n",
       "      <td>bioconductor-degseq</td>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>822</th>\n",
       "      <td>bioconductor-delayedarray</td>\n",
       "      <td>11131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>823</th>\n",
       "      <td>bioconductor-delayeddataframe</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>824</th>\n",
       "      <td>bioconductor-delayedmatrixstats</td>\n",
       "      <td>762</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>825</th>\n",
       "      <td>bioconductor-deltacapturec</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>826</th>\n",
       "      <td>bioconductor-deltagseg</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>827</th>\n",
       "      <td>bioconductor-demand</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>828</th>\n",
       "      <td>bioconductor-demixt</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>829</th>\n",
       "      <td>bioconductor-dep</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>830</th>\n",
       "      <td>bioconductor-depecher</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>831</th>\n",
       "      <td>bioconductor-depmap</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>832</th>\n",
       "      <td>bioconductor-deqms</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>833</th>\n",
       "      <td>bioconductor-derfinder</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>834</th>\n",
       "      <td>bioconductor-derfinderdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>835</th>\n",
       "      <td>bioconductor-derfinderhelper</td>\n",
       "      <td>378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>836</th>\n",
       "      <td>bioconductor-derfinderplot</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>837</th>\n",
       "      <td>bioconductor-descan2</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>838</th>\n",
       "      <td>bioconductor-deseq</td>\n",
       "      <td>1082</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>839</th>\n",
       "      <td>bioconductor-deseq2</td>\n",
       "      <td>3163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>840</th>\n",
       "      <td>bioconductor-desingle</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>841</th>\n",
       "      <td>bioconductor-desousa2013</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>842</th>\n",
       "      <td>bioconductor-destiny</td>\n",
       "      <td>307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>843</th>\n",
       "      <td>bioconductor-desubs</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>844</th>\n",
       "      <td>bioconductor-dewseq</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>845</th>\n",
       "      <td>bioconductor-dexseq</td>\n",
       "      <td>763</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>846</th>\n",
       "      <td>bioconductor-dexus</td>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>847</th>\n",
       "      <td>bioconductor-dfp</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>848</th>\n",
       "      <td>bioconductor-diffbind</td>\n",
       "      <td>1086</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>849</th>\n",
       "      <td>bioconductor-diffcoexp</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>850</th>\n",
       "      <td>bioconductor-diffcyt</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>851</th>\n",
       "      <td>bioconductor-diffgeneanalysis</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>852</th>\n",
       "      <td>bioconductor-diffhic</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>853</th>\n",
       "      <td>bioconductor-difflogo</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>854</th>\n",
       "      <td>bioconductor-diffloop</td>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>855</th>\n",
       "      <td>bioconductor-diffloopdata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>856</th>\n",
       "      <td>bioconductor-diffustats</td>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>857</th>\n",
       "      <td>bioconductor-diggit</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>858</th>\n",
       "      <td>bioconductor-diggitdata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>859</th>\n",
       "      <td>bioconductor-director</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>860</th>\n",
       "      <td>bioconductor-dirichletmultinomial</td>\n",
       "      <td>786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>861</th>\n",
       "      <td>bioconductor-discordant</td>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>862</th>\n",
       "      <td>bioconductor-discorhythm</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>863</th>\n",
       "      <td>bioconductor-divergence</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>864</th>\n",
       "      <td>bioconductor-dks</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>865</th>\n",
       "      <td>bioconductor-dlbcl</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>866</th>\n",
       "      <td>bioconductor-dmcfb</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>867</th>\n",
       "      <td>bioconductor-dmchmm</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>868</th>\n",
       "      <td>bioconductor-dmelsgi</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>869</th>\n",
       "      <td>bioconductor-dmrcaller</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>870</th>\n",
       "      <td>bioconductor-dmrcate</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>871</th>\n",
       "      <td>bioconductor-dmrcatedata</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>872</th>\n",
       "      <td>bioconductor-dmrforpairs</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>873</th>\n",
       "      <td>bioconductor-dmrscan</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>874</th>\n",
       "      <td>bioconductor-dmrseq</td>\n",
       "      <td>172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>875</th>\n",
       "      <td>bioconductor-dnabarcodecompatibility</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>876</th>\n",
       "      <td>bioconductor-dnabarcodes</td>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>877</th>\n",
       "      <td>bioconductor-dnacopy</td>\n",
       "      <td>1390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>878</th>\n",
       "      <td>bioconductor-dnashaper</td>\n",
       "      <td>563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>879</th>\n",
       "      <td>bioconductor-do.db</td>\n",
       "      <td>405</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>880</th>\n",
       "      <td>bioconductor-dominoeffect</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>881</th>\n",
       "      <td>bioconductor-donapllp2013</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>882</th>\n",
       "      <td>bioconductor-doppelgangr</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>883</th>\n",
       "      <td>bioconductor-doscheda</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>884</th>\n",
       "      <td>bioconductor-dose</td>\n",
       "      <td>1033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>885</th>\n",
       "      <td>bioconductor-doser</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>886</th>\n",
       "      <td>bioconductor-drawproteins</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>887</th>\n",
       "      <td>bioconductor-dream4</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>888</th>\n",
       "      <td>bioconductor-dresscheck</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>889</th>\n",
       "      <td>bioconductor-drimseq</td>\n",
       "      <td>245</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>890</th>\n",
       "      <td>bioconductor-drivernet</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>891</th>\n",
       "      <td>bioconductor-dropletutils</td>\n",
       "      <td>517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>892</th>\n",
       "      <td>bioconductor-drosgenome1cdf</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>893</th>\n",
       "      <td>bioconductor-drosgenome1.db</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>894</th>\n",
       "      <td>bioconductor-drosgenome1probe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>895</th>\n",
       "      <td>bioconductor-drosophila2cdf</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>896</th>\n",
       "      <td>bioconductor-drosophila2.db</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>897</th>\n",
       "      <td>bioconductor-drosophila2probe</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>898</th>\n",
       "      <td>bioconductor-drugvsdisease</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>899</th>\n",
       "      <td>bioconductor-drugvsdiseasedata</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>900</th>\n",
       "      <td>bioconductor-dsqtl</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>901</th>\n",
       "      <td>bioconductor-dss</td>\n",
       "      <td>485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>902</th>\n",
       "      <td>bioconductor-dta</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>903</th>\n",
       "      <td>bioconductor-dualks</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>904</th>\n",
       "      <td>bioconductor-duoclustering2018</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>905</th>\n",
       "      <td>bioconductor-dupchecker</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>906</th>\n",
       "      <td>bioconductor-dupradar</td>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>907</th>\n",
       "      <td>bioconductor-dvddata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>908</th>\n",
       "      <td>bioconductor-dyebias</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>909</th>\n",
       "      <td>bioconductor-dyebiasexamples</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>910</th>\n",
       "      <td>bioconductor-dyndoc</td>\n",
       "      <td>213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>911</th>\n",
       "      <td>bioconductor-easyqpcr</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>912</th>\n",
       "      <td>bioconductor-easyrnaseq</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>913</th>\n",
       "      <td>bioconductor-eatonetalchipseq</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>914</th>\n",
       "      <td>bioconductor-ebarrays</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>915</th>\n",
       "      <td>bioconductor-ebcoexpress</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>916</th>\n",
       "      <td>bioconductor-ebimage</td>\n",
       "      <td>862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>917</th>\n",
       "      <td>bioconductor-ebsea</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>918</th>\n",
       "      <td>bioconductor-ebseq</td>\n",
       "      <td>792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>919</th>\n",
       "      <td>bioconductor-ebseqhmm</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>920</th>\n",
       "      <td>bioconductor-ecoli2cdf</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>921</th>\n",
       "      <td>bioconductor-ecoli2.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>922</th>\n",
       "      <td>bioconductor-ecoli2probe</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>923</th>\n",
       "      <td>bioconductor-ecoliasv2cdf</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>924</th>\n",
       "      <td>bioconductor-ecoliasv2probe</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>925</th>\n",
       "      <td>bioconductor-ecolicdf</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>926</th>\n",
       "      <td>bioconductor-ecolik12.db0</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>927</th>\n",
       "      <td>bioconductor-ecolileucine</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>928</th>\n",
       "      <td>bioconductor-ecoliprobe</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>929</th>\n",
       "      <td>bioconductor-ecolisakai.db0</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>930</th>\n",
       "      <td>bioconductor-ecolitk</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>931</th>\n",
       "      <td>bioconductor-edaseq</td>\n",
       "      <td>422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>932</th>\n",
       "      <td>bioconductor-edda</td>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>933</th>\n",
       "      <td>bioconductor-edge</td>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>934</th>\n",
       "      <td>bioconductor-edger</td>\n",
       "      <td>3070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>935</th>\n",
       "      <td>bioconductor-eegc</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>936</th>\n",
       "      <td>bioconductor-egad</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>937</th>\n",
       "      <td>bioconductor-egsea</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>938</th>\n",
       "      <td>bioconductor-egseadata</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>939</th>\n",
       "      <td>bioconductor-eir</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>940</th>\n",
       "      <td>bioconductor-eisa</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>941</th>\n",
       "      <td>bioconductor-elbow</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>942</th>\n",
       "      <td>bioconductor-elmer</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>943</th>\n",
       "      <td>bioconductor-elmer.data</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>944</th>\n",
       "      <td>bioconductor-emdomics</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>945</th>\n",
       "      <td>bioconductor-empiricalbrownsmethod</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>946</th>\n",
       "      <td>bioconductor-encodexplorer</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>947</th>\n",
       "      <td>bioconductor-encodexplorerdata</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>948</th>\n",
       "      <td>bioconductor-enhancedvolcano</td>\n",
       "      <td>75</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>949</th>\n",
       "      <td>bioconductor-enmix</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>950</th>\n",
       "      <td>bioconductor-enrichedheatmap</td>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>951</th>\n",
       "      <td>bioconductor-enrichmentbrowser</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>952</th>\n",
       "      <td>bioconductor-enrichplot</td>\n",
       "      <td>367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>953</th>\n",
       "      <td>bioconductor-enrichtf</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>954</th>\n",
       "      <td>bioconductor-ensdb.hsapiens.v75</td>\n",
       "      <td>222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>955</th>\n",
       "      <td>bioconductor-ensdb.hsapiens.v79</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>956</th>\n",
       "      <td>bioconductor-ensdb.hsapiens.v86</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>957</th>\n",
       "      <td>bioconductor-ensdb.mmusculus.v75</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>958</th>\n",
       "      <td>bioconductor-ensdb.mmusculus.v79</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>959</th>\n",
       "      <td>bioconductor-ensdb.rnorvegicus.v75</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>960</th>\n",
       "      <td>bioconductor-ensdb.rnorvegicus.v79</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>961</th>\n",
       "      <td>bioconductor-ensembldb</td>\n",
       "      <td>1146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>962</th>\n",
       "      <td>bioconductor-ensemblvep</td>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>963</th>\n",
       "      <td>bioconductor-envisionquery</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>964</th>\n",
       "      <td>bioconductor-epidish</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>965</th>\n",
       "      <td>bioconductor-epigenomix</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>966</th>\n",
       "      <td>bioconductor-epihet</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>967</th>\n",
       "      <td>bioconductor-epivizr</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>968</th>\n",
       "      <td>bioconductor-epivizrchart</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>969</th>\n",
       "      <td>bioconductor-epivizrdata</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>970</th>\n",
       "      <td>bioconductor-epivizrserver</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>971</th>\n",
       "      <td>bioconductor-epivizrstandalone</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>972</th>\n",
       "      <td>bioconductor-erccdashboard</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>973</th>\n",
       "      <td>bioconductor-erma</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>974</th>\n",
       "      <td>bioconductor-erssa</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>975</th>\n",
       "      <td>bioconductor-esatac</td>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>976</th>\n",
       "      <td>bioconductor-esetvis</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>977</th>\n",
       "      <td>bioconductor-estrogen</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>978</th>\n",
       "      <td>bioconductor-etec16s</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>979</th>\n",
       "      <td>bioconductor-eudysbiome</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>980</th>\n",
       "      <td>bioconductor-eupathdb</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>981</th>\n",
       "      <td>bioconductor-evaluomer</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>982</th>\n",
       "      <td>bioconductor-eventpointer</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>983</th>\n",
       "      <td>bioconductor-excluster</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>984</th>\n",
       "      <td>bioconductor-eximir</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>985</th>\n",
       "      <td>bioconductor-exomecopy</td>\n",
       "      <td>671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>986</th>\n",
       "      <td>bioconductor-experimenthub</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>987</th>\n",
       "      <td>bioconductor-experimenthubdata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>988</th>\n",
       "      <td>bioconductor-explorase</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>989</th>\n",
       "      <td>bioconductor-expressionatlas</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>990</th>\n",
       "      <td>bioconductor-expressionview</td>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>991</th>\n",
       "      <td>bioconductor-faahko</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>992</th>\n",
       "      <td>bioconductor-fabia</td>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>993</th>\n",
       "      <td>bioconductor-fabiadata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>994</th>\n",
       "      <td>bioconductor-facsdorit</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>bioconductor-factdesign</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>bioconductor-famagg</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>bioconductor-fantom3and4cage</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>bioconductor-farms</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>bioconductor-fastliquidassociation</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1000</th>\n",
       "      <td>bioconductor-fastqcleaner</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1001</th>\n",
       "      <td>bioconductor-fastseg</td>\n",
       "      <td>685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1002</th>\n",
       "      <td>bioconductor-fcbf</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1003</th>\n",
       "      <td>bioconductor-fccac</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1004</th>\n",
       "      <td>bioconductor-fci</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1005</th>\n",
       "      <td>bioconductor-fcoex</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1006</th>\n",
       "      <td>bioconductor-fcscan</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1007</th>\n",
       "      <td>bioconductor-fdb.fantom4.promoters.hg19</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1008</th>\n",
       "      <td>bioconductor-fdb.infiniummethylation.hg18</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1009</th>\n",
       "      <td>bioconductor-fdb.infiniummethylation.hg19</td>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1010</th>\n",
       "      <td>bioconductor-fdb.ucsc.snp135common.hg19</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1011</th>\n",
       "      <td>bioconductor-fdb.ucsc.snp137common.hg19</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1012</th>\n",
       "      <td>bioconductor-fdb.ucsc.trnas</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1013</th>\n",
       "      <td>bioconductor-fdrame</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1014</th>\n",
       "      <td>bioconductor-fella</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1015</th>\n",
       "      <td>bioconductor-fem</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1016</th>\n",
       "      <td>bioconductor-ffpe</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1017</th>\n",
       "      <td>bioconductor-ffpeexampledata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1018</th>\n",
       "      <td>bioconductor-fgnet</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1019</th>\n",
       "      <td>bioconductor-fgsea</td>\n",
       "      <td>899</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1020</th>\n",
       "      <td>bioconductor-fibroeset</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1021</th>\n",
       "      <td>bioconductor-findmyfriends</td>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1022</th>\n",
       "      <td>bioconductor-fis</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1023</th>\n",
       "      <td>bioconductor-fishalyser</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1024</th>\n",
       "      <td>bioconductor-fishpond</td>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1025</th>\n",
       "      <td>bioconductor-fission</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1026</th>\n",
       "      <td>bioconductor-fitcons.ucsc.hg19</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1027</th>\n",
       "      <td>bioconductor-fithic</td>\n",
       "      <td>580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1028</th>\n",
       "      <td>bioconductor-flagme</td>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1029</th>\n",
       "      <td>bioconductor-fletcher2013a</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1030</th>\n",
       "      <td>bioconductor-fletcher2013b</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1031</th>\n",
       "      <td>bioconductor-flowai</td>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1032</th>\n",
       "      <td>bioconductor-flowbeads</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1033</th>\n",
       "      <td>bioconductor-flowbin</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1034</th>\n",
       "      <td>bioconductor-flowcatchr</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1035</th>\n",
       "      <td>bioconductor-flowchic</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1036</th>\n",
       "      <td>bioconductor-flowcl</td>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1037</th>\n",
       "      <td>bioconductor-flowclean</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1038</th>\n",
       "      <td>bioconductor-flowclust</td>\n",
       "      <td>301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1039</th>\n",
       "      <td>bioconductor-flowcore</td>\n",
       "      <td>879</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1040</th>\n",
       "      <td>bioconductor-flowcybar</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1041</th>\n",
       "      <td>bioconductor-flowdensity</td>\n",
       "      <td>383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1042</th>\n",
       "      <td>bioconductor-flowfit</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1043</th>\n",
       "      <td>bioconductor-flowfitexampledata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1044</th>\n",
       "      <td>bioconductor-flowfp</td>\n",
       "      <td>658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1045</th>\n",
       "      <td>bioconductor-flowmap</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1046</th>\n",
       "      <td>bioconductor-flowmatch</td>\n",
       "      <td>473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1047</th>\n",
       "      <td>bioconductor-flowmeans</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1048</th>\n",
       "      <td>bioconductor-flowmerge</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1049</th>\n",
       "      <td>bioconductor-flowpeaks</td>\n",
       "      <td>591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1050</th>\n",
       "      <td>bioconductor-flowploidy</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1051</th>\n",
       "      <td>bioconductor-flowploidydata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1052</th>\n",
       "      <td>bioconductor-flowplots</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1053</th>\n",
       "      <td>bioconductor-flowrepositoryr</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1054</th>\n",
       "      <td>bioconductor-flowsom</td>\n",
       "      <td>687</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1055</th>\n",
       "      <td>bioconductor-flowsorted.blood.450k</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1056</th>\n",
       "      <td>bioconductor-flowsorted.blood.epic</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1057</th>\n",
       "      <td>bioconductor-flowsorted.cordblood.450k</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1058</th>\n",
       "      <td>bioconductor-flowsorted.cordbloodcombined.450k</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1059</th>\n",
       "      <td>bioconductor-flowsorted.cordbloodnorway.450k</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1060</th>\n",
       "      <td>bioconductor-flowsorted.dlpfc.450k</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1061</th>\n",
       "      <td>bioconductor-flowspecs</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1062</th>\n",
       "      <td>bioconductor-flowspy</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1063</th>\n",
       "      <td>bioconductor-flowstats</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1064</th>\n",
       "      <td>bioconductor-flowtime</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1065</th>\n",
       "      <td>bioconductor-flowtrans</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1066</th>\n",
       "      <td>bioconductor-flowtype</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1067</th>\n",
       "      <td>bioconductor-flowutils</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1068</th>\n",
       "      <td>bioconductor-flowviz</td>\n",
       "      <td>411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1069</th>\n",
       "      <td>bioconductor-flowvs</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1070</th>\n",
       "      <td>bioconductor-flowworkspace</td>\n",
       "      <td>676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1071</th>\n",
       "      <td>bioconductor-flowworkspacedata</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1072</th>\n",
       "      <td>bioconductor-fly.db0</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1073</th>\n",
       "      <td>bioconductor-fmcsr</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1074</th>\n",
       "      <td>bioconductor-focalcall</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1075</th>\n",
       "      <td>bioconductor-foldgo</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1076</th>\n",
       "      <td>bioconductor-fourcseq</td>\n",
       "      <td>386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1077</th>\n",
       "      <td>bioconductor-frgepistasis</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1078</th>\n",
       "      <td>bioconductor-frma</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1079</th>\n",
       "      <td>bioconductor-frmaexampledata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1080</th>\n",
       "      <td>bioconductor-frmatools</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1081</th>\n",
       "      <td>bioconductor-funchip</td>\n",
       "      <td>472</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1082</th>\n",
       "      <td>bioconductor-funcisnp</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1083</th>\n",
       "      <td>bioconductor-funcisnp.data</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1084</th>\n",
       "      <td>bioconductor-funtoonorm</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1085</th>\n",
       "      <td>bioconductor-furrowseg</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1086</th>\n",
       "      <td>bioconductor-ga4ghclient</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1087</th>\n",
       "      <td>bioconductor-ga4ghshiny</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1088</th>\n",
       "      <td>bioconductor-gaga</td>\n",
       "      <td>662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1089</th>\n",
       "      <td>bioconductor-gage</td>\n",
       "      <td>401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1090</th>\n",
       "      <td>bioconductor-gagedata</td>\n",
       "      <td>489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1091</th>\n",
       "      <td>bioconductor-gaggle</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1092</th>\n",
       "      <td>bioconductor-gaia</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1093</th>\n",
       "      <td>bioconductor-gapgom</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1094</th>\n",
       "      <td>bioconductor-gaprediction</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1095</th>\n",
       "      <td>bioconductor-garfield</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1096</th>\n",
       "      <td>bioconductor-gars</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1097</th>\n",
       "      <td>bioconductor-gaschyhs</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1098</th>\n",
       "      <td>bioconductor-gatefinder</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1099</th>\n",
       "      <td>bioconductor-gatingmldata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1100</th>\n",
       "      <td>bioconductor-gcapc</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1101</th>\n",
       "      <td>bioconductor-gcatest</td>\n",
       "      <td>483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1102</th>\n",
       "      <td>bioconductor-gcmap</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1103</th>\n",
       "      <td>bioconductor-gcmapweb</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1104</th>\n",
       "      <td>bioconductor-gcrisprtools</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1105</th>\n",
       "      <td>bioconductor-gcrma</td>\n",
       "      <td>588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1106</th>\n",
       "      <td>bioconductor-gcspikelite</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1107</th>\n",
       "      <td>bioconductor-gcsscore</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1108</th>\n",
       "      <td>bioconductor-gdcrnatools</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1109</th>\n",
       "      <td>bioconductor-gdsarray</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1110</th>\n",
       "      <td>bioconductor-gdsfmt</td>\n",
       "      <td>596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1111</th>\n",
       "      <td>bioconductor-geecc</td>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1112</th>\n",
       "      <td>bioconductor-gem</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1113</th>\n",
       "      <td>bioconductor-gemini</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1114</th>\n",
       "      <td>bioconductor-genarise</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1115</th>\n",
       "      <td>bioconductor-genbankr</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1116</th>\n",
       "      <td>bioconductor-geneaccord</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1117</th>\n",
       "      <td>bioconductor-geneanswers</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1118</th>\n",
       "      <td>bioconductor-geneattribution</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1119</th>\n",
       "      <td>bioconductor-genebreak</td>\n",
       "      <td>378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1120</th>\n",
       "      <td>bioconductor-geneclassifiers</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1121</th>\n",
       "      <td>bioconductor-geneexpressionsignature</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1122</th>\n",
       "      <td>bioconductor-genefilter</td>\n",
       "      <td>3648</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1123</th>\n",
       "      <td>bioconductor-genefu</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1124</th>\n",
       "      <td>bioconductor-genega</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1125</th>\n",
       "      <td>bioconductor-genegeneinter</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1126</th>\n",
       "      <td>bioconductor-genelendatabase</td>\n",
       "      <td>666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1127</th>\n",
       "      <td>bioconductor-genemeta</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1128</th>\n",
       "      <td>bioconductor-genenetworkbuilder</td>\n",
       "      <td>406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1129</th>\n",
       "      <td>bioconductor-geneoverlap</td>\n",
       "      <td>212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1130</th>\n",
       "      <td>bioconductor-geneplast</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1131</th>\n",
       "      <td>bioconductor-geneplast.data.string.v91</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1132</th>\n",
       "      <td>bioconductor-geneplotter</td>\n",
       "      <td>2919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1133</th>\n",
       "      <td>bioconductor-generecommender</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1134</th>\n",
       "      <td>bioconductor-generegionscan</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1135</th>\n",
       "      <td>bioconductor-generxcluster</td>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1136</th>\n",
       "      <td>bioconductor-geneselectmmd</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1137</th>\n",
       "      <td>bioconductor-genesis</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1138</th>\n",
       "      <td>bioconductor-genestructuretools</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1139</th>\n",
       "      <td>bioconductor-genetclassifier</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1140</th>\n",
       "      <td>bioconductor-geneticsdesign</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1141</th>\n",
       "      <td>bioconductor-geneticsped</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1142</th>\n",
       "      <td>bioconductor-genextender</td>\n",
       "      <td>367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1143</th>\n",
       "      <td>bioconductor-genie3</td>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1144</th>\n",
       "      <td>bioconductor-genocn</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1145</th>\n",
       "      <td>bioconductor-genogam</td>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1146</th>\n",
       "      <td>bioconductor-genomation</td>\n",
       "      <td>860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1147</th>\n",
       "      <td>bioconductor-genomationdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1148</th>\n",
       "      <td>bioconductor-genomeinfodb</td>\n",
       "      <td>12210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1149</th>\n",
       "      <td>bioconductor-genomeinfodbdata</td>\n",
       "      <td>11516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1150</th>\n",
       "      <td>bioconductor-genomeintervals</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1151</th>\n",
       "      <td>bioconductor-genomes</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1152</th>\n",
       "      <td>bioconductor-genomewidesnp5crlmm</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1153</th>\n",
       "      <td>bioconductor-genomewidesnp6crlmm</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1154</th>\n",
       "      <td>bioconductor-genomicalignments</td>\n",
       "      <td>10675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1155</th>\n",
       "      <td>bioconductor-genomicdatacommons</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1156</th>\n",
       "      <td>bioconductor-genomicfeatures</td>\n",
       "      <td>2617</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1157</th>\n",
       "      <td>bioconductor-genomicfiles</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1158</th>\n",
       "      <td>bioconductor-genomicinteractions</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1159</th>\n",
       "      <td>bioconductor-genomicozone</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1160</th>\n",
       "      <td>bioconductor-genomicranges</td>\n",
       "      <td>12857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1161</th>\n",
       "      <td>bioconductor-genomicscores</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1162</th>\n",
       "      <td>bioconductor-genomictuples</td>\n",
       "      <td>553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1163</th>\n",
       "      <td>bioconductor-genoset</td>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1164</th>\n",
       "      <td>bioconductor-genotypeeval</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1165</th>\n",
       "      <td>bioconductor-genphen</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1166</th>\n",
       "      <td>bioconductor-genrank</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1167</th>\n",
       "      <td>bioconductor-genvisr</td>\n",
       "      <td>307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1168</th>\n",
       "      <td>bioconductor-geometadb</td>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1169</th>\n",
       "      <td>bioconductor-geoquery</td>\n",
       "      <td>739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1170</th>\n",
       "      <td>bioconductor-geosubmission</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1171</th>\n",
       "      <td>bioconductor-gep2pep</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1172</th>\n",
       "      <td>bioconductor-gesper</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1173</th>\n",
       "      <td>bioconductor-geuvadistranscriptexpr</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1174</th>\n",
       "      <td>bioconductor-geuvpack</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1175</th>\n",
       "      <td>bioconductor-geuvstore2</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1176</th>\n",
       "      <td>bioconductor-gewist</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1177</th>\n",
       "      <td>bioconductor-ggbase</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1178</th>\n",
       "      <td>bioconductor-ggbio</td>\n",
       "      <td>801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1179</th>\n",
       "      <td>bioconductor-ggcyto</td>\n",
       "      <td>410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1180</th>\n",
       "      <td>bioconductor-ggdata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1181</th>\n",
       "      <td>bioconductor-gghumanmethcancerpanelv1.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1182</th>\n",
       "      <td>bioconductor-ggtools</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1183</th>\n",
       "      <td>bioconductor-ggtree</td>\n",
       "      <td>603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1184</th>\n",
       "      <td>bioconductor-gigsea</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1185</th>\n",
       "      <td>bioconductor-gigseadata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1186</th>\n",
       "      <td>bioconductor-girafe</td>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1187</th>\n",
       "      <td>bioconductor-gispa</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1188</th>\n",
       "      <td>bioconductor-glad</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1189</th>\n",
       "      <td>bioconductor-gladiatox</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1190</th>\n",
       "      <td>bioconductor-glimma</td>\n",
       "      <td>239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1191</th>\n",
       "      <td>bioconductor-glmsparsenet</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1192</th>\n",
       "      <td>bioconductor-globalancova</td>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1193</th>\n",
       "      <td>bioconductor-globalseq</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1194</th>\n",
       "      <td>bioconductor-globaltest</td>\n",
       "      <td>303</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1195</th>\n",
       "      <td>bioconductor-gmapr</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1196</th>\n",
       "      <td>bioconductor-gmicr</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1197</th>\n",
       "      <td>bioconductor-gmrp</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>bioconductor-gnet2</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>bioconductor-go.db</td>\n",
       "      <td>1270</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>bioconductor-goexpress</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>bioconductor-gofuncr</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1202</th>\n",
       "      <td>bioconductor-gofunction</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1203</th>\n",
       "      <td>bioconductor-golubesets</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1204</th>\n",
       "      <td>bioconductor-gopro</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1205</th>\n",
       "      <td>bioconductor-goprofiles</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1206</th>\n",
       "      <td>bioconductor-gosemsim</td>\n",
       "      <td>1148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1207</th>\n",
       "      <td>bioconductor-goseq</td>\n",
       "      <td>672</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1208</th>\n",
       "      <td>bioconductor-gosim</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1209</th>\n",
       "      <td>bioconductor-gostag</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1210</th>\n",
       "      <td>bioconductor-gostats</td>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1211</th>\n",
       "      <td>bioconductor-gosummaries</td>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1212</th>\n",
       "      <td>bioconductor-gothic</td>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1213</th>\n",
       "      <td>bioconductor-gotools</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1214</th>\n",
       "      <td>bioconductor-gp53cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1215</th>\n",
       "      <td>bioconductor-gpart</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1216</th>\n",
       "      <td>bioconductor-gpls</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1217</th>\n",
       "      <td>bioconductor-gprege</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1218</th>\n",
       "      <td>bioconductor-gqtlbase</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1219</th>\n",
       "      <td>bioconductor-gqtlstats</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1220</th>\n",
       "      <td>bioconductor-gramm4r</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1221</th>\n",
       "      <td>bioconductor-graper</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1222</th>\n",
       "      <td>bioconductor-graph</td>\n",
       "      <td>1787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1223</th>\n",
       "      <td>bioconductor-graphalignment</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1224</th>\n",
       "      <td>bioconductor-graphat</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1225</th>\n",
       "      <td>bioconductor-graphite</td>\n",
       "      <td>469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1226</th>\n",
       "      <td>bioconductor-graphpac</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1227</th>\n",
       "      <td>bioconductor-grasp2db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1228</th>\n",
       "      <td>bioconductor-greengenes13.5mgdb</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1229</th>\n",
       "      <td>bioconductor-grenits</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1230</th>\n",
       "      <td>bioconductor-greylistchip</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1231</th>\n",
       "      <td>bioconductor-grmetrics</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1232</th>\n",
       "      <td>bioconductor-grndata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1233</th>\n",
       "      <td>bioconductor-grohmm</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1234</th>\n",
       "      <td>bioconductor-grridge</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1235</th>\n",
       "      <td>bioconductor-gsalightning</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1236</th>\n",
       "      <td>bioconductor-gsar</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1237</th>\n",
       "      <td>bioconductor-gsbenchmark</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1238</th>\n",
       "      <td>bioconductor-gsca</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1239</th>\n",
       "      <td>bioconductor-gscreend</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1240</th>\n",
       "      <td>bioconductor-gse62944</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1241</th>\n",
       "      <td>bioconductor-gseabase</td>\n",
       "      <td>845</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1242</th>\n",
       "      <td>bioconductor-gseabenchmarker</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1243</th>\n",
       "      <td>bioconductor-gsealm</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1244</th>\n",
       "      <td>bioconductor-gsean</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1245</th>\n",
       "      <td>bioconductor-gskb</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1246</th>\n",
       "      <td>bioconductor-gsreg</td>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1247</th>\n",
       "      <td>bioconductor-gsri</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1248</th>\n",
       "      <td>bioconductor-gsva</td>\n",
       "      <td>879</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1249</th>\n",
       "      <td>bioconductor-gsvadata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1250</th>\n",
       "      <td>bioconductor-gtrellis</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1251</th>\n",
       "      <td>bioconductor-guideseq</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1252</th>\n",
       "      <td>bioconductor-guitar</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1253</th>\n",
       "      <td>bioconductor-gviz</td>\n",
       "      <td>738</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1254</th>\n",
       "      <td>bioconductor-gwascat</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1255</th>\n",
       "      <td>bioconductor-gwasdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1256</th>\n",
       "      <td>bioconductor-gwastools</td>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1257</th>\n",
       "      <td>bioconductor-gwasurvivr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1258</th>\n",
       "      <td>bioconductor-h10kcod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1259</th>\n",
       "      <td>bioconductor-h20kcod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1260</th>\n",
       "      <td>bioconductor-h5vc</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1261</th>\n",
       "      <td>bioconductor-h5vcdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1262</th>\n",
       "      <td>bioconductor-hapfabia</td>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1263</th>\n",
       "      <td>bioconductor-hapmap100khind</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1264</th>\n",
       "      <td>bioconductor-hapmap100kxba</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1265</th>\n",
       "      <td>bioconductor-hapmap370k</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1266</th>\n",
       "      <td>bioconductor-hapmap500knsp</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1267</th>\n",
       "      <td>bioconductor-hapmap500ksty</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1268</th>\n",
       "      <td>bioconductor-hapmapsnp5</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1269</th>\n",
       "      <td>bioconductor-hapmapsnp6</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1270</th>\n",
       "      <td>bioconductor-harbchip</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1271</th>\n",
       "      <td>bioconductor-harman</td>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1272</th>\n",
       "      <td>bioconductor-harmandata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1273</th>\n",
       "      <td>bioconductor-harmonizedtcgadata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1274</th>\n",
       "      <td>bioconductor-harshlight</td>\n",
       "      <td>467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1275</th>\n",
       "      <td>bioconductor-hcabrowser</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1276</th>\n",
       "      <td>bioconductor-hcadata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1277</th>\n",
       "      <td>bioconductor-hcaexplorer</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1278</th>\n",
       "      <td>bioconductor-hcg110cdf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1279</th>\n",
       "      <td>bioconductor-hcg110.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1280</th>\n",
       "      <td>bioconductor-hcg110probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1281</th>\n",
       "      <td>bioconductor-hd2013sgi</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1282</th>\n",
       "      <td>bioconductor-hdcytodata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1283</th>\n",
       "      <td>bioconductor-hdf5array</td>\n",
       "      <td>1169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1284</th>\n",
       "      <td>bioconductor-hdtd</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1285</th>\n",
       "      <td>bioconductor-healthyflowdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1286</th>\n",
       "      <td>bioconductor-heatmaps</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1287</th>\n",
       "      <td>bioconductor-heatplus</td>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1288</th>\n",
       "      <td>bioconductor-heebodata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1289</th>\n",
       "      <td>bioconductor-helloranges</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1290</th>\n",
       "      <td>bioconductor-hellorangesdata</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1291</th>\n",
       "      <td>bioconductor-help</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1292</th>\n",
       "      <td>bioconductor-hem</td>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1293</th>\n",
       "      <td>bioconductor-hgfocuscdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1294</th>\n",
       "      <td>bioconductor-hgfocus.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1295</th>\n",
       "      <td>bioconductor-hgfocusprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1296</th>\n",
       "      <td>bioconductor-hgu133a2cdf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1297</th>\n",
       "      <td>bioconductor-hgu133a2.db</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1298</th>\n",
       "      <td>bioconductor-hgu133a2frmavecs</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1299</th>\n",
       "      <td>bioconductor-hgu133a2probe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1300</th>\n",
       "      <td>bioconductor-hgu133abarcodevecs</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1301</th>\n",
       "      <td>bioconductor-hgu133acdf</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1302</th>\n",
       "      <td>bioconductor-hgu133a.db</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1303</th>\n",
       "      <td>bioconductor-hgu133afrmavecs</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1304</th>\n",
       "      <td>bioconductor-hgu133aprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1305</th>\n",
       "      <td>bioconductor-hgu133atagcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1306</th>\n",
       "      <td>bioconductor-hgu133atagprobe</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1307</th>\n",
       "      <td>bioconductor-hgu133bcdf</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1308</th>\n",
       "      <td>bioconductor-hgu133b.db</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1309</th>\n",
       "      <td>bioconductor-hgu133bprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1310</th>\n",
       "      <td>bioconductor-hgu133plus2barcodevecs</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1311</th>\n",
       "      <td>bioconductor-hgu133plus2cdf</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1312</th>\n",
       "      <td>bioconductor-hgu133plus2cellscore</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1313</th>\n",
       "      <td>bioconductor-hgu133plus2.db</td>\n",
       "      <td>130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1314</th>\n",
       "      <td>bioconductor-hgu133plus2frmavecs</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1315</th>\n",
       "      <td>bioconductor-hgu133plus2probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1316</th>\n",
       "      <td>bioconductor-hgu219cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1317</th>\n",
       "      <td>bioconductor-hgu219.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1318</th>\n",
       "      <td>bioconductor-hgu219probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1319</th>\n",
       "      <td>bioconductor-hgu2beta7</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1320</th>\n",
       "      <td>bioconductor-hgu95acdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1321</th>\n",
       "      <td>bioconductor-hgu95a.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1322</th>\n",
       "      <td>bioconductor-hgu95aprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1323</th>\n",
       "      <td>bioconductor-hgu95av2</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1324</th>\n",
       "      <td>bioconductor-hgu95av2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1325</th>\n",
       "      <td>bioconductor-hgu95av2.db</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1326</th>\n",
       "      <td>bioconductor-hgu95av2probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1327</th>\n",
       "      <td>bioconductor-hgu95bcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1328</th>\n",
       "      <td>bioconductor-hgu95b.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1329</th>\n",
       "      <td>bioconductor-hgu95bprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1330</th>\n",
       "      <td>bioconductor-hgu95ccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1331</th>\n",
       "      <td>bioconductor-hgu95c.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1332</th>\n",
       "      <td>bioconductor-hgu95cprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1333</th>\n",
       "      <td>bioconductor-hgu95dcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1334</th>\n",
       "      <td>bioconductor-hgu95d.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1335</th>\n",
       "      <td>bioconductor-hgu95dprobe</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1336</th>\n",
       "      <td>bioconductor-hgu95ecdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1337</th>\n",
       "      <td>bioconductor-hgu95e.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1338</th>\n",
       "      <td>bioconductor-hgu95eprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1339</th>\n",
       "      <td>bioconductor-hguatlas13k.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1340</th>\n",
       "      <td>bioconductor-hgubeta7.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1341</th>\n",
       "      <td>bioconductor-hgudkfz31.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1342</th>\n",
       "      <td>bioconductor-hgug4100a.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1343</th>\n",
       "      <td>bioconductor-hgug4101a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1344</th>\n",
       "      <td>bioconductor-hgug4110b.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1345</th>\n",
       "      <td>bioconductor-hgug4111a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1346</th>\n",
       "      <td>bioconductor-hgug4112a.db</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1347</th>\n",
       "      <td>bioconductor-hgug4845a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1348</th>\n",
       "      <td>bioconductor-hguqiagenv3.db</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1349</th>\n",
       "      <td>bioconductor-hi16cod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1350</th>\n",
       "      <td>bioconductor-hiannotator</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1351</th>\n",
       "      <td>bioconductor-hibag</td>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1352</th>\n",
       "      <td>bioconductor-hicbricks</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1353</th>\n",
       "      <td>bioconductor-hiccompare</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1354</th>\n",
       "      <td>bioconductor-hicdatahumanimr90</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1355</th>\n",
       "      <td>bioconductor-hicdatalymphoblast</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1356</th>\n",
       "      <td>bioconductor-hicrep</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1357</th>\n",
       "      <td>bioconductor-hiergwas</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1358</th>\n",
       "      <td>bioconductor-hierinf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1359</th>\n",
       "      <td>bioconductor-hiiragi2013</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1360</th>\n",
       "      <td>bioconductor-hilbertcurve</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1361</th>\n",
       "      <td>bioconductor-hilbertvis</td>\n",
       "      <td>678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1362</th>\n",
       "      <td>bioconductor-hilda</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1363</th>\n",
       "      <td>bioconductor-hipathia</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1364</th>\n",
       "      <td>bioconductor-hireadsprocessor</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1365</th>\n",
       "      <td>bioconductor-hireewas</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1366</th>\n",
       "      <td>bioconductor-hitc</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1367</th>\n",
       "      <td>bioconductor-hivcdnavantwout03</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1368</th>\n",
       "      <td>bioconductor-hivprtplus2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1369</th>\n",
       "      <td>bioconductor-hmdbquery</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1370</th>\n",
       "      <td>bioconductor-hmmcopy</td>\n",
       "      <td>585</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1371</th>\n",
       "      <td>bioconductor-hmp16sdata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1372</th>\n",
       "      <td>bioconductor-hmp2data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1373</th>\n",
       "      <td>bioconductor-hmyrib36</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1374</th>\n",
       "      <td>bioconductor-hom.at.inp.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1375</th>\n",
       "      <td>bioconductor-hom.ce.inp.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1376</th>\n",
       "      <td>bioconductor-hom.dm.inp.db</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1377</th>\n",
       "      <td>bioconductor-hom.dr.inp.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1378</th>\n",
       "      <td>bioconductor-hom.hs.inp.db</td>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1379</th>\n",
       "      <td>bioconductor-hom.mm.inp.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1380</th>\n",
       "      <td>bioconductor-homo.sapiens</td>\n",
       "      <td>141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1381</th>\n",
       "      <td>bioconductor-hom.rn.inp.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1382</th>\n",
       "      <td>bioconductor-hom.sc.inp.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1383</th>\n",
       "      <td>bioconductor-hopach</td>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1384</th>\n",
       "      <td>bioconductor-hpaanalyze</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1385</th>\n",
       "      <td>bioconductor-hpannot</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1386</th>\n",
       "      <td>bioconductor-hpar</td>\n",
       "      <td>497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1387</th>\n",
       "      <td>bioconductor-hs25kresogen.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1388</th>\n",
       "      <td>bioconductor-hs6ug171.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1389</th>\n",
       "      <td>bioconductor-hsagilentdesign026652.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1390</th>\n",
       "      <td>bioconductor-hsmmsinglecell</td>\n",
       "      <td>594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1391</th>\n",
       "      <td>bioconductor-hspec</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1392</th>\n",
       "      <td>bioconductor-hspeccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1393</th>\n",
       "      <td>bioconductor-hta20probeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1394</th>\n",
       "      <td>bioconductor-hta20transcriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1395</th>\n",
       "      <td>bioconductor-hthgu133acdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1396</th>\n",
       "      <td>bioconductor-hthgu133a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1397</th>\n",
       "      <td>bioconductor-hthgu133afrmavecs</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1398</th>\n",
       "      <td>bioconductor-hthgu133aprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1399</th>\n",
       "      <td>bioconductor-hthgu133bcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1400</th>\n",
       "      <td>bioconductor-hthgu133b.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1401</th>\n",
       "      <td>bioconductor-hthgu133bprobe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1402</th>\n",
       "      <td>bioconductor-hthgu133pluspmcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1403</th>\n",
       "      <td>bioconductor-hthgu133pluspmprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1404</th>\n",
       "      <td>bioconductor-htmg430acdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1405</th>\n",
       "      <td>bioconductor-htmg430aprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1406</th>\n",
       "      <td>bioconductor-htmg430bcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1407</th>\n",
       "      <td>bioconductor-htmg430bprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1408</th>\n",
       "      <td>bioconductor-htmg430pmcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1409</th>\n",
       "      <td>bioconductor-htmg430pmprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1410</th>\n",
       "      <td>bioconductor-htqpcr</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1411</th>\n",
       "      <td>bioconductor-htrat230pmcdf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1412</th>\n",
       "      <td>bioconductor-htrat230pmprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1413</th>\n",
       "      <td>bioconductor-htratfocuscdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1414</th>\n",
       "      <td>bioconductor-htratfocusprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1415</th>\n",
       "      <td>bioconductor-htseqgenie</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1416</th>\n",
       "      <td>bioconductor-htsfilter</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1417</th>\n",
       "      <td>bioconductor-hu35ksubacdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1418</th>\n",
       "      <td>bioconductor-hu35ksuba.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1419</th>\n",
       "      <td>bioconductor-hu35ksubaprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1420</th>\n",
       "      <td>bioconductor-hu35ksubbcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1421</th>\n",
       "      <td>bioconductor-hu35ksubb.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1422</th>\n",
       "      <td>bioconductor-hu35ksubbprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1423</th>\n",
       "      <td>bioconductor-hu35ksubccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1424</th>\n",
       "      <td>bioconductor-hu35ksubc.db</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1425</th>\n",
       "      <td>bioconductor-hu35ksubcprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1426</th>\n",
       "      <td>bioconductor-hu35ksubdcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1427</th>\n",
       "      <td>bioconductor-hu35ksubd.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1428</th>\n",
       "      <td>bioconductor-hu35ksubdprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1429</th>\n",
       "      <td>bioconductor-hu6800cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1430</th>\n",
       "      <td>bioconductor-hu6800.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1431</th>\n",
       "      <td>bioconductor-hu6800probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1432</th>\n",
       "      <td>bioconductor-hu6800subacdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1433</th>\n",
       "      <td>bioconductor-hu6800subbcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1434</th>\n",
       "      <td>bioconductor-hu6800subccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1435</th>\n",
       "      <td>bioconductor-hu6800subdcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1436</th>\n",
       "      <td>bioconductor-huex10stprobeset.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1437</th>\n",
       "      <td>bioconductor-huex10sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1438</th>\n",
       "      <td>bioconductor-huex.1.0.st.v2frmavecs</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1439</th>\n",
       "      <td>bioconductor-huexexonprobesetlocation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1440</th>\n",
       "      <td>bioconductor-huexexonprobesetlocationhg18</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1441</th>\n",
       "      <td>bioconductor-huexexonprobesetlocationhg19</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1442</th>\n",
       "      <td>bioconductor-hugene10stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1443</th>\n",
       "      <td>bioconductor-hugene10sttranscriptcluster.db</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1444</th>\n",
       "      <td>bioconductor-hugene10stv1cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1445</th>\n",
       "      <td>bioconductor-hugene.1.0.st.v1frmavecs</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1446</th>\n",
       "      <td>bioconductor-hugene10stv1probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1447</th>\n",
       "      <td>bioconductor-hugene11stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1448</th>\n",
       "      <td>bioconductor-hugene11sttranscriptcluster.db</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1449</th>\n",
       "      <td>bioconductor-hugene20stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1450</th>\n",
       "      <td>bioconductor-hugene20sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1451</th>\n",
       "      <td>bioconductor-hugene21stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1452</th>\n",
       "      <td>bioconductor-hugene21sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1453</th>\n",
       "      <td>bioconductor-human1mduov3bcrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1454</th>\n",
       "      <td>bioconductor-human1mv1ccrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1455</th>\n",
       "      <td>bioconductor-human370quadv3ccrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1456</th>\n",
       "      <td>bioconductor-human370v1ccrlmm</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1457</th>\n",
       "      <td>bioconductor-human550v3bcrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1458</th>\n",
       "      <td>bioconductor-human610quadv1bcrlmm</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1459</th>\n",
       "      <td>bioconductor-human650v3acrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1460</th>\n",
       "      <td>bioconductor-human660quadv1acrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1461</th>\n",
       "      <td>bioconductor-humanaffydata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1462</th>\n",
       "      <td>bioconductor-humanchrloc</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1463</th>\n",
       "      <td>bioconductor-humancytosnp12v2p1hcrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1464</th>\n",
       "      <td>bioconductor-human.db0</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1465</th>\n",
       "      <td>bioconductor-humanomni1quadv1bcrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1466</th>\n",
       "      <td>bioconductor-humanomni25quadv1bcrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1467</th>\n",
       "      <td>bioconductor-humanomni5quadv1bcrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1468</th>\n",
       "      <td>bioconductor-humanomniexpress12v1bcrlmm</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1469</th>\n",
       "      <td>bioconductor-humanstemcell</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1470</th>\n",
       "      <td>bioconductor-humantranscriptomecompendium</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1471</th>\n",
       "      <td>bioconductor-huo22.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1472</th>\n",
       "      <td>bioconductor-hwgcod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1473</th>\n",
       "      <td>bioconductor-hybridmtest</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1474</th>\n",
       "      <td>bioconductor-hyper</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1475</th>\n",
       "      <td>bioconductor-hyperdraw</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1476</th>\n",
       "      <td>bioconductor-hypergraph</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1477</th>\n",
       "      <td>bioconductor-iaseq</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1478</th>\n",
       "      <td>bioconductor-iasva</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1479</th>\n",
       "      <td>bioconductor-ibbig</td>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1480</th>\n",
       "      <td>bioconductor-ibh</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1481</th>\n",
       "      <td>bioconductor-ibmq</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1482</th>\n",
       "      <td>bioconductor-icare</td>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1483</th>\n",
       "      <td>bioconductor-icens</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1484</th>\n",
       "      <td>bioconductor-icetea</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1485</th>\n",
       "      <td>bioconductor-icheck</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1486</th>\n",
       "      <td>bioconductor-ichip</td>\n",
       "      <td>481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1487</th>\n",
       "      <td>bioconductor-iclusterplus</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1488</th>\n",
       "      <td>bioconductor-icnv</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1489</th>\n",
       "      <td>bioconductor-icobra</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1490</th>\n",
       "      <td>bioconductor-ideal</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1491</th>\n",
       "      <td>bioconductor-ideoviz</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1492</th>\n",
       "      <td>bioconductor-idiogram</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1493</th>\n",
       "      <td>bioconductor-idr2d</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1494</th>\n",
       "      <td>bioconductor-igc</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1495</th>\n",
       "      <td>bioconductor-iggeneusage</td>\n",
       "      <td>91</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1496</th>\n",
       "      <td>bioconductor-igvr</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1497</th>\n",
       "      <td>bioconductor-ihw</td>\n",
       "      <td>444</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1498</th>\n",
       "      <td>bioconductor-ihwpaper</td>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1499</th>\n",
       "      <td>bioconductor-illumina450probevariants.db</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1500</th>\n",
       "      <td>bioconductor-illuminadatatestfiles</td>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1501</th>\n",
       "      <td>bioconductor-illuminahumanmethylation27kanno.i...</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1502</th>\n",
       "      <td>bioconductor-illuminahumanmethylation27k.db</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1503</th>\n",
       "      <td>bioconductor-illuminahumanmethylation27kmanifest</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1504</th>\n",
       "      <td>bioconductor-illuminahumanmethylation450kanno....</td>\n",
       "      <td>164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1505</th>\n",
       "      <td>bioconductor-illuminahumanmethylation450kmanifest</td>\n",
       "      <td>140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1506</th>\n",
       "      <td>bioconductor-illuminahumanmethylation450kprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1507</th>\n",
       "      <td>bioconductor-illuminahumanmethylationepicanno....</td>\n",
       "      <td>111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1508</th>\n",
       "      <td>bioconductor-illuminahumanmethylationepicanno....</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1509</th>\n",
       "      <td>bioconductor-illuminahumanmethylationepicanno....</td>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1510</th>\n",
       "      <td>bioconductor-illuminahumanmethylationepicmanifest</td>\n",
       "      <td>123</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1511</th>\n",
       "      <td>bioconductor-illuminahumanv1.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1512</th>\n",
       "      <td>bioconductor-illuminahumanv2beadid.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1513</th>\n",
       "      <td>bioconductor-illuminahumanv2.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1514</th>\n",
       "      <td>bioconductor-illuminahumanv3.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1515</th>\n",
       "      <td>bioconductor-illuminahumanv4.db</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1516</th>\n",
       "      <td>bioconductor-illuminahumanwgdaslv3.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1517</th>\n",
       "      <td>bioconductor-illuminahumanwgdaslv4.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1518</th>\n",
       "      <td>bioconductor-illuminaio</td>\n",
       "      <td>848</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1519</th>\n",
       "      <td>bioconductor-illuminamousev1.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1520</th>\n",
       "      <td>bioconductor-illuminamousev1p1.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1521</th>\n",
       "      <td>bioconductor-illuminamousev2.db</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1522</th>\n",
       "      <td>bioconductor-illuminaratv1.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1523</th>\n",
       "      <td>bioconductor-imagehts</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1524</th>\n",
       "      <td>bioconductor-imas</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1525</th>\n",
       "      <td>bioconductor-imetagene</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1526</th>\n",
       "      <td>bioconductor-imman</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1527</th>\n",
       "      <td>bioconductor-immunespacer</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1528</th>\n",
       "      <td>bioconductor-immunoclust</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1529</th>\n",
       "      <td>bioconductor-impcdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1530</th>\n",
       "      <td>bioconductor-impulsede</td>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1531</th>\n",
       "      <td>bioconductor-impulsede2</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1532</th>\n",
       "      <td>bioconductor-impute</td>\n",
       "      <td>1192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1533</th>\n",
       "      <td>bioconductor-indac.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1534</th>\n",
       "      <td>bioconductor-indeed</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1535</th>\n",
       "      <td>bioconductor-infercnv</td>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1536</th>\n",
       "      <td>bioconductor-inpas</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1537</th>\n",
       "      <td>bioconductor-inpower</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1538</th>\n",
       "      <td>bioconductor-inspect</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1539</th>\n",
       "      <td>bioconductor-intad</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1540</th>\n",
       "      <td>bioconductor-intansv</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1541</th>\n",
       "      <td>bioconductor-interactionset</td>\n",
       "      <td>561</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1542</th>\n",
       "      <td>bioconductor-interactivedisplay</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1543</th>\n",
       "      <td>bioconductor-interactivedisplaybase</td>\n",
       "      <td>795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1544</th>\n",
       "      <td>bioconductor-interest</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1545</th>\n",
       "      <td>bioconductor-interminer</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1546</th>\n",
       "      <td>bioconductor-intramirexplorer</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1547</th>\n",
       "      <td>bioconductor-inversion</td>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1548</th>\n",
       "      <td>bioconductor-ioniser</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1549</th>\n",
       "      <td>bioconductor-ipac</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1550</th>\n",
       "      <td>bioconductor-ipddb</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1551</th>\n",
       "      <td>bioconductor-ipo</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1552</th>\n",
       "      <td>bioconductor-ippd</td>\n",
       "      <td>467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1553</th>\n",
       "      <td>bioconductor-iranges</td>\n",
       "      <td>13965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1554</th>\n",
       "      <td>bioconductor-isee</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1555</th>\n",
       "      <td>bioconductor-iseq</td>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1556</th>\n",
       "      <td>bioconductor-isobar</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1557</th>\n",
       "      <td>bioconductor-isocorrector</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1558</th>\n",
       "      <td>bioconductor-isocorrectorgui</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1559</th>\n",
       "      <td>bioconductor-isoformswitchanalyzer</td>\n",
       "      <td>481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1560</th>\n",
       "      <td>bioconductor-isogenegui</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1561</th>\n",
       "      <td>bioconductor-isolde</td>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1562</th>\n",
       "      <td>bioconductor-isomirs</td>\n",
       "      <td>476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1563</th>\n",
       "      <td>bioconductor-italics</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1564</th>\n",
       "      <td>bioconductor-italicsdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1565</th>\n",
       "      <td>bioconductor-iterativebma</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1566</th>\n",
       "      <td>bioconductor-iterativebmasurv</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1567</th>\n",
       "      <td>bioconductor-iterclust</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1568</th>\n",
       "      <td>bioconductor-iteremoval</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1569</th>\n",
       "      <td>bioconductor-ivas</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1570</th>\n",
       "      <td>bioconductor-ivygapse</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1571</th>\n",
       "      <td>bioconductor-iwtomics</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1572</th>\n",
       "      <td>bioconductor-iyer517</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1573</th>\n",
       "      <td>bioconductor-jaspar2014</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1574</th>\n",
       "      <td>bioconductor-jaspar2016</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1575</th>\n",
       "      <td>bioconductor-jaspar2018</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1576</th>\n",
       "      <td>bioconductor-jazaerimetadata.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1577</th>\n",
       "      <td>bioconductor-jctseqdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1578</th>\n",
       "      <td>bioconductor-joda</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1579</th>\n",
       "      <td>bioconductor-junctionseq</td>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1580</th>\n",
       "      <td>bioconductor-karyoploter</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1581</th>\n",
       "      <td>bioconductor-kcsmart</td>\n",
       "      <td>377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1582</th>\n",
       "      <td>bioconductor-kebabs</td>\n",
       "      <td>483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1583</th>\n",
       "      <td>bioconductor-keggandmetacoredzpathwaysgeo</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1584</th>\n",
       "      <td>bioconductor-kegg.db</td>\n",
       "      <td>231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1585</th>\n",
       "      <td>bioconductor-keggdzpathwaysgeo</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1586</th>\n",
       "      <td>bioconductor-kegggraph</td>\n",
       "      <td>634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1587</th>\n",
       "      <td>bioconductor-kegglincs</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1588</th>\n",
       "      <td>bioconductor-keggorthology</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1589</th>\n",
       "      <td>bioconductor-keggprofile</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1590</th>\n",
       "      <td>bioconductor-keggrest</td>\n",
       "      <td>771</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1591</th>\n",
       "      <td>bioconductor-kidpack</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1592</th>\n",
       "      <td>bioconductor-kimod</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1593</th>\n",
       "      <td>bioconductor-kinswingr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1594</th>\n",
       "      <td>bioconductor-kissde</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1595</th>\n",
       "      <td>bioconductor-knowseq</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1596</th>\n",
       "      <td>bioconductor-kodata</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1597</th>\n",
       "      <td>bioconductor-lapmix</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1598</th>\n",
       "      <td>bioconductor-lapointe.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1599</th>\n",
       "      <td>bioconductor-lbe</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1600</th>\n",
       "      <td>bioconductor-ldblock</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1601</th>\n",
       "      <td>bioconductor-lea</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1602</th>\n",
       "      <td>bioconductor-ledpred</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1603</th>\n",
       "      <td>bioconductor-leebamviews</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1604</th>\n",
       "      <td>bioconductor-les</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1605</th>\n",
       "      <td>bioconductor-leukemiaseset</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1606</th>\n",
       "      <td>bioconductor-levi</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1607</th>\n",
       "      <td>bioconductor-lfa</td>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1608</th>\n",
       "      <td>bioconductor-liebermanaidenhic2009</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1609</th>\n",
       "      <td>bioconductor-limma</td>\n",
       "      <td>4835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1610</th>\n",
       "      <td>bioconductor-limmagui</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1611</th>\n",
       "      <td>bioconductor-linc</td>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1612</th>\n",
       "      <td>bioconductor-lineagepulse</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1613</th>\n",
       "      <td>bioconductor-linkhd</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1614</th>\n",
       "      <td>bioconductor-linnorm</td>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1615</th>\n",
       "      <td>bioconductor-lionessr</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1616</th>\n",
       "      <td>bioconductor-lipidr</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1617</th>\n",
       "      <td>bioconductor-liquidassociation</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1618</th>\n",
       "      <td>bioconductor-listeretalbsseq</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1619</th>\n",
       "      <td>bioconductor-lmdme</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1620</th>\n",
       "      <td>bioconductor-lmgene</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1621</th>\n",
       "      <td>bioconductor-lobstahs</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1622</th>\n",
       "      <td>bioconductor-loci2path</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1623</th>\n",
       "      <td>bioconductor-logicfs</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1624</th>\n",
       "      <td>bioconductor-logitt</td>\n",
       "      <td>460</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1625</th>\n",
       "      <td>bioconductor-logolas</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1626</th>\n",
       "      <td>bioconductor-lola</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1627</th>\n",
       "      <td>bioconductor-loomexperiment</td>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1628</th>\n",
       "      <td>bioconductor-lowmaca</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1629</th>\n",
       "      <td>bioconductor-lowmacaannotation</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1630</th>\n",
       "      <td>bioconductor-lpe</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1631</th>\n",
       "      <td>bioconductor-lpeadj</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1632</th>\n",
       "      <td>bioconductor-lpnet</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1633</th>\n",
       "      <td>bioconductor-lpsymphony</td>\n",
       "      <td>725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1634</th>\n",
       "      <td>bioconductor-lrbase.ath.eg.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1635</th>\n",
       "      <td>bioconductor-lrbase.bta.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1636</th>\n",
       "      <td>bioconductor-lrbase.cel.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1637</th>\n",
       "      <td>bioconductor-lrbasedbi</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1638</th>\n",
       "      <td>bioconductor-lrbase.dme.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1639</th>\n",
       "      <td>bioconductor-lrbase.dre.eg.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1640</th>\n",
       "      <td>bioconductor-lrbase.gga.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1641</th>\n",
       "      <td>bioconductor-lrbase.hsa.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1642</th>\n",
       "      <td>bioconductor-lrbase.mmu.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1643</th>\n",
       "      <td>bioconductor-lrbase.pab.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1644</th>\n",
       "      <td>bioconductor-lrbase.rno.eg.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1645</th>\n",
       "      <td>bioconductor-lrbase.ssc.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1646</th>\n",
       "      <td>bioconductor-lrbase.xtr.eg.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1647</th>\n",
       "      <td>bioconductor-lumi</td>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1648</th>\n",
       "      <td>bioconductor-lumibarnes</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1649</th>\n",
       "      <td>bioconductor-lumihumanall.db</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1650</th>\n",
       "      <td>bioconductor-lumihumanidmapping</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1651</th>\n",
       "      <td>bioconductor-lumimouseall.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1652</th>\n",
       "      <td>bioconductor-lumimouseidmapping</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1653</th>\n",
       "      <td>bioconductor-lumiratall.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1654</th>\n",
       "      <td>bioconductor-lumiratidmapping</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1655</th>\n",
       "      <td>bioconductor-lungcanceracvssccgeo</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1656</th>\n",
       "      <td>bioconductor-lungcancerlines</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1657</th>\n",
       "      <td>bioconductor-lungexpression</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1658</th>\n",
       "      <td>bioconductor-lydata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1659</th>\n",
       "      <td>bioconductor-lymphoseq</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1660</th>\n",
       "      <td>bioconductor-lymphoseqdb</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1661</th>\n",
       "      <td>bioconductor-m10kcod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1662</th>\n",
       "      <td>bioconductor-m20kcod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1663</th>\n",
       "      <td>bioconductor-m3c</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1664</th>\n",
       "      <td>bioconductor-m3d</td>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1665</th>\n",
       "      <td>bioconductor-m3dexampledata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1666</th>\n",
       "      <td>bioconductor-m3drop</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1667</th>\n",
       "      <td>bioconductor-maanova</td>\n",
       "      <td>557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1668</th>\n",
       "      <td>bioconductor-maaslin2</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1669</th>\n",
       "      <td>bioconductor-macat</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1670</th>\n",
       "      <td>bioconductor-macorrplot</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1671</th>\n",
       "      <td>bioconductor-macpet</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1672</th>\n",
       "      <td>bioconductor-macrophage</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1673</th>\n",
       "      <td>bioconductor-macsquantifyr</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1674</th>\n",
       "      <td>bioconductor-made4</td>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1675</th>\n",
       "      <td>bioconductor-madseq</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1676</th>\n",
       "      <td>bioconductor-mafdb.1kgenomes.phase1.grch38</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1677</th>\n",
       "      <td>bioconductor-mafdb.1kgenomes.phase1.hs37d5</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1678</th>\n",
       "      <td>bioconductor-mafdb.1kgenomes.phase3.grch38</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1679</th>\n",
       "      <td>bioconductor-mafdb.1kgenomes.phase3.hs37d5</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1680</th>\n",
       "      <td>bioconductor-mafdb.esp6500si.v2.ssa137.grch38</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1681</th>\n",
       "      <td>bioconductor-mafdb.esp6500si.v2.ssa137.hs37d5</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1682</th>\n",
       "      <td>bioconductor-mafdb.exac.r1.0.grch38</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1683</th>\n",
       "      <td>bioconductor-mafdb.exac.r1.0.hs37d5</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1684</th>\n",
       "      <td>bioconductor-mafdb.exac.r1.0.nontcga.grch38</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1685</th>\n",
       "      <td>bioconductor-mafdb.exac.r1.0.nontcga.hs37d5</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1686</th>\n",
       "      <td>bioconductor-mafdb.gnomadex.r2.1.grch38</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1687</th>\n",
       "      <td>bioconductor-mafdb.gnomadex.r2.1.hs37d5</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1688</th>\n",
       "      <td>bioconductor-mafdb.gnomad.r2.1.grch38</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1689</th>\n",
       "      <td>bioconductor-mafdb.gnomad.r2.1.hs37d5</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1690</th>\n",
       "      <td>bioconductor-mafdb.topmed.freeze5.hg19</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1691</th>\n",
       "      <td>bioconductor-mafdb.topmed.freeze5.hg38</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1692</th>\n",
       "      <td>bioconductor-maftools</td>\n",
       "      <td>243</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1693</th>\n",
       "      <td>bioconductor-mageckflute</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1694</th>\n",
       "      <td>bioconductor-maigespack</td>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1695</th>\n",
       "      <td>bioconductor-mait</td>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1696</th>\n",
       "      <td>bioconductor-maizecdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1697</th>\n",
       "      <td>bioconductor-maizeprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1698</th>\n",
       "      <td>bioconductor-makecdfenv</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1699</th>\n",
       "      <td>bioconductor-malaria.db0</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1700</th>\n",
       "      <td>bioconductor-mammaprintdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1701</th>\n",
       "      <td>bioconductor-manor</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1702</th>\n",
       "      <td>bioconductor-mantelcorr</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1703</th>\n",
       "      <td>bioconductor-mapkl</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1704</th>\n",
       "      <td>bioconductor-mapkldata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1705</th>\n",
       "      <td>bioconductor-mapredictdsc</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1706</th>\n",
       "      <td>bioconductor-mapscape</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1707</th>\n",
       "      <td>bioconductor-maqcexpression4plex</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1708</th>\n",
       "      <td>bioconductor-maqcsubset</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1709</th>\n",
       "      <td>bioconductor-maqcsubsetafx</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1710</th>\n",
       "      <td>bioconductor-maqcsubsetilm</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1711</th>\n",
       "      <td>bioconductor-marray</td>\n",
       "      <td>453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1712</th>\n",
       "      <td>bioconductor-martini</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1713</th>\n",
       "      <td>bioconductor-maser</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1714</th>\n",
       "      <td>bioconductor-masigpro</td>\n",
       "      <td>662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1715</th>\n",
       "      <td>bioconductor-maskbad</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1716</th>\n",
       "      <td>bioconductor-massarray</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1717</th>\n",
       "      <td>bioconductor-massir</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1718</th>\n",
       "      <td>bioconductor-massspecwavelet</td>\n",
       "      <td>978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1719</th>\n",
       "      <td>bioconductor-mast</td>\n",
       "      <td>304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1720</th>\n",
       "      <td>bioconductor-matchbox</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1721</th>\n",
       "      <td>bioconductor-matrixrider</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1722</th>\n",
       "      <td>bioconductor-matter</td>\n",
       "      <td>660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1723</th>\n",
       "      <td>bioconductor-maxcontrastprojection</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1724</th>\n",
       "      <td>bioconductor-mbamethyl</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1725</th>\n",
       "      <td>bioconductor-mbased</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1726</th>\n",
       "      <td>bioconductor-mbcb</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1727</th>\n",
       "      <td>bioconductor-mbkmeans</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1728</th>\n",
       "      <td>bioconductor-mbpcr</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1729</th>\n",
       "      <td>bioconductor-mbttest</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1730</th>\n",
       "      <td>bioconductor-mcagui</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1731</th>\n",
       "      <td>bioconductor-mcbiclust</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1732</th>\n",
       "      <td>bioconductor-mcsea</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1733</th>\n",
       "      <td>bioconductor-mcseadata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1734</th>\n",
       "      <td>bioconductor-mcsurvdata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1735</th>\n",
       "      <td>bioconductor-mdgsa</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1736</th>\n",
       "      <td>bioconductor-mdp</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1737</th>\n",
       "      <td>bioconductor-mdqc</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1738</th>\n",
       "      <td>bioconductor-mdts</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1739</th>\n",
       "      <td>bioconductor-meal</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1740</th>\n",
       "      <td>bioconductor-measurementerror.cor</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1741</th>\n",
       "      <td>bioconductor-meb</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1742</th>\n",
       "      <td>bioconductor-medicagocdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1743</th>\n",
       "      <td>bioconductor-medicagoprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1744</th>\n",
       "      <td>bioconductor-medips</td>\n",
       "      <td>383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1745</th>\n",
       "      <td>bioconductor-medipsdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1746</th>\n",
       "      <td>bioconductor-medme</td>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1747</th>\n",
       "      <td>bioconductor-meebodata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1748</th>\n",
       "      <td>bioconductor-meigor</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1749</th>\n",
       "      <td>bioconductor-melissa</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1750</th>\n",
       "      <td>bioconductor-mergemaid</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1751</th>\n",
       "      <td>bioconductor-mergeomics</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1752</th>\n",
       "      <td>bioconductor-mesh.aca.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1753</th>\n",
       "      <td>bioconductor-mesh.aga.pest.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1754</th>\n",
       "      <td>bioconductor-mesh.ame.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1755</th>\n",
       "      <td>bioconductor-mesh.aml.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1756</th>\n",
       "      <td>bioconductor-mesh.ana.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1757</th>\n",
       "      <td>bioconductor-mesh.ani.fgsc.eg.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1758</th>\n",
       "      <td>bioconductor-mesh.aor.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1759</th>\n",
       "      <td>bioconductor-mesh.ath.eg.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1760</th>\n",
       "      <td>bioconductor-mesh.bfl.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1761</th>\n",
       "      <td>bioconductor-mesh.bsu.168.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1762</th>\n",
       "      <td>bioconductor-mesh.bta.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1763</th>\n",
       "      <td>bioconductor-mesh.cal.sc5314.eg.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1764</th>\n",
       "      <td>bioconductor-mesh.cbr.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1765</th>\n",
       "      <td>bioconductor-mesh.cel.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1766</th>\n",
       "      <td>bioconductor-mesh.cfa.eg.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1767</th>\n",
       "      <td>bioconductor-mesh.cin.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1768</th>\n",
       "      <td>bioconductor-mesh.cja.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1769</th>\n",
       "      <td>bioconductor-mesh.cpo.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1770</th>\n",
       "      <td>bioconductor-mesh.cre.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1771</th>\n",
       "      <td>bioconductor-mesh.dan.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1772</th>\n",
       "      <td>bioconductor-mesh.db</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1773</th>\n",
       "      <td>bioconductor-meshdbi</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1774</th>\n",
       "      <td>bioconductor-mesh.dda.3937.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1775</th>\n",
       "      <td>bioconductor-mesh.ddi.ax4.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1776</th>\n",
       "      <td>bioconductor-mesh.der.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1777</th>\n",
       "      <td>bioconductor-mesh.dgr.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1778</th>\n",
       "      <td>bioconductor-mesh.dme.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1779</th>\n",
       "      <td>bioconductor-mesh.dmo.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1780</th>\n",
       "      <td>bioconductor-mesh.dpe.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1781</th>\n",
       "      <td>bioconductor-mesh.dre.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1782</th>\n",
       "      <td>bioconductor-mesh.dse.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1783</th>\n",
       "      <td>bioconductor-mesh.dsi.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1784</th>\n",
       "      <td>bioconductor-mesh.dvi.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1785</th>\n",
       "      <td>bioconductor-mesh.dya.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1786</th>\n",
       "      <td>bioconductor-mesh.eco.55989.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1787</th>\n",
       "      <td>bioconductor-mesh.eco.ed1a.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1788</th>\n",
       "      <td>bioconductor-mesh.eco.iai39.eg.db</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1789</th>\n",
       "      <td>bioconductor-mesh.eco.k12.mg1655.eg.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1790</th>\n",
       "      <td>bioconductor-mesh.eco.o157.h7.sakai.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1791</th>\n",
       "      <td>bioconductor-mesh.eco.umn026.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1792</th>\n",
       "      <td>bioconductor-mesh.eqc.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1793</th>\n",
       "      <td>bioconductor-meshes</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1794</th>\n",
       "      <td>bioconductor-mesh.gga.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1795</th>\n",
       "      <td>bioconductor-mesh.gma.eg.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1796</th>\n",
       "      <td>bioconductor-mesh.hsa.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1797</th>\n",
       "      <td>bioconductor-mesh.laf.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1798</th>\n",
       "      <td>bioconductor-mesh.lma.eg.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1799</th>\n",
       "      <td>bioconductor-mesh.mdo.eg.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1800</th>\n",
       "      <td>bioconductor-mesh.mes.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1801</th>\n",
       "      <td>bioconductor-mesh.mga.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1802</th>\n",
       "      <td>bioconductor-mesh.miy.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1803</th>\n",
       "      <td>bioconductor-mesh.mml.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1804</th>\n",
       "      <td>bioconductor-mesh.mmu.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1805</th>\n",
       "      <td>bioconductor-mesh.mtr.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1806</th>\n",
       "      <td>bioconductor-mesh.nle.eg.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1807</th>\n",
       "      <td>bioconductor-mesh.oan.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1808</th>\n",
       "      <td>bioconductor-mesh.ocu.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1809</th>\n",
       "      <td>bioconductor-mesh.oni.eg.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1810</th>\n",
       "      <td>bioconductor-mesh.osa.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1811</th>\n",
       "      <td>bioconductor-mesh.pab.eg.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1812</th>\n",
       "      <td>bioconductor-mesh.pae.pao1.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1813</th>\n",
       "      <td>bioconductor-mesh.pcr.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1814</th>\n",
       "      <td>bioconductor-mesh.pfa.3d7.eg.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1815</th>\n",
       "      <td>bioconductor-mesh.pto.eg.db</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1816</th>\n",
       "      <td>bioconductor-mesh.ptr.eg.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1817</th>\n",
       "      <td>bioconductor-meshr</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1818</th>\n",
       "      <td>bioconductor-mesh.rno.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1819</th>\n",
       "      <td>bioconductor-mesh.sce.s288c.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1820</th>\n",
       "      <td>bioconductor-mesh.sco.a32.eg.db</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1821</th>\n",
       "      <td>bioconductor-mesh.sil.eg.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1822</th>\n",
       "      <td>bioconductor-mesh.spu.eg.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1823</th>\n",
       "      <td>bioconductor-mesh.ssc.eg.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1824</th>\n",
       "      <td>bioconductor-mesh.syn.eg.db</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1825</th>\n",
       "      <td>bioconductor-mesh.tbr.9274.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1826</th>\n",
       "      <td>bioconductor-mesh.tgo.me49.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1827</th>\n",
       "      <td>bioconductor-mesh.tgu.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1828</th>\n",
       "      <td>bioconductor-mesh.vvi.eg.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1829</th>\n",
       "      <td>bioconductor-mesh.xla.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1830</th>\n",
       "      <td>bioconductor-mesh.xtr.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1831</th>\n",
       "      <td>bioconductor-mesh.zma.eg.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1832</th>\n",
       "      <td>bioconductor-messina</td>\n",
       "      <td>569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1833</th>\n",
       "      <td>bioconductor-metaarray</td>\n",
       "      <td>569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1834</th>\n",
       "      <td>bioconductor-metab</td>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1835</th>\n",
       "      <td>bioconductor-metabomxtr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1836</th>\n",
       "      <td>bioconductor-metabosignal</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1837</th>\n",
       "      <td>bioconductor-metacca</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1838</th>\n",
       "      <td>bioconductor-metacyto</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1839</th>\n",
       "      <td>bioconductor-metagene2</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1840</th>\n",
       "      <td>bioconductor-metagene</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1841</th>\n",
       "      <td>bioconductor-metagenomefeatures</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1842</th>\n",
       "      <td>bioconductor-metagenomeseq</td>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1843</th>\n",
       "      <td>bioconductor-metagxbreast</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1844</th>\n",
       "      <td>bioconductor-metagxovarian</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1845</th>\n",
       "      <td>bioconductor-metagxpancreas</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1846</th>\n",
       "      <td>bioconductor-metahdep</td>\n",
       "      <td>575</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1847</th>\n",
       "      <td>bioconductor-metams</td>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1848</th>\n",
       "      <td>bioconductor-metamsdata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1849</th>\n",
       "      <td>bioconductor-metaneighbor</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1850</th>\n",
       "      <td>bioconductor-metaseq</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1851</th>\n",
       "      <td>bioconductor-metaseqr</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1852</th>\n",
       "      <td>bioconductor-metavizr</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1853</th>\n",
       "      <td>bioconductor-metavolcanor</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1854</th>\n",
       "      <td>bioconductor-metcirc</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1855</th>\n",
       "      <td>bioconductor-methcp</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1856</th>\n",
       "      <td>bioconductor-methimpute</td>\n",
       "      <td>369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1857</th>\n",
       "      <td>bioconductor-methinheritsim</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1858</th>\n",
       "      <td>bioconductor-methped</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1859</th>\n",
       "      <td>bioconductor-methrix</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1860</th>\n",
       "      <td>bioconductor-methtargetedngs</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1861</th>\n",
       "      <td>bioconductor-methvisual</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1862</th>\n",
       "      <td>bioconductor-methyanalysis</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1863</th>\n",
       "      <td>bioconductor-methylaid</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1864</th>\n",
       "      <td>bioconductor-methylaiddata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1865</th>\n",
       "      <td>bioconductor-methylcc</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1866</th>\n",
       "      <td>bioconductor-methylgsa</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1867</th>\n",
       "      <td>bioconductor-methylinheritance</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1868</th>\n",
       "      <td>bioconductor-methylkit</td>\n",
       "      <td>866</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1869</th>\n",
       "      <td>bioconductor-methylmix</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1870</th>\n",
       "      <td>bioconductor-methylmnm</td>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1871</th>\n",
       "      <td>bioconductor-methylpipe</td>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1872</th>\n",
       "      <td>bioconductor-methylseekr</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1873</th>\n",
       "      <td>bioconductor-methylumi</td>\n",
       "      <td>322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1874</th>\n",
       "      <td>bioconductor-methyvim</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1875</th>\n",
       "      <td>bioconductor-methyvimdata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1876</th>\n",
       "      <td>bioconductor-metid</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1877</th>\n",
       "      <td>bioconductor-metnet</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1878</th>\n",
       "      <td>bioconductor-mfa</td>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1879</th>\n",
       "      <td>bioconductor-mfuzz</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1880</th>\n",
       "      <td>bioconductor-mgfm</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1881</th>\n",
       "      <td>bioconductor-mgfr</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1882</th>\n",
       "      <td>bioconductor-mgsa</td>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1883</th>\n",
       "      <td>bioconductor-mgu74acdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1884</th>\n",
       "      <td>bioconductor-mgu74a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1885</th>\n",
       "      <td>bioconductor-mgu74aprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1886</th>\n",
       "      <td>bioconductor-mgu74av2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1887</th>\n",
       "      <td>bioconductor-mgu74av2.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1888</th>\n",
       "      <td>bioconductor-mgu74av2probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1889</th>\n",
       "      <td>bioconductor-mgu74bcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1890</th>\n",
       "      <td>bioconductor-mgu74b.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1891</th>\n",
       "      <td>bioconductor-mgu74bprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1892</th>\n",
       "      <td>bioconductor-mgu74bv2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1893</th>\n",
       "      <td>bioconductor-mgu74bv2.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1894</th>\n",
       "      <td>bioconductor-mgu74bv2probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1895</th>\n",
       "      <td>bioconductor-mgu74ccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1896</th>\n",
       "      <td>bioconductor-mgu74c.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1897</th>\n",
       "      <td>bioconductor-mgu74cprobe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1898</th>\n",
       "      <td>bioconductor-mgu74cv2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1899</th>\n",
       "      <td>bioconductor-mgu74cv2.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1900</th>\n",
       "      <td>bioconductor-mgu74cv2probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1901</th>\n",
       "      <td>bioconductor-mguatlas5k.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1902</th>\n",
       "      <td>bioconductor-mgug4104a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1903</th>\n",
       "      <td>bioconductor-mgug4120a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1904</th>\n",
       "      <td>bioconductor-mgug4121a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1905</th>\n",
       "      <td>bioconductor-mgug4122a.db</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1906</th>\n",
       "      <td>bioconductor-mi16cod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1907</th>\n",
       "      <td>bioconductor-michip</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1908</th>\n",
       "      <td>bioconductor-microbiome</td>\n",
       "      <td>213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1909</th>\n",
       "      <td>bioconductor-microbiomedasim</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1910</th>\n",
       "      <td>bioconductor-microrna</td>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1911</th>\n",
       "      <td>bioconductor-micrornaome</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1912</th>\n",
       "      <td>bioconductor-migsa</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1913</th>\n",
       "      <td>bioconductor-migsadata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1914</th>\n",
       "      <td>bioconductor-mimager</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1915</th>\n",
       "      <td>bioconductor-mimosa</td>\n",
       "      <td>377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1916</th>\n",
       "      <td>bioconductor-mineica</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1917</th>\n",
       "      <td>bioconductor-minet</td>\n",
       "      <td>483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1918</th>\n",
       "      <td>bioconductor-minfi</td>\n",
       "      <td>562</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1919</th>\n",
       "      <td>bioconductor-minfidata</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1920</th>\n",
       "      <td>bioconductor-minfidataepic</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1921</th>\n",
       "      <td>bioconductor-minimumdistance</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1922</th>\n",
       "      <td>bioconductor-minionsummarydata</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1923</th>\n",
       "      <td>bioconductor-mipp</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1924</th>\n",
       "      <td>bioconductor-mira</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1925</th>\n",
       "      <td>bioconductor-mirage</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1926</th>\n",
       "      <td>bioconductor-mirbaseconverter</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1927</th>\n",
       "      <td>bioconductor-mirbase.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1928</th>\n",
       "      <td>bioconductor-mirbaseversions.db</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1929</th>\n",
       "      <td>bioconductor-mircomp</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1930</th>\n",
       "      <td>bioconductor-mircompdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1931</th>\n",
       "      <td>bioconductor-mirintegrator</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1932</th>\n",
       "      <td>bioconductor-mirmine</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1933</th>\n",
       "      <td>bioconductor-mirna102xgaincdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1934</th>\n",
       "      <td>bioconductor-mirna10cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1935</th>\n",
       "      <td>bioconductor-mirna10probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1936</th>\n",
       "      <td>bioconductor-mirna20cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1937</th>\n",
       "      <td>bioconductor-mirnameconverter</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1938</th>\n",
       "      <td>bioconductor-mirnapath</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1939</th>\n",
       "      <td>bioconductor-mirnatap</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1940</th>\n",
       "      <td>bioconductor-mirnatap.db</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1941</th>\n",
       "      <td>bioconductor-mirnatarget</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1942</th>\n",
       "      <td>bioconductor-mirsm</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1943</th>\n",
       "      <td>bioconductor-mirsponger</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1944</th>\n",
       "      <td>bioconductor-mirsynergy</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1945</th>\n",
       "      <td>bioconductor-missmethyl</td>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1946</th>\n",
       "      <td>bioconductor-missrows</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1947</th>\n",
       "      <td>bioconductor-mitoode</td>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1948</th>\n",
       "      <td>bioconductor-mitoodedata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1949</th>\n",
       "      <td>bioconductor-mixomics</td>\n",
       "      <td>35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1950</th>\n",
       "      <td>bioconductor-mlinterfaces</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1951</th>\n",
       "      <td>bioconductor-mlp</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1952</th>\n",
       "      <td>bioconductor-mlseq</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1953</th>\n",
       "      <td>bioconductor-mm24kresogen.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1954</th>\n",
       "      <td>bioconductor-mmagilentdesign026655.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1955</th>\n",
       "      <td>bioconductor-mmappr2</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1956</th>\n",
       "      <td>bioconductor-mmappr2data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1957</th>\n",
       "      <td>bioconductor-mmdiff2</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1958</th>\n",
       "      <td>bioconductor-mmdiffbamsubset</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1959</th>\n",
       "      <td>bioconductor-mmpalatemirna</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1960</th>\n",
       "      <td>bioconductor-mmuphin</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1961</th>\n",
       "      <td>bioconductor-moda</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1962</th>\n",
       "      <td>bioconductor-modstrings</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1963</th>\n",
       "      <td>bioconductor-moe430acdf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1964</th>\n",
       "      <td>bioconductor-moe430a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1965</th>\n",
       "      <td>bioconductor-moe430aprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1966</th>\n",
       "      <td>bioconductor-moe430bcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1967</th>\n",
       "      <td>bioconductor-moe430b.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1968</th>\n",
       "      <td>bioconductor-moe430bprobe</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1969</th>\n",
       "      <td>bioconductor-moex10stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1970</th>\n",
       "      <td>bioconductor-moex10sttranscriptcluster.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1971</th>\n",
       "      <td>bioconductor-moexexonprobesetlocation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1972</th>\n",
       "      <td>bioconductor-mofa</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1973</th>\n",
       "      <td>bioconductor-mofadata</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1974</th>\n",
       "      <td>bioconductor-mogene10stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1975</th>\n",
       "      <td>bioconductor-mogene10sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1976</th>\n",
       "      <td>bioconductor-mogene10stv1cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1977</th>\n",
       "      <td>bioconductor-mogene.1.0.st.v1frmavecs</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1978</th>\n",
       "      <td>bioconductor-mogene10stv1probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1979</th>\n",
       "      <td>bioconductor-mogene11stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1980</th>\n",
       "      <td>bioconductor-mogene11sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1981</th>\n",
       "      <td>bioconductor-mogene20stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1982</th>\n",
       "      <td>bioconductor-mogene20sttranscriptcluster.db</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1983</th>\n",
       "      <td>bioconductor-mogene21stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1984</th>\n",
       "      <td>bioconductor-mogene21sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1985</th>\n",
       "      <td>bioconductor-mogsa</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1986</th>\n",
       "      <td>bioconductor-monocle</td>\n",
       "      <td>872</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1987</th>\n",
       "      <td>bioconductor-moonlightr</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1988</th>\n",
       "      <td>bioconductor-mops</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1989</th>\n",
       "      <td>bioconductor-mosaics</td>\n",
       "      <td>744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1990</th>\n",
       "      <td>bioconductor-mosaicsexample</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1991</th>\n",
       "      <td>bioconductor-mosim</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1992</th>\n",
       "      <td>bioconductor-motifbreakr</td>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1993</th>\n",
       "      <td>bioconductor-motifcounter</td>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1994</th>\n",
       "      <td>bioconductor-motifdb</td>\n",
       "      <td>391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1995</th>\n",
       "      <td>bioconductor-motifmatchr</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1996</th>\n",
       "      <td>bioconductor-motifrg</td>\n",
       "      <td>473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1997</th>\n",
       "      <td>bioconductor-motifstack</td>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1998</th>\n",
       "      <td>bioconductor-motiv</td>\n",
       "      <td>660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1999</th>\n",
       "      <td>bioconductor-mouse4302barcodevecs</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2000</th>\n",
       "      <td>bioconductor-mouse4302cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2001</th>\n",
       "      <td>bioconductor-mouse4302.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2002</th>\n",
       "      <td>bioconductor-mouse4302frmavecs</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2003</th>\n",
       "      <td>bioconductor-mouse4302probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2004</th>\n",
       "      <td>bioconductor-mouse430a2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2005</th>\n",
       "      <td>bioconductor-mouse430a2.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2006</th>\n",
       "      <td>bioconductor-mouse430a2frmavecs</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2007</th>\n",
       "      <td>bioconductor-mouse430a2probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2008</th>\n",
       "      <td>bioconductor-mousechrloc</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2009</th>\n",
       "      <td>bioconductor-mouse.db0</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2010</th>\n",
       "      <td>bioconductor-mousegastrulationdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2011</th>\n",
       "      <td>bioconductor-mpedbarray.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2012</th>\n",
       "      <td>bioconductor-mpfe</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2013</th>\n",
       "      <td>bioconductor-mpra</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2014</th>\n",
       "      <td>bioconductor-mpranalyze</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2015</th>\n",
       "      <td>bioconductor-msa</td>\n",
       "      <td>309</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2016</th>\n",
       "      <td>bioconductor-msd16s</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2017</th>\n",
       "      <td>bioconductor-msdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2018</th>\n",
       "      <td>bioconductor-msgbsr</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2019</th>\n",
       "      <td>bioconductor-msgfgui</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2020</th>\n",
       "      <td>bioconductor-msgfplus</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2021</th>\n",
       "      <td>bioconductor-msmb</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2022</th>\n",
       "      <td>bioconductor-msmseda</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2023</th>\n",
       "      <td>bioconductor-msmstests</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2024</th>\n",
       "      <td>bioconductor-msnbase</td>\n",
       "      <td>794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2025</th>\n",
       "      <td>bioconductor-msnid</td>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2026</th>\n",
       "      <td>bioconductor-mspurity</td>\n",
       "      <td>661</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2027</th>\n",
       "      <td>bioconductor-mspuritydata</td>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2028</th>\n",
       "      <td>bioconductor-msqc1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2029</th>\n",
       "      <td>bioconductor-msstats</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2030</th>\n",
       "      <td>bioconductor-msstatsbiodata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2031</th>\n",
       "      <td>bioconductor-msstatsqc</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2032</th>\n",
       "      <td>bioconductor-msstatsqcgui</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2033</th>\n",
       "      <td>bioconductor-msstatssamplesize</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2034</th>\n",
       "      <td>bioconductor-msstatstmt</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2035</th>\n",
       "      <td>bioconductor-mta10probeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2036</th>\n",
       "      <td>bioconductor-mta10transcriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2037</th>\n",
       "      <td>bioconductor-mtbls2</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2038</th>\n",
       "      <td>bioconductor-mtseeker</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2039</th>\n",
       "      <td>bioconductor-mtseekerdata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2040</th>\n",
       "      <td>bioconductor-mu11ksubacdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2041</th>\n",
       "      <td>bioconductor-mu11ksuba.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2042</th>\n",
       "      <td>bioconductor-mu11ksubaprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2043</th>\n",
       "      <td>bioconductor-mu11ksubbcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2044</th>\n",
       "      <td>bioconductor-mu11ksubb.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2045</th>\n",
       "      <td>bioconductor-mu11ksubbprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2046</th>\n",
       "      <td>bioconductor-mu15v1.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2047</th>\n",
       "      <td>bioconductor-mu19ksubacdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2048</th>\n",
       "      <td>bioconductor-mu19ksuba.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2049</th>\n",
       "      <td>bioconductor-mu19ksubbcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2050</th>\n",
       "      <td>bioconductor-mu19ksubb.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2051</th>\n",
       "      <td>bioconductor-mu19ksubccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2052</th>\n",
       "      <td>bioconductor-mu19ksubc.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2053</th>\n",
       "      <td>bioconductor-mu22v3.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2054</th>\n",
       "      <td>bioconductor-mu6500subacdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2055</th>\n",
       "      <td>bioconductor-mu6500subbcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2056</th>\n",
       "      <td>bioconductor-mu6500subccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2057</th>\n",
       "      <td>bioconductor-mu6500subdcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2058</th>\n",
       "      <td>bioconductor-mugaexampledata</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2059</th>\n",
       "      <td>bioconductor-mulcom</td>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2060</th>\n",
       "      <td>bioconductor-mulder2012</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2061</th>\n",
       "      <td>bioconductor-multiassayexperiment</td>\n",
       "      <td>212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2062</th>\n",
       "      <td>bioconductor-multiclust</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2063</th>\n",
       "      <td>bioconductor-multidataset</td>\n",
       "      <td>306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2064</th>\n",
       "      <td>bioconductor-multihiccompare</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2065</th>\n",
       "      <td>bioconductor-multimed</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2066</th>\n",
       "      <td>bioconductor-multimir</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2067</th>\n",
       "      <td>bioconductor-multiomicsviz</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2068</th>\n",
       "      <td>bioconductor-multiscan</td>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2069</th>\n",
       "      <td>bioconductor-multtest</td>\n",
       "      <td>1275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2070</th>\n",
       "      <td>bioconductor-muscat</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2071</th>\n",
       "      <td>bioconductor-muscdata</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2072</th>\n",
       "      <td>bioconductor-muscle</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2073</th>\n",
       "      <td>bioconductor-mus.musculus</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2074</th>\n",
       "      <td>bioconductor-mutationalpatterns</td>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2075</th>\n",
       "      <td>bioconductor-mvcclass</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2076</th>\n",
       "      <td>bioconductor-mvoutdata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2077</th>\n",
       "      <td>bioconductor-mwastools</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2078</th>\n",
       "      <td>bioconductor-mwgcod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2079</th>\n",
       "      <td>bioconductor-mygene</td>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2080</th>\n",
       "      <td>bioconductor-myvariant</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2081</th>\n",
       "      <td>bioconductor-mzid</td>\n",
       "      <td>510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2082</th>\n",
       "      <td>bioconductor-mzr</td>\n",
       "      <td>984</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2083</th>\n",
       "      <td>bioconductor-nadfinder</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2084</th>\n",
       "      <td>bioconductor-nanostringdiff</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2085</th>\n",
       "      <td>bioconductor-nanostringqcpro</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2086</th>\n",
       "      <td>bioconductor-nanotator</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2087</th>\n",
       "      <td>bioconductor-nanotubes</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2088</th>\n",
       "      <td>bioconductor-narrowpeaks</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2089</th>\n",
       "      <td>bioconductor-nbamseq</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2090</th>\n",
       "      <td>bioconductor-nbsplice</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2091</th>\n",
       "      <td>bioconductor-ncdfflow</td>\n",
       "      <td>775</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2092</th>\n",
       "      <td>bioconductor-ncgtw</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2093</th>\n",
       "      <td>bioconductor-ncigraph</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2094</th>\n",
       "      <td>bioconductor-ncigraphdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2095</th>\n",
       "      <td>bioconductor-ndexr</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2096</th>\n",
       "      <td>bioconductor-neighbornet</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2097</th>\n",
       "      <td>bioconductor-nestlink</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2098</th>\n",
       "      <td>bioconductor-netbenchmark</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2099</th>\n",
       "      <td>bioconductor-netbiov</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2100</th>\n",
       "      <td>bioconductor-netboost</td>\n",
       "      <td>183</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2101</th>\n",
       "      <td>bioconductor-nethet</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2102</th>\n",
       "      <td>bioconductor-netpathminer</td>\n",
       "      <td>565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2103</th>\n",
       "      <td>bioconductor-netprior</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2104</th>\n",
       "      <td>bioconductor-netreg</td>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2105</th>\n",
       "      <td>bioconductor-netresponse</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2106</th>\n",
       "      <td>bioconductor-netsam</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2107</th>\n",
       "      <td>bioconductor-netsmooth</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2108</th>\n",
       "      <td>bioconductor-networkbma</td>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2109</th>\n",
       "      <td>bioconductor-neve2006</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2110</th>\n",
       "      <td>bioconductor-ngscopydata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2111</th>\n",
       "      <td>bioconductor-ngsreports</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2112</th>\n",
       "      <td>bioconductor-nnnorm</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2113</th>\n",
       "      <td>bioconductor-noiseq</td>\n",
       "      <td>5679</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2114</th>\n",
       "      <td>bioconductor-nondetects</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2115</th>\n",
       "      <td>bioconductor-normalize450k</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2116</th>\n",
       "      <td>bioconductor-normalyzerde</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2117</th>\n",
       "      <td>bioconductor-normqpcr</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2118</th>\n",
       "      <td>bioconductor-normr</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2119</th>\n",
       "      <td>bioconductor-norway981.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2120</th>\n",
       "      <td>bioconductor-npgsea</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2121</th>\n",
       "      <td>bioconductor-ntw</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2122</th>\n",
       "      <td>bioconductor-nucleosim</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2123</th>\n",
       "      <td>bioconductor-nucler</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2124</th>\n",
       "      <td>bioconductor-nucpos</td>\n",
       "      <td>393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2125</th>\n",
       "      <td>bioconductor-nugohs1a520180cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2126</th>\n",
       "      <td>bioconductor-nugohs1a520180.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2127</th>\n",
       "      <td>bioconductor-nugohs1a520180probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2128</th>\n",
       "      <td>bioconductor-nugomm1a520177cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2129</th>\n",
       "      <td>bioconductor-nugomm1a520177.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2130</th>\n",
       "      <td>bioconductor-nugomm1a520177probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2131</th>\n",
       "      <td>bioconductor-nupop</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2132</th>\n",
       "      <td>bioconductor-occugene</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2133</th>\n",
       "      <td>bioconductor-ocplus</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2134</th>\n",
       "      <td>bioconductor-oct4</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2135</th>\n",
       "      <td>bioconductor-odseq</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2136</th>\n",
       "      <td>bioconductor-ogsa</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2137</th>\n",
       "      <td>bioconductor-oligo</td>\n",
       "      <td>682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2138</th>\n",
       "      <td>bioconductor-oligoclasses</td>\n",
       "      <td>417</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2139</th>\n",
       "      <td>bioconductor-oligodata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2140</th>\n",
       "      <td>bioconductor-olin</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2141</th>\n",
       "      <td>bioconductor-olingui</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2142</th>\n",
       "      <td>bioconductor-omadb</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2143</th>\n",
       "      <td>bioconductor-omicade4</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2144</th>\n",
       "      <td>bioconductor-omiccircos</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2145</th>\n",
       "      <td>bioconductor-omicplotr</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2146</th>\n",
       "      <td>bioconductor-omicrexposome</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2147</th>\n",
       "      <td>bioconductor-omicslonda</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2148</th>\n",
       "      <td>bioconductor-omicsmarker</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2149</th>\n",
       "      <td>bioconductor-omicspca</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2150</th>\n",
       "      <td>bioconductor-omicspcadata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2151</th>\n",
       "      <td>bioconductor-omicsprint</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2152</th>\n",
       "      <td>bioconductor-omnipathr</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2153</th>\n",
       "      <td>bioconductor-onassis</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2154</th>\n",
       "      <td>bioconductor-onassisjavalibs</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2155</th>\n",
       "      <td>bioconductor-oncomix</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2156</th>\n",
       "      <td>bioconductor-oncoscore</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2157</th>\n",
       "      <td>bioconductor-oncosimulr</td>\n",
       "      <td>473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2158</th>\n",
       "      <td>bioconductor-onesense</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2159</th>\n",
       "      <td>bioconductor-onlinefdr</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2160</th>\n",
       "      <td>bioconductor-ontoproc</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2161</th>\n",
       "      <td>bioconductor-opencyto</td>\n",
       "      <td>308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2162</th>\n",
       "      <td>bioconductor-openprimer</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2163</th>\n",
       "      <td>bioconductor-openprimerui</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2164</th>\n",
       "      <td>bioconductor-operonhumanv3.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2165</th>\n",
       "      <td>bioconductor-opossom</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2166</th>\n",
       "      <td>bioconductor-oppar</td>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2167</th>\n",
       "      <td>bioconductor-oppti</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2168</th>\n",
       "      <td>bioconductor-opweight</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2169</th>\n",
       "      <td>bioconductor-orderedlist</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2170</th>\n",
       "      <td>bioconductor-orfik</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2171</th>\n",
       "      <td>bioconductor-org.ag.eg.db</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2172</th>\n",
       "      <td>bioconductor-organismdbi</td>\n",
       "      <td>678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2173</th>\n",
       "      <td>bioconductor-organism.dplyr</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2174</th>\n",
       "      <td>bioconductor-org.at.tair.db</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2175</th>\n",
       "      <td>bioconductor-org.bt.eg.db</td>\n",
       "      <td>392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2176</th>\n",
       "      <td>bioconductor-org.ce.eg.db</td>\n",
       "      <td>415</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2177</th>\n",
       "      <td>bioconductor-org.cf.eg.db</td>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2178</th>\n",
       "      <td>bioconductor-org.dm.eg.db</td>\n",
       "      <td>718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2179</th>\n",
       "      <td>bioconductor-org.dr.eg.db</td>\n",
       "      <td>459</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2180</th>\n",
       "      <td>bioconductor-org.eck12.eg.db</td>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2181</th>\n",
       "      <td>bioconductor-org.ecsakai.eg.db</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2182</th>\n",
       "      <td>bioconductor-org.gg.eg.db</td>\n",
       "      <td>404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2183</th>\n",
       "      <td>bioconductor-org.hs.eg.db</td>\n",
       "      <td>1190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2184</th>\n",
       "      <td>bioconductor-org.mm.eg.db</td>\n",
       "      <td>1033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2185</th>\n",
       "      <td>bioconductor-org.mmu.eg.db</td>\n",
       "      <td>51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2186</th>\n",
       "      <td>bioconductor-org.mxanthus.db</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2187</th>\n",
       "      <td>bioconductor-org.pf.plasmo.db</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2188</th>\n",
       "      <td>bioconductor-org.pt.eg.db</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2189</th>\n",
       "      <td>bioconductor-org.rn.eg.db</td>\n",
       "      <td>420</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2190</th>\n",
       "      <td>bioconductor-org.sc.sgd.db</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2191</th>\n",
       "      <td>bioconductor-org.ss.eg.db</td>\n",
       "      <td>397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2192</th>\n",
       "      <td>bioconductor-org.xl.eg.db</td>\n",
       "      <td>26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2193</th>\n",
       "      <td>bioconductor-osat</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2194</th>\n",
       "      <td>bioconductor-oscope</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2195</th>\n",
       "      <td>bioconductor-otubase</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2196</th>\n",
       "      <td>bioconductor-outlierd</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2197</th>\n",
       "      <td>bioconductor-outrider</td>\n",
       "      <td>378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2198</th>\n",
       "      <td>bioconductor-oveseg</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2199</th>\n",
       "      <td>bioconductor-paa</td>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2200</th>\n",
       "      <td>bioconductor-padog</td>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2201</th>\n",
       "      <td>bioconductor-paeg1acdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2202</th>\n",
       "      <td>bioconductor-paeg1aprobe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2203</th>\n",
       "      <td>bioconductor-pairadise</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2204</th>\n",
       "      <td>bioconductor-paircompviz</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2205</th>\n",
       "      <td>bioconductor-pandar</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2206</th>\n",
       "      <td>bioconductor-panelcn.mops</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2207</th>\n",
       "      <td>bioconductor-panp</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2208</th>\n",
       "      <td>bioconductor-panr</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2209</th>\n",
       "      <td>bioconductor-panther.db</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2210</th>\n",
       "      <td>bioconductor-panvizgenerator</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2211</th>\n",
       "      <td>bioconductor-parathyroidse</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2212</th>\n",
       "      <td>bioconductor-parglms</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2213</th>\n",
       "      <td>bioconductor-parody</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2214</th>\n",
       "      <td>bioconductor-partheenmetadata.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2215</th>\n",
       "      <td>bioconductor-pasilla</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2216</th>\n",
       "      <td>bioconductor-pasillabamsubset</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2217</th>\n",
       "      <td>bioconductor-pasillatranscriptexpr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2218</th>\n",
       "      <td>bioconductor-past</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2219</th>\n",
       "      <td>bioconductor-path2ppi</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2220</th>\n",
       "      <td>bioconductor-pathifier</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2221</th>\n",
       "      <td>bioconductor-pathnet</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2222</th>\n",
       "      <td>bioconductor-pathnetdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2223</th>\n",
       "      <td>bioconductor-pathostat</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2224</th>\n",
       "      <td>bioconductor-pathprint</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2225</th>\n",
       "      <td>bioconductor-pathprintgeodata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2226</th>\n",
       "      <td>bioconductor-pathrender</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2227</th>\n",
       "      <td>bioconductor-pathvar</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2228</th>\n",
       "      <td>bioconductor-pathview</td>\n",
       "      <td>516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2229</th>\n",
       "      <td>bioconductor-pathwaypca</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2230</th>\n",
       "      <td>bioconductor-pathwaysplice</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2231</th>\n",
       "      <td>bioconductor-paxtoolsr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2232</th>\n",
       "      <td>bioconductor-pcaexplorer</td>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2233</th>\n",
       "      <td>bioconductor-pcagopromoter.hs.hg19</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2234</th>\n",
       "      <td>bioconductor-pcagopromoter.mm.mm9</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2235</th>\n",
       "      <td>bioconductor-pcagopromoter.rn.rn4</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2236</th>\n",
       "      <td>bioconductor-pcamethods</td>\n",
       "      <td>981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2237</th>\n",
       "      <td>bioconductor-pcan</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2238</th>\n",
       "      <td>bioconductor-pcatools</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2239</th>\n",
       "      <td>bioconductor-pchicdata</td>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2240</th>\n",
       "      <td>bioconductor-pcot2</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2241</th>\n",
       "      <td>bioconductor-pcpheno</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2242</th>\n",
       "      <td>bioconductor-pcxn</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2243</th>\n",
       "      <td>bioconductor-pcxndata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2244</th>\n",
       "      <td>bioconductor-pd.081229.hg18.promoter.medip.hx1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2245</th>\n",
       "      <td>bioconductor-pd.2006.07.18.hg18.refseq.promoter</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2246</th>\n",
       "      <td>bioconductor-pd.2006.07.18.mm8.refseq.promoter</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2247</th>\n",
       "      <td>bioconductor-pd.2006.10.31.rn34.refseq.promoter</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2248</th>\n",
       "      <td>bioconductor-pd.ag</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2249</th>\n",
       "      <td>bioconductor-pd.aragene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2250</th>\n",
       "      <td>bioconductor-pd.aragene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2251</th>\n",
       "      <td>bioconductor-pd.atdschip.tiling</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2252</th>\n",
       "      <td>bioconductor-pd.ath1.121501</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2253</th>\n",
       "      <td>bioconductor-pd.barley1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2254</th>\n",
       "      <td>bioconductor-pd.bovgene.1.0.st</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2255</th>\n",
       "      <td>bioconductor-pd.bovgene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2256</th>\n",
       "      <td>bioconductor-pd.bovine</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2257</th>\n",
       "      <td>bioconductor-pd.bsubtilis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2258</th>\n",
       "      <td>bioconductor-pd.cangene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2259</th>\n",
       "      <td>bioconductor-pd.cangene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2260</th>\n",
       "      <td>bioconductor-pd.canine.2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2261</th>\n",
       "      <td>bioconductor-pd.canine</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2262</th>\n",
       "      <td>bioconductor-pd.celegans</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2263</th>\n",
       "      <td>bioconductor-pd.charm.hg18.example</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2264</th>\n",
       "      <td>bioconductor-pd.chicken</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2265</th>\n",
       "      <td>bioconductor-pd.chigene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2266</th>\n",
       "      <td>bioconductor-pd.chigene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2267</th>\n",
       "      <td>bioconductor-pd.chogene.2.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2268</th>\n",
       "      <td>bioconductor-pd.chogene.2.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2269</th>\n",
       "      <td>bioconductor-pd.citrus</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2270</th>\n",
       "      <td>bioconductor-pd.clariom.d.human</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2271</th>\n",
       "      <td>bioconductor-pd.clariom.s.human</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2272</th>\n",
       "      <td>bioconductor-pd.clariom.s.human.ht</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2273</th>\n",
       "      <td>bioconductor-pd.clariom.s.mouse</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2274</th>\n",
       "      <td>bioconductor-pd.clariom.s.mouse.ht</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2275</th>\n",
       "      <td>bioconductor-pd.clariom.s.rat</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2276</th>\n",
       "      <td>bioconductor-pd.clariom.s.rat.ht</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2277</th>\n",
       "      <td>bioconductor-pd.cotton</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2278</th>\n",
       "      <td>bioconductor-pd.cyngene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2279</th>\n",
       "      <td>bioconductor-pd.cyngene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2280</th>\n",
       "      <td>bioconductor-pd.cyrgene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2281</th>\n",
       "      <td>bioconductor-pd.cyrgene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2282</th>\n",
       "      <td>bioconductor-pd.cytogenetics.array</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2283</th>\n",
       "      <td>bioconductor-pd.drogene.1.0.st</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2284</th>\n",
       "      <td>bioconductor-pd.drogene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2285</th>\n",
       "      <td>bioconductor-pd.drosgenome1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2286</th>\n",
       "      <td>bioconductor-pd.drosophila.2</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2287</th>\n",
       "      <td>bioconductor-pd.e.coli.2</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2288</th>\n",
       "      <td>bioconductor-pd.ecoli</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2289</th>\n",
       "      <td>bioconductor-pd.ecoli.asv2</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2290</th>\n",
       "      <td>bioconductor-pd.elegene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2291</th>\n",
       "      <td>bioconductor-pd.elegene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2292</th>\n",
       "      <td>bioconductor-pd.equgene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2293</th>\n",
       "      <td>bioconductor-pd.equgene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2294</th>\n",
       "      <td>bioconductor-pd.feinberg.hg18.me.hx1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2295</th>\n",
       "      <td>bioconductor-pd.feinberg.mm8.me.hx1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2296</th>\n",
       "      <td>bioconductor-pd.felgene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2297</th>\n",
       "      <td>bioconductor-pd.felgene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2298</th>\n",
       "      <td>bioconductor-pd.fingene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2299</th>\n",
       "      <td>bioconductor-pd.fingene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2300</th>\n",
       "      <td>bioconductor-pd.genomewidesnp.5</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2301</th>\n",
       "      <td>bioconductor-pd.genomewidesnp.6</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2302</th>\n",
       "      <td>bioconductor-pd.guigene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2303</th>\n",
       "      <td>bioconductor-pd.guigene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2304</th>\n",
       "      <td>bioconductor-pd.hc.g110</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2305</th>\n",
       "      <td>bioconductor-pd.hg18.60mer.expr</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2306</th>\n",
       "      <td>bioconductor-pd.hg.focus</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2307</th>\n",
       "      <td>bioconductor-pd.hg.u133a.2</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2308</th>\n",
       "      <td>bioconductor-pd.hg.u133a</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2309</th>\n",
       "      <td>bioconductor-pd.hg.u133a.tag</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2310</th>\n",
       "      <td>bioconductor-pd.hg.u133b</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2311</th>\n",
       "      <td>bioconductor-pd.hg.u133.plus.2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2312</th>\n",
       "      <td>bioconductor-pd.hg.u219</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2313</th>\n",
       "      <td>bioconductor-pd.hg.u95a</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2314</th>\n",
       "      <td>bioconductor-pd.hg.u95av2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2315</th>\n",
       "      <td>bioconductor-pd.hg.u95b</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2316</th>\n",
       "      <td>bioconductor-pd.hg.u95c</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2317</th>\n",
       "      <td>bioconductor-pd.hg.u95d</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2318</th>\n",
       "      <td>bioconductor-pd.hg.u95e</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2319</th>\n",
       "      <td>bioconductor-pd.hta.2.0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2320</th>\n",
       "      <td>bioconductor-pd.ht.hg.u133a</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2321</th>\n",
       "      <td>bioconductor-pd.ht.hg.u133.plus.pm</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2322</th>\n",
       "      <td>bioconductor-pd.ht.mg.430a</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2323</th>\n",
       "      <td>bioconductor-pd.hu6800</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2324</th>\n",
       "      <td>bioconductor-pd.huex.1.0.st.v2</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2325</th>\n",
       "      <td>bioconductor-pd.hugene.1.0.st.v1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2326</th>\n",
       "      <td>bioconductor-pd.hugene.1.1.st.v1</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2327</th>\n",
       "      <td>bioconductor-pd.hugene.2.0.st</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2328</th>\n",
       "      <td>bioconductor-pd.hugene.2.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2329</th>\n",
       "      <td>bioconductor-pdinfobuilder</td>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2330</th>\n",
       "      <td>bioconductor-pd.maize</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2331</th>\n",
       "      <td>bioconductor-pd.mapping250k.nsp</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2332</th>\n",
       "      <td>bioconductor-pd.mapping250k.sty</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2333</th>\n",
       "      <td>bioconductor-pd.mapping50k.hind240</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2334</th>\n",
       "      <td>bioconductor-pd.mapping50k.xba240</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2335</th>\n",
       "      <td>bioconductor-pd.margene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2336</th>\n",
       "      <td>bioconductor-pd.margene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2337</th>\n",
       "      <td>bioconductor-pd.medgene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2338</th>\n",
       "      <td>bioconductor-pd.medgene.1.1.st</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2339</th>\n",
       "      <td>bioconductor-pd.medicago</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2340</th>\n",
       "      <td>bioconductor-pd.mg.u74a</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2341</th>\n",
       "      <td>bioconductor-pd.mg.u74av2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2342</th>\n",
       "      <td>bioconductor-pd.mg.u74b</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2343</th>\n",
       "      <td>bioconductor-pd.mg.u74bv2</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2344</th>\n",
       "      <td>bioconductor-pd.mg.u74c</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2345</th>\n",
       "      <td>bioconductor-pd.mg.u74cv2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2346</th>\n",
       "      <td>bioconductor-pd.mirna.1.0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2347</th>\n",
       "      <td>bioconductor-pd.mirna.2.0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2348</th>\n",
       "      <td>bioconductor-pd.mirna.3.0</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2349</th>\n",
       "      <td>bioconductor-pd.mirna.3.1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2350</th>\n",
       "      <td>bioconductor-pd.mirna.4.0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2351</th>\n",
       "      <td>bioconductor-pd.moe430a</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2352</th>\n",
       "      <td>bioconductor-pd.moe430b</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2353</th>\n",
       "      <td>bioconductor-pd.moex.1.0.st.v1</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2354</th>\n",
       "      <td>bioconductor-pd.mogene.1.0.st.v1</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2355</th>\n",
       "      <td>bioconductor-pd.mogene.1.1.st.v1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2356</th>\n",
       "      <td>bioconductor-pd.mogene.2.0.st</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2357</th>\n",
       "      <td>bioconductor-pd.mogene.2.1.st</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2358</th>\n",
       "      <td>bioconductor-pd.mouse430.2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2359</th>\n",
       "      <td>bioconductor-pd.mouse430a.2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2360</th>\n",
       "      <td>bioconductor-pd.mta.1.0</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2361</th>\n",
       "      <td>bioconductor-pd.mu11ksuba</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2362</th>\n",
       "      <td>bioconductor-pd.mu11ksubb</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2363</th>\n",
       "      <td>bioconductor-pd.nugo.hs1a520180</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2364</th>\n",
       "      <td>bioconductor-pd.nugo.mm1a520177</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2365</th>\n",
       "      <td>bioconductor-pd.ovigene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2366</th>\n",
       "      <td>bioconductor-pd.ovigene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2367</th>\n",
       "      <td>bioconductor-pd.pae.g1a</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2368</th>\n",
       "      <td>bioconductor-pd.plasmodium.anopheles</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2369</th>\n",
       "      <td>bioconductor-pd.poplar</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2370</th>\n",
       "      <td>bioconductor-pd.porcine</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2371</th>\n",
       "      <td>bioconductor-pd.porgene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2372</th>\n",
       "      <td>bioconductor-pd.porgene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2373</th>\n",
       "      <td>bioconductor-pd.rabgene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2374</th>\n",
       "      <td>bioconductor-pd.rabgene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2375</th>\n",
       "      <td>bioconductor-pd.rae230a</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2376</th>\n",
       "      <td>bioconductor-pd.rae230b</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2377</th>\n",
       "      <td>bioconductor-pd.raex.1.0.st.v1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2378</th>\n",
       "      <td>bioconductor-pd.ragene.1.0.st.v1</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2379</th>\n",
       "      <td>bioconductor-pd.ragene.1.1.st.v1</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2380</th>\n",
       "      <td>bioconductor-pd.ragene.2.0.st</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2381</th>\n",
       "      <td>bioconductor-pd.ragene.2.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2382</th>\n",
       "      <td>bioconductor-pd.rat230.2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2383</th>\n",
       "      <td>bioconductor-pd.rcngene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2384</th>\n",
       "      <td>bioconductor-pd.rcngene.1.1.st</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2385</th>\n",
       "      <td>bioconductor-pd.rg.u34a</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2386</th>\n",
       "      <td>bioconductor-pd.rg.u34b</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2387</th>\n",
       "      <td>bioconductor-pd.rg.u34c</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2388</th>\n",
       "      <td>bioconductor-pd.rhegene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2389</th>\n",
       "      <td>bioconductor-pd.rhegene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2390</th>\n",
       "      <td>bioconductor-pd.rhesus</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2391</th>\n",
       "      <td>bioconductor-pd.rice</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2392</th>\n",
       "      <td>bioconductor-pd.rjpgene.1.0.st</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2393</th>\n",
       "      <td>bioconductor-pd.rjpgene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2394</th>\n",
       "      <td>bioconductor-pd.rn.u34</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2395</th>\n",
       "      <td>bioconductor-pd.rta.1.0</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2396</th>\n",
       "      <td>bioconductor-pd.rusgene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2397</th>\n",
       "      <td>bioconductor-pd.rusgene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2398</th>\n",
       "      <td>bioconductor-pd.s.aureus</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2399</th>\n",
       "      <td>bioconductor-pd.soybean</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2400</th>\n",
       "      <td>bioconductor-pd.soygene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2401</th>\n",
       "      <td>bioconductor-pd.soygene.1.1.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2402</th>\n",
       "      <td>bioconductor-pd.sugar.cane</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2403</th>\n",
       "      <td>bioconductor-pd.tomato</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2404</th>\n",
       "      <td>bioconductor-pd.u133.x3p</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2405</th>\n",
       "      <td>bioconductor-pd.vitis.vinifera</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2406</th>\n",
       "      <td>bioconductor-pd.wheat</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2407</th>\n",
       "      <td>bioconductor-pd.xenopus.laevis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2408</th>\n",
       "      <td>bioconductor-pd.x.laevis.2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2409</th>\n",
       "      <td>bioconductor-pd.x.tropicalis</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2410</th>\n",
       "      <td>bioconductor-pd.yeast.2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2411</th>\n",
       "      <td>bioconductor-pd.yg.s98</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2412</th>\n",
       "      <td>bioconductor-pd.zebgene.1.0.st</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2413</th>\n",
       "      <td>bioconductor-pd.zebgene.1.1.st</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2414</th>\n",
       "      <td>bioconductor-pd.zebrafish</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2415</th>\n",
       "      <td>bioconductor-peakpanther</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2416</th>\n",
       "      <td>bioconductor-peca</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2417</th>\n",
       "      <td>bioconductor-pedbarrayv10.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2418</th>\n",
       "      <td>bioconductor-pedbarrayv9.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2419</th>\n",
       "      <td>bioconductor-pepdat</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2420</th>\n",
       "      <td>bioconductor-pepsnmr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2421</th>\n",
       "      <td>bioconductor-pepsnmrdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2422</th>\n",
       "      <td>bioconductor-pepstat</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2423</th>\n",
       "      <td>bioconductor-pepxmltab</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2424</th>\n",
       "      <td>bioconductor-perfect</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2425</th>\n",
       "      <td>bioconductor-perturbatr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2426</th>\n",
       "      <td>bioconductor-pfam.db</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2427</th>\n",
       "      <td>bioconductor-pga</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2428</th>\n",
       "      <td>bioconductor-pgca</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2429</th>\n",
       "      <td>bioconductor-pgsea</td>\n",
       "      <td>372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2430</th>\n",
       "      <td>bioconductor-phantasus</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2431</th>\n",
       "      <td>bioconductor-pharmacogx</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2432</th>\n",
       "      <td>bioconductor-phastcons100way.ucsc.hg19</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2433</th>\n",
       "      <td>bioconductor-phastcons100way.ucsc.hg38</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2434</th>\n",
       "      <td>bioconductor-phastcons7way.ucsc.hg38</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2435</th>\n",
       "      <td>bioconductor-phemd</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2436</th>\n",
       "      <td>bioconductor-phenopath</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2437</th>\n",
       "      <td>bioconductor-phenotest</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2438</th>\n",
       "      <td>bioconductor-phenstat</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2439</th>\n",
       "      <td>bioconductor-philr</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2440</th>\n",
       "      <td>bioconductor-phosphonormalizer</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2441</th>\n",
       "      <td>bioconductor-phyloprofile</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2442</th>\n",
       "      <td>bioconductor-phyloprofiledata</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2443</th>\n",
       "      <td>bioconductor-phyloseq</td>\n",
       "      <td>699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2444</th>\n",
       "      <td>bioconductor-pi</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2445</th>\n",
       "      <td>bioconductor-piano</td>\n",
       "      <td>382</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2446</th>\n",
       "      <td>bioconductor-pickgene</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2447</th>\n",
       "      <td>bioconductor-pics</td>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2448</th>\n",
       "      <td>bioconductor-pig.db0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2449</th>\n",
       "      <td>bioconductor-pigengene</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2450</th>\n",
       "      <td>bioconductor-ping</td>\n",
       "      <td>561</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2451</th>\n",
       "      <td>bioconductor-pipeframe</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2452</th>\n",
       "      <td>bioconductor-pkgdeptools</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2453</th>\n",
       "      <td>bioconductor-plasfia</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2454</th>\n",
       "      <td>bioconductor-plasmodiumanophelescdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2455</th>\n",
       "      <td>bioconductor-plasmodiumanophelesprobe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2456</th>\n",
       "      <td>bioconductor-plethy</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2457</th>\n",
       "      <td>bioconductor-plgem</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2458</th>\n",
       "      <td>bioconductor-plier</td>\n",
       "      <td>572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2459</th>\n",
       "      <td>bioconductor-plotgrouper</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2460</th>\n",
       "      <td>bioconductor-plpe</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2461</th>\n",
       "      <td>bioconductor-plrs</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2462</th>\n",
       "      <td>bioconductor-plw</td>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2463</th>\n",
       "      <td>bioconductor-plyranges</td>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2464</th>\n",
       "      <td>bioconductor-pmm</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2465</th>\n",
       "      <td>bioconductor-pocrcannotation.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2466</th>\n",
       "      <td>bioconductor-podkat</td>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2467</th>\n",
       "      <td>bioconductor-pogos</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2468</th>\n",
       "      <td>bioconductor-polyester</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2469</th>\n",
       "      <td>bioconductor-polyfit</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2470</th>\n",
       "      <td>bioconductor-polyphen.hsapiens.dbsnp131</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2471</th>\n",
       "      <td>bioconductor-poplarcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2472</th>\n",
       "      <td>bioconductor-poplarprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2473</th>\n",
       "      <td>bioconductor-porcinecdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2474</th>\n",
       "      <td>bioconductor-porcine.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2475</th>\n",
       "      <td>bioconductor-porcineprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2476</th>\n",
       "      <td>bioconductor-post</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2477</th>\n",
       "      <td>bioconductor-potra</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2478</th>\n",
       "      <td>bioconductor-powerexplorer</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2479</th>\n",
       "      <td>bioconductor-powertcr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2480</th>\n",
       "      <td>bioconductor-ppidata</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2481</th>\n",
       "      <td>bioconductor-ppinfer</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2482</th>\n",
       "      <td>bioconductor-ppistats</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2483</th>\n",
       "      <td>bioconductor-pqsfinder</td>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2484</th>\n",
       "      <td>bioconductor-prada</td>\n",
       "      <td>681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2485</th>\n",
       "      <td>bioconductor-pram</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2486</th>\n",
       "      <td>bioconductor-prebs</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2487</th>\n",
       "      <td>bioconductor-prebsdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2488</th>\n",
       "      <td>bioconductor-precisiontrialdrawer</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2489</th>\n",
       "      <td>bioconductor-preda</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2490</th>\n",
       "      <td>bioconductor-predasampledata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2491</th>\n",
       "      <td>bioconductor-predictionet</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2492</th>\n",
       "      <td>bioconductor-preprocesscore</td>\n",
       "      <td>1617</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2493</th>\n",
       "      <td>bioconductor-primeviewcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2494</th>\n",
       "      <td>bioconductor-primeviewprobe</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2495</th>\n",
       "      <td>bioconductor-primirtss</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2496</th>\n",
       "      <td>bioconductor-prince</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2497</th>\n",
       "      <td>bioconductor-prize</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2498</th>\n",
       "      <td>bioconductor-probamr</td>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2499</th>\n",
       "      <td>bioconductor-probatch</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2500</th>\n",
       "      <td>bioconductor-process</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2501</th>\n",
       "      <td>bioconductor-procoil</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2502</th>\n",
       "      <td>bioconductor-proda</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2503</th>\n",
       "      <td>bioconductor-prodata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2504</th>\n",
       "      <td>bioconductor-profia</td>\n",
       "      <td>558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2505</th>\n",
       "      <td>bioconductor-profileplyr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2506</th>\n",
       "      <td>bioconductor-profilescoredist</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2507</th>\n",
       "      <td>bioconductor-progeny</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2508</th>\n",
       "      <td>bioconductor-projectr</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2509</th>\n",
       "      <td>bioconductor-proloc</td>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2510</th>\n",
       "      <td>bioconductor-prolocdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2511</th>\n",
       "      <td>bioconductor-prolocgui</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2512</th>\n",
       "      <td>bioconductor-promise</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2513</th>\n",
       "      <td>bioconductor-proper</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2514</th>\n",
       "      <td>bioconductor-props</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2515</th>\n",
       "      <td>bioconductor-prostar</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2516</th>\n",
       "      <td>bioconductor-prostatecancercamcap</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2517</th>\n",
       "      <td>bioconductor-prostatecancergrasso</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2518</th>\n",
       "      <td>bioconductor-prostatecancerstockholm</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2519</th>\n",
       "      <td>bioconductor-prostatecancertaylor</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2520</th>\n",
       "      <td>bioconductor-prostatecancervarambally</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2521</th>\n",
       "      <td>bioconductor-proteinprofiles</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2522</th>\n",
       "      <td>bioconductor-proteomicsannotationhubdata</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2523</th>\n",
       "      <td>bioconductor-proteomm</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2524</th>\n",
       "      <td>bioconductor-protgenerics</td>\n",
       "      <td>1300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2525</th>\n",
       "      <td>bioconductor-psea</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2526</th>\n",
       "      <td>bioconductor-psichomics</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2527</th>\n",
       "      <td>bioconductor-psicquic</td>\n",
       "      <td>337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2528</th>\n",
       "      <td>bioconductor-psygenet2r</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2529</th>\n",
       "      <td>bioconductor-pth2o2lipids</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2530</th>\n",
       "      <td>bioconductor-pulsedsilac</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2531</th>\n",
       "      <td>bioconductor-puma</td>\n",
       "      <td>469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2532</th>\n",
       "      <td>bioconductor-pumadata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2533</th>\n",
       "      <td>bioconductor-purecn</td>\n",
       "      <td>609</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2534</th>\n",
       "      <td>bioconductor-pvac</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2535</th>\n",
       "      <td>bioconductor-pvca</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2536</th>\n",
       "      <td>bioconductor-pviz</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2537</th>\n",
       "      <td>bioconductor-pwmenrich</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2538</th>\n",
       "      <td>bioconductor-pwmenrich.dmelanogaster.background</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2539</th>\n",
       "      <td>bioconductor-pwmenrich.hsapiens.background</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2540</th>\n",
       "      <td>bioconductor-pwmenrich.mmusculus.background</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2541</th>\n",
       "      <td>bioconductor-pwomics</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2542</th>\n",
       "      <td>bioconductor-pwrewas</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2543</th>\n",
       "      <td>bioconductor-pwrewas.data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2544</th>\n",
       "      <td>bioconductor-qckitfastq</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2545</th>\n",
       "      <td>bioconductor-qcmetrics</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2546</th>\n",
       "      <td>bioconductor-qdnaseq</td>\n",
       "      <td>523</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2547</th>\n",
       "      <td>bioconductor-qdnaseq.hg19</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2548</th>\n",
       "      <td>bioconductor-qdnaseq.mm10</td>\n",
       "      <td>389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2549</th>\n",
       "      <td>bioconductor-qpcrnorm</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2550</th>\n",
       "      <td>bioconductor-qpgraph</td>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2551</th>\n",
       "      <td>bioconductor-qplexanalyzer</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2552</th>\n",
       "      <td>bioconductor-qplexdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2553</th>\n",
       "      <td>bioconductor-qrqc</td>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2554</th>\n",
       "      <td>bioconductor-qsea</td>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2555</th>\n",
       "      <td>bioconductor-qsmooth</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2556</th>\n",
       "      <td>bioconductor-qsutils</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2557</th>\n",
       "      <td>bioconductor-qtlizer</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2558</th>\n",
       "      <td>bioconductor-qualifier</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2559</th>\n",
       "      <td>bioconductor-quantro</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2560</th>\n",
       "      <td>bioconductor-quantsmooth</td>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2561</th>\n",
       "      <td>bioconductor-quartpac</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2562</th>\n",
       "      <td>bioconductor-quasr</td>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2563</th>\n",
       "      <td>bioconductor-quaternaryprod</td>\n",
       "      <td>482</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2564</th>\n",
       "      <td>bioconductor-qubic</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2565</th>\n",
       "      <td>bioconductor-qubicdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2566</th>\n",
       "      <td>bioconductor-qusage</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2567</th>\n",
       "      <td>bioconductor-qvalue</td>\n",
       "      <td>1201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2568</th>\n",
       "      <td>bioconductor-r10kcod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2569</th>\n",
       "      <td>bioconductor-r3cpet</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2570</th>\n",
       "      <td>bioconductor-r3cseq</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2571</th>\n",
       "      <td>bioconductor-r453plus1toolbox</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2572</th>\n",
       "      <td>bioconductor-r4rna</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2573</th>\n",
       "      <td>bioconductor-rae230acdf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2574</th>\n",
       "      <td>bioconductor-rae230a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2575</th>\n",
       "      <td>bioconductor-rae230aprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2576</th>\n",
       "      <td>bioconductor-rae230bcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2577</th>\n",
       "      <td>bioconductor-rae230b.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2578</th>\n",
       "      <td>bioconductor-rae230bprobe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2579</th>\n",
       "      <td>bioconductor-raex10stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2580</th>\n",
       "      <td>bioconductor-raex10sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2581</th>\n",
       "      <td>bioconductor-raexexonprobesetlocation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2582</th>\n",
       "      <td>bioconductor-ragene10stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2583</th>\n",
       "      <td>bioconductor-ragene10sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2584</th>\n",
       "      <td>bioconductor-ragene10stv1cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2585</th>\n",
       "      <td>bioconductor-ragene10stv1probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2586</th>\n",
       "      <td>bioconductor-ragene11stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2587</th>\n",
       "      <td>bioconductor-ragene11sttranscriptcluster.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2588</th>\n",
       "      <td>bioconductor-ragene20stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2589</th>\n",
       "      <td>bioconductor-ragene20sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2590</th>\n",
       "      <td>bioconductor-ragene21stprobeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2591</th>\n",
       "      <td>bioconductor-ragene21sttranscriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2592</th>\n",
       "      <td>bioconductor-raggedexperiment</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2593</th>\n",
       "      <td>bioconductor-rain</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2594</th>\n",
       "      <td>bioconductor-rama</td>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2595</th>\n",
       "      <td>bioconductor-ramwas</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2596</th>\n",
       "      <td>bioconductor-randomwalkrestartmh</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2597</th>\n",
       "      <td>bioconductor-randpack</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2598</th>\n",
       "      <td>bioconductor-rankprod</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2599</th>\n",
       "      <td>bioconductor-rarevariantvis</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2600</th>\n",
       "      <td>bioconductor-rariant</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2601</th>\n",
       "      <td>bioconductor-rat2302cdf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2602</th>\n",
       "      <td>bioconductor-rat2302.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2603</th>\n",
       "      <td>bioconductor-rat2302frmavecs</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2604</th>\n",
       "      <td>bioconductor-rat2302probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2605</th>\n",
       "      <td>bioconductor-ratchrloc</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2606</th>\n",
       "      <td>bioconductor-rat.db0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2607</th>\n",
       "      <td>bioconductor-rattoxfxcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2608</th>\n",
       "      <td>bioconductor-rattoxfxprobe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2609</th>\n",
       "      <td>bioconductor-rattus.norvegicus</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2610</th>\n",
       "      <td>bioconductor-rbcbook1</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2611</th>\n",
       "      <td>bioconductor-rbgl</td>\n",
       "      <td>1470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2612</th>\n",
       "      <td>bioconductor-rbioinf</td>\n",
       "      <td>563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2613</th>\n",
       "      <td>bioconductor-rbiopaxparser</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2614</th>\n",
       "      <td>bioconductor-rbm</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2615</th>\n",
       "      <td>bioconductor-rbowtie2</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2616</th>\n",
       "      <td>bioconductor-rbowtie</td>\n",
       "      <td>584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2617</th>\n",
       "      <td>bioconductor-rbsurv</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2618</th>\n",
       "      <td>bioconductor-rcade</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2619</th>\n",
       "      <td>bioconductor-rcas</td>\n",
       "      <td>658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2620</th>\n",
       "      <td>bioconductor-rcaspar</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2621</th>\n",
       "      <td>bioconductor-rcellminer</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2622</th>\n",
       "      <td>bioconductor-rcellminerdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2623</th>\n",
       "      <td>bioconductor-rcgh</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2624</th>\n",
       "      <td>bioconductor-rchyoptimyx</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2625</th>\n",
       "      <td>bioconductor-rcistarget</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2626</th>\n",
       "      <td>bioconductor-rcistarget.hg19.motifdbs.cisbponl...</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2627</th>\n",
       "      <td>bioconductor-rcm</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2628</th>\n",
       "      <td>bioconductor-rcpi</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2629</th>\n",
       "      <td>bioconductor-rcy3</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2630</th>\n",
       "      <td>bioconductor-rcyjs</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2631</th>\n",
       "      <td>bioconductor-rdavidwebservice</td>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2632</th>\n",
       "      <td>bioconductor-rdgidb</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2633</th>\n",
       "      <td>bioconductor-rdisop</td>\n",
       "      <td>367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2634</th>\n",
       "      <td>bioconductor-rdrtoolbox</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2635</th>\n",
       "      <td>bioconductor-reactome.db</td>\n",
       "      <td>330</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2636</th>\n",
       "      <td>bioconductor-reactomegsa</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2637</th>\n",
       "      <td>bioconductor-reactomegsa.data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2638</th>\n",
       "      <td>bioconductor-reactomepa</td>\n",
       "      <td>266</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2639</th>\n",
       "      <td>bioconductor-readqpcr</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2640</th>\n",
       "      <td>bioconductor-reb</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2641</th>\n",
       "      <td>bioconductor-rebet</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2642</th>\n",
       "      <td>bioconductor-recount</td>\n",
       "      <td>317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2643</th>\n",
       "      <td>bioconductor-recoup</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2644</th>\n",
       "      <td>bioconductor-reder</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2645</th>\n",
       "      <td>bioconductor-redseq</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2646</th>\n",
       "      <td>bioconductor-refnet</td>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2647</th>\n",
       "      <td>bioconductor-refplus</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2648</th>\n",
       "      <td>bioconductor-regioner</td>\n",
       "      <td>764</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2649</th>\n",
       "      <td>bioconductor-regionreport</td>\n",
       "      <td>60</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2650</th>\n",
       "      <td>bioconductor-regparallel</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2651</th>\n",
       "      <td>bioconductor-regsplice</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2652</th>\n",
       "      <td>bioconductor-remp</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2653</th>\n",
       "      <td>bioconductor-repitools</td>\n",
       "      <td>673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2654</th>\n",
       "      <td>bioconductor-reportingtools</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2655</th>\n",
       "      <td>bioconductor-repviz</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2656</th>\n",
       "      <td>bioconductor-reqon</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2657</th>\n",
       "      <td>bioconductor-restfulse</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2658</th>\n",
       "      <td>bioconductor-restfulsedata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2659</th>\n",
       "      <td>bioconductor-rexposome</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2660</th>\n",
       "      <td>bioconductor-rforproteomics</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2661</th>\n",
       "      <td>bioconductor-rfpred</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2662</th>\n",
       "      <td>bioconductor-rgadem</td>\n",
       "      <td>657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2663</th>\n",
       "      <td>bioconductor-rgalaxy</td>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2664</th>\n",
       "      <td>bioconductor-rgin</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2665</th>\n",
       "      <td>bioconductor-rgmql</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2666</th>\n",
       "      <td>bioconductor-rgmqllib</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2667</th>\n",
       "      <td>bioconductor-rgraph2js</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2668</th>\n",
       "      <td>bioconductor-rgraphviz</td>\n",
       "      <td>1163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2669</th>\n",
       "      <td>bioconductor-rgreat</td>\n",
       "      <td>306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2670</th>\n",
       "      <td>bioconductor-rgsea</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2671</th>\n",
       "      <td>bioconductor-rgsepd</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2672</th>\n",
       "      <td>bioconductor-rgu34acdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2673</th>\n",
       "      <td>bioconductor-rgu34a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2674</th>\n",
       "      <td>bioconductor-rgu34aprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2675</th>\n",
       "      <td>bioconductor-rgu34bcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2676</th>\n",
       "      <td>bioconductor-rgu34b.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2677</th>\n",
       "      <td>bioconductor-rgu34bprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2678</th>\n",
       "      <td>bioconductor-rgu34ccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2679</th>\n",
       "      <td>bioconductor-rgu34c.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2680</th>\n",
       "      <td>bioconductor-rgu34cprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2681</th>\n",
       "      <td>bioconductor-rguatlas4k.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2682</th>\n",
       "      <td>bioconductor-rgug4105a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2683</th>\n",
       "      <td>bioconductor-rgug4130a.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2684</th>\n",
       "      <td>bioconductor-rgug4131a.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2685</th>\n",
       "      <td>bioconductor-rhdf5</td>\n",
       "      <td>2665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2686</th>\n",
       "      <td>bioconductor-rhdf5client</td>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2687</th>\n",
       "      <td>bioconductor-rhdf5lib</td>\n",
       "      <td>2426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2688</th>\n",
       "      <td>bioconductor-rhesuscdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2689</th>\n",
       "      <td>bioconductor-rhesus.db0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2690</th>\n",
       "      <td>bioconductor-rhesusprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2691</th>\n",
       "      <td>bioconductor-rheumaticconditionwollbold</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2692</th>\n",
       "      <td>bioconductor-rhisat2</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2693</th>\n",
       "      <td>bioconductor-rhtslib</td>\n",
       "      <td>7232</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2694</th>\n",
       "      <td>bioconductor-ri16cod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2695</th>\n",
       "      <td>bioconductor-riboprofiling</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2696</th>\n",
       "      <td>bioconductor-riboseqr</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2697</th>\n",
       "      <td>bioconductor-ribosomaldatabaseproject11.5mgdb</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2698</th>\n",
       "      <td>bioconductor-ricecdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2699</th>\n",
       "      <td>bioconductor-riceprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2700</th>\n",
       "      <td>bioconductor-rimmport</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2701</th>\n",
       "      <td>bioconductor-ringo</td>\n",
       "      <td>676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2702</th>\n",
       "      <td>bioconductor-risa</td>\n",
       "      <td>479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2703</th>\n",
       "      <td>bioconductor-ritan</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2704</th>\n",
       "      <td>bioconductor-ritandata</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2705</th>\n",
       "      <td>bioconductor-river</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2706</th>\n",
       "      <td>bioconductor-rjmcmcnucleosomes</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2707</th>\n",
       "      <td>bioconductor-rlmm</td>\n",
       "      <td>301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2708</th>\n",
       "      <td>bioconductor-rmagpie</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2709</th>\n",
       "      <td>bioconductor-rmassbank</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2710</th>\n",
       "      <td>bioconductor-rmassbankdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2711</th>\n",
       "      <td>bioconductor-rmelting</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2712</th>\n",
       "      <td>bioconductor-rmir</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2713</th>\n",
       "      <td>bioconductor-rmir.hsa</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2714</th>\n",
       "      <td>bioconductor-rmir.hs.mirna</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2715</th>\n",
       "      <td>bioconductor-rmmquant</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2716</th>\n",
       "      <td>bioconductor-rnadecay</td>\n",
       "      <td>369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2717</th>\n",
       "      <td>bioconductor-rnagilentdesign028282.db</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2718</th>\n",
       "      <td>bioconductor-rnainteract</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2719</th>\n",
       "      <td>bioconductor-rnainteractmapk</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2720</th>\n",
       "      <td>bioconductor-rnaither</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2721</th>\n",
       "      <td>bioconductor-rnamodr</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2722</th>\n",
       "      <td>bioconductor-rnamodr.alkanilineseq</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2723</th>\n",
       "      <td>bioconductor-rnamodr.data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2724</th>\n",
       "      <td>bioconductor-rnamodr.ml</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2725</th>\n",
       "      <td>bioconductor-rnamodr.ribomethseq</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2726</th>\n",
       "      <td>bioconductor-rnaprobr</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2727</th>\n",
       "      <td>bioconductor-rnasense</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2728</th>\n",
       "      <td>bioconductor-rnaseqcomp</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2729</th>\n",
       "      <td>bioconductor-rnaseqdata.hnrnpc.bam.chr14</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2730</th>\n",
       "      <td>bioconductor-rnaseqmap</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2731</th>\n",
       "      <td>bioconductor-rnaseqpower</td>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2732</th>\n",
       "      <td>bioconductor-rnaseqr</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2733</th>\n",
       "      <td>bioconductor-rnaseqrdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2734</th>\n",
       "      <td>bioconductor-rnaseqsamplesizedata</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2735</th>\n",
       "      <td>bioconductor-rnbeads</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2736</th>\n",
       "      <td>bioconductor-rnbeads.hg19</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2737</th>\n",
       "      <td>bioconductor-rnbeads.hg38</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2738</th>\n",
       "      <td>bioconductor-rnbeads.mm10</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2739</th>\n",
       "      <td>bioconductor-rnbeads.mm9</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2740</th>\n",
       "      <td>bioconductor-rnbeads.rn5</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2741</th>\n",
       "      <td>bioconductor-rnits</td>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2742</th>\n",
       "      <td>bioconductor-rnu34cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2743</th>\n",
       "      <td>bioconductor-rnu34.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2744</th>\n",
       "      <td>bioconductor-rnu34probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2745</th>\n",
       "      <td>bioconductor-roar</td>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2746</th>\n",
       "      <td>bioconductor-roberts2005annotation.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2747</th>\n",
       "      <td>bioconductor-roc</td>\n",
       "      <td>577</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2748</th>\n",
       "      <td>bioconductor-roleswitch</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2749</th>\n",
       "      <td>bioconductor-rols</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2750</th>\n",
       "      <td>bioconductor-rontotools</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2751</th>\n",
       "      <td>bioconductor-ropls</td>\n",
       "      <td>814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2752</th>\n",
       "      <td>bioconductor-rots</td>\n",
       "      <td>751</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2753</th>\n",
       "      <td>bioconductor-rpa</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2754</th>\n",
       "      <td>bioconductor-rprotobuflib</td>\n",
       "      <td>496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2755</th>\n",
       "      <td>bioconductor-rpsixml</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2756</th>\n",
       "      <td>bioconductor-rpx</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2757</th>\n",
       "      <td>bioconductor-rqc</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2758</th>\n",
       "      <td>bioconductor-rqt</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2759</th>\n",
       "      <td>bioconductor-rqubic</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2760</th>\n",
       "      <td>bioconductor-rrbsdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2761</th>\n",
       "      <td>bioconductor-rrdp</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2762</th>\n",
       "      <td>bioconductor-rrdpdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2763</th>\n",
       "      <td>bioconductor-rrho</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2764</th>\n",
       "      <td>bioconductor-rsamtools</td>\n",
       "      <td>10490</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2765</th>\n",
       "      <td>bioconductor-rscudo</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2766</th>\n",
       "      <td>bioconductor-rseqan</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2767</th>\n",
       "      <td>bioconductor-rsubread</td>\n",
       "      <td>1437</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2768</th>\n",
       "      <td>bioconductor-rsvsim</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2769</th>\n",
       "      <td>bioconductor-rta10probeset.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2770</th>\n",
       "      <td>bioconductor-rta10transcriptcluster.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2771</th>\n",
       "      <td>bioconductor-rtandem</td>\n",
       "      <td>386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2772</th>\n",
       "      <td>bioconductor-rtca</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2773</th>\n",
       "      <td>bioconductor-rtcga</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2774</th>\n",
       "      <td>bioconductor-rtcga.clinical</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2775</th>\n",
       "      <td>bioconductor-rtcga.cnv</td>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2776</th>\n",
       "      <td>bioconductor-rtcga.methylation</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2777</th>\n",
       "      <td>bioconductor-rtcga.mirnaseq</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2778</th>\n",
       "      <td>bioconductor-rtcga.mrna</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2779</th>\n",
       "      <td>bioconductor-rtcga.mutations</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2780</th>\n",
       "      <td>bioconductor-rtcga.pancan12</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2781</th>\n",
       "      <td>bioconductor-rtcga.rnaseq</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2782</th>\n",
       "      <td>bioconductor-rtcga.rppa</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2783</th>\n",
       "      <td>bioconductor-rtcgatoolbox</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2784</th>\n",
       "      <td>bioconductor-rtn</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2785</th>\n",
       "      <td>bioconductor-rtnduals</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2786</th>\n",
       "      <td>bioconductor-rtnsurvival</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2787</th>\n",
       "      <td>bioconductor-rtopper</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2788</th>\n",
       "      <td>bioconductor-rtracklayer</td>\n",
       "      <td>8090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2789</th>\n",
       "      <td>bioconductor-rtreemix</td>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2790</th>\n",
       "      <td>bioconductor-rtrm</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2791</th>\n",
       "      <td>bioconductor-rtrmui</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2792</th>\n",
       "      <td>bioconductor-rtu34cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2793</th>\n",
       "      <td>bioconductor-rtu34.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2794</th>\n",
       "      <td>bioconductor-rtu34probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2795</th>\n",
       "      <td>bioconductor-runibic</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2796</th>\n",
       "      <td>bioconductor-ruvcorr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2797</th>\n",
       "      <td>bioconductor-ruvnormalize</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2798</th>\n",
       "      <td>bioconductor-ruvnormalizedata</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2799</th>\n",
       "      <td>bioconductor-ruvseq</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2800</th>\n",
       "      <td>bioconductor-rvs</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2801</th>\n",
       "      <td>bioconductor-rwgcod.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2802</th>\n",
       "      <td>bioconductor-rwikipathways</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2803</th>\n",
       "      <td>bioconductor-s4vectors</td>\n",
       "      <td>13820</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2804</th>\n",
       "      <td>bioconductor-safe</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2805</th>\n",
       "      <td>bioconductor-sagenhaft</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2806</th>\n",
       "      <td>bioconductor-sagx</td>\n",
       "      <td>557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2807</th>\n",
       "      <td>bioconductor-saigegds</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2808</th>\n",
       "      <td>bioconductor-samexplorer</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2809</th>\n",
       "      <td>bioconductor-sampleclassifier</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2810</th>\n",
       "      <td>bioconductor-sampleclassifierdata</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2811</th>\n",
       "      <td>bioconductor-samspectral</td>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2812</th>\n",
       "      <td>bioconductor-sangerseqr</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2813</th>\n",
       "      <td>bioconductor-sapfinder</td>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2814</th>\n",
       "      <td>bioconductor-saureuscdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2815</th>\n",
       "      <td>bioconductor-saureusprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2816</th>\n",
       "      <td>bioconductor-savr</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2817</th>\n",
       "      <td>bioconductor-sbgnview</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2818</th>\n",
       "      <td>bioconductor-sbgnview.data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2819</th>\n",
       "      <td>bioconductor-sbmlr</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2820</th>\n",
       "      <td>bioconductor-sc3</td>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2821</th>\n",
       "      <td>bioconductor-scale4c</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2822</th>\n",
       "      <td>bioconductor-scalign</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2823</th>\n",
       "      <td>bioconductor-scan.upc</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2824</th>\n",
       "      <td>bioconductor-scanvis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2825</th>\n",
       "      <td>bioconductor-scater</td>\n",
       "      <td>1055</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2826</th>\n",
       "      <td>bioconductor-scbfa</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2827</th>\n",
       "      <td>bioconductor-scbn</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2828</th>\n",
       "      <td>bioconductor-scdblfinder</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2829</th>\n",
       "      <td>bioconductor-scdd</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2830</th>\n",
       "      <td>bioconductor-scde</td>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2831</th>\n",
       "      <td>bioconductor-scds</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2832</th>\n",
       "      <td>bioconductor-scfeaturefilter</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2833</th>\n",
       "      <td>bioconductor-scfind</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2834</th>\n",
       "      <td>bioconductor-scgps</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2835</th>\n",
       "      <td>bioconductor-schex</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2836</th>\n",
       "      <td>bioconductor-scisi</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2837</th>\n",
       "      <td>bioconductor-sclcbam</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2838</th>\n",
       "      <td>bioconductor-scmap</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2839</th>\n",
       "      <td>bioconductor-scmerge</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2840</th>\n",
       "      <td>bioconductor-scmeth</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2841</th>\n",
       "      <td>bioconductor-scnorm</td>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2842</th>\n",
       "      <td>bioconductor-scone</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2843</th>\n",
       "      <td>bioconductor-sconify</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2844</th>\n",
       "      <td>bioconductor-scoreinvhap</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2845</th>\n",
       "      <td>bioconductor-scpca</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2846</th>\n",
       "      <td>bioconductor-scpipe</td>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2847</th>\n",
       "      <td>bioconductor-scran</td>\n",
       "      <td>954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2848</th>\n",
       "      <td>bioconductor-screcover</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2849</th>\n",
       "      <td>bioconductor-scrnaseq</td>\n",
       "      <td>111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2850</th>\n",
       "      <td>bioconductor-scruff</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2851</th>\n",
       "      <td>bioconductor-scsr</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2852</th>\n",
       "      <td>bioconductor-sctensor</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2853</th>\n",
       "      <td>bioconductor-sctgif</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2854</th>\n",
       "      <td>bioconductor-sdams</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2855</th>\n",
       "      <td>bioconductor-segmentseq</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2856</th>\n",
       "      <td>bioconductor-selex</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2857</th>\n",
       "      <td>bioconductor-semdist</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2858</th>\n",
       "      <td>bioconductor-semisup</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2859</th>\n",
       "      <td>bioconductor-sepira</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2860</th>\n",
       "      <td>bioconductor-seq2pathway</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2861</th>\n",
       "      <td>bioconductor-seq2pathway.data</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2862</th>\n",
       "      <td>bioconductor-seqarray</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2863</th>\n",
       "      <td>bioconductor-seqbias</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2864</th>\n",
       "      <td>bioconductor-seqc</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2865</th>\n",
       "      <td>bioconductor-seqcat</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2866</th>\n",
       "      <td>bioconductor-seqcna</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2867</th>\n",
       "      <td>bioconductor-seqcna.annot</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2868</th>\n",
       "      <td>bioconductor-seqcombo</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2869</th>\n",
       "      <td>bioconductor-seqgsea</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2870</th>\n",
       "      <td>bioconductor-seqlogo</td>\n",
       "      <td>588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2871</th>\n",
       "      <td>bioconductor-seqpattern</td>\n",
       "      <td>576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2872</th>\n",
       "      <td>bioconductor-seqplots</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2873</th>\n",
       "      <td>bioconductor-seqsetvis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2874</th>\n",
       "      <td>bioconductor-seqsqc</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2875</th>\n",
       "      <td>bioconductor-seqtools</td>\n",
       "      <td>583</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2876</th>\n",
       "      <td>bioconductor-seqvartools</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2877</th>\n",
       "      <td>bioconductor-serumstimulation</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2878</th>\n",
       "      <td>bioconductor-sesame</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2879</th>\n",
       "      <td>bioconductor-sesamedata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2880</th>\n",
       "      <td>bioconductor-setools</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2881</th>\n",
       "      <td>bioconductor-sevenbridges</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2882</th>\n",
       "      <td>bioconductor-sevenc</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2883</th>\n",
       "      <td>bioconductor-seventygenedata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2884</th>\n",
       "      <td>bioconductor-sgseq</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2885</th>\n",
       "      <td>bioconductor-sharedobject</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2886</th>\n",
       "      <td>bioconductor-shdz.db</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2887</th>\n",
       "      <td>bioconductor-shinymethyl</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2888</th>\n",
       "      <td>bioconductor-shinymethyldata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2889</th>\n",
       "      <td>bioconductor-shinytandem</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2890</th>\n",
       "      <td>bioconductor-shortread</td>\n",
       "      <td>3404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2891</th>\n",
       "      <td>bioconductor-siamcat</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2892</th>\n",
       "      <td>bioconductor-sictools</td>\n",
       "      <td>273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2893</th>\n",
       "      <td>bioconductor-sift.hsapiens.dbsnp132</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2894</th>\n",
       "      <td>bioconductor-sift.hsapiens.dbsnp137</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2895</th>\n",
       "      <td>bioconductor-sigar</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2896</th>\n",
       "      <td>bioconductor-sigcheck</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2897</th>\n",
       "      <td>bioconductor-sigfeature</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2898</th>\n",
       "      <td>bioconductor-sigfuge</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2899</th>\n",
       "      <td>bioconductor-siggenes</td>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2900</th>\n",
       "      <td>bioconductor-sights</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2901</th>\n",
       "      <td>bioconductor-signaturesearch</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2902</th>\n",
       "      <td>bioconductor-signaturesearchdata</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2903</th>\n",
       "      <td>bioconductor-signer</td>\n",
       "      <td>651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2904</th>\n",
       "      <td>bioconductor-signet</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2905</th>\n",
       "      <td>bioconductor-sigpathway</td>\n",
       "      <td>562</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2906</th>\n",
       "      <td>bioconductor-sigspack</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2907</th>\n",
       "      <td>bioconductor-sigsquared</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2908</th>\n",
       "      <td>bioconductor-silva128.1mgdb</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2909</th>\n",
       "      <td>bioconductor-sim</td>\n",
       "      <td>464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2910</th>\n",
       "      <td>bioconductor-simat</td>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2911</th>\n",
       "      <td>bioconductor-simbindprofiles</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2912</th>\n",
       "      <td>bioconductor-simd</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2913</th>\n",
       "      <td>bioconductor-similarpeak</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2914</th>\n",
       "      <td>bioconductor-simlr</td>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2915</th>\n",
       "      <td>bioconductor-simpintlists</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2916</th>\n",
       "      <td>bioconductor-simpleaffy</td>\n",
       "      <td>569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2917</th>\n",
       "      <td>bioconductor-simulatorz</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2918</th>\n",
       "      <td>bioconductor-sincell</td>\n",
       "      <td>573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2919</th>\n",
       "      <td>bioconductor-singlecellexperiment</td>\n",
       "      <td>871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2920</th>\n",
       "      <td>bioconductor-singlecellexperiment-scripts</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2921</th>\n",
       "      <td>bioconductor-singlecelltk</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2922</th>\n",
       "      <td>bioconductor-single.mtec.transcriptomes</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2923</th>\n",
       "      <td>bioconductor-singler</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2924</th>\n",
       "      <td>bioconductor-singscore</td>\n",
       "      <td>29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2925</th>\n",
       "      <td>bioconductor-sispa</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2926</th>\n",
       "      <td>bioconductor-sitepath</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2927</th>\n",
       "      <td>bioconductor-sizepower</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2928</th>\n",
       "      <td>bioconductor-skewr</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2929</th>\n",
       "      <td>bioconductor-slalom</td>\n",
       "      <td>377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2930</th>\n",
       "      <td>bioconductor-slgi</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2931</th>\n",
       "      <td>bioconductor-slingshot</td>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2932</th>\n",
       "      <td>bioconductor-slinky</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2933</th>\n",
       "      <td>bioconductor-slqpcr</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2934</th>\n",
       "      <td>bioconductor-smad</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2935</th>\n",
       "      <td>bioconductor-smap</td>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2936</th>\n",
       "      <td>bioconductor-smite</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2937</th>\n",
       "      <td>bioconductor-snadata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2938</th>\n",
       "      <td>bioconductor-snagee</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2939</th>\n",
       "      <td>bioconductor-snageedata</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2940</th>\n",
       "      <td>bioconductor-snapcgh</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2941</th>\n",
       "      <td>bioconductor-snm</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2942</th>\n",
       "      <td>bioconductor-snpediar</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2943</th>\n",
       "      <td>bioconductor-snphood</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2944</th>\n",
       "      <td>bioconductor-snphooddata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2945</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp141.grch38</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2946</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp142.grch37</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2947</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp144.grch37</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2948</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp144.grch38</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2949</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp149.grch38</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2950</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp150.grch38</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2951</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp151.grch38</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2952</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp.20101109</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2953</th>\n",
       "      <td>bioconductor-snplocs.hsapiens.dbsnp.20120608</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2954</th>\n",
       "      <td>bioconductor-snprelate</td>\n",
       "      <td>584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2955</th>\n",
       "      <td>bioconductor-snpstats</td>\n",
       "      <td>588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2956</th>\n",
       "      <td>bioconductor-soggi</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2957</th>\n",
       "      <td>bioconductor-sojourner</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2958</th>\n",
       "      <td>bioconductor-somaticadata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2959</th>\n",
       "      <td>bioconductor-somaticcanceralterations</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2960</th>\n",
       "      <td>bioconductor-somaticsignatures</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2961</th>\n",
       "      <td>bioconductor-soybeancdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2962</th>\n",
       "      <td>bioconductor-soybeanprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2963</th>\n",
       "      <td>bioconductor-spacepac</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2964</th>\n",
       "      <td>bioconductor-spaniel</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2965</th>\n",
       "      <td>bioconductor-sparsedossa</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2966</th>\n",
       "      <td>bioconductor-sparsenetgls</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2967</th>\n",
       "      <td>bioconductor-sparsesignatures</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2968</th>\n",
       "      <td>bioconductor-spatialcpie</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2969</th>\n",
       "      <td>bioconductor-specl</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2970</th>\n",
       "      <td>bioconductor-specond</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2971</th>\n",
       "      <td>bioconductor-spectraltad</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2972</th>\n",
       "      <td>bioconductor-spem</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2973</th>\n",
       "      <td>bioconductor-spia</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2974</th>\n",
       "      <td>bioconductor-spidermir</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2975</th>\n",
       "      <td>bioconductor-spikein</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2976</th>\n",
       "      <td>bioconductor-spikeinsubset</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2977</th>\n",
       "      <td>bioconductor-spikeli</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2978</th>\n",
       "      <td>bioconductor-spktools</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2979</th>\n",
       "      <td>bioconductor-splatter</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2980</th>\n",
       "      <td>bioconductor-splicinggraphs</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2981</th>\n",
       "      <td>bioconductor-splinetimer</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2982</th>\n",
       "      <td>bioconductor-splinter</td>\n",
       "      <td>303</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2983</th>\n",
       "      <td>bioconductor-splots</td>\n",
       "      <td>313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2984</th>\n",
       "      <td>bioconductor-sponge</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2985</th>\n",
       "      <td>bioconductor-spotsegmentation</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2986</th>\n",
       "      <td>bioconductor-sqldataframe</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2987</th>\n",
       "      <td>bioconductor-squadd</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2988</th>\n",
       "      <td>bioconductor-sracipe</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2989</th>\n",
       "      <td>bioconductor-sradb</td>\n",
       "      <td>481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2990</th>\n",
       "      <td>bioconductor-srap</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2991</th>\n",
       "      <td>bioconductor-srgnet</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2992</th>\n",
       "      <td>bioconductor-srnadiff</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2993</th>\n",
       "      <td>bioconductor-sscore</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2994</th>\n",
       "      <td>bioconductor-sscu</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2995</th>\n",
       "      <td>bioconductor-sseq</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2996</th>\n",
       "      <td>bioconductor-ssize</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2997</th>\n",
       "      <td>bioconductor-sspa</td>\n",
       "      <td>673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2998</th>\n",
       "      <td>bioconductor-sspaths</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2999</th>\n",
       "      <td>bioconductor-ssrch</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3000</th>\n",
       "      <td>bioconductor-ssviz</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3001</th>\n",
       "      <td>bioconductor-stager</td>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3002</th>\n",
       "      <td>bioconductor-stan</td>\n",
       "      <td>482</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3003</th>\n",
       "      <td>bioconductor-starank</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3004</th>\n",
       "      <td>bioconductor-starbiotrek</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3005</th>\n",
       "      <td>bioconductor-starr</td>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3006</th>\n",
       "      <td>bioconductor-stategra</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3007</th>\n",
       "      <td>bioconductor-stattarget</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3008</th>\n",
       "      <td>bioconductor-stemhypoxia</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3009</th>\n",
       "      <td>bioconductor-stepnorm</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3010</th>\n",
       "      <td>bioconductor-stjudem</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3011</th>\n",
       "      <td>bioconductor-strandcheckr</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3012</th>\n",
       "      <td>bioconductor-streamer</td>\n",
       "      <td>568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3013</th>\n",
       "      <td>bioconductor-stringdb</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3014</th>\n",
       "      <td>bioconductor-stroma4</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3015</th>\n",
       "      <td>bioconductor-structstrings</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3016</th>\n",
       "      <td>bioconductor-structuralvariantannotation</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3017</th>\n",
       "      <td>bioconductor-subcellbarcode</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3018</th>\n",
       "      <td>bioconductor-subseq</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3019</th>\n",
       "      <td>bioconductor-sugarcanecdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3020</th>\n",
       "      <td>bioconductor-sugarcaneprobe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3021</th>\n",
       "      <td>bioconductor-summarizedbenchmark</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3022</th>\n",
       "      <td>bioconductor-summarizedexperiment</td>\n",
       "      <td>11621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3023</th>\n",
       "      <td>bioconductor-suprahex</td>\n",
       "      <td>321</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3024</th>\n",
       "      <td>bioconductor-survcomp</td>\n",
       "      <td>485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3025</th>\n",
       "      <td>bioconductor-survtype</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3026</th>\n",
       "      <td>bioconductor-sushi</td>\n",
       "      <td>311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3027</th>\n",
       "      <td>bioconductor-sva</td>\n",
       "      <td>1253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3028</th>\n",
       "      <td>bioconductor-svaplsseq</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3029</th>\n",
       "      <td>bioconductor-svm2crmdata</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3030</th>\n",
       "      <td>bioconductor-swath2stats</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3031</th>\n",
       "      <td>bioconductor-swathxtend</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3032</th>\n",
       "      <td>bioconductor-swfdr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3033</th>\n",
       "      <td>bioconductor-swimr</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3034</th>\n",
       "      <td>bioconductor-switchbox</td>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3035</th>\n",
       "      <td>bioconductor-switchde</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3036</th>\n",
       "      <td>bioconductor-synapter</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3037</th>\n",
       "      <td>bioconductor-synapterdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3038</th>\n",
       "      <td>bioconductor-synergyfinder</td>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3039</th>\n",
       "      <td>bioconductor-synlet</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3040</th>\n",
       "      <td>bioconductor-synmut</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3041</th>\n",
       "      <td>bioconductor-systempiper</td>\n",
       "      <td>615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3042</th>\n",
       "      <td>bioconductor-systempiperdata</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3043</th>\n",
       "      <td>bioconductor-tabulamurisdata</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3044</th>\n",
       "      <td>bioconductor-target</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3045</th>\n",
       "      <td>bioconductor-targetscan.hs.eg.db</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3046</th>\n",
       "      <td>bioconductor-targetscan.mm.eg.db</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3047</th>\n",
       "      <td>bioconductor-targetscore</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3048</th>\n",
       "      <td>bioconductor-targetscoredata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3049</th>\n",
       "      <td>bioconductor-targetsearch</td>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3050</th>\n",
       "      <td>bioconductor-targetsearchdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3051</th>\n",
       "      <td>bioconductor-tarseqqc</td>\n",
       "      <td>372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3052</th>\n",
       "      <td>bioconductor-tartare</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3053</th>\n",
       "      <td>bioconductor-tbx20bamsubset</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3054</th>\n",
       "      <td>bioconductor-tcc</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3055</th>\n",
       "      <td>bioconductor-tcgabiolinks</td>\n",
       "      <td>219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3056</th>\n",
       "      <td>bioconductor-tcgabiolinksgui</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3057</th>\n",
       "      <td>bioconductor-tcgabiolinksgui.data</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3058</th>\n",
       "      <td>bioconductor-tcgacrcmirna</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3059</th>\n",
       "      <td>bioconductor-tcgacrcmrna</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3060</th>\n",
       "      <td>bioconductor-tcgamethylation450k</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3061</th>\n",
       "      <td>bioconductor-tcgautils</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3062</th>\n",
       "      <td>bioconductor-tcgawgbsdata.hg19</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3063</th>\n",
       "      <td>bioconductor-tcgaworkflowdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3064</th>\n",
       "      <td>bioconductor-tcseq</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3065</th>\n",
       "      <td>bioconductor-tdaracne</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3066</th>\n",
       "      <td>bioconductor-tenxbraindata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3067</th>\n",
       "      <td>bioconductor-tenxbusdata</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3068</th>\n",
       "      <td>bioconductor-tenxpbmcdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3069</th>\n",
       "      <td>bioconductor-tenxplore</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3070</th>\n",
       "      <td>bioconductor-teqc</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3071</th>\n",
       "      <td>bioconductor-ternarynet</td>\n",
       "      <td>559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3072</th>\n",
       "      <td>bioconductor-test1cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3073</th>\n",
       "      <td>bioconductor-test2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3074</th>\n",
       "      <td>bioconductor-test3cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3075</th>\n",
       "      <td>bioconductor-test3probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3076</th>\n",
       "      <td>bioconductor-tfarm</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3077</th>\n",
       "      <td>bioconductor-tfbstools</td>\n",
       "      <td>769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3078</th>\n",
       "      <td>bioconductor-tfea.chip</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3079</th>\n",
       "      <td>bioconductor-tfhaz</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3080</th>\n",
       "      <td>bioconductor-tfutils</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3081</th>\n",
       "      <td>bioconductor-tigre</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3082</th>\n",
       "      <td>bioconductor-tilingarray</td>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3083</th>\n",
       "      <td>bioconductor-timecourse</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3084</th>\n",
       "      <td>bioconductor-timerquant</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3085</th>\n",
       "      <td>bioconductor-timescape</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3086</th>\n",
       "      <td>bioconductor-timeseriesexperiment</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3087</th>\n",
       "      <td>bioconductor-tin</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3088</th>\n",
       "      <td>bioconductor-tinesath1cdf</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3089</th>\n",
       "      <td>bioconductor-tinesath1probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3090</th>\n",
       "      <td>bioconductor-tissueenrich</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3091</th>\n",
       "      <td>bioconductor-tissuetreg</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3092</th>\n",
       "      <td>bioconductor-titancna</td>\n",
       "      <td>1076</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3093</th>\n",
       "      <td>bioconductor-tkwidgets</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3094</th>\n",
       "      <td>bioconductor-tmixclust</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3095</th>\n",
       "      <td>bioconductor-tnbc.cms</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3096</th>\n",
       "      <td>bioconductor-tnt</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3097</th>\n",
       "      <td>bioconductor-toast</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3098</th>\n",
       "      <td>bioconductor-tofsims</td>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3099</th>\n",
       "      <td>bioconductor-tofsimsdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3100</th>\n",
       "      <td>bioconductor-tomatocdf</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3101</th>\n",
       "      <td>bioconductor-tomatoprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3102</th>\n",
       "      <td>bioconductor-topaseq</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3103</th>\n",
       "      <td>bioconductor-topconfects</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3104</th>\n",
       "      <td>bioconductor-topdownr</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3105</th>\n",
       "      <td>bioconductor-topdownrdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3106</th>\n",
       "      <td>bioconductor-topgo</td>\n",
       "      <td>644</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3107</th>\n",
       "      <td>bioconductor-tpp2d</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3108</th>\n",
       "      <td>bioconductor-tpp</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3109</th>\n",
       "      <td>bioconductor-tracktables</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3110</th>\n",
       "      <td>bioconductor-trackviewer</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3111</th>\n",
       "      <td>bioconductor-tradeseq</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3112</th>\n",
       "      <td>bioconductor-transcriptogramer</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3113</th>\n",
       "      <td>bioconductor-transcriptr</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3114</th>\n",
       "      <td>bioconductor-transite</td>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3115</th>\n",
       "      <td>bioconductor-translatome</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3116</th>\n",
       "      <td>bioconductor-transview</td>\n",
       "      <td>558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3117</th>\n",
       "      <td>bioconductor-traser</td>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3118</th>\n",
       "      <td>bioconductor-treeio</td>\n",
       "      <td>520</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3119</th>\n",
       "      <td>bioconductor-treesummarizedexperiment</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3120</th>\n",
       "      <td>bioconductor-trena</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3121</th>\n",
       "      <td>bioconductor-trendy</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3122</th>\n",
       "      <td>bioconductor-triform</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3123</th>\n",
       "      <td>bioconductor-trigger</td>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3124</th>\n",
       "      <td>bioconductor-trio</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3125</th>\n",
       "      <td>bioconductor-triplex</td>\n",
       "      <td>565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3126</th>\n",
       "      <td>bioconductor-trna</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3127</th>\n",
       "      <td>bioconductor-trnadbimport</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3128</th>\n",
       "      <td>bioconductor-trnascanimport</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3129</th>\n",
       "      <td>bioconductor-tronco</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3130</th>\n",
       "      <td>bioconductor-tscan</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3131</th>\n",
       "      <td>bioconductor-tspair</td>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3132</th>\n",
       "      <td>bioconductor-tsrchitect</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3133</th>\n",
       "      <td>bioconductor-ttmap</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3134</th>\n",
       "      <td>bioconductor-turbonorm</td>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3135</th>\n",
       "      <td>bioconductor-tvtb</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3136</th>\n",
       "      <td>bioconductor-tweedeseq</td>\n",
       "      <td>647</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3137</th>\n",
       "      <td>bioconductor-tweedeseqcountdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3138</th>\n",
       "      <td>bioconductor-twilight</td>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3139</th>\n",
       "      <td>bioconductor-twoddpcr</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3140</th>\n",
       "      <td>bioconductor-txdb.athaliana.biomart.plantsmart22</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3141</th>\n",
       "      <td>bioconductor-txdb.athaliana.biomart.plantsmart25</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3142</th>\n",
       "      <td>bioconductor-txdb.athaliana.biomart.plantsmart28</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3143</th>\n",
       "      <td>bioconductor-txdb.btaurus.ucsc.bostau8.refgene</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3144</th>\n",
       "      <td>bioconductor-txdb.btaurus.ucsc.bostau9.refgene</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3145</th>\n",
       "      <td>bioconductor-txdb.celegans.ucsc.ce11.ensgene</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3146</th>\n",
       "      <td>bioconductor-txdb.celegans.ucsc.ce11.refgene</td>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3147</th>\n",
       "      <td>bioconductor-txdb.celegans.ucsc.ce6.ensgene</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3148</th>\n",
       "      <td>bioconductor-txdb.cfamiliaris.ucsc.canfam3.ref...</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3149</th>\n",
       "      <td>bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene</td>\n",
       "      <td>129</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3150</th>\n",
       "      <td>bioconductor-txdb.dmelanogaster.ucsc.dm6.ensgene</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3151</th>\n",
       "      <td>bioconductor-txdb.drerio.ucsc.danrer10.refgene</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3152</th>\n",
       "      <td>bioconductor-txdb.drerio.ucsc.danrer11.refgene</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3153</th>\n",
       "      <td>bioconductor-txdb.ggallus.ucsc.galgal4.refgene</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3154</th>\n",
       "      <td>bioconductor-txdb.ggallus.ucsc.galgal5.refgene</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3155</th>\n",
       "      <td>bioconductor-txdb.ggallus.ucsc.galgal6.refgene</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3156</th>\n",
       "      <td>bioconductor-txdb.hsapiens.biomart.igis</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3157</th>\n",
       "      <td>bioconductor-txdb.hsapiens.ucsc.hg18.knowngene</td>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3158</th>\n",
       "      <td>bioconductor-txdb.hsapiens.ucsc.hg19.knowngene</td>\n",
       "      <td>336</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3159</th>\n",
       "      <td>bioconductor-txdb.hsapiens.ucsc.hg19.lincrnast...</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3160</th>\n",
       "      <td>bioconductor-txdb.hsapiens.ucsc.hg38.knowngene</td>\n",
       "      <td>249</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3161</th>\n",
       "      <td>bioconductor-txdb.mmulatta.ucsc.rhemac10.refgene</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3162</th>\n",
       "      <td>bioconductor-txdb.mmulatta.ucsc.rhemac3.refgene</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3163</th>\n",
       "      <td>bioconductor-txdb.mmulatta.ucsc.rhemac8.refgene</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3164</th>\n",
       "      <td>bioconductor-txdb.mmusculus.ucsc.mm10.ensgene</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3165</th>\n",
       "      <td>bioconductor-txdb.mmusculus.ucsc.mm10.knowngene</td>\n",
       "      <td>171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3166</th>\n",
       "      <td>bioconductor-txdb.mmusculus.ucsc.mm9.knowngene</td>\n",
       "      <td>177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3167</th>\n",
       "      <td>bioconductor-txdb.ptroglodytes.ucsc.pantro4.re...</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3168</th>\n",
       "      <td>bioconductor-txdb.ptroglodytes.ucsc.pantro5.re...</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3169</th>\n",
       "      <td>bioconductor-txdb.ptroglodytes.ucsc.pantro6.re...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3170</th>\n",
       "      <td>bioconductor-txdb.rnorvegicus.biomart.igis</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3171</th>\n",
       "      <td>bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene</td>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3172</th>\n",
       "      <td>bioconductor-txdb.rnorvegicus.ucsc.rn5.refgene</td>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3173</th>\n",
       "      <td>bioconductor-txdb.rnorvegicus.ucsc.rn6.refgene</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3174</th>\n",
       "      <td>bioconductor-txdb.scerevisiae.ucsc.saccer2.sgd...</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3175</th>\n",
       "      <td>bioconductor-txdb.scerevisiae.ucsc.saccer3.sgd...</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3176</th>\n",
       "      <td>bioconductor-txdb.sscrofa.ucsc.susscr11.refgene</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3177</th>\n",
       "      <td>bioconductor-txdb.sscrofa.ucsc.susscr3.refgene</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3178</th>\n",
       "      <td>bioconductor-tximeta</td>\n",
       "      <td>62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3179</th>\n",
       "      <td>bioconductor-tximport</td>\n",
       "      <td>1498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3180</th>\n",
       "      <td>bioconductor-tximportdata</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3181</th>\n",
       "      <td>bioconductor-txreginfra</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3182</th>\n",
       "      <td>bioconductor-typeinfo</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3183</th>\n",
       "      <td>bioconductor-u133aaofav2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3184</th>\n",
       "      <td>bioconductor-u133x3pcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3185</th>\n",
       "      <td>bioconductor-u133x3p.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3186</th>\n",
       "      <td>bioconductor-u133x3pprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3187</th>\n",
       "      <td>bioconductor-ularcirc</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3188</th>\n",
       "      <td>bioconductor-undo</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3189</th>\n",
       "      <td>bioconductor-unifiedwmwqpcr</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3190</th>\n",
       "      <td>bioconductor-uniprot.ws</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3191</th>\n",
       "      <td>bioconductor-uniquorn</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3192</th>\n",
       "      <td>bioconductor-universalmotif</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3193</th>\n",
       "      <td>bioconductor-usort</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3194</th>\n",
       "      <td>bioconductor-vanillaice</td>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3195</th>\n",
       "      <td>bioconductor-variancepartition</td>\n",
       "      <td>225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3196</th>\n",
       "      <td>bioconductor-variantannotation</td>\n",
       "      <td>1845</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3197</th>\n",
       "      <td>bioconductor-variantexperiment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3198</th>\n",
       "      <td>bioconductor-variantfiltering</td>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3199</th>\n",
       "      <td>bioconductor-varianttools</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3200</th>\n",
       "      <td>bioconductor-varianttoolsdata</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3201</th>\n",
       "      <td>bioconductor-vbmp</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3202</th>\n",
       "      <td>bioconductor-vcfarray</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3203</th>\n",
       "      <td>bioconductor-vega</td>\n",
       "      <td>572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3204</th>\n",
       "      <td>bioconductor-vegamc</td>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3205</th>\n",
       "      <td>bioconductor-venndetail</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3206</th>\n",
       "      <td>bioconductor-vidger</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3207</th>\n",
       "      <td>bioconductor-viper</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3208</th>\n",
       "      <td>bioconductor-viseago</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3209</th>\n",
       "      <td>bioconductor-vitisviniferacdf</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3210</th>\n",
       "      <td>bioconductor-vitisviniferaprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3211</th>\n",
       "      <td>bioconductor-vsn</td>\n",
       "      <td>1188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3212</th>\n",
       "      <td>bioconductor-vtpnet</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3213</th>\n",
       "      <td>bioconductor-vulcan</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3214</th>\n",
       "      <td>bioconductor-vulcandata</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3215</th>\n",
       "      <td>bioconductor-waddr</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3216</th>\n",
       "      <td>bioconductor-watermelon</td>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3217</th>\n",
       "      <td>bioconductor-wavcluster</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3218</th>\n",
       "      <td>bioconductor-wavetilingdata</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3219</th>\n",
       "      <td>bioconductor-weaver</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3220</th>\n",
       "      <td>bioconductor-webbioc</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3221</th>\n",
       "      <td>bioconductor-wes.1kg.wugsc</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3222</th>\n",
       "      <td>bioconductor-wheatcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3223</th>\n",
       "      <td>bioconductor-wheatprobe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3224</th>\n",
       "      <td>bioconductor-widgettools</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3225</th>\n",
       "      <td>bioconductor-wiggleplotr</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3226</th>\n",
       "      <td>bioconductor-worm.db0</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3227</th>\n",
       "      <td>bioconductor-wrench</td>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3228</th>\n",
       "      <td>bioconductor-xbseq</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3229</th>\n",
       "      <td>bioconductor-xcir</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3230</th>\n",
       "      <td>bioconductor-xcms</td>\n",
       "      <td>1253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3231</th>\n",
       "      <td>bioconductor-xde</td>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3232</th>\n",
       "      <td>bioconductor-xenopus.db0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3233</th>\n",
       "      <td>bioconductor-xenopuslaeviscdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3234</th>\n",
       "      <td>bioconductor-xenopuslaevisprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3235</th>\n",
       "      <td>bioconductor-xeva</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3236</th>\n",
       "      <td>bioconductor-xhybcasneuf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3237</th>\n",
       "      <td>bioconductor-xina</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3238</th>\n",
       "      <td>bioconductor-xlaevis2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3239</th>\n",
       "      <td>bioconductor-xlaevis2probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3240</th>\n",
       "      <td>bioconductor-xlaevis.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3241</th>\n",
       "      <td>bioconductor-xmapbridge</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3242</th>\n",
       "      <td>bioconductor-xtrasnplocs.hsapiens.dbsnp141.grch38</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3243</th>\n",
       "      <td>bioconductor-xtrasnplocs.hsapiens.dbsnp144.grch37</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3244</th>\n",
       "      <td>bioconductor-xtrasnplocs.hsapiens.dbsnp144.grch38</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3245</th>\n",
       "      <td>bioconductor-xtropicaliscdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3246</th>\n",
       "      <td>bioconductor-xtropicalisprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3247</th>\n",
       "      <td>bioconductor-xvector</td>\n",
       "      <td>12556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3248</th>\n",
       "      <td>bioconductor-yamss</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3249</th>\n",
       "      <td>bioconductor-yapsa</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3250</th>\n",
       "      <td>bioconductor-yaqcaffy</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3251</th>\n",
       "      <td>bioconductor-yarn</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3252</th>\n",
       "      <td>bioconductor-ye6100subacdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3253</th>\n",
       "      <td>bioconductor-ye6100subbcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3254</th>\n",
       "      <td>bioconductor-ye6100subccdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3255</th>\n",
       "      <td>bioconductor-ye6100subdcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3256</th>\n",
       "      <td>bioconductor-yeast2cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3257</th>\n",
       "      <td>bioconductor-yeast2.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3258</th>\n",
       "      <td>bioconductor-yeast2probe</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3259</th>\n",
       "      <td>bioconductor-yeastcc</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3260</th>\n",
       "      <td>bioconductor-yeast.db0</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3261</th>\n",
       "      <td>bioconductor-yeastexpdata</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3262</th>\n",
       "      <td>bioconductor-yeastgsdata</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3263</th>\n",
       "      <td>bioconductor-yeastnagalakshmi</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3264</th>\n",
       "      <td>bioconductor-yeastrnaseq</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3265</th>\n",
       "      <td>bioconductor-ygs98cdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3266</th>\n",
       "      <td>bioconductor-ygs98.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3267</th>\n",
       "      <td>bioconductor-ygs98frmavecs</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3268</th>\n",
       "      <td>bioconductor-ygs98probe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3269</th>\n",
       "      <td>bioconductor-yri1kgv</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3270</th>\n",
       "      <td>bioconductor-yrimulti</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3271</th>\n",
       "      <td>bioconductor-zebrafishcdf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3272</th>\n",
       "      <td>bioconductor-zebrafish.db0</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3273</th>\n",
       "      <td>bioconductor-zebrafish.db</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3274</th>\n",
       "      <td>bioconductor-zebrafishprobe</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3275</th>\n",
       "      <td>bioconductor-zebrafishrnaseq</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3276</th>\n",
       "      <td>bioconductor-zfpkm</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3277</th>\n",
       "      <td>bioconductor-zinbwave</td>\n",
       "      <td>274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3278</th>\n",
       "      <td>bioconductor-zlibbioc</td>\n",
       "      <td>12826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3279</th>\n",
       "      <td>bioconvert</td>\n",
       "      <td>326</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3280</th>\n",
       "      <td>biodiff</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3281</th>\n",
       "      <td>bioexcel_seqqc</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3282</th>\n",
       "      <td>biofluff</td>\n",
       "      <td>849</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3283</th>\n",
       "      <td>bioformats</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3284</th>\n",
       "      <td>biogridpy</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3285</th>\n",
       "      <td>bio_hansel</td>\n",
       "      <td>1252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3286</th>\n",
       "      <td>biokit</td>\n",
       "      <td>589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3287</th>\n",
       "      <td>biolib</td>\n",
       "      <td>135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3288</th>\n",
       "      <td>biolite</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3289</th>\n",
       "      <td>biolite-tools</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3290</th>\n",
       "      <td>biomaj</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3291</th>\n",
       "      <td>biopet</td>\n",
       "      <td>316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3292</th>\n",
       "      <td>biopet-bamstats</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3293</th>\n",
       "      <td>biopet-basecounter</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3294</th>\n",
       "      <td>biopet-extractadaptersfastqc</td>\n",
       "      <td>212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3295</th>\n",
       "      <td>biopet-fastqsplitter</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3296</th>\n",
       "      <td>biopet-sampleconfig</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3297</th>\n",
       "      <td>biopet-scatterregions</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3298</th>\n",
       "      <td>biopet-seattleseqkit</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3299</th>\n",
       "      <td>biopet-seqstat</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3300</th>\n",
       "      <td>biopet-validateannotation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3301</th>\n",
       "      <td>biopet-validatefastq</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3302</th>\n",
       "      <td>biopet-validatevcf</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3303</th>\n",
       "      <td>biopet-vcffilter</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3304</th>\n",
       "      <td>biopet-vcfstats</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3305</th>\n",
       "      <td>biopython.convert</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3306</th>\n",
       "      <td>bioservices</td>\n",
       "      <td>796</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3307</th>\n",
       "      <td>biotdg</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3308</th>\n",
       "      <td>biothings_client</td>\n",
       "      <td>497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3309</th>\n",
       "      <td>biotradis</td>\n",
       "      <td>228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3310</th>\n",
       "      <td>bio-vcf</td>\n",
       "      <td>383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3311</th>\n",
       "      <td>biowdl-input-converter</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3312</th>\n",
       "      <td>biskit</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3313</th>\n",
       "      <td>bismark</td>\n",
       "      <td>1307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3314</th>\n",
       "      <td>bis-snp</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3315</th>\n",
       "      <td>bis-snp-utils</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3316</th>\n",
       "      <td>bitmapperbs</td>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3317</th>\n",
       "      <td>bitstring</td>\n",
       "      <td>127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3318</th>\n",
       "      <td>blasr</td>\n",
       "      <td>707</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3319</th>\n",
       "      <td>blasr_libcpp</td>\n",
       "      <td>584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3320</th>\n",
       "      <td>blast</td>\n",
       "      <td>10108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3321</th>\n",
       "      <td>blastalign</td>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3322</th>\n",
       "      <td>blastbesties</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3323</th>\n",
       "      <td>blast-legacy</td>\n",
       "      <td>857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3324</th>\n",
       "      <td>blksheep</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3325</th>\n",
       "      <td>blobtools</td>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3326</th>\n",
       "      <td>blockbuster</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3327</th>\n",
       "      <td>blockclust</td>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3328</th>\n",
       "      <td>bloocoo</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3329</th>\n",
       "      <td>bloomfiltertrie</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3330</th>\n",
       "      <td>bmfilter</td>\n",
       "      <td>302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3331</th>\n",
       "      <td>bmtagger</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3332</th>\n",
       "      <td>bmtool</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3333</th>\n",
       "      <td>bohra</td>\n",
       "      <td>62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3334</th>\n",
       "      <td>bold-identification</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3335</th>\n",
       "      <td>bolt</td>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3336</th>\n",
       "      <td>booster</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3337</th>\n",
       "      <td>borf</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3338</th>\n",
       "      <td>bowtie</td>\n",
       "      <td>11858</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3339</th>\n",
       "      <td>bowtie2</td>\n",
       "      <td>8426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3340</th>\n",
       "      <td>bpipe</td>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3341</th>\n",
       "      <td>bpp-core</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3342</th>\n",
       "      <td>bpp-phyl</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3343</th>\n",
       "      <td>bpp-popgen</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3344</th>\n",
       "      <td>bpp-seq</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3345</th>\n",
       "      <td>bracken</td>\n",
       "      <td>407</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3346</th>\n",
       "      <td>braker</td>\n",
       "      <td>163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3347</th>\n",
       "      <td>braker2</td>\n",
       "      <td>165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3348</th>\n",
       "      <td>brass</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3349</th>\n",
       "      <td>breakdancer</td>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3350</th>\n",
       "      <td>break-point-inspector</td>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3351</th>\n",
       "      <td>breakseq2</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3352</th>\n",
       "      <td>breseq</td>\n",
       "      <td>713</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3353</th>\n",
       "      <td>brockman-pipeline</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3354</th>\n",
       "      <td>bs_call</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3355</th>\n",
       "      <td>bsmap</td>\n",
       "      <td>132</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3356</th>\n",
       "      <td>bs-seeker2</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3357</th>\n",
       "      <td>btrim</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3358</th>\n",
       "      <td>bufet</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3359</th>\n",
       "      <td>bumbershoot</td>\n",
       "      <td>581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3360</th>\n",
       "      <td>burrito</td>\n",
       "      <td>254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3361</th>\n",
       "      <td>burrito-fillings</td>\n",
       "      <td>244</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3362</th>\n",
       "      <td>busco</td>\n",
       "      <td>1475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3363</th>\n",
       "      <td>bustools</td>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3364</th>\n",
       "      <td>bwa</td>\n",
       "      <td>14995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3365</th>\n",
       "      <td>bwakit</td>\n",
       "      <td>361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3366</th>\n",
       "      <td>bwameth</td>\n",
       "      <td>428</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3367</th>\n",
       "      <td>bwapy</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3368</th>\n",
       "      <td>bwise</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3369</th>\n",
       "      <td>bx-python</td>\n",
       "      <td>4947</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3370</th>\n",
       "      <td>bxtools</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3371</th>\n",
       "      <td>cactus</td>\n",
       "      <td>142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3372</th>\n",
       "      <td>cafe</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3373</th>\n",
       "      <td>cage</td>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3374</th>\n",
       "      <td>calib</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3375</th>\n",
       "      <td>calour</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3376</th>\n",
       "      <td>cameo</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3377</th>\n",
       "      <td>cami-opal</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3378</th>\n",
       "      <td>cancerit-allelecount</td>\n",
       "      <td>332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3379</th>\n",
       "      <td>cannoli</td>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3380</th>\n",
       "      <td>cansam</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3381</th>\n",
       "      <td>cansnper</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3382</th>\n",
       "      <td>canu</td>\n",
       "      <td>1254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3383</th>\n",
       "      <td>canvas</td>\n",
       "      <td>238</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3384</th>\n",
       "      <td>cap3</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3385</th>\n",
       "      <td>capc-map</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3386</th>\n",
       "      <td>caper</td>\n",
       "      <td>142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3387</th>\n",
       "      <td>cap-mirseq</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3388</th>\n",
       "      <td>card_trick</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3389</th>\n",
       "      <td>c-ares</td>\n",
       "      <td>267</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3390</th>\n",
       "      <td>carna</td>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3391</th>\n",
       "      <td>carnac-lr</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3392</th>\n",
       "      <td>cas-offinder</td>\n",
       "      <td>179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3393</th>\n",
       "      <td>cassiopee</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3394</th>\n",
       "      <td>cat</td>\n",
       "      <td>336</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3395</th>\n",
       "      <td>catch</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3396</th>\n",
       "      <td>ccat</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3397</th>\n",
       "      <td>cdbtools</td>\n",
       "      <td>397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3398</th>\n",
       "      <td>cd-hit</td>\n",
       "      <td>1752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3399</th>\n",
       "      <td>cd-hit-auxtools</td>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3400</th>\n",
       "      <td>cdna_cupcake</td>\n",
       "      <td>479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3401</th>\n",
       "      <td>ceas</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3402</th>\n",
       "      <td>centreseq</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3403</th>\n",
       "      <td>centrifuge</td>\n",
       "      <td>704</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3404</th>\n",
       "      <td>centroid_rna_package</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3405</th>\n",
       "      <td>centrosome</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3406</th>\n",
       "      <td>cesm</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3407</th>\n",
       "      <td>cfm</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3408</th>\n",
       "      <td>cgat-apps</td>\n",
       "      <td>509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3409</th>\n",
       "      <td>cgatcore</td>\n",
       "      <td>1072</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3410</th>\n",
       "      <td>cgat-daisy</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3411</th>\n",
       "      <td>cgat-report</td>\n",
       "      <td>312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3412</th>\n",
       "      <td>cgat-scripts</td>\n",
       "      <td>485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3413</th>\n",
       "      <td>cgcloud-lib</td>\n",
       "      <td>234</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3414</th>\n",
       "      <td>cgecore</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3415</th>\n",
       "      <td>cgpbigwig</td>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3416</th>\n",
       "      <td>cgview</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3417</th>\n",
       "      <td>chado-tools</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3418</th>\n",
       "      <td>changeo</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3419</th>\n",
       "      <td>chanjo</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3420</th>\n",
       "      <td>checkm-genome</td>\n",
       "      <td>962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3421</th>\n",
       "      <td>check-sort-order</td>\n",
       "      <td>429</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3422</th>\n",
       "      <td>chemfp</td>\n",
       "      <td>477</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3423</th>\n",
       "      <td>chewbbaca</td>\n",
       "      <td>590</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3424</th>\n",
       "      <td>chexmix</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3425</th>\n",
       "      <td>chicagotools</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3426</th>\n",
       "      <td>chipseq-greylist</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3427</th>\n",
       "      <td>chira</td>\n",
       "      <td>61</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3428</th>\n",
       "      <td>chiron</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3429</th>\n",
       "      <td>chorus2</td>\n",
       "      <td>114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3430</th>\n",
       "      <td>chromatiblock</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3431</th>\n",
       "      <td>chromhmm</td>\n",
       "      <td>356</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3432</th>\n",
       "      <td>chromosight</td>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3433</th>\n",
       "      <td>chromosomer</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3434</th>\n",
       "      <td>chunked-scatter</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3435</th>\n",
       "      <td>cigar</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3436</th>\n",
       "      <td>circexplorer2</td>\n",
       "      <td>1883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3437</th>\n",
       "      <td>circexplorer</td>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3438</th>\n",
       "      <td>circlator</td>\n",
       "      <td>822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3439</th>\n",
       "      <td>circle-map</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3440</th>\n",
       "      <td>circos</td>\n",
       "      <td>1460</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3441</th>\n",
       "      <td>circos-tools</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3442</th>\n",
       "      <td>circularmapper</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3443</th>\n",
       "      <td>cistrome_beta</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3444</th>\n",
       "      <td>cistrome-ceas</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3445</th>\n",
       "      <td>cityhash</td>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3446</th>\n",
       "      <td>clair</td>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3447</th>\n",
       "      <td>clairvoyante</td>\n",
       "      <td>302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3448</th>\n",
       "      <td>clan</td>\n",
       "      <td>83</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3449</th>\n",
       "      <td>clark</td>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3450</th>\n",
       "      <td>clearcut</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3451</th>\n",
       "      <td>clever-toolkit</td>\n",
       "      <td>573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3452</th>\n",
       "      <td>clinod</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3453</th>\n",
       "      <td>clinvar-tsv</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3454</th>\n",
       "      <td>clipandmerge</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3455</th>\n",
       "      <td>clonalframeml</td>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3456</th>\n",
       "      <td>clove</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3457</th>\n",
       "      <td>clsify</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3458</th>\n",
       "      <td>clust</td>\n",
       "      <td>395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3459</th>\n",
       "      <td>clustalo</td>\n",
       "      <td>502</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3460</th>\n",
       "      <td>clustalw</td>\n",
       "      <td>3296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3461</th>\n",
       "      <td>clustergrammer</td>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3462</th>\n",
       "      <td>clusterpicker</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3463</th>\n",
       "      <td>cmappy</td>\n",
       "      <td>1246</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3464</th>\n",
       "      <td>cmfinder</td>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3465</th>\n",
       "      <td>cmph</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3466</th>\n",
       "      <td>cmsip</td>\n",
       "      <td>44</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3467</th>\n",
       "      <td>cmv</td>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3468</th>\n",
       "      <td>cnvetti</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3469</th>\n",
       "      <td>cnv_facets</td>\n",
       "      <td>725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3470</th>\n",
       "      <td>cnvkit</td>\n",
       "      <td>2842</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3471</th>\n",
       "      <td>cobra</td>\n",
       "      <td>411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3472</th>\n",
       "      <td>cocoscore</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3473</th>\n",
       "      <td>codingquarry</td>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3474</th>\n",
       "      <td>codonw</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3475</th>\n",
       "      <td>cofold</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3476</th>\n",
       "      <td>cogent</td>\n",
       "      <td>489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3477</th>\n",
       "      <td>coinfinder</td>\n",
       "      <td>316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3478</th>\n",
       "      <td>collect-columns</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3479</th>\n",
       "      <td>collectl</td>\n",
       "      <td>343</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3480</th>\n",
       "      <td>collect_mgf</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3481</th>\n",
       "      <td>colombo</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3482</th>\n",
       "      <td>colorbrewer</td>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3483</th>\n",
       "      <td>combined-pvalues</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3484</th>\n",
       "      <td>comet-ms</td>\n",
       "      <td>847</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3485</th>\n",
       "      <td>commet</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3486</th>\n",
       "      <td>compalignp</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3487</th>\n",
       "      <td>comparative-annotation-toolkit</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3488</th>\n",
       "      <td>comparems2</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3489</th>\n",
       "      <td>compare-reads</td>\n",
       "      <td>119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3490</th>\n",
       "      <td>concoct</td>\n",
       "      <td>785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3491</th>\n",
       "      <td>confindr</td>\n",
       "      <td>144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3492</th>\n",
       "      <td>connectome-workbench</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3493</th>\n",
       "      <td>connor</td>\n",
       "      <td>214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3494</th>\n",
       "      <td>consan</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3495</th>\n",
       "      <td>consel</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3496</th>\n",
       "      <td>consensusfixer</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3497</th>\n",
       "      <td>conterminator</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3498</th>\n",
       "      <td>control-freec</td>\n",
       "      <td>401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3499</th>\n",
       "      <td>conus</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3500</th>\n",
       "      <td>convert_zero_one_based</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3501</th>\n",
       "      <td>cooler</td>\n",
       "      <td>2887</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3502</th>\n",
       "      <td>coprarna</td>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3503</th>\n",
       "      <td>coral</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3504</th>\n",
       "      <td>corset</td>\n",
       "      <td>343</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3505</th>\n",
       "      <td>cortex_con</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3506</th>\n",
       "      <td>cortexpy</td>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3507</th>\n",
       "      <td>cosi2</td>\n",
       "      <td>664</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3508</th>\n",
       "      <td>count_constant_sites</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3509</th>\n",
       "      <td>counterr</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3510</th>\n",
       "      <td>coverm</td>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3511</th>\n",
       "      <td>covtobed</td>\n",
       "      <td>529</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3512</th>\n",
       "      <td>cpat</td>\n",
       "      <td>367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3513</th>\n",
       "      <td>cpinsim</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3514</th>\n",
       "      <td>cramtools</td>\n",
       "      <td>308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3515</th>\n",
       "      <td>crass</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3516</th>\n",
       "      <td>crb-blast</td>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3517</th>\n",
       "      <td>crimson</td>\n",
       "      <td>213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3518</th>\n",
       "      <td>crisper_recognition_tool</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3519</th>\n",
       "      <td>crisprbact</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3520</th>\n",
       "      <td>crispresso2</td>\n",
       "      <td>669</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3521</th>\n",
       "      <td>crispresso</td>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3522</th>\n",
       "      <td>crispritz</td>\n",
       "      <td>762</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3523</th>\n",
       "      <td>crispr_studio</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3524</th>\n",
       "      <td>cromshell</td>\n",
       "      <td>26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3525</th>\n",
       "      <td>cromwell-tools</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3526</th>\n",
       "      <td>croo</td>\n",
       "      <td>137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3527</th>\n",
       "      <td>crossmap</td>\n",
       "      <td>911</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3528</th>\n",
       "      <td>crumble</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3529</th>\n",
       "      <td>crux-toolkit</td>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3530</th>\n",
       "      <td>cryfa</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3531</th>\n",
       "      <td>crypto_typer</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3532</th>\n",
       "      <td>csb</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3533</th>\n",
       "      <td>csvtk</td>\n",
       "      <td>2339</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3534</th>\n",
       "      <td>ctat-lncrna</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3535</th>\n",
       "      <td>ctat-metagenomics</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3536</th>\n",
       "      <td>ctat-mutations</td>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3537</th>\n",
       "      <td>cufflinks</td>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3538</th>\n",
       "      <td>curesim</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3539</th>\n",
       "      <td>curlywhirly</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3540</th>\n",
       "      <td>cutadapt</td>\n",
       "      <td>14265</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3541</th>\n",
       "      <td>cutesv</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3542</th>\n",
       "      <td>cvbio</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3543</th>\n",
       "      <td>cwl2wdl</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3544</th>\n",
       "      <td>cwltest</td>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3545</th>\n",
       "      <td>cytoscape</td>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3546</th>\n",
       "      <td>cyvcf2</td>\n",
       "      <td>8829</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3547</th>\n",
       "      <td>daisysuite</td>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3548</th>\n",
       "      <td>daligner</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3549</th>\n",
       "      <td>damageprofiler</td>\n",
       "      <td>222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3550</th>\n",
       "      <td>damasker</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3551</th>\n",
       "      <td>damidseq_pipeline</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3552</th>\n",
       "      <td>dammet</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3553</th>\n",
       "      <td>dammit</td>\n",
       "      <td>498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3554</th>\n",
       "      <td>danpos</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3555</th>\n",
       "      <td>dart</td>\n",
       "      <td>431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3556</th>\n",
       "      <td>dascrubber</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3557</th>\n",
       "      <td>dashing</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3558</th>\n",
       "      <td>das_tool</td>\n",
       "      <td>172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3559</th>\n",
       "      <td>data_hacks</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3560</th>\n",
       "      <td>dawg</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3561</th>\n",
       "      <td>dazz_db</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3562</th>\n",
       "      <td>dbg2olc</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3563</th>\n",
       "      <td>dbgraph</td>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3564</th>\n",
       "      <td>dca</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3565</th>\n",
       "      <td>dcmtk</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3566</th>\n",
       "      <td>dcplib</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3567</th>\n",
       "      <td>ddocent</td>\n",
       "      <td>1868</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3568</th>\n",
       "      <td>ddrage</td>\n",
       "      <td>845</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3569</th>\n",
       "      <td>deap</td>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3570</th>\n",
       "      <td>deblur</td>\n",
       "      <td>2644</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3571</th>\n",
       "      <td>debwt</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3572</th>\n",
       "      <td>decoypyrat</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3573</th>\n",
       "      <td>dedup</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3574</th>\n",
       "      <td>deepac</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3575</th>\n",
       "      <td>deepbgc</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3576</th>\n",
       "      <td>deepbinner</td>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3577</th>\n",
       "      <td>deeplc</td>\n",
       "      <td>41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3578</th>\n",
       "      <td>deeplift</td>\n",
       "      <td>30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3579</th>\n",
       "      <td>deepmedic</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3580</th>\n",
       "      <td>deepstats</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3581</th>\n",
       "      <td>deepsvr</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3582</th>\n",
       "      <td>deeptools</td>\n",
       "      <td>5174</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3583</th>\n",
       "      <td>deeptoolsintervals</td>\n",
       "      <td>1728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3584</th>\n",
       "      <td>deepvariant</td>\n",
       "      <td>908</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3585</th>\n",
       "      <td>defiant</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3586</th>\n",
       "      <td>dehomopolymerate</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3587</th>\n",
       "      <td>delegation</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3588</th>\n",
       "      <td>delly</td>\n",
       "      <td>1063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3589</th>\n",
       "      <td>deltabs</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3590</th>\n",
       "      <td>demuxlet</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3591</th>\n",
       "      <td>dendropy</td>\n",
       "      <td>4080</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3592</th>\n",
       "      <td>denovogear</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3593</th>\n",
       "      <td>deploid</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3594</th>\n",
       "      <td>desalt</td>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3595</th>\n",
       "      <td>desman</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3596</th>\n",
       "      <td>detonate</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3597</th>\n",
       "      <td>dextractor</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3598</th>\n",
       "      <td>dfam</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3599</th>\n",
       "      <td>dfast</td>\n",
       "      <td>307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3600</th>\n",
       "      <td>dialign2</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3601</th>\n",
       "      <td>dialign-tx</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3602</th>\n",
       "      <td>diamond</td>\n",
       "      <td>4416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3603</th>\n",
       "      <td>diamond_add_taxonomy</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3604</th>\n",
       "      <td>dia_umpire</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3605</th>\n",
       "      <td>dicey</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3606</th>\n",
       "      <td>dig2</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3607</th>\n",
       "      <td>digestiflow-cli</td>\n",
       "      <td>1305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3608</th>\n",
       "      <td>digestiflow-demux</td>\n",
       "      <td>52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3609</th>\n",
       "      <td>dimspy</td>\n",
       "      <td>382</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3610</th>\n",
       "      <td>dinamo</td>\n",
       "      <td>166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3611</th>\n",
       "      <td>dinopy</td>\n",
       "      <td>625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3612</th>\n",
       "      <td>dinosaur</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3613</th>\n",
       "      <td>discasm</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3614</th>\n",
       "      <td>disco</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3615</th>\n",
       "      <td>discosnp</td>\n",
       "      <td>309</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3616</th>\n",
       "      <td>discovar</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3617</th>\n",
       "      <td>disty</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3618</th>\n",
       "      <td>ditasic</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3619</th>\n",
       "      <td>divvier</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3620</th>\n",
       "      <td>dkfz-bias-filter</td>\n",
       "      <td>165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3621</th>\n",
       "      <td>dlcpar</td>\n",
       "      <td>172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3622</th>\n",
       "      <td>dnaio</td>\n",
       "      <td>10175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3623</th>\n",
       "      <td>dnapi</td>\n",
       "      <td>389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3624</th>\n",
       "      <td>dnp-binstrings</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3625</th>\n",
       "      <td>dnp-corrprofile</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3626</th>\n",
       "      <td>dnp-diprofile</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3627</th>\n",
       "      <td>dnp-fourier</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3628</th>\n",
       "      <td>domclust</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3629</th>\n",
       "      <td>downpore</td>\n",
       "      <td>373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3630</th>\n",
       "      <td>drax</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3631</th>\n",
       "      <td>dr-disco</td>\n",
       "      <td>922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3632</th>\n",
       "      <td>dreamtools</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3633</th>\n",
       "      <td>drep</td>\n",
       "      <td>275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3634</th>\n",
       "      <td>drive</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3635</th>\n",
       "      <td>dropest</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3636</th>\n",
       "      <td>dropletutils-scripts</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3637</th>\n",
       "      <td>dropseq_tools</td>\n",
       "      <td>228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3638</th>\n",
       "      <td>dsh-bio</td>\n",
       "      <td>224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3639</th>\n",
       "      <td>dsk</td>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3640</th>\n",
       "      <td>dsrc</td>\n",
       "      <td>219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3641</th>\n",
       "      <td>dudes</td>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3642</th>\n",
       "      <td>dunovo</td>\n",
       "      <td>1113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3643</th>\n",
       "      <td>duphold</td>\n",
       "      <td>1144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3644</th>\n",
       "      <td>dupre</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3645</th>\n",
       "      <td>dwgsim</td>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3646</th>\n",
       "      <td>dx-cwl</td>\n",
       "      <td>1135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3647</th>\n",
       "      <td>dxpy</td>\n",
       "      <td>937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3648</th>\n",
       "      <td>dxua</td>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3649</th>\n",
       "      <td>eagle</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3650</th>\n",
       "      <td>easel</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3651</th>\n",
       "      <td>ea-utils</td>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3652</th>\n",
       "      <td>ebfilter</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3653</th>\n",
       "      <td>ebisearch</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3654</th>\n",
       "      <td>ecmwfapi</td>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3655</th>\n",
       "      <td>ecopcr</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3656</th>\n",
       "      <td>ecoprimers</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3657</th>\n",
       "      <td>ecopy</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3658</th>\n",
       "      <td>ectyper</td>\n",
       "      <td>223</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3659</th>\n",
       "      <td>edd</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3660</th>\n",
       "      <td>eden</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3661</th>\n",
       "      <td>edena</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3662</th>\n",
       "      <td>edittag</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3663</th>\n",
       "      <td>edlib</td>\n",
       "      <td>503</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3664</th>\n",
       "      <td>edta</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3665</th>\n",
       "      <td>effectivet3</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3666</th>\n",
       "      <td>ega2</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3667</th>\n",
       "      <td>egglib</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3668</th>\n",
       "      <td>eggnog-mapper</td>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3669</th>\n",
       "      <td>eigensoft</td>\n",
       "      <td>236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3670</th>\n",
       "      <td>elasticluster</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3671</th>\n",
       "      <td>elector</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3672</th>\n",
       "      <td>elprep</td>\n",
       "      <td>662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3673</th>\n",
       "      <td>ema</td>\n",
       "      <td>4952</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3674</th>\n",
       "      <td>embassy-phylip</td>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3675</th>\n",
       "      <td>embl-api-validator</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3676</th>\n",
       "      <td>emblmygff3</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3677</th>\n",
       "      <td>emboss</td>\n",
       "      <td>1368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3678</th>\n",
       "      <td>emirge</td>\n",
       "      <td>135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3679</th>\n",
       "      <td>emmix</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3680</th>\n",
       "      <td>emmtyper</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3681</th>\n",
       "      <td>emperor</td>\n",
       "      <td>329</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3682</th>\n",
       "      <td>enabrowsertools</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3683</th>\n",
       "      <td>ena-dl</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3684</th>\n",
       "      <td>enasearch</td>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3685</th>\n",
       "      <td>encode-blacklist</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3686</th>\n",
       "      <td>encyclopedia</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3687</th>\n",
       "      <td>enrichm</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3688</th>\n",
       "      <td>ensembl-vep</td>\n",
       "      <td>3846</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3689</th>\n",
       "      <td>entrez-direct</td>\n",
       "      <td>6557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3690</th>\n",
       "      <td>entrezpy</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3691</th>\n",
       "      <td>eoulsan</td>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3692</th>\n",
       "      <td>epa-ng</td>\n",
       "      <td>775</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3693</th>\n",
       "      <td>e-pcr</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3694</th>\n",
       "      <td>ephemeris</td>\n",
       "      <td>1457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3695</th>\n",
       "      <td>epic</td>\n",
       "      <td>1561</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3696</th>\n",
       "      <td>epic2</td>\n",
       "      <td>1087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3697</th>\n",
       "      <td>epicseg</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3698</th>\n",
       "      <td>epydoc</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3699</th>\n",
       "      <td>eqtlbma</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3700</th>\n",
       "      <td>erds</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3701</th>\n",
       "      <td>ericscript</td>\n",
       "      <td>175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3702</th>\n",
       "      <td>erne</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3703</th>\n",
       "      <td>escher</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3704</th>\n",
       "      <td>esimsa</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3705</th>\n",
       "      <td>esmre</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3706</th>\n",
       "      <td>estmapper</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3707</th>\n",
       "      <td>estscan</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3708</th>\n",
       "      <td>eta</td>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3709</th>\n",
       "      <td>ete2</td>\n",
       "      <td>361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3710</th>\n",
       "      <td>eval</td>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3711</th>\n",
       "      <td>evidencemodeler</td>\n",
       "      <td>502</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3712</th>\n",
       "      <td>evofold2</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3713</th>\n",
       "      <td>exonerate</td>\n",
       "      <td>764</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3714</th>\n",
       "      <td>expansionhunter</td>\n",
       "      <td>535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3715</th>\n",
       "      <td>exparna</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3716</th>\n",
       "      <td>export2graphlan</td>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3717</th>\n",
       "      <td>express</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3718</th>\n",
       "      <td>extract-codon-alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3719</th>\n",
       "      <td>extract_fasta_seq</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3720</th>\n",
       "      <td>extract_fullseq</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3721</th>\n",
       "      <td>extract_genome_region</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3722</th>\n",
       "      <td>extract-sv-reads</td>\n",
       "      <td>476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3723</th>\n",
       "      <td>ezclermont</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3724</th>\n",
       "      <td>fadapa</td>\n",
       "      <td>382</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3725</th>\n",
       "      <td>falco</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3726</th>\n",
       "      <td>famsa</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3727</th>\n",
       "      <td>famseq</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3728</th>\n",
       "      <td>fann</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3729</th>\n",
       "      <td>faqcs</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3730</th>\n",
       "      <td>fargene</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3731</th>\n",
       "      <td>fast5</td>\n",
       "      <td>1060</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3732</th>\n",
       "      <td>fast5-research</td>\n",
       "      <td>161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3733</th>\n",
       "      <td>fast5seek</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3734</th>\n",
       "      <td>fasta3</td>\n",
       "      <td>401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3735</th>\n",
       "      <td>fastahack</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3736</th>\n",
       "      <td>fastaindex</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3737</th>\n",
       "      <td>fastalite</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3738</th>\n",
       "      <td>fastani</td>\n",
       "      <td>523</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3739</th>\n",
       "      <td>fastaptamer</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3740</th>\n",
       "      <td>fasta-splitter</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3741</th>\n",
       "      <td>fasta_ushuffle</td>\n",
       "      <td>122</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3742</th>\n",
       "      <td>fasten</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3743</th>\n",
       "      <td>fastindep</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3744</th>\n",
       "      <td>fastlmm</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3745</th>\n",
       "      <td>fastme</td>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3746</th>\n",
       "      <td>fastml</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3747</th>\n",
       "      <td>fastobo</td>\n",
       "      <td>359</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3748</th>\n",
       "      <td>fastool</td>\n",
       "      <td>345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3749</th>\n",
       "      <td>fastp</td>\n",
       "      <td>7260</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3750</th>\n",
       "      <td>fastphylo</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3751</th>\n",
       "      <td>fastq-and-furious</td>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3752</th>\n",
       "      <td>fastq-anonymous</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3753</th>\n",
       "      <td>fastqc</td>\n",
       "      <td>14258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3754</th>\n",
       "      <td>fastq-count</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3755</th>\n",
       "      <td>fastq-dl</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3756</th>\n",
       "      <td>fastqe</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3757</th>\n",
       "      <td>fastq-join</td>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3758</th>\n",
       "      <td>fastqmetrics</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3759</th>\n",
       "      <td>fastq-multx</td>\n",
       "      <td>267</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3760</th>\n",
       "      <td>fastq-pair</td>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3761</th>\n",
       "      <td>fastqpuri</td>\n",
       "      <td>395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3762</th>\n",
       "      <td>fastq-scan</td>\n",
       "      <td>451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3763</th>\n",
       "      <td>fastq-screen</td>\n",
       "      <td>859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3764</th>\n",
       "      <td>fastqsplitter</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3765</th>\n",
       "      <td>fastq-tools</td>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3766</th>\n",
       "      <td>fastq_utils</td>\n",
       "      <td>684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3767</th>\n",
       "      <td>fastsimbac</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3768</th>\n",
       "      <td>fastspar</td>\n",
       "      <td>310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3769</th>\n",
       "      <td>faststructure</td>\n",
       "      <td>118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3770</th>\n",
       "      <td>fasttree</td>\n",
       "      <td>3658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3771</th>\n",
       "      <td>fastuniq</td>\n",
       "      <td>114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3772</th>\n",
       "      <td>fastutils</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3773</th>\n",
       "      <td>fastx_toolkit</td>\n",
       "      <td>611</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3774</th>\n",
       "      <td>fcp</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3775</th>\n",
       "      <td>fcsparser</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3776</th>\n",
       "      <td>feature_merge</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3777</th>\n",
       "      <td>feelnc</td>\n",
       "      <td>141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3778</th>\n",
       "      <td>feht</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3779</th>\n",
       "      <td>fermi</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3780</th>\n",
       "      <td>fermi2</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3781</th>\n",
       "      <td>fermikit</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3782</th>\n",
       "      <td>fermi-lite</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3783</th>\n",
       "      <td>fgap</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3784</th>\n",
       "      <td>fgbio</td>\n",
       "      <td>1106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3785</th>\n",
       "      <td>fgmp</td>\n",
       "      <td>237</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3786</th>\n",
       "      <td>ficus</td>\n",
       "      <td>217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3787</th>\n",
       "      <td>fido</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3788</th>\n",
       "      <td>figtree</td>\n",
       "      <td>62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3789</th>\n",
       "      <td>fiji</td>\n",
       "      <td>318</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3790</th>\n",
       "      <td>filtlong</td>\n",
       "      <td>494</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3791</th>\n",
       "      <td>find_differential_primers</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3792</th>\n",
       "      <td>fineradstructure</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3793</th>\n",
       "      <td>finestructure</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3794</th>\n",
       "      <td>fings</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3795</th>\n",
       "      <td>firecloud</td>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3796</th>\n",
       "      <td>fishtaco</td>\n",
       "      <td>144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3797</th>\n",
       "      <td>fithic</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3798</th>\n",
       "      <td>fit_nbinom</td>\n",
       "      <td>914</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3799</th>\n",
       "      <td>fitter</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3800</th>\n",
       "      <td>flaimapper</td>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3801</th>\n",
       "      <td>flair</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3802</th>\n",
       "      <td>flapjack</td>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3803</th>\n",
       "      <td>flash2</td>\n",
       "      <td>257</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3804</th>\n",
       "      <td>flash</td>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3805</th>\n",
       "      <td>flashlfq</td>\n",
       "      <td>705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3806</th>\n",
       "      <td>flexbar</td>\n",
       "      <td>317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3807</th>\n",
       "      <td>flextaxd</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3808</th>\n",
       "      <td>flock</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3809</th>\n",
       "      <td>flowcraft</td>\n",
       "      <td>695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3810</th>\n",
       "      <td>flux-simulator</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3811</th>\n",
       "      <td>flye</td>\n",
       "      <td>1596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3812</th>\n",
       "      <td>fmlrc</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3813</th>\n",
       "      <td>focus</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3814</th>\n",
       "      <td>footprint</td>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3815</th>\n",
       "      <td>forgi</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3816</th>\n",
       "      <td>forked-path</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3817</th>\n",
       "      <td>fpa</td>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3818</th>\n",
       "      <td>fqtools</td>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3819</th>\n",
       "      <td>fqtrim</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3820</th>\n",
       "      <td>fqzcomp</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3821</th>\n",
       "      <td>fraggenescan</td>\n",
       "      <td>439</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3822</th>\n",
       "      <td>frc</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3823</th>\n",
       "      <td>fred2</td>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3824</th>\n",
       "      <td>freebayes</td>\n",
       "      <td>3075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3825</th>\n",
       "      <td>frogs</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3826</th>\n",
       "      <td>fsa</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3827</th>\n",
       "      <td>fseq</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3828</th>\n",
       "      <td>fsm-lite</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3829</th>\n",
       "      <td>fsnviz</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3830</th>\n",
       "      <td>fuma</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3831</th>\n",
       "      <td>funannotate</td>\n",
       "      <td>452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3832</th>\n",
       "      <td>funcannot</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3833</th>\n",
       "      <td>funnel</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3834</th>\n",
       "      <td>fusioncatcher</td>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3835</th>\n",
       "      <td>fusioncatcher-seqtk</td>\n",
       "      <td>118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3836</th>\n",
       "      <td>fusion-filter</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3837</th>\n",
       "      <td>fusion-inspector</td>\n",
       "      <td>314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3838</th>\n",
       "      <td>fusion-report</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3839</th>\n",
       "      <td>fwdpp</td>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3840</th>\n",
       "      <td>fwdpy11</td>\n",
       "      <td>2315</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3841</th>\n",
       "      <td>fwdpy</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3842</th>\n",
       "      <td>ga4ghmongo</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3843</th>\n",
       "      <td>gadem</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3844</th>\n",
       "      <td>galaxy-containers</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3845</th>\n",
       "      <td>galaxy-ie-helpers</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3846</th>\n",
       "      <td>galaxy-lib</td>\n",
       "      <td>3664</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3847</th>\n",
       "      <td>galaxy-ml</td>\n",
       "      <td>1321</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3848</th>\n",
       "      <td>galaxy-ops</td>\n",
       "      <td>239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3849</th>\n",
       "      <td>galaxy-parsec</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3850</th>\n",
       "      <td>galaxy_sequence_utils</td>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3851</th>\n",
       "      <td>galaxy-tool-util</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3852</th>\n",
       "      <td>galaxy-util</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3853</th>\n",
       "      <td>ganon</td>\n",
       "      <td>489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3854</th>\n",
       "      <td>gap2seq</td>\n",
       "      <td>320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3855</th>\n",
       "      <td>gapfiller</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3856</th>\n",
       "      <td>gappa</td>\n",
       "      <td>865</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3857</th>\n",
       "      <td>garnet</td>\n",
       "      <td>472</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3858</th>\n",
       "      <td>garnett-cli</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3859</th>\n",
       "      <td>gat</td>\n",
       "      <td>443</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3860</th>\n",
       "      <td>gatb</td>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3861</th>\n",
       "      <td>gatk</td>\n",
       "      <td>1703</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3862</th>\n",
       "      <td>gatk4</td>\n",
       "      <td>3636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3863</th>\n",
       "      <td>gatk-framework</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3864</th>\n",
       "      <td>gatktool</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3865</th>\n",
       "      <td>gblocks</td>\n",
       "      <td>247</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3866</th>\n",
       "      <td>gbmunge</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3867</th>\n",
       "      <td>gbsx</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3868</th>\n",
       "      <td>gb_taxonomy_tools</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3869</th>\n",
       "      <td>gcs-oauth2-boto-plugin</td>\n",
       "      <td>138</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3870</th>\n",
       "      <td>gdc</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3871</th>\n",
       "      <td>gdc-client</td>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3872</th>\n",
       "      <td>gdsctools</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3873</th>\n",
       "      <td>geco2</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3874</th>\n",
       "      <td>gecode</td>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3875</th>\n",
       "      <td>gem3-mapper</td>\n",
       "      <td>214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3876</th>\n",
       "      <td>gembs</td>\n",
       "      <td>119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3877</th>\n",
       "      <td>gemini</td>\n",
       "      <td>1688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3878</th>\n",
       "      <td>gemma</td>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3879</th>\n",
       "      <td>genblasta</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3880</th>\n",
       "      <td>genblastg</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3881</th>\n",
       "      <td>genclust</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3882</th>\n",
       "      <td>gencore</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3883</th>\n",
       "      <td>genefuse</td>\n",
       "      <td>88</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3884</th>\n",
       "      <td>geneimpacts</td>\n",
       "      <td>2254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3885</th>\n",
       "      <td>genenotebook</td>\n",
       "      <td>1519</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3886</th>\n",
       "      <td>genepender</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3887</th>\n",
       "      <td>genepop</td>\n",
       "      <td>1635</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3888</th>\n",
       "      <td>generax</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3889</th>\n",
       "      <td>genericrepeatfinder</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3890</th>\n",
       "      <td>genesplicer</td>\n",
       "      <td>328</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3891</th>\n",
       "      <td>genform</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3892</th>\n",
       "      <td>genie</td>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3893</th>\n",
       "      <td>genmap</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3894</th>\n",
       "      <td>genologics</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3895</th>\n",
       "      <td>genomebaser</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3896</th>\n",
       "      <td>genomeconstellation</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3897</th>\n",
       "      <td>genomedata</td>\n",
       "      <td>645</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3898</th>\n",
       "      <td>genomelake</td>\n",
       "      <td>448</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3899</th>\n",
       "      <td>genomepy</td>\n",
       "      <td>863</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3900</th>\n",
       "      <td>genomestrip</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3901</th>\n",
       "      <td>genometester4</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3902</th>\n",
       "      <td>genomethreader</td>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3903</th>\n",
       "      <td>genometools</td>\n",
       "      <td>129</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3904</th>\n",
       "      <td>genometools-genometools</td>\n",
       "      <td>666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3905</th>\n",
       "      <td>genome_updater</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3906</th>\n",
       "      <td>genomicassertions</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3907</th>\n",
       "      <td>genomicconsensus</td>\n",
       "      <td>364</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3908</th>\n",
       "      <td>genonets</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3909</th>\n",
       "      <td>genrich</td>\n",
       "      <td>241</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3910</th>\n",
       "      <td>geodl</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3911</th>\n",
       "      <td>geosketch</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3912</th>\n",
       "      <td>gerp</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3913</th>\n",
       "      <td>gfapy</td>\n",
       "      <td>317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3914</th>\n",
       "      <td>gff3sort</td>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3915</th>\n",
       "      <td>gff3toembl</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3916</th>\n",
       "      <td>gffcompare</td>\n",
       "      <td>878</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3917</th>\n",
       "      <td>gffmunger</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3918</th>\n",
       "      <td>gffread</td>\n",
       "      <td>1034</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3919</th>\n",
       "      <td>gffutils</td>\n",
       "      <td>2548</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3920</th>\n",
       "      <td>gfinisher</td>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3921</th>\n",
       "      <td>gfold</td>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3922</th>\n",
       "      <td>ggd</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3923</th>\n",
       "      <td>ghm</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3924</th>\n",
       "      <td>ghmm</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3925</th>\n",
       "      <td>ghost-tree</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3926</th>\n",
       "      <td>ghostx</td>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3927</th>\n",
       "      <td>ghostz</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3928</th>\n",
       "      <td>gimmemotifs</td>\n",
       "      <td>1812</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3929</th>\n",
       "      <td>glimmer</td>\n",
       "      <td>212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3930</th>\n",
       "      <td>glimmerhmm</td>\n",
       "      <td>1076</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3931</th>\n",
       "      <td>gmap</td>\n",
       "      <td>2006</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3932</th>\n",
       "      <td>gmap-fusion</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3933</th>\n",
       "      <td>gmcloser</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3934</th>\n",
       "      <td>gmtk</td>\n",
       "      <td>137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3935</th>\n",
       "      <td>gneiss</td>\n",
       "      <td>2516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3936</th>\n",
       "      <td>gnomic</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3937</th>\n",
       "      <td>gnu-getopt</td>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3938</th>\n",
       "      <td>gnuplot-py</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3939</th>\n",
       "      <td>gnu-wget</td>\n",
       "      <td>141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3940</th>\n",
       "      <td>goalign</td>\n",
       "      <td>340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3941</th>\n",
       "      <td>goatools</td>\n",
       "      <td>711</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3942</th>\n",
       "      <td>gocr</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3943</th>\n",
       "      <td>goenrichment</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3944</th>\n",
       "      <td>goleft</td>\n",
       "      <td>1781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3945</th>\n",
       "      <td>goslimmer</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3946</th>\n",
       "      <td>gotohscan</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3947</th>\n",
       "      <td>gotree</td>\n",
       "      <td>139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3948</th>\n",
       "      <td>gottcha</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3949</th>\n",
       "      <td>gprofiler-official</td>\n",
       "      <td>120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3950</th>\n",
       "      <td>gqt</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3951</th>\n",
       "      <td>grabix</td>\n",
       "      <td>653</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3952</th>\n",
       "      <td>graftm</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3953</th>\n",
       "      <td>graph2pro-var</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3954</th>\n",
       "      <td>graphaligner</td>\n",
       "      <td>963</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3955</th>\n",
       "      <td>graphclust-wrappers</td>\n",
       "      <td>1447</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3956</th>\n",
       "      <td>graph_embed</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3957</th>\n",
       "      <td>graphlan</td>\n",
       "      <td>261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3958</th>\n",
       "      <td>graphlite</td>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3959</th>\n",
       "      <td>graphmap</td>\n",
       "      <td>407</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3960</th>\n",
       "      <td>graphprot</td>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3961</th>\n",
       "      <td>grep</td>\n",
       "      <td>994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3962</th>\n",
       "      <td>gretel</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3963</th>\n",
       "      <td>grid</td>\n",
       "      <td>306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3964</th>\n",
       "      <td>gridss</td>\n",
       "      <td>1304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3965</th>\n",
       "      <td>gromacs</td>\n",
       "      <td>1390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3966</th>\n",
       "      <td>groopm</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3967</th>\n",
       "      <td>groot</td>\n",
       "      <td>1230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3968</th>\n",
       "      <td>group_humann2_uniref_abundances_to_go</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3969</th>\n",
       "      <td>gsalign</td>\n",
       "      <td>1231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3970</th>\n",
       "      <td>gseapy</td>\n",
       "      <td>1942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3971</th>\n",
       "      <td>gsort</td>\n",
       "      <td>865</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3972</th>\n",
       "      <td>gtdbtk</td>\n",
       "      <td>934</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3973</th>\n",
       "      <td>gtfparse</td>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3974</th>\n",
       "      <td>gtf_to_genes</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3975</th>\n",
       "      <td>gubbins</td>\n",
       "      <td>811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3976</th>\n",
       "      <td>guessmylt</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3977</th>\n",
       "      <td>guidescan</td>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3978</th>\n",
       "      <td>gustaf</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3979</th>\n",
       "      <td>gvcf2bed</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3980</th>\n",
       "      <td>gvcfgenotyper</td>\n",
       "      <td>314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3981</th>\n",
       "      <td>gvcf-regions</td>\n",
       "      <td>546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3982</th>\n",
       "      <td>gvcftools</td>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3983</th>\n",
       "      <td>gwama</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3984</th>\n",
       "      <td>gxformat2</td>\n",
       "      <td>1311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3985</th>\n",
       "      <td>halla</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3986</th>\n",
       "      <td>hanselx</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3987</th>\n",
       "      <td>hapbin</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3988</th>\n",
       "      <td>hapcut2</td>\n",
       "      <td>427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3989</th>\n",
       "      <td>hapflk</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3990</th>\n",
       "      <td>haploclique</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3991</th>\n",
       "      <td>haploconduct</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3992</th>\n",
       "      <td>haplotype-lso</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3993</th>\n",
       "      <td>happer</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3994</th>\n",
       "      <td>hap.py</td>\n",
       "      <td>409</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3995</th>\n",
       "      <td>hardklor</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3996</th>\n",
       "      <td>harvesttools</td>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3997</th>\n",
       "      <td>haslr</td>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3998</th>\n",
       "      <td>haystack_bio</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3999</th>\n",
       "      <td>hca</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4000</th>\n",
       "      <td>hca-matrix-downloader</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4001</th>\n",
       "      <td>hclust2</td>\n",
       "      <td>254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4002</th>\n",
       "      <td>hcluster_sg</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4003</th>\n",
       "      <td>heinz</td>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4004</th>\n",
       "      <td>helperlibs</td>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4005</th>\n",
       "      <td>hera</td>\n",
       "      <td>304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4006</th>\n",
       "      <td>hg-color</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4007</th>\n",
       "      <td>hgtools</td>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4008</th>\n",
       "      <td>hhsuite</td>\n",
       "      <td>357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4009</th>\n",
       "      <td>hic2cool</td>\n",
       "      <td>1064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4010</th>\n",
       "      <td>hicap</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4011</th>\n",
       "      <td>hicbrowser</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4012</th>\n",
       "      <td>hicexplorer</td>\n",
       "      <td>2404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4013</th>\n",
       "      <td>hichipper</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4014</th>\n",
       "      <td>hicmatrix</td>\n",
       "      <td>1371</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4015</th>\n",
       "      <td>hicup</td>\n",
       "      <td>302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4016</th>\n",
       "      <td>hiddendomains</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4017</th>\n",
       "      <td>hifive</td>\n",
       "      <td>753</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4018</th>\n",
       "      <td>hilive2</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4019</th>\n",
       "      <td>hint</td>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4020</th>\n",
       "      <td>hisat2</td>\n",
       "      <td>3302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4021</th>\n",
       "      <td>hits</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4022</th>\n",
       "      <td>hivtrace</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4023</th>\n",
       "      <td>hla-la</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4024</th>\n",
       "      <td>hlama</td>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4025</th>\n",
       "      <td>hlaprofiler</td>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4026</th>\n",
       "      <td>hmftools-amber</td>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4027</th>\n",
       "      <td>hmftools-cobalt</td>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4028</th>\n",
       "      <td>hmftools-linx</td>\n",
       "      <td>52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4029</th>\n",
       "      <td>hmftools-purple</td>\n",
       "      <td>256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4030</th>\n",
       "      <td>hmftools-sage</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4031</th>\n",
       "      <td>hmmcopy</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4032</th>\n",
       "      <td>hmmer2</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4033</th>\n",
       "      <td>hmmer</td>\n",
       "      <td>7583</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4034</th>\n",
       "      <td>hmmratac</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4035</th>\n",
       "      <td>homer</td>\n",
       "      <td>896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4036</th>\n",
       "      <td>hops</td>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4037</th>\n",
       "      <td>hotspot3d</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4038</th>\n",
       "      <td>howdesbt</td>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4039</th>\n",
       "      <td>hr2</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4040</th>\n",
       "      <td>hs-blastn</td>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4041</th>\n",
       "      <td>htsbox</td>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4042</th>\n",
       "      <td>htseq</td>\n",
       "      <td>1945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4043</th>\n",
       "      <td>htslib</td>\n",
       "      <td>61473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4044</th>\n",
       "      <td>hts-nim-tools</td>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4045</th>\n",
       "      <td>htstream</td>\n",
       "      <td>221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4046</th>\n",
       "      <td>hubward</td>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4047</th>\n",
       "      <td>hubward-all</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4048</th>\n",
       "      <td>hulk</td>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4049</th>\n",
       "      <td>humann2</td>\n",
       "      <td>807</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4050</th>\n",
       "      <td>hurry.filesize</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4051</th>\n",
       "      <td>hypercluster</td>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4052</th>\n",
       "      <td>hyphy</td>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4053</th>\n",
       "      <td>hypo</td>\n",
       "      <td>227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4054</th>\n",
       "      <td>ibdne</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4055</th>\n",
       "      <td>ibdseq</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4056</th>\n",
       "      <td>iced</td>\n",
       "      <td>322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4057</th>\n",
       "      <td>iclipro</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4058</th>\n",
       "      <td>icount</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4059</th>\n",
       "      <td>icqsol</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4060</th>\n",
       "      <td>idba</td>\n",
       "      <td>432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4061</th>\n",
       "      <td>idba_subasm</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4062</th>\n",
       "      <td>ideas</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4063</th>\n",
       "      <td>idr</td>\n",
       "      <td>509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4064</th>\n",
       "      <td>idr-py</td>\n",
       "      <td>312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4065</th>\n",
       "      <td>igblast</td>\n",
       "      <td>553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4066</th>\n",
       "      <td>ig-checkfcs</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4067</th>\n",
       "      <td>ig-checkflowtypes</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4068</th>\n",
       "      <td>igdiscover</td>\n",
       "      <td>886</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4069</th>\n",
       "      <td>ig-flowtools</td>\n",
       "      <td>720</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4070</th>\n",
       "      <td>igor_vdj</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4071</th>\n",
       "      <td>igphyml</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4072</th>\n",
       "      <td>igv</td>\n",
       "      <td>730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4073</th>\n",
       "      <td>igv-reports</td>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4074</th>\n",
       "      <td>igvtools</td>\n",
       "      <td>670</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4075</th>\n",
       "      <td>illumina-cleanup</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4076</th>\n",
       "      <td>illumina-interop</td>\n",
       "      <td>562</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4077</th>\n",
       "      <td>illuminate</td>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4078</th>\n",
       "      <td>illumina-utils</td>\n",
       "      <td>258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4079</th>\n",
       "      <td>illumiprocessor</td>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4080</th>\n",
       "      <td>imctools</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4081</th>\n",
       "      <td>imfusion</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4082</th>\n",
       "      <td>impute2</td>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4083</th>\n",
       "      <td>imseq</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4084</th>\n",
       "      <td>imsindel</td>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4085</th>\n",
       "      <td>indelfixer</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4086</th>\n",
       "      <td>infernal</td>\n",
       "      <td>2367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4087</th>\n",
       "      <td>influx_si</td>\n",
       "      <td>43</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4088</th>\n",
       "      <td>inforna</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4089</th>\n",
       "      <td>inheritance</td>\n",
       "      <td>948</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4090</th>\n",
       "      <td>insilicoseq</td>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4091</th>\n",
       "      <td>intarna</td>\n",
       "      <td>1767</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4092</th>\n",
       "      <td>intemap</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4093</th>\n",
       "      <td>interleafq</td>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4094</th>\n",
       "      <td>intermine</td>\n",
       "      <td>370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4095</th>\n",
       "      <td>intervalstats</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4096</th>\n",
       "      <td>intervaltree_bio</td>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4097</th>\n",
       "      <td>intervene</td>\n",
       "      <td>429</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4098</th>\n",
       "      <td>involucro</td>\n",
       "      <td>855</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4099</th>\n",
       "      <td>ipyrad</td>\n",
       "      <td>393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4100</th>\n",
       "      <td>ipython-cluster-helper</td>\n",
       "      <td>1680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4101</th>\n",
       "      <td>iqtree</td>\n",
       "      <td>4687</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4102</th>\n",
       "      <td>ir</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4103</th>\n",
       "      <td>irep</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4104</th>\n",
       "      <td>irfinder</td>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4105</th>\n",
       "      <td>irida-linker</td>\n",
       "      <td>633</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4106</th>\n",
       "      <td>irida-sistr-results</td>\n",
       "      <td>388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4107</th>\n",
       "      <td>irida-uploader</td>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4108</th>\n",
       "      <td>isatools</td>\n",
       "      <td>424</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4109</th>\n",
       "      <td>isescan</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4110</th>\n",
       "      <td>islandpath</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4111</th>\n",
       "      <td>ismapper</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4112</th>\n",
       "      <td>isocor</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4113</th>\n",
       "      <td>isoseq3</td>\n",
       "      <td>655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4114</th>\n",
       "      <td>ispcr</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4115</th>\n",
       "      <td>itero</td>\n",
       "      <td>595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4116</th>\n",
       "      <td>itsx</td>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4117</th>\n",
       "      <td>itsxpress</td>\n",
       "      <td>624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4118</th>\n",
       "      <td>iucn_sim</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4119</th>\n",
       "      <td>iva</td>\n",
       "      <td>305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4120</th>\n",
       "      <td>ivar</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4121</th>\n",
       "      <td>jaffa</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4122</th>\n",
       "      <td>jali</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4123</th>\n",
       "      <td>jalview</td>\n",
       "      <td>687</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4124</th>\n",
       "      <td>jamm</td>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4125</th>\n",
       "      <td>jannovar-cli</td>\n",
       "      <td>649</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4126</th>\n",
       "      <td>jarvis</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4127</th>\n",
       "      <td>jasmine</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4128</th>\n",
       "      <td>jass</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4129</th>\n",
       "      <td>jass_preprocessing</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4130</th>\n",
       "      <td>jbrowse</td>\n",
       "      <td>1166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4131</th>\n",
       "      <td>jcvi</td>\n",
       "      <td>936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4132</th>\n",
       "      <td>je-suite</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4133</th>\n",
       "      <td>jmztab-m</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4134</th>\n",
       "      <td>jobtree</td>\n",
       "      <td>46</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4135</th>\n",
       "      <td>jolytree</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4136</th>\n",
       "      <td>jpredapi</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4137</th>\n",
       "      <td>json_collect_data_source</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4138</th>\n",
       "      <td>jupyterngsplugin</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4139</th>\n",
       "      <td>justbackoff</td>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4140</th>\n",
       "      <td>jvarkit-bam2svg</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4141</th>\n",
       "      <td>jvarkit-bam2wig</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4142</th>\n",
       "      <td>jvarkit-msa2vcf</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4143</th>\n",
       "      <td>kaiju</td>\n",
       "      <td>1133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4144</th>\n",
       "      <td>kalign2</td>\n",
       "      <td>118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4145</th>\n",
       "      <td>kallisto</td>\n",
       "      <td>2958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4146</th>\n",
       "      <td>kaptive</td>\n",
       "      <td>330</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4147</th>\n",
       "      <td>karect</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4148</th>\n",
       "      <td>kart</td>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4149</th>\n",
       "      <td>karyopype</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4150</th>\n",
       "      <td>kat</td>\n",
       "      <td>675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4151</th>\n",
       "      <td>kfoots</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4152</th>\n",
       "      <td>kggseq</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4153</th>\n",
       "      <td>khmer</td>\n",
       "      <td>1068</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4154</th>\n",
       "      <td>kid</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4155</th>\n",
       "      <td>kinsimriboswitch</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4156</th>\n",
       "      <td>kipoi</td>\n",
       "      <td>158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4157</th>\n",
       "      <td>kipoiseq</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4158</th>\n",
       "      <td>kipoi-utils</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4159</th>\n",
       "      <td>kipoi_veff</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4160</th>\n",
       "      <td>kissplice</td>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4161</th>\n",
       "      <td>kiwidist</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4162</th>\n",
       "      <td>kleborate</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4163</th>\n",
       "      <td>kma</td>\n",
       "      <td>1834</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4164</th>\n",
       "      <td>kmasker</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4165</th>\n",
       "      <td>kmc</td>\n",
       "      <td>835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4166</th>\n",
       "      <td>kmergenie</td>\n",
       "      <td>130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4167</th>\n",
       "      <td>kmerinshort</td>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4168</th>\n",
       "      <td>kmer-jellyfish</td>\n",
       "      <td>1306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4169</th>\n",
       "      <td>kneaddata</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4170</th>\n",
       "      <td>knock-knock</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4171</th>\n",
       "      <td>knot</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4172</th>\n",
       "      <td>kobas</td>\n",
       "      <td>249</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4173</th>\n",
       "      <td>kodoja</td>\n",
       "      <td>230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4174</th>\n",
       "      <td>koeken</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4175</th>\n",
       "      <td>kofamscan</td>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4176</th>\n",
       "      <td>kopt</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4177</th>\n",
       "      <td>kounta</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4178</th>\n",
       "      <td>kpal</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4179</th>\n",
       "      <td>kraken</td>\n",
       "      <td>1017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4180</th>\n",
       "      <td>kraken2</td>\n",
       "      <td>752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4181</th>\n",
       "      <td>kraken-biom</td>\n",
       "      <td>137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4182</th>\n",
       "      <td>kraken-ea</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4183</th>\n",
       "      <td>krakenuniq</td>\n",
       "      <td>628</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4184</th>\n",
       "      <td>krbalancing</td>\n",
       "      <td>1208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4185</th>\n",
       "      <td>krocus</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4186</th>\n",
       "      <td>krona</td>\n",
       "      <td>1011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4187</th>\n",
       "      <td>kronik</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4188</th>\n",
       "      <td>kronos</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4189</th>\n",
       "      <td>k-slam</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4190</th>\n",
       "      <td>kssd</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4191</th>\n",
       "      <td>ksw</td>\n",
       "      <td>376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4192</th>\n",
       "      <td>kwip</td>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4193</th>\n",
       "      <td>kyototycoon</td>\n",
       "      <td>146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4194</th>\n",
       "      <td>lace</td>\n",
       "      <td>346</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4195</th>\n",
       "      <td>lambda</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4196</th>\n",
       "      <td>lap</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4197</th>\n",
       "      <td>last</td>\n",
       "      <td>1461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4198</th>\n",
       "      <td>lastz</td>\n",
       "      <td>416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4199</th>\n",
       "      <td>latentstrainanalysis</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4200</th>\n",
       "      <td>lca</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4201</th>\n",
       "      <td>lcfit</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4202</th>\n",
       "      <td>ldhelmet</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4203</th>\n",
       "      <td>ldsc</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4204</th>\n",
       "      <td>lefse</td>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4205</th>\n",
       "      <td>legsta</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4206</th>\n",
       "      <td>lexmapr</td>\n",
       "      <td>938</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4207</th>\n",
       "      <td>libbambamc</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4208</th>\n",
       "      <td>libbigwig</td>\n",
       "      <td>589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4209</th>\n",
       "      <td>libdivsufsort</td>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4210</th>\n",
       "      <td>libgenome</td>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4211</th>\n",
       "      <td>libgtextutils</td>\n",
       "      <td>537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4212</th>\n",
       "      <td>libidn</td>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4213</th>\n",
       "      <td>libis</td>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4214</th>\n",
       "      <td>libmems</td>\n",
       "      <td>217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4215</th>\n",
       "      <td>libmuscle</td>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4216</th>\n",
       "      <td>libsbml</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4217</th>\n",
       "      <td>libsequence</td>\n",
       "      <td>958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4218</th>\n",
       "      <td>libssw</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4219</th>\n",
       "      <td>libstatgen</td>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4220</th>\n",
       "      <td>lightassembler</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4221</th>\n",
       "      <td>lighter</td>\n",
       "      <td>332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4222</th>\n",
       "      <td>lightning</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4223</th>\n",
       "      <td>lightstringgraph</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4224</th>\n",
       "      <td>lima</td>\n",
       "      <td>743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4225</th>\n",
       "      <td>linkage2allegro</td>\n",
       "      <td>311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4226</th>\n",
       "      <td>links</td>\n",
       "      <td>344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4227</th>\n",
       "      <td>livekraken</td>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4228</th>\n",
       "      <td>locarna</td>\n",
       "      <td>1166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4229</th>\n",
       "      <td>locus_processing</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4230</th>\n",
       "      <td>lofreq</td>\n",
       "      <td>521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4231</th>\n",
       "      <td>logol</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4232</th>\n",
       "      <td>logomaker</td>\n",
       "      <td>138</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4233</th>\n",
       "      <td>lohhla</td>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4234</th>\n",
       "      <td>longshot</td>\n",
       "      <td>616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4235</th>\n",
       "      <td>lordec</td>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4236</th>\n",
       "      <td>lordfast</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4237</th>\n",
       "      <td>lorikeet</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4238</th>\n",
       "      <td>lorma</td>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4239</th>\n",
       "      <td>lrez</td>\n",
       "      <td>527</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4240</th>\n",
       "      <td>lrzip</td>\n",
       "      <td>1729</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4241</th>\n",
       "      <td>lrzip</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4242</th>\n",
       "      <td>lsc</td>\n",
       "      <td>1627</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4243</th>\n",
       "      <td>lsc</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4244</th>\n",
       "      <td>lsd</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4245</th>\n",
       "      <td>lsd</td>\n",
       "      <td>1911</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4246</th>\n",
       "      <td>ls-gkm</td>\n",
       "      <td>139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4247</th>\n",
       "      <td>ls-gkm</td>\n",
       "      <td>2985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4248</th>\n",
       "      <td>ltr_finder</td>\n",
       "      <td>1222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4249</th>\n",
       "      <td>ltr_finder</td>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4250</th>\n",
       "      <td>ltr_retriever</td>\n",
       "      <td>1522</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4251</th>\n",
       "      <td>ltr_retriever</td>\n",
       "      <td>53</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4252</th>\n",
       "      <td>luciphor2</td>\n",
       "      <td>45</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4253</th>\n",
       "      <td>luciphor2</td>\n",
       "      <td>7580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4254</th>\n",
       "      <td>lumpy-sv</td>\n",
       "      <td>22513</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4255</th>\n",
       "      <td>lumpy-sv</td>\n",
       "      <td>790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4256</th>\n",
       "      <td>lusstr</td>\n",
       "      <td>66</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4257</th>\n",
       "      <td>lyner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4258</th>\n",
       "      <td>lyner</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4259</th>\n",
       "      <td>lz4-bin</td>\n",
       "      <td>122</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4260</th>\n",
       "      <td>lz4-bin</td>\n",
       "      <td>8631</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4261</th>\n",
       "      <td>ma</td>\n",
       "      <td>1329</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4262</th>\n",
       "      <td>ma</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4263</th>\n",
       "      <td>maaslin</td>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4264</th>\n",
       "      <td>maaslin2</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4265</th>\n",
       "      <td>maaslin2</td>\n",
       "      <td>378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4266</th>\n",
       "      <td>maaslin</td>\n",
       "      <td>3673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4267</th>\n",
       "      <td>macaron</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4268</th>\n",
       "      <td>macaron</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4269</th>\n",
       "      <td>mace</td>\n",
       "      <td>1140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4270</th>\n",
       "      <td>mace</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4271</th>\n",
       "      <td>macrel</td>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4272</th>\n",
       "      <td>macrel</td>\n",
       "      <td>2453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4273</th>\n",
       "      <td>macs2</td>\n",
       "      <td>2064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4274</th>\n",
       "      <td>macs2</td>\n",
       "      <td>49852</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4275</th>\n",
       "      <td>macse</td>\n",
       "      <td>37</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4276</th>\n",
       "      <td>mafft</td>\n",
       "      <td>174314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4277</th>\n",
       "      <td>mafft</td>\n",
       "      <td>5345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4278</th>\n",
       "      <td>mageck</td>\n",
       "      <td>1513</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4279</th>\n",
       "      <td>mageck</td>\n",
       "      <td>31030</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4280</th>\n",
       "      <td>mageck-vispr</td>\n",
       "      <td>13335</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4281</th>\n",
       "      <td>mageck-vispr</td>\n",
       "      <td>667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4282</th>\n",
       "      <td>magicblast</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4283</th>\n",
       "      <td>magicblast</td>\n",
       "      <td>2219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4284</th>\n",
       "      <td>magpurify</td>\n",
       "      <td>3211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4285</th>\n",
       "      <td>makehub</td>\n",
       "      <td>1365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4286</th>\n",
       "      <td>maker</td>\n",
       "      <td>21204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4287</th>\n",
       "      <td>malder</td>\n",
       "      <td>1431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4288</th>\n",
       "      <td>malt</td>\n",
       "      <td>2572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4289</th>\n",
       "      <td>malva</td>\n",
       "      <td>4573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4290</th>\n",
       "      <td>mamotif</td>\n",
       "      <td>3323</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4291</th>\n",
       "      <td>mango</td>\n",
       "      <td>244</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4292</th>\n",
       "      <td>manorm</td>\n",
       "      <td>8489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4293</th>\n",
       "      <td>manta</td>\n",
       "      <td>43748</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4294</th>\n",
       "      <td>mantis</td>\n",
       "      <td>1486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4295</th>\n",
       "      <td>mapcaller</td>\n",
       "      <td>17608</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4296</th>\n",
       "      <td>mapdamage2</td>\n",
       "      <td>3044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4297</th>\n",
       "      <td>mapdia</td>\n",
       "      <td>3169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4298</th>\n",
       "      <td>mapgl</td>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4299</th>\n",
       "      <td>mapping-iterative-assembler</td>\n",
       "      <td>1270</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4300</th>\n",
       "      <td>mappy</td>\n",
       "      <td>44044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4301</th>\n",
       "      <td>mapsembler2</td>\n",
       "      <td>2169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4302</th>\n",
       "      <td>mapsplice</td>\n",
       "      <td>4582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4303</th>\n",
       "      <td>marge</td>\n",
       "      <td>2013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4304</th>\n",
       "      <td>mark-nonconverted-reads</td>\n",
       "      <td>168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4305</th>\n",
       "      <td>marvel</td>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4306</th>\n",
       "      <td>mash</td>\n",
       "      <td>32784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4307</th>\n",
       "      <td>mashmap</td>\n",
       "      <td>5341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4308</th>\n",
       "      <td>mashtree</td>\n",
       "      <td>21657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4309</th>\n",
       "      <td>maskrc-svg</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4310</th>\n",
       "      <td>mason</td>\n",
       "      <td>5302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4311</th>\n",
       "      <td>masurca</td>\n",
       "      <td>13922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4312</th>\n",
       "      <td>matam</td>\n",
       "      <td>942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4313</th>\n",
       "      <td>mathstats</td>\n",
       "      <td>4486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4314</th>\n",
       "      <td>matlock</td>\n",
       "      <td>1491</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4315</th>\n",
       "      <td>mauve</td>\n",
       "      <td>4514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4316</th>\n",
       "      <td>mauvealigner</td>\n",
       "      <td>3949</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4317</th>\n",
       "      <td>mavis</td>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4318</th>\n",
       "      <td>mawk</td>\n",
       "      <td>6831</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4319</th>\n",
       "      <td>maxbin2</td>\n",
       "      <td>11184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4320</th>\n",
       "      <td>maxentpy</td>\n",
       "      <td>4593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4321</th>\n",
       "      <td>maxentscan</td>\n",
       "      <td>7595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4322</th>\n",
       "      <td>maxquant</td>\n",
       "      <td>3944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4323</th>\n",
       "      <td>mbg</td>\n",
       "      <td>183</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4324</th>\n",
       "      <td>mbuffer</td>\n",
       "      <td>7388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4325</th>\n",
       "      <td>mccortex</td>\n",
       "      <td>2617</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4326</th>\n",
       "      <td>mcl</td>\n",
       "      <td>30556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4327</th>\n",
       "      <td>mdasim</td>\n",
       "      <td>7871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4328</th>\n",
       "      <td>mdust</td>\n",
       "      <td>2844</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4329</th>\n",
       "      <td>mea</td>\n",
       "      <td>2337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4330</th>\n",
       "      <td>medaka</td>\n",
       "      <td>29619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4331</th>\n",
       "      <td>medpy</td>\n",
       "      <td>3118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4332</th>\n",
       "      <td>medusa</td>\n",
       "      <td>178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4333</th>\n",
       "      <td>medusa-data-fusion</td>\n",
       "      <td>2185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4334</th>\n",
       "      <td>megagta</td>\n",
       "      <td>1680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4335</th>\n",
       "      <td>megahit</td>\n",
       "      <td>43017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4336</th>\n",
       "      <td>megalodon</td>\n",
       "      <td>2383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4337</th>\n",
       "      <td>megan</td>\n",
       "      <td>2393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4338</th>\n",
       "      <td>melt</td>\n",
       "      <td>2832</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4339</th>\n",
       "      <td>meme</td>\n",
       "      <td>46318</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4340</th>\n",
       "      <td>mendelscan</td>\n",
       "      <td>1801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4341</th>\n",
       "      <td>meneco</td>\n",
       "      <td>1935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4342</th>\n",
       "      <td>menetools</td>\n",
       "      <td>1955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4343</th>\n",
       "      <td>mentalist</td>\n",
       "      <td>22971</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4344</th>\n",
       "      <td>meraculous</td>\n",
       "      <td>6600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4345</th>\n",
       "      <td>mercat</td>\n",
       "      <td>3950</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4346</th>\n",
       "      <td>merfishtools</td>\n",
       "      <td>13680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4347</th>\n",
       "      <td>mergenotcombined</td>\n",
       "      <td>401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4348</th>\n",
       "      <td>mergevcf</td>\n",
       "      <td>1835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4349</th>\n",
       "      <td>merlin</td>\n",
       "      <td>3353</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4350</th>\n",
       "      <td>merqury</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4351</th>\n",
       "      <td>meryl</td>\n",
       "      <td>2168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4352</th>\n",
       "      <td>metabat2</td>\n",
       "      <td>12728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4353</th>\n",
       "      <td>metabinkit</td>\n",
       "      <td>826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4354</th>\n",
       "      <td>metacherchant</td>\n",
       "      <td>1854</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4355</th>\n",
       "      <td>metacluster</td>\n",
       "      <td>1777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4356</th>\n",
       "      <td>metaeuk</td>\n",
       "      <td>1292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4357</th>\n",
       "      <td>metagene_annotator</td>\n",
       "      <td>3960</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4358</th>\n",
       "      <td>metagenome-atlas</td>\n",
       "      <td>7499</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4359</th>\n",
       "      <td>metalign</td>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4360</th>\n",
       "      <td>metamaps</td>\n",
       "      <td>495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4361</th>\n",
       "      <td>metameta</td>\n",
       "      <td>7344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4362</th>\n",
       "      <td>metametamerge</td>\n",
       "      <td>4251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4363</th>\n",
       "      <td>metaphlan2</td>\n",
       "      <td>119507</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4364</th>\n",
       "      <td>metaphlan</td>\n",
       "      <td>2546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4365</th>\n",
       "      <td>metaphyler</td>\n",
       "      <td>1802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4366</th>\n",
       "      <td>metapop</td>\n",
       "      <td>23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4367</th>\n",
       "      <td>metaprob</td>\n",
       "      <td>1774</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4368</th>\n",
       "      <td>metaprokka</td>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4369</th>\n",
       "      <td>metaquant</td>\n",
       "      <td>1714</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4370</th>\n",
       "      <td>metaquantome</td>\n",
       "      <td>1320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4371</th>\n",
       "      <td>metaseq</td>\n",
       "      <td>4921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4372</th>\n",
       "      <td>metaseq-all</td>\n",
       "      <td>2009</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4373</th>\n",
       "      <td>metasnv</td>\n",
       "      <td>4595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4374</th>\n",
       "      <td>meta-sparse</td>\n",
       "      <td>4904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4375</th>\n",
       "      <td>metasv</td>\n",
       "      <td>8319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4376</th>\n",
       "      <td>metatree</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4377</th>\n",
       "      <td>metavelvet</td>\n",
       "      <td>4116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4378</th>\n",
       "      <td>metavelvet-annois</td>\n",
       "      <td>2013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4379</th>\n",
       "      <td>metavelvet-sl</td>\n",
       "      <td>1871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4380</th>\n",
       "      <td>metavelvet-sl-feature-extraction</td>\n",
       "      <td>2234</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4381</th>\n",
       "      <td>metavelvet-sl-pipeline</td>\n",
       "      <td>1558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4382</th>\n",
       "      <td>metawatt</td>\n",
       "      <td>1974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4383</th>\n",
       "      <td>metawrap</td>\n",
       "      <td>4248</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4384</th>\n",
       "      <td>metaxa</td>\n",
       "      <td>1826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4385</th>\n",
       "      <td>metfrag</td>\n",
       "      <td>5507</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4386</th>\n",
       "      <td>methplotlib</td>\n",
       "      <td>427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4387</th>\n",
       "      <td>methyldackel</td>\n",
       "      <td>12831</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4388</th>\n",
       "      <td>methylextract</td>\n",
       "      <td>1752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4389</th>\n",
       "      <td>methylpy</td>\n",
       "      <td>2180</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4390</th>\n",
       "      <td>metilene</td>\n",
       "      <td>6151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4391</th>\n",
       "      <td>mfqe</td>\n",
       "      <td>3152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4392</th>\n",
       "      <td>mgf-formatter</td>\n",
       "      <td>2790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4393</th>\n",
       "      <td>mgkit</td>\n",
       "      <td>9904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4394</th>\n",
       "      <td>mglex</td>\n",
       "      <td>2064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4395</th>\n",
       "      <td>mgltools</td>\n",
       "      <td>6253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4396</th>\n",
       "      <td>mgs-canopy</td>\n",
       "      <td>1869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4397</th>\n",
       "      <td>mhap</td>\n",
       "      <td>3717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4398</th>\n",
       "      <td>mhcflurry</td>\n",
       "      <td>1161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4399</th>\n",
       "      <td>mhcnames</td>\n",
       "      <td>769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4400</th>\n",
       "      <td>mhcnuggets</td>\n",
       "      <td>1843</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4401</th>\n",
       "      <td>micall-lite</td>\n",
       "      <td>4276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4402</th>\n",
       "      <td>microbecensus</td>\n",
       "      <td>3705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4403</th>\n",
       "      <td>microhapdb</td>\n",
       "      <td>400</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4404</th>\n",
       "      <td>microhapulator</td>\n",
       "      <td>332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4405</th>\n",
       "      <td>micropita</td>\n",
       "      <td>1319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4406</th>\n",
       "      <td>micro_razers</td>\n",
       "      <td>1773</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4407</th>\n",
       "      <td>midas</td>\n",
       "      <td>5962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4408</th>\n",
       "      <td>migmap</td>\n",
       "      <td>5834</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4409</th>\n",
       "      <td>migraine</td>\n",
       "      <td>3222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4410</th>\n",
       "      <td>migrate-n</td>\n",
       "      <td>1750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4411</th>\n",
       "      <td>mikado</td>\n",
       "      <td>10671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4412</th>\n",
       "      <td>mimeo</td>\n",
       "      <td>2055</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4413</th>\n",
       "      <td>mimodd</td>\n",
       "      <td>6862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4414</th>\n",
       "      <td>mimseq</td>\n",
       "      <td>120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4415</th>\n",
       "      <td>minced</td>\n",
       "      <td>35151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4416</th>\n",
       "      <td>mindthegap</td>\n",
       "      <td>1998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4417</th>\n",
       "      <td>minia</td>\n",
       "      <td>3591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4418</th>\n",
       "      <td>minialign</td>\n",
       "      <td>27269</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4419</th>\n",
       "      <td>miniasm</td>\n",
       "      <td>36543</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4420</th>\n",
       "      <td>minimap2</td>\n",
       "      <td>118765</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4421</th>\n",
       "      <td>minimap</td>\n",
       "      <td>4646</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4422</th>\n",
       "      <td>minimock</td>\n",
       "      <td>2242</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4423</th>\n",
       "      <td>minipolish</td>\n",
       "      <td>150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4424</th>\n",
       "      <td>minirmd</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4425</th>\n",
       "      <td>minnow</td>\n",
       "      <td>4042</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4426</th>\n",
       "      <td>minorseq</td>\n",
       "      <td>2440</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4427</th>\n",
       "      <td>mintmap</td>\n",
       "      <td>6147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4428</th>\n",
       "      <td>minvar</td>\n",
       "      <td>18552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4429</th>\n",
       "      <td>minys</td>\n",
       "      <td>1402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4430</th>\n",
       "      <td>mira</td>\n",
       "      <td>9216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4431</th>\n",
       "      <td>miranda</td>\n",
       "      <td>2898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4432</th>\n",
       "      <td>mirdeep2</td>\n",
       "      <td>12705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4433</th>\n",
       "      <td>mirfix</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4434</th>\n",
       "      <td>mirge</td>\n",
       "      <td>11618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4435</th>\n",
       "      <td>mir-prefer</td>\n",
       "      <td>5429</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4436</th>\n",
       "      <td>mirtop</td>\n",
       "      <td>25084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4437</th>\n",
       "      <td>mirtrace</td>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4438</th>\n",
       "      <td>misopy</td>\n",
       "      <td>2806</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4439</th>\n",
       "      <td>mitos</td>\n",
       "      <td>1272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4440</th>\n",
       "      <td>mlpy</td>\n",
       "      <td>1275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4441</th>\n",
       "      <td>mlrho</td>\n",
       "      <td>1754</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4442</th>\n",
       "      <td>mlst</td>\n",
       "      <td>27672</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4443</th>\n",
       "      <td>mmseqs2</td>\n",
       "      <td>23434</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4444</th>\n",
       "      <td>mmvc</td>\n",
       "      <td>1699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4445</th>\n",
       "      <td>mnnpy</td>\n",
       "      <td>333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4446</th>\n",
       "      <td>moabs</td>\n",
       "      <td>14268</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4447</th>\n",
       "      <td>mobidic-mpa</td>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4448</th>\n",
       "      <td>mobster</td>\n",
       "      <td>9231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4449</th>\n",
       "      <td>mob_suite</td>\n",
       "      <td>9622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4450</th>\n",
       "      <td>moca</td>\n",
       "      <td>7954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4451</th>\n",
       "      <td>mockinbird</td>\n",
       "      <td>2393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4452</th>\n",
       "      <td>modeltest-ng</td>\n",
       "      <td>2581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4453</th>\n",
       "      <td>modlamp</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4454</th>\n",
       "      <td>mofapy</td>\n",
       "      <td>538</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4455</th>\n",
       "      <td>moff</td>\n",
       "      <td>9928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4456</th>\n",
       "      <td>molpopgen-analysis</td>\n",
       "      <td>2019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4457</th>\n",
       "      <td>monocle3-cli</td>\n",
       "      <td>3251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4458</th>\n",
       "      <td>monovar</td>\n",
       "      <td>2032</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4459</th>\n",
       "      <td>moods</td>\n",
       "      <td>7790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4460</th>\n",
       "      <td>moreutils</td>\n",
       "      <td>8400</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4461</th>\n",
       "      <td>morpheus</td>\n",
       "      <td>3571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4462</th>\n",
       "      <td>mosaik</td>\n",
       "      <td>3603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4463</th>\n",
       "      <td>mosca</td>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4464</th>\n",
       "      <td>mosdepth</td>\n",
       "      <td>42114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4465</th>\n",
       "      <td>mothur</td>\n",
       "      <td>27931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4466</th>\n",
       "      <td>motifraptor</td>\n",
       "      <td>2075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4467</th>\n",
       "      <td>motifscan</td>\n",
       "      <td>4306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4468</th>\n",
       "      <td>motus</td>\n",
       "      <td>9593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4469</th>\n",
       "      <td>mp3treesim</td>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4470</th>\n",
       "      <td>mpa-portable</td>\n",
       "      <td>2479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4471</th>\n",
       "      <td>mpa-server</td>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4472</th>\n",
       "      <td>mpra-data-access-portal</td>\n",
       "      <td>90</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4473</th>\n",
       "      <td>mqc</td>\n",
       "      <td>15800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4474</th>\n",
       "      <td>mrbayes</td>\n",
       "      <td>5568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4475</th>\n",
       "      <td>mreps</td>\n",
       "      <td>4007</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4476</th>\n",
       "      <td>mrsfast</td>\n",
       "      <td>1276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4477</th>\n",
       "      <td>ms</td>\n",
       "      <td>1634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4478</th>\n",
       "      <td>ms2pip</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4479</th>\n",
       "      <td>msaboot</td>\n",
       "      <td>5937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4480</th>\n",
       "      <td>msaconverter</td>\n",
       "      <td>74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4481</th>\n",
       "      <td>msalign2</td>\n",
       "      <td>1541</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4482</th>\n",
       "      <td>msaprobs</td>\n",
       "      <td>445</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4483</th>\n",
       "      <td>msgf_plus</td>\n",
       "      <td>17142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4484</th>\n",
       "      <td>msisensor</td>\n",
       "      <td>19094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4485</th>\n",
       "      <td>msisensor-pro</td>\n",
       "      <td>3441</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4486</th>\n",
       "      <td>msms</td>\n",
       "      <td>3456</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4487</th>\n",
       "      <td>msp2db</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4488</th>\n",
       "      <td>msproteomicstools</td>\n",
       "      <td>3554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4489</th>\n",
       "      <td>msstitch</td>\n",
       "      <td>20195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4490</th>\n",
       "      <td>mtbseq</td>\n",
       "      <td>2076</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4491</th>\n",
       "      <td>mtb-snp-it</td>\n",
       "      <td>441</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4492</th>\n",
       "      <td>mtglink</td>\n",
       "      <td>75</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4493</th>\n",
       "      <td>mtnucratio</td>\n",
       "      <td>3613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4494</th>\n",
       "      <td>mtsv</td>\n",
       "      <td>5181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4495</th>\n",
       "      <td>mtsv-tools</td>\n",
       "      <td>1220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4496</th>\n",
       "      <td>mugsy</td>\n",
       "      <td>2217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4497</th>\n",
       "      <td>multigps</td>\n",
       "      <td>7714</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4498</th>\n",
       "      <td>multiqc</td>\n",
       "      <td>187074</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4499</th>\n",
       "      <td>multiqc-bcbio</td>\n",
       "      <td>47178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4500</th>\n",
       "      <td>multivcfanalyzer</td>\n",
       "      <td>2416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4501</th>\n",
       "      <td>multiz</td>\n",
       "      <td>1432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4502</th>\n",
       "      <td>mummer</td>\n",
       "      <td>38688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4503</th>\n",
       "      <td>mummer4</td>\n",
       "      <td>17301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4504</th>\n",
       "      <td>muscle</td>\n",
       "      <td>73723</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4505</th>\n",
       "      <td>muse</td>\n",
       "      <td>2539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4506</th>\n",
       "      <td>music</td>\n",
       "      <td>2000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4507</th>\n",
       "      <td>musicc</td>\n",
       "      <td>4274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4508</th>\n",
       "      <td>music-deconvolution</td>\n",
       "      <td>88</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4509</th>\n",
       "      <td>mustang</td>\n",
       "      <td>1931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4510</th>\n",
       "      <td>mutmap</td>\n",
       "      <td>819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4511</th>\n",
       "      <td>mutscan</td>\n",
       "      <td>669</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4512</th>\n",
       "      <td>mvicuna</td>\n",
       "      <td>11906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4513</th>\n",
       "      <td>mvp</td>\n",
       "      <td>3889</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4514</th>\n",
       "      <td>mwga-utils</td>\n",
       "      <td>2121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4515</th>\n",
       "      <td>mygene</td>\n",
       "      <td>27902</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4516</th>\n",
       "      <td>mykatlas</td>\n",
       "      <td>2995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4517</th>\n",
       "      <td>mykrobe</td>\n",
       "      <td>8996</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4518</th>\n",
       "      <td>myriad</td>\n",
       "      <td>3430</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4519</th>\n",
       "      <td>mzml2isa</td>\n",
       "      <td>4650</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4520</th>\n",
       "      <td>mztosqlite</td>\n",
       "      <td>4441</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4521</th>\n",
       "      <td>n50</td>\n",
       "      <td>2051</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4522</th>\n",
       "      <td>nanoblaster</td>\n",
       "      <td>1767</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4523</th>\n",
       "      <td>nanocall</td>\n",
       "      <td>11316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4524</th>\n",
       "      <td>nanocomp</td>\n",
       "      <td>17974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4525</th>\n",
       "      <td>nanocompore</td>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4526</th>\n",
       "      <td>nanofilt</td>\n",
       "      <td>19048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4527</th>\n",
       "      <td>nanoget</td>\n",
       "      <td>29192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4528</th>\n",
       "      <td>nanolyse</td>\n",
       "      <td>9098</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4529</th>\n",
       "      <td>nanomath</td>\n",
       "      <td>25320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4530</th>\n",
       "      <td>nanomod</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4531</th>\n",
       "      <td>nanonet</td>\n",
       "      <td>1906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4532</th>\n",
       "      <td>nanoplexer</td>\n",
       "      <td>1787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4533</th>\n",
       "      <td>nanoplot</td>\n",
       "      <td>30660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4534</th>\n",
       "      <td>nanoplotter</td>\n",
       "      <td>29508</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4535</th>\n",
       "      <td>nanopolish</td>\n",
       "      <td>36116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4536</th>\n",
       "      <td>nanopolishcomp</td>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4537</th>\n",
       "      <td>nanopore_simulation</td>\n",
       "      <td>1982</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4538</th>\n",
       "      <td>nanoqc</td>\n",
       "      <td>11656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4539</th>\n",
       "      <td>nanoraw</td>\n",
       "      <td>8755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4540</th>\n",
       "      <td>nanosim</td>\n",
       "      <td>9324</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4541</th>\n",
       "      <td>nanosim-h</td>\n",
       "      <td>5432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4542</th>\n",
       "      <td>nanosplit</td>\n",
       "      <td>1724</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4543</th>\n",
       "      <td>nanostat</td>\n",
       "      <td>13733</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4544</th>\n",
       "      <td>nanosv</td>\n",
       "      <td>9471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4545</th>\n",
       "      <td>nanovar</td>\n",
       "      <td>3297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4546</th>\n",
       "      <td>nasp</td>\n",
       "      <td>7287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4547</th>\n",
       "      <td>ncbi-acc-download</td>\n",
       "      <td>718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4548</th>\n",
       "      <td>ncbi-amrfinderplus</td>\n",
       "      <td>12208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4549</th>\n",
       "      <td>ncbi-genome-download</td>\n",
       "      <td>22755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4550</th>\n",
       "      <td>ncbimeta</td>\n",
       "      <td>688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4551</th>\n",
       "      <td>ncbi-ngs-sdk</td>\n",
       "      <td>35929</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4552</th>\n",
       "      <td>ncbitk</td>\n",
       "      <td>59</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4553</th>\n",
       "      <td>ncbi-util-legacy</td>\n",
       "      <td>1297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4554</th>\n",
       "      <td>ncbi-vdb</td>\n",
       "      <td>11333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4555</th>\n",
       "      <td>ncfp</td>\n",
       "      <td>2011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4556</th>\n",
       "      <td>ncls</td>\n",
       "      <td>37462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4557</th>\n",
       "      <td>ncrf</td>\n",
       "      <td>3938</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4558</th>\n",
       "      <td>ndex-python</td>\n",
       "      <td>1981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4559</th>\n",
       "      <td>neatms</td>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4560</th>\n",
       "      <td>nebulizer</td>\n",
       "      <td>86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4561</th>\n",
       "      <td>necat</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4562</th>\n",
       "      <td>negative_training_sampler</td>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4563</th>\n",
       "      <td>nemo</td>\n",
       "      <td>1475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4564</th>\n",
       "      <td>nemo-age</td>\n",
       "      <td>165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4565</th>\n",
       "      <td>neptune</td>\n",
       "      <td>5416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4566</th>\n",
       "      <td>netcdf-metadata-info</td>\n",
       "      <td>1760</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4567</th>\n",
       "      <td>networkxgmml</td>\n",
       "      <td>2077</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4568</th>\n",
       "      <td>newick_utils</td>\n",
       "      <td>4752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4569</th>\n",
       "      <td>nextflow</td>\n",
       "      <td>89664</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4570</th>\n",
       "      <td>nextgenmap</td>\n",
       "      <td>6928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4571</th>\n",
       "      <td>nf-core</td>\n",
       "      <td>1511</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4572</th>\n",
       "      <td>ngless</td>\n",
       "      <td>21918</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4573</th>\n",
       "      <td>ngmerge</td>\n",
       "      <td>2104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4574</th>\n",
       "      <td>ngmlr</td>\n",
       "      <td>13147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4575</th>\n",
       "      <td>ngs-bits</td>\n",
       "      <td>15231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4576</th>\n",
       "      <td>ngsderive</td>\n",
       "      <td>139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4577</th>\n",
       "      <td>ngs-disambiguate</td>\n",
       "      <td>19377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4578</th>\n",
       "      <td>ngsep</td>\n",
       "      <td>26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4579</th>\n",
       "      <td>ngseqbasic</td>\n",
       "      <td>1684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4580</th>\n",
       "      <td>ngshmmalign</td>\n",
       "      <td>1808</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4581</th>\n",
       "      <td>ngsutils</td>\n",
       "      <td>2819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4582</th>\n",
       "      <td>niemads</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4583</th>\n",
       "      <td>nim-falcon</td>\n",
       "      <td>10824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4584</th>\n",
       "      <td>nimnexus</td>\n",
       "      <td>2837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4585</th>\n",
       "      <td>nlstradamus</td>\n",
       "      <td>1976</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4586</th>\n",
       "      <td>nmrglue</td>\n",
       "      <td>6164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4587</th>\n",
       "      <td>nmrml2isa</td>\n",
       "      <td>1903</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4588</th>\n",
       "      <td>nmrpro</td>\n",
       "      <td>2048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4589</th>\n",
       "      <td>nonpareil</td>\n",
       "      <td>11092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4590</th>\n",
       "      <td>norns</td>\n",
       "      <td>9751</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4591</th>\n",
       "      <td>nose-capturestderr</td>\n",
       "      <td>3369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4592</th>\n",
       "      <td>novasplice</td>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4593</th>\n",
       "      <td>novoalign</td>\n",
       "      <td>29615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4594</th>\n",
       "      <td>novobreak</td>\n",
       "      <td>2222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4595</th>\n",
       "      <td>novoplasty</td>\n",
       "      <td>2275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4596</th>\n",
       "      <td>npinv</td>\n",
       "      <td>256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4597</th>\n",
       "      <td>np-likeness-scorer</td>\n",
       "      <td>2162</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4598</th>\n",
       "      <td>npyscreen</td>\n",
       "      <td>1953</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4599</th>\n",
       "      <td>nseg</td>\n",
       "      <td>2505</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4600</th>\n",
       "      <td>nspdk</td>\n",
       "      <td>3074</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4601</th>\n",
       "      <td>ntcard</td>\n",
       "      <td>3243</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4602</th>\n",
       "      <td>ntedit</td>\n",
       "      <td>1058</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4603</th>\n",
       "      <td>nthits</td>\n",
       "      <td>678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4604</th>\n",
       "      <td>ntjoin</td>\n",
       "      <td>1488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4605</th>\n",
       "      <td>nucamino</td>\n",
       "      <td>1694</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4606</th>\n",
       "      <td>nucdiff</td>\n",
       "      <td>4155</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4607</th>\n",
       "      <td>nucleoatac</td>\n",
       "      <td>4002</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4608</th>\n",
       "      <td>nucleosome_prediction</td>\n",
       "      <td>1600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4609</th>\n",
       "      <td>nudup</td>\n",
       "      <td>6931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4610</th>\n",
       "      <td>nullarbor</td>\n",
       "      <td>2227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4611</th>\n",
       "      <td>nvc</td>\n",
       "      <td>3591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4612</th>\n",
       "      <td>nxtrim</td>\n",
       "      <td>883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4613</th>\n",
       "      <td>oases</td>\n",
       "      <td>4030</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4614</th>\n",
       "      <td>obitools</td>\n",
       "      <td>6945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4615</th>\n",
       "      <td>ococo</td>\n",
       "      <td>7084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4616</th>\n",
       "      <td>ocrad</td>\n",
       "      <td>1980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4617</th>\n",
       "      <td>octopus</td>\n",
       "      <td>15638</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4618</th>\n",
       "      <td>odgi</td>\n",
       "      <td>1673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4619</th>\n",
       "      <td>odose</td>\n",
       "      <td>1598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4620</th>\n",
       "      <td>ogdf</td>\n",
       "      <td>1843</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4621</th>\n",
       "      <td>oligoarrayaux</td>\n",
       "      <td>1397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4622</th>\n",
       "      <td>oligotyping</td>\n",
       "      <td>3276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4623</th>\n",
       "      <td>omero-py</td>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4624</th>\n",
       "      <td>omssa</td>\n",
       "      <td>7798</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4625</th>\n",
       "      <td>oncofuse</td>\n",
       "      <td>9614</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4626</th>\n",
       "      <td>oncogemini</td>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4627</th>\n",
       "      <td>oncopipe</td>\n",
       "      <td>67</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4628</th>\n",
       "      <td>oncotator</td>\n",
       "      <td>1103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4629</th>\n",
       "      <td>ont-fast5-api</td>\n",
       "      <td>10166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4630</th>\n",
       "      <td>onto2nx</td>\n",
       "      <td>2746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4631</th>\n",
       "      <td>ont-tombo</td>\n",
       "      <td>22094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4632</th>\n",
       "      <td>opencontactcli</td>\n",
       "      <td>1699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4633</th>\n",
       "      <td>openduck</td>\n",
       "      <td>135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4634</th>\n",
       "      <td>openmg</td>\n",
       "      <td>2305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4635</th>\n",
       "      <td>openslide-python</td>\n",
       "      <td>6361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4636</th>\n",
       "      <td>opentargets</td>\n",
       "      <td>86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4637</th>\n",
       "      <td>opentargets-urlzsource</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4638</th>\n",
       "      <td>opentargets-validator</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4639</th>\n",
       "      <td>opsin</td>\n",
       "      <td>5885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4640</th>\n",
       "      <td>optbuild</td>\n",
       "      <td>2658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4641</th>\n",
       "      <td>optimir</td>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4642</th>\n",
       "      <td>optitype</td>\n",
       "      <td>18313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4643</th>\n",
       "      <td>optplus</td>\n",
       "      <td>2774</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4644</th>\n",
       "      <td>orfm</td>\n",
       "      <td>3889</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4645</th>\n",
       "      <td>orfquant</td>\n",
       "      <td>52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4646</th>\n",
       "      <td>orthofinder</td>\n",
       "      <td>20333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4647</th>\n",
       "      <td>orthomcl</td>\n",
       "      <td>3176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4648</th>\n",
       "      <td>osra</td>\n",
       "      <td>5639</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4649</th>\n",
       "      <td>outrigger</td>\n",
       "      <td>2215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4650</th>\n",
       "      <td>pacbio_falcon</td>\n",
       "      <td>2103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4651</th>\n",
       "      <td>pafpy</td>\n",
       "      <td>63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4652</th>\n",
       "      <td>pairix</td>\n",
       "      <td>51497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4653</th>\n",
       "      <td>pairsnp</td>\n",
       "      <td>964</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4654</th>\n",
       "      <td>pairtools</td>\n",
       "      <td>8578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4655</th>\n",
       "      <td>pal2nal</td>\n",
       "      <td>824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4656</th>\n",
       "      <td>paladin</td>\n",
       "      <td>5099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4657</th>\n",
       "      <td>pal_finder</td>\n",
       "      <td>2006</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4658</th>\n",
       "      <td>paml</td>\n",
       "      <td>80195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4659</th>\n",
       "      <td>panaroo</td>\n",
       "      <td>610</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4660</th>\n",
       "      <td>pancake</td>\n",
       "      <td>2037</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4661</th>\n",
       "      <td>pandaseq</td>\n",
       "      <td>7844</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4662</th>\n",
       "      <td>pangolin</td>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4663</th>\n",
       "      <td>panoct</td>\n",
       "      <td>1781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4664</th>\n",
       "      <td>panpasco</td>\n",
       "      <td>76</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4665</th>\n",
       "      <td>panphlan</td>\n",
       "      <td>2033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4666</th>\n",
       "      <td>pan-plaster</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4667</th>\n",
       "      <td>pantools</td>\n",
       "      <td>2155</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4668</th>\n",
       "      <td>panx</td>\n",
       "      <td>3375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4669</th>\n",
       "      <td>paraclu</td>\n",
       "      <td>336</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4670</th>\n",
       "      <td>parafly</td>\n",
       "      <td>13643</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4671</th>\n",
       "      <td>paragraph</td>\n",
       "      <td>3159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4672</th>\n",
       "      <td>parallel-fastq-dump</td>\n",
       "      <td>17800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4673</th>\n",
       "      <td>paralyzer</td>\n",
       "      <td>3001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4674</th>\n",
       "      <td>parasail-python</td>\n",
       "      <td>17766</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4675</th>\n",
       "      <td>parent-map</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4676</th>\n",
       "      <td>parsnp</td>\n",
       "      <td>3890</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4677</th>\n",
       "      <td>pasa</td>\n",
       "      <td>4919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4678</th>\n",
       "      <td>pash</td>\n",
       "      <td>1612</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4679</th>\n",
       "      <td>pasta</td>\n",
       "      <td>4476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4680</th>\n",
       "      <td>pathogen-profiler</td>\n",
       "      <td>1919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4681</th>\n",
       "      <td>pathogist</td>\n",
       "      <td>519</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4682</th>\n",
       "      <td>patholive</td>\n",
       "      <td>55</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4683</th>\n",
       "      <td>pathoscope</td>\n",
       "      <td>3725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4684</th>\n",
       "      <td>pathracer</td>\n",
       "      <td>143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4685</th>\n",
       "      <td>pathwaymatcher</td>\n",
       "      <td>2611</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4686</th>\n",
       "      <td>pauda</td>\n",
       "      <td>1707</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4687</th>\n",
       "      <td>pauvre</td>\n",
       "      <td>14450</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4688</th>\n",
       "      <td>pbalign</td>\n",
       "      <td>6283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4689</th>\n",
       "      <td>pb-assembly</td>\n",
       "      <td>15176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4690</th>\n",
       "      <td>pbbam</td>\n",
       "      <td>21989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4691</th>\n",
       "      <td>pbccs</td>\n",
       "      <td>11619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4692</th>\n",
       "      <td>pbcommand</td>\n",
       "      <td>10742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4693</th>\n",
       "      <td>pbcopper</td>\n",
       "      <td>12710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4694</th>\n",
       "      <td>pbcore</td>\n",
       "      <td>13175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4695</th>\n",
       "      <td>pbcoretools</td>\n",
       "      <td>4317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4696</th>\n",
       "      <td>pbdagcon</td>\n",
       "      <td>2221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4697</th>\n",
       "      <td>pb-dazzler</td>\n",
       "      <td>6361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4698</th>\n",
       "      <td>pb-falcon</td>\n",
       "      <td>31787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4699</th>\n",
       "      <td>pb-falconc</td>\n",
       "      <td>1868</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4700</th>\n",
       "      <td>pb-falcon-phase</td>\n",
       "      <td>1837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4701</th>\n",
       "      <td>pbgcpp</td>\n",
       "      <td>5107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4702</th>\n",
       "      <td>pbgzip</td>\n",
       "      <td>73784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4703</th>\n",
       "      <td>pbh5tools</td>\n",
       "      <td>2346</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4704</th>\n",
       "      <td>pbhoover</td>\n",
       "      <td>3318</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4705</th>\n",
       "      <td>pbipa</td>\n",
       "      <td>1453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4706</th>\n",
       "      <td>pblaa</td>\n",
       "      <td>2148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4707</th>\n",
       "      <td>pblat</td>\n",
       "      <td>943</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4708</th>\n",
       "      <td>pbmarkdup</td>\n",
       "      <td>1259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4709</th>\n",
       "      <td>pbmm2</td>\n",
       "      <td>23306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4710</th>\n",
       "      <td>pbsim</td>\n",
       "      <td>2259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4711</th>\n",
       "      <td>pbsv</td>\n",
       "      <td>11940</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4712</th>\n",
       "      <td>pbwt</td>\n",
       "      <td>1714</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4713</th>\n",
       "      <td>pdfkit</td>\n",
       "      <td>5298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4714</th>\n",
       "      <td>peakachu</td>\n",
       "      <td>3045</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4715</th>\n",
       "      <td>peakranger</td>\n",
       "      <td>3476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4716</th>\n",
       "      <td>peaksql</td>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4717</th>\n",
       "      <td>peakzilla</td>\n",
       "      <td>2358</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4718</th>\n",
       "      <td>pear</td>\n",
       "      <td>12803</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4719</th>\n",
       "      <td>peddy</td>\n",
       "      <td>31278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4720</th>\n",
       "      <td>ped_parser</td>\n",
       "      <td>3699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4721</th>\n",
       "      <td>pepgenome</td>\n",
       "      <td>129</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4722</th>\n",
       "      <td>pepnovo</td>\n",
       "      <td>9187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4723</th>\n",
       "      <td>pepquery</td>\n",
       "      <td>389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4724</th>\n",
       "      <td>pepr</td>\n",
       "      <td>5949</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4725</th>\n",
       "      <td>peptide-shaker</td>\n",
       "      <td>45590</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4726</th>\n",
       "      <td>perbase</td>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4727</th>\n",
       "      <td>percolator</td>\n",
       "      <td>12501</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4728</th>\n",
       "      <td>perl-aceperl</td>\n",
       "      <td>91047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4729</th>\n",
       "      <td>perl-acme-damn</td>\n",
       "      <td>5166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4730</th>\n",
       "      <td>perl-algorithm-cluster</td>\n",
       "      <td>9711</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4731</th>\n",
       "      <td>perl-algorithm-dependency</td>\n",
       "      <td>3683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4732</th>\n",
       "      <td>perl-algorithm-diff</td>\n",
       "      <td>82117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4733</th>\n",
       "      <td>perl-algorithm-munkres</td>\n",
       "      <td>91185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4734</th>\n",
       "      <td>perl-aliased</td>\n",
       "      <td>1987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4735</th>\n",
       "      <td>perl-apache-test</td>\n",
       "      <td>91574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4736</th>\n",
       "      <td>perl-appconfig</td>\n",
       "      <td>91165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4737</th>\n",
       "      <td>perl-app-cpanminus</td>\n",
       "      <td>256642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4738</th>\n",
       "      <td>perl-archive-extract</td>\n",
       "      <td>30932</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4739</th>\n",
       "      <td>perl-archive-tar</td>\n",
       "      <td>183935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4740</th>\n",
       "      <td>perl-archive-tar-wrapper</td>\n",
       "      <td>1194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4741</th>\n",
       "      <td>perl-archive-zip</td>\n",
       "      <td>34402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4742</th>\n",
       "      <td>perl-array-compare</td>\n",
       "      <td>94236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4743</th>\n",
       "      <td>perl-array-set</td>\n",
       "      <td>1486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4744</th>\n",
       "      <td>perl-array-utils</td>\n",
       "      <td>13317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4745</th>\n",
       "      <td>perl-attribute-handlers</td>\n",
       "      <td>4814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4746</th>\n",
       "      <td>perl-authen-sasl-saslprep</td>\n",
       "      <td>3356</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4747</th>\n",
       "      <td>perl-autodie</td>\n",
       "      <td>1685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4748</th>\n",
       "      <td>perl-autoloader</td>\n",
       "      <td>94532</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4749</th>\n",
       "      <td>perl-b</td>\n",
       "      <td>6746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4750</th>\n",
       "      <td>perl-b-debug</td>\n",
       "      <td>1130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4751</th>\n",
       "      <td>perl-b-hooks-endofscope</td>\n",
       "      <td>8354</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4752</th>\n",
       "      <td>perl-bignum</td>\n",
       "      <td>1164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4753</th>\n",
       "      <td>perl-bio-asn1-entrezgene</td>\n",
       "      <td>94397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4754</th>\n",
       "      <td>perl-bio-automatedannotation</td>\n",
       "      <td>1623</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4755</th>\n",
       "      <td>perl-bio-bigfile</td>\n",
       "      <td>1680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4756</th>\n",
       "      <td>perl-bio-cigar</td>\n",
       "      <td>2342</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4757</th>\n",
       "      <td>perl-bio-coordinate</td>\n",
       "      <td>75221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4758</th>\n",
       "      <td>perl-bio-das</td>\n",
       "      <td>171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4759</th>\n",
       "      <td>perl-bio-db-hts</td>\n",
       "      <td>45855</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4760</th>\n",
       "      <td>perl-bio-db-sam</td>\n",
       "      <td>8449</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4761</th>\n",
       "      <td>perl-bio-eutilities</td>\n",
       "      <td>2383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4762</th>\n",
       "      <td>perl-bio-featureio</td>\n",
       "      <td>89260</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4763</th>\n",
       "      <td>perl-bio-gff3</td>\n",
       "      <td>5044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4764</th>\n",
       "      <td>perl-bio-mlst-check</td>\n",
       "      <td>1998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4765</th>\n",
       "      <td>perl-bioperl</td>\n",
       "      <td>127487</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4766</th>\n",
       "      <td>perl-bioperl-core</td>\n",
       "      <td>100182</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4767</th>\n",
       "      <td>perl-bioperl-run</td>\n",
       "      <td>92658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4768</th>\n",
       "      <td>perl-bio-phylo</td>\n",
       "      <td>93600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4769</th>\n",
       "      <td>perl-bio-phylo-cipres</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4770</th>\n",
       "      <td>perl-bio-rna-rnaalisplit</td>\n",
       "      <td>5756</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4771</th>\n",
       "      <td>perl-biosails</td>\n",
       "      <td>1319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4772</th>\n",
       "      <td>perl-bio-samtools</td>\n",
       "      <td>88294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4773</th>\n",
       "      <td>perl-bio-tools-phylo-paml</td>\n",
       "      <td>75262</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4774</th>\n",
       "      <td>perl-bio-tools-run-alignment-clustalw</td>\n",
       "      <td>75208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4775</th>\n",
       "      <td>perl-bio-tools-run-alignment-tcoffee</td>\n",
       "      <td>72117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4776</th>\n",
       "      <td>perl-bio-trace-abif</td>\n",
       "      <td>581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4777</th>\n",
       "      <td>perl-bio-viennangs</td>\n",
       "      <td>6521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4778</th>\n",
       "      <td>perl-biox-seq</td>\n",
       "      <td>2055</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4779</th>\n",
       "      <td>perl-biox-workflow-command</td>\n",
       "      <td>7872</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4780</th>\n",
       "      <td>perl-bit-vector</td>\n",
       "      <td>5553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4781</th>\n",
       "      <td>perl-blib</td>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4782</th>\n",
       "      <td>perl-bloom-faster</td>\n",
       "      <td>1884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4783</th>\n",
       "      <td>perl-border-style</td>\n",
       "      <td>2063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4784</th>\n",
       "      <td>perl-btlib</td>\n",
       "      <td>1944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4785</th>\n",
       "      <td>perl-bundle-bioperl</td>\n",
       "      <td>1755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4786</th>\n",
       "      <td>perl-business-isbn</td>\n",
       "      <td>180995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4787</th>\n",
       "      <td>perl-business-isbn-data</td>\n",
       "      <td>180987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4788</th>\n",
       "      <td>perl-cache-cache</td>\n",
       "      <td>79812</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4789</th>\n",
       "      <td>perl-cairo</td>\n",
       "      <td>1736</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4790</th>\n",
       "      <td>perl-canary-stability</td>\n",
       "      <td>5907</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4791</th>\n",
       "      <td>perl-capture-tiny</td>\n",
       "      <td>97832</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4792</th>\n",
       "      <td>perl-carp</td>\n",
       "      <td>316240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4793</th>\n",
       "      <td>perl-carp-clan</td>\n",
       "      <td>3987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4794</th>\n",
       "      <td>perl-cgi</td>\n",
       "      <td>121797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4795</th>\n",
       "      <td>perl-cg-pipeline</td>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4796</th>\n",
       "      <td>perl-class-accessor</td>\n",
       "      <td>3266</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4797</th>\n",
       "      <td>perl-class-data-inheritable</td>\n",
       "      <td>83971</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4798</th>\n",
       "      <td>perl-class-inspector</td>\n",
       "      <td>97153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4799</th>\n",
       "      <td>perl-class-load</td>\n",
       "      <td>87930</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4800</th>\n",
       "      <td>perl-class-load-xs</td>\n",
       "      <td>89272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4801</th>\n",
       "      <td>perl-class-methodmaker</td>\n",
       "      <td>2061</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4802</th>\n",
       "      <td>perl-class-method-modifiers</td>\n",
       "      <td>101876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4803</th>\n",
       "      <td>perl-class-singleton</td>\n",
       "      <td>1544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4804</th>\n",
       "      <td>perl-class-std</td>\n",
       "      <td>2033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4805</th>\n",
       "      <td>perl-class-std-fast</td>\n",
       "      <td>1905</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4806</th>\n",
       "      <td>perl-class-xsaccessor</td>\n",
       "      <td>2293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4807</th>\n",
       "      <td>perl-clone</td>\n",
       "      <td>114111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4808</th>\n",
       "      <td>perl-clone-choose</td>\n",
       "      <td>7044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4809</th>\n",
       "      <td>perl-clone-pp</td>\n",
       "      <td>3401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4810</th>\n",
       "      <td>perl-color-theme</td>\n",
       "      <td>3517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4811</th>\n",
       "      <td>perl-common-sense</td>\n",
       "      <td>194862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4812</th>\n",
       "      <td>perl-compress-bgzf</td>\n",
       "      <td>2049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4813</th>\n",
       "      <td>perl-compress-raw-bzip2</td>\n",
       "      <td>229972</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4814</th>\n",
       "      <td>perl-compress-raw-zlib</td>\n",
       "      <td>262613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4815</th>\n",
       "      <td>perl-config-any</td>\n",
       "      <td>4874</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4816</th>\n",
       "      <td>perl-config-autoconf</td>\n",
       "      <td>3362</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4817</th>\n",
       "      <td>perl-config-general</td>\n",
       "      <td>30726</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4818</th>\n",
       "      <td>perl-config-simple</td>\n",
       "      <td>3695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4819</th>\n",
       "      <td>perl-config-tiny</td>\n",
       "      <td>1889</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4820</th>\n",
       "      <td>perl-constant</td>\n",
       "      <td>220149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4821</th>\n",
       "      <td>perl-const-fast</td>\n",
       "      <td>2926</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4822</th>\n",
       "      <td>perl-convert-binary-c</td>\n",
       "      <td>92837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4823</th>\n",
       "      <td>perl-convert-binhex</td>\n",
       "      <td>91670</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4824</th>\n",
       "      <td>perl-cpan-meta</td>\n",
       "      <td>41828</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4825</th>\n",
       "      <td>perl-cpan-meta-check</td>\n",
       "      <td>8419</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4826</th>\n",
       "      <td>perl-cpan-meta-requirements</td>\n",
       "      <td>37325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4827</th>\n",
       "      <td>perl-cpan-meta-validator</td>\n",
       "      <td>1738</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4828</th>\n",
       "      <td>perl-cpan-meta-yaml</td>\n",
       "      <td>35229</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4829</th>\n",
       "      <td>perl-cpan-shell</td>\n",
       "      <td>2601</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4830</th>\n",
       "      <td>perl-crypt-openssl-random</td>\n",
       "      <td>1683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4831</th>\n",
       "      <td>perl-crypt-openssl-rsa</td>\n",
       "      <td>1670</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4832</th>\n",
       "      <td>perl-crypt-rc4</td>\n",
       "      <td>91188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4833</th>\n",
       "      <td>perl-data-compare</td>\n",
       "      <td>2988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4834</th>\n",
       "      <td>perl-data-dump</td>\n",
       "      <td>2549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4835</th>\n",
       "      <td>perl-data-dumper</td>\n",
       "      <td>236600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4836</th>\n",
       "      <td>perl-data-lock</td>\n",
       "      <td>74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4837</th>\n",
       "      <td>perl-data-match</td>\n",
       "      <td>2814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4838</th>\n",
       "      <td>perl-data-munge</td>\n",
       "      <td>1569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4839</th>\n",
       "      <td>perl-data-optlist</td>\n",
       "      <td>92281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4840</th>\n",
       "      <td>perl-data-predicate</td>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4841</th>\n",
       "      <td>perl-data-stag</td>\n",
       "      <td>93048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4842</th>\n",
       "      <td>perl-data-utilities</td>\n",
       "      <td>1744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4843</th>\n",
       "      <td>perl-data-uuid</td>\n",
       "      <td>8988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4844</th>\n",
       "      <td>perl-data-visitor</td>\n",
       "      <td>1983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4845</th>\n",
       "      <td>perl-data-walk</td>\n",
       "      <td>1948</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4846</th>\n",
       "      <td>perl-date-format</td>\n",
       "      <td>79203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4847</th>\n",
       "      <td>perl-date-manip</td>\n",
       "      <td>7564</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4848</th>\n",
       "      <td>perl-datetime</td>\n",
       "      <td>6498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4849</th>\n",
       "      <td>perl-datetime-format-strptime</td>\n",
       "      <td>3489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4850</th>\n",
       "      <td>perl-datetime-locale</td>\n",
       "      <td>6425</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4851</th>\n",
       "      <td>perl-datetime-timezone</td>\n",
       "      <td>6422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4852</th>\n",
       "      <td>perl-dbd-mysql</td>\n",
       "      <td>46038</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4853</th>\n",
       "      <td>perl-dbd-pg</td>\n",
       "      <td>9457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4854</th>\n",
       "      <td>perl-dbd-sqlite</td>\n",
       "      <td>104450</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4855</th>\n",
       "      <td>perl-db-file</td>\n",
       "      <td>113622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4856</th>\n",
       "      <td>perl-dbi</td>\n",
       "      <td>136618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4857</th>\n",
       "      <td>perl-dbm-deep</td>\n",
       "      <td>2154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4858</th>\n",
       "      <td>perl-devel-assert</td>\n",
       "      <td>1754</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4859</th>\n",
       "      <td>perl-devel-checkbin</td>\n",
       "      <td>2273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4860</th>\n",
       "      <td>perl-devel-checkos</td>\n",
       "      <td>1230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4861</th>\n",
       "      <td>perl-devel-cover</td>\n",
       "      <td>1186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4862</th>\n",
       "      <td>perl-devel-cycle</td>\n",
       "      <td>1631</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4863</th>\n",
       "      <td>perl-devel-globaldestruction</td>\n",
       "      <td>103763</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4864</th>\n",
       "      <td>perl-devel-overloadinfo</td>\n",
       "      <td>85669</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4865</th>\n",
       "      <td>perl-devel-size</td>\n",
       "      <td>7607</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4866</th>\n",
       "      <td>perl-devel-stacktrace</td>\n",
       "      <td>99551</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4867</th>\n",
       "      <td>perl-devel-symdump</td>\n",
       "      <td>9039</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4868</th>\n",
       "      <td>perl-digest-crc32</td>\n",
       "      <td>4974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4869</th>\n",
       "      <td>perl-digest-hmac</td>\n",
       "      <td>190957</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4870</th>\n",
       "      <td>perl-digest-md5</td>\n",
       "      <td>167946</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4871</th>\n",
       "      <td>perl-digest-md5-file</td>\n",
       "      <td>9816</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4872</th>\n",
       "      <td>perl-digest-perl-md5</td>\n",
       "      <td>104292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4873</th>\n",
       "      <td>perl-digest-sha</td>\n",
       "      <td>1754</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4874</th>\n",
       "      <td>perl-digest-sha1</td>\n",
       "      <td>81192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4875</th>\n",
       "      <td>perl-dist-checkconflicts</td>\n",
       "      <td>88864</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4876</th>\n",
       "      <td>perl-dumbbench</td>\n",
       "      <td>1187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4877</th>\n",
       "      <td>perl-dynaloader</td>\n",
       "      <td>93137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4878</th>\n",
       "      <td>perl-email-date-format</td>\n",
       "      <td>92325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4879</th>\n",
       "      <td>perl-email-simple</td>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4880</th>\n",
       "      <td>perl-encode</td>\n",
       "      <td>220236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4881</th>\n",
       "      <td>perl-encode-locale</td>\n",
       "      <td>198252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4882</th>\n",
       "      <td>perl-ensembl-api</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4883</th>\n",
       "      <td>perl-ensembl-compara</td>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4884</th>\n",
       "      <td>perl-ensembl-core</td>\n",
       "      <td>253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4885</th>\n",
       "      <td>perl-ensembl-funcgen</td>\n",
       "      <td>217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4886</th>\n",
       "      <td>perl-ensembl-genomes</td>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4887</th>\n",
       "      <td>perl-ensembl-io</td>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4888</th>\n",
       "      <td>perl-ensembl-variation</td>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4889</th>\n",
       "      <td>perl-env</td>\n",
       "      <td>1870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4890</th>\n",
       "      <td>perl-env-path</td>\n",
       "      <td>5236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4891</th>\n",
       "      <td>perl-error</td>\n",
       "      <td>94005</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4892</th>\n",
       "      <td>perl-escape-houdini</td>\n",
       "      <td>1221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4893</th>\n",
       "      <td>perl-estscan1</td>\n",
       "      <td>1699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4894</th>\n",
       "      <td>perl-estscan2</td>\n",
       "      <td>1958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4895</th>\n",
       "      <td>perl-eval-closure</td>\n",
       "      <td>90096</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4896</th>\n",
       "      <td>perl-excel-writer-xlsx</td>\n",
       "      <td>5241</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4897</th>\n",
       "      <td>perl-exception-class</td>\n",
       "      <td>85919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4898</th>\n",
       "      <td>perl-experimental</td>\n",
       "      <td>1728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4899</th>\n",
       "      <td>perl-exporter</td>\n",
       "      <td>311641</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4900</th>\n",
       "      <td>perl-exporter-tidy</td>\n",
       "      <td>1739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4901</th>\n",
       "      <td>perl-exporter-tiny</td>\n",
       "      <td>259127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4902</th>\n",
       "      <td>perl-extutils-cbuilder</td>\n",
       "      <td>38990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4903</th>\n",
       "      <td>perl-extutils-config</td>\n",
       "      <td>2277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4904</th>\n",
       "      <td>perl-extutils-constant</td>\n",
       "      <td>3020</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4905</th>\n",
       "      <td>perl-extutils-cppguess</td>\n",
       "      <td>17689</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4906</th>\n",
       "      <td>perl-extutils-depends</td>\n",
       "      <td>3091</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4907</th>\n",
       "      <td>perl-extutils-helpers</td>\n",
       "      <td>5095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4908</th>\n",
       "      <td>perl-extutils-installpaths</td>\n",
       "      <td>3597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4909</th>\n",
       "      <td>perl-extutils-makemaker</td>\n",
       "      <td>330596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4910</th>\n",
       "      <td>perl-extutils-manifest</td>\n",
       "      <td>42384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4911</th>\n",
       "      <td>perl-extutils-parsexs</td>\n",
       "      <td>40488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4912</th>\n",
       "      <td>perl-extutils-pkgconfig</td>\n",
       "      <td>1941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4913</th>\n",
       "      <td>perl-fastx-abi</td>\n",
       "      <td>615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4914</th>\n",
       "      <td>perl-fastx-reader</td>\n",
       "      <td>2415</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4915</th>\n",
       "      <td>perl-file-chdir</td>\n",
       "      <td>2350</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4916</th>\n",
       "      <td>perl-file-compare</td>\n",
       "      <td>1969</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4917</th>\n",
       "      <td>perl-file-copy-link</td>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4918</th>\n",
       "      <td>perl-file-copy-recursive</td>\n",
       "      <td>31912</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4919</th>\n",
       "      <td>perl-file-copy-recursive-reduced</td>\n",
       "      <td>1544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4920</th>\n",
       "      <td>perl-file-details</td>\n",
       "      <td>1947</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4921</th>\n",
       "      <td>perl-filedirutil</td>\n",
       "      <td>1924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4922</th>\n",
       "      <td>perl-file-fetch</td>\n",
       "      <td>30484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4923</th>\n",
       "      <td>perl-file-find</td>\n",
       "      <td>1980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4924</th>\n",
       "      <td>perl-file-find-rule</td>\n",
       "      <td>16942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4925</th>\n",
       "      <td>perl-file-grep</td>\n",
       "      <td>10755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4926</th>\n",
       "      <td>perl-file-homedir</td>\n",
       "      <td>5864</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4927</th>\n",
       "      <td>perl-file-listing</td>\n",
       "      <td>187767</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4928</th>\n",
       "      <td>perl-file-next</td>\n",
       "      <td>4732</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4929</th>\n",
       "      <td>perl-file-path</td>\n",
       "      <td>220225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4930</th>\n",
       "      <td>perl-file-pushd</td>\n",
       "      <td>1733</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4931</th>\n",
       "      <td>perl-file-readbackwards</td>\n",
       "      <td>4929</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4932</th>\n",
       "      <td>perl-file-remove</td>\n",
       "      <td>1970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4933</th>\n",
       "      <td>perl-file-share</td>\n",
       "      <td>3327</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4934</th>\n",
       "      <td>perl-file-sharedir</td>\n",
       "      <td>32273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4935</th>\n",
       "      <td>perl-file-sharedir-install</td>\n",
       "      <td>35651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4936</th>\n",
       "      <td>perl-file-slurp</td>\n",
       "      <td>13981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4937</th>\n",
       "      <td>perl-file-slurper</td>\n",
       "      <td>12398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4938</th>\n",
       "      <td>perl-file-slurp-tiny</td>\n",
       "      <td>93455</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4939</th>\n",
       "      <td>perl-file-sort</td>\n",
       "      <td>88187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4940</th>\n",
       "      <td>perl-file-spec</td>\n",
       "      <td>11280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4941</th>\n",
       "      <td>perl-filesys-df</td>\n",
       "      <td>11290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4942</th>\n",
       "      <td>perl-file-tee</td>\n",
       "      <td>2300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4943</th>\n",
       "      <td>perl-file-temp</td>\n",
       "      <td>209533</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4944</th>\n",
       "      <td>perl-file-touch</td>\n",
       "      <td>1770</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4945</th>\n",
       "      <td>perl-file-util</td>\n",
       "      <td>2319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4946</th>\n",
       "      <td>perl-file-which</td>\n",
       "      <td>114116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4947</th>\n",
       "      <td>perl-filter-simple</td>\n",
       "      <td>1761</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4948</th>\n",
       "      <td>perl-findbin</td>\n",
       "      <td>11258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4949</th>\n",
       "      <td>perl-findbin-libs</td>\n",
       "      <td>3560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4950</th>\n",
       "      <td>perl-findbin-real</td>\n",
       "      <td>1574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4951</th>\n",
       "      <td>perl-font-afm</td>\n",
       "      <td>91898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4952</th>\n",
       "      <td>perl-font-ttf</td>\n",
       "      <td>99495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4953</th>\n",
       "      <td>perl-forks</td>\n",
       "      <td>5647</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4954</th>\n",
       "      <td>perl-gd</td>\n",
       "      <td>147090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4955</th>\n",
       "      <td>perl-gdgraph</td>\n",
       "      <td>52384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4956</th>\n",
       "      <td>perl-gdgraph-histogram</td>\n",
       "      <td>27932</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4957</th>\n",
       "      <td>perl-gd-svg</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4958</th>\n",
       "      <td>perl-gdtextutil</td>\n",
       "      <td>50657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4959</th>\n",
       "      <td>perl-getopt-argvfile</td>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4960</th>\n",
       "      <td>perl-getopt-long</td>\n",
       "      <td>123163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4961</th>\n",
       "      <td>perl-getopt-long-descriptive</td>\n",
       "      <td>2057</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4962</th>\n",
       "      <td>perl-gfacs</td>\n",
       "      <td>30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4963</th>\n",
       "      <td>perl-git-wrapper</td>\n",
       "      <td>3798</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4964</th>\n",
       "      <td>perl-git-wrapper-plus</td>\n",
       "      <td>3550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4965</th>\n",
       "      <td>perl-go-perl</td>\n",
       "      <td>3100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4966</th>\n",
       "      <td>perl-graph</td>\n",
       "      <td>93377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4967</th>\n",
       "      <td>perl-graphics-colornames</td>\n",
       "      <td>4671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4968</th>\n",
       "      <td>perl-graphics-colorobject</td>\n",
       "      <td>4673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4969</th>\n",
       "      <td>perl-graph-readwrite</td>\n",
       "      <td>10414</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4970</th>\n",
       "      <td>perl-graphviz</td>\n",
       "      <td>93671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4971</th>\n",
       "      <td>perl-hash-merge</td>\n",
       "      <td>10380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4972</th>\n",
       "      <td>perl-hash-util-fieldhash-compat</td>\n",
       "      <td>1949</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4973</th>\n",
       "      <td>perl-heap</td>\n",
       "      <td>1677</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4974</th>\n",
       "      <td>perl-heap-simple</td>\n",
       "      <td>4777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4975</th>\n",
       "      <td>perl-heap-simple-perl</td>\n",
       "      <td>4857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4976</th>\n",
       "      <td>perl-heap-simple-xs</td>\n",
       "      <td>5060</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4977</th>\n",
       "      <td>perl-hook-lexwrap</td>\n",
       "      <td>2162</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4978</th>\n",
       "      <td>perl-hpc-runner-command</td>\n",
       "      <td>21989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4979</th>\n",
       "      <td>perl-hpc-runner-command-plugin-logger-sqlite</td>\n",
       "      <td>1820</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4980</th>\n",
       "      <td>perl-html-element-extended</td>\n",
       "      <td>91000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4981</th>\n",
       "      <td>perl-html-entities-numbered</td>\n",
       "      <td>91765</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4982</th>\n",
       "      <td>perl-html-form</td>\n",
       "      <td>2693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4983</th>\n",
       "      <td>perl-html-formatter</td>\n",
       "      <td>93783</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4984</th>\n",
       "      <td>perl-html-parser</td>\n",
       "      <td>204107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4985</th>\n",
       "      <td>perl-html-tableextract</td>\n",
       "      <td>91010</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4986</th>\n",
       "      <td>perl-html-tagset</td>\n",
       "      <td>201341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4987</th>\n",
       "      <td>perl-html-template</td>\n",
       "      <td>22164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4988</th>\n",
       "      <td>perl-html-tidy</td>\n",
       "      <td>93685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4989</th>\n",
       "      <td>perl-html-tree</td>\n",
       "      <td>187437</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4990</th>\n",
       "      <td>perl-html-treebuilder-xpath</td>\n",
       "      <td>91107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4991</th>\n",
       "      <td>perl-http-cookies</td>\n",
       "      <td>189959</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4992</th>\n",
       "      <td>perl-http-daemon</td>\n",
       "      <td>187757</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4993</th>\n",
       "      <td>perl-http-date</td>\n",
       "      <td>190549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4994</th>\n",
       "      <td>perl-http-message</td>\n",
       "      <td>190192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4995</th>\n",
       "      <td>perl-http-negotiate</td>\n",
       "      <td>189739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4996</th>\n",
       "      <td>perl-http-server-simple</td>\n",
       "      <td>2065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4997</th>\n",
       "      <td>perl-http-tiny</td>\n",
       "      <td>214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4998</th>\n",
       "      <td>perl-if</td>\n",
       "      <td>44</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4999</th>\n",
       "      <td>perl-image-exiftool</td>\n",
       "      <td>6708</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5000</th>\n",
       "      <td>perl-image-info</td>\n",
       "      <td>91184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5001</th>\n",
       "      <td>perl-image-size</td>\n",
       "      <td>91277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5002</th>\n",
       "      <td>perl-importer</td>\n",
       "      <td>20006</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5003</th>\n",
       "      <td>perl-import-into</td>\n",
       "      <td>1650</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5004</th>\n",
       "      <td>perl-indirect</td>\n",
       "      <td>1778</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5005</th>\n",
       "      <td>perl-inline</td>\n",
       "      <td>8219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5006</th>\n",
       "      <td>perl-inline-c</td>\n",
       "      <td>7387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5007</th>\n",
       "      <td>perl-integer</td>\n",
       "      <td>1739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5008</th>\n",
       "      <td>perl-io-all</td>\n",
       "      <td>7774</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5009</th>\n",
       "      <td>perl-io-compress</td>\n",
       "      <td>229748</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5010</th>\n",
       "      <td>perl-io-compress-deflate</td>\n",
       "      <td>1685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5011</th>\n",
       "      <td>perl-io-gzip</td>\n",
       "      <td>12452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5012</th>\n",
       "      <td>perl-io-handle</td>\n",
       "      <td>3801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5013</th>\n",
       "      <td>perl-io-html</td>\n",
       "      <td>190396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5014</th>\n",
       "      <td>perl-io-interactive</td>\n",
       "      <td>4930</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5015</th>\n",
       "      <td>perl-io-null</td>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5016</th>\n",
       "      <td>perl-io-prompt</td>\n",
       "      <td>5348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5017</th>\n",
       "      <td>perl-io-scalar</td>\n",
       "      <td>1789</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5018</th>\n",
       "      <td>perl-io-sessiondata</td>\n",
       "      <td>91486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5019</th>\n",
       "      <td>perl-io-socket-inet6</td>\n",
       "      <td>1936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5020</th>\n",
       "      <td>perl-io-socket-ssl</td>\n",
       "      <td>194780</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5021</th>\n",
       "      <td>perl-io-string</td>\n",
       "      <td>110358</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5022</th>\n",
       "      <td>perl-io-stringy</td>\n",
       "      <td>91783</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5023</th>\n",
       "      <td>perl-io-tty</td>\n",
       "      <td>93230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5024</th>\n",
       "      <td>perl-io-uncompress-bunzip2</td>\n",
       "      <td>1880</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5025</th>\n",
       "      <td>perl-io-uncompress-gunzip</td>\n",
       "      <td>4306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5026</th>\n",
       "      <td>perl-io-uncompress-rawinflate</td>\n",
       "      <td>1718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5027</th>\n",
       "      <td>perl-io-zlib</td>\n",
       "      <td>177731</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5028</th>\n",
       "      <td>perl-ipc-cmd</td>\n",
       "      <td>63204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5029</th>\n",
       "      <td>perl-ipc-run3</td>\n",
       "      <td>1582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5030</th>\n",
       "      <td>perl-ipc-run</td>\n",
       "      <td>95251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5031</th>\n",
       "      <td>perl-ipc-sharelite</td>\n",
       "      <td>81362</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5032</th>\n",
       "      <td>perl-ipc-system-simple</td>\n",
       "      <td>29888</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5033</th>\n",
       "      <td>perl-jcode</td>\n",
       "      <td>92240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5034</th>\n",
       "      <td>perl-json</td>\n",
       "      <td>192951</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5035</th>\n",
       "      <td>perl-json-create</td>\n",
       "      <td>1793</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5036</th>\n",
       "      <td>perl-json-maybexs</td>\n",
       "      <td>3818</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5037</th>\n",
       "      <td>perl-json-parse</td>\n",
       "      <td>3719</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5038</th>\n",
       "      <td>perl-json-pp</td>\n",
       "      <td>40737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5039</th>\n",
       "      <td>perl-json-xs</td>\n",
       "      <td>201633</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5040</th>\n",
       "      <td>perl-lib</td>\n",
       "      <td>83391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5041</th>\n",
       "      <td>perl-libwww-perl</td>\n",
       "      <td>193203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5042</th>\n",
       "      <td>perl-libxml-perl</td>\n",
       "      <td>93352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5043</th>\n",
       "      <td>perl-list-compare</td>\n",
       "      <td>2105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5044</th>\n",
       "      <td>perl-list-moreutils</td>\n",
       "      <td>260626</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5045</th>\n",
       "      <td>perl-list-moreutils-xs</td>\n",
       "      <td>219721</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5046</th>\n",
       "      <td>perl-list-someutils</td>\n",
       "      <td>3780</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5047</th>\n",
       "      <td>perl-list-uniq</td>\n",
       "      <td>2179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5048</th>\n",
       "      <td>perl-locale</td>\n",
       "      <td>1752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5049</th>\n",
       "      <td>perl-locale-maketext-simple</td>\n",
       "      <td>62273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5050</th>\n",
       "      <td>perl-local-lib</td>\n",
       "      <td>3468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5051</th>\n",
       "      <td>perl-lockfile-simple</td>\n",
       "      <td>176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5052</th>\n",
       "      <td>perl-log-any</td>\n",
       "      <td>1757</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5053</th>\n",
       "      <td>perl-logger-simple</td>\n",
       "      <td>3959</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5054</th>\n",
       "      <td>perl-log-log4perl</td>\n",
       "      <td>25395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5055</th>\n",
       "      <td>perl-lwp-mediatypes</td>\n",
       "      <td>193795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5056</th>\n",
       "      <td>perl-lwp-protocol-https</td>\n",
       "      <td>184087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5057</th>\n",
       "      <td>perl-lwp-simple</td>\n",
       "      <td>104368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5058</th>\n",
       "      <td>perl-lyve-set</td>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5059</th>\n",
       "      <td>perl-mac-propertylist</td>\n",
       "      <td>1596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5060</th>\n",
       "      <td>perl-mac-systemdirectory</td>\n",
       "      <td>1393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5061</th>\n",
       "      <td>perl-mailtools</td>\n",
       "      <td>94965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5062</th>\n",
       "      <td>perl-math-base-convert</td>\n",
       "      <td>2435</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5063</th>\n",
       "      <td>perl-math-bezier</td>\n",
       "      <td>28922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5064</th>\n",
       "      <td>perl-math-bigint</td>\n",
       "      <td>3070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5065</th>\n",
       "      <td>perl-math-bigrat</td>\n",
       "      <td>1137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5066</th>\n",
       "      <td>perl-math-cdf</td>\n",
       "      <td>103084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5067</th>\n",
       "      <td>perl-math-combinatorics</td>\n",
       "      <td>1833</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5068</th>\n",
       "      <td>perl-math-complex</td>\n",
       "      <td>1705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5069</th>\n",
       "      <td>perl-math-derivative</td>\n",
       "      <td>93593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5070</th>\n",
       "      <td>perl-math-matrix</td>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5071</th>\n",
       "      <td>perl-math-matrixreal</td>\n",
       "      <td>80</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5072</th>\n",
       "      <td>perl-math-random</td>\n",
       "      <td>92981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5073</th>\n",
       "      <td>perl-math-random-mt-auto</td>\n",
       "      <td>3290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5074</th>\n",
       "      <td>perl-math-round</td>\n",
       "      <td>31721</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5075</th>\n",
       "      <td>perl-math-spline</td>\n",
       "      <td>90994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5076</th>\n",
       "      <td>perl-math-utils</td>\n",
       "      <td>1615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5077</th>\n",
       "      <td>perl-math-vecstat</td>\n",
       "      <td>29049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5078</th>\n",
       "      <td>perl-mce</td>\n",
       "      <td>7989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5079</th>\n",
       "      <td>perl-mce-shared</td>\n",
       "      <td>6265</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5080</th>\n",
       "      <td>perl-memoize</td>\n",
       "      <td>5252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5081</th>\n",
       "      <td>perl-metabolomics-fragment-annotation</td>\n",
       "      <td>150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5082</th>\n",
       "      <td>perl-mime-base64</td>\n",
       "      <td>194696</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5083</th>\n",
       "      <td>perl-mime-lite</td>\n",
       "      <td>91243</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5084</th>\n",
       "      <td>perl-mime-quotedprint</td>\n",
       "      <td>1704</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5085</th>\n",
       "      <td>perl-mime-tools</td>\n",
       "      <td>94390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5086</th>\n",
       "      <td>perl-mime-types</td>\n",
       "      <td>93251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5087</th>\n",
       "      <td>perl-mixin-linewise</td>\n",
       "      <td>3121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5088</th>\n",
       "      <td>perl-mldbm</td>\n",
       "      <td>93352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5089</th>\n",
       "      <td>perl-mldbm-sync</td>\n",
       "      <td>2223</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5090</th>\n",
       "      <td>perl-module-build</td>\n",
       "      <td>56819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5091</th>\n",
       "      <td>perl-module-build-tiny</td>\n",
       "      <td>2731</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5092</th>\n",
       "      <td>perl-module-corelist</td>\n",
       "      <td>70935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5093</th>\n",
       "      <td>perl-module-extract-use</td>\n",
       "      <td>1859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5094</th>\n",
       "      <td>perl-module-fromperlver</td>\n",
       "      <td>3567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5095</th>\n",
       "      <td>perl-module-implementation</td>\n",
       "      <td>102398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5096</th>\n",
       "      <td>perl-module-list</td>\n",
       "      <td>2000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5097</th>\n",
       "      <td>perl-module-load</td>\n",
       "      <td>62370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5098</th>\n",
       "      <td>perl-module-load-conditional</td>\n",
       "      <td>63428</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5099</th>\n",
       "      <td>perl-module-loaded</td>\n",
       "      <td>4713</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5100</th>\n",
       "      <td>perl-module-metadata</td>\n",
       "      <td>65483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5101</th>\n",
       "      <td>perl-module-pluggable</td>\n",
       "      <td>15663</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5102</th>\n",
       "      <td>perl-module-runtime</td>\n",
       "      <td>119313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5103</th>\n",
       "      <td>perl-module-runtime-conflicts</td>\n",
       "      <td>85671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5104</th>\n",
       "      <td>perl-module-scandeps</td>\n",
       "      <td>6139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5105</th>\n",
       "      <td>perl-module-util</td>\n",
       "      <td>1684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5106</th>\n",
       "      <td>perl-moo</td>\n",
       "      <td>104481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5107</th>\n",
       "      <td>perl-moose</td>\n",
       "      <td>94442</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5108</th>\n",
       "      <td>perl-moosex-app</td>\n",
       "      <td>6550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5109</th>\n",
       "      <td>perl-moosex-app-role-log4perl</td>\n",
       "      <td>2005</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5110</th>\n",
       "      <td>perl-moosex-clone</td>\n",
       "      <td>1986</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5111</th>\n",
       "      <td>perl-moosex-fileattribute</td>\n",
       "      <td>3859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5112</th>\n",
       "      <td>perl-moosex-getopt</td>\n",
       "      <td>5876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5113</th>\n",
       "      <td>perl-moosex-nonmoose</td>\n",
       "      <td>2071</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5114</th>\n",
       "      <td>perl-moosex-object-pluggable</td>\n",
       "      <td>1325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5115</th>\n",
       "      <td>perl-moosex-role-parameterized</td>\n",
       "      <td>1955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5116</th>\n",
       "      <td>perl-moosex-role-withoverloading</td>\n",
       "      <td>1869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5117</th>\n",
       "      <td>perl-moosex-singleton</td>\n",
       "      <td>1710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5118</th>\n",
       "      <td>perl-moosex-types</td>\n",
       "      <td>5016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5119</th>\n",
       "      <td>perl-moosex-types-path-class</td>\n",
       "      <td>3326</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5120</th>\n",
       "      <td>perl-moosex-types-path-tiny</td>\n",
       "      <td>2144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5121</th>\n",
       "      <td>perl-moosex-types-stringlike</td>\n",
       "      <td>1705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5122</th>\n",
       "      <td>perl-mozilla-ca</td>\n",
       "      <td>170070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5123</th>\n",
       "      <td>perl-mro-compat</td>\n",
       "      <td>85682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5124</th>\n",
       "      <td>perl-ms</td>\n",
       "      <td>127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5125</th>\n",
       "      <td>perl-namespace-autoclean</td>\n",
       "      <td>7894</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5126</th>\n",
       "      <td>perl-namespace-clean</td>\n",
       "      <td>8345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5127</th>\n",
       "      <td>perl-net-ftp</td>\n",
       "      <td>1915</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5128</th>\n",
       "      <td>perl-net-ftp-recursive</td>\n",
       "      <td>1847</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5129</th>\n",
       "      <td>perl-net-http</td>\n",
       "      <td>191945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5130</th>\n",
       "      <td>perl-net-netrc</td>\n",
       "      <td>1750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5131</th>\n",
       "      <td>perl-net-ssleay</td>\n",
       "      <td>203568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5132</th>\n",
       "      <td>perl-ntlm</td>\n",
       "      <td>189800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5133</th>\n",
       "      <td>perl-number-compare</td>\n",
       "      <td>16392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5134</th>\n",
       "      <td>perl-number-format</td>\n",
       "      <td>43136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5135</th>\n",
       "      <td>perl-number-misc</td>\n",
       "      <td>2306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5136</th>\n",
       "      <td>perl-number-range</td>\n",
       "      <td>1784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5137</th>\n",
       "      <td>perl-number-witherror</td>\n",
       "      <td>1227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5138</th>\n",
       "      <td>perl-object-insideout</td>\n",
       "      <td>6087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5139</th>\n",
       "      <td>perl-obogaf-parser</td>\n",
       "      <td>177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5140</th>\n",
       "      <td>perl-ole-storage_lite</td>\n",
       "      <td>92678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5141</th>\n",
       "      <td>perl-onto-perl</td>\n",
       "      <td>2158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5142</th>\n",
       "      <td>perl-package-deprecationmanager</td>\n",
       "      <td>85656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5143</th>\n",
       "      <td>perl-package-stash</td>\n",
       "      <td>92054</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5144</th>\n",
       "      <td>perl-package-stash-xs</td>\n",
       "      <td>87537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5145</th>\n",
       "      <td>perl-padwalker</td>\n",
       "      <td>2043</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5146</th>\n",
       "      <td>perl-par</td>\n",
       "      <td>1772</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5147</th>\n",
       "      <td>perl-parallel-forkmanager</td>\n",
       "      <td>22749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5148</th>\n",
       "      <td>perl-parallel-iterator</td>\n",
       "      <td>1148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5149</th>\n",
       "      <td>perl-parallel-loops</td>\n",
       "      <td>3787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5150</th>\n",
       "      <td>perl-params-check</td>\n",
       "      <td>62114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5151</th>\n",
       "      <td>perl-params-coerce</td>\n",
       "      <td>1980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5152</th>\n",
       "      <td>perl-params-util</td>\n",
       "      <td>94560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5153</th>\n",
       "      <td>perl-params-validate</td>\n",
       "      <td>33495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5154</th>\n",
       "      <td>perl-params-validationcompiler</td>\n",
       "      <td>6550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5155</th>\n",
       "      <td>perl-par-dist</td>\n",
       "      <td>1775</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5156</th>\n",
       "      <td>perl-parent</td>\n",
       "      <td>230774</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5157</th>\n",
       "      <td>perl-par-packer</td>\n",
       "      <td>1816</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5158</th>\n",
       "      <td>perl-parse-recdescent</td>\n",
       "      <td>94668</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5159</th>\n",
       "      <td>perl-parse-yapp</td>\n",
       "      <td>12019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5160</th>\n",
       "      <td>perl-path-class</td>\n",
       "      <td>6644</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5161</th>\n",
       "      <td>perl-path-tiny</td>\n",
       "      <td>10366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5162</th>\n",
       "      <td>perl-pathtools</td>\n",
       "      <td>252174</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5163</th>\n",
       "      <td>perl-pbkdf2-tiny</td>\n",
       "      <td>1684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5164</th>\n",
       "      <td>perl-pcap</td>\n",
       "      <td>8356</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5165</th>\n",
       "      <td>perl-pdf-api2</td>\n",
       "      <td>99799</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5166</th>\n",
       "      <td>perl-pdf-table</td>\n",
       "      <td>743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5167</th>\n",
       "      <td>perl-pegex</td>\n",
       "      <td>5582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5168</th>\n",
       "      <td>perl-perl4-corelibs</td>\n",
       "      <td>1981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5169</th>\n",
       "      <td>perl-perlio</td>\n",
       "      <td>2123</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5170</th>\n",
       "      <td>perl-perlio-encoding</td>\n",
       "      <td>1716</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5171</th>\n",
       "      <td>perl-perlio-gzip</td>\n",
       "      <td>72763</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5172</th>\n",
       "      <td>perl-perlio-utf8_strict</td>\n",
       "      <td>12756</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5173</th>\n",
       "      <td>perl-perl-osnames</td>\n",
       "      <td>2170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5174</th>\n",
       "      <td>perl-perl-ostype</td>\n",
       "      <td>39683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5175</th>\n",
       "      <td>perl-perl-unsafe-signals</td>\n",
       "      <td>5628</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5176</th>\n",
       "      <td>perl-perl-version</td>\n",
       "      <td>4305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5177</th>\n",
       "      <td>perl-pod-checker</td>\n",
       "      <td>1765</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5178</th>\n",
       "      <td>perl-pod-coverage</td>\n",
       "      <td>2052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5179</th>\n",
       "      <td>perl-pod-coverage-trustpod</td>\n",
       "      <td>1715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5180</th>\n",
       "      <td>perl-pod-elemental</td>\n",
       "      <td>3207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5181</th>\n",
       "      <td>perl-pod-escapes</td>\n",
       "      <td>83011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5182</th>\n",
       "      <td>perl-pod-parser</td>\n",
       "      <td>1941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5183</th>\n",
       "      <td>perl-pod-plaintext</td>\n",
       "      <td>1745</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5184</th>\n",
       "      <td>perl-pod-simple</td>\n",
       "      <td>1838</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5185</th>\n",
       "      <td>perl-pod-simple-text</td>\n",
       "      <td>1794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5186</th>\n",
       "      <td>perl-pod-usage</td>\n",
       "      <td>82784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5187</th>\n",
       "      <td>perl-posix</td>\n",
       "      <td>6118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5188</th>\n",
       "      <td>perl-postscript</td>\n",
       "      <td>91100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5189</th>\n",
       "      <td>perl-ppi</td>\n",
       "      <td>2037</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5190</th>\n",
       "      <td>perl-prefork</td>\n",
       "      <td>1224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5191</th>\n",
       "      <td>perl-probe-perl</td>\n",
       "      <td>1465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5192</th>\n",
       "      <td>perl-proc-fork</td>\n",
       "      <td>3212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5193</th>\n",
       "      <td>perl-readonly</td>\n",
       "      <td>31496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5194</th>\n",
       "      <td>perl-regexp-common</td>\n",
       "      <td>32725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5195</th>\n",
       "      <td>perl-rest-client</td>\n",
       "      <td>175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5196</th>\n",
       "      <td>perl-return-multilevel</td>\n",
       "      <td>1559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5197</th>\n",
       "      <td>perl-role-tiny</td>\n",
       "      <td>104105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5198</th>\n",
       "      <td>perl-role-tiny-with</td>\n",
       "      <td>1748</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5199</th>\n",
       "      <td>perl-safe</td>\n",
       "      <td>1918</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5200</th>\n",
       "      <td>perl-sanger-cgp-allelecount</td>\n",
       "      <td>9599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5201</th>\n",
       "      <td>perl-sanger-cgp-battenberg</td>\n",
       "      <td>7368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5202</th>\n",
       "      <td>perl-sanger-cgp-vagrent</td>\n",
       "      <td>5470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5203</th>\n",
       "      <td>perl-sanger-cgp-vcf</td>\n",
       "      <td>9384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5204</th>\n",
       "      <td>perl-scalar-list-utils</td>\n",
       "      <td>300579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5205</th>\n",
       "      <td>perl-scalar-util-numeric</td>\n",
       "      <td>7194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5206</th>\n",
       "      <td>perl-scope-guard</td>\n",
       "      <td>15822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5207</th>\n",
       "      <td>perl-sereal</td>\n",
       "      <td>38743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5208</th>\n",
       "      <td>perl-sereal-decoder</td>\n",
       "      <td>39884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5209</th>\n",
       "      <td>perl-sereal-encoder</td>\n",
       "      <td>39795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5210</th>\n",
       "      <td>perl-set-intervaltree</td>\n",
       "      <td>54140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5211</th>\n",
       "      <td>perl-set-intspan</td>\n",
       "      <td>28823</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5212</th>\n",
       "      <td>perl-set-scalar</td>\n",
       "      <td>91067</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5213</th>\n",
       "      <td>perl-slurp</td>\n",
       "      <td>1612</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5214</th>\n",
       "      <td>perl-snap</td>\n",
       "      <td>1544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5215</th>\n",
       "      <td>perl-soap-lite</td>\n",
       "      <td>91141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5216</th>\n",
       "      <td>perl-socket</td>\n",
       "      <td>164299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5217</th>\n",
       "      <td>perl-socket6</td>\n",
       "      <td>3567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5218</th>\n",
       "      <td>perl-sort-naturally</td>\n",
       "      <td>92589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5219</th>\n",
       "      <td>perl-sort-versions</td>\n",
       "      <td>2450</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5220</th>\n",
       "      <td>perl-specio</td>\n",
       "      <td>3151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5221</th>\n",
       "      <td>perl-specio-exporter</td>\n",
       "      <td>6801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5222</th>\n",
       "      <td>perl-spiffy</td>\n",
       "      <td>2611</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5223</th>\n",
       "      <td>perl-spreadsheet-parseexcel</td>\n",
       "      <td>91046</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5224</th>\n",
       "      <td>perl-spreadsheet-writeexcel</td>\n",
       "      <td>92232</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5225</th>\n",
       "      <td>perl-sql-statement</td>\n",
       "      <td>2457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5226</th>\n",
       "      <td>perl-statistics-basic</td>\n",
       "      <td>32796</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5227</th>\n",
       "      <td>perl-statistics-caseresampling</td>\n",
       "      <td>1222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5228</th>\n",
       "      <td>perl-statistics-descriptive</td>\n",
       "      <td>101117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5229</th>\n",
       "      <td>perl-statistics-distributions</td>\n",
       "      <td>5892</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5230</th>\n",
       "      <td>perl-statistics-frequency</td>\n",
       "      <td>1749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5231</th>\n",
       "      <td>perl-statistics-lite</td>\n",
       "      <td>1912</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5232</th>\n",
       "      <td>perl-statistics-r</td>\n",
       "      <td>4291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5233</th>\n",
       "      <td>perl-statistics-ttest</td>\n",
       "      <td>4469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5234</th>\n",
       "      <td>perl-storable</td>\n",
       "      <td>173738</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5235</th>\n",
       "      <td>perl-string-approx</td>\n",
       "      <td>2530</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5236</th>\n",
       "      <td>perl-string-diff</td>\n",
       "      <td>558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5237</th>\n",
       "      <td>perl-string-escape</td>\n",
       "      <td>2869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5238</th>\n",
       "      <td>perl-string-random</td>\n",
       "      <td>1797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5239</th>\n",
       "      <td>perl-string-rewriteprefix</td>\n",
       "      <td>2136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5240</th>\n",
       "      <td>perl-string-truncate</td>\n",
       "      <td>2147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5241</th>\n",
       "      <td>perl-string-util</td>\n",
       "      <td>2311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5242</th>\n",
       "      <td>perl-sub-attribute</td>\n",
       "      <td>2089</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5243</th>\n",
       "      <td>perl-sub-exporter</td>\n",
       "      <td>90883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5244</th>\n",
       "      <td>perl-sub-exporter-formethods</td>\n",
       "      <td>2489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5245</th>\n",
       "      <td>perl-sub-exporter-progressive</td>\n",
       "      <td>104944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5246</th>\n",
       "      <td>perl-sub-identify</td>\n",
       "      <td>91132</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5247</th>\n",
       "      <td>perl-sub-info</td>\n",
       "      <td>15343</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5248</th>\n",
       "      <td>perl-sub-install</td>\n",
       "      <td>92496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5249</th>\n",
       "      <td>perl-sub-name</td>\n",
       "      <td>86227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5250</th>\n",
       "      <td>perl-sub-quote</td>\n",
       "      <td>85871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5251</th>\n",
       "      <td>perl-sub-uplevel</td>\n",
       "      <td>86539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5252</th>\n",
       "      <td>perl-super</td>\n",
       "      <td>1819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5253</th>\n",
       "      <td>perl-svg</td>\n",
       "      <td>107203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5254</th>\n",
       "      <td>perl-svg-graph</td>\n",
       "      <td>92603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5255</th>\n",
       "      <td>perl-symbol</td>\n",
       "      <td>1785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5256</th>\n",
       "      <td>perl-symbol-util</td>\n",
       "      <td>1755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5257</th>\n",
       "      <td>perl-sys-info</td>\n",
       "      <td>3230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5258</th>\n",
       "      <td>perl-sys-info-base</td>\n",
       "      <td>3274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5259</th>\n",
       "      <td>perl-sys-info-driver-linux</td>\n",
       "      <td>4505</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5260</th>\n",
       "      <td>perl-sys-info-driver-osx</td>\n",
       "      <td>2653</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5261</th>\n",
       "      <td>perl-sys-sigaction</td>\n",
       "      <td>4904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5262</th>\n",
       "      <td>perl-tap-harness-env</td>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5263</th>\n",
       "      <td>perl-task-weaken</td>\n",
       "      <td>93722</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5264</th>\n",
       "      <td>perl-template-toolkit</td>\n",
       "      <td>91274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5265</th>\n",
       "      <td>perl-term-app-roles</td>\n",
       "      <td>3635</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5266</th>\n",
       "      <td>perl-term-detect-software</td>\n",
       "      <td>2215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5267</th>\n",
       "      <td>perl-term-encoding</td>\n",
       "      <td>3000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5268</th>\n",
       "      <td>perl-term-progressbar</td>\n",
       "      <td>3517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5269</th>\n",
       "      <td>perl-termreadkey</td>\n",
       "      <td>10924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5270</th>\n",
       "      <td>perl-term-table</td>\n",
       "      <td>17085</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5271</th>\n",
       "      <td>perl-test2</td>\n",
       "      <td>8812</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5272</th>\n",
       "      <td>perl-test2-suite</td>\n",
       "      <td>22953</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5273</th>\n",
       "      <td>perl-test</td>\n",
       "      <td>83033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5274</th>\n",
       "      <td>perl-test-base</td>\n",
       "      <td>3674</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5275</th>\n",
       "      <td>perl-test-builder-tester</td>\n",
       "      <td>5391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5276</th>\n",
       "      <td>perl-test-classapi</td>\n",
       "      <td>3564</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5277</th>\n",
       "      <td>perl-test-class-moose</td>\n",
       "      <td>6605</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5278</th>\n",
       "      <td>perl-test-cleannamespaces</td>\n",
       "      <td>3022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5279</th>\n",
       "      <td>perl-test-cpan-meta</td>\n",
       "      <td>2189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5280</th>\n",
       "      <td>perl-test-deep</td>\n",
       "      <td>70414</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5281</th>\n",
       "      <td>perl-test-differences</td>\n",
       "      <td>83164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5282</th>\n",
       "      <td>perl-test-eol</td>\n",
       "      <td>3661</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5283</th>\n",
       "      <td>perl-test-exception</td>\n",
       "      <td>82724</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5284</th>\n",
       "      <td>perl-test-exec</td>\n",
       "      <td>1681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5285</th>\n",
       "      <td>perl-test-fatal</td>\n",
       "      <td>7465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5286</th>\n",
       "      <td>perl-test-file</td>\n",
       "      <td>1164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5287</th>\n",
       "      <td>perl-test-file-contents</td>\n",
       "      <td>1962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5288</th>\n",
       "      <td>perl-test-files</td>\n",
       "      <td>5184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5289</th>\n",
       "      <td>perl-test-fork</td>\n",
       "      <td>1615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5290</th>\n",
       "      <td>perl-test-harness</td>\n",
       "      <td>88682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5291</th>\n",
       "      <td>perl-test-inter</td>\n",
       "      <td>4563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5292</th>\n",
       "      <td>perl-test-leaktrace</td>\n",
       "      <td>95230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5293</th>\n",
       "      <td>perl-test-lectrotest</td>\n",
       "      <td>1193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5294</th>\n",
       "      <td>perl-test-longstring</td>\n",
       "      <td>1931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5295</th>\n",
       "      <td>perl-test-memory-cycle</td>\n",
       "      <td>1767</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5296</th>\n",
       "      <td>perl-test-mockmodule</td>\n",
       "      <td>1942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5297</th>\n",
       "      <td>perl-test-most</td>\n",
       "      <td>83449</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5298</th>\n",
       "      <td>perl-test-needs</td>\n",
       "      <td>2931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5299</th>\n",
       "      <td>perl-test-notabs</td>\n",
       "      <td>3789</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5300</th>\n",
       "      <td>perl-test-nowarnings</td>\n",
       "      <td>3651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5301</th>\n",
       "      <td>perl-test-object</td>\n",
       "      <td>1990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5302</th>\n",
       "      <td>perl-test-output</td>\n",
       "      <td>6474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5303</th>\n",
       "      <td>perl-test-pod</td>\n",
       "      <td>18121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5304</th>\n",
       "      <td>perl-test-pod-coverage</td>\n",
       "      <td>1980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5305</th>\n",
       "      <td>perl-test-prereq</td>\n",
       "      <td>1831</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5306</th>\n",
       "      <td>perl-test-requires</td>\n",
       "      <td>2870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5307</th>\n",
       "      <td>perl-test-requiresinternet</td>\n",
       "      <td>163206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5308</th>\n",
       "      <td>perl-test-script</td>\n",
       "      <td>1460</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5309</th>\n",
       "      <td>perl-test-simple</td>\n",
       "      <td>20050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5310</th>\n",
       "      <td>perl-test-subcalls</td>\n",
       "      <td>1906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5311</th>\n",
       "      <td>perl-test-sys-info</td>\n",
       "      <td>3065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5312</th>\n",
       "      <td>perl-test-toolbox</td>\n",
       "      <td>2273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5313</th>\n",
       "      <td>perl-test-trap</td>\n",
       "      <td>3699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5314</th>\n",
       "      <td>perl-test-unit-lite</td>\n",
       "      <td>1734</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5315</th>\n",
       "      <td>perl-test-utf8</td>\n",
       "      <td>1171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5316</th>\n",
       "      <td>perl-test-warn</td>\n",
       "      <td>85125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5317</th>\n",
       "      <td>perl-test-warnings</td>\n",
       "      <td>7174</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5318</th>\n",
       "      <td>perl-test-without-module</td>\n",
       "      <td>1552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5319</th>\n",
       "      <td>perl-test-xml</td>\n",
       "      <td>2195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5320</th>\n",
       "      <td>perl-test-yaml</td>\n",
       "      <td>3730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5321</th>\n",
       "      <td>perl-text-abbrev</td>\n",
       "      <td>38557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5322</th>\n",
       "      <td>perl-text-ansitable</td>\n",
       "      <td>1976</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5323</th>\n",
       "      <td>perl-text-asciitable</td>\n",
       "      <td>3962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5324</th>\n",
       "      <td>perl-text-balanced</td>\n",
       "      <td>4597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5325</th>\n",
       "      <td>perl-text-csv</td>\n",
       "      <td>47770</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5326</th>\n",
       "      <td>perl-text-csv_xs</td>\n",
       "      <td>656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5327</th>\n",
       "      <td>perl-text-diff</td>\n",
       "      <td>84023</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5328</th>\n",
       "      <td>perl-text-format</td>\n",
       "      <td>29286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5329</th>\n",
       "      <td>perl-text-glob</td>\n",
       "      <td>19464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5330</th>\n",
       "      <td>perl-text-levenshtein</td>\n",
       "      <td>2159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5331</th>\n",
       "      <td>perl-text-levenshteinxs</td>\n",
       "      <td>1182</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5332</th>\n",
       "      <td>perl-text-nsp</td>\n",
       "      <td>2111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5333</th>\n",
       "      <td>perl-text-parsewords</td>\n",
       "      <td>41822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5334</th>\n",
       "      <td>perl-text-tabs</td>\n",
       "      <td>1755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5335</th>\n",
       "      <td>perl-text-tabs-wrap</td>\n",
       "      <td>3169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5336</th>\n",
       "      <td>perl-text-template</td>\n",
       "      <td>1956</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5337</th>\n",
       "      <td>perl-text-template-simple</td>\n",
       "      <td>1589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5338</th>\n",
       "      <td>perl-text-wrap</td>\n",
       "      <td>4108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5339</th>\n",
       "      <td>perl-threaded</td>\n",
       "      <td>222698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5340</th>\n",
       "      <td>perl-tie-cache</td>\n",
       "      <td>2305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5341</th>\n",
       "      <td>perl-tie-cacher</td>\n",
       "      <td>1878</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5342</th>\n",
       "      <td>perl-tie-hash</td>\n",
       "      <td>1788</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5343</th>\n",
       "      <td>perl-tie-hash-indexed</td>\n",
       "      <td>1853</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5344</th>\n",
       "      <td>perl-tie-ixhash</td>\n",
       "      <td>92287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5345</th>\n",
       "      <td>perl-tie-log4perl</td>\n",
       "      <td>2613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5346</th>\n",
       "      <td>perl-tie-refhash</td>\n",
       "      <td>2214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5347</th>\n",
       "      <td>perl-tie-refhash-weak</td>\n",
       "      <td>2049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5348</th>\n",
       "      <td>perl-tie-toobject</td>\n",
       "      <td>2005</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5349</th>\n",
       "      <td>perl-timedate</td>\n",
       "      <td>91562</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5350</th>\n",
       "      <td>perl-time-hires</td>\n",
       "      <td>109294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5351</th>\n",
       "      <td>perl-time-local</td>\n",
       "      <td>166078</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5352</th>\n",
       "      <td>perl-time-piece</td>\n",
       "      <td>8634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5353</th>\n",
       "      <td>perl-tree-dag_node</td>\n",
       "      <td>93585</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5354</th>\n",
       "      <td>perl-try-tiny</td>\n",
       "      <td>185653</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5355</th>\n",
       "      <td>perl-try-tiny-retry</td>\n",
       "      <td>3386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5356</th>\n",
       "      <td>perl-types-serialiser</td>\n",
       "      <td>176717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5357</th>\n",
       "      <td>perl-type-tiny</td>\n",
       "      <td>97815</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5358</th>\n",
       "      <td>perl-unicode-map</td>\n",
       "      <td>92909</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5359</th>\n",
       "      <td>perl-unicode-normalize</td>\n",
       "      <td>4715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5360</th>\n",
       "      <td>perl-unicode-stringprep</td>\n",
       "      <td>1867</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5361</th>\n",
       "      <td>perl-unicode-utf8</td>\n",
       "      <td>1718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5362</th>\n",
       "      <td>perl-uri</td>\n",
       "      <td>223700</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5363</th>\n",
       "      <td>perl-util-properties</td>\n",
       "      <td>171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5364</th>\n",
       "      <td>perl-variable-magic</td>\n",
       "      <td>8456</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5365</th>\n",
       "      <td>perl-vars</td>\n",
       "      <td>1785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5366</th>\n",
       "      <td>perl-vcftools-vcf</td>\n",
       "      <td>50943</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5367</th>\n",
       "      <td>perl-velvetoptimiser</td>\n",
       "      <td>5062</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5368</th>\n",
       "      <td>perl-version</td>\n",
       "      <td>62151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5369</th>\n",
       "      <td>perl-version-next</td>\n",
       "      <td>4214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5370</th>\n",
       "      <td>perl-want</td>\n",
       "      <td>5390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5371</th>\n",
       "      <td>perl-warnings-register</td>\n",
       "      <td>6539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5372</th>\n",
       "      <td>perl-www-mechanize</td>\n",
       "      <td>1591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5373</th>\n",
       "      <td>perl-www-robotrules</td>\n",
       "      <td>188830</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5374</th>\n",
       "      <td>perl-xml-dom</td>\n",
       "      <td>93322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5375</th>\n",
       "      <td>perl-xml-dom-xpath</td>\n",
       "      <td>91013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5376</th>\n",
       "      <td>perl-xml-entities</td>\n",
       "      <td>1701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5377</th>\n",
       "      <td>perl-xml-filter-buffertext</td>\n",
       "      <td>91224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5378</th>\n",
       "      <td>perl-xml-libxml</td>\n",
       "      <td>121576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5379</th>\n",
       "      <td>perl-xml-libxslt</td>\n",
       "      <td>93334</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5380</th>\n",
       "      <td>perl-xml-namespacesupport</td>\n",
       "      <td>205411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5381</th>\n",
       "      <td>perl-xml-parser</td>\n",
       "      <td>122792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5382</th>\n",
       "      <td>perl-xml-parser-lite</td>\n",
       "      <td>3722</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5383</th>\n",
       "      <td>perl-xml-regexp</td>\n",
       "      <td>91679</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5384</th>\n",
       "      <td>perl-xml-sax</td>\n",
       "      <td>209529</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5385</th>\n",
       "      <td>perl-xml-sax-base</td>\n",
       "      <td>205378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5386</th>\n",
       "      <td>perl-xml-sax-expat</td>\n",
       "      <td>192576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5387</th>\n",
       "      <td>perl-xml-sax-writer</td>\n",
       "      <td>93440</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5388</th>\n",
       "      <td>perl-xml-semanticdiff</td>\n",
       "      <td>3836</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5389</th>\n",
       "      <td>perl-xml-simple</td>\n",
       "      <td>195056</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5390</th>\n",
       "      <td>perl-xml-twig</td>\n",
       "      <td>95408</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5391</th>\n",
       "      <td>perl-xml-writer</td>\n",
       "      <td>91377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5392</th>\n",
       "      <td>perl-xml-xpath</td>\n",
       "      <td>93518</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5393</th>\n",
       "      <td>perl-xml-xpathengine</td>\n",
       "      <td>92050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5394</th>\n",
       "      <td>perl-xsloader</td>\n",
       "      <td>277632</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5395</th>\n",
       "      <td>perl-yaml</td>\n",
       "      <td>151011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5396</th>\n",
       "      <td>perl-yaml-libyaml</td>\n",
       "      <td>7989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5397</th>\n",
       "      <td>perl-yaml-pp</td>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5398</th>\n",
       "      <td>perl-yaml-tiny</td>\n",
       "      <td>4175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5399</th>\n",
       "      <td>pfam_scan</td>\n",
       "      <td>3965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5400</th>\n",
       "      <td>pftools</td>\n",
       "      <td>488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5401</th>\n",
       "      <td>pgcgap</td>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5402</th>\n",
       "      <td>pgdspider</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5403</th>\n",
       "      <td>pgma-simple</td>\n",
       "      <td>3269</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5404</th>\n",
       "      <td>pgsa</td>\n",
       "      <td>1916</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5405</th>\n",
       "      <td>pgx-pipe-helper</td>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5406</th>\n",
       "      <td>pgx-variant-tools</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5407</th>\n",
       "      <td>phame</td>\n",
       "      <td>8216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5408</th>\n",
       "      <td>phantompeakqualtools</td>\n",
       "      <td>4860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5409</th>\n",
       "      <td>phast</td>\n",
       "      <td>1790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5410</th>\n",
       "      <td>phastaf</td>\n",
       "      <td>55</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5411</th>\n",
       "      <td>phate</td>\n",
       "      <td>376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5412</th>\n",
       "      <td>pheniqs</td>\n",
       "      <td>4024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5413</th>\n",
       "      <td>phenix</td>\n",
       "      <td>1642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5414</th>\n",
       "      <td>phigaro</td>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5415</th>\n",
       "      <td>phipack</td>\n",
       "      <td>3018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5416</th>\n",
       "      <td>phispy</td>\n",
       "      <td>3247</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5417</th>\n",
       "      <td>phizz</td>\n",
       "      <td>2103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5418</th>\n",
       "      <td>phylics</td>\n",
       "      <td>2111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5419</th>\n",
       "      <td>phylip</td>\n",
       "      <td>6999</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5420</th>\n",
       "      <td>phylocsf</td>\n",
       "      <td>1053</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5421</th>\n",
       "      <td>phylodm</td>\n",
       "      <td>872</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5422</th>\n",
       "      <td>phyloflash</td>\n",
       "      <td>5737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5423</th>\n",
       "      <td>phylogenetics</td>\n",
       "      <td>81</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5424</th>\n",
       "      <td>phylommand</td>\n",
       "      <td>1166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5425</th>\n",
       "      <td>phylopandas</td>\n",
       "      <td>4469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5426</th>\n",
       "      <td>phylophlan</td>\n",
       "      <td>1989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5427</th>\n",
       "      <td>phylorank</td>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5428</th>\n",
       "      <td>phylotoast</td>\n",
       "      <td>4007</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5429</th>\n",
       "      <td>phylovega</td>\n",
       "      <td>236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5430</th>\n",
       "      <td>phylowgs</td>\n",
       "      <td>10000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5431</th>\n",
       "      <td>phyluce</td>\n",
       "      <td>11135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5432</th>\n",
       "      <td>phyml</td>\n",
       "      <td>14348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5433</th>\n",
       "      <td>phyphy</td>\n",
       "      <td>51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5434</th>\n",
       "      <td>picard</td>\n",
       "      <td>264985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5435</th>\n",
       "      <td>picard-slim</td>\n",
       "      <td>2229</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5436</th>\n",
       "      <td>picopore</td>\n",
       "      <td>12268</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5437</th>\n",
       "      <td>picrust</td>\n",
       "      <td>14723</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5438</th>\n",
       "      <td>picrust2</td>\n",
       "      <td>7276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5439</th>\n",
       "      <td>piler</td>\n",
       "      <td>2154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5440</th>\n",
       "      <td>pileuppy</td>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5441</th>\n",
       "      <td>pilon</td>\n",
       "      <td>29346</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5442</th>\n",
       "      <td>pindel</td>\n",
       "      <td>48830</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5443</th>\n",
       "      <td>pipelign</td>\n",
       "      <td>1826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5444</th>\n",
       "      <td>piper</td>\n",
       "      <td>390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5445</th>\n",
       "      <td>pipits</td>\n",
       "      <td>9567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5446</th>\n",
       "      <td>pipmir</td>\n",
       "      <td>2551</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5447</th>\n",
       "      <td>piranha</td>\n",
       "      <td>3413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5448</th>\n",
       "      <td>pirate</td>\n",
       "      <td>489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5449</th>\n",
       "      <td>piret</td>\n",
       "      <td>1473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5450</th>\n",
       "      <td>pirs</td>\n",
       "      <td>1960</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5451</th>\n",
       "      <td>pisces</td>\n",
       "      <td>4158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5452</th>\n",
       "      <td>pispino</td>\n",
       "      <td>4733</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5453</th>\n",
       "      <td>pizzly</td>\n",
       "      <td>9161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5454</th>\n",
       "      <td>pkiss</td>\n",
       "      <td>1921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5455</th>\n",
       "      <td>planemo</td>\n",
       "      <td>44322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5456</th>\n",
       "      <td>plant_tribes_assembly_post_processor</td>\n",
       "      <td>8095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5457</th>\n",
       "      <td>plant_tribes_gene_family_aligner</td>\n",
       "      <td>5062</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5458</th>\n",
       "      <td>plant_tribes_gene_family_classifier</td>\n",
       "      <td>10177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5459</th>\n",
       "      <td>plant_tribes_gene_family_integrator</td>\n",
       "      <td>5263</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5460</th>\n",
       "      <td>plant_tribes_gene_family_phylogeny_builder</td>\n",
       "      <td>5403</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5461</th>\n",
       "      <td>plant_tribes_kaks_analysis</td>\n",
       "      <td>5122</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5462</th>\n",
       "      <td>plasclass</td>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5463</th>\n",
       "      <td>plascope</td>\n",
       "      <td>3256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5464</th>\n",
       "      <td>plasflow</td>\n",
       "      <td>2388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5465</th>\n",
       "      <td>plasmidfinder</td>\n",
       "      <td>369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5466</th>\n",
       "      <td>plasmidid</td>\n",
       "      <td>1179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5467</th>\n",
       "      <td>plasmidtron</td>\n",
       "      <td>5270</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5468</th>\n",
       "      <td>plass</td>\n",
       "      <td>2995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5469</th>\n",
       "      <td>plastid</td>\n",
       "      <td>6996</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5470</th>\n",
       "      <td>platon</td>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5471</th>\n",
       "      <td>platypus-conquistador</td>\n",
       "      <td>1790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5472</th>\n",
       "      <td>platypus-variant</td>\n",
       "      <td>12299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5473</th>\n",
       "      <td>plek</td>\n",
       "      <td>2407</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5474</th>\n",
       "      <td>plink</td>\n",
       "      <td>14533</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5475</th>\n",
       "      <td>plink2</td>\n",
       "      <td>6890</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5476</th>\n",
       "      <td>plncpro</td>\n",
       "      <td>504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5477</th>\n",
       "      <td>pmdtools</td>\n",
       "      <td>3107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5478</th>\n",
       "      <td>pmmrcalculator</td>\n",
       "      <td>118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5479</th>\n",
       "      <td>pmx_biobb</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5480</th>\n",
       "      <td>pneumocat</td>\n",
       "      <td>3673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5481</th>\n",
       "      <td>poa</td>\n",
       "      <td>2333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5482</th>\n",
       "      <td>polymutt</td>\n",
       "      <td>1549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5483</th>\n",
       "      <td>polystest</td>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5484</th>\n",
       "      <td>pomoxis</td>\n",
       "      <td>1317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5485</th>\n",
       "      <td>popdel</td>\n",
       "      <td>17489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5486</th>\n",
       "      <td>popera</td>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5487</th>\n",
       "      <td>popgen-entropy</td>\n",
       "      <td>552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5488</th>\n",
       "      <td>poppunk</td>\n",
       "      <td>4383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5489</th>\n",
       "      <td>popscle</td>\n",
       "      <td>936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5490</th>\n",
       "      <td>porechop</td>\n",
       "      <td>8438</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5491</th>\n",
       "      <td>poretools</td>\n",
       "      <td>10546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5492</th>\n",
       "      <td>portcullis</td>\n",
       "      <td>9791</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5493</th>\n",
       "      <td>potrace</td>\n",
       "      <td>2487</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5494</th>\n",
       "      <td>pourrna</td>\n",
       "      <td>3902</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5495</th>\n",
       "      <td>pout2mzid</td>\n",
       "      <td>1978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5496</th>\n",
       "      <td>pp</td>\n",
       "      <td>4246</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5497</th>\n",
       "      <td>ppanggolin</td>\n",
       "      <td>4658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5498</th>\n",
       "      <td>ppanini</td>\n",
       "      <td>3381</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5499</th>\n",
       "      <td>pplacer</td>\n",
       "      <td>14967</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5500</th>\n",
       "      <td>pp-sketchlib</td>\n",
       "      <td>2206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5501</th>\n",
       "      <td>prank</td>\n",
       "      <td>14366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5502</th>\n",
       "      <td>prefersim</td>\n",
       "      <td>964</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5503</th>\n",
       "      <td>preseq</td>\n",
       "      <td>26951</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5504</th>\n",
       "      <td>presto</td>\n",
       "      <td>4860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5505</th>\n",
       "      <td>pretextgraph</td>\n",
       "      <td>787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5506</th>\n",
       "      <td>pretextmap</td>\n",
       "      <td>1580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5507</th>\n",
       "      <td>pretextsnapshot</td>\n",
       "      <td>972</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5508</th>\n",
       "      <td>pretext-suite</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5509</th>\n",
       "      <td>primedrpa</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5510</th>\n",
       "      <td>primer3</td>\n",
       "      <td>13576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5511</th>\n",
       "      <td>primer3-py</td>\n",
       "      <td>10369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5512</th>\n",
       "      <td>primerclip</td>\n",
       "      <td>876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5513</th>\n",
       "      <td>primerprospector</td>\n",
       "      <td>2001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5514</th>\n",
       "      <td>prince</td>\n",
       "      <td>894</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5515</th>\n",
       "      <td>prinseq</td>\n",
       "      <td>18170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5516</th>\n",
       "      <td>prinseq-plus-plus</td>\n",
       "      <td>170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5517</th>\n",
       "      <td>probabilistic2020</td>\n",
       "      <td>710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5518</th>\n",
       "      <td>probamconvert</td>\n",
       "      <td>4944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5519</th>\n",
       "      <td>probcons</td>\n",
       "      <td>1322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5520</th>\n",
       "      <td>proda</td>\n",
       "      <td>1219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5521</th>\n",
       "      <td>prodigal</td>\n",
       "      <td>70937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5522</th>\n",
       "      <td>progressivemauve</td>\n",
       "      <td>3177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5523</th>\n",
       "      <td>prokaryote</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5524</th>\n",
       "      <td>prokka</td>\n",
       "      <td>47219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5525</th>\n",
       "      <td>pronto</td>\n",
       "      <td>6313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5526</th>\n",
       "      <td>proot</td>\n",
       "      <td>1802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5527</th>\n",
       "      <td>prophane</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5528</th>\n",
       "      <td>prophasm</td>\n",
       "      <td>1782</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5529</th>\n",
       "      <td>prophex</td>\n",
       "      <td>3448</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5530</th>\n",
       "      <td>prophyle</td>\n",
       "      <td>24637</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5531</th>\n",
       "      <td>prosampler</td>\n",
       "      <td>2740</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5532</th>\n",
       "      <td>prosic</td>\n",
       "      <td>6146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5533</th>\n",
       "      <td>prosolo</td>\n",
       "      <td>9580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5534</th>\n",
       "      <td>proteinortho</td>\n",
       "      <td>19577</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5535</th>\n",
       "      <td>proteowizard</td>\n",
       "      <td>2984</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5536</th>\n",
       "      <td>protk</td>\n",
       "      <td>1683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5537</th>\n",
       "      <td>protmapper</td>\n",
       "      <td>147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5538</th>\n",
       "      <td>protrac</td>\n",
       "      <td>4642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5539</th>\n",
       "      <td>psascan</td>\n",
       "      <td>1242</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5540</th>\n",
       "      <td>psass</td>\n",
       "      <td>486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5541</th>\n",
       "      <td>psm_fragments</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5542</th>\n",
       "      <td>ptrimmer</td>\n",
       "      <td>1168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5543</th>\n",
       "      <td>pubchempy</td>\n",
       "      <td>187185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5544</th>\n",
       "      <td>pubmlst_client</td>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5545</th>\n",
       "      <td>pullseq</td>\n",
       "      <td>3786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5546</th>\n",
       "      <td>pureclip</td>\n",
       "      <td>12591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5547</th>\n",
       "      <td>purge_haplotigs</td>\n",
       "      <td>5784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5548</th>\n",
       "      <td>purple-bio</td>\n",
       "      <td>143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5549</th>\n",
       "      <td>pvacseq</td>\n",
       "      <td>3723</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5550</th>\n",
       "      <td>py2bit</td>\n",
       "      <td>67566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5551</th>\n",
       "      <td>pyaavf</td>\n",
       "      <td>348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5552</th>\n",
       "      <td>pyani</td>\n",
       "      <td>13422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5553</th>\n",
       "      <td>pyasp</td>\n",
       "      <td>2319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5554</th>\n",
       "      <td>pyasr</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5555</th>\n",
       "      <td>pybamparser</td>\n",
       "      <td>2007</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5556</th>\n",
       "      <td>pybamtools</td>\n",
       "      <td>3755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5557</th>\n",
       "      <td>pybda</td>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5558</th>\n",
       "      <td>pybedgraph</td>\n",
       "      <td>5090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5559</th>\n",
       "      <td>pybedtools</td>\n",
       "      <td>225121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5560</th>\n",
       "      <td>pybel</td>\n",
       "      <td>6131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5561</th>\n",
       "      <td>pybigwig</td>\n",
       "      <td>198714</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5562</th>\n",
       "      <td>pybiomart</td>\n",
       "      <td>181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5563</th>\n",
       "      <td>pychopper</td>\n",
       "      <td>1488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5564</th>\n",
       "      <td>pycli</td>\n",
       "      <td>5063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5565</th>\n",
       "      <td>pyclone</td>\n",
       "      <td>119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5566</th>\n",
       "      <td>pycluster</td>\n",
       "      <td>5109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5567</th>\n",
       "      <td>pycoqc</td>\n",
       "      <td>3679</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5568</th>\n",
       "      <td>pycsg</td>\n",
       "      <td>2372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5569</th>\n",
       "      <td>pydemult</td>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5570</th>\n",
       "      <td>pydna</td>\n",
       "      <td>3870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5571</th>\n",
       "      <td>pydnase</td>\n",
       "      <td>4104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5572</th>\n",
       "      <td>pydownsampler</td>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5573</th>\n",
       "      <td>pydp</td>\n",
       "      <td>142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5574</th>\n",
       "      <td>pydpi</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5575</th>\n",
       "      <td>pyega3</td>\n",
       "      <td>3150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5576</th>\n",
       "      <td>pyensembl</td>\n",
       "      <td>4189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5577</th>\n",
       "      <td>pyexcelerator</td>\n",
       "      <td>4233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5578</th>\n",
       "      <td>pyfaidx</td>\n",
       "      <td>146823</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5579</th>\n",
       "      <td>pyfasta</td>\n",
       "      <td>9633</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5580</th>\n",
       "      <td>pyfastaq</td>\n",
       "      <td>12246</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5581</th>\n",
       "      <td>pyfastx</td>\n",
       "      <td>5697</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5582</th>\n",
       "      <td>pygenometracks</td>\n",
       "      <td>25683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5583</th>\n",
       "      <td>pygmes</td>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5584</th>\n",
       "      <td>pygresql</td>\n",
       "      <td>3331</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5585</th>\n",
       "      <td>pygtftk</td>\n",
       "      <td>9354</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5586</th>\n",
       "      <td>pygvcf2coverage</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5587</th>\n",
       "      <td>pyhashxx</td>\n",
       "      <td>13317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5588</th>\n",
       "      <td>pyimagingmspec</td>\n",
       "      <td>1882</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5589</th>\n",
       "      <td>pyimzml</td>\n",
       "      <td>2355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5590</th>\n",
       "      <td>pylca</td>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5591</th>\n",
       "      <td>pylibseq</td>\n",
       "      <td>8928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5592</th>\n",
       "      <td>pyliftover</td>\n",
       "      <td>11520</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5593</th>\n",
       "      <td>pyloh</td>\n",
       "      <td>9095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5594</th>\n",
       "      <td>pylprotpredictor</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5595</th>\n",
       "      <td>pymisc-utils</td>\n",
       "      <td>3464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5596</th>\n",
       "      <td>pymix</td>\n",
       "      <td>1934</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5597</th>\n",
       "      <td>pymot</td>\n",
       "      <td>1815</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5598</th>\n",
       "      <td>pymummer</td>\n",
       "      <td>16117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5599</th>\n",
       "      <td>pymvpa</td>\n",
       "      <td>6436</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5600</th>\n",
       "      <td>pymzml</td>\n",
       "      <td>10626</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5601</th>\n",
       "      <td>pynast</td>\n",
       "      <td>30800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5602</th>\n",
       "      <td>pynnotator</td>\n",
       "      <td>363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5603</th>\n",
       "      <td>pyomero-upload</td>\n",
       "      <td>343</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5604</th>\n",
       "      <td>pyopt</td>\n",
       "      <td>1663</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5605</th>\n",
       "      <td>pypairs</td>\n",
       "      <td>725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5606</th>\n",
       "      <td>pypeflow</td>\n",
       "      <td>12069</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5607</th>\n",
       "      <td>pypgatk</td>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5608</th>\n",
       "      <td>pypiper</td>\n",
       "      <td>3383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5609</th>\n",
       "      <td>pypore</td>\n",
       "      <td>5364</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5610</th>\n",
       "      <td>pyprophet</td>\n",
       "      <td>6276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5611</th>\n",
       "      <td>pyquant-ms</td>\n",
       "      <td>7514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5612</th>\n",
       "      <td>pyrad</td>\n",
       "      <td>3416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5613</th>\n",
       "      <td>pyranges</td>\n",
       "      <td>35568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5614</th>\n",
       "      <td>pyrle</td>\n",
       "      <td>28693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5615</th>\n",
       "      <td>pyrodigal</td>\n",
       "      <td>360</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5616</th>\n",
       "      <td>pyrpipe</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5617</th>\n",
       "      <td>pysam</td>\n",
       "      <td>1415206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5618</th>\n",
       "      <td>pysamstats</td>\n",
       "      <td>18500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5619</th>\n",
       "      <td>pyscaf</td>\n",
       "      <td>1867</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5620</th>\n",
       "      <td>pyseer</td>\n",
       "      <td>6921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5621</th>\n",
       "      <td>pysnptools</td>\n",
       "      <td>3760</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5622</th>\n",
       "      <td>pysradb</td>\n",
       "      <td>5658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5623</th>\n",
       "      <td>pysvg</td>\n",
       "      <td>4221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5624</th>\n",
       "      <td>pysvmlight</td>\n",
       "      <td>1928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5625</th>\n",
       "      <td>pytabix</td>\n",
       "      <td>71032</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5626</th>\n",
       "      <td>pytaxonkit</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5627</th>\n",
       "      <td>pyteomics</td>\n",
       "      <td>23409</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5628</th>\n",
       "      <td>pytest-marks</td>\n",
       "      <td>3065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5629</th>\n",
       "      <td>pythomics</td>\n",
       "      <td>3987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5630</th>\n",
       "      <td>python-bioext</td>\n",
       "      <td>5556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5631</th>\n",
       "      <td>python-bioformats</td>\n",
       "      <td>331</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5632</th>\n",
       "      <td>python-chado</td>\n",
       "      <td>10172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5633</th>\n",
       "      <td>python-cluster</td>\n",
       "      <td>4297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5634</th>\n",
       "      <td>python-consensuscore</td>\n",
       "      <td>12253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5635</th>\n",
       "      <td>python-consensuscore2</td>\n",
       "      <td>6189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5636</th>\n",
       "      <td>python-edlib</td>\n",
       "      <td>20187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5637</th>\n",
       "      <td>python-eel</td>\n",
       "      <td>966</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5638</th>\n",
       "      <td>python-fakemp</td>\n",
       "      <td>2915</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5639</th>\n",
       "      <td>python-goenrichment</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5640</th>\n",
       "      <td>python-hivclustering</td>\n",
       "      <td>7798</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5641</th>\n",
       "      <td>python-hppy</td>\n",
       "      <td>3600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5642</th>\n",
       "      <td>python-htsget</td>\n",
       "      <td>917</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5643</th>\n",
       "      <td>python-hyphy-python</td>\n",
       "      <td>7274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5644</th>\n",
       "      <td>python-intervaltree</td>\n",
       "      <td>4586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5645</th>\n",
       "      <td>python-libsbml</td>\n",
       "      <td>4972</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5646</th>\n",
       "      <td>python-mailund-newick</td>\n",
       "      <td>1971</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5647</th>\n",
       "      <td>python-msgpack</td>\n",
       "      <td>8375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5648</th>\n",
       "      <td>python-newick</td>\n",
       "      <td>4088</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5649</th>\n",
       "      <td>pythonpy</td>\n",
       "      <td>9922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5650</th>\n",
       "      <td>python-rnacentral</td>\n",
       "      <td>90</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5651</th>\n",
       "      <td>python-sortedcontainers</td>\n",
       "      <td>4690</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5652</th>\n",
       "      <td>python-tripal</td>\n",
       "      <td>14176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5653</th>\n",
       "      <td>pytriangle</td>\n",
       "      <td>4727</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5654</th>\n",
       "      <td>pywdl</td>\n",
       "      <td>2131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5655</th>\n",
       "      <td>pyx</td>\n",
       "      <td>4410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5656</th>\n",
       "      <td>qcat</td>\n",
       "      <td>2261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5657</th>\n",
       "      <td>qcli</td>\n",
       "      <td>31867</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5658</th>\n",
       "      <td>qcumber</td>\n",
       "      <td>1656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5659</th>\n",
       "      <td>qfilt</td>\n",
       "      <td>2109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5660</th>\n",
       "      <td>qgrs-cpp</td>\n",
       "      <td>1554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5661</th>\n",
       "      <td>qiime</td>\n",
       "      <td>46811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5662</th>\n",
       "      <td>qiime-default-reference</td>\n",
       "      <td>26606</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5663</th>\n",
       "      <td>qiimetomaaslin</td>\n",
       "      <td>1752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5664</th>\n",
       "      <td>qorts</td>\n",
       "      <td>4406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5665</th>\n",
       "      <td>qsignature</td>\n",
       "      <td>8331</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5666</th>\n",
       "      <td>qtip</td>\n",
       "      <td>2276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5667</th>\n",
       "      <td>qtlseq</td>\n",
       "      <td>715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5668</th>\n",
       "      <td>quake</td>\n",
       "      <td>2075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5669</th>\n",
       "      <td>qualimap</td>\n",
       "      <td>721</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5670</th>\n",
       "      <td>qualimap</td>\n",
       "      <td>74370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5671</th>\n",
       "      <td>quantiprot</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5672</th>\n",
       "      <td>quantwiz-iq</td>\n",
       "      <td>328</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5673</th>\n",
       "      <td>quasildr</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5674</th>\n",
       "      <td>quasirecomb</td>\n",
       "      <td>76</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5675</th>\n",
       "      <td>quasitools</td>\n",
       "      <td>23372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5676</th>\n",
       "      <td>quast</td>\n",
       "      <td>121985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5677</th>\n",
       "      <td>query_phenomizer</td>\n",
       "      <td>2203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5678</th>\n",
       "      <td>quickmerge</td>\n",
       "      <td>685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5679</th>\n",
       "      <td>quicksect</td>\n",
       "      <td>14408</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5680</th>\n",
       "      <td>quicktree</td>\n",
       "      <td>5615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5681</th>\n",
       "      <td>quorum</td>\n",
       "      <td>2094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5682</th>\n",
       "      <td>r2r</td>\n",
       "      <td>3205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5683</th>\n",
       "      <td>ra</td>\n",
       "      <td>1792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5684</th>\n",
       "      <td>rabix-bunny</td>\n",
       "      <td>15977</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5685</th>\n",
       "      <td>r-acidbase</td>\n",
       "      <td>1427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5686</th>\n",
       "      <td>r-acidgenerics</td>\n",
       "      <td>964</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5687</th>\n",
       "      <td>r-acidplots</td>\n",
       "      <td>1178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5688</th>\n",
       "      <td>racon</td>\n",
       "      <td>52675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5689</th>\n",
       "      <td>rad_haplotyper</td>\n",
       "      <td>5095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5690</th>\n",
       "      <td>radsex</td>\n",
       "      <td>2056</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5691</th>\n",
       "      <td>ragout</td>\n",
       "      <td>8007</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5692</th>\n",
       "      <td>ragtag</td>\n",
       "      <td>236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5693</th>\n",
       "      <td>rainbow</td>\n",
       "      <td>5337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5694</th>\n",
       "      <td>ra-integrate</td>\n",
       "      <td>1553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5695</th>\n",
       "      <td>raiss</td>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5696</th>\n",
       "      <td>rambo-k</td>\n",
       "      <td>1942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5697</th>\n",
       "      <td>r-ampvis</td>\n",
       "      <td>2012</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5698</th>\n",
       "      <td>randfold</td>\n",
       "      <td>3954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5699</th>\n",
       "      <td>r-annotables</td>\n",
       "      <td>1862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5700</th>\n",
       "      <td>rapclust</td>\n",
       "      <td>3949</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5701</th>\n",
       "      <td>rapid</td>\n",
       "      <td>14449</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5702</th>\n",
       "      <td>rapidnj</td>\n",
       "      <td>3258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5703</th>\n",
       "      <td>rapifilt</td>\n",
       "      <td>444</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5704</th>\n",
       "      <td>rapmap</td>\n",
       "      <td>13737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5705</th>\n",
       "      <td>rappas</td>\n",
       "      <td>310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5706</th>\n",
       "      <td>rapsearch</td>\n",
       "      <td>4378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5707</th>\n",
       "      <td>r-aroma.affymetrix</td>\n",
       "      <td>7435</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5708</th>\n",
       "      <td>r-aroma.core</td>\n",
       "      <td>7659</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5709</th>\n",
       "      <td>rascaf</td>\n",
       "      <td>1887</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5710</th>\n",
       "      <td>rasusa</td>\n",
       "      <td>1505</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5711</th>\n",
       "      <td>ratt</td>\n",
       "      <td>153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5712</th>\n",
       "      <td>raven-assembler</td>\n",
       "      <td>5280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5713</th>\n",
       "      <td>rawtools</td>\n",
       "      <td>520</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5714</th>\n",
       "      <td>raxml</td>\n",
       "      <td>112386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5715</th>\n",
       "      <td>raxml-ng</td>\n",
       "      <td>3117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5716</th>\n",
       "      <td>ray</td>\n",
       "      <td>3923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5717</th>\n",
       "      <td>razers3</td>\n",
       "      <td>19593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5718</th>\n",
       "      <td>r-basejump</td>\n",
       "      <td>16258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5719</th>\n",
       "      <td>r-bcbiobase</td>\n",
       "      <td>14046</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5720</th>\n",
       "      <td>r-bcbiornaseq</td>\n",
       "      <td>14616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5721</th>\n",
       "      <td>r-bcbiosinglecell</td>\n",
       "      <td>439</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5722</th>\n",
       "      <td>r-biodb</td>\n",
       "      <td>10998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5723</th>\n",
       "      <td>r-bisquerna</td>\n",
       "      <td>272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5724</th>\n",
       "      <td>r-cate</td>\n",
       "      <td>1627</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5725</th>\n",
       "      <td>r-cellassign</td>\n",
       "      <td>4555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5726</th>\n",
       "      <td>r-chbutils</td>\n",
       "      <td>8897</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5727</th>\n",
       "      <td>r-cimpl</td>\n",
       "      <td>1987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5728</th>\n",
       "      <td>r-classdiscovery</td>\n",
       "      <td>4173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5729</th>\n",
       "      <td>r-coloc</td>\n",
       "      <td>9818</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5730</th>\n",
       "      <td>rcorrector</td>\n",
       "      <td>8461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5731</th>\n",
       "      <td>r-cp4p</td>\n",
       "      <td>381</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5732</th>\n",
       "      <td>r-dartr</td>\n",
       "      <td>3445</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5733</th>\n",
       "      <td>r-ddir</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5734</th>\n",
       "      <td>r-deconstructsigs</td>\n",
       "      <td>7099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5735</th>\n",
       "      <td>r-deseqanalysis</td>\n",
       "      <td>1088</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5736</th>\n",
       "      <td>rdfextras</td>\n",
       "      <td>2036</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5737</th>\n",
       "      <td>r-diffcorr</td>\n",
       "      <td>240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5738</th>\n",
       "      <td>r-dimsum</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5739</th>\n",
       "      <td>r-dnet</td>\n",
       "      <td>366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5740</th>\n",
       "      <td>rdock</td>\n",
       "      <td>2392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5741</th>\n",
       "      <td>rdp_classifier</td>\n",
       "      <td>2251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5742</th>\n",
       "      <td>r-dpeak</td>\n",
       "      <td>2145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5743</th>\n",
       "      <td>rdp-readseq</td>\n",
       "      <td>1737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5744</th>\n",
       "      <td>rdptools</td>\n",
       "      <td>5094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5745</th>\n",
       "      <td>r-eacon</td>\n",
       "      <td>54</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5746</th>\n",
       "      <td>readfq</td>\n",
       "      <td>1598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5747</th>\n",
       "      <td>readseq</td>\n",
       "      <td>6084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5748</th>\n",
       "      <td>readucks</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5749</th>\n",
       "      <td>reago</td>\n",
       "      <td>2140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5750</th>\n",
       "      <td>real</td>\n",
       "      <td>1677</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5751</th>\n",
       "      <td>realphy</td>\n",
       "      <td>1789</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5752</th>\n",
       "      <td>reaper</td>\n",
       "      <td>1994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5753</th>\n",
       "      <td>rebaler</td>\n",
       "      <td>5436</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5754</th>\n",
       "      <td>recalladapters</td>\n",
       "      <td>2102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5755</th>\n",
       "      <td>recentrifuge</td>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5756</th>\n",
       "      <td>recognizer</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5757</th>\n",
       "      <td>recon</td>\n",
       "      <td>5068</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5758</th>\n",
       "      <td>recycler</td>\n",
       "      <td>6630</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5759</th>\n",
       "      <td>red</td>\n",
       "      <td>1648</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5760</th>\n",
       "      <td>referenceseeker</td>\n",
       "      <td>568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5761</th>\n",
       "      <td>refgenconf</td>\n",
       "      <td>609</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5762</th>\n",
       "      <td>refgenie</td>\n",
       "      <td>592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5763</th>\n",
       "      <td>refgenieserver</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5764</th>\n",
       "      <td>refinem</td>\n",
       "      <td>563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5765</th>\n",
       "      <td>refseq_masher</td>\n",
       "      <td>4112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5766</th>\n",
       "      <td>regional</td>\n",
       "      <td>874</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5767</th>\n",
       "      <td>regtools</td>\n",
       "      <td>4896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5768</th>\n",
       "      <td>relocate2</td>\n",
       "      <td>3550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5769</th>\n",
       "      <td>r-empiricalfdr.deseq2</td>\n",
       "      <td>2485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5770</th>\n",
       "      <td>remurna</td>\n",
       "      <td>1869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5771</th>\n",
       "      <td>rename</td>\n",
       "      <td>5196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5772</th>\n",
       "      <td>repaq</td>\n",
       "      <td>656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5773</th>\n",
       "      <td>reparation_blast</td>\n",
       "      <td>12179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5774</th>\n",
       "      <td>repdenovo</td>\n",
       "      <td>873</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5775</th>\n",
       "      <td>repeatmasker</td>\n",
       "      <td>19287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5776</th>\n",
       "      <td>repeatmodeler</td>\n",
       "      <td>7016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5777</th>\n",
       "      <td>repeatscout</td>\n",
       "      <td>5561</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5778</th>\n",
       "      <td>repenrich</td>\n",
       "      <td>1777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5779</th>\n",
       "      <td>r-epic</td>\n",
       "      <td>2064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5780</th>\n",
       "      <td>req</td>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5781</th>\n",
       "      <td>revbayes</td>\n",
       "      <td>944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5782</th>\n",
       "      <td>reveal</td>\n",
       "      <td>2130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5783</th>\n",
       "      <td>revtrans</td>\n",
       "      <td>1619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5784</th>\n",
       "      <td>r-exomedepth</td>\n",
       "      <td>5068</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5785</th>\n",
       "      <td>r-facets</td>\n",
       "      <td>2676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5786</th>\n",
       "      <td>r-fastbaps</td>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5787</th>\n",
       "      <td>r-fgwas</td>\n",
       "      <td>3522</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5788</th>\n",
       "      <td>r-firebrowser</td>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5789</th>\n",
       "      <td>rfmix</td>\n",
       "      <td>1703</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5790</th>\n",
       "      <td>r-garnett</td>\n",
       "      <td>328</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5791</th>\n",
       "      <td>rgccacmd</td>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5792</th>\n",
       "      <td>r-genometricorr</td>\n",
       "      <td>1581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5793</th>\n",
       "      <td>r-genomictools</td>\n",
       "      <td>348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5794</th>\n",
       "      <td>r-genomictools.filehandler</td>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5795</th>\n",
       "      <td>r-george</td>\n",
       "      <td>1705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5796</th>\n",
       "      <td>r-ggbiplot</td>\n",
       "      <td>2161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5797</th>\n",
       "      <td>rgi</td>\n",
       "      <td>22986</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5798</th>\n",
       "      <td>r-gkmsvm</td>\n",
       "      <td>4103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5799</th>\n",
       "      <td>r-gmwt</td>\n",
       "      <td>348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5800</th>\n",
       "      <td>r-goalie</td>\n",
       "      <td>6719</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5801</th>\n",
       "      <td>r-grain</td>\n",
       "      <td>1598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5802</th>\n",
       "      <td>r-grbase</td>\n",
       "      <td>1761</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5803</th>\n",
       "      <td>rgt</td>\n",
       "      <td>3920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5804</th>\n",
       "      <td>r-gwpcr</td>\n",
       "      <td>3777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5805</th>\n",
       "      <td>r-harmony</td>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5806</th>\n",
       "      <td>r-hemdag</td>\n",
       "      <td>11575</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5807</th>\n",
       "      <td>r-histonehmm</td>\n",
       "      <td>7294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5808</th>\n",
       "      <td>r-htscluster</td>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5809</th>\n",
       "      <td>r-htssip</td>\n",
       "      <td>2318</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5810</th>\n",
       "      <td>ribocode</td>\n",
       "      <td>9401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5811</th>\n",
       "      <td>ribodiff</td>\n",
       "      <td>1896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5812</th>\n",
       "      <td>riboplot</td>\n",
       "      <td>1937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5813</th>\n",
       "      <td>riboraptor</td>\n",
       "      <td>1887</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5814</th>\n",
       "      <td>riborex</td>\n",
       "      <td>1629</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5815</th>\n",
       "      <td>riboseed</td>\n",
       "      <td>13897</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5816</th>\n",
       "      <td>riboseqc</td>\n",
       "      <td>79</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5817</th>\n",
       "      <td>riboseq-rust</td>\n",
       "      <td>1618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5818</th>\n",
       "      <td>ribotaper</td>\n",
       "      <td>4598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5819</th>\n",
       "      <td>ribotish</td>\n",
       "      <td>4647</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5820</th>\n",
       "      <td>ribotricer</td>\n",
       "      <td>1535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5821</th>\n",
       "      <td>r-ic10</td>\n",
       "      <td>3725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5822</th>\n",
       "      <td>r-ichorcna</td>\n",
       "      <td>2276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5823</th>\n",
       "      <td>rilseq</td>\n",
       "      <td>326</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5824</th>\n",
       "      <td>r-immunedeconv</td>\n",
       "      <td>625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5825</th>\n",
       "      <td>r-imputelcmd</td>\n",
       "      <td>357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5826</th>\n",
       "      <td>r-intego</td>\n",
       "      <td>1777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5827</th>\n",
       "      <td>r-intlim</td>\n",
       "      <td>84</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5828</th>\n",
       "      <td>ripser</td>\n",
       "      <td>1153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5829</th>\n",
       "      <td>r-isogene</td>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5830</th>\n",
       "      <td>r-isva</td>\n",
       "      <td>392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5831</th>\n",
       "      <td>r-jackstraw</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5832</th>\n",
       "      <td>rkp</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5833</th>\n",
       "      <td>r-leapp</td>\n",
       "      <td>1619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5834</th>\n",
       "      <td>r-leidenbase</td>\n",
       "      <td>2075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5835</th>\n",
       "      <td>r-liger</td>\n",
       "      <td>1205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5836</th>\n",
       "      <td>r-lme4qtl</td>\n",
       "      <td>1615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5837</th>\n",
       "      <td>r-lncpipereporter</td>\n",
       "      <td>1897</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5838</th>\n",
       "      <td>r-loom</td>\n",
       "      <td>2004</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5839</th>\n",
       "      <td>r-loomr</td>\n",
       "      <td>155</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5840</th>\n",
       "      <td>rmap</td>\n",
       "      <td>2000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5841</th>\n",
       "      <td>rmats2sashimiplot</td>\n",
       "      <td>3982</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5842</th>\n",
       "      <td>rmats</td>\n",
       "      <td>9355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5843</th>\n",
       "      <td>rmblast</td>\n",
       "      <td>16134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5844</th>\n",
       "      <td>r-mcpcounter</td>\n",
       "      <td>2105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5845</th>\n",
       "      <td>r-metaboanalyst</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5846</th>\n",
       "      <td>r-metacoder</td>\n",
       "      <td>1261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5847</th>\n",
       "      <td>r-metalonda</td>\n",
       "      <td>2256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5848</th>\n",
       "      <td>r-metama</td>\n",
       "      <td>2093</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5849</th>\n",
       "      <td>rmetl</td>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5850</th>\n",
       "      <td>r-mgsz</td>\n",
       "      <td>71</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5851</th>\n",
       "      <td>r-minems2</td>\n",
       "      <td>4097</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5852</th>\n",
       "      <td>r-minionqc</td>\n",
       "      <td>1530</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5853</th>\n",
       "      <td>r-misha</td>\n",
       "      <td>3370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5854</th>\n",
       "      <td>r-mixkernel</td>\n",
       "      <td>1391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5855</th>\n",
       "      <td>r-mkmisc</td>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5856</th>\n",
       "      <td>r-mmgenome</td>\n",
       "      <td>3675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5857</th>\n",
       "      <td>r-monocle3</td>\n",
       "      <td>2614</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5858</th>\n",
       "      <td>r-motifbinner</td>\n",
       "      <td>970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5859</th>\n",
       "      <td>r-msqrob</td>\n",
       "      <td>3106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5860</th>\n",
       "      <td>r-mutoss</td>\n",
       "      <td>4253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5861</th>\n",
       "      <td>rnabloom</td>\n",
       "      <td>177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5862</th>\n",
       "      <td>rnablueprint</td>\n",
       "      <td>5503</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5863</th>\n",
       "      <td>rnabob</td>\n",
       "      <td>2521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5864</th>\n",
       "      <td>rna-cd</td>\n",
       "      <td>166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5865</th>\n",
       "      <td>rnachipintegrator</td>\n",
       "      <td>3680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5866</th>\n",
       "      <td>rnaclust</td>\n",
       "      <td>1627</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5867</th>\n",
       "      <td>rnacode</td>\n",
       "      <td>2932</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5868</th>\n",
       "      <td>rnalien</td>\n",
       "      <td>10660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5869</th>\n",
       "      <td>r-nanopore</td>\n",
       "      <td>3154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5870</th>\n",
       "      <td>r-nanostringnorm</td>\n",
       "      <td>4303</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5871</th>\n",
       "      <td>rnaquast</td>\n",
       "      <td>3024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5872</th>\n",
       "      <td>rnaredprint</td>\n",
       "      <td>498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5873</th>\n",
       "      <td>rnasamba</td>\n",
       "      <td>4537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5874</th>\n",
       "      <td>rna-seqc</td>\n",
       "      <td>11028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5875</th>\n",
       "      <td>rnashapes</td>\n",
       "      <td>5982</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5876</th>\n",
       "      <td>rnasketch</td>\n",
       "      <td>3378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5877</th>\n",
       "      <td>rnasnp</td>\n",
       "      <td>2427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5878</th>\n",
       "      <td>r-nastiseq</td>\n",
       "      <td>2537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5879</th>\n",
       "      <td>rnastructure</td>\n",
       "      <td>6003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5880</th>\n",
       "      <td>rnaz</td>\n",
       "      <td>6658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5881</th>\n",
       "      <td>r-nbpseq</td>\n",
       "      <td>1627</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5882</th>\n",
       "      <td>rnftools</td>\n",
       "      <td>9690</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5883</th>\n",
       "      <td>r-ngsplot</td>\n",
       "      <td>2702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5884</th>\n",
       "      <td>roary</td>\n",
       "      <td>20426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5885</th>\n",
       "      <td>roary2fripan.py</td>\n",
       "      <td>1649</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5886</th>\n",
       "      <td>r-ontologyplot</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5887</th>\n",
       "      <td>rop</td>\n",
       "      <td>1826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5888</th>\n",
       "      <td>ropebwt2</td>\n",
       "      <td>2669</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5889</th>\n",
       "      <td>roprofile</td>\n",
       "      <td>1592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5890</th>\n",
       "      <td>r-orqa</td>\n",
       "      <td>1621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5891</th>\n",
       "      <td>r-pctgcdata</td>\n",
       "      <td>2728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5892</th>\n",
       "      <td>r-peer</td>\n",
       "      <td>2153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5893</th>\n",
       "      <td>r-perfmeas</td>\n",
       "      <td>2022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5894</th>\n",
       "      <td>rpf-count-cds</td>\n",
       "      <td>182</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5895</th>\n",
       "      <td>r-phewas</td>\n",
       "      <td>1609</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5896</th>\n",
       "      <td>r-phyext2</td>\n",
       "      <td>2158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5897</th>\n",
       "      <td>r-phylomeasures</td>\n",
       "      <td>2043</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5898</th>\n",
       "      <td>r-phytools</td>\n",
       "      <td>4622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5899</th>\n",
       "      <td>r-pipette</td>\n",
       "      <td>678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5900</th>\n",
       "      <td>rpkmforgenes</td>\n",
       "      <td>1921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5901</th>\n",
       "      <td>r-popgenreport</td>\n",
       "      <td>1262</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5902</th>\n",
       "      <td>r-pophelper</td>\n",
       "      <td>1557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5903</th>\n",
       "      <td>r-pore</td>\n",
       "      <td>3666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5904</th>\n",
       "      <td>r-prestor</td>\n",
       "      <td>231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5905</th>\n",
       "      <td>r-probmetab</td>\n",
       "      <td>4004</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5906</th>\n",
       "      <td>r-pscbs</td>\n",
       "      <td>18787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5907</th>\n",
       "      <td>r-qorts</td>\n",
       "      <td>2154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5908</th>\n",
       "      <td>r-r4cker</td>\n",
       "      <td>162</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5909</th>\n",
       "      <td>r-rapidr</td>\n",
       "      <td>1985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5910</th>\n",
       "      <td>r-riborex</td>\n",
       "      <td>1376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5911</th>\n",
       "      <td>r-rphast</td>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5912</th>\n",
       "      <td>r-rrbgen</td>\n",
       "      <td>1848</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5913</th>\n",
       "      <td>r-rtassel</td>\n",
       "      <td>67</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5914</th>\n",
       "      <td>r-rtfbs</td>\n",
       "      <td>2339</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5915</th>\n",
       "      <td>r-rubic</td>\n",
       "      <td>4782</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5916</th>\n",
       "      <td>r-samr</td>\n",
       "      <td>5819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5917</th>\n",
       "      <td>r-sartools</td>\n",
       "      <td>12797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5918</th>\n",
       "      <td>rsat-core</td>\n",
       "      <td>1998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5919</th>\n",
       "      <td>r-sbpiper</td>\n",
       "      <td>3521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5920</th>\n",
       "      <td>rscape</td>\n",
       "      <td>12824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5921</th>\n",
       "      <td>r-sceasy</td>\n",
       "      <td>494</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5922</th>\n",
       "      <td>r-scimpute</td>\n",
       "      <td>3376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5923</th>\n",
       "      <td>r-scpred</td>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5924</th>\n",
       "      <td>rseg</td>\n",
       "      <td>2078</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5925</th>\n",
       "      <td>rsem</td>\n",
       "      <td>32054</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5926</th>\n",
       "      <td>rseqc</td>\n",
       "      <td>71908</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5927</th>\n",
       "      <td>r-sequenza</td>\n",
       "      <td>2553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5928</th>\n",
       "      <td>r-sew</td>\n",
       "      <td>1181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5929</th>\n",
       "      <td>r-sgtr</td>\n",
       "      <td>1534</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5930</th>\n",
       "      <td>r-shaman</td>\n",
       "      <td>1388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5931</th>\n",
       "      <td>rsidx</td>\n",
       "      <td>155</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5932</th>\n",
       "      <td>r-sigqc</td>\n",
       "      <td>3365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5933</th>\n",
       "      <td>r-sigtree</td>\n",
       "      <td>2235</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5934</th>\n",
       "      <td>r-singlecellnet</td>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5935</th>\n",
       "      <td>r-sleuth</td>\n",
       "      <td>20440</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5936</th>\n",
       "      <td>r-smartsva</td>\n",
       "      <td>1534</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5937</th>\n",
       "      <td>r-soap-nmr</td>\n",
       "      <td>2100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5938</th>\n",
       "      <td>r-speaq</td>\n",
       "      <td>7441</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5939</th>\n",
       "      <td>r-spieceasi</td>\n",
       "      <td>2061</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5940</th>\n",
       "      <td>r-spp</td>\n",
       "      <td>18131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5941</th>\n",
       "      <td>r-statvisual</td>\n",
       "      <td>151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5942</th>\n",
       "      <td>r-stitch</td>\n",
       "      <td>6260</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5943</th>\n",
       "      <td>r-structssi</td>\n",
       "      <td>2294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5944</th>\n",
       "      <td>r-swamp</td>\n",
       "      <td>3818</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5945</th>\n",
       "      <td>r-syntactic</td>\n",
       "      <td>5665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5946</th>\n",
       "      <td>r-tailfindr</td>\n",
       "      <td>501</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5947</th>\n",
       "      <td>r-taxonomizr</td>\n",
       "      <td>2509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5948</th>\n",
       "      <td>r-tcga2stat</td>\n",
       "      <td>2303</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5949</th>\n",
       "      <td>r-tcr</td>\n",
       "      <td>3504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5950</th>\n",
       "      <td>r-tftargets</td>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5951</th>\n",
       "      <td>rtg-tools</td>\n",
       "      <td>30573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5952</th>\n",
       "      <td>r-tidygenomics</td>\n",
       "      <td>2924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5953</th>\n",
       "      <td>rtk</td>\n",
       "      <td>1542</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5954</th>\n",
       "      <td>r-tmae</td>\n",
       "      <td>451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5955</th>\n",
       "      <td>r-txrevise</td>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5956</th>\n",
       "      <td>ruby-dna-tools</td>\n",
       "      <td>2027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5957</th>\n",
       "      <td>rucs</td>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5958</th>\n",
       "      <td>ruffus</td>\n",
       "      <td>16028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5959</th>\n",
       "      <td>r-umi4c</td>\n",
       "      <td>1417</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5960</th>\n",
       "      <td>rust-bio-tools</td>\n",
       "      <td>31528</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5961</th>\n",
       "      <td>rust-ncbitaxonomy</td>\n",
       "      <td>6786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5962</th>\n",
       "      <td>rust-overlaps</td>\n",
       "      <td>2454</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5963</th>\n",
       "      <td>r-velocyto.r</td>\n",
       "      <td>805</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5964</th>\n",
       "      <td>r-virfinder</td>\n",
       "      <td>1261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5965</th>\n",
       "      <td>r-vision</td>\n",
       "      <td>826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5966</th>\n",
       "      <td>rvtests</td>\n",
       "      <td>2968</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5967</th>\n",
       "      <td>r-wasabi</td>\n",
       "      <td>17869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5968</th>\n",
       "      <td>r-wgcna</td>\n",
       "      <td>14044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5969</th>\n",
       "      <td>r-whopgenome</td>\n",
       "      <td>1918</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5970</th>\n",
       "      <td>r-workflowscriptscommon</td>\n",
       "      <td>3306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5971</th>\n",
       "      <td>r-xcell</td>\n",
       "      <td>1892</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5972</th>\n",
       "      <td>rxdock</td>\n",
       "      <td>1264</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5973</th>\n",
       "      <td>r-xgr</td>\n",
       "      <td>366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5974</th>\n",
       "      <td>r-xmlrpc</td>\n",
       "      <td>2345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5975</th>\n",
       "      <td>r-zerone</td>\n",
       "      <td>1758</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5976</th>\n",
       "      <td>s3gof3r</td>\n",
       "      <td>7168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5977</th>\n",
       "      <td>sabre</td>\n",
       "      <td>1333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5978</th>\n",
       "      <td>saffrontree</td>\n",
       "      <td>1675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5979</th>\n",
       "      <td>sailfish</td>\n",
       "      <td>14884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5980</th>\n",
       "      <td>sak</td>\n",
       "      <td>1904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5981</th>\n",
       "      <td>salmid</td>\n",
       "      <td>230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5982</th>\n",
       "      <td>salmon</td>\n",
       "      <td>116310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5983</th>\n",
       "      <td>sam</td>\n",
       "      <td>2361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5984</th>\n",
       "      <td>sambamba</td>\n",
       "      <td>202487</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5985</th>\n",
       "      <td>samblaster</td>\n",
       "      <td>32239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5986</th>\n",
       "      <td>samclip</td>\n",
       "      <td>10145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5987</th>\n",
       "      <td>sampei</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5988</th>\n",
       "      <td>sample-sheet</td>\n",
       "      <td>2696</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5989</th>\n",
       "      <td>samplot</td>\n",
       "      <td>17946</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5990</th>\n",
       "      <td>samsift</td>\n",
       "      <td>12485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5991</th>\n",
       "      <td>samsifter</td>\n",
       "      <td>2177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5992</th>\n",
       "      <td>samstats</td>\n",
       "      <td>1212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5993</th>\n",
       "      <td>samsum</td>\n",
       "      <td>1284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5994</th>\n",
       "      <td>samtools</td>\n",
       "      <td>1836974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5995</th>\n",
       "      <td>sanitizeme</td>\n",
       "      <td>146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5996</th>\n",
       "      <td>sarscov2formatter</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5997</th>\n",
       "      <td>sarscov2summary</td>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5998</th>\n",
       "      <td>satrap</td>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5999</th>\n",
       "      <td>satsuma2</td>\n",
       "      <td>1712</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6000</th>\n",
       "      <td>savage</td>\n",
       "      <td>7836</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6001</th>\n",
       "      <td>sbg-cwl-runner</td>\n",
       "      <td>2720</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6002</th>\n",
       "      <td>sbpipe</td>\n",
       "      <td>2053</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6003</th>\n",
       "      <td>sc3-scripts</td>\n",
       "      <td>275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6004</th>\n",
       "      <td>scaden</td>\n",
       "      <td>253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6005</th>\n",
       "      <td>scaffold_builder</td>\n",
       "      <td>1599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6006</th>\n",
       "      <td>scagaire</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6007</th>\n",
       "      <td>scala</td>\n",
       "      <td>3063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6008</th>\n",
       "      <td>scallop</td>\n",
       "      <td>2709</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6009</th>\n",
       "      <td>scallop-lr</td>\n",
       "      <td>1050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6010</th>\n",
       "      <td>scalpel</td>\n",
       "      <td>33981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6011</th>\n",
       "      <td>scanindel</td>\n",
       "      <td>1696</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6012</th>\n",
       "      <td>scanorama</td>\n",
       "      <td>312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6013</th>\n",
       "      <td>scanpy</td>\n",
       "      <td>25595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6014</th>\n",
       "      <td>scanpy-scripts</td>\n",
       "      <td>1695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6015</th>\n",
       "      <td>scapp</td>\n",
       "      <td>64</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6016</th>\n",
       "      <td>scater-scripts</td>\n",
       "      <td>137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6017</th>\n",
       "      <td>sccaf</td>\n",
       "      <td>604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6018</th>\n",
       "      <td>sccaller</td>\n",
       "      <td>3313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6019</th>\n",
       "      <td>scelvis</td>\n",
       "      <td>1067</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6020</th>\n",
       "      <td>scg</td>\n",
       "      <td>1818</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6021</th>\n",
       "      <td>schavott</td>\n",
       "      <td>7769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6022</th>\n",
       "      <td>schicexplorer</td>\n",
       "      <td>477</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6023</th>\n",
       "      <td>scikits-datasmooth</td>\n",
       "      <td>2384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6024</th>\n",
       "      <td>sciphi</td>\n",
       "      <td>2296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6025</th>\n",
       "      <td>scirpy</td>\n",
       "      <td>166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6026</th>\n",
       "      <td>scmap-cli</td>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6027</th>\n",
       "      <td>scnic</td>\n",
       "      <td>802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6028</th>\n",
       "      <td>scoary</td>\n",
       "      <td>4367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6029</th>\n",
       "      <td>scoop</td>\n",
       "      <td>4462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6030</th>\n",
       "      <td>scpred-cli</td>\n",
       "      <td>333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6031</th>\n",
       "      <td>scprep</td>\n",
       "      <td>1236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6032</th>\n",
       "      <td>scran-cli</td>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6033</th>\n",
       "      <td>scrappie</td>\n",
       "      <td>5220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6034</th>\n",
       "      <td>screamingbackpack</td>\n",
       "      <td>4388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6035</th>\n",
       "      <td>scrm</td>\n",
       "      <td>6861</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6036</th>\n",
       "      <td>scrublet</td>\n",
       "      <td>1819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6037</th>\n",
       "      <td>scssim</td>\n",
       "      <td>749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6038</th>\n",
       "      <td>sctools</td>\n",
       "      <td>624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6039</th>\n",
       "      <td>scvelo</td>\n",
       "      <td>732</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6040</th>\n",
       "      <td>scvi</td>\n",
       "      <td>3193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6041</th>\n",
       "      <td>scvis</td>\n",
       "      <td>120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6042</th>\n",
       "      <td>scvis_galaxy</td>\n",
       "      <td>86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6043</th>\n",
       "      <td>scvi-tools</td>\n",
       "      <td>87</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6044</th>\n",
       "      <td>scxa-plots</td>\n",
       "      <td>41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6045</th>\n",
       "      <td>sdm</td>\n",
       "      <td>1037</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6046</th>\n",
       "      <td>sdrf-pipelines</td>\n",
       "      <td>500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6047</th>\n",
       "      <td>seacr</td>\n",
       "      <td>487</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6048</th>\n",
       "      <td>searchgui</td>\n",
       "      <td>34090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6049</th>\n",
       "      <td>secapr</td>\n",
       "      <td>17312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6050</th>\n",
       "      <td>secimtools</td>\n",
       "      <td>2248</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6051</th>\n",
       "      <td>seedme</td>\n",
       "      <td>3410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6052</th>\n",
       "      <td>seer</td>\n",
       "      <td>1648</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6053</th>\n",
       "      <td>segemehl</td>\n",
       "      <td>6795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6054</th>\n",
       "      <td>segmentation-fold</td>\n",
       "      <td>4048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6055</th>\n",
       "      <td>segtools</td>\n",
       "      <td>4075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6056</th>\n",
       "      <td>segway</td>\n",
       "      <td>10739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6057</th>\n",
       "      <td>selectfasta</td>\n",
       "      <td>438</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6058</th>\n",
       "      <td>selectsequencesfrommsa</td>\n",
       "      <td>4308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6059</th>\n",
       "      <td>selene-sdk</td>\n",
       "      <td>13595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6060</th>\n",
       "      <td>selscan</td>\n",
       "      <td>4540</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6061</th>\n",
       "      <td>semeta</td>\n",
       "      <td>1649</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6062</th>\n",
       "      <td>semidbm</td>\n",
       "      <td>2980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6063</th>\n",
       "      <td>sensv</td>\n",
       "      <td>871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6064</th>\n",
       "      <td>sentieon</td>\n",
       "      <td>43429</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6065</th>\n",
       "      <td>sepp</td>\n",
       "      <td>34758</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6066</th>\n",
       "      <td>sepp-refgg138</td>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6067</th>\n",
       "      <td>sepp-refsilva128</td>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6068</th>\n",
       "      <td>seq2c</td>\n",
       "      <td>12771</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6069</th>\n",
       "      <td>seq2hla</td>\n",
       "      <td>3403</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6070</th>\n",
       "      <td>seq2science</td>\n",
       "      <td>335</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6071</th>\n",
       "      <td>seqan</td>\n",
       "      <td>26220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6072</th>\n",
       "      <td>seqan3</td>\n",
       "      <td>242</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6073</th>\n",
       "      <td>seqan_tcoffee</td>\n",
       "      <td>2352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6074</th>\n",
       "      <td>seqbuster</td>\n",
       "      <td>16181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6075</th>\n",
       "      <td>seqcluster</td>\n",
       "      <td>40295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6076</th>\n",
       "      <td>seqfu</td>\n",
       "      <td>698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6077</th>\n",
       "      <td>seq-gen</td>\n",
       "      <td>3433</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6078</th>\n",
       "      <td>seqhax</td>\n",
       "      <td>1233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6079</th>\n",
       "      <td>seqkit</td>\n",
       "      <td>80431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6080</th>\n",
       "      <td>seqlogo</td>\n",
       "      <td>9514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6081</th>\n",
       "      <td>seqmagick</td>\n",
       "      <td>5049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6082</th>\n",
       "      <td>seqmap</td>\n",
       "      <td>1655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6083</th>\n",
       "      <td>seqprep</td>\n",
       "      <td>5744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6084</th>\n",
       "      <td>seqscreen</td>\n",
       "      <td>80</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6085</th>\n",
       "      <td>seq-seq-pan</td>\n",
       "      <td>4000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6086</th>\n",
       "      <td>seqsero2</td>\n",
       "      <td>398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6087</th>\n",
       "      <td>seqstats</td>\n",
       "      <td>469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6088</th>\n",
       "      <td>seqtk</td>\n",
       "      <td>92153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6089</th>\n",
       "      <td>seq-to-first-iso</td>\n",
       "      <td>231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6090</th>\n",
       "      <td>sequana</td>\n",
       "      <td>24584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6091</th>\n",
       "      <td>sequencetools</td>\n",
       "      <td>4726</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6092</th>\n",
       "      <td>sequenza-utils</td>\n",
       "      <td>3438</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6093</th>\n",
       "      <td>sequnwinder</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6094</th>\n",
       "      <td>seqwish</td>\n",
       "      <td>2445</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6095</th>\n",
       "      <td>seqyclean</td>\n",
       "      <td>3277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6096</th>\n",
       "      <td>seroba</td>\n",
       "      <td>2221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6097</th>\n",
       "      <td>sesimcmc</td>\n",
       "      <td>993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6098</th>\n",
       "      <td>setuptools_cython</td>\n",
       "      <td>2709</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6099</th>\n",
       "      <td>seurat-scripts</td>\n",
       "      <td>331</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6100</th>\n",
       "      <td>sevenbridges-python</td>\n",
       "      <td>32373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6101</th>\n",
       "      <td>sexdeterrmine</td>\n",
       "      <td>2601</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6102</th>\n",
       "      <td>sff2fastq</td>\n",
       "      <td>2120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6103</th>\n",
       "      <td>sfld</td>\n",
       "      <td>417</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6104</th>\n",
       "      <td>sfs_code</td>\n",
       "      <td>1740</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6105</th>\n",
       "      <td>sga</td>\n",
       "      <td>4725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6106</th>\n",
       "      <td>shannon</td>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6107</th>\n",
       "      <td>shannon_cpp</td>\n",
       "      <td>1361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6108</th>\n",
       "      <td>shape_it</td>\n",
       "      <td>2347</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6109</th>\n",
       "      <td>shapeit4</td>\n",
       "      <td>2274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6110</th>\n",
       "      <td>shapemapper</td>\n",
       "      <td>1730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6111</th>\n",
       "      <td>shapeshifter</td>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6112</th>\n",
       "      <td>shapeshifter-cli</td>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6113</th>\n",
       "      <td>sharedmem</td>\n",
       "      <td>3638</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6114</th>\n",
       "      <td>shark</td>\n",
       "      <td>1313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6115</th>\n",
       "      <td>sherpas</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6116</th>\n",
       "      <td>shigatyper</td>\n",
       "      <td>145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6117</th>\n",
       "      <td>shiver</td>\n",
       "      <td>7320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6118</th>\n",
       "      <td>shmlast</td>\n",
       "      <td>6546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6119</th>\n",
       "      <td>shorah</td>\n",
       "      <td>12621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6120</th>\n",
       "      <td>shortstack</td>\n",
       "      <td>5118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6121</th>\n",
       "      <td>shovill</td>\n",
       "      <td>14189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6122</th>\n",
       "      <td>showit</td>\n",
       "      <td>1081</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6123</th>\n",
       "      <td>shustring</td>\n",
       "      <td>4783</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6124</th>\n",
       "      <td>sibelia</td>\n",
       "      <td>4345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6125</th>\n",
       "      <td>sicer2</td>\n",
       "      <td>1467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6126</th>\n",
       "      <td>sicer</td>\n",
       "      <td>4361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6127</th>\n",
       "      <td>sickle</td>\n",
       "      <td>7857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6128</th>\n",
       "      <td>sickle-trim</td>\n",
       "      <td>59616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6129</th>\n",
       "      <td>sidr</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6130</th>\n",
       "      <td>sierrapy</td>\n",
       "      <td>607</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6131</th>\n",
       "      <td>sift4g</td>\n",
       "      <td>1840</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6132</th>\n",
       "      <td>sigmut</td>\n",
       "      <td>268</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6133</th>\n",
       "      <td>sim4db</td>\n",
       "      <td>1785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6134</th>\n",
       "      <td>simka</td>\n",
       "      <td>4016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6135</th>\n",
       "      <td>simlord</td>\n",
       "      <td>7491</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6136</th>\n",
       "      <td>simplesam</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6137</th>\n",
       "      <td>simple_sv_annotation</td>\n",
       "      <td>24941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6138</th>\n",
       "      <td>simwalk2</td>\n",
       "      <td>1660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6139</th>\n",
       "      <td>sina</td>\n",
       "      <td>67310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6140</th>\n",
       "      <td>singlecell-barcodes</td>\n",
       "      <td>8705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6141</th>\n",
       "      <td>singlem</td>\n",
       "      <td>79</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6142</th>\n",
       "      <td>sirius-csifingerid</td>\n",
       "      <td>8017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6143</th>\n",
       "      <td>sis</td>\n",
       "      <td>3594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6144</th>\n",
       "      <td>sistr_cmd</td>\n",
       "      <td>11307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6145</th>\n",
       "      <td>sixgill</td>\n",
       "      <td>4433</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6146</th>\n",
       "      <td>ska</td>\n",
       "      <td>2217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6147</th>\n",
       "      <td>skesa</td>\n",
       "      <td>9840</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6148</th>\n",
       "      <td>sketchy</td>\n",
       "      <td>1490</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6149</th>\n",
       "      <td>skewer</td>\n",
       "      <td>11993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6150</th>\n",
       "      <td>skmer</td>\n",
       "      <td>4404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6151</th>\n",
       "      <td>slamdunk</td>\n",
       "      <td>5996</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6152</th>\n",
       "      <td>slclust</td>\n",
       "      <td>15084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6153</th>\n",
       "      <td>slimm</td>\n",
       "      <td>1655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6154</th>\n",
       "      <td>slivar</td>\n",
       "      <td>10960</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6155</th>\n",
       "      <td>slncky</td>\n",
       "      <td>1978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6156</th>\n",
       "      <td>sloika</td>\n",
       "      <td>1624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6157</th>\n",
       "      <td>smaca</td>\n",
       "      <td>258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6158</th>\n",
       "      <td>smafa</td>\n",
       "      <td>1015</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6159</th>\n",
       "      <td>smallgenomeutilities</td>\n",
       "      <td>4112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6160</th>\n",
       "      <td>smalt</td>\n",
       "      <td>4399</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6161</th>\n",
       "      <td>smartdenovo</td>\n",
       "      <td>2314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6162</th>\n",
       "      <td>smashbenchmarking</td>\n",
       "      <td>1604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6163</th>\n",
       "      <td>smashpp</td>\n",
       "      <td>1098</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6164</th>\n",
       "      <td>smcounter2</td>\n",
       "      <td>3777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6165</th>\n",
       "      <td>smeg</td>\n",
       "      <td>7705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6166</th>\n",
       "      <td>smhasher</td>\n",
       "      <td>2745</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6167</th>\n",
       "      <td>smina</td>\n",
       "      <td>3295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6168</th>\n",
       "      <td>smof</td>\n",
       "      <td>254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6169</th>\n",
       "      <td>smoove</td>\n",
       "      <td>16139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6170</th>\n",
       "      <td>smudgeplot</td>\n",
       "      <td>2313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6171</th>\n",
       "      <td>snakefmt</td>\n",
       "      <td>147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6172</th>\n",
       "      <td>snakemake</td>\n",
       "      <td>261989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6173</th>\n",
       "      <td>snakeparse</td>\n",
       "      <td>3411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6174</th>\n",
       "      <td>snap</td>\n",
       "      <td>8310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6175</th>\n",
       "      <td>snap-aligner</td>\n",
       "      <td>11222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6176</th>\n",
       "      <td>snaptools</td>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6177</th>\n",
       "      <td>sniffles</td>\n",
       "      <td>13847</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6178</th>\n",
       "      <td>snippy</td>\n",
       "      <td>30762</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6179</th>\n",
       "      <td>snmf</td>\n",
       "      <td>1894</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6180</th>\n",
       "      <td>snns</td>\n",
       "      <td>2113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6181</th>\n",
       "      <td>snoscan</td>\n",
       "      <td>6777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6182</th>\n",
       "      <td>snostrip</td>\n",
       "      <td>1127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6183</th>\n",
       "      <td>snp-dists</td>\n",
       "      <td>10769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6184</th>\n",
       "      <td>snpeff</td>\n",
       "      <td>87645</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6185</th>\n",
       "      <td>snpgenie</td>\n",
       "      <td>46</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6186</th>\n",
       "      <td>snpiphy</td>\n",
       "      <td>4905</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6187</th>\n",
       "      <td>snp-mutator</td>\n",
       "      <td>768</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6188</th>\n",
       "      <td>snpomatic</td>\n",
       "      <td>1715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6189</th>\n",
       "      <td>snp-pileup</td>\n",
       "      <td>4014</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6190</th>\n",
       "      <td>snp-pipeline</td>\n",
       "      <td>6321</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6191</th>\n",
       "      <td>snpsift</td>\n",
       "      <td>26730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6192</th>\n",
       "      <td>snp-sites</td>\n",
       "      <td>15811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6193</th>\n",
       "      <td>snpsplit</td>\n",
       "      <td>6181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6194</th>\n",
       "      <td>snver</td>\n",
       "      <td>175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6195</th>\n",
       "      <td>snvphyl-tools</td>\n",
       "      <td>3105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6196</th>\n",
       "      <td>soapaligner</td>\n",
       "      <td>2062</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6197</th>\n",
       "      <td>soapcoverage</td>\n",
       "      <td>1794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6198</th>\n",
       "      <td>soapdenovo2</td>\n",
       "      <td>4161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6199</th>\n",
       "      <td>soapdenovo2-errorcorrection</td>\n",
       "      <td>2152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6200</th>\n",
       "      <td>soapdenovo2-gapcloser</td>\n",
       "      <td>2395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6201</th>\n",
       "      <td>soapdenovo2-prepare</td>\n",
       "      <td>2077</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6202</th>\n",
       "      <td>soapdenovo-trans</td>\n",
       "      <td>2206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6203</th>\n",
       "      <td>soapec</td>\n",
       "      <td>2324</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6204</th>\n",
       "      <td>soapsplice</td>\n",
       "      <td>1822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6205</th>\n",
       "      <td>socksipy-branch</td>\n",
       "      <td>5296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6206</th>\n",
       "      <td>socru</td>\n",
       "      <td>6240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6207</th>\n",
       "      <td>solexaqa</td>\n",
       "      <td>1727</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6208</th>\n",
       "      <td>solvebio</td>\n",
       "      <td>8647</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6209</th>\n",
       "      <td>somaticseq</td>\n",
       "      <td>1993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6210</th>\n",
       "      <td>somatic-sniper</td>\n",
       "      <td>2221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6211</th>\n",
       "      <td>sonicparanoid</td>\n",
       "      <td>4145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6212</th>\n",
       "      <td>sonlib</td>\n",
       "      <td>2289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6213</th>\n",
       "      <td>sorted_nearest</td>\n",
       "      <td>35031</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6214</th>\n",
       "      <td>sortmerna</td>\n",
       "      <td>107451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6215</th>\n",
       "      <td>sourcetracker</td>\n",
       "      <td>168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6216</th>\n",
       "      <td>sourmash</td>\n",
       "      <td>45667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6217</th>\n",
       "      <td>spacepharer</td>\n",
       "      <td>658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6218</th>\n",
       "      <td>spades</td>\n",
       "      <td>97089</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6219</th>\n",
       "      <td>spaln</td>\n",
       "      <td>10605</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6220</th>\n",
       "      <td>spanki</td>\n",
       "      <td>1643</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6221</th>\n",
       "      <td>sparc</td>\n",
       "      <td>2552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6222</th>\n",
       "      <td>sparcc</td>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6223</th>\n",
       "      <td>sparseassembler</td>\n",
       "      <td>2497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6224</th>\n",
       "      <td>sparse-neighbors-search</td>\n",
       "      <td>5047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6225</th>\n",
       "      <td>spectacle</td>\n",
       "      <td>1711</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6226</th>\n",
       "      <td>spectra-cluster-cli</td>\n",
       "      <td>2096</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6227</th>\n",
       "      <td>spectral_hk</td>\n",
       "      <td>2056</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6228</th>\n",
       "      <td>spectrassembler</td>\n",
       "      <td>1961</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6229</th>\n",
       "      <td>spectrum_utils</td>\n",
       "      <td>1768</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6230</th>\n",
       "      <td>spine</td>\n",
       "      <td>1821</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6231</th>\n",
       "      <td>spingo</td>\n",
       "      <td>2162</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6232</th>\n",
       "      <td>spliceai</td>\n",
       "      <td>785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6233</th>\n",
       "      <td>spliceai-wrapper</td>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6234</th>\n",
       "      <td>splicemap</td>\n",
       "      <td>1727</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6235</th>\n",
       "      <td>splitmem</td>\n",
       "      <td>1750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6236</th>\n",
       "      <td>spoa</td>\n",
       "      <td>3071</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6237</th>\n",
       "      <td>spotyping</td>\n",
       "      <td>2149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6238</th>\n",
       "      <td>spotyping3</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6239</th>\n",
       "      <td>sprai</td>\n",
       "      <td>1700</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6240</th>\n",
       "      <td>spring</td>\n",
       "      <td>1063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6241</th>\n",
       "      <td>sprinkles</td>\n",
       "      <td>2744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6242</th>\n",
       "      <td>spydrpick</td>\n",
       "      <td>4545</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6243</th>\n",
       "      <td>sqlalchemy-datatables</td>\n",
       "      <td>3925</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6244</th>\n",
       "      <td>sqlitebrowser</td>\n",
       "      <td>1735</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6245</th>\n",
       "      <td>sqt</td>\n",
       "      <td>8981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6246</th>\n",
       "      <td>squeakr</td>\n",
       "      <td>3027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6247</th>\n",
       "      <td>squid</td>\n",
       "      <td>3479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6248</th>\n",
       "      <td>squizz</td>\n",
       "      <td>3718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6249</th>\n",
       "      <td>sra-tools</td>\n",
       "      <td>94909</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6250</th>\n",
       "      <td>srax</td>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6251</th>\n",
       "      <td>srnapipe</td>\n",
       "      <td>1533</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6252</th>\n",
       "      <td>srprism</td>\n",
       "      <td>17983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6253</th>\n",
       "      <td>srst2</td>\n",
       "      <td>6801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6254</th>\n",
       "      <td>ssake</td>\n",
       "      <td>1807</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6255</th>\n",
       "      <td>sshmm</td>\n",
       "      <td>1981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6256</th>\n",
       "      <td>ssu-align</td>\n",
       "      <td>1989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6257</th>\n",
       "      <td>ssuissero</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6258</th>\n",
       "      <td>stacks</td>\n",
       "      <td>39603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6259</th>\n",
       "      <td>stacks_summary</td>\n",
       "      <td>3913</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6260</th>\n",
       "      <td>staden_io_lib</td>\n",
       "      <td>2402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6261</th>\n",
       "      <td>stamp</td>\n",
       "      <td>3535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6262</th>\n",
       "      <td>star</td>\n",
       "      <td>267257</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6263</th>\n",
       "      <td>staramr</td>\n",
       "      <td>13956</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6264</th>\n",
       "      <td>starcode</td>\n",
       "      <td>8817</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6265</th>\n",
       "      <td>starfish</td>\n",
       "      <td>2241</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6266</th>\n",
       "      <td>star-fusion</td>\n",
       "      <td>30292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6267</th>\n",
       "      <td>stark</td>\n",
       "      <td>642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6268</th>\n",
       "      <td>starseqr</td>\n",
       "      <td>7746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6269</th>\n",
       "      <td>start-asap</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6270</th>\n",
       "      <td>stellar</td>\n",
       "      <td>1958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6271</th>\n",
       "      <td>stereogene</td>\n",
       "      <td>1769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6272</th>\n",
       "      <td>stoatydive</td>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6273</th>\n",
       "      <td>strainest</td>\n",
       "      <td>4351</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6274</th>\n",
       "      <td>strainseeker</td>\n",
       "      <td>394</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6275</th>\n",
       "      <td>straitrazor</td>\n",
       "      <td>1751</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6276</th>\n",
       "      <td>stream</td>\n",
       "      <td>31573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6277</th>\n",
       "      <td>stream_atac</td>\n",
       "      <td>12740</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6278</th>\n",
       "      <td>strelka</td>\n",
       "      <td>21613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6279</th>\n",
       "      <td>stride</td>\n",
       "      <td>2084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6280</th>\n",
       "      <td>strike</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6281</th>\n",
       "      <td>stringdecomposer</td>\n",
       "      <td>306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6282</th>\n",
       "      <td>stringmeup</td>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6283</th>\n",
       "      <td>stringmlst</td>\n",
       "      <td>15028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6284</th>\n",
       "      <td>stringtie</td>\n",
       "      <td>63535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6285</th>\n",
       "      <td>strip_it</td>\n",
       "      <td>2281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6286</th>\n",
       "      <td>strling</td>\n",
       "      <td>939</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6287</th>\n",
       "      <td>structure</td>\n",
       "      <td>3214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6288</th>\n",
       "      <td>structureharvester</td>\n",
       "      <td>2114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6289</th>\n",
       "      <td>strudel</td>\n",
       "      <td>1693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6290</th>\n",
       "      <td>sts-smctc</td>\n",
       "      <td>1903</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6291</th>\n",
       "      <td>subread</td>\n",
       "      <td>132027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6292</th>\n",
       "      <td>sumaclust</td>\n",
       "      <td>2096</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6293</th>\n",
       "      <td>suma_package</td>\n",
       "      <td>1817</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6294</th>\n",
       "      <td>super_distance</td>\n",
       "      <td>2262</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6295</th>\n",
       "      <td>super-focus</td>\n",
       "      <td>5394</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6296</th>\n",
       "      <td>suppa</td>\n",
       "      <td>2749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6297</th>\n",
       "      <td>sure</td>\n",
       "      <td>2001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6298</th>\n",
       "      <td>surpyvor</td>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6299</th>\n",
       "      <td>survivor</td>\n",
       "      <td>13287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6300</th>\n",
       "      <td>sv2</td>\n",
       "      <td>1406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6301</th>\n",
       "      <td>svaba</td>\n",
       "      <td>1202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6302</th>\n",
       "      <td>svanalyzer</td>\n",
       "      <td>885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6303</th>\n",
       "      <td>svdb</td>\n",
       "      <td>9689</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6304</th>\n",
       "      <td>svict</td>\n",
       "      <td>2462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6305</th>\n",
       "      <td>svim</td>\n",
       "      <td>1805</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6306</th>\n",
       "      <td>svim-asm</td>\n",
       "      <td>63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6307</th>\n",
       "      <td>svist4get</td>\n",
       "      <td>666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6308</th>\n",
       "      <td>svjedi</td>\n",
       "      <td>45</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6309</th>\n",
       "      <td>svmlight</td>\n",
       "      <td>1869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6310</th>\n",
       "      <td>svtk</td>\n",
       "      <td>493</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6311</th>\n",
       "      <td>svtools</td>\n",
       "      <td>17094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6312</th>\n",
       "      <td>svtyper</td>\n",
       "      <td>19689</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6313</th>\n",
       "      <td>svviz</td>\n",
       "      <td>5631</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6314</th>\n",
       "      <td>swalign</td>\n",
       "      <td>4154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6315</th>\n",
       "      <td>swarm</td>\n",
       "      <td>13352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6316</th>\n",
       "      <td>sweepfinder2</td>\n",
       "      <td>1978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6317</th>\n",
       "      <td>swga</td>\n",
       "      <td>5105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6318</th>\n",
       "      <td>swiftlink</td>\n",
       "      <td>2143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6319</th>\n",
       "      <td>swipe</td>\n",
       "      <td>3372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6320</th>\n",
       "      <td>sword</td>\n",
       "      <td>4027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6321</th>\n",
       "      <td>synapseclient</td>\n",
       "      <td>14180</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6322</th>\n",
       "      <td>tabix</td>\n",
       "      <td>101027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6323</th>\n",
       "      <td>tablet</td>\n",
       "      <td>3819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6324</th>\n",
       "      <td>tabview</td>\n",
       "      <td>2746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6325</th>\n",
       "      <td>taco</td>\n",
       "      <td>3665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6326</th>\n",
       "      <td>tadarida-c</td>\n",
       "      <td>4859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6327</th>\n",
       "      <td>tadarida-d</td>\n",
       "      <td>6461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6328</th>\n",
       "      <td>tadbit</td>\n",
       "      <td>623</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6329</th>\n",
       "      <td>tadtool</td>\n",
       "      <td>2375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6330</th>\n",
       "      <td>taeper</td>\n",
       "      <td>1923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6331</th>\n",
       "      <td>tag</td>\n",
       "      <td>400</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6332</th>\n",
       "      <td>tagger</td>\n",
       "      <td>1710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6333</th>\n",
       "      <td>tagore</td>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6334</th>\n",
       "      <td>takeabreak</td>\n",
       "      <td>2181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6335</th>\n",
       "      <td>talloc</td>\n",
       "      <td>1743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6336</th>\n",
       "      <td>tandem-genotypes</td>\n",
       "      <td>377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6337</th>\n",
       "      <td>tantan</td>\n",
       "      <td>3273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6338</th>\n",
       "      <td>tapestry</td>\n",
       "      <td>440</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6339</th>\n",
       "      <td>tardis</td>\n",
       "      <td>12546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6340</th>\n",
       "      <td>targetdb</td>\n",
       "      <td>74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6341</th>\n",
       "      <td>targetfinder</td>\n",
       "      <td>2973</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6342</th>\n",
       "      <td>targqc</td>\n",
       "      <td>2688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6343</th>\n",
       "      <td>tasmanian-mismatch</td>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6344</th>\n",
       "      <td>tassel</td>\n",
       "      <td>5571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6345</th>\n",
       "      <td>taxator-tk</td>\n",
       "      <td>1761</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6346</th>\n",
       "      <td>taxmapper</td>\n",
       "      <td>5345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6347</th>\n",
       "      <td>taxonkit</td>\n",
       "      <td>13187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6348</th>\n",
       "      <td>taxonomy</td>\n",
       "      <td>1143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6349</th>\n",
       "      <td>taxonomy_ranks</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6350</th>\n",
       "      <td>taxopy</td>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6351</th>\n",
       "      <td>taxsbp</td>\n",
       "      <td>402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6352</th>\n",
       "      <td>taxtastic</td>\n",
       "      <td>5480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6353</th>\n",
       "      <td>tbl2asn</td>\n",
       "      <td>47855</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6354</th>\n",
       "      <td>tbl2asn-forever</td>\n",
       "      <td>8605</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6355</th>\n",
       "      <td>tbox-scan</td>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6356</th>\n",
       "      <td>tb-profiler</td>\n",
       "      <td>3171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6357</th>\n",
       "      <td>tb_variant_filter</td>\n",
       "      <td>837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6358</th>\n",
       "      <td>tbvcfreport</td>\n",
       "      <td>390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6359</th>\n",
       "      <td>t-coffee</td>\n",
       "      <td>4671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6360</th>\n",
       "      <td>t_coffee</td>\n",
       "      <td>81184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6361</th>\n",
       "      <td>tdrmapper</td>\n",
       "      <td>9594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6362</th>\n",
       "      <td>tedna</td>\n",
       "      <td>1885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6363</th>\n",
       "      <td>telescope</td>\n",
       "      <td>1732</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6364</th>\n",
       "      <td>teloclip</td>\n",
       "      <td>1717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6365</th>\n",
       "      <td>telseq</td>\n",
       "      <td>3581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6366</th>\n",
       "      <td>tendo</td>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6367</th>\n",
       "      <td>tepeaks</td>\n",
       "      <td>795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6368</th>\n",
       "      <td>tepid</td>\n",
       "      <td>3350</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6369</th>\n",
       "      <td>terminus</td>\n",
       "      <td>604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6370</th>\n",
       "      <td>tesorter</td>\n",
       "      <td>695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6371</th>\n",
       "      <td>tetranscripts</td>\n",
       "      <td>2194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6372</th>\n",
       "      <td>tetyper</td>\n",
       "      <td>353</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6373</th>\n",
       "      <td>textinput</td>\n",
       "      <td>3202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6374</th>\n",
       "      <td>tgsgapcloser</td>\n",
       "      <td>421</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6375</th>\n",
       "      <td>tgt</td>\n",
       "      <td>4101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6376</th>\n",
       "      <td>thapbi-pict</td>\n",
       "      <td>2439</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6377</th>\n",
       "      <td>thermorawfileparser</td>\n",
       "      <td>12636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6378</th>\n",
       "      <td>thesias</td>\n",
       "      <td>631</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6379</th>\n",
       "      <td>theta2</td>\n",
       "      <td>6992</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6380</th>\n",
       "      <td>tiddit</td>\n",
       "      <td>6666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6381</th>\n",
       "      <td>tidehunter</td>\n",
       "      <td>2018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6382</th>\n",
       "      <td>tidyp</td>\n",
       "      <td>91896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6383</th>\n",
       "      <td>tigmint</td>\n",
       "      <td>328</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6384</th>\n",
       "      <td>tinscan</td>\n",
       "      <td>1876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6385</th>\n",
       "      <td>tinyalign</td>\n",
       "      <td>920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6386</th>\n",
       "      <td>tinyfasta</td>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6387</th>\n",
       "      <td>tinysink</td>\n",
       "      <td>1701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6388</th>\n",
       "      <td>tiptoft</td>\n",
       "      <td>3755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6389</th>\n",
       "      <td>tirmite</td>\n",
       "      <td>5298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6390</th>\n",
       "      <td>tmalign</td>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6391</th>\n",
       "      <td>tn93</td>\n",
       "      <td>11851</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6392</th>\n",
       "      <td>tobias</td>\n",
       "      <td>8379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6393</th>\n",
       "      <td>toil</td>\n",
       "      <td>29386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6394</th>\n",
       "      <td>toolshed</td>\n",
       "      <td>20049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6395</th>\n",
       "      <td>topas</td>\n",
       "      <td>178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6396</th>\n",
       "      <td>tophat</td>\n",
       "      <td>33640</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6397</th>\n",
       "      <td>tophat-recondition</td>\n",
       "      <td>17041</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6398</th>\n",
       "      <td>toulligqc</td>\n",
       "      <td>7106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6399</th>\n",
       "      <td>tpmcalculator</td>\n",
       "      <td>1526</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6400</th>\n",
       "      <td>tpp</td>\n",
       "      <td>3310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6401</th>\n",
       "      <td>tqdist</td>\n",
       "      <td>1683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6402</th>\n",
       "      <td>tracer</td>\n",
       "      <td>4023</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6403</th>\n",
       "      <td>trackhub</td>\n",
       "      <td>6884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6404</th>\n",
       "      <td>tracy</td>\n",
       "      <td>11815</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6405</th>\n",
       "      <td>traitar</td>\n",
       "      <td>75</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6406</th>\n",
       "      <td>transabyss</td>\n",
       "      <td>6478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6407</th>\n",
       "      <td>transcomb</td>\n",
       "      <td>1701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6408</th>\n",
       "      <td>transcov</td>\n",
       "      <td>234</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6409</th>\n",
       "      <td>transdecoder</td>\n",
       "      <td>23937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6410</th>\n",
       "      <td>transgenescan</td>\n",
       "      <td>1608</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6411</th>\n",
       "      <td>transindel</td>\n",
       "      <td>69</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6412</th>\n",
       "      <td>transit</td>\n",
       "      <td>5501</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6413</th>\n",
       "      <td>translate-gard</td>\n",
       "      <td>3232</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6414</th>\n",
       "      <td>translatorx</td>\n",
       "      <td>1916</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6415</th>\n",
       "      <td>translig</td>\n",
       "      <td>801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6416</th>\n",
       "      <td>transposcope</td>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6417</th>\n",
       "      <td>transposonpsi</td>\n",
       "      <td>82</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6418</th>\n",
       "      <td>transrate</td>\n",
       "      <td>885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6419</th>\n",
       "      <td>transrate-tools</td>\n",
       "      <td>2339</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6420</th>\n",
       "      <td>transtermhp</td>\n",
       "      <td>2218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6421</th>\n",
       "      <td>trawler</td>\n",
       "      <td>5288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6422</th>\n",
       "      <td>treebest</td>\n",
       "      <td>4945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6423</th>\n",
       "      <td>treecluster</td>\n",
       "      <td>35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6424</th>\n",
       "      <td>treekin</td>\n",
       "      <td>3465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6425</th>\n",
       "      <td>treemaker</td>\n",
       "      <td>2064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6426</th>\n",
       "      <td>treemix</td>\n",
       "      <td>4704</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6427</th>\n",
       "      <td>treerecs</td>\n",
       "      <td>314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6428</th>\n",
       "      <td>treesapp</td>\n",
       "      <td>2165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6429</th>\n",
       "      <td>treeswift</td>\n",
       "      <td>445</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6430</th>\n",
       "      <td>treetime</td>\n",
       "      <td>7993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6431</th>\n",
       "      <td>trf</td>\n",
       "      <td>18658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6432</th>\n",
       "      <td>triform2</td>\n",
       "      <td>2042</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6433</th>\n",
       "      <td>trimadap</td>\n",
       "      <td>6015</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6434</th>\n",
       "      <td>trimal</td>\n",
       "      <td>18495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6435</th>\n",
       "      <td>trim-galore</td>\n",
       "      <td>61752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6436</th>\n",
       "      <td>trimmomatic</td>\n",
       "      <td>108735</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6437</th>\n",
       "      <td>trinculo</td>\n",
       "      <td>1841</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6438</th>\n",
       "      <td>trinity</td>\n",
       "      <td>58666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6439</th>\n",
       "      <td>trinotate</td>\n",
       "      <td>9766</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6440</th>\n",
       "      <td>triqler</td>\n",
       "      <td>259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6441</th>\n",
       "      <td>trnanalysis</td>\n",
       "      <td>6120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6442</th>\n",
       "      <td>trnascan-se</td>\n",
       "      <td>17057</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6443</th>\n",
       "      <td>trtools</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6444</th>\n",
       "      <td>trumicount</td>\n",
       "      <td>9512</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6445</th>\n",
       "      <td>truvari</td>\n",
       "      <td>1720</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6446</th>\n",
       "      <td>trycycler</td>\n",
       "      <td>67</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6447</th>\n",
       "      <td>tspex</td>\n",
       "      <td>757</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6448</th>\n",
       "      <td>tssar</td>\n",
       "      <td>1914</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6449</th>\n",
       "      <td>twobitreader</td>\n",
       "      <td>7717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6450</th>\n",
       "      <td>twopaco</td>\n",
       "      <td>1741</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6451</th>\n",
       "      <td>tximport-scripts</td>\n",
       "      <td>87</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6452</th>\n",
       "      <td>ucsc-addcols</td>\n",
       "      <td>8000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6453</th>\n",
       "      <td>ucsc-ameme</td>\n",
       "      <td>6451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6454</th>\n",
       "      <td>ucsc-autodtd</td>\n",
       "      <td>6542</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6455</th>\n",
       "      <td>ucsc-autosql</td>\n",
       "      <td>6474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6456</th>\n",
       "      <td>ucsc-autoxml</td>\n",
       "      <td>6630</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6457</th>\n",
       "      <td>ucsc-avecols</td>\n",
       "      <td>6390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6458</th>\n",
       "      <td>ucsc-axtchain</td>\n",
       "      <td>7087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6459</th>\n",
       "      <td>ucsc-axtsort</td>\n",
       "      <td>6576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6460</th>\n",
       "      <td>ucsc-axtswap</td>\n",
       "      <td>6570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6461</th>\n",
       "      <td>ucsc-axttomaf</td>\n",
       "      <td>8043</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6462</th>\n",
       "      <td>ucsc-axttopsl</td>\n",
       "      <td>8008</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6463</th>\n",
       "      <td>ucsc-bamtopsl</td>\n",
       "      <td>5000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6464</th>\n",
       "      <td>ucsc-bedclip</td>\n",
       "      <td>14210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6465</th>\n",
       "      <td>ucsc-bedcommonregions</td>\n",
       "      <td>8087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6466</th>\n",
       "      <td>ucsc-bedcoverage</td>\n",
       "      <td>8052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6467</th>\n",
       "      <td>ucsc-bedextendranges</td>\n",
       "      <td>8095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6468</th>\n",
       "      <td>ucsc-bedgeneparts</td>\n",
       "      <td>7957</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6469</th>\n",
       "      <td>ucsc-bedgraphpack</td>\n",
       "      <td>8130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6470</th>\n",
       "      <td>ucsc-bedgraphtobigwig</td>\n",
       "      <td>72277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6471</th>\n",
       "      <td>ucsc-bedintersect</td>\n",
       "      <td>8003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6472</th>\n",
       "      <td>ucsc-beditemoverlapcount</td>\n",
       "      <td>8233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6473</th>\n",
       "      <td>ucsc-bedjointaboffset</td>\n",
       "      <td>2935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6474</th>\n",
       "      <td>ucsc-bedpileups</td>\n",
       "      <td>7980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6475</th>\n",
       "      <td>ucsc-bedremoveoverlap</td>\n",
       "      <td>8024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6476</th>\n",
       "      <td>ucsc-bedrestricttopositions</td>\n",
       "      <td>8152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6477</th>\n",
       "      <td>ucsc-bedsort</td>\n",
       "      <td>14341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6478</th>\n",
       "      <td>ucsc-bedtobigbed</td>\n",
       "      <td>35333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6479</th>\n",
       "      <td>ucsc-bedtogenepred</td>\n",
       "      <td>16092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6480</th>\n",
       "      <td>ucsc-bedtopsl</td>\n",
       "      <td>7966</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6481</th>\n",
       "      <td>ucsc-bedweedoverlapping</td>\n",
       "      <td>7983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6482</th>\n",
       "      <td>ucsc-bigbedinfo</td>\n",
       "      <td>14280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6483</th>\n",
       "      <td>ucsc-bigbednameditems</td>\n",
       "      <td>8136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6484</th>\n",
       "      <td>ucsc-bigbedsummary</td>\n",
       "      <td>13821</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6485</th>\n",
       "      <td>ucsc-bigbedtobed</td>\n",
       "      <td>19934</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6486</th>\n",
       "      <td>ucsc-bigmaftomaf</td>\n",
       "      <td>4741</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6487</th>\n",
       "      <td>ucsc-bigpsltopsl</td>\n",
       "      <td>6582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6488</th>\n",
       "      <td>ucsc-bigwigaverageoverbed</td>\n",
       "      <td>16017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6489</th>\n",
       "      <td>ucsc-bigwigcat</td>\n",
       "      <td>8293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6490</th>\n",
       "      <td>ucsc-bigwigcluster</td>\n",
       "      <td>8240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6491</th>\n",
       "      <td>ucsc-bigwigcorrelate</td>\n",
       "      <td>8166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6492</th>\n",
       "      <td>ucsc-bigwiginfo</td>\n",
       "      <td>20321</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6493</th>\n",
       "      <td>ucsc-bigwigmerge</td>\n",
       "      <td>10077</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6494</th>\n",
       "      <td>ucsc-bigwigsummary</td>\n",
       "      <td>27755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6495</th>\n",
       "      <td>ucsc-bigwigtobedgraph</td>\n",
       "      <td>20848</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6496</th>\n",
       "      <td>ucsc-bigwigtowig</td>\n",
       "      <td>16153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6497</th>\n",
       "      <td>ucsc-blasttopsl</td>\n",
       "      <td>8146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6498</th>\n",
       "      <td>ucsc-blat</td>\n",
       "      <td>5863</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6499</th>\n",
       "      <td>ucsc-catdir</td>\n",
       "      <td>6473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6500</th>\n",
       "      <td>ucsc-catuncomment</td>\n",
       "      <td>6406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6501</th>\n",
       "      <td>ucsc-cell-browser</td>\n",
       "      <td>1739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6502</th>\n",
       "      <td>ucsc-chainantirepeat</td>\n",
       "      <td>7941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6503</th>\n",
       "      <td>ucsc-chainbridge</td>\n",
       "      <td>1359</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6504</th>\n",
       "      <td>ucsc-chainfilter</td>\n",
       "      <td>8027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6505</th>\n",
       "      <td>ucsc-chainmergesort</td>\n",
       "      <td>8479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6506</th>\n",
       "      <td>ucsc-chainnet</td>\n",
       "      <td>8325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6507</th>\n",
       "      <td>ucsc-chainprenet</td>\n",
       "      <td>8130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6508</th>\n",
       "      <td>ucsc-chainsort</td>\n",
       "      <td>8057</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6509</th>\n",
       "      <td>ucsc-chainsplit</td>\n",
       "      <td>8247</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6510</th>\n",
       "      <td>ucsc-chainstitchid</td>\n",
       "      <td>8154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6511</th>\n",
       "      <td>ucsc-chainswap</td>\n",
       "      <td>7968</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6512</th>\n",
       "      <td>ucsc-chaintoaxt</td>\n",
       "      <td>8032</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6513</th>\n",
       "      <td>ucsc-chaintopsl</td>\n",
       "      <td>7955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6514</th>\n",
       "      <td>ucsc-chaintopslbasic</td>\n",
       "      <td>8035</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6515</th>\n",
       "      <td>ucsc-checkagpandfa</td>\n",
       "      <td>6496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6516</th>\n",
       "      <td>ucsc-checkcoveragegaps</td>\n",
       "      <td>6402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6517</th>\n",
       "      <td>ucsc-checkhgfindspec</td>\n",
       "      <td>6358</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6518</th>\n",
       "      <td>ucsc-checktablecoords</td>\n",
       "      <td>6413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6519</th>\n",
       "      <td>ucsc-chopfalines</td>\n",
       "      <td>6415</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6520</th>\n",
       "      <td>ucsc-chromgraphfrombin</td>\n",
       "      <td>6500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6521</th>\n",
       "      <td>ucsc-chromgraphtobin</td>\n",
       "      <td>6574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6522</th>\n",
       "      <td>ucsc-clustergenes</td>\n",
       "      <td>4784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6523</th>\n",
       "      <td>ucsc-coltransform</td>\n",
       "      <td>6432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6524</th>\n",
       "      <td>ucsc-countchars</td>\n",
       "      <td>6375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6525</th>\n",
       "      <td>ucsc-crtreeindexbed</td>\n",
       "      <td>8058</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6526</th>\n",
       "      <td>ucsc-crtreesearchbed</td>\n",
       "      <td>8047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6527</th>\n",
       "      <td>ucsc-dbsnoop</td>\n",
       "      <td>6539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6528</th>\n",
       "      <td>ucsc-dbtrash</td>\n",
       "      <td>6519</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6529</th>\n",
       "      <td>ucsc-endsinlf</td>\n",
       "      <td>2937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6530</th>\n",
       "      <td>ucsc-estorient</td>\n",
       "      <td>7728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6531</th>\n",
       "      <td>ucsc-expmatrixtobarchartbed</td>\n",
       "      <td>4931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6532</th>\n",
       "      <td>ucsc-faalign</td>\n",
       "      <td>6473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6533</th>\n",
       "      <td>ucsc-facmp</td>\n",
       "      <td>6381</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6534</th>\n",
       "      <td>ucsc-facount</td>\n",
       "      <td>7163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6535</th>\n",
       "      <td>ucsc-fafilter</td>\n",
       "      <td>6532</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6536</th>\n",
       "      <td>ucsc-fafiltern</td>\n",
       "      <td>6388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6537</th>\n",
       "      <td>ucsc-fafrag</td>\n",
       "      <td>6425</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6538</th>\n",
       "      <td>ucsc-fanoise</td>\n",
       "      <td>6396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6539</th>\n",
       "      <td>ucsc-faonerecord</td>\n",
       "      <td>8231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6540</th>\n",
       "      <td>ucsc-fapolyasizes</td>\n",
       "      <td>8154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6541</th>\n",
       "      <td>ucsc-farandomize</td>\n",
       "      <td>7950</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6542</th>\n",
       "      <td>ucsc-farc</td>\n",
       "      <td>7968</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6543</th>\n",
       "      <td>ucsc-fasize</td>\n",
       "      <td>8342</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6544</th>\n",
       "      <td>ucsc-fasomerecords</td>\n",
       "      <td>9000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6545</th>\n",
       "      <td>ucsc-fasplit</td>\n",
       "      <td>9048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6546</th>\n",
       "      <td>ucsc-fastqstatsandsubsample</td>\n",
       "      <td>4799</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6547</th>\n",
       "      <td>ucsc-fastqtofa</td>\n",
       "      <td>8119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6548</th>\n",
       "      <td>ucsc-fatofastq</td>\n",
       "      <td>8160</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6549</th>\n",
       "      <td>ucsc-fatotab</td>\n",
       "      <td>7962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6550</th>\n",
       "      <td>ucsc-fatotwobit</td>\n",
       "      <td>48589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6551</th>\n",
       "      <td>ucsc-fatrans</td>\n",
       "      <td>8153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6552</th>\n",
       "      <td>ucsc-featurebits</td>\n",
       "      <td>7992</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6553</th>\n",
       "      <td>ucsc-fetchchromsizes</td>\n",
       "      <td>16862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6554</th>\n",
       "      <td>ucsc-findmotif</td>\n",
       "      <td>7786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6555</th>\n",
       "      <td>ucsc-gaptolift</td>\n",
       "      <td>7997</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6556</th>\n",
       "      <td>ucsc-genepredcheck</td>\n",
       "      <td>9301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6557</th>\n",
       "      <td>ucsc-genepredfilter</td>\n",
       "      <td>8099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6558</th>\n",
       "      <td>ucsc-genepredhisto</td>\n",
       "      <td>7931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6559</th>\n",
       "      <td>ucsc-genepredsinglecover</td>\n",
       "      <td>8199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6560</th>\n",
       "      <td>ucsc-genepredtobed</td>\n",
       "      <td>24900</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6561</th>\n",
       "      <td>ucsc-genepredtobiggenepred</td>\n",
       "      <td>9386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6562</th>\n",
       "      <td>ucsc-genepredtofakepsl</td>\n",
       "      <td>9824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6563</th>\n",
       "      <td>ucsc-genepredtogtf</td>\n",
       "      <td>17857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6564</th>\n",
       "      <td>ucsc-genepredtomafframes</td>\n",
       "      <td>9594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6565</th>\n",
       "      <td>ucsc-genepredtoprot</td>\n",
       "      <td>4698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6566</th>\n",
       "      <td>ucsc-gensub2</td>\n",
       "      <td>2881</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6567</th>\n",
       "      <td>ucsc-getrna</td>\n",
       "      <td>7995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6568</th>\n",
       "      <td>ucsc-getrnapred</td>\n",
       "      <td>7985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6569</th>\n",
       "      <td>ucsc-gff3togenepred</td>\n",
       "      <td>18400</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6570</th>\n",
       "      <td>ucsc-gff3topsl</td>\n",
       "      <td>6531</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6571</th>\n",
       "      <td>ucsc-gmtime</td>\n",
       "      <td>4651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6572</th>\n",
       "      <td>ucsc-gtftogenepred</td>\n",
       "      <td>38615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6573</th>\n",
       "      <td>ucsc-headrest</td>\n",
       "      <td>4580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6574</th>\n",
       "      <td>ucsc-hgbbidblink</td>\n",
       "      <td>4618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6575</th>\n",
       "      <td>ucsc-hgfakeagp</td>\n",
       "      <td>4711</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6576</th>\n",
       "      <td>ucsc-hgfindspec</td>\n",
       "      <td>4600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6577</th>\n",
       "      <td>ucsc-hggcpercent</td>\n",
       "      <td>10363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6578</th>\n",
       "      <td>ucsc-hggoldgapgl</td>\n",
       "      <td>1373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6579</th>\n",
       "      <td>ucsc-hgloadbed</td>\n",
       "      <td>6357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6580</th>\n",
       "      <td>ucsc-hgloadchain</td>\n",
       "      <td>6453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6581</th>\n",
       "      <td>ucsc-hgloadmaf</td>\n",
       "      <td>6363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6582</th>\n",
       "      <td>ucsc-hgloadnet</td>\n",
       "      <td>4735</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6583</th>\n",
       "      <td>ucsc-hgloadout</td>\n",
       "      <td>4755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6584</th>\n",
       "      <td>ucsc-hgloadoutjoined</td>\n",
       "      <td>4673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6585</th>\n",
       "      <td>ucsc-hgloadsqltab</td>\n",
       "      <td>2896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6586</th>\n",
       "      <td>ucsc-hgloadwiggle</td>\n",
       "      <td>6480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6587</th>\n",
       "      <td>ucsc-hgspeciesrna</td>\n",
       "      <td>4686</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6588</th>\n",
       "      <td>ucsc-hgsqldump</td>\n",
       "      <td>4673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6589</th>\n",
       "      <td>ucsc-hgtrackdb</td>\n",
       "      <td>4595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6590</th>\n",
       "      <td>ucsc-hgvstovcf</td>\n",
       "      <td>1349</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6591</th>\n",
       "      <td>ucsc-htmlcheck</td>\n",
       "      <td>4608</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6592</th>\n",
       "      <td>ucsc-hubcheck</td>\n",
       "      <td>4792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6593</th>\n",
       "      <td>ucsc-hubpubliccheck</td>\n",
       "      <td>4544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6594</th>\n",
       "      <td>ucsc-ixixx</td>\n",
       "      <td>6962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6595</th>\n",
       "      <td>ucsc-lavtoaxt</td>\n",
       "      <td>4726</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6596</th>\n",
       "      <td>ucsc-lavtopsl</td>\n",
       "      <td>6475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6597</th>\n",
       "      <td>ucsc-ldhggene</td>\n",
       "      <td>4641</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6598</th>\n",
       "      <td>ucsc-liftover</td>\n",
       "      <td>39204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6599</th>\n",
       "      <td>ucsc-liftup</td>\n",
       "      <td>6543</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6600</th>\n",
       "      <td>ucsc-linestora</td>\n",
       "      <td>4717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6601</th>\n",
       "      <td>ucsc-localtime</td>\n",
       "      <td>4563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6602</th>\n",
       "      <td>ucsc-mafaddirows</td>\n",
       "      <td>6317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6603</th>\n",
       "      <td>ucsc-mafaddqrows</td>\n",
       "      <td>6527</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6604</th>\n",
       "      <td>ucsc-mafcoverage</td>\n",
       "      <td>6434</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6605</th>\n",
       "      <td>ucsc-maffetch</td>\n",
       "      <td>6397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6606</th>\n",
       "      <td>ucsc-maffilter</td>\n",
       "      <td>6588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6607</th>\n",
       "      <td>ucsc-maffrag</td>\n",
       "      <td>6373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6608</th>\n",
       "      <td>ucsc-maffrags</td>\n",
       "      <td>6414</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6609</th>\n",
       "      <td>ucsc-mafgene</td>\n",
       "      <td>6422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6610</th>\n",
       "      <td>ucsc-mafmefirst</td>\n",
       "      <td>6365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6611</th>\n",
       "      <td>ucsc-maforder</td>\n",
       "      <td>6347</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6612</th>\n",
       "      <td>ucsc-mafranges</td>\n",
       "      <td>6370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6613</th>\n",
       "      <td>ucsc-mafsinregion</td>\n",
       "      <td>6654</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6614</th>\n",
       "      <td>ucsc-mafspecieslist</td>\n",
       "      <td>6535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6615</th>\n",
       "      <td>ucsc-mafspeciessubset</td>\n",
       "      <td>6529</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6616</th>\n",
       "      <td>ucsc-mafsplit</td>\n",
       "      <td>6377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6617</th>\n",
       "      <td>ucsc-mafsplitpos</td>\n",
       "      <td>6565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6618</th>\n",
       "      <td>ucsc-maftoaxt</td>\n",
       "      <td>6485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6619</th>\n",
       "      <td>ucsc-maftobigmaf</td>\n",
       "      <td>4716</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6620</th>\n",
       "      <td>ucsc-maftopsl</td>\n",
       "      <td>6357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6621</th>\n",
       "      <td>ucsc-maftosnpbed</td>\n",
       "      <td>6517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6622</th>\n",
       "      <td>ucsc-maketablelist</td>\n",
       "      <td>4604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6623</th>\n",
       "      <td>ucsc-maskoutfa</td>\n",
       "      <td>8051</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6624</th>\n",
       "      <td>ucsc-mktime</td>\n",
       "      <td>4587</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6625</th>\n",
       "      <td>ucsc-mrnatogene</td>\n",
       "      <td>4701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6626</th>\n",
       "      <td>ucsc-netchainsubset</td>\n",
       "      <td>6483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6627</th>\n",
       "      <td>ucsc-netclass</td>\n",
       "      <td>8020</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6628</th>\n",
       "      <td>ucsc-netfilter</td>\n",
       "      <td>6493</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6629</th>\n",
       "      <td>ucsc-netsplit</td>\n",
       "      <td>4693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6630</th>\n",
       "      <td>ucsc-netsyntenic</td>\n",
       "      <td>7945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6631</th>\n",
       "      <td>ucsc-nettoaxt</td>\n",
       "      <td>7995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6632</th>\n",
       "      <td>ucsc-nettobed</td>\n",
       "      <td>9592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6633</th>\n",
       "      <td>ucsc-newprog</td>\n",
       "      <td>4602</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6634</th>\n",
       "      <td>ucsc-newpythonprog</td>\n",
       "      <td>9586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6635</th>\n",
       "      <td>ucsc-nibfrag</td>\n",
       "      <td>12491</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6636</th>\n",
       "      <td>ucsc-nibsize</td>\n",
       "      <td>9563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6637</th>\n",
       "      <td>ucsc-oligomatch</td>\n",
       "      <td>10938</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6638</th>\n",
       "      <td>ucsc-overlapselect</td>\n",
       "      <td>3355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6639</th>\n",
       "      <td>ucsc-para</td>\n",
       "      <td>2931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6640</th>\n",
       "      <td>ucsc-parafetch</td>\n",
       "      <td>9613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6641</th>\n",
       "      <td>ucsc-parahub</td>\n",
       "      <td>2868</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6642</th>\n",
       "      <td>ucsc-parahubstop</td>\n",
       "      <td>2945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6643</th>\n",
       "      <td>ucsc-paranode</td>\n",
       "      <td>2937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6644</th>\n",
       "      <td>ucsc-paranodestart</td>\n",
       "      <td>2882</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6645</th>\n",
       "      <td>ucsc-paranodestatus</td>\n",
       "      <td>2895</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6646</th>\n",
       "      <td>ucsc-paranodestop</td>\n",
       "      <td>2970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6647</th>\n",
       "      <td>ucsc-parasol</td>\n",
       "      <td>2893</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6648</th>\n",
       "      <td>ucsc-parasync</td>\n",
       "      <td>9622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6649</th>\n",
       "      <td>ucsc-paratestjob</td>\n",
       "      <td>2940</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6650</th>\n",
       "      <td>ucsc-positionaltblcheck</td>\n",
       "      <td>9844</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6651</th>\n",
       "      <td>ucsc-pslcat</td>\n",
       "      <td>9636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6652</th>\n",
       "      <td>ucsc-pslcdnafilter</td>\n",
       "      <td>11975</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6653</th>\n",
       "      <td>ucsc-pslcheck</td>\n",
       "      <td>10082</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6654</th>\n",
       "      <td>ucsc-psldropoverlap</td>\n",
       "      <td>9623</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6655</th>\n",
       "      <td>ucsc-pslfilter</td>\n",
       "      <td>9681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6656</th>\n",
       "      <td>ucsc-pslhisto</td>\n",
       "      <td>9485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6657</th>\n",
       "      <td>ucsc-pslliftsubrangeblat</td>\n",
       "      <td>9504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6658</th>\n",
       "      <td>ucsc-pslmap</td>\n",
       "      <td>8529</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6659</th>\n",
       "      <td>ucsc-pslmappostchain</td>\n",
       "      <td>5035</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6660</th>\n",
       "      <td>ucsc-pslmrnacover</td>\n",
       "      <td>9836</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6661</th>\n",
       "      <td>ucsc-pslpairs</td>\n",
       "      <td>9766</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6662</th>\n",
       "      <td>ucsc-pslpartition</td>\n",
       "      <td>9613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6663</th>\n",
       "      <td>ucsc-pslpostarget</td>\n",
       "      <td>9912</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6664</th>\n",
       "      <td>ucsc-pslpretty</td>\n",
       "      <td>9622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6665</th>\n",
       "      <td>ucsc-pslrc</td>\n",
       "      <td>4610</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6666</th>\n",
       "      <td>ucsc-pslrecalcmatch</td>\n",
       "      <td>10357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6667</th>\n",
       "      <td>ucsc-pslreps</td>\n",
       "      <td>9673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6668</th>\n",
       "      <td>ucsc-pslscore</td>\n",
       "      <td>9464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6669</th>\n",
       "      <td>ucsc-pslselect</td>\n",
       "      <td>9607</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6670</th>\n",
       "      <td>ucsc-pslsomerecords</td>\n",
       "      <td>2928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6671</th>\n",
       "      <td>ucsc-pslsort</td>\n",
       "      <td>10227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6672</th>\n",
       "      <td>ucsc-pslstats</td>\n",
       "      <td>9520</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6673</th>\n",
       "      <td>ucsc-pslswap</td>\n",
       "      <td>9629</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6674</th>\n",
       "      <td>ucsc-psltobed</td>\n",
       "      <td>9797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6675</th>\n",
       "      <td>ucsc-psltobigpsl</td>\n",
       "      <td>9857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6676</th>\n",
       "      <td>ucsc-psltochain</td>\n",
       "      <td>9958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6677</th>\n",
       "      <td>ucsc-psltopslx</td>\n",
       "      <td>9720</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6678</th>\n",
       "      <td>ucsc-pslxtofa</td>\n",
       "      <td>9489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6679</th>\n",
       "      <td>ucsc-qacagplift</td>\n",
       "      <td>9493</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6680</th>\n",
       "      <td>ucsc-qactoqa</td>\n",
       "      <td>9634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6681</th>\n",
       "      <td>ucsc-qactowig</td>\n",
       "      <td>9877</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6682</th>\n",
       "      <td>ucsc-qatoqac</td>\n",
       "      <td>9639</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6683</th>\n",
       "      <td>ucsc-randomlines</td>\n",
       "      <td>9578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6684</th>\n",
       "      <td>ucsc-rasqlquery</td>\n",
       "      <td>9521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6685</th>\n",
       "      <td>ucsc-ratolines</td>\n",
       "      <td>9535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6686</th>\n",
       "      <td>ucsc-ratotab</td>\n",
       "      <td>9481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6687</th>\n",
       "      <td>ucsc-rmfadups</td>\n",
       "      <td>9651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6688</th>\n",
       "      <td>ucsc-rowstocols</td>\n",
       "      <td>9560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6689</th>\n",
       "      <td>ucsc-spacedtotab</td>\n",
       "      <td>9503</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6690</th>\n",
       "      <td>ucsc-splitfile</td>\n",
       "      <td>9695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6691</th>\n",
       "      <td>ucsc-splitfilebycolumn</td>\n",
       "      <td>10028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6692</th>\n",
       "      <td>ucsc-sqltoxml</td>\n",
       "      <td>9639</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6693</th>\n",
       "      <td>ucsc-stringify</td>\n",
       "      <td>9510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6694</th>\n",
       "      <td>ucsc-subchar</td>\n",
       "      <td>9700</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6695</th>\n",
       "      <td>ucsc-subcolumn</td>\n",
       "      <td>9545</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6696</th>\n",
       "      <td>ucsc-taillines</td>\n",
       "      <td>9578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6697</th>\n",
       "      <td>ucsc-tdbquery</td>\n",
       "      <td>9765</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6698</th>\n",
       "      <td>ucsc-texthistogram</td>\n",
       "      <td>9457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6699</th>\n",
       "      <td>ucsc-ticktodate</td>\n",
       "      <td>9702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6700</th>\n",
       "      <td>ucsc-tolower</td>\n",
       "      <td>9840</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6701</th>\n",
       "      <td>ucsc-toupper</td>\n",
       "      <td>9624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6702</th>\n",
       "      <td>ucsc-transmappsltogenepred</td>\n",
       "      <td>5086</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6703</th>\n",
       "      <td>ucsc-trfbig</td>\n",
       "      <td>9601</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6704</th>\n",
       "      <td>ucsc-twobitdup</td>\n",
       "      <td>9507</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6705</th>\n",
       "      <td>ucsc-twobitinfo</td>\n",
       "      <td>15618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6706</th>\n",
       "      <td>ucsc-twobitmask</td>\n",
       "      <td>9676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6707</th>\n",
       "      <td>ucsc-twobittofa</td>\n",
       "      <td>27327</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6708</th>\n",
       "      <td>ucsc-validatefiles</td>\n",
       "      <td>9592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6709</th>\n",
       "      <td>ucsc-validatemanifest</td>\n",
       "      <td>9514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6710</th>\n",
       "      <td>ucsc-websync</td>\n",
       "      <td>2923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6711</th>\n",
       "      <td>ucsc-wigcorrelate</td>\n",
       "      <td>9906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6712</th>\n",
       "      <td>ucsc-wigtobigwig</td>\n",
       "      <td>53810</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6713</th>\n",
       "      <td>ucsc-wordline</td>\n",
       "      <td>9426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6714</th>\n",
       "      <td>ucsc-xmlcat</td>\n",
       "      <td>9662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6715</th>\n",
       "      <td>ucsc-xmltosql</td>\n",
       "      <td>9724</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6716</th>\n",
       "      <td>udocker</td>\n",
       "      <td>3398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6717</th>\n",
       "      <td>ultraheatmap</td>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6718</th>\n",
       "      <td>umap</td>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6719</th>\n",
       "      <td>umis</td>\n",
       "      <td>34432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6720</th>\n",
       "      <td>umitools</td>\n",
       "      <td>3829</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6721</th>\n",
       "      <td>umi_tools</td>\n",
       "      <td>38413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6722</th>\n",
       "      <td>unfazed</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6723</th>\n",
       "      <td>unicycler</td>\n",
       "      <td>19892</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6724</th>\n",
       "      <td>unifrac</td>\n",
       "      <td>52573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6725</th>\n",
       "      <td>uniprot</td>\n",
       "      <td>2186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6726</th>\n",
       "      <td>unitas</td>\n",
       "      <td>1557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6727</th>\n",
       "      <td>unitig-caller</td>\n",
       "      <td>3140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6728</th>\n",
       "      <td>unitig-counter</td>\n",
       "      <td>2976</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6729</th>\n",
       "      <td>unmerge</td>\n",
       "      <td>395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6730</th>\n",
       "      <td>upimapi</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6731</th>\n",
       "      <td>uropa</td>\n",
       "      <td>10470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6732</th>\n",
       "      <td>ushuffle</td>\n",
       "      <td>4301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6733</th>\n",
       "      <td>uvp</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6734</th>\n",
       "      <td>valet</td>\n",
       "      <td>2134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6735</th>\n",
       "      <td>validate-fasta-database</td>\n",
       "      <td>2218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6736</th>\n",
       "      <td>vamb</td>\n",
       "      <td>2365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6737</th>\n",
       "      <td>var-agg</td>\n",
       "      <td>2965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6738</th>\n",
       "      <td>varda2-client</td>\n",
       "      <td>150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6739</th>\n",
       "      <td>vardict</td>\n",
       "      <td>52342</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6740</th>\n",
       "      <td>vardict-java</td>\n",
       "      <td>42794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6741</th>\n",
       "      <td>varfish-annotator-cli</td>\n",
       "      <td>883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6742</th>\n",
       "      <td>vargeno</td>\n",
       "      <td>1621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6743</th>\n",
       "      <td>variantbam</td>\n",
       "      <td>14183</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6744</th>\n",
       "      <td>variantbreak</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6745</th>\n",
       "      <td>variant_tools</td>\n",
       "      <td>4576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6746</th>\n",
       "      <td>varifier</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6747</th>\n",
       "      <td>varlociraptor</td>\n",
       "      <td>15567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6748</th>\n",
       "      <td>varscan</td>\n",
       "      <td>42424</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6749</th>\n",
       "      <td>vawk</td>\n",
       "      <td>7412</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6750</th>\n",
       "      <td>vcf2db</td>\n",
       "      <td>29462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6751</th>\n",
       "      <td>vcf2genome</td>\n",
       "      <td>4704</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6752</th>\n",
       "      <td>vcf2maf</td>\n",
       "      <td>29421</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6753</th>\n",
       "      <td>vcf2variants</td>\n",
       "      <td>63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6754</th>\n",
       "      <td>vcfanno</td>\n",
       "      <td>36457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6755</th>\n",
       "      <td>vcf-annotator</td>\n",
       "      <td>1890</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6756</th>\n",
       "      <td>vcfkit</td>\n",
       "      <td>3874</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6757</th>\n",
       "      <td>vcflatten</td>\n",
       "      <td>1801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6758</th>\n",
       "      <td>vcflib</td>\n",
       "      <td>36923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6759</th>\n",
       "      <td>vcfpy</td>\n",
       "      <td>6497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6760</th>\n",
       "      <td>vcfsamplecompare</td>\n",
       "      <td>3013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6761</th>\n",
       "      <td>vcftoolbox</td>\n",
       "      <td>2177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6762</th>\n",
       "      <td>vcftools</td>\n",
       "      <td>88217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6763</th>\n",
       "      <td>vcontact2</td>\n",
       "      <td>531</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6764</th>\n",
       "      <td>vdjer</td>\n",
       "      <td>1781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6765</th>\n",
       "      <td>velocyto.py</td>\n",
       "      <td>3123</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6766</th>\n",
       "      <td>velvet</td>\n",
       "      <td>19866</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6767</th>\n",
       "      <td>velvet-sc</td>\n",
       "      <td>1854</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6768</th>\n",
       "      <td>vembrane</td>\n",
       "      <td>271</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6769</th>\n",
       "      <td>verifybamid2</td>\n",
       "      <td>8156</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6770</th>\n",
       "      <td>verifybamid</td>\n",
       "      <td>7200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6771</th>\n",
       "      <td>verse</td>\n",
       "      <td>1925</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6772</th>\n",
       "      <td>vg</td>\n",
       "      <td>2559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6773</th>\n",
       "      <td>vibrant</td>\n",
       "      <td>642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6774</th>\n",
       "      <td>viennarna</td>\n",
       "      <td>92753</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6775</th>\n",
       "      <td>virchip</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6776</th>\n",
       "      <td>virema</td>\n",
       "      <td>1688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6777</th>\n",
       "      <td>virmet</td>\n",
       "      <td>6572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6778</th>\n",
       "      <td>virsorter</td>\n",
       "      <td>2904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6779</th>\n",
       "      <td>visceral-evaluatesegmentation</td>\n",
       "      <td>3559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6780</th>\n",
       "      <td>vispr</td>\n",
       "      <td>18940</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6781</th>\n",
       "      <td>vkmz</td>\n",
       "      <td>490</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6782</th>\n",
       "      <td>vmatch</td>\n",
       "      <td>1319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6783</th>\n",
       "      <td>vnl</td>\n",
       "      <td>1563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6784</th>\n",
       "      <td>vphaser2</td>\n",
       "      <td>12731</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6785</th>\n",
       "      <td>vpolo</td>\n",
       "      <td>682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6786</th>\n",
       "      <td>vqsr_cnn</td>\n",
       "      <td>4579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6787</th>\n",
       "      <td>vsearch</td>\n",
       "      <td>162465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6788</th>\n",
       "      <td>vsnp</td>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6789</th>\n",
       "      <td>vt</td>\n",
       "      <td>36218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6790</th>\n",
       "      <td>vtools</td>\n",
       "      <td>2061</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6791</th>\n",
       "      <td>w4mclassfilter</td>\n",
       "      <td>16151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6792</th>\n",
       "      <td>w4mclstrpeakpics</td>\n",
       "      <td>2062</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6793</th>\n",
       "      <td>wade</td>\n",
       "      <td>460</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6794</th>\n",
       "      <td>watchdog-wms</td>\n",
       "      <td>8702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6795</th>\n",
       "      <td>wbuild</td>\n",
       "      <td>324</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6796</th>\n",
       "      <td>wdltool</td>\n",
       "      <td>6284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6797</th>\n",
       "      <td>weblogo</td>\n",
       "      <td>12163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6798</th>\n",
       "      <td>weeder</td>\n",
       "      <td>5984</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6799</th>\n",
       "      <td>wes-service-client</td>\n",
       "      <td>275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6800</th>\n",
       "      <td>wg-blimp</td>\n",
       "      <td>706</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6801</th>\n",
       "      <td>wgfast</td>\n",
       "      <td>1853</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6802</th>\n",
       "      <td>wgs2ncbi</td>\n",
       "      <td>2452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6803</th>\n",
       "      <td>wgs-assembler</td>\n",
       "      <td>1785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6804</th>\n",
       "      <td>wgsim</td>\n",
       "      <td>3606</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6805</th>\n",
       "      <td>wham</td>\n",
       "      <td>13584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6806</th>\n",
       "      <td>whatsgnu</td>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6807</th>\n",
       "      <td>whatshap</td>\n",
       "      <td>28093</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6808</th>\n",
       "      <td>wheezy.template</td>\n",
       "      <td>2098</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6809</th>\n",
       "      <td>wiggletools</td>\n",
       "      <td>7983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6810</th>\n",
       "      <td>wise2</td>\n",
       "      <td>1185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6811</th>\n",
       "      <td>wisecondorx</td>\n",
       "      <td>36190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6812</th>\n",
       "      <td>wisestork</td>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6813</th>\n",
       "      <td>wlogdate</td>\n",
       "      <td>61</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6814</th>\n",
       "      <td>womtool</td>\n",
       "      <td>3600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6815</th>\n",
       "      <td>workspace</td>\n",
       "      <td>2617</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6816</th>\n",
       "      <td>wtdbg</td>\n",
       "      <td>9156</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6817</th>\n",
       "      <td>wtforms-alchemy</td>\n",
       "      <td>5509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6818</th>\n",
       "      <td>wtforms-components</td>\n",
       "      <td>2454</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6819</th>\n",
       "      <td>wub</td>\n",
       "      <td>4442</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6820</th>\n",
       "      <td>xatlas</td>\n",
       "      <td>3482</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6821</th>\n",
       "      <td>xbioc</td>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6822</th>\n",
       "      <td>xcftools</td>\n",
       "      <td>1702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6823</th>\n",
       "      <td>xhmm</td>\n",
       "      <td>1743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6824</th>\n",
       "      <td>xmatchview</td>\n",
       "      <td>309</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6825</th>\n",
       "      <td>xmlbuilder</td>\n",
       "      <td>2402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6826</th>\n",
       "      <td>xmltramp2</td>\n",
       "      <td>14139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6827</th>\n",
       "      <td>xpclr</td>\n",
       "      <td>543</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6828</th>\n",
       "      <td>xpressplot</td>\n",
       "      <td>657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6829</th>\n",
       "      <td>xsd</td>\n",
       "      <td>9896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6830</th>\n",
       "      <td>xs-sim</td>\n",
       "      <td>884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6831</th>\n",
       "      <td>xtail</td>\n",
       "      <td>2022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6832</th>\n",
       "      <td>xtandem</td>\n",
       "      <td>10579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6833</th>\n",
       "      <td>xtermcolor</td>\n",
       "      <td>2127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6834</th>\n",
       "      <td>xunit-wrapper</td>\n",
       "      <td>2105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6835</th>\n",
       "      <td>xxmotif</td>\n",
       "      <td>4852</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6836</th>\n",
       "      <td>xyalign</td>\n",
       "      <td>5143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6837</th>\n",
       "      <td>yacrd</td>\n",
       "      <td>10903</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6838</th>\n",
       "      <td>yaggo</td>\n",
       "      <td>5213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6839</th>\n",
       "      <td>yaha</td>\n",
       "      <td>1911</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6840</th>\n",
       "      <td>yahmm</td>\n",
       "      <td>3199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6841</th>\n",
       "      <td>yak</td>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6842</th>\n",
       "      <td>yamda</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6843</th>\n",
       "      <td>yanagiba</td>\n",
       "      <td>3616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6844</th>\n",
       "      <td>yanc</td>\n",
       "      <td>2299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6845</th>\n",
       "      <td>yapc</td>\n",
       "      <td>44</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6846</th>\n",
       "      <td>yara</td>\n",
       "      <td>7599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6847</th>\n",
       "      <td>yass</td>\n",
       "      <td>1954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6848</th>\n",
       "      <td>ymp</td>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6849</th>\n",
       "      <td>zagros</td>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6850</th>\n",
       "      <td>zerone</td>\n",
       "      <td>1898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6851</th>\n",
       "      <td>zga</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6852</th>\n",
       "      <td>zifa</td>\n",
       "      <td>2052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6853</th>\n",
       "      <td>zorro</td>\n",
       "      <td>1305</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                 module  usage_2019_01\n",
       "0                                         2pg_cartesian             99\n",
       "1                                              a5-miseq            108\n",
       "2                                               abawaca             94\n",
       "3                                                abeona           1618\n",
       "4                                                 abra2            215\n",
       "5                                              abricate           1187\n",
       "6                                              abritamr              2\n",
       "7                                          abundancebin           3422\n",
       "8                                                 abyss           1633\n",
       "9                                                    ac             16\n",
       "10                                           ac-diamond             94\n",
       "11                                                 adam            305\n",
       "12                                       adapterremoval           4908\n",
       "13                              adapterremovalfixprefix            199\n",
       "14                                                addrg            284\n",
       "15                                           admixtools            291\n",
       "16                                            admixture            156\n",
       "17                                               advntr            794\n",
       "18                                               afplot            104\n",
       "19                                              afterqc            204\n",
       "20                                                 agat             38\n",
       "21                                           age-metasv            199\n",
       "22                                             agfusion            211\n",
       "23                                                  agg            185\n",
       "24                                                 airr             14\n",
       "25                                                  akt            193\n",
       "26                                           albatradis            484\n",
       "27                                                alder             98\n",
       "28                                                  ale            194\n",
       "29                                               alfred           1244\n",
       "30                                             align_it            231\n",
       "31                                        alignlib-lite            475\n",
       "32                                           alignstats            479\n",
       "33                                              allegro            189\n",
       "34                                             altamisa             20\n",
       "35                                                 amap             93\n",
       "36                                                 amas             10\n",
       "37                                                 amos            131\n",
       "38                                                amptk            402\n",
       "39                                               amused             96\n",
       "40                                                 andi            287\n",
       "41                                                angsd            516\n",
       "42                                          anise_basil            104\n",
       "43                                           anndata2ri             60\n",
       "44                                                  ant            213\n",
       "45                                              antarna             94\n",
       "46                                            antismash            452\n",
       "47                                                anvio           1327\n",
       "48                                        anvio-minimal            683\n",
       "49                                            any2fasta            385\n",
       "50                                                 apoc            100\n",
       "51                                               apollo             22\n",
       "52                               apt-probeset-summarize             93\n",
       "53                                               aquila              4\n",
       "54                                         aquila_stlfr             25\n",
       "55                                          aquila_umap              4\n",
       "56                                              aragorn           1500\n",
       "57                                              arb-bio            133\n",
       "58                                             arboreto             51\n",
       "59                                                 arcs            195\n",
       "60                                                 arem             99\n",
       "61                                                aria2            324\n",
       "62                                                ariba           1376\n",
       "63                                         array-as-vcf              4\n",
       "64                                               arriba            432\n",
       "65                                                  art            402\n",
       "66                                              artemis            364\n",
       "67                                          arvados-cli             92\n",
       "68                                   arvados-cwl-runner           2537\n",
       "69                                arvados-python-client           2510\n",
       "70                                                ascat            319\n",
       "71                                          asciigenome            838\n",
       "72                                              askocli            484\n",
       "73                                               asn2gb             95\n",
       "74                                               asqcan            189\n",
       "75                                        assemblerflow            284\n",
       "76                                        assembly-scan            104\n",
       "77                                       assembly-stats            250\n",
       "78                                         assemblytics             20\n",
       "79                                      assemblyutility            109\n",
       "80                                          astalavista            373\n",
       "81                                                 atac             95\n",
       "82                                               atactk            108\n",
       "83                                                ataqv            420\n",
       "84                                          athena_meta            193\n",
       "85                             atlas-metadata-validator              8\n",
       "86                                              atropos           1420\n",
       "87                                                augur             53\n",
       "88                                             augustus           1994\n",
       "89                                        autodock-vina            203\n",
       "90                                              autolog            115\n",
       "91                                             avro-cwl            107\n",
       "92                                         avro-python2            431\n",
       "93                                         avro-python3            416\n",
       "94                                               axiome             98\n",
       "95                                            azure-cli            104\n",
       "96                                           backspinpy            100\n",
       "97                                             bactopia             24\n",
       "98                                           baitfisher             95\n",
       "99                                            bam2fasta             39\n",
       "100                                           bam2fastx            250\n",
       "101                                             bambamc            163\n",
       "102                                          bamclipper             99\n",
       "103                                             bamhash            201\n",
       "104                                                bamm            104\n",
       "105                                             bamread            334\n",
       "106                                       bam-readcount           2597\n",
       "107                                            bamscale            210\n",
       "108                                            bamtools           4141\n",
       "109                                             bamutil            391\n",
       "110                                             bandage            224\n",
       "111                                              banner              6\n",
       "112                                    barcode_splitter             13\n",
       "113                                            barriers            285\n",
       "114                                             barrnap           1993\n",
       "115                                      barrnap-python            102\n",
       "116                                               basic            211\n",
       "117                                               batvi             94\n",
       "118                                             bax2bam            169\n",
       "119                                            bayescan            101\n",
       "120                                          bayestyper             95\n",
       "121                                               bazam              9\n",
       "122                                               bbknn            308\n",
       "123                                               bbmap           7568\n",
       "124                                               bcalm            226\n",
       "125                                           bcbio-gff            166\n",
       "126                                            bcbiogff            538\n",
       "127                                       bcbio_monitor            192\n",
       "128                                       bcbio-nextgen           4567\n",
       "129                                    bcbio-nextgen-vm            835\n",
       "130                                    bcbio-prioritize            665\n",
       "131                                        bcbio-rnaseq            465\n",
       "132                                     bcbio-variation            194\n",
       "133                              bcbio-variation-recall            592\n",
       "134                                           bcbreport            961\n",
       "135                                            bcftools          44152\n",
       "136                             bcftools-gtc2vcf-plugin             97\n",
       "137                             bcftools-snvphyl-plugin            382\n",
       "138                                   bcl2fastq-nextseq            225\n",
       "139                                               bcool            100\n",
       "140                                                bcov             93\n",
       "141                                             bctools            206\n",
       "142                                     bd2k-python-lib            709\n",
       "143                                              beagle            348\n",
       "144                                          beagle-lib            625\n",
       "145                                              beast2            224\n",
       "146                                               beast            722\n",
       "147                                              bedops           2257\n",
       "148                                            bedtools          24168\n",
       "149                                               behst           1116\n",
       "150                                       bellmans-gapc             95\n",
       "151                                       bel-resources             28\n",
       "152                                             berokka            203\n",
       "153                                               besst            298\n",
       "154                                                 bfc            104\n",
       "155                                             bftools              7\n",
       "156                                              bgreat             97\n",
       "157                                                 bgt            188\n",
       "158                                            biasaway              7\n",
       "159                                          bibliospec             93\n",
       "160                                        bicseq2-norm             95\n",
       "161                                         bicseq2-seg             96\n",
       "162                                               bigsi              4\n",
       "163                                             bindash             94\n",
       "164                                           binsanity            149\n",
       "165                             bio_assembly_refinement            250\n",
       "166                                              bioawk            500\n",
       "167                                           biobambam           1597\n",
       "168                                    biobasehttptools             16\n",
       "169                                      biobb_adapters              2\n",
       "170                                      biobb_analysis            115\n",
       "171                                     biobb_chemistry             27\n",
       "172                                        biobb_common            112\n",
       "173                                            biobb_io            102\n",
       "174                                            biobb_md            112\n",
       "175                                         biobb_model            100\n",
       "176                                           biobb_pmx             25\n",
       "177                            biobb_structure_checking             65\n",
       "178                             biobb_structure_manager             60\n",
       "179                               biobb_structure_utils              5\n",
       "180                                  biobb_wf_mutations              3\n",
       "181                                            bioblend           1965\n",
       "182                                      bioconda-utils           1024\n",
       "183                                     bioconductor-a4             10\n",
       "184                                 bioconductor-a4base            202\n",
       "185                              bioconductor-a4classif              7\n",
       "186                                 bioconductor-a4core            293\n",
       "187                              bioconductor-a4preproc            298\n",
       "188                            bioconductor-a4reporting            205\n",
       "189                                bioconductor-abadata            207\n",
       "190                          bioconductor-abaenrichment            478\n",
       "191                                bioconductor-abarray             10\n",
       "192                                 bioconductor-abseqr             12\n",
       "193                                 bioconductor-absseq            294\n",
       "194                                   bioconductor-acde            298\n",
       "195                                    bioconductor-ace              8\n",
       "196                                   bioconductor-acgh            576\n",
       "197                                   bioconductor-acme            574\n",
       "198                                bioconductor-adacgh2            188\n",
       "199                                   bioconductor-adam            186\n",
       "200                                bioconductor-adamgui              4\n",
       "201                               bioconductor-adaptest              8\n",
       "202                             bioconductor-adductdata              6\n",
       "203                           bioconductor-adductomicsr              6\n",
       "204                           bioconductor-adme16cod.db              7\n",
       "205                                bioconductor-adsplit            571\n",
       "206                               bioconductor-affixcan              8\n",
       "207                             bioconductor-affxparser            626\n",
       "208                                   bioconductor-affy           1354\n",
       "209                               bioconductor-affycomp            296\n",
       "210                         bioconductor-affycompatible            294\n",
       "211                           bioconductor-affycompdata              7\n",
       "212                             bioconductor-affycontam            288\n",
       "213                          bioconductor-affycoretools            198\n",
       "214                               bioconductor-affydata            296\n",
       "215                            bioconductor-affyexpress            289\n",
       "216                       bioconductor-affyhgu133a2expr             12\n",
       "217                        bioconductor-affyhgu133aexpr             10\n",
       "218                    bioconductor-affyhgu133plus2expr              9\n",
       "219                                bioconductor-affyilm            289\n",
       "220                                 bioconductor-affyio           1368\n",
       "221                              bioconductor-affylmgui              7\n",
       "222                bioconductor-affymetrixdatatestfiles             12\n",
       "223                        bioconductor-affymoe4302expr             10\n",
       "224                               bioconductor-affypara              7\n",
       "225                                bioconductor-affyplm            499\n",
       "226                           bioconductor-affyqcreport              9\n",
       "227                     bioconductor-affyrnadegradation            289\n",
       "228                                  bioconductor-agcdf              9\n",
       "229                                  bioconductor-ag.db             11\n",
       "230                                  bioconductor-agdex            290\n",
       "231                                  bioconductor-agilp            299\n",
       "232                            bioconductor-agimicrorna              8\n",
       "233                                bioconductor-agprobe              8\n",
       "234                            bioconductor-ahcytobands             11\n",
       "235                               bioconductor-ahensdbs              8\n",
       "236                                   bioconductor-aims            299\n",
       "237                                 bioconductor-airway             13\n",
       "238                                 bioconductor-aldex2            312\n",
       "239                               bioconductor-alevinqc              7\n",
       "240                                    bioconductor-all            504\n",
       "241                       bioconductor-allelicimbalance             10\n",
       "242                                 bioconductor-allmll              9\n",
       "243                              bioconductor-alphabeta              2\n",
       "244                                 bioconductor-alpine            384\n",
       "245                             bioconductor-alpinedata              8\n",
       "246                                   bioconductor-alps              2\n",
       "247                                 bioconductor-alsace             11\n",
       "248                             bioconductor-altcdfenvs            194\n",
       "249         bioconductor-alternativesplicingevents.hg19              8\n",
       "250         bioconductor-alternativesplicingevents.hg38             11\n",
       "251                               bioconductor-amaretto            191\n",
       "252                              bioconductor-amountain            573\n",
       "253                         bioconductor-ampaffyexample              8\n",
       "254                               bioconductor-amplican            196\n",
       "255                     bioconductor-analysispageserver            285\n",
       "256                                bioconductor-anaquin             10\n",
       "257                             bioconductor-aneufinder            287\n",
       "258                         bioconductor-aneufinderdata             12\n",
       "259                                    bioconductor-anf              8\n",
       "260                            bioconductor-animalcules              4\n",
       "261                                bioconductor-annaffy            300\n",
       "262                                 bioconductor-annmap            293\n",
       "263                               bioconductor-annotate           3317\n",
       "264                          bioconductor-annotationdbi           4575\n",
       "265                       bioconductor-annotationfilter            873\n",
       "266                        bioconductor-annotationforge            686\n",
       "267                        bioconductor-annotationfuncs            341\n",
       "268                          bioconductor-annotationhub            904\n",
       "269                      bioconductor-annotationhubdata            200\n",
       "270                        bioconductor-annotationtools            293\n",
       "271                               bioconductor-annotatr            175\n",
       "272                          bioconductor-anopheles.db0             12\n",
       "273                                  bioconductor-anota            294\n",
       "274                              bioconductor-anota2seq              6\n",
       "275                           bioconductor-antiprofiles            297\n",
       "276                       bioconductor-antiprofilesdata              8\n",
       "277                               bioconductor-apalyzer              2\n",
       "278                              bioconductor-apcomplex            197\n",
       "279                                 bioconductor-apeglm            685\n",
       "280                              bioconductor-appreci8r              7\n",
       "281                        bioconductor-arabidopsis.db0             11\n",
       "282                        bioconductor-aracne.networks             15\n",
       "283                            bioconductor-aroma.light            570\n",
       "284                           bioconductor-arrayexpress            283\n",
       "285                        bioconductor-arrayexpresshts            380\n",
       "286                             bioconductor-arraymvout              9\n",
       "287                           bioconductor-arrayquality            299\n",
       "288                    bioconductor-arrayqualitymetrics            203\n",
       "289                             bioconductor-arraytools            295\n",
       "290                                bioconductor-arraytv              7\n",
       "291                               bioconductor-arrmdata            203\n",
       "292                      bioconductor-arrmnormalization            205\n",
       "293                                  bioconductor-artms             10\n",
       "294                                  bioconductor-asafe            296\n",
       "295                                   bioconductor-aseb            584\n",
       "296                                 bioconductor-asgsca            301\n",
       "297                     bioconductor-ashkenazimsonchr21             11\n",
       "298                                  bioconductor-asics              8\n",
       "299                              bioconductor-asicsdata              9\n",
       "300                              bioconductor-aspediafi              2\n",
       "301                                  bioconductor-aspli            290\n",
       "302                              bioconductor-assessorf             10\n",
       "303                          bioconductor-assessorfdata             10\n",
       "304                                  bioconductor-asset             12\n",
       "305                                 bioconductor-assign              8\n",
       "306                              bioconductor-atacseqqc            296\n",
       "307                          bioconductor-ath1121501cdf              8\n",
       "308                          bioconductor-ath1121501.db              6\n",
       "309                        bioconductor-ath1121501probe              6\n",
       "310                                  bioconductor-atsnp            188\n",
       "311                                bioconductor-attract              9\n",
       "312                                 bioconductor-aucell             20\n",
       "313                              bioconductor-autotuner              2\n",
       "314                               bioconductor-awfisher             94\n",
       "315                               bioconductor-baalchip            195\n",
       "316                                    bioconductor-bac            567\n",
       "317                                  bioconductor-bacon            676\n",
       "318                                  bioconductor-bader            660\n",
       "319                        bioconductor-badregionfinder            292\n",
       "320                                   bioconductor-bags            289\n",
       "321                               bioconductor-ballgown            432\n",
       "322                             bioconductor-bamsignals            503\n",
       "323                                bioconductor-bandits            188\n",
       "324                                 bioconductor-banocc            305\n",
       "325                             bioconductor-barley1cdf              7\n",
       "326                           bioconductor-barley1probe              9\n",
       "327                             bioconductor-basecallqc            204\n",
       "328                             bioconductor-basespacer            295\n",
       "329                             bioconductor-basic4cseq            391\n",
       "330                                 bioconductor-basics            282\n",
       "331                          bioconductor-basicstarrseq             11\n",
       "332                              bioconductor-batchelor            259\n",
       "333                                bioconductor-batchqc            201\n",
       "334                         bioconductor-bayesknockdown              8\n",
       "335                              bioconductor-bayespeak            675\n",
       "336                                bioconductor-baynorm            282\n",
       "337                                 bioconductor-bayseq            325\n",
       "338                            bioconductor-bbcanalyzer            295\n",
       "339                             bioconductor-bcellviper             14\n",
       "340                                 bioconductor-bcrank            565\n",
       "341                                  bioconductor-bcseq            282\n",
       "342                           bioconductor-bdmmacorrect            286\n",
       "343                               bioconductor-beachmat           1158\n",
       "344                              bioconductor-beadarray            585\n",
       "345                   bioconductor-beadarrayexampledata              8\n",
       "346                           bioconductor-beadarraysnp            294\n",
       "347                      bioconductor-beadarrayusecases              7\n",
       "348                          bioconductor-beaddatapackr            572\n",
       "349                                bioconductor-bearscc              8\n",
       "350                                   bioconductor-beat            297\n",
       "351                                bioconductor-beclear            488\n",
       "352                   bioconductor-benchmarkfdrdata2019              2\n",
       "353                                  bioconductor-beta7             14\n",
       "354                                 bioconductor-bgafun            297\n",
       "355                                 bioconductor-bgeedb            294\n",
       "356                                  bioconductor-bgmix            560\n",
       "357                                    bioconductor-bgx            184\n",
       "358                                    bioconductor-bhc            567\n",
       "359                                 bioconductor-bicare            572\n",
       "360                                  bioconductor-bifet              8\n",
       "361                               bioconductor-bigmelon             22\n",
       "362                        bioconductor-bigmemoryextras             11\n",
       "363                                bioconductor-bigpint              6\n",
       "364                              bioconductor-bioassayr             11\n",
       "365                                bioconductor-biobase          12941\n",
       "366                               bioconductor-biobroom              9\n",
       "367                              bioconductor-biocancer             10\n",
       "368                        bioconductor-bioccasestudies              9\n",
       "369                              bioconductor-bioccheck              9\n",
       "370                          bioconductor-biocfilecache           1225\n",
       "371                           bioconductor-biocgenerics          13832\n",
       "372                              bioconductor-biocgraph            194\n",
       "373                          bioconductor-biocneighbors            643\n",
       "374                             bioconductor-bioconcotk              7\n",
       "375                                 bioconductor-biocor              9\n",
       "376                           bioconductor-biocparallel          12476\n",
       "377                           bioconductor-biocpkgtools              8\n",
       "378                           bioconductor-biocsingular            461\n",
       "379                            bioconductor-biocsklearn              8\n",
       "380                              bioconductor-biocstyle            919\n",
       "381                            bioconductor-biocversion            160\n",
       "382                              bioconductor-biocviews            601\n",
       "383                      bioconductor-biocworkflowtools            296\n",
       "384                                bioconductor-biodist            304\n",
       "385                                bioconductor-biomart           2686\n",
       "386                             bioconductor-biomformat            655\n",
       "387                                  bioconductor-biomm              4\n",
       "388                            bioconductor-biomvcclass            196\n",
       "389                              bioconductor-biomvrcns            279\n",
       "390                                 bioconductor-bionet            391\n",
       "391                             bioconductor-bionetstat              6\n",
       "392                                  bioconductor-bioqc            467\n",
       "393                            bioconductor-bioseqclass              6\n",
       "394                              bioconductor-biosigner            552\n",
       "395                             bioconductor-biostrings          11390\n",
       "396                                 bioconductor-biosvd            205\n",
       "397                                 bioconductor-biotip              2\n",
       "398                                bioconductor-biotmle              9\n",
       "399                            bioconductor-biotmledata             10\n",
       "400                             bioconductor-biovizbase           1204\n",
       "401                               bioconductor-birewire            488\n",
       "402                                  bioconductor-birta            557\n",
       "403                             bioconductor-biscuiteer              3\n",
       "404                         bioconductor-biscuiteerdata              2\n",
       "405                                  bioconductor-biseq            287\n",
       "406                                 bioconductor-bitseq            560\n",
       "407                            bioconductor-blacksheepr              2\n",
       "408                           bioconductor-bladderbatch            209\n",
       "409                                  bioconductor-blima            571\n",
       "410                       bioconductor-blimatestingdata              9\n",
       "411                                   bioconductor-blma              9\n",
       "412              bioconductor-bloodcancermultiomics2017              5\n",
       "413                                   bioconductor-bnbc            286\n",
       "414                             bioconductor-bodymaprat              5\n",
       "415                              bioconductor-bovinecdf              7\n",
       "416                             bioconductor-bovine.db0              8\n",
       "417                              bioconductor-bovine.db              7\n",
       "418                            bioconductor-bovineprobe              8\n",
       "419                                bioconductor-bprmeth            279\n",
       "420                                  bioconductor-brain            205\n",
       "421                        bioconductor-brainimagerdata             10\n",
       "422                             bioconductor-brainstars            203\n",
       "423                          bioconductor-branchpointer              9\n",
       "424                            bioconductor-breakpointr              9\n",
       "425                        bioconductor-breakpointrdata             10\n",
       "426                      bioconductor-breastcancermainz              8\n",
       "427                        bioconductor-breastcancernki             12\n",
       "428                   bioconductor-breastcancertransbig             10\n",
       "429                        bioconductor-breastcancerunt             10\n",
       "430                        bioconductor-breastcancerupp             10\n",
       "431                        bioconductor-breastcancervdx              9\n",
       "432                               bioconductor-brendadb             93\n",
       "433                               bioconductor-brgedata              8\n",
       "434                                 bioconductor-bridge            471\n",
       "435                              bioconductor-bridgedbr             13\n",
       "436                          bioconductor-bronchialil13             10\n",
       "437                             bioconductor-browserviz            199\n",
       "438                               bioconductor-bsgenome           1622\n",
       "439                bioconductor-bsgenome.alyrata.jgi.v1              9\n",
       "440   bioconductor-bsgenome.amellifera.beebase.assem...              9\n",
       "441       bioconductor-bsgenome.amellifera.ucsc.apimel2              8\n",
       "442   bioconductor-bsgenome.amellifera.ucsc.apimel2....              6\n",
       "443          bioconductor-bsgenome.aofficinalis.ncbi.v1              8\n",
       "444       bioconductor-bsgenome.athaliana.tair.04232008              8\n",
       "445          bioconductor-bsgenome.athaliana.tair.tair9             11\n",
       "446          bioconductor-bsgenome.btaurus.ucsc.bostau3              5\n",
       "447   bioconductor-bsgenome.btaurus.ucsc.bostau3.masked              6\n",
       "448          bioconductor-bsgenome.btaurus.ucsc.bostau4              7\n",
       "449   bioconductor-bsgenome.btaurus.ucsc.bostau4.masked              6\n",
       "450          bioconductor-bsgenome.btaurus.ucsc.bostau6              6\n",
       "451   bioconductor-bsgenome.btaurus.ucsc.bostau6.masked              6\n",
       "452          bioconductor-bsgenome.btaurus.ucsc.bostau8              5\n",
       "453            bioconductor-bsgenome.carietinum.ncbi.v1              6\n",
       "454            bioconductor-bsgenome.celegans.ucsc.ce10            108\n",
       "455            bioconductor-bsgenome.celegans.ucsc.ce11             10\n",
       "456             bioconductor-bsgenome.celegans.ucsc.ce2              6\n",
       "457             bioconductor-bsgenome.celegans.ucsc.ce6            106\n",
       "458      bioconductor-bsgenome.cfamiliaris.ucsc.canfam2              5\n",
       "459   bioconductor-bsgenome.cfamiliaris.ucsc.canfam2...              6\n",
       "460      bioconductor-bsgenome.cfamiliaris.ucsc.canfam3              8\n",
       "461   bioconductor-bsgenome.cfamiliaris.ucsc.canfam3...              6\n",
       "462         bioconductor-bsgenome.cjacchus.ucsc.caljac3              4\n",
       "463        bioconductor-bsgenome.dmelanogaster.ucsc.dm2              7\n",
       "464   bioconductor-bsgenome.dmelanogaster.ucsc.dm2.m...              6\n",
       "465        bioconductor-bsgenome.dmelanogaster.ucsc.dm3            114\n",
       "466   bioconductor-bsgenome.dmelanogaster.ucsc.dm3.m...              9\n",
       "467        bioconductor-bsgenome.dmelanogaster.ucsc.dm6              6\n",
       "468          bioconductor-bsgenome.drerio.ucsc.danrer10              6\n",
       "469           bioconductor-bsgenome.drerio.ucsc.danrer5              5\n",
       "470    bioconductor-bsgenome.drerio.ucsc.danrer5.masked              6\n",
       "471           bioconductor-bsgenome.drerio.ucsc.danrer6              5\n",
       "472    bioconductor-bsgenome.drerio.ucsc.danrer6.masked              6\n",
       "473           bioconductor-bsgenome.drerio.ucsc.danrer7            105\n",
       "474    bioconductor-bsgenome.drerio.ucsc.danrer7.masked              6\n",
       "475           bioconductor-bsgenome.ecoli.ncbi.20080805            115\n",
       "476       bioconductor-bsgenome.gaculeatus.ucsc.gasacu1              6\n",
       "477   bioconductor-bsgenome.gaculeatus.ucsc.gasacu1....              5\n",
       "478          bioconductor-bsgenome.ggallus.ucsc.galgal3              5\n",
       "479   bioconductor-bsgenome.ggallus.ucsc.galgal3.masked              5\n",
       "480          bioconductor-bsgenome.ggallus.ucsc.galgal4              5\n",
       "481   bioconductor-bsgenome.ggallus.ucsc.galgal4.masked              8\n",
       "482          bioconductor-bsgenome.ggallus.ucsc.galgal5              6\n",
       "483   bioconductor-bsgenome.hsapiens.1000genomes.hs37d5             10\n",
       "484          bioconductor-bsgenome.hsapiens.ncbi.grch38              6\n",
       "485            bioconductor-bsgenome.hsapiens.ucsc.hg17              6\n",
       "486     bioconductor-bsgenome.hsapiens.ucsc.hg17.masked              8\n",
       "487            bioconductor-bsgenome.hsapiens.ucsc.hg18            110\n",
       "488     bioconductor-bsgenome.hsapiens.ucsc.hg18.masked              6\n",
       "489            bioconductor-bsgenome.hsapiens.ucsc.hg19            355\n",
       "490     bioconductor-bsgenome.hsapiens.ucsc.hg19.masked              8\n",
       "491            bioconductor-bsgenome.hsapiens.ucsc.hg38            373\n",
       "492     bioconductor-bsgenome.hsapiens.ucsc.hg38.masked              7\n",
       "493       bioconductor-bsgenome.mdomestica.ucsc.mondom5              4\n",
       "494        bioconductor-bsgenome.mfascicularis.ncbi.5.0              7\n",
       "495            bioconductor-bsgenome.mfuro.ucsc.musfur1              7\n",
       "496         bioconductor-bsgenome.mmulatta.ucsc.rhemac2              8\n",
       "497   bioconductor-bsgenome.mmulatta.ucsc.rhemac2.ma...              6\n",
       "498         bioconductor-bsgenome.mmulatta.ucsc.rhemac3              5\n",
       "499   bioconductor-bsgenome.mmulatta.ucsc.rhemac3.ma...              7\n",
       "500         bioconductor-bsgenome.mmulatta.ucsc.rhemac8              7\n",
       "501           bioconductor-bsgenome.mmusculus.ucsc.mm10            173\n",
       "502    bioconductor-bsgenome.mmusculus.ucsc.mm10.masked              8\n",
       "503            bioconductor-bsgenome.mmusculus.ucsc.mm8              8\n",
       "504     bioconductor-bsgenome.mmusculus.ucsc.mm8.masked              5\n",
       "505            bioconductor-bsgenome.mmusculus.ucsc.mm9            152\n",
       "506     bioconductor-bsgenome.mmusculus.ucsc.mm9.masked              6\n",
       "507              bioconductor-bsgenome.osativa.msu.msu7              6\n",
       "508     bioconductor-bsgenome.ptroglodytes.ucsc.pantro2              6\n",
       "509   bioconductor-bsgenome.ptroglodytes.ucsc.pantro...              7\n",
       "510     bioconductor-bsgenome.ptroglodytes.ucsc.pantro3              7\n",
       "511   bioconductor-bsgenome.ptroglodytes.ucsc.pantro...             10\n",
       "512     bioconductor-bsgenome.ptroglodytes.ucsc.pantro5              6\n",
       "513     bioconductor-bsgenome.ptroglodytes.ucsc.pantro6              8\n",
       "514          bioconductor-bsgenome.rnorvegicus.ucsc.rn4              9\n",
       "515   bioconductor-bsgenome.rnorvegicus.ucsc.rn4.masked              9\n",
       "516          bioconductor-bsgenome.rnorvegicus.ucsc.rn5              9\n",
       "517   bioconductor-bsgenome.rnorvegicus.ucsc.rn5.masked              5\n",
       "518          bioconductor-bsgenome.rnorvegicus.ucsc.rn6              5\n",
       "519      bioconductor-bsgenome.scerevisiae.ucsc.saccer1              7\n",
       "520      bioconductor-bsgenome.scerevisiae.ucsc.saccer2              7\n",
       "521      bioconductor-bsgenome.scerevisiae.ucsc.saccer3              5\n",
       "522         bioconductor-bsgenome.sscrofa.ucsc.susscr11              8\n",
       "523          bioconductor-bsgenome.sscrofa.ucsc.susscr3              7\n",
       "524   bioconductor-bsgenome.sscrofa.ucsc.susscr3.masked              7\n",
       "525            bioconductor-bsgenome.tgondii.toxodb.7.0              9\n",
       "526         bioconductor-bsgenome.tguttata.ucsc.taegut1              6\n",
       "527   bioconductor-bsgenome.tguttata.ucsc.taegut1.ma...              6\n",
       "528         bioconductor-bsgenome.tguttata.ucsc.taegut2              8\n",
       "529      bioconductor-bsgenome.vvinifera.urgi.iggp12xv0              9\n",
       "530      bioconductor-bsgenome.vvinifera.urgi.iggp12xv2             10\n",
       "531         bioconductor-bsgenome.vvinifera.urgi.iggp8x              8\n",
       "532                                  bioconductor-bsseq            641\n",
       "533                              bioconductor-bsseqdata              9\n",
       "534                           bioconductor-bsubtiliscdf              7\n",
       "535                         bioconductor-bsubtilisprobe              9\n",
       "536                             bioconductor-bubbletree            587\n",
       "537                         bioconductor-bufferedmatrix            465\n",
       "538                  bioconductor-bufferedmatrixmethods            464\n",
       "539                                 bioconductor-bumhmm              9\n",
       "540                             bioconductor-bumphunter            811\n",
       "541                                    bioconductor-bus            468\n",
       "542                             bioconductor-buscorrect            276\n",
       "543                               bioconductor-busparse            104\n",
       "544                                   bioconductor-cafe              7\n",
       "545                             bioconductor-cagefightr              8\n",
       "546                                  bioconductor-cager            383\n",
       "547                                   bioconductor-calm              2\n",
       "548                                 bioconductor-camera            920\n",
       "549                                 bioconductor-camthc              8\n",
       "550                            bioconductor-cancerclass            276\n",
       "551                             bioconductor-cancerdata              8\n",
       "552                         bioconductor-cancerinsilico            558\n",
       "553                 bioconductor-cancermutationanalysis            556\n",
       "554                         bioconductor-cancersubtypes            199\n",
       "555                                   bioconductor-cand            293\n",
       "556                             bioconductor-canine2cdf              8\n",
       "557                             bioconductor-canine2.db              9\n",
       "558                           bioconductor-canine2probe              8\n",
       "559                              bioconductor-caninecdf              7\n",
       "560                             bioconductor-canine.db0              9\n",
       "561                              bioconductor-canine.db              8\n",
       "562                            bioconductor-canineprobe              7\n",
       "563                               bioconductor-caomicsv             12\n",
       "564                               bioconductor-cardinal            763\n",
       "565                      bioconductor-cardinalworkflows             10\n",
       "566                                 bioconductor-casper            554\n",
       "567                               bioconductor-catalyst             11\n",
       "568                               bioconductor-category            594\n",
       "569                        bioconductor-categorycompare              8\n",
       "570                                bioconductor-causalr            293\n",
       "571                                   bioconductor-cbaf              8\n",
       "572                                 bioconductor-ccdata             12\n",
       "573                                bioconductor-ccfindr            276\n",
       "574                                   bioconductor-ccl4              8\n",
       "575                                  bioconductor-ccmap             14\n",
       "576                              bioconductor-ccpromise              7\n",
       "577                                 bioconductor-ccrepe             11\n",
       "578                             bioconductor-cctutorial              8\n",
       "579                                bioconductor-celaref              9\n",
       "580                            bioconductor-celarefdata              9\n",
       "581                                  bioconductor-celda            184\n",
       "582                            bioconductor-celeganscdf              8\n",
       "583                            bioconductor-celegans.db              8\n",
       "584                          bioconductor-celegansprobe              8\n",
       "585                              bioconductor-cellbaser              7\n",
       "586                              bioconductor-cellbench              4\n",
       "587                               bioconductor-cellhts2            301\n",
       "588                                bioconductor-cellity              9\n",
       "589                             bioconductor-cellmapper            284\n",
       "590                         bioconductor-cellmapperdata              8\n",
       "591                               bioconductor-cellmixs              4\n",
       "592                              bioconductor-cellnoptr            646\n",
       "593                              bioconductor-cellscape             12\n",
       "594                              bioconductor-cellscore              7\n",
       "595                             bioconductor-celltrails              9\n",
       "596                               bioconductor-celltree              8\n",
       "597                               bioconductor-cemitool             11\n",
       "598                                 bioconductor-ceu1kg              8\n",
       "599                                bioconductor-ceu1kgv              8\n",
       "600                                 bioconductor-ceuhm3              8\n",
       "601                                  bioconductor-cexor            385\n",
       "602                                bioconductor-cfassay            293\n",
       "603                                 bioconductor-cgdv17              6\n",
       "604                                   bioconductor-cgen            277\n",
       "605                                bioconductor-cghbase            422\n",
       "606                                bioconductor-cghcall            511\n",
       "607                                 bioconductor-cghmcr            288\n",
       "608                          bioconductor-cghnormaliter            196\n",
       "609                             bioconductor-cghregions            293\n",
       "610                                  bioconductor-champ             13\n",
       "611                              bioconductor-champdata             16\n",
       "612                                 bioconductor-charge              8\n",
       "613                          bioconductor-chemminedrugs              8\n",
       "614                              bioconductor-chemminer            573\n",
       "615                                 bioconductor-chetah              4\n",
       "616                                   bioconductor-chic             11\n",
       "617                                bioconductor-chicago            389\n",
       "618                              bioconductor-chic.data             11\n",
       "619                             bioconductor-chickencdf             11\n",
       "620                            bioconductor-chicken.db0              6\n",
       "621                             bioconductor-chicken.db             13\n",
       "622                           bioconductor-chickenprobe             10\n",
       "623                                bioconductor-chimera            475\n",
       "624                             bioconductor-chimeraviz            198\n",
       "625                              bioconductor-chimp.db0              7\n",
       "626                    bioconductor-chimphumanbraindata              8\n",
       "627                           bioconductor-chipanalyser              9\n",
       "628                               bioconductor-chipcomp            652\n",
       "629                             bioconductor-chipenrich              9\n",
       "630                        bioconductor-chipenrich.data              7\n",
       "631                            bioconductor-chipexoqual              8\n",
       "632                     bioconductor-chipexoqualexample             10\n",
       "633                           bioconductor-chippeakanno            320\n",
       "634                                 bioconductor-chipqc            398\n",
       "635                             bioconductor-chipseeker            632\n",
       "636                                bioconductor-chipseq            697\n",
       "637                          bioconductor-chipseqdbdata              5\n",
       "638                               bioconductor-chipseqr            277\n",
       "639                           bioconductor-chipseqspike              9\n",
       "640                                bioconductor-chipsim            201\n",
       "641                             bioconductor-chipxpress            193\n",
       "642                         bioconductor-chipxpressdata            201\n",
       "643                             bioconductor-chopsticks            468\n",
       "644                              bioconductor-chromdraw            556\n",
       "645                           bioconductor-chromheatmap            200\n",
       "646                              bioconductor-chromplot            297\n",
       "647                              bioconductor-chromstar            567\n",
       "648                          bioconductor-chromstardata            204\n",
       "649                            bioconductor-chromswitch             10\n",
       "650                               bioconductor-chromvar            506\n",
       "651                                bioconductor-chronos            287\n",
       "652                                 bioconductor-cicero             19\n",
       "653                                 bioconductor-cindex            203\n",
       "654                        bioconductor-circrnaprofiler              2\n",
       "655                                bioconductor-cispath            463\n",
       "656                              bioconductor-citruscdf              8\n",
       "657                            bioconductor-citrusprobe              7\n",
       "658               bioconductor-clariomdhumanprobeset.db              7\n",
       "659      bioconductor-clariomdhumantranscriptcluster.db              8\n",
       "660    bioconductor-clariomshumanhttranscriptcluster.db              7\n",
       "661      bioconductor-clariomshumantranscriptcluster.db              6\n",
       "662    bioconductor-clariomsmousehttranscriptcluster.db              8\n",
       "663      bioconductor-clariomsmousetranscriptcluster.db              6\n",
       "664      bioconductor-clariomsrathttranscriptcluster.db              8\n",
       "665        bioconductor-clariomsrattranscriptcluster.db              6\n",
       "666                              bioconductor-classifyr            199\n",
       "667                           bioconductor-cleanupdtseq            197\n",
       "668                                bioconductor-cleaver            297\n",
       "669                                bioconductor-clippda              7\n",
       "670                                bioconductor-clipper            286\n",
       "671                                    bioconductor-cll             20\n",
       "672                         bioconductor-cllmethylation              7\n",
       "673                                bioconductor-clomial            293\n",
       "674                              bioconductor-clonality            292\n",
       "675                             bioconductor-clonotyper            200\n",
       "676                                   bioconductor-clst            206\n",
       "677                              bioconductor-clstutils            204\n",
       "678                                bioconductor-clumsid              4\n",
       "679                            bioconductor-clumsiddata              4\n",
       "680                              bioconductor-clustcomp            205\n",
       "681                      bioconductor-clusterexperiment            363\n",
       "682                           bioconductor-clusterjudge             11\n",
       "683                        bioconductor-clusterprofiler            905\n",
       "684                             bioconductor-clusterseq              6\n",
       "685                    bioconductor-clustersignificance            203\n",
       "686                            bioconductor-clusterstab            201\n",
       "687                                    bioconductor-cma            228\n",
       "688                                   bioconductor-cmap            101\n",
       "689                              bioconductor-cmap2data            197\n",
       "690                                bioconductor-cnanorm            554\n",
       "691                                   bioconductor-cner            681\n",
       "692                               bioconductor-cn.farms            472\n",
       "693                                bioconductor-cn.mops            862\n",
       "694                                 bioconductor-cnordt            369\n",
       "695                             bioconductor-cnorfeeder              7\n",
       "696                              bioconductor-cnorfuzzy            465\n",
       "697                                bioconductor-cnorode            469\n",
       "698                                bioconductor-cntools            574\n",
       "699                              bioconductor-cnvfilter              2\n",
       "700                                 bioconductor-cnvgsa            196\n",
       "701                             bioconductor-cnvgsadata              7\n",
       "702                           bioconductor-cnvpanelizer            197\n",
       "703                              bioconductor-cnvranger              4\n",
       "704                                 bioconductor-cnvrd2              8\n",
       "705                               bioconductor-cnvtools            462\n",
       "706                                bioconductor-cobindr             10\n",
       "707                            bioconductor-cocitestats              9\n",
       "708                                  bioconductor-cocoa              9\n",
       "709                               bioconductor-codelink            200\n",
       "710                                  bioconductor-codex            300\n",
       "711                                bioconductor-coexnet              7\n",
       "712                                 bioconductor-cogaps            276\n",
       "713                                 bioconductor-cogena            290\n",
       "714                                  bioconductor-cogps            203\n",
       "715                                 bioconductor-cohcap            474\n",
       "716                             bioconductor-cohcapanno            202\n",
       "717                                   bioconductor-cola            185\n",
       "718                                bioconductor-colonca              9\n",
       "719                                  bioconductor-comet              5\n",
       "720                             bioconductor-compartmap              7\n",
       "721                                bioconductor-compass            674\n",
       "722                              bioconductor-compcoder             14\n",
       "723                           bioconductor-compepitools            291\n",
       "724                                 bioconductor-compgo              8\n",
       "725                         bioconductor-complexheatmap            759\n",
       "726                                bioconductor-confess              8\n",
       "727                            bioconductor-confessdata             10\n",
       "728                        bioconductor-connectivitymap             11\n",
       "729                              bioconductor-consensus             11\n",
       "730                   bioconductor-consensusclusterplus            488\n",
       "731                            bioconductor-consensusde              7\n",
       "732                            bioconductor-consensusov            200\n",
       "733                        bioconductor-consensusseeker            203\n",
       "734                              bioconductor-contibait            471\n",
       "735                                bioconductor-conumee              5\n",
       "736                                bioconductor-convert            199\n",
       "737                                   bioconductor-copa            284\n",
       "738              bioconductor-copdsexualdimorphism.data             10\n",
       "739                             bioconductor-copyhelper            210\n",
       "740                         bioconductor-copyneutralima              9\n",
       "741                             bioconductor-copynumber            491\n",
       "742                        bioconductor-copynumberplots              6\n",
       "743                             bioconductor-copywriter            203\n",
       "744                                 bioconductor-cordon              6\n",
       "745                              bioconductor-coregflux              4\n",
       "746                               bioconductor-coregnet            277\n",
       "747                               bioconductor-cormotif            196\n",
       "748                                 bioconductor-cormut             12\n",
       "749                                 bioconductor-correp            202\n",
       "750                                  bioconductor-coseq              9\n",
       "751                              bioconductor-cosmic.67              7\n",
       "752                                 bioconductor-cosmiq            470\n",
       "753                                 bioconductor-cosnet            286\n",
       "754                              bioconductor-cottoncdf              7\n",
       "755                            bioconductor-cottonprobe              8\n",
       "756                             bioconductor-countclust            191\n",
       "757                             bioconductor-countsimqc              7\n",
       "758                                  bioconductor-coveb             11\n",
       "759                           bioconductor-coverageview              9\n",
       "760                                 bioconductor-covrna            199\n",
       "761                                 bioconductor-cpvsnp            196\n",
       "762                                    bioconductor-cqn            300\n",
       "763                                 bioconductor-crcl18             11\n",
       "764                                bioconductor-crimage            195\n",
       "765                             bioconductor-crisprseek            240\n",
       "766                         bioconductor-crisprseekplus            195\n",
       "767                         bioconductor-crisprvariants            198\n",
       "768                                  bioconductor-crlmm            471\n",
       "769                               bioconductor-crossicc              2\n",
       "770                              bioconductor-crossmeta              9\n",
       "771                                   bioconductor-csar            277\n",
       "772                                   bioconductor-csaw           1092\n",
       "773                                   bioconductor-cssp            285\n",
       "774                                    bioconductor-ctc            223\n",
       "775                                  bioconductor-ctrap              8\n",
       "776                                  bioconductor-ctsge            206\n",
       "777                             bioconductor-cummerbund            621\n",
       "778                       bioconductor-curatedadipochip              6\n",
       "779                        bioconductor-curatedadiporna              4\n",
       "780                     bioconductor-curatedbladderdata              8\n",
       "781                      bioconductor-curatedbreastdata              8\n",
       "782                         bioconductor-curatedcrcdata              8\n",
       "783                 bioconductor-curatedmetagenomicdata            194\n",
       "784                     bioconductor-curatedovariandata            288\n",
       "785                        bioconductor-curatedtcgadata             10\n",
       "786                            bioconductor-customprodb            119\n",
       "787                                  bioconductor-cycle              8\n",
       "788                                  bioconductor-cydar            276\n",
       "789                              bioconductor-cyp450cdf              8\n",
       "790                                 bioconductor-cytodx             12\n",
       "791                               bioconductor-cytofast              4\n",
       "792                                bioconductor-cytolib            239\n",
       "793                                 bioconductor-cytoml            216\n",
       "794                                  bioconductor-dada2           3751\n",
       "795                                bioconductor-daglogo              7\n",
       "796                                   bioconductor-dama            203\n",
       "797                               bioconductor-damirseq              8\n",
       "798                                  bioconductor-dapar             11\n",
       "799                              bioconductor-dapardata             10\n",
       "800                                   bioconductor-dart             12\n",
       "801                            bioconductor-davidtiling             10\n",
       "802                                 bioconductor-dbchip            200\n",
       "803                                  bioconductor-dcanr              4\n",
       "804                                  bioconductor-dcgsa            194\n",
       "805                               bioconductor-dchiprep            192\n",
       "806                                   bioconductor-ddct            194\n",
       "807                             bioconductor-ddpcrclust              6\n",
       "808                                 bioconductor-debcam              1\n",
       "809                              bioconductor-debrowser              5\n",
       "810                               bioconductor-decipher            510\n",
       "811                                   bioconductor-deco              2\n",
       "812                       bioconductor-decomplexdisease            276\n",
       "813                        bioconductor-decomptumor2sig              4\n",
       "814                            bioconductor-deconrnaseq              9\n",
       "815                               bioconductor-decontam             21\n",
       "816                              bioconductor-deepbluer            195\n",
       "817                                bioconductor-deepsnv            652\n",
       "818                              bioconductor-deformats            259\n",
       "819                                bioconductor-degraph            193\n",
       "820                              bioconductor-degreport            632\n",
       "821                                 bioconductor-degseq            571\n",
       "822                           bioconductor-delayedarray          11131\n",
       "823                       bioconductor-delayeddataframe              2\n",
       "824                     bioconductor-delayedmatrixstats            762\n",
       "825                          bioconductor-deltacapturec              1\n",
       "826                              bioconductor-deltagseg              7\n",
       "827                                 bioconductor-demand            199\n",
       "828                                 bioconductor-demixt            184\n",
       "829                                    bioconductor-dep              9\n",
       "830                               bioconductor-depecher            188\n",
       "831                                 bioconductor-depmap              2\n",
       "832                                  bioconductor-deqms            202\n",
       "833                              bioconductor-derfinder            287\n",
       "834                          bioconductor-derfinderdata              8\n",
       "835                        bioconductor-derfinderhelper            378\n",
       "836                          bioconductor-derfinderplot            191\n",
       "837                                bioconductor-descan2            276\n",
       "838                                  bioconductor-deseq           1082\n",
       "839                                 bioconductor-deseq2           3163\n",
       "840                               bioconductor-desingle             12\n",
       "841                            bioconductor-desousa2013              7\n",
       "842                                bioconductor-destiny            307\n",
       "843                                 bioconductor-desubs              7\n",
       "844                                 bioconductor-dewseq              1\n",
       "845                                 bioconductor-dexseq            763\n",
       "846                                  bioconductor-dexus            574\n",
       "847                                    bioconductor-dfp            196\n",
       "848                               bioconductor-diffbind           1086\n",
       "849                              bioconductor-diffcoexp              4\n",
       "850                                bioconductor-diffcyt              7\n",
       "851                       bioconductor-diffgeneanalysis            198\n",
       "852                                bioconductor-diffhic            285\n",
       "853                               bioconductor-difflogo              8\n",
       "854                               bioconductor-diffloop            207\n",
       "855                           bioconductor-diffloopdata              6\n",
       "856                             bioconductor-diffustats            463\n",
       "857                                 bioconductor-diggit              9\n",
       "858                             bioconductor-diggitdata              6\n",
       "859                               bioconductor-director            199\n",
       "860                   bioconductor-dirichletmultinomial            786\n",
       "861                             bioconductor-discordant            368\n",
       "862                            bioconductor-discorhythm              1\n",
       "863                             bioconductor-divergence              2\n",
       "864                                    bioconductor-dks              7\n",
       "865                                  bioconductor-dlbcl              6\n",
       "866                                  bioconductor-dmcfb              1\n",
       "867                                 bioconductor-dmchmm              5\n",
       "868                                bioconductor-dmelsgi              7\n",
       "869                              bioconductor-dmrcaller            287\n",
       "870                                bioconductor-dmrcate            205\n",
       "871                            bioconductor-dmrcatedata            200\n",
       "872                            bioconductor-dmrforpairs              5\n",
       "873                                bioconductor-dmrscan              8\n",
       "874                                 bioconductor-dmrseq            172\n",
       "875                bioconductor-dnabarcodecompatibility              2\n",
       "876                            bioconductor-dnabarcodes            463\n",
       "877                                bioconductor-dnacopy           1390\n",
       "878                              bioconductor-dnashaper            563\n",
       "879                                  bioconductor-do.db            405\n",
       "880                           bioconductor-dominoeffect              8\n",
       "881                           bioconductor-donapllp2013              5\n",
       "882                            bioconductor-doppelgangr            289\n",
       "883                               bioconductor-doscheda              6\n",
       "884                                   bioconductor-dose           1033\n",
       "885                                  bioconductor-doser              2\n",
       "886                           bioconductor-drawproteins              8\n",
       "887                                 bioconductor-dream4              8\n",
       "888                             bioconductor-dresscheck              4\n",
       "889                                bioconductor-drimseq            245\n",
       "890                              bioconductor-drivernet            199\n",
       "891                           bioconductor-dropletutils            517\n",
       "892                         bioconductor-drosgenome1cdf              5\n",
       "893                         bioconductor-drosgenome1.db              5\n",
       "894                       bioconductor-drosgenome1probe              4\n",
       "895                         bioconductor-drosophila2cdf              9\n",
       "896                         bioconductor-drosophila2.db              9\n",
       "897                       bioconductor-drosophila2probe              9\n",
       "898                          bioconductor-drugvsdisease            195\n",
       "899                      bioconductor-drugvsdiseasedata            198\n",
       "900                                  bioconductor-dsqtl              7\n",
       "901                                    bioconductor-dss            485\n",
       "902                                    bioconductor-dta            197\n",
       "903                                 bioconductor-dualks              7\n",
       "904                      bioconductor-duoclustering2018              6\n",
       "905                             bioconductor-dupchecker            200\n",
       "906                               bioconductor-dupradar            484\n",
       "907                                bioconductor-dvddata              8\n",
       "908                                bioconductor-dyebias            197\n",
       "909                        bioconductor-dyebiasexamples              6\n",
       "910                                 bioconductor-dyndoc            213\n",
       "911                               bioconductor-easyqpcr              2\n",
       "912                             bioconductor-easyrnaseq            198\n",
       "913                       bioconductor-eatonetalchipseq              7\n",
       "914                               bioconductor-ebarrays            554\n",
       "915                            bioconductor-ebcoexpress            276\n",
       "916                                bioconductor-ebimage            862\n",
       "917                                  bioconductor-ebsea            199\n",
       "918                                  bioconductor-ebseq            792\n",
       "919                               bioconductor-ebseqhmm             10\n",
       "920                              bioconductor-ecoli2cdf              6\n",
       "921                              bioconductor-ecoli2.db              6\n",
       "922                            bioconductor-ecoli2probe              6\n",
       "923                           bioconductor-ecoliasv2cdf              5\n",
       "924                         bioconductor-ecoliasv2probe              5\n",
       "925                               bioconductor-ecolicdf              5\n",
       "926                           bioconductor-ecolik12.db0              5\n",
       "927                           bioconductor-ecolileucine              7\n",
       "928                             bioconductor-ecoliprobe              6\n",
       "929                         bioconductor-ecolisakai.db0              6\n",
       "930                                bioconductor-ecolitk              7\n",
       "931                                 bioconductor-edaseq            422\n",
       "932                                   bioconductor-edda            556\n",
       "933                                   bioconductor-edge            279\n",
       "934                                  bioconductor-edger           3070\n",
       "935                                   bioconductor-eegc              6\n",
       "936                                   bioconductor-egad              5\n",
       "937                                  bioconductor-egsea            199\n",
       "938                              bioconductor-egseadata            206\n",
       "939                                    bioconductor-eir            276\n",
       "940                                   bioconductor-eisa              6\n",
       "941                                  bioconductor-elbow             11\n",
       "942                                  bioconductor-elmer            108\n",
       "943                             bioconductor-elmer.data            104\n",
       "944                               bioconductor-emdomics             10\n",
       "945                  bioconductor-empiricalbrownsmethod            197\n",
       "946                          bioconductor-encodexplorer            200\n",
       "947                      bioconductor-encodexplorerdata              2\n",
       "948                        bioconductor-enhancedvolcano             75\n",
       "949                                  bioconductor-enmix              7\n",
       "950                        bioconductor-enrichedheatmap            279\n",
       "951                      bioconductor-enrichmentbrowser             99\n",
       "952                             bioconductor-enrichplot            367\n",
       "953                               bioconductor-enrichtf              2\n",
       "954                     bioconductor-ensdb.hsapiens.v75            222\n",
       "955                     bioconductor-ensdb.hsapiens.v79              9\n",
       "956                     bioconductor-ensdb.hsapiens.v86             38\n",
       "957                    bioconductor-ensdb.mmusculus.v75              6\n",
       "958                    bioconductor-ensdb.mmusculus.v79              9\n",
       "959                  bioconductor-ensdb.rnorvegicus.v75              9\n",
       "960                  bioconductor-ensdb.rnorvegicus.v79              5\n",
       "961                              bioconductor-ensembldb           1146\n",
       "962                             bioconductor-ensemblvep            218\n",
       "963                          bioconductor-envisionquery              7\n",
       "964                                bioconductor-epidish             10\n",
       "965                             bioconductor-epigenomix            193\n",
       "966                                 bioconductor-epihet              6\n",
       "967                                bioconductor-epivizr            193\n",
       "968                           bioconductor-epivizrchart              7\n",
       "969                            bioconductor-epivizrdata            196\n",
       "970                          bioconductor-epivizrserver            201\n",
       "971                      bioconductor-epivizrstandalone            195\n",
       "972                          bioconductor-erccdashboard             17\n",
       "973                                   bioconductor-erma              5\n",
       "974                                  bioconductor-erssa              6\n",
       "975                                 bioconductor-esatac            380\n",
       "976                                bioconductor-esetvis            192\n",
       "977                               bioconductor-estrogen              8\n",
       "978                                bioconductor-etec16s              5\n",
       "979                             bioconductor-eudysbiome            204\n",
       "980                               bioconductor-eupathdb              7\n",
       "981                              bioconductor-evaluomer              2\n",
       "982                           bioconductor-eventpointer              7\n",
       "983                              bioconductor-excluster              5\n",
       "984                                 bioconductor-eximir            197\n",
       "985                              bioconductor-exomecopy            671\n",
       "986                          bioconductor-experimenthub            208\n",
       "987                      bioconductor-experimenthubdata              6\n",
       "988                              bioconductor-explorase              3\n",
       "989                        bioconductor-expressionatlas              6\n",
       "990                         bioconductor-expressionview            279\n",
       "991                                 bioconductor-faahko            102\n",
       "992                                  bioconductor-fabia            470\n",
       "993                              bioconductor-fabiadata             10\n",
       "994                              bioconductor-facsdorit              6\n",
       "995                             bioconductor-factdesign              7\n",
       "996                                 bioconductor-famagg              7\n",
       "997                        bioconductor-fantom3and4cage              8\n",
       "998                                  bioconductor-farms            289\n",
       "999                  bioconductor-fastliquidassociation              8\n",
       "1000                          bioconductor-fastqcleaner            277\n",
       "1001                               bioconductor-fastseg            685\n",
       "1002                                  bioconductor-fcbf              9\n",
       "1003                                 bioconductor-fccac            194\n",
       "1004                                   bioconductor-fci             10\n",
       "1005                                 bioconductor-fcoex              2\n",
       "1006                                bioconductor-fcscan              1\n",
       "1007            bioconductor-fdb.fantom4.promoters.hg19              4\n",
       "1008          bioconductor-fdb.infiniummethylation.hg18              6\n",
       "1009          bioconductor-fdb.infiniummethylation.hg19            136\n",
       "1010            bioconductor-fdb.ucsc.snp135common.hg19              3\n",
       "1011            bioconductor-fdb.ucsc.snp137common.hg19              5\n",
       "1012                        bioconductor-fdb.ucsc.trnas              7\n",
       "1013                                bioconductor-fdrame            277\n",
       "1014                                 bioconductor-fella              8\n",
       "1015                                   bioconductor-fem              9\n",
       "1016                                  bioconductor-ffpe            196\n",
       "1017                       bioconductor-ffpeexampledata              6\n",
       "1018                                 bioconductor-fgnet              8\n",
       "1019                                 bioconductor-fgsea            899\n",
       "1020                             bioconductor-fibroeset              8\n",
       "1021                         bioconductor-findmyfriends            475\n",
       "1022                                   bioconductor-fis            192\n",
       "1023                            bioconductor-fishalyser            300\n",
       "1024                              bioconductor-fishpond            106\n",
       "1025                               bioconductor-fission              8\n",
       "1026                     bioconductor-fitcons.ucsc.hg19              8\n",
       "1027                                bioconductor-fithic            580\n",
       "1028                                bioconductor-flagme            465\n",
       "1029                         bioconductor-fletcher2013a              5\n",
       "1030                         bioconductor-fletcher2013b              5\n",
       "1031                                bioconductor-flowai            560\n",
       "1032                             bioconductor-flowbeads            198\n",
       "1033                               bioconductor-flowbin              6\n",
       "1034                            bioconductor-flowcatchr              5\n",
       "1035                              bioconductor-flowchic              8\n",
       "1036                                bioconductor-flowcl            475\n",
       "1037                             bioconductor-flowclean            196\n",
       "1038                             bioconductor-flowclust            301\n",
       "1039                              bioconductor-flowcore            879\n",
       "1040                             bioconductor-flowcybar              8\n",
       "1041                           bioconductor-flowdensity            383\n",
       "1042                               bioconductor-flowfit              7\n",
       "1043                    bioconductor-flowfitexampledata              7\n",
       "1044                                bioconductor-flowfp            658\n",
       "1045                               bioconductor-flowmap            189\n",
       "1046                             bioconductor-flowmatch            473\n",
       "1047                             bioconductor-flowmeans              7\n",
       "1048                             bioconductor-flowmerge              7\n",
       "1049                             bioconductor-flowpeaks            591\n",
       "1050                            bioconductor-flowploidy            197\n",
       "1051                        bioconductor-flowploidydata              7\n",
       "1052                             bioconductor-flowplots              8\n",
       "1053                       bioconductor-flowrepositoryr            290\n",
       "1054                               bioconductor-flowsom            687\n",
       "1055                 bioconductor-flowsorted.blood.450k            193\n",
       "1056                 bioconductor-flowsorted.blood.epic              9\n",
       "1057             bioconductor-flowsorted.cordblood.450k              5\n",
       "1058     bioconductor-flowsorted.cordbloodcombined.450k              2\n",
       "1059       bioconductor-flowsorted.cordbloodnorway.450k              4\n",
       "1060                 bioconductor-flowsorted.dlpfc.450k              7\n",
       "1061                             bioconductor-flowspecs              1\n",
       "1062                               bioconductor-flowspy             92\n",
       "1063                             bioconductor-flowstats             33\n",
       "1064                              bioconductor-flowtime              5\n",
       "1065                             bioconductor-flowtrans              7\n",
       "1066                              bioconductor-flowtype            280\n",
       "1067                             bioconductor-flowutils            292\n",
       "1068                               bioconductor-flowviz            411\n",
       "1069                                bioconductor-flowvs              8\n",
       "1070                         bioconductor-flowworkspace            676\n",
       "1071                     bioconductor-flowworkspacedata             31\n",
       "1072                               bioconductor-fly.db0              6\n",
       "1073                                 bioconductor-fmcsr            284\n",
       "1074                             bioconductor-focalcall            196\n",
       "1075                                bioconductor-foldgo              6\n",
       "1076                              bioconductor-fourcseq            386\n",
       "1077                          bioconductor-frgepistasis             10\n",
       "1078                                  bioconductor-frma            193\n",
       "1079                       bioconductor-frmaexampledata              7\n",
       "1080                             bioconductor-frmatools            193\n",
       "1081                               bioconductor-funchip            472\n",
       "1082                              bioconductor-funcisnp              5\n",
       "1083                         bioconductor-funcisnp.data              6\n",
       "1084                            bioconductor-funtoonorm              5\n",
       "1085                             bioconductor-furrowseg              6\n",
       "1086                           bioconductor-ga4ghclient              7\n",
       "1087                            bioconductor-ga4ghshiny              5\n",
       "1088                                  bioconductor-gaga            662\n",
       "1089                                  bioconductor-gage            401\n",
       "1090                              bioconductor-gagedata            489\n",
       "1091                                bioconductor-gaggle              7\n",
       "1092                                  bioconductor-gaia              8\n",
       "1093                                bioconductor-gapgom              2\n",
       "1094                          bioconductor-gaprediction            193\n",
       "1095                              bioconductor-garfield            289\n",
       "1096                                  bioconductor-gars              6\n",
       "1097                              bioconductor-gaschyhs              7\n",
       "1098                            bioconductor-gatefinder              7\n",
       "1099                          bioconductor-gatingmldata              8\n",
       "1100                                 bioconductor-gcapc              6\n",
       "1101                               bioconductor-gcatest            483\n",
       "1102                                 bioconductor-gcmap              6\n",
       "1103                              bioconductor-gcmapweb              6\n",
       "1104                          bioconductor-gcrisprtools            196\n",
       "1105                                 bioconductor-gcrma            588\n",
       "1106                           bioconductor-gcspikelite            298\n",
       "1107                              bioconductor-gcsscore              1\n",
       "1108                           bioconductor-gdcrnatools              6\n",
       "1109                              bioconductor-gdsarray              9\n",
       "1110                                bioconductor-gdsfmt            596\n",
       "1111                                 bioconductor-geecc            368\n",
       "1112                                   bioconductor-gem              6\n",
       "1113                                bioconductor-gemini              1\n",
       "1114                              bioconductor-genarise             10\n",
       "1115                              bioconductor-genbankr            201\n",
       "1116                            bioconductor-geneaccord              5\n",
       "1117                           bioconductor-geneanswers              4\n",
       "1118                       bioconductor-geneattribution            199\n",
       "1119                             bioconductor-genebreak            378\n",
       "1120                       bioconductor-geneclassifiers              5\n",
       "1121               bioconductor-geneexpressionsignature              6\n",
       "1122                            bioconductor-genefilter           3648\n",
       "1123                                bioconductor-genefu            201\n",
       "1124                                bioconductor-genega            201\n",
       "1125                         bioconductor-genegeneinter            187\n",
       "1126                       bioconductor-genelendatabase            666\n",
       "1127                              bioconductor-genemeta            197\n",
       "1128                    bioconductor-genenetworkbuilder            406\n",
       "1129                           bioconductor-geneoverlap            212\n",
       "1130                             bioconductor-geneplast            198\n",
       "1131             bioconductor-geneplast.data.string.v91              9\n",
       "1132                           bioconductor-geneplotter           2919\n",
       "1133                       bioconductor-generecommender            201\n",
       "1134                        bioconductor-generegionscan            200\n",
       "1135                         bioconductor-generxcluster            462\n",
       "1136                         bioconductor-geneselectmmd            286\n",
       "1137                               bioconductor-genesis            100\n",
       "1138                    bioconductor-genestructuretools              8\n",
       "1139                       bioconductor-genetclassifier            195\n",
       "1140                        bioconductor-geneticsdesign            198\n",
       "1141                           bioconductor-geneticsped            277\n",
       "1142                           bioconductor-genextender            367\n",
       "1143                                bioconductor-genie3            396\n",
       "1144                                bioconductor-genocn            289\n",
       "1145                               bioconductor-genogam            466\n",
       "1146                            bioconductor-genomation            860\n",
       "1147                        bioconductor-genomationdata              8\n",
       "1148                          bioconductor-genomeinfodb          12210\n",
       "1149                      bioconductor-genomeinfodbdata          11516\n",
       "1150                       bioconductor-genomeintervals            199\n",
       "1151                               bioconductor-genomes              8\n",
       "1152                   bioconductor-genomewidesnp5crlmm              8\n",
       "1153                   bioconductor-genomewidesnp6crlmm              9\n",
       "1154                     bioconductor-genomicalignments          10675\n",
       "1155                    bioconductor-genomicdatacommons            297\n",
       "1156                       bioconductor-genomicfeatures           2617\n",
       "1157                          bioconductor-genomicfiles            292\n",
       "1158                   bioconductor-genomicinteractions            291\n",
       "1159                          bioconductor-genomicozone              1\n",
       "1160                         bioconductor-genomicranges          12857\n",
       "1161                         bioconductor-genomicscores            298\n",
       "1162                         bioconductor-genomictuples            553\n",
       "1163                               bioconductor-genoset            461\n",
       "1164                          bioconductor-genotypeeval              6\n",
       "1165                               bioconductor-genphen            189\n",
       "1166                               bioconductor-genrank            199\n",
       "1167                               bioconductor-genvisr            307\n",
       "1168                             bioconductor-geometadb            209\n",
       "1169                              bioconductor-geoquery            739\n",
       "1170                         bioconductor-geosubmission            192\n",
       "1171                               bioconductor-gep2pep              9\n",
       "1172                                bioconductor-gesper              7\n",
       "1173                bioconductor-geuvadistranscriptexpr              7\n",
       "1174                              bioconductor-geuvpack              7\n",
       "1175                            bioconductor-geuvstore2              6\n",
       "1176                                bioconductor-gewist            203\n",
       "1177                                bioconductor-ggbase            199\n",
       "1178                                 bioconductor-ggbio            801\n",
       "1179                                bioconductor-ggcyto            410\n",
       "1180                                bioconductor-ggdata              7\n",
       "1181           bioconductor-gghumanmethcancerpanelv1.db              3\n",
       "1182                               bioconductor-ggtools            197\n",
       "1183                                bioconductor-ggtree            603\n",
       "1184                                bioconductor-gigsea              8\n",
       "1185                            bioconductor-gigseadata              8\n",
       "1186                                bioconductor-girafe            466\n",
       "1187                                 bioconductor-gispa              8\n",
       "1188                                  bioconductor-glad            185\n",
       "1189                             bioconductor-gladiatox              4\n",
       "1190                                bioconductor-glimma            239\n",
       "1191                          bioconductor-glmsparsenet              6\n",
       "1192                          bioconductor-globalancova            474\n",
       "1193                             bioconductor-globalseq            289\n",
       "1194                            bioconductor-globaltest            303\n",
       "1195                                 bioconductor-gmapr            284\n",
       "1196                                 bioconductor-gmicr              1\n",
       "1197                                  bioconductor-gmrp            195\n",
       "1198                                 bioconductor-gnet2            184\n",
       "1199                                 bioconductor-go.db           1270\n",
       "1200                             bioconductor-goexpress              7\n",
       "1201                               bioconductor-gofuncr            276\n",
       "1202                            bioconductor-gofunction            195\n",
       "1203                            bioconductor-golubesets            204\n",
       "1204                                 bioconductor-gopro            276\n",
       "1205                            bioconductor-goprofiles             12\n",
       "1206                              bioconductor-gosemsim           1148\n",
       "1207                                 bioconductor-goseq            672\n",
       "1208                                 bioconductor-gosim            287\n",
       "1209                                bioconductor-gostag              6\n",
       "1210                               bioconductor-gostats            574\n",
       "1211                           bioconductor-gosummaries            278\n",
       "1212                                bioconductor-gothic            384\n",
       "1213                               bioconductor-gotools            201\n",
       "1214                               bioconductor-gp53cdf              3\n",
       "1215                                 bioconductor-gpart            276\n",
       "1216                                  bioconductor-gpls            189\n",
       "1217                                bioconductor-gprege              4\n",
       "1218                              bioconductor-gqtlbase              3\n",
       "1219                             bioconductor-gqtlstats              6\n",
       "1220                               bioconductor-gramm4r              1\n",
       "1221                                bioconductor-graper            185\n",
       "1222                                 bioconductor-graph           1787\n",
       "1223                        bioconductor-graphalignment            280\n",
       "1224                               bioconductor-graphat            191\n",
       "1225                              bioconductor-graphite            469\n",
       "1226                              bioconductor-graphpac            192\n",
       "1227                              bioconductor-grasp2db              3\n",
       "1228                    bioconductor-greengenes13.5mgdb              3\n",
       "1229                               bioconductor-grenits            276\n",
       "1230                          bioconductor-greylistchip              3\n",
       "1231                             bioconductor-grmetrics            198\n",
       "1232                               bioconductor-grndata              4\n",
       "1233                                bioconductor-grohmm            286\n",
       "1234                               bioconductor-grridge              3\n",
       "1235                          bioconductor-gsalightning            192\n",
       "1236                                  bioconductor-gsar             11\n",
       "1237                           bioconductor-gsbenchmark              4\n",
       "1238                                  bioconductor-gsca            202\n",
       "1239                              bioconductor-gscreend              1\n",
       "1240                              bioconductor-gse62944              3\n",
       "1241                              bioconductor-gseabase            845\n",
       "1242                       bioconductor-gseabenchmarker              6\n",
       "1243                                bioconductor-gsealm            190\n",
       "1244                                 bioconductor-gsean              3\n",
       "1245                                  bioconductor-gskb             10\n",
       "1246                                 bioconductor-gsreg            462\n",
       "1247                                  bioconductor-gsri            188\n",
       "1248                                  bioconductor-gsva            879\n",
       "1249                              bioconductor-gsvadata              9\n",
       "1250                              bioconductor-gtrellis            193\n",
       "1251                              bioconductor-guideseq            196\n",
       "1252                                bioconductor-guitar              9\n",
       "1253                                  bioconductor-gviz            738\n",
       "1254                               bioconductor-gwascat              7\n",
       "1255                              bioconductor-gwasdata              3\n",
       "1256                             bioconductor-gwastools            116\n",
       "1257                            bioconductor-gwasurvivr              4\n",
       "1258                            bioconductor-h10kcod.db              3\n",
       "1259                            bioconductor-h20kcod.db              3\n",
       "1260                                  bioconductor-h5vc            276\n",
       "1261                              bioconductor-h5vcdata              4\n",
       "1262                              bioconductor-hapfabia            480\n",
       "1263                        bioconductor-hapmap100khind              4\n",
       "1264                         bioconductor-hapmap100kxba              4\n",
       "1265                            bioconductor-hapmap370k              4\n",
       "1266                         bioconductor-hapmap500knsp              4\n",
       "1267                         bioconductor-hapmap500ksty              4\n",
       "1268                            bioconductor-hapmapsnp5              4\n",
       "1269                            bioconductor-hapmapsnp6              4\n",
       "1270                              bioconductor-harbchip              3\n",
       "1271                                bioconductor-harman            480\n",
       "1272                            bioconductor-harmandata              8\n",
       "1273                    bioconductor-harmonizedtcgadata              7\n",
       "1274                            bioconductor-harshlight            467\n",
       "1275                            bioconductor-hcabrowser              2\n",
       "1276                               bioconductor-hcadata              3\n",
       "1277                           bioconductor-hcaexplorer              1\n",
       "1278                             bioconductor-hcg110cdf              4\n",
       "1279                             bioconductor-hcg110.db              3\n",
       "1280                           bioconductor-hcg110probe              3\n",
       "1281                             bioconductor-hd2013sgi              3\n",
       "1282                            bioconductor-hdcytodata              3\n",
       "1283                             bioconductor-hdf5array           1169\n",
       "1284                                  bioconductor-hdtd            284\n",
       "1285                       bioconductor-healthyflowdata              3\n",
       "1286                              bioconductor-heatmaps            294\n",
       "1287                              bioconductor-heatplus            387\n",
       "1288                             bioconductor-heebodata              4\n",
       "1289                           bioconductor-helloranges            197\n",
       "1290                       bioconductor-hellorangesdata             11\n",
       "1291                                  bioconductor-help            202\n",
       "1292                                   bioconductor-hem            465\n",
       "1293                            bioconductor-hgfocuscdf              3\n",
       "1294                            bioconductor-hgfocus.db              6\n",
       "1295                          bioconductor-hgfocusprobe              3\n",
       "1296                           bioconductor-hgu133a2cdf              4\n",
       "1297                           bioconductor-hgu133a2.db            107\n",
       "1298                      bioconductor-hgu133a2frmavecs              4\n",
       "1299                         bioconductor-hgu133a2probe              4\n",
       "1300                    bioconductor-hgu133abarcodevecs              9\n",
       "1301                            bioconductor-hgu133acdf              7\n",
       "1302                            bioconductor-hgu133a.db            126\n",
       "1303                       bioconductor-hgu133afrmavecs              7\n",
       "1304                          bioconductor-hgu133aprobe              3\n",
       "1305                         bioconductor-hgu133atagcdf              3\n",
       "1306                       bioconductor-hgu133atagprobe              6\n",
       "1307                            bioconductor-hgu133bcdf              6\n",
       "1308                            bioconductor-hgu133b.db             10\n",
       "1309                          bioconductor-hgu133bprobe              3\n",
       "1310                bioconductor-hgu133plus2barcodevecs              6\n",
       "1311                        bioconductor-hgu133plus2cdf             10\n",
       "1312                  bioconductor-hgu133plus2cellscore              3\n",
       "1313                        bioconductor-hgu133plus2.db            130\n",
       "1314                   bioconductor-hgu133plus2frmavecs              5\n",
       "1315                      bioconductor-hgu133plus2probe              3\n",
       "1316                             bioconductor-hgu219cdf              3\n",
       "1317                             bioconductor-hgu219.db              3\n",
       "1318                           bioconductor-hgu219probe              3\n",
       "1319                             bioconductor-hgu2beta7              4\n",
       "1320                             bioconductor-hgu95acdf              3\n",
       "1321                             bioconductor-hgu95a.db              6\n",
       "1322                           bioconductor-hgu95aprobe              3\n",
       "1323                              bioconductor-hgu95av2              9\n",
       "1324                           bioconductor-hgu95av2cdf              3\n",
       "1325                           bioconductor-hgu95av2.db            107\n",
       "1326                         bioconductor-hgu95av2probe              3\n",
       "1327                             bioconductor-hgu95bcdf              3\n",
       "1328                             bioconductor-hgu95b.db              3\n",
       "1329                           bioconductor-hgu95bprobe              3\n",
       "1330                             bioconductor-hgu95ccdf              3\n",
       "1331                             bioconductor-hgu95c.db              3\n",
       "1332                           bioconductor-hgu95cprobe              3\n",
       "1333                             bioconductor-hgu95dcdf              3\n",
       "1334                             bioconductor-hgu95d.db              3\n",
       "1335                           bioconductor-hgu95dprobe              6\n",
       "1336                             bioconductor-hgu95ecdf              3\n",
       "1337                             bioconductor-hgu95e.db              3\n",
       "1338                           bioconductor-hgu95eprobe              3\n",
       "1339                        bioconductor-hguatlas13k.db              4\n",
       "1340                           bioconductor-hgubeta7.db              3\n",
       "1341                          bioconductor-hgudkfz31.db              3\n",
       "1342                          bioconductor-hgug4100a.db              4\n",
       "1343                          bioconductor-hgug4101a.db              3\n",
       "1344                          bioconductor-hgug4110b.db              3\n",
       "1345                          bioconductor-hgug4111a.db              3\n",
       "1346                          bioconductor-hgug4112a.db              5\n",
       "1347                          bioconductor-hgug4845a.db              3\n",
       "1348                        bioconductor-hguqiagenv3.db              5\n",
       "1349                            bioconductor-hi16cod.db              3\n",
       "1350                           bioconductor-hiannotator            202\n",
       "1351                                 bioconductor-hibag            463\n",
       "1352                             bioconductor-hicbricks              3\n",
       "1353                            bioconductor-hiccompare            204\n",
       "1354                     bioconductor-hicdatahumanimr90              7\n",
       "1355                    bioconductor-hicdatalymphoblast              9\n",
       "1356                                bioconductor-hicrep            281\n",
       "1357                              bioconductor-hiergwas              5\n",
       "1358                               bioconductor-hierinf              4\n",
       "1359                           bioconductor-hiiragi2013              3\n",
       "1360                          bioconductor-hilbertcurve              3\n",
       "1361                            bioconductor-hilbertvis            678\n",
       "1362                                 bioconductor-hilda             92\n",
       "1363                              bioconductor-hipathia              7\n",
       "1364                      bioconductor-hireadsprocessor              3\n",
       "1365                              bioconductor-hireewas            277\n",
       "1366                                  bioconductor-hitc            195\n",
       "1367                     bioconductor-hivcdnavantwout03              4\n",
       "1368                        bioconductor-hivprtplus2cdf              3\n",
       "1369                             bioconductor-hmdbquery              3\n",
       "1370                               bioconductor-hmmcopy            585\n",
       "1371                            bioconductor-hmp16sdata              7\n",
       "1372                              bioconductor-hmp2data              1\n",
       "1373                              bioconductor-hmyrib36              3\n",
       "1374                         bioconductor-hom.at.inp.db              3\n",
       "1375                         bioconductor-hom.ce.inp.db              3\n",
       "1376                         bioconductor-hom.dm.inp.db              9\n",
       "1377                         bioconductor-hom.dr.inp.db              6\n",
       "1378                         bioconductor-hom.hs.inp.db            149\n",
       "1379                         bioconductor-hom.mm.inp.db              3\n",
       "1380                          bioconductor-homo.sapiens            141\n",
       "1381                         bioconductor-hom.rn.inp.db              3\n",
       "1382                         bioconductor-hom.sc.inp.db              3\n",
       "1383                                bioconductor-hopach            462\n",
       "1384                            bioconductor-hpaanalyze              4\n",
       "1385                               bioconductor-hpannot              3\n",
       "1386                                  bioconductor-hpar            497\n",
       "1387                       bioconductor-hs25kresogen.db              3\n",
       "1388                           bioconductor-hs6ug171.db              3\n",
       "1389              bioconductor-hsagilentdesign026652.db              3\n",
       "1390                        bioconductor-hsmmsinglecell            594\n",
       "1391                                 bioconductor-hspec              3\n",
       "1392                              bioconductor-hspeccdf              3\n",
       "1393                      bioconductor-hta20probeset.db              3\n",
       "1394             bioconductor-hta20transcriptcluster.db              3\n",
       "1395                          bioconductor-hthgu133acdf              3\n",
       "1396                          bioconductor-hthgu133a.db              3\n",
       "1397                     bioconductor-hthgu133afrmavecs              8\n",
       "1398                        bioconductor-hthgu133aprobe              3\n",
       "1399                          bioconductor-hthgu133bcdf              3\n",
       "1400                          bioconductor-hthgu133b.db              3\n",
       "1401                        bioconductor-hthgu133bprobe              4\n",
       "1402                     bioconductor-hthgu133pluspmcdf              3\n",
       "1403                   bioconductor-hthgu133pluspmprobe              3\n",
       "1404                           bioconductor-htmg430acdf              3\n",
       "1405                         bioconductor-htmg430aprobe              3\n",
       "1406                           bioconductor-htmg430bcdf              3\n",
       "1407                         bioconductor-htmg430bprobe              3\n",
       "1408                          bioconductor-htmg430pmcdf              3\n",
       "1409                        bioconductor-htmg430pmprobe              3\n",
       "1410                                bioconductor-htqpcr            195\n",
       "1411                         bioconductor-htrat230pmcdf              4\n",
       "1412                       bioconductor-htrat230pmprobe              3\n",
       "1413                         bioconductor-htratfocuscdf              3\n",
       "1414                       bioconductor-htratfocusprobe              3\n",
       "1415                            bioconductor-htseqgenie              3\n",
       "1416                             bioconductor-htsfilter            204\n",
       "1417                          bioconductor-hu35ksubacdf              3\n",
       "1418                          bioconductor-hu35ksuba.db              3\n",
       "1419                        bioconductor-hu35ksubaprobe              3\n",
       "1420                          bioconductor-hu35ksubbcdf              3\n",
       "1421                          bioconductor-hu35ksubb.db              3\n",
       "1422                        bioconductor-hu35ksubbprobe              3\n",
       "1423                          bioconductor-hu35ksubccdf              3\n",
       "1424                          bioconductor-hu35ksubc.db              5\n",
       "1425                        bioconductor-hu35ksubcprobe              3\n",
       "1426                          bioconductor-hu35ksubdcdf              3\n",
       "1427                          bioconductor-hu35ksubd.db              3\n",
       "1428                        bioconductor-hu35ksubdprobe              3\n",
       "1429                             bioconductor-hu6800cdf              3\n",
       "1430                             bioconductor-hu6800.db              3\n",
       "1431                           bioconductor-hu6800probe              3\n",
       "1432                         bioconductor-hu6800subacdf              3\n",
       "1433                         bioconductor-hu6800subbcdf              3\n",
       "1434                         bioconductor-hu6800subccdf              3\n",
       "1435                         bioconductor-hu6800subdcdf              3\n",
       "1436                   bioconductor-huex10stprobeset.db              4\n",
       "1437          bioconductor-huex10sttranscriptcluster.db              3\n",
       "1438                bioconductor-huex.1.0.st.v2frmavecs              4\n",
       "1439              bioconductor-huexexonprobesetlocation              3\n",
       "1440          bioconductor-huexexonprobesetlocationhg18              3\n",
       "1441          bioconductor-huexexonprobesetlocationhg19              3\n",
       "1442                 bioconductor-hugene10stprobeset.db              3\n",
       "1443        bioconductor-hugene10sttranscriptcluster.db             11\n",
       "1444                       bioconductor-hugene10stv1cdf              3\n",
       "1445              bioconductor-hugene.1.0.st.v1frmavecs              4\n",
       "1446                     bioconductor-hugene10stv1probe              3\n",
       "1447                 bioconductor-hugene11stprobeset.db              3\n",
       "1448        bioconductor-hugene11sttranscriptcluster.db             10\n",
       "1449                 bioconductor-hugene20stprobeset.db              3\n",
       "1450        bioconductor-hugene20sttranscriptcluster.db              3\n",
       "1451                 bioconductor-hugene21stprobeset.db              3\n",
       "1452        bioconductor-hugene21sttranscriptcluster.db              3\n",
       "1453                    bioconductor-human1mduov3bcrlmm              4\n",
       "1454                       bioconductor-human1mv1ccrlmm              4\n",
       "1455                  bioconductor-human370quadv3ccrlmm              4\n",
       "1456                      bioconductor-human370v1ccrlmm              7\n",
       "1457                      bioconductor-human550v3bcrlmm              4\n",
       "1458                  bioconductor-human610quadv1bcrlmm              6\n",
       "1459                      bioconductor-human650v3acrlmm              4\n",
       "1460                  bioconductor-human660quadv1acrlmm              4\n",
       "1461                         bioconductor-humanaffydata              3\n",
       "1462                           bioconductor-humanchrloc              4\n",
       "1463              bioconductor-humancytosnp12v2p1hcrlmm              4\n",
       "1464                             bioconductor-human.db0              6\n",
       "1465                bioconductor-humanomni1quadv1bcrlmm              4\n",
       "1466               bioconductor-humanomni25quadv1bcrlmm              4\n",
       "1467                bioconductor-humanomni5quadv1bcrlmm              4\n",
       "1468            bioconductor-humanomniexpress12v1bcrlmm              4\n",
       "1469                         bioconductor-humanstemcell              3\n",
       "1470          bioconductor-humantranscriptomecompendium              2\n",
       "1471                              bioconductor-huo22.db              3\n",
       "1472                             bioconductor-hwgcod.db              3\n",
       "1473                           bioconductor-hybridmtest            189\n",
       "1474                                 bioconductor-hyper              2\n",
       "1475                             bioconductor-hyperdraw            192\n",
       "1476                            bioconductor-hypergraph            189\n",
       "1477                                 bioconductor-iaseq            191\n",
       "1478                                 bioconductor-iasva              3\n",
       "1479                                 bioconductor-ibbig            278\n",
       "1480                                   bioconductor-ibh            190\n",
       "1481                                  bioconductor-ibmq            282\n",
       "1482                                 bioconductor-icare            380\n",
       "1483                                 bioconductor-icens            195\n",
       "1484                                bioconductor-icetea              6\n",
       "1485                                bioconductor-icheck              7\n",
       "1486                                 bioconductor-ichip            481\n",
       "1487                          bioconductor-iclusterplus             98\n",
       "1488                                  bioconductor-icnv              4\n",
       "1489                                bioconductor-icobra            190\n",
       "1490                                 bioconductor-ideal            194\n",
       "1491                               bioconductor-ideoviz            189\n",
       "1492                              bioconductor-idiogram            202\n",
       "1493                                 bioconductor-idr2d              1\n",
       "1494                                   bioconductor-igc              8\n",
       "1495                           bioconductor-iggeneusage             91\n",
       "1496                                  bioconductor-igvr              6\n",
       "1497                                   bioconductor-ihw            444\n",
       "1498                              bioconductor-ihwpaper            278\n",
       "1499           bioconductor-illumina450probevariants.db              9\n",
       "1500                 bioconductor-illuminadatatestfiles             15\n",
       "1501  bioconductor-illuminahumanmethylation27kanno.i...             10\n",
       "1502        bioconductor-illuminahumanmethylation27k.db              8\n",
       "1503   bioconductor-illuminahumanmethylation27kmanifest              7\n",
       "1504  bioconductor-illuminahumanmethylation450kanno....            164\n",
       "1505  bioconductor-illuminahumanmethylation450kmanifest            140\n",
       "1506     bioconductor-illuminahumanmethylation450kprobe              3\n",
       "1507  bioconductor-illuminahumanmethylationepicanno....            111\n",
       "1508  bioconductor-illuminahumanmethylationepicanno....            103\n",
       "1509  bioconductor-illuminahumanmethylationepicanno....            128\n",
       "1510  bioconductor-illuminahumanmethylationepicmanifest            123\n",
       "1511                    bioconductor-illuminahumanv1.db              3\n",
       "1512              bioconductor-illuminahumanv2beadid.db              3\n",
       "1513                    bioconductor-illuminahumanv2.db              3\n",
       "1514                    bioconductor-illuminahumanv3.db              6\n",
       "1515                    bioconductor-illuminahumanv4.db             12\n",
       "1516              bioconductor-illuminahumanwgdaslv3.db              3\n",
       "1517              bioconductor-illuminahumanwgdaslv4.db              3\n",
       "1518                            bioconductor-illuminaio            848\n",
       "1519                    bioconductor-illuminamousev1.db              3\n",
       "1520                  bioconductor-illuminamousev1p1.db              3\n",
       "1521                    bioconductor-illuminamousev2.db            110\n",
       "1522                      bioconductor-illuminaratv1.db              3\n",
       "1523                              bioconductor-imagehts            186\n",
       "1524                                  bioconductor-imas            199\n",
       "1525                             bioconductor-imetagene              5\n",
       "1526                                 bioconductor-imman              5\n",
       "1527                          bioconductor-immunespacer              9\n",
       "1528                           bioconductor-immunoclust            277\n",
       "1529                              bioconductor-impcdata              4\n",
       "1530                             bioconductor-impulsede            207\n",
       "1531                            bioconductor-impulsede2             10\n",
       "1532                                bioconductor-impute           1192\n",
       "1533                              bioconductor-indac.db              3\n",
       "1534                                bioconductor-indeed              5\n",
       "1535                              bioconductor-infercnv             19\n",
       "1536                                 bioconductor-inpas              3\n",
       "1537                               bioconductor-inpower            201\n",
       "1538                               bioconductor-inspect            200\n",
       "1539                                 bioconductor-intad              3\n",
       "1540                               bioconductor-intansv              3\n",
       "1541                        bioconductor-interactionset            561\n",
       "1542                    bioconductor-interactivedisplay            185\n",
       "1543                bioconductor-interactivedisplaybase            795\n",
       "1544                              bioconductor-interest            189\n",
       "1545                            bioconductor-interminer            196\n",
       "1546                      bioconductor-intramirexplorer              9\n",
       "1547                             bioconductor-inversion            463\n",
       "1548                               bioconductor-ioniser            189\n",
       "1549                                  bioconductor-ipac            200\n",
       "1550                                 bioconductor-ipddb              6\n",
       "1551                                   bioconductor-ipo            289\n",
       "1552                                  bioconductor-ippd            467\n",
       "1553                               bioconductor-iranges          13965\n",
       "1554                                  bioconductor-isee            107\n",
       "1555                                  bioconductor-iseq            468\n",
       "1556                                bioconductor-isobar              3\n",
       "1557                          bioconductor-isocorrector              4\n",
       "1558                       bioconductor-isocorrectorgui              2\n",
       "1559                 bioconductor-isoformswitchanalyzer            481\n",
       "1560                            bioconductor-isogenegui            280\n",
       "1561                                bioconductor-isolde            556\n",
       "1562                               bioconductor-isomirs            476\n",
       "1563                               bioconductor-italics              4\n",
       "1564                           bioconductor-italicsdata              4\n",
       "1565                          bioconductor-iterativebma            194\n",
       "1566                      bioconductor-iterativebmasurv            190\n",
       "1567                             bioconductor-iterclust              7\n",
       "1568                            bioconductor-iteremoval              3\n",
       "1569                                  bioconductor-ivas            188\n",
       "1570                              bioconductor-ivygapse              6\n",
       "1571                              bioconductor-iwtomics            294\n",
       "1572                               bioconductor-iyer517              3\n",
       "1573                            bioconductor-jaspar2014            281\n",
       "1574                            bioconductor-jaspar2016            292\n",
       "1575                            bioconductor-jaspar2018            197\n",
       "1576                    bioconductor-jazaerimetadata.db              3\n",
       "1577                            bioconductor-jctseqdata              4\n",
       "1578                                  bioconductor-joda              3\n",
       "1579                           bioconductor-junctionseq            480\n",
       "1580                           bioconductor-karyoploter             17\n",
       "1581                               bioconductor-kcsmart            377\n",
       "1582                                bioconductor-kebabs            483\n",
       "1583          bioconductor-keggandmetacoredzpathwaysgeo              3\n",
       "1584                               bioconductor-kegg.db            231\n",
       "1585                     bioconductor-keggdzpathwaysgeo            198\n",
       "1586                             bioconductor-kegggraph            634\n",
       "1587                             bioconductor-kegglincs            199\n",
       "1588                         bioconductor-keggorthology            195\n",
       "1589                           bioconductor-keggprofile              6\n",
       "1590                              bioconductor-keggrest            771\n",
       "1591                               bioconductor-kidpack              3\n",
       "1592                                 bioconductor-kimod            199\n",
       "1593                             bioconductor-kinswingr              4\n",
       "1594                                bioconductor-kissde              6\n",
       "1595                               bioconductor-knowseq              2\n",
       "1596                                bioconductor-kodata            191\n",
       "1597                                bioconductor-lapmix            188\n",
       "1598                           bioconductor-lapointe.db              4\n",
       "1599                                   bioconductor-lbe            192\n",
       "1600                               bioconductor-ldblock              3\n",
       "1601                                   bioconductor-lea            289\n",
       "1602                               bioconductor-ledpred              4\n",
       "1603                           bioconductor-leebamviews              3\n",
       "1604                                   bioconductor-les            192\n",
       "1605                         bioconductor-leukemiaseset              4\n",
       "1606                                  bioconductor-levi            276\n",
       "1607                                   bioconductor-lfa            484\n",
       "1608                 bioconductor-liebermanaidenhic2009              3\n",
       "1609                                 bioconductor-limma           4835\n",
       "1610                              bioconductor-limmagui              4\n",
       "1611                                  bioconductor-linc            474\n",
       "1612                          bioconductor-lineagepulse              3\n",
       "1613                                bioconductor-linkhd              1\n",
       "1614                               bioconductor-linnorm            387\n",
       "1615                              bioconductor-lionessr              1\n",
       "1616                                bioconductor-lipidr              2\n",
       "1617                     bioconductor-liquidassociation            189\n",
       "1618                       bioconductor-listeretalbsseq              4\n",
       "1619                                 bioconductor-lmdme            201\n",
       "1620                                bioconductor-lmgene            188\n",
       "1621                              bioconductor-lobstahs            103\n",
       "1622                             bioconductor-loci2path              4\n",
       "1623                               bioconductor-logicfs              4\n",
       "1624                                bioconductor-logitt            460\n",
       "1625                               bioconductor-logolas              3\n",
       "1626                                  bioconductor-lola            198\n",
       "1627                        bioconductor-loomexperiment             36\n",
       "1628                               bioconductor-lowmaca              3\n",
       "1629                     bioconductor-lowmacaannotation              4\n",
       "1630                                   bioconductor-lpe            202\n",
       "1631                                bioconductor-lpeadj            189\n",
       "1632                                 bioconductor-lpnet            196\n",
       "1633                            bioconductor-lpsymphony            725\n",
       "1634                      bioconductor-lrbase.ath.eg.db              7\n",
       "1635                      bioconductor-lrbase.bta.eg.db              3\n",
       "1636                      bioconductor-lrbase.cel.eg.db              3\n",
       "1637                             bioconductor-lrbasedbi              3\n",
       "1638                      bioconductor-lrbase.dme.eg.db              3\n",
       "1639                      bioconductor-lrbase.dre.eg.db              7\n",
       "1640                      bioconductor-lrbase.gga.eg.db              3\n",
       "1641                      bioconductor-lrbase.hsa.eg.db              3\n",
       "1642                      bioconductor-lrbase.mmu.eg.db              3\n",
       "1643                      bioconductor-lrbase.pab.eg.db              3\n",
       "1644                      bioconductor-lrbase.rno.eg.db              4\n",
       "1645                      bioconductor-lrbase.ssc.eg.db              3\n",
       "1646                      bioconductor-lrbase.xtr.eg.db              6\n",
       "1647                                  bioconductor-lumi            215\n",
       "1648                            bioconductor-lumibarnes              8\n",
       "1649                       bioconductor-lumihumanall.db            104\n",
       "1650                    bioconductor-lumihumanidmapping              6\n",
       "1651                       bioconductor-lumimouseall.db              3\n",
       "1652                    bioconductor-lumimouseidmapping              7\n",
       "1653                         bioconductor-lumiratall.db              3\n",
       "1654                      bioconductor-lumiratidmapping              5\n",
       "1655                  bioconductor-lungcanceracvssccgeo            202\n",
       "1656                       bioconductor-lungcancerlines              3\n",
       "1657                        bioconductor-lungexpression              3\n",
       "1658                                bioconductor-lydata              4\n",
       "1659                             bioconductor-lymphoseq              7\n",
       "1660                           bioconductor-lymphoseqdb              5\n",
       "1661                            bioconductor-m10kcod.db              3\n",
       "1662                            bioconductor-m20kcod.db              3\n",
       "1663                                   bioconductor-m3c              5\n",
       "1664                                   bioconductor-m3d            466\n",
       "1665                        bioconductor-m3dexampledata              4\n",
       "1666                                bioconductor-m3drop            297\n",
       "1667                               bioconductor-maanova            557\n",
       "1668                              bioconductor-maaslin2              1\n",
       "1669                                 bioconductor-macat            290\n",
       "1670                            bioconductor-macorrplot            283\n",
       "1671                                bioconductor-macpet            276\n",
       "1672                            bioconductor-macrophage              3\n",
       "1673                         bioconductor-macsquantifyr              1\n",
       "1674                                 bioconductor-made4            379\n",
       "1675                                bioconductor-madseq            195\n",
       "1676         bioconductor-mafdb.1kgenomes.phase1.grch38              3\n",
       "1677         bioconductor-mafdb.1kgenomes.phase1.hs37d5              3\n",
       "1678         bioconductor-mafdb.1kgenomes.phase3.grch38              4\n",
       "1679         bioconductor-mafdb.1kgenomes.phase3.hs37d5              3\n",
       "1680      bioconductor-mafdb.esp6500si.v2.ssa137.grch38              6\n",
       "1681      bioconductor-mafdb.esp6500si.v2.ssa137.hs37d5              3\n",
       "1682                bioconductor-mafdb.exac.r1.0.grch38              5\n",
       "1683                bioconductor-mafdb.exac.r1.0.hs37d5              3\n",
       "1684        bioconductor-mafdb.exac.r1.0.nontcga.grch38              4\n",
       "1685        bioconductor-mafdb.exac.r1.0.nontcga.hs37d5              6\n",
       "1686            bioconductor-mafdb.gnomadex.r2.1.grch38              2\n",
       "1687            bioconductor-mafdb.gnomadex.r2.1.hs37d5              3\n",
       "1688              bioconductor-mafdb.gnomad.r2.1.grch38              1\n",
       "1689              bioconductor-mafdb.gnomad.r2.1.hs37d5              4\n",
       "1690             bioconductor-mafdb.topmed.freeze5.hg19              6\n",
       "1691             bioconductor-mafdb.topmed.freeze5.hg38              6\n",
       "1692                              bioconductor-maftools            243\n",
       "1693                           bioconductor-mageckflute             10\n",
       "1694                            bioconductor-maigespack            471\n",
       "1695                                  bioconductor-mait            368\n",
       "1696                              bioconductor-maizecdf              3\n",
       "1697                            bioconductor-maizeprobe              3\n",
       "1698                            bioconductor-makecdfenv            554\n",
       "1699                           bioconductor-malaria.db0              4\n",
       "1700                        bioconductor-mammaprintdata              4\n",
       "1701                                 bioconductor-manor            185\n",
       "1702                            bioconductor-mantelcorr            293\n",
       "1703                                 bioconductor-mapkl             94\n",
       "1704                             bioconductor-mapkldata              4\n",
       "1705                          bioconductor-mapredictdsc            200\n",
       "1706                              bioconductor-mapscape              8\n",
       "1707                   bioconductor-maqcexpression4plex              4\n",
       "1708                            bioconductor-maqcsubset              7\n",
       "1709                         bioconductor-maqcsubsetafx              4\n",
       "1710                         bioconductor-maqcsubsetilm              6\n",
       "1711                                bioconductor-marray            453\n",
       "1712                               bioconductor-martini            277\n",
       "1713                                 bioconductor-maser              6\n",
       "1714                              bioconductor-masigpro            662\n",
       "1715                               bioconductor-maskbad            187\n",
       "1716                             bioconductor-massarray            282\n",
       "1717                                bioconductor-massir            281\n",
       "1718                       bioconductor-massspecwavelet            978\n",
       "1719                                  bioconductor-mast            304\n",
       "1720                              bioconductor-matchbox              4\n",
       "1721                           bioconductor-matrixrider            286\n",
       "1722                                bioconductor-matter            660\n",
       "1723                 bioconductor-maxcontrastprojection              4\n",
       "1724                             bioconductor-mbamethyl            283\n",
       "1725                                bioconductor-mbased            290\n",
       "1726                                  bioconductor-mbcb              5\n",
       "1727                              bioconductor-mbkmeans            189\n",
       "1728                                 bioconductor-mbpcr              3\n",
       "1729                               bioconductor-mbttest            285\n",
       "1730                                bioconductor-mcagui              2\n",
       "1731                             bioconductor-mcbiclust              6\n",
       "1732                                 bioconductor-mcsea              6\n",
       "1733                             bioconductor-mcseadata              9\n",
       "1734                            bioconductor-mcsurvdata              9\n",
       "1735                                 bioconductor-mdgsa            282\n",
       "1736                                   bioconductor-mdp              5\n",
       "1737                                  bioconductor-mdqc            291\n",
       "1738                                  bioconductor-mdts              6\n",
       "1739                                  bioconductor-meal              6\n",
       "1740                  bioconductor-measurementerror.cor            188\n",
       "1741                                   bioconductor-meb              1\n",
       "1742                           bioconductor-medicagocdf              3\n",
       "1743                         bioconductor-medicagoprobe              3\n",
       "1744                                bioconductor-medips            383\n",
       "1745                            bioconductor-medipsdata              4\n",
       "1746                                 bioconductor-medme            461\n",
       "1747                             bioconductor-meebodata              4\n",
       "1748                                bioconductor-meigor            188\n",
       "1749                               bioconductor-melissa              2\n",
       "1750                             bioconductor-mergemaid            281\n",
       "1751                            bioconductor-mergeomics            291\n",
       "1752                        bioconductor-mesh.aca.eg.db              3\n",
       "1753                   bioconductor-mesh.aga.pest.eg.db              3\n",
       "1754                        bioconductor-mesh.ame.eg.db              3\n",
       "1755                        bioconductor-mesh.aml.eg.db              3\n",
       "1756                        bioconductor-mesh.ana.eg.db              3\n",
       "1757                   bioconductor-mesh.ani.fgsc.eg.db              6\n",
       "1758                           bioconductor-mesh.aor.db              7\n",
       "1759                        bioconductor-mesh.ath.eg.db              7\n",
       "1760                        bioconductor-mesh.bfl.eg.db              3\n",
       "1761                    bioconductor-mesh.bsu.168.eg.db              3\n",
       "1762                        bioconductor-mesh.bta.eg.db              3\n",
       "1763                 bioconductor-mesh.cal.sc5314.eg.db              7\n",
       "1764                        bioconductor-mesh.cbr.eg.db              3\n",
       "1765                        bioconductor-mesh.cel.eg.db              3\n",
       "1766                        bioconductor-mesh.cfa.eg.db              7\n",
       "1767                        bioconductor-mesh.cin.eg.db              3\n",
       "1768                        bioconductor-mesh.cja.eg.db              3\n",
       "1769                        bioconductor-mesh.cpo.eg.db              3\n",
       "1770                        bioconductor-mesh.cre.eg.db              3\n",
       "1771                        bioconductor-mesh.dan.eg.db              3\n",
       "1772                               bioconductor-mesh.db            188\n",
       "1773                               bioconductor-meshdbi            281\n",
       "1774                   bioconductor-mesh.dda.3937.eg.db              3\n",
       "1775                    bioconductor-mesh.ddi.ax4.eg.db              3\n",
       "1776                        bioconductor-mesh.der.eg.db              3\n",
       "1777                        bioconductor-mesh.dgr.eg.db              3\n",
       "1778                        bioconductor-mesh.dme.eg.db              3\n",
       "1779                        bioconductor-mesh.dmo.eg.db              3\n",
       "1780                        bioconductor-mesh.dpe.eg.db              3\n",
       "1781                        bioconductor-mesh.dre.eg.db              3\n",
       "1782                        bioconductor-mesh.dse.eg.db              3\n",
       "1783                        bioconductor-mesh.dsi.eg.db              3\n",
       "1784                        bioconductor-mesh.dvi.eg.db              3\n",
       "1785                        bioconductor-mesh.dya.eg.db              3\n",
       "1786                  bioconductor-mesh.eco.55989.eg.db              3\n",
       "1787                   bioconductor-mesh.eco.ed1a.eg.db              3\n",
       "1788                  bioconductor-mesh.eco.iai39.eg.db              5\n",
       "1789             bioconductor-mesh.eco.k12.mg1655.eg.db              6\n",
       "1790          bioconductor-mesh.eco.o157.h7.sakai.eg.db              3\n",
       "1791                 bioconductor-mesh.eco.umn026.eg.db              3\n",
       "1792                        bioconductor-mesh.eqc.eg.db              3\n",
       "1793                                bioconductor-meshes            195\n",
       "1794                        bioconductor-mesh.gga.eg.db              3\n",
       "1795                        bioconductor-mesh.gma.eg.db              6\n",
       "1796                        bioconductor-mesh.hsa.eg.db              3\n",
       "1797                        bioconductor-mesh.laf.eg.db              3\n",
       "1798                        bioconductor-mesh.lma.eg.db              4\n",
       "1799                        bioconductor-mesh.mdo.eg.db              4\n",
       "1800                        bioconductor-mesh.mes.eg.db              3\n",
       "1801                        bioconductor-mesh.mga.eg.db              3\n",
       "1802                        bioconductor-mesh.miy.eg.db              3\n",
       "1803                        bioconductor-mesh.mml.eg.db              3\n",
       "1804                        bioconductor-mesh.mmu.eg.db              3\n",
       "1805                        bioconductor-mesh.mtr.eg.db              3\n",
       "1806                        bioconductor-mesh.nle.eg.db              4\n",
       "1807                        bioconductor-mesh.oan.eg.db              3\n",
       "1808                        bioconductor-mesh.ocu.eg.db              3\n",
       "1809                        bioconductor-mesh.oni.eg.db              7\n",
       "1810                        bioconductor-mesh.osa.eg.db              3\n",
       "1811                        bioconductor-mesh.pab.eg.db              4\n",
       "1812                   bioconductor-mesh.pae.pao1.eg.db              3\n",
       "1813                           bioconductor-mesh.pcr.db              3\n",
       "1814                    bioconductor-mesh.pfa.3d7.eg.db              6\n",
       "1815                        bioconductor-mesh.pto.eg.db              9\n",
       "1816                        bioconductor-mesh.ptr.eg.db              4\n",
       "1817                                 bioconductor-meshr              8\n",
       "1818                        bioconductor-mesh.rno.eg.db              3\n",
       "1819                  bioconductor-mesh.sce.s288c.eg.db              3\n",
       "1820                    bioconductor-mesh.sco.a32.eg.db              8\n",
       "1821                        bioconductor-mesh.sil.eg.db              6\n",
       "1822                        bioconductor-mesh.spu.eg.db              4\n",
       "1823                        bioconductor-mesh.ssc.eg.db              7\n",
       "1824                        bioconductor-mesh.syn.eg.db              9\n",
       "1825                   bioconductor-mesh.tbr.9274.eg.db              3\n",
       "1826                   bioconductor-mesh.tgo.me49.eg.db              3\n",
       "1827                        bioconductor-mesh.tgu.eg.db              3\n",
       "1828                        bioconductor-mesh.vvi.eg.db              4\n",
       "1829                        bioconductor-mesh.xla.eg.db              3\n",
       "1830                        bioconductor-mesh.xtr.eg.db              3\n",
       "1831                        bioconductor-mesh.zma.eg.db              3\n",
       "1832                               bioconductor-messina            569\n",
       "1833                             bioconductor-metaarray            569\n",
       "1834                                 bioconductor-metab            124\n",
       "1835                            bioconductor-metabomxtr              4\n",
       "1836                          bioconductor-metabosignal            190\n",
       "1837                               bioconductor-metacca            283\n",
       "1838                              bioconductor-metacyto              4\n",
       "1839                             bioconductor-metagene2              2\n",
       "1840                              bioconductor-metagene              7\n",
       "1841                    bioconductor-metagenomefeatures              4\n",
       "1842                         bioconductor-metagenomeseq            446\n",
       "1843                          bioconductor-metagxbreast              3\n",
       "1844                         bioconductor-metagxovarian              6\n",
       "1845                        bioconductor-metagxpancreas              3\n",
       "1846                              bioconductor-metahdep            575\n",
       "1847                                bioconductor-metams            374\n",
       "1848                            bioconductor-metamsdata              9\n",
       "1849                          bioconductor-metaneighbor            185\n",
       "1850                               bioconductor-metaseq            554\n",
       "1851                              bioconductor-metaseqr              7\n",
       "1852                              bioconductor-metavizr            102\n",
       "1853                          bioconductor-metavolcanor              1\n",
       "1854                               bioconductor-metcirc            297\n",
       "1855                                bioconductor-methcp              1\n",
       "1856                            bioconductor-methimpute            369\n",
       "1857                        bioconductor-methinheritsim              7\n",
       "1858                               bioconductor-methped            293\n",
       "1859                               bioconductor-methrix              1\n",
       "1860                       bioconductor-methtargetedngs            282\n",
       "1861                            bioconductor-methvisual              4\n",
       "1862                         bioconductor-methyanalysis            193\n",
       "1863                             bioconductor-methylaid              9\n",
       "1864                         bioconductor-methylaiddata              3\n",
       "1865                              bioconductor-methylcc              1\n",
       "1866                             bioconductor-methylgsa              6\n",
       "1867                     bioconductor-methylinheritance              3\n",
       "1868                             bioconductor-methylkit            866\n",
       "1869                             bioconductor-methylmix              9\n",
       "1870                             bioconductor-methylmnm            555\n",
       "1871                            bioconductor-methylpipe            570\n",
       "1872                           bioconductor-methylseekr            285\n",
       "1873                             bioconductor-methylumi            322\n",
       "1874                              bioconductor-methyvim              7\n",
       "1875                          bioconductor-methyvimdata              7\n",
       "1876                                 bioconductor-metid              6\n",
       "1877                                bioconductor-metnet              7\n",
       "1878                                   bioconductor-mfa            374\n",
       "1879                                 bioconductor-mfuzz             21\n",
       "1880                                  bioconductor-mgfm            288\n",
       "1881                                  bioconductor-mgfr            283\n",
       "1882                                  bioconductor-mgsa            556\n",
       "1883                             bioconductor-mgu74acdf              3\n",
       "1884                             bioconductor-mgu74a.db              3\n",
       "1885                           bioconductor-mgu74aprobe              3\n",
       "1886                           bioconductor-mgu74av2cdf              3\n",
       "1887                           bioconductor-mgu74av2.db              3\n",
       "1888                         bioconductor-mgu74av2probe              3\n",
       "1889                             bioconductor-mgu74bcdf              3\n",
       "1890                             bioconductor-mgu74b.db              3\n",
       "1891                           bioconductor-mgu74bprobe              3\n",
       "1892                           bioconductor-mgu74bv2cdf              3\n",
       "1893                           bioconductor-mgu74bv2.db              3\n",
       "1894                         bioconductor-mgu74bv2probe              3\n",
       "1895                             bioconductor-mgu74ccdf              3\n",
       "1896                             bioconductor-mgu74c.db              3\n",
       "1897                           bioconductor-mgu74cprobe              4\n",
       "1898                           bioconductor-mgu74cv2cdf              3\n",
       "1899                           bioconductor-mgu74cv2.db              6\n",
       "1900                         bioconductor-mgu74cv2probe              3\n",
       "1901                         bioconductor-mguatlas5k.db              3\n",
       "1902                          bioconductor-mgug4104a.db              3\n",
       "1903                          bioconductor-mgug4120a.db              3\n",
       "1904                          bioconductor-mgug4121a.db              3\n",
       "1905                          bioconductor-mgug4122a.db            193\n",
       "1906                            bioconductor-mi16cod.db              3\n",
       "1907                                bioconductor-michip            292\n",
       "1908                            bioconductor-microbiome            213\n",
       "1909                       bioconductor-microbiomedasim              1\n",
       "1910                              bioconductor-microrna            555\n",
       "1911                           bioconductor-micrornaome              5\n",
       "1912                                 bioconductor-migsa              3\n",
       "1913                             bioconductor-migsadata              4\n",
       "1914                               bioconductor-mimager              3\n",
       "1915                                bioconductor-mimosa            377\n",
       "1916                               bioconductor-mineica            101\n",
       "1917                                 bioconductor-minet            483\n",
       "1918                                 bioconductor-minfi            562\n",
       "1919                             bioconductor-minfidata             10\n",
       "1920                         bioconductor-minfidataepic              9\n",
       "1921                       bioconductor-minimumdistance            189\n",
       "1922                     bioconductor-minionsummarydata              5\n",
       "1923                                  bioconductor-mipp            281\n",
       "1924                                  bioconductor-mira              3\n",
       "1925                                bioconductor-mirage            282\n",
       "1926                      bioconductor-mirbaseconverter              6\n",
       "1927                            bioconductor-mirbase.db              7\n",
       "1928                    bioconductor-mirbaseversions.db            203\n",
       "1929                               bioconductor-mircomp              3\n",
       "1930                           bioconductor-mircompdata              4\n",
       "1931                         bioconductor-mirintegrator            188\n",
       "1932                               bioconductor-mirmine              3\n",
       "1933                      bioconductor-mirna102xgaincdf              3\n",
       "1934                            bioconductor-mirna10cdf              3\n",
       "1935                          bioconductor-mirna10probe              3\n",
       "1936                            bioconductor-mirna20cdf              3\n",
       "1937                      bioconductor-mirnameconverter            188\n",
       "1938                             bioconductor-mirnapath            285\n",
       "1939                              bioconductor-mirnatap            293\n",
       "1940                           bioconductor-mirnatap.db             97\n",
       "1941                           bioconductor-mirnatarget              3\n",
       "1942                                 bioconductor-mirsm            186\n",
       "1943                            bioconductor-mirsponger            188\n",
       "1944                            bioconductor-mirsynergy            282\n",
       "1945                            bioconductor-missmethyl            207\n",
       "1946                              bioconductor-missrows              3\n",
       "1947                               bioconductor-mitoode            385\n",
       "1948                           bioconductor-mitoodedata              4\n",
       "1949                              bioconductor-mixomics             35\n",
       "1950                          bioconductor-mlinterfaces            196\n",
       "1951                                   bioconductor-mlp            285\n",
       "1952                                 bioconductor-mlseq            195\n",
       "1953                       bioconductor-mm24kresogen.db              3\n",
       "1954              bioconductor-mmagilentdesign026655.db              3\n",
       "1955                               bioconductor-mmappr2              2\n",
       "1956                           bioconductor-mmappr2data              1\n",
       "1957                               bioconductor-mmdiff2            189\n",
       "1958                       bioconductor-mmdiffbamsubset              7\n",
       "1959                         bioconductor-mmpalatemirna              3\n",
       "1960                               bioconductor-mmuphin              1\n",
       "1961                                  bioconductor-moda              3\n",
       "1962                            bioconductor-modstrings              2\n",
       "1963                            bioconductor-moe430acdf              4\n",
       "1964                            bioconductor-moe430a.db              3\n",
       "1965                          bioconductor-moe430aprobe              3\n",
       "1966                            bioconductor-moe430bcdf              3\n",
       "1967                            bioconductor-moe430b.db              3\n",
       "1968                          bioconductor-moe430bprobe              8\n",
       "1969                   bioconductor-moex10stprobeset.db              3\n",
       "1970          bioconductor-moex10sttranscriptcluster.db              4\n",
       "1971              bioconductor-moexexonprobesetlocation              3\n",
       "1972                                  bioconductor-mofa              6\n",
       "1973                              bioconductor-mofadata              5\n",
       "1974                 bioconductor-mogene10stprobeset.db              3\n",
       "1975        bioconductor-mogene10sttranscriptcluster.db              3\n",
       "1976                       bioconductor-mogene10stv1cdf              3\n",
       "1977              bioconductor-mogene.1.0.st.v1frmavecs              6\n",
       "1978                     bioconductor-mogene10stv1probe              3\n",
       "1979                 bioconductor-mogene11stprobeset.db              3\n",
       "1980        bioconductor-mogene11sttranscriptcluster.db              3\n",
       "1981                 bioconductor-mogene20stprobeset.db              3\n",
       "1982        bioconductor-mogene20sttranscriptcluster.db              6\n",
       "1983                 bioconductor-mogene21stprobeset.db              3\n",
       "1984        bioconductor-mogene21sttranscriptcluster.db              3\n",
       "1985                                 bioconductor-mogsa             10\n",
       "1986                               bioconductor-monocle            872\n",
       "1987                            bioconductor-moonlightr              6\n",
       "1988                                  bioconductor-mops            281\n",
       "1989                               bioconductor-mosaics            744\n",
       "1990                        bioconductor-mosaicsexample              4\n",
       "1991                                 bioconductor-mosim              1\n",
       "1992                           bioconductor-motifbreakr            374\n",
       "1993                          bioconductor-motifcounter            480\n",
       "1994                               bioconductor-motifdb            391\n",
       "1995                           bioconductor-motifmatchr            284\n",
       "1996                               bioconductor-motifrg            473\n",
       "1997                            bioconductor-motifstack            387\n",
       "1998                                 bioconductor-motiv            660\n",
       "1999                  bioconductor-mouse4302barcodevecs             10\n",
       "2000                          bioconductor-mouse4302cdf              3\n",
       "2001                          bioconductor-mouse4302.db              4\n",
       "2002                     bioconductor-mouse4302frmavecs              4\n",
       "2003                        bioconductor-mouse4302probe              3\n",
       "2004                         bioconductor-mouse430a2cdf              3\n",
       "2005                         bioconductor-mouse430a2.db              4\n",
       "2006                    bioconductor-mouse430a2frmavecs              8\n",
       "2007                       bioconductor-mouse430a2probe              3\n",
       "2008                           bioconductor-mousechrloc              4\n",
       "2009                             bioconductor-mouse.db0              4\n",
       "2010                 bioconductor-mousegastrulationdata              3\n",
       "2011                         bioconductor-mpedbarray.db              4\n",
       "2012                                  bioconductor-mpfe            282\n",
       "2013                                  bioconductor-mpra              6\n",
       "2014                            bioconductor-mpranalyze              3\n",
       "2015                                   bioconductor-msa            309\n",
       "2016                                bioconductor-msd16s              6\n",
       "2017                                bioconductor-msdata              8\n",
       "2018                                bioconductor-msgbsr              3\n",
       "2019                               bioconductor-msgfgui              1\n",
       "2020                              bioconductor-msgfplus              4\n",
       "2021                                  bioconductor-msmb              5\n",
       "2022                               bioconductor-msmseda             98\n",
       "2023                             bioconductor-msmstests              5\n",
       "2024                               bioconductor-msnbase            794\n",
       "2025                                 bioconductor-msnid            368\n",
       "2026                              bioconductor-mspurity            661\n",
       "2027                          bioconductor-mspuritydata            112\n",
       "2028                                 bioconductor-msqc1              3\n",
       "2029                               bioconductor-msstats             18\n",
       "2030                        bioconductor-msstatsbiodata              4\n",
       "2031                             bioconductor-msstatsqc              3\n",
       "2032                          bioconductor-msstatsqcgui              3\n",
       "2033                     bioconductor-msstatssamplesize              1\n",
       "2034                            bioconductor-msstatstmt              4\n",
       "2035                      bioconductor-mta10probeset.db              3\n",
       "2036             bioconductor-mta10transcriptcluster.db              3\n",
       "2037                                bioconductor-mtbls2              4\n",
       "2038                              bioconductor-mtseeker              4\n",
       "2039                          bioconductor-mtseekerdata              6\n",
       "2040                          bioconductor-mu11ksubacdf              3\n",
       "2041                          bioconductor-mu11ksuba.db              3\n",
       "2042                        bioconductor-mu11ksubaprobe              3\n",
       "2043                          bioconductor-mu11ksubbcdf              3\n",
       "2044                          bioconductor-mu11ksubb.db              3\n",
       "2045                        bioconductor-mu11ksubbprobe              3\n",
       "2046                             bioconductor-mu15v1.db              4\n",
       "2047                          bioconductor-mu19ksubacdf              3\n",
       "2048                          bioconductor-mu19ksuba.db              3\n",
       "2049                          bioconductor-mu19ksubbcdf              3\n",
       "2050                          bioconductor-mu19ksubb.db              3\n",
       "2051                          bioconductor-mu19ksubccdf              3\n",
       "2052                          bioconductor-mu19ksubc.db              3\n",
       "2053                             bioconductor-mu22v3.db              3\n",
       "2054                         bioconductor-mu6500subacdf              3\n",
       "2055                         bioconductor-mu6500subbcdf              3\n",
       "2056                         bioconductor-mu6500subccdf              3\n",
       "2057                         bioconductor-mu6500subdcdf              3\n",
       "2058                       bioconductor-mugaexampledata              5\n",
       "2059                                bioconductor-mulcom            462\n",
       "2060                            bioconductor-mulder2012              6\n",
       "2061                  bioconductor-multiassayexperiment            212\n",
       "2062                            bioconductor-multiclust            194\n",
       "2063                          bioconductor-multidataset            306\n",
       "2064                       bioconductor-multihiccompare              5\n",
       "2065                              bioconductor-multimed            283\n",
       "2066                              bioconductor-multimir              9\n",
       "2067                         bioconductor-multiomicsviz            285\n",
       "2068                             bioconductor-multiscan            566\n",
       "2069                              bioconductor-multtest           1275\n",
       "2070                                bioconductor-muscat              3\n",
       "2071                              bioconductor-muscdata              2\n",
       "2072                                bioconductor-muscle            282\n",
       "2073                          bioconductor-mus.musculus             14\n",
       "2074                    bioconductor-mutationalpatterns            374\n",
       "2075                              bioconductor-mvcclass            283\n",
       "2076                             bioconductor-mvoutdata              7\n",
       "2077                             bioconductor-mwastools            193\n",
       "2078                             bioconductor-mwgcod.db              3\n",
       "2079                                bioconductor-mygene            210\n",
       "2080                             bioconductor-myvariant            291\n",
       "2081                                  bioconductor-mzid            510\n",
       "2082                                   bioconductor-mzr            984\n",
       "2083                             bioconductor-nadfinder            190\n",
       "2084                        bioconductor-nanostringdiff            280\n",
       "2085                       bioconductor-nanostringqcpro              4\n",
       "2086                             bioconductor-nanotator              3\n",
       "2087                             bioconductor-nanotubes              2\n",
       "2088                           bioconductor-narrowpeaks            280\n",
       "2089                               bioconductor-nbamseq              2\n",
       "2090                              bioconductor-nbsplice              5\n",
       "2091                              bioconductor-ncdfflow            775\n",
       "2092                                 bioconductor-ncgtw             92\n",
       "2093                              bioconductor-ncigraph            282\n",
       "2094                          bioconductor-ncigraphdata              4\n",
       "2095                                 bioconductor-ndexr            197\n",
       "2096                           bioconductor-neighbornet              4\n",
       "2097                              bioconductor-nestlink              4\n",
       "2098                          bioconductor-netbenchmark            277\n",
       "2099                               bioconductor-netbiov            290\n",
       "2100                              bioconductor-netboost            183\n",
       "2101                                bioconductor-nethet            280\n",
       "2102                          bioconductor-netpathminer            565\n",
       "2103                              bioconductor-netprior              4\n",
       "2104                                bioconductor-netreg            462\n",
       "2105                           bioconductor-netresponse            285\n",
       "2106                                bioconductor-netsam              3\n",
       "2107                             bioconductor-netsmooth              4\n",
       "2108                            bioconductor-networkbma            462\n",
       "2109                              bioconductor-neve2006              3\n",
       "2110                           bioconductor-ngscopydata              4\n",
       "2111                            bioconductor-ngsreports              3\n",
       "2112                                bioconductor-nnnorm            282\n",
       "2113                                bioconductor-noiseq           5679\n",
       "2114                            bioconductor-nondetects            203\n",
       "2115                         bioconductor-normalize450k              3\n",
       "2116                          bioconductor-normalyzerde              3\n",
       "2117                              bioconductor-normqpcr              3\n",
       "2118                                 bioconductor-normr            284\n",
       "2119                          bioconductor-norway981.db              3\n",
       "2120                                bioconductor-npgsea            286\n",
       "2121                                   bioconductor-ntw            289\n",
       "2122                             bioconductor-nucleosim            280\n",
       "2123                                bioconductor-nucler            281\n",
       "2124                                bioconductor-nucpos            393\n",
       "2125                     bioconductor-nugohs1a520180cdf              3\n",
       "2126                     bioconductor-nugohs1a520180.db              4\n",
       "2127                   bioconductor-nugohs1a520180probe              3\n",
       "2128                     bioconductor-nugomm1a520177cdf              3\n",
       "2129                     bioconductor-nugomm1a520177.db              3\n",
       "2130                   bioconductor-nugomm1a520177probe              3\n",
       "2131                                 bioconductor-nupop            288\n",
       "2132                              bioconductor-occugene            292\n",
       "2133                                bioconductor-ocplus              3\n",
       "2134                                  bioconductor-oct4              3\n",
       "2135                                 bioconductor-odseq              8\n",
       "2136                                  bioconductor-ogsa            281\n",
       "2137                                 bioconductor-oligo            682\n",
       "2138                          bioconductor-oligoclasses            417\n",
       "2139                             bioconductor-oligodata              7\n",
       "2140                                  bioconductor-olin            282\n",
       "2141                               bioconductor-olingui              4\n",
       "2142                                 bioconductor-omadb              5\n",
       "2143                              bioconductor-omicade4            283\n",
       "2144                            bioconductor-omiccircos            283\n",
       "2145                             bioconductor-omicplotr              5\n",
       "2146                         bioconductor-omicrexposome              6\n",
       "2147                            bioconductor-omicslonda              2\n",
       "2148                           bioconductor-omicsmarker              5\n",
       "2149                              bioconductor-omicspca              1\n",
       "2150                          bioconductor-omicspcadata              3\n",
       "2151                            bioconductor-omicsprint              9\n",
       "2152                             bioconductor-omnipathr              1\n",
       "2153                               bioconductor-onassis              5\n",
       "2154                       bioconductor-onassisjavalibs              7\n",
       "2155                               bioconductor-oncomix              7\n",
       "2156                             bioconductor-oncoscore              9\n",
       "2157                            bioconductor-oncosimulr            473\n",
       "2158                              bioconductor-onesense              7\n",
       "2159                             bioconductor-onlinefdr              8\n",
       "2160                              bioconductor-ontoproc              6\n",
       "2161                              bioconductor-opencyto            308\n",
       "2162                            bioconductor-openprimer              8\n",
       "2163                          bioconductor-openprimerui              6\n",
       "2164                      bioconductor-operonhumanv3.db              3\n",
       "2165                               bioconductor-opossom            284\n",
       "2166                                 bioconductor-oppar            555\n",
       "2167                                 bioconductor-oppti              1\n",
       "2168                              bioconductor-opweight              5\n",
       "2169                           bioconductor-orderedlist            195\n",
       "2170                                 bioconductor-orfik            282\n",
       "2171                          bioconductor-org.ag.eg.db             12\n",
       "2172                           bioconductor-organismdbi            678\n",
       "2173                        bioconductor-organism.dplyr              7\n",
       "2174                        bioconductor-org.at.tair.db             38\n",
       "2175                          bioconductor-org.bt.eg.db            392\n",
       "2176                          bioconductor-org.ce.eg.db            415\n",
       "2177                          bioconductor-org.cf.eg.db            385\n",
       "2178                          bioconductor-org.dm.eg.db            718\n",
       "2179                          bioconductor-org.dr.eg.db            459\n",
       "2180                       bioconductor-org.eck12.eg.db             19\n",
       "2181                     bioconductor-org.ecsakai.eg.db             12\n",
       "2182                          bioconductor-org.gg.eg.db            404\n",
       "2183                          bioconductor-org.hs.eg.db           1190\n",
       "2184                          bioconductor-org.mm.eg.db           1033\n",
       "2185                         bioconductor-org.mmu.eg.db             51\n",
       "2186                       bioconductor-org.mxanthus.db              1\n",
       "2187                      bioconductor-org.pf.plasmo.db             16\n",
       "2188                          bioconductor-org.pt.eg.db             12\n",
       "2189                          bioconductor-org.rn.eg.db            420\n",
       "2190                         bioconductor-org.sc.sgd.db            208\n",
       "2191                          bioconductor-org.ss.eg.db            397\n",
       "2192                          bioconductor-org.xl.eg.db             26\n",
       "2193                                  bioconductor-osat            294\n",
       "2194                                bioconductor-oscope            295\n",
       "2195                               bioconductor-otubase            281\n",
       "2196                              bioconductor-outlierd            288\n",
       "2197                              bioconductor-outrider            378\n",
       "2198                                bioconductor-oveseg            184\n",
       "2199                                   bioconductor-paa            278\n",
       "2200                                 bioconductor-padog            210\n",
       "2201                             bioconductor-paeg1acdf              3\n",
       "2202                           bioconductor-paeg1aprobe              4\n",
       "2203                             bioconductor-pairadise              3\n",
       "2204                           bioconductor-paircompviz            281\n",
       "2205                                bioconductor-pandar            198\n",
       "2206                          bioconductor-panelcn.mops             11\n",
       "2207                                  bioconductor-panp            290\n",
       "2208                                  bioconductor-panr            199\n",
       "2209                            bioconductor-panther.db            108\n",
       "2210                       bioconductor-panvizgenerator              4\n",
       "2211                         bioconductor-parathyroidse              3\n",
       "2212                               bioconductor-parglms              3\n",
       "2213                                bioconductor-parody            291\n",
       "2214                   bioconductor-partheenmetadata.db              3\n",
       "2215                               bioconductor-pasilla              6\n",
       "2216                      bioconductor-pasillabamsubset             11\n",
       "2217                 bioconductor-pasillatranscriptexpr              4\n",
       "2218                                  bioconductor-past              2\n",
       "2219                              bioconductor-path2ppi            286\n",
       "2220                             bioconductor-pathifier            198\n",
       "2221                               bioconductor-pathnet            283\n",
       "2222                           bioconductor-pathnetdata              4\n",
       "2223                             bioconductor-pathostat            195\n",
       "2224                             bioconductor-pathprint            192\n",
       "2225                      bioconductor-pathprintgeodata              8\n",
       "2226                            bioconductor-pathrender            188\n",
       "2227                               bioconductor-pathvar            189\n",
       "2228                              bioconductor-pathview            516\n",
       "2229                            bioconductor-pathwaypca              4\n",
       "2230                         bioconductor-pathwaysplice              6\n",
       "2231                             bioconductor-paxtoolsr              4\n",
       "2232                           bioconductor-pcaexplorer            396\n",
       "2233                 bioconductor-pcagopromoter.hs.hg19              4\n",
       "2234                  bioconductor-pcagopromoter.mm.mm9              4\n",
       "2235                  bioconductor-pcagopromoter.rn.rn4             11\n",
       "2236                            bioconductor-pcamethods            981\n",
       "2237                                  bioconductor-pcan            291\n",
       "2238                              bioconductor-pcatools            105\n",
       "2239                             bioconductor-pchicdata            385\n",
       "2240                                 bioconductor-pcot2            280\n",
       "2241                               bioconductor-pcpheno            188\n",
       "2242                                  bioconductor-pcxn              8\n",
       "2243                              bioconductor-pcxndata              9\n",
       "2244     bioconductor-pd.081229.hg18.promoter.medip.hx1              3\n",
       "2245    bioconductor-pd.2006.07.18.hg18.refseq.promoter              3\n",
       "2246     bioconductor-pd.2006.07.18.mm8.refseq.promoter              3\n",
       "2247    bioconductor-pd.2006.10.31.rn34.refseq.promoter              3\n",
       "2248                                 bioconductor-pd.ag              4\n",
       "2249                     bioconductor-pd.aragene.1.0.st              3\n",
       "2250                     bioconductor-pd.aragene.1.1.st              3\n",
       "2251                    bioconductor-pd.atdschip.tiling              4\n",
       "2252                        bioconductor-pd.ath1.121501              9\n",
       "2253                            bioconductor-pd.barley1              3\n",
       "2254                     bioconductor-pd.bovgene.1.0.st              4\n",
       "2255                     bioconductor-pd.bovgene.1.1.st              3\n",
       "2256                             bioconductor-pd.bovine              3\n",
       "2257                          bioconductor-pd.bsubtilis              3\n",
       "2258                     bioconductor-pd.cangene.1.0.st              3\n",
       "2259                     bioconductor-pd.cangene.1.1.st              3\n",
       "2260                           bioconductor-pd.canine.2              3\n",
       "2261                             bioconductor-pd.canine              4\n",
       "2262                           bioconductor-pd.celegans              3\n",
       "2263                 bioconductor-pd.charm.hg18.example              6\n",
       "2264                            bioconductor-pd.chicken              7\n",
       "2265                     bioconductor-pd.chigene.1.0.st              3\n",
       "2266                     bioconductor-pd.chigene.1.1.st              3\n",
       "2267                     bioconductor-pd.chogene.2.0.st              3\n",
       "2268                     bioconductor-pd.chogene.2.1.st              3\n",
       "2269                             bioconductor-pd.citrus              3\n",
       "2270                    bioconductor-pd.clariom.d.human              8\n",
       "2271                    bioconductor-pd.clariom.s.human              6\n",
       "2272                 bioconductor-pd.clariom.s.human.ht              8\n",
       "2273                    bioconductor-pd.clariom.s.mouse              8\n",
       "2274                 bioconductor-pd.clariom.s.mouse.ht              8\n",
       "2275                      bioconductor-pd.clariom.s.rat              6\n",
       "2276                   bioconductor-pd.clariom.s.rat.ht              7\n",
       "2277                             bioconductor-pd.cotton              3\n",
       "2278                     bioconductor-pd.cyngene.1.0.st              3\n",
       "2279                     bioconductor-pd.cyngene.1.1.st              3\n",
       "2280                     bioconductor-pd.cyrgene.1.0.st              3\n",
       "2281                     bioconductor-pd.cyrgene.1.1.st              3\n",
       "2282                 bioconductor-pd.cytogenetics.array              7\n",
       "2283                     bioconductor-pd.drogene.1.0.st              4\n",
       "2284                     bioconductor-pd.drogene.1.1.st              3\n",
       "2285                        bioconductor-pd.drosgenome1              3\n",
       "2286                       bioconductor-pd.drosophila.2              4\n",
       "2287                           bioconductor-pd.e.coli.2              4\n",
       "2288                              bioconductor-pd.ecoli              3\n",
       "2289                         bioconductor-pd.ecoli.asv2              8\n",
       "2290                     bioconductor-pd.elegene.1.0.st              3\n",
       "2291                     bioconductor-pd.elegene.1.1.st              3\n",
       "2292                     bioconductor-pd.equgene.1.0.st              3\n",
       "2293                     bioconductor-pd.equgene.1.1.st              3\n",
       "2294               bioconductor-pd.feinberg.hg18.me.hx1              3\n",
       "2295                bioconductor-pd.feinberg.mm8.me.hx1              3\n",
       "2296                     bioconductor-pd.felgene.1.0.st              3\n",
       "2297                     bioconductor-pd.felgene.1.1.st              3\n",
       "2298                     bioconductor-pd.fingene.1.0.st              3\n",
       "2299                     bioconductor-pd.fingene.1.1.st              3\n",
       "2300                    bioconductor-pd.genomewidesnp.5              4\n",
       "2301                    bioconductor-pd.genomewidesnp.6              4\n",
       "2302                     bioconductor-pd.guigene.1.0.st              3\n",
       "2303                     bioconductor-pd.guigene.1.1.st              3\n",
       "2304                            bioconductor-pd.hc.g110              3\n",
       "2305                    bioconductor-pd.hg18.60mer.expr              3\n",
       "2306                           bioconductor-pd.hg.focus              7\n",
       "2307                         bioconductor-pd.hg.u133a.2              7\n",
       "2308                           bioconductor-pd.hg.u133a              8\n",
       "2309                       bioconductor-pd.hg.u133a.tag              3\n",
       "2310                           bioconductor-pd.hg.u133b              3\n",
       "2311                     bioconductor-pd.hg.u133.plus.2              3\n",
       "2312                            bioconductor-pd.hg.u219              3\n",
       "2313                            bioconductor-pd.hg.u95a              3\n",
       "2314                          bioconductor-pd.hg.u95av2              3\n",
       "2315                            bioconductor-pd.hg.u95b              3\n",
       "2316                            bioconductor-pd.hg.u95c              3\n",
       "2317                            bioconductor-pd.hg.u95d              3\n",
       "2318                            bioconductor-pd.hg.u95e              3\n",
       "2319                            bioconductor-pd.hta.2.0              3\n",
       "2320                        bioconductor-pd.ht.hg.u133a              6\n",
       "2321                 bioconductor-pd.ht.hg.u133.plus.pm              6\n",
       "2322                         bioconductor-pd.ht.mg.430a              3\n",
       "2323                             bioconductor-pd.hu6800              3\n",
       "2324                     bioconductor-pd.huex.1.0.st.v2              8\n",
       "2325                   bioconductor-pd.hugene.1.0.st.v1              9\n",
       "2326                   bioconductor-pd.hugene.1.1.st.v1             10\n",
       "2327                      bioconductor-pd.hugene.2.0.st              8\n",
       "2328                      bioconductor-pd.hugene.2.1.st              3\n",
       "2329                         bioconductor-pdinfobuilder            466\n",
       "2330                              bioconductor-pd.maize              3\n",
       "2331                    bioconductor-pd.mapping250k.nsp              5\n",
       "2332                    bioconductor-pd.mapping250k.sty              7\n",
       "2333                 bioconductor-pd.mapping50k.hind240              3\n",
       "2334                  bioconductor-pd.mapping50k.xba240              3\n",
       "2335                     bioconductor-pd.margene.1.0.st              3\n",
       "2336                     bioconductor-pd.margene.1.1.st              3\n",
       "2337                     bioconductor-pd.medgene.1.0.st              3\n",
       "2338                     bioconductor-pd.medgene.1.1.st              6\n",
       "2339                           bioconductor-pd.medicago              4\n",
       "2340                            bioconductor-pd.mg.u74a              3\n",
       "2341                          bioconductor-pd.mg.u74av2              3\n",
       "2342                            bioconductor-pd.mg.u74b              3\n",
       "2343                          bioconductor-pd.mg.u74bv2              4\n",
       "2344                            bioconductor-pd.mg.u74c              3\n",
       "2345                          bioconductor-pd.mg.u74cv2              3\n",
       "2346                          bioconductor-pd.mirna.1.0              3\n",
       "2347                          bioconductor-pd.mirna.2.0              3\n",
       "2348                          bioconductor-pd.mirna.3.0              4\n",
       "2349                          bioconductor-pd.mirna.3.1              3\n",
       "2350                          bioconductor-pd.mirna.4.0              3\n",
       "2351                            bioconductor-pd.moe430a              4\n",
       "2352                            bioconductor-pd.moe430b              3\n",
       "2353                     bioconductor-pd.moex.1.0.st.v1              8\n",
       "2354                   bioconductor-pd.mogene.1.0.st.v1              7\n",
       "2355                   bioconductor-pd.mogene.1.1.st.v1              9\n",
       "2356                      bioconductor-pd.mogene.2.0.st              5\n",
       "2357                      bioconductor-pd.mogene.2.1.st              7\n",
       "2358                         bioconductor-pd.mouse430.2              3\n",
       "2359                        bioconductor-pd.mouse430a.2              3\n",
       "2360                            bioconductor-pd.mta.1.0              7\n",
       "2361                          bioconductor-pd.mu11ksuba              3\n",
       "2362                          bioconductor-pd.mu11ksubb              3\n",
       "2363                    bioconductor-pd.nugo.hs1a520180              4\n",
       "2364                    bioconductor-pd.nugo.mm1a520177              3\n",
       "2365                     bioconductor-pd.ovigene.1.0.st              3\n",
       "2366                     bioconductor-pd.ovigene.1.1.st              3\n",
       "2367                            bioconductor-pd.pae.g1a              3\n",
       "2368               bioconductor-pd.plasmodium.anopheles              3\n",
       "2369                             bioconductor-pd.poplar              7\n",
       "2370                            bioconductor-pd.porcine              3\n",
       "2371                     bioconductor-pd.porgene.1.0.st              3\n",
       "2372                     bioconductor-pd.porgene.1.1.st              3\n",
       "2373                     bioconductor-pd.rabgene.1.0.st              3\n",
       "2374                     bioconductor-pd.rabgene.1.1.st              3\n",
       "2375                            bioconductor-pd.rae230a              5\n",
       "2376                            bioconductor-pd.rae230b              6\n",
       "2377                     bioconductor-pd.raex.1.0.st.v1              3\n",
       "2378                   bioconductor-pd.ragene.1.0.st.v1              5\n",
       "2379                   bioconductor-pd.ragene.1.1.st.v1              4\n",
       "2380                      bioconductor-pd.ragene.2.0.st              4\n",
       "2381                      bioconductor-pd.ragene.2.1.st              3\n",
       "2382                           bioconductor-pd.rat230.2              3\n",
       "2383                     bioconductor-pd.rcngene.1.0.st              3\n",
       "2384                     bioconductor-pd.rcngene.1.1.st              4\n",
       "2385                            bioconductor-pd.rg.u34a              3\n",
       "2386                            bioconductor-pd.rg.u34b              3\n",
       "2387                            bioconductor-pd.rg.u34c              3\n",
       "2388                     bioconductor-pd.rhegene.1.0.st              3\n",
       "2389                     bioconductor-pd.rhegene.1.1.st              3\n",
       "2390                             bioconductor-pd.rhesus              3\n",
       "2391                               bioconductor-pd.rice              8\n",
       "2392                     bioconductor-pd.rjpgene.1.0.st              4\n",
       "2393                     bioconductor-pd.rjpgene.1.1.st              3\n",
       "2394                             bioconductor-pd.rn.u34              3\n",
       "2395                            bioconductor-pd.rta.1.0              8\n",
       "2396                     bioconductor-pd.rusgene.1.0.st              3\n",
       "2397                     bioconductor-pd.rusgene.1.1.st              3\n",
       "2398                           bioconductor-pd.s.aureus              4\n",
       "2399                            bioconductor-pd.soybean              3\n",
       "2400                     bioconductor-pd.soygene.1.0.st              3\n",
       "2401                     bioconductor-pd.soygene.1.1.st              3\n",
       "2402                         bioconductor-pd.sugar.cane              7\n",
       "2403                             bioconductor-pd.tomato              6\n",
       "2404                           bioconductor-pd.u133.x3p              4\n",
       "2405                     bioconductor-pd.vitis.vinifera              3\n",
       "2406                              bioconductor-pd.wheat              4\n",
       "2407                     bioconductor-pd.xenopus.laevis              3\n",
       "2408                         bioconductor-pd.x.laevis.2              3\n",
       "2409                       bioconductor-pd.x.tropicalis              6\n",
       "2410                            bioconductor-pd.yeast.2              3\n",
       "2411                             bioconductor-pd.yg.s98              3\n",
       "2412                     bioconductor-pd.zebgene.1.0.st              3\n",
       "2413                     bioconductor-pd.zebgene.1.1.st              4\n",
       "2414                          bioconductor-pd.zebrafish              4\n",
       "2415                           bioconductor-peakpanther              1\n",
       "2416                                  bioconductor-peca            281\n",
       "2417                       bioconductor-pedbarrayv10.db              3\n",
       "2418                        bioconductor-pedbarrayv9.db              3\n",
       "2419                                bioconductor-pepdat              3\n",
       "2420                               bioconductor-pepsnmr              4\n",
       "2421                           bioconductor-pepsnmrdata              4\n",
       "2422                               bioconductor-pepstat            282\n",
       "2423                             bioconductor-pepxmltab            287\n",
       "2424                               bioconductor-perfect              1\n",
       "2425                            bioconductor-perturbatr              4\n",
       "2426                               bioconductor-pfam.db            203\n",
       "2427                                   bioconductor-pga              8\n",
       "2428                                  bioconductor-pgca              8\n",
       "2429                                 bioconductor-pgsea            372\n",
       "2430                             bioconductor-phantasus              9\n",
       "2431                            bioconductor-pharmacogx              4\n",
       "2432             bioconductor-phastcons100way.ucsc.hg19            192\n",
       "2433             bioconductor-phastcons100way.ucsc.hg38              3\n",
       "2434               bioconductor-phastcons7way.ucsc.hg38              7\n",
       "2435                                 bioconductor-phemd              2\n",
       "2436                             bioconductor-phenopath            283\n",
       "2437                             bioconductor-phenotest              3\n",
       "2438                              bioconductor-phenstat            186\n",
       "2439                                 bioconductor-philr            199\n",
       "2440                     bioconductor-phosphonormalizer            192\n",
       "2441                          bioconductor-phyloprofile              1\n",
       "2442                      bioconductor-phyloprofiledata              2\n",
       "2443                              bioconductor-phyloseq            699\n",
       "2444                                    bioconductor-pi              6\n",
       "2445                                 bioconductor-piano            382\n",
       "2446                              bioconductor-pickgene            282\n",
       "2447                                  bioconductor-pics            574\n",
       "2448                               bioconductor-pig.db0              3\n",
       "2449                             bioconductor-pigengene              3\n",
       "2450                                  bioconductor-ping            561\n",
       "2451                             bioconductor-pipeframe              2\n",
       "2452                           bioconductor-pkgdeptools            281\n",
       "2453                               bioconductor-plasfia              3\n",
       "2454                bioconductor-plasmodiumanophelescdf              3\n",
       "2455              bioconductor-plasmodiumanophelesprobe              4\n",
       "2456                                bioconductor-plethy            282\n",
       "2457                                 bioconductor-plgem            282\n",
       "2458                                 bioconductor-plier            572\n",
       "2459                           bioconductor-plotgrouper              4\n",
       "2460                                  bioconductor-plpe            198\n",
       "2461                                  bioconductor-plrs              6\n",
       "2462                                   bioconductor-plw            571\n",
       "2463                             bioconductor-plyranges             78\n",
       "2464                                   bioconductor-pmm            192\n",
       "2465                    bioconductor-pocrcannotation.db              3\n",
       "2466                                bioconductor-podkat            555\n",
       "2467                                 bioconductor-pogos              3\n",
       "2468                             bioconductor-polyester             20\n",
       "2469                               bioconductor-polyfit            282\n",
       "2470            bioconductor-polyphen.hsapiens.dbsnp131              7\n",
       "2471                             bioconductor-poplarcdf              3\n",
       "2472                           bioconductor-poplarprobe              3\n",
       "2473                            bioconductor-porcinecdf              3\n",
       "2474                            bioconductor-porcine.db              4\n",
       "2475                          bioconductor-porcineprobe              3\n",
       "2476                                  bioconductor-post              5\n",
       "2477                                 bioconductor-potra              3\n",
       "2478                         bioconductor-powerexplorer              4\n",
       "2479                              bioconductor-powertcr              4\n",
       "2480                               bioconductor-ppidata            188\n",
       "2481                               bioconductor-ppinfer            289\n",
       "2482                              bioconductor-ppistats            196\n",
       "2483                             bioconductor-pqsfinder            478\n",
       "2484                                 bioconductor-prada            681\n",
       "2485                                  bioconductor-pram              2\n",
       "2486                                 bioconductor-prebs            193\n",
       "2487                             bioconductor-prebsdata              4\n",
       "2488                  bioconductor-precisiontrialdrawer              2\n",
       "2489                                 bioconductor-preda            193\n",
       "2490                       bioconductor-predasampledata              3\n",
       "2491                          bioconductor-predictionet            276\n",
       "2492                        bioconductor-preprocesscore           1617\n",
       "2493                          bioconductor-primeviewcdf              3\n",
       "2494                        bioconductor-primeviewprobe              6\n",
       "2495                             bioconductor-primirtss              5\n",
       "2496                                bioconductor-prince              2\n",
       "2497                                 bioconductor-prize            189\n",
       "2498                               bioconductor-probamr            374\n",
       "2499                              bioconductor-probatch              2\n",
       "2500                               bioconductor-process            189\n",
       "2501                               bioconductor-procoil            199\n",
       "2502                                 bioconductor-proda              1\n",
       "2503                               bioconductor-prodata              6\n",
       "2504                                bioconductor-profia            558\n",
       "2505                           bioconductor-profileplyr              4\n",
       "2506                      bioconductor-profilescoredist            276\n",
       "2507                               bioconductor-progeny              5\n",
       "2508                              bioconductor-projectr              5\n",
       "2509                                bioconductor-proloc            471\n",
       "2510                            bioconductor-prolocdata              3\n",
       "2511                             bioconductor-prolocgui            102\n",
       "2512                               bioconductor-promise            281\n",
       "2513                                bioconductor-proper            297\n",
       "2514                                 bioconductor-props              3\n",
       "2515                               bioconductor-prostar              5\n",
       "2516                  bioconductor-prostatecancercamcap              3\n",
       "2517                  bioconductor-prostatecancergrasso              3\n",
       "2518               bioconductor-prostatecancerstockholm              3\n",
       "2519                  bioconductor-prostatecancertaylor              3\n",
       "2520              bioconductor-prostatecancervarambally              6\n",
       "2521                       bioconductor-proteinprofiles              4\n",
       "2522           bioconductor-proteomicsannotationhubdata             97\n",
       "2523                              bioconductor-proteomm              3\n",
       "2524                          bioconductor-protgenerics           1300\n",
       "2525                                  bioconductor-psea            295\n",
       "2526                            bioconductor-psichomics            283\n",
       "2527                              bioconductor-psicquic            337\n",
       "2528                            bioconductor-psygenet2r            195\n",
       "2529                          bioconductor-pth2o2lipids              6\n",
       "2530                           bioconductor-pulsedsilac              1\n",
       "2531                                  bioconductor-puma            469\n",
       "2532                              bioconductor-pumadata              3\n",
       "2533                                bioconductor-purecn            609\n",
       "2534                                  bioconductor-pvac            288\n",
       "2535                                  bioconductor-pvca            197\n",
       "2536                                  bioconductor-pviz            286\n",
       "2537                             bioconductor-pwmenrich            197\n",
       "2538    bioconductor-pwmenrich.dmelanogaster.background              3\n",
       "2539         bioconductor-pwmenrich.hsapiens.background              3\n",
       "2540        bioconductor-pwmenrich.mmusculus.background              3\n",
       "2541                               bioconductor-pwomics            193\n",
       "2542                               bioconductor-pwrewas              2\n",
       "2543                          bioconductor-pwrewas.data              1\n",
       "2544                            bioconductor-qckitfastq            185\n",
       "2545                             bioconductor-qcmetrics            189\n",
       "2546                               bioconductor-qdnaseq            523\n",
       "2547                          bioconductor-qdnaseq.hg19             13\n",
       "2548                          bioconductor-qdnaseq.mm10            389\n",
       "2549                              bioconductor-qpcrnorm            292\n",
       "2550                               bioconductor-qpgraph            474\n",
       "2551                         bioconductor-qplexanalyzer              3\n",
       "2552                             bioconductor-qplexdata              3\n",
       "2553                                  bioconductor-qrqc            567\n",
       "2554                                  bioconductor-qsea            471\n",
       "2555                               bioconductor-qsmooth              3\n",
       "2556                               bioconductor-qsutils              3\n",
       "2557                               bioconductor-qtlizer              1\n",
       "2558                             bioconductor-qualifier            280\n",
       "2559                               bioconductor-quantro            297\n",
       "2560                           bioconductor-quantsmooth            375\n",
       "2561                              bioconductor-quartpac            188\n",
       "2562                                 bioconductor-quasr            466\n",
       "2563                        bioconductor-quaternaryprod            482\n",
       "2564                                 bioconductor-qubic            295\n",
       "2565                             bioconductor-qubicdata              8\n",
       "2566                                bioconductor-qusage            300\n",
       "2567                                bioconductor-qvalue           1201\n",
       "2568                            bioconductor-r10kcod.db              3\n",
       "2569                                bioconductor-r3cpet            282\n",
       "2570                                bioconductor-r3cseq            284\n",
       "2571                      bioconductor-r453plus1toolbox            276\n",
       "2572                                 bioconductor-r4rna            280\n",
       "2573                            bioconductor-rae230acdf              4\n",
       "2574                            bioconductor-rae230a.db              3\n",
       "2575                          bioconductor-rae230aprobe              3\n",
       "2576                            bioconductor-rae230bcdf              3\n",
       "2577                            bioconductor-rae230b.db              3\n",
       "2578                          bioconductor-rae230bprobe              4\n",
       "2579                   bioconductor-raex10stprobeset.db              3\n",
       "2580          bioconductor-raex10sttranscriptcluster.db              3\n",
       "2581              bioconductor-raexexonprobesetlocation              3\n",
       "2582                 bioconductor-ragene10stprobeset.db              3\n",
       "2583        bioconductor-ragene10sttranscriptcluster.db              3\n",
       "2584                       bioconductor-ragene10stv1cdf              3\n",
       "2585                     bioconductor-ragene10stv1probe              3\n",
       "2586                 bioconductor-ragene11stprobeset.db              3\n",
       "2587        bioconductor-ragene11sttranscriptcluster.db              4\n",
       "2588                 bioconductor-ragene20stprobeset.db              3\n",
       "2589        bioconductor-ragene20sttranscriptcluster.db              3\n",
       "2590                 bioconductor-ragene21stprobeset.db              3\n",
       "2591        bioconductor-ragene21sttranscriptcluster.db              3\n",
       "2592                      bioconductor-raggedexperiment            195\n",
       "2593                                  bioconductor-rain            284\n",
       "2594                                  bioconductor-rama            556\n",
       "2595                                bioconductor-ramwas            277\n",
       "2596                   bioconductor-randomwalkrestartmh              7\n",
       "2597                              bioconductor-randpack            291\n",
       "2598                              bioconductor-rankprod            203\n",
       "2599                        bioconductor-rarevariantvis            194\n",
       "2600                               bioconductor-rariant            187\n",
       "2601                            bioconductor-rat2302cdf              4\n",
       "2602                            bioconductor-rat2302.db              4\n",
       "2603                       bioconductor-rat2302frmavecs              2\n",
       "2604                          bioconductor-rat2302probe              3\n",
       "2605                             bioconductor-ratchrloc              4\n",
       "2606                               bioconductor-rat.db0              3\n",
       "2607                           bioconductor-rattoxfxcdf              3\n",
       "2608                         bioconductor-rattoxfxprobe              4\n",
       "2609                     bioconductor-rattus.norvegicus              8\n",
       "2610                              bioconductor-rbcbook1            281\n",
       "2611                                  bioconductor-rbgl           1470\n",
       "2612                               bioconductor-rbioinf            563\n",
       "2613                         bioconductor-rbiopaxparser            284\n",
       "2614                                   bioconductor-rbm            296\n",
       "2615                              bioconductor-rbowtie2            287\n",
       "2616                               bioconductor-rbowtie            584\n",
       "2617                                bioconductor-rbsurv            293\n",
       "2618                                 bioconductor-rcade              6\n",
       "2619                                  bioconductor-rcas            658\n",
       "2620                               bioconductor-rcaspar            282\n",
       "2621                            bioconductor-rcellminer              3\n",
       "2622                        bioconductor-rcellminerdata              4\n",
       "2623                                  bioconductor-rcgh            193\n",
       "2624                           bioconductor-rchyoptimyx            276\n",
       "2625                            bioconductor-rcistarget             13\n",
       "2626  bioconductor-rcistarget.hg19.motifdbs.cisbponl...              9\n",
       "2627                                   bioconductor-rcm              2\n",
       "2628                                  bioconductor-rcpi              7\n",
       "2629                                  bioconductor-rcy3            203\n",
       "2630                                 bioconductor-rcyjs              6\n",
       "2631                      bioconductor-rdavidwebservice             19\n",
       "2632                                bioconductor-rdgidb            282\n",
       "2633                                bioconductor-rdisop            367\n",
       "2634                            bioconductor-rdrtoolbox              8\n",
       "2635                           bioconductor-reactome.db            330\n",
       "2636                           bioconductor-reactomegsa              1\n",
       "2637                      bioconductor-reactomegsa.data              1\n",
       "2638                            bioconductor-reactomepa            266\n",
       "2639                              bioconductor-readqpcr            280\n",
       "2640                                   bioconductor-reb            276\n",
       "2641                                 bioconductor-rebet            288\n",
       "2642                               bioconductor-recount            317\n",
       "2643                                bioconductor-recoup            281\n",
       "2644                                 bioconductor-reder            195\n",
       "2645                                bioconductor-redseq              3\n",
       "2646                                bioconductor-refnet            279\n",
       "2647                               bioconductor-refplus            197\n",
       "2648                              bioconductor-regioner            764\n",
       "2649                          bioconductor-regionreport             60\n",
       "2650                           bioconductor-regparallel              4\n",
       "2651                             bioconductor-regsplice            193\n",
       "2652                                  bioconductor-remp              4\n",
       "2653                             bioconductor-repitools            673\n",
       "2654                        bioconductor-reportingtools            200\n",
       "2655                                bioconductor-repviz              2\n",
       "2656                                 bioconductor-reqon              6\n",
       "2657                             bioconductor-restfulse              3\n",
       "2658                         bioconductor-restfulsedata              6\n",
       "2659                             bioconductor-rexposome              3\n",
       "2660                        bioconductor-rforproteomics             10\n",
       "2661                                bioconductor-rfpred            554\n",
       "2662                                bioconductor-rgadem            657\n",
       "2663                               bioconductor-rgalaxy            384\n",
       "2664                                  bioconductor-rgin            283\n",
       "2665                                 bioconductor-rgmql              6\n",
       "2666                              bioconductor-rgmqllib              5\n",
       "2667                             bioconductor-rgraph2js            281\n",
       "2668                             bioconductor-rgraphviz           1163\n",
       "2669                                bioconductor-rgreat            306\n",
       "2670                                 bioconductor-rgsea            283\n",
       "2671                                bioconductor-rgsepd            298\n",
       "2672                             bioconductor-rgu34acdf              3\n",
       "2673                             bioconductor-rgu34a.db              3\n",
       "2674                           bioconductor-rgu34aprobe              3\n",
       "2675                             bioconductor-rgu34bcdf              3\n",
       "2676                             bioconductor-rgu34b.db              3\n",
       "2677                           bioconductor-rgu34bprobe              3\n",
       "2678                             bioconductor-rgu34ccdf              3\n",
       "2679                             bioconductor-rgu34c.db              3\n",
       "2680                           bioconductor-rgu34cprobe              3\n",
       "2681                         bioconductor-rguatlas4k.db              4\n",
       "2682                          bioconductor-rgug4105a.db              3\n",
       "2683                          bioconductor-rgug4130a.db              3\n",
       "2684                          bioconductor-rgug4131a.db              4\n",
       "2685                                 bioconductor-rhdf5           2665\n",
       "2686                           bioconductor-rhdf5client            368\n",
       "2687                              bioconductor-rhdf5lib           2426\n",
       "2688                             bioconductor-rhesuscdf              3\n",
       "2689                            bioconductor-rhesus.db0              3\n",
       "2690                           bioconductor-rhesusprobe              3\n",
       "2691            bioconductor-rheumaticconditionwollbold              4\n",
       "2692                               bioconductor-rhisat2            184\n",
       "2693                               bioconductor-rhtslib           7232\n",
       "2694                            bioconductor-ri16cod.db              3\n",
       "2695                         bioconductor-riboprofiling            287\n",
       "2696                              bioconductor-riboseqr            281\n",
       "2697      bioconductor-ribosomaldatabaseproject11.5mgdb              3\n",
       "2698                               bioconductor-ricecdf              3\n",
       "2699                             bioconductor-riceprobe              3\n",
       "2700                              bioconductor-rimmport            282\n",
       "2701                                 bioconductor-ringo            676\n",
       "2702                                  bioconductor-risa            479\n",
       "2703                                 bioconductor-ritan              9\n",
       "2704                             bioconductor-ritandata              5\n",
       "2705                                 bioconductor-river              3\n",
       "2706                     bioconductor-rjmcmcnucleosomes            276\n",
       "2707                                  bioconductor-rlmm            301\n",
       "2708                               bioconductor-rmagpie              3\n",
       "2709                             bioconductor-rmassbank            277\n",
       "2710                         bioconductor-rmassbankdata              4\n",
       "2711                              bioconductor-rmelting              3\n",
       "2712                                  bioconductor-rmir            189\n",
       "2713                              bioconductor-rmir.hsa              3\n",
       "2714                         bioconductor-rmir.hs.mirna            101\n",
       "2715                              bioconductor-rmmquant            280\n",
       "2716                              bioconductor-rnadecay            369\n",
       "2717              bioconductor-rnagilentdesign028282.db              8\n",
       "2718                           bioconductor-rnainteract            191\n",
       "2719                       bioconductor-rnainteractmapk              3\n",
       "2720                              bioconductor-rnaither            188\n",
       "2721                               bioconductor-rnamodr              1\n",
       "2722                 bioconductor-rnamodr.alkanilineseq              1\n",
       "2723                          bioconductor-rnamodr.data              1\n",
       "2724                            bioconductor-rnamodr.ml              1\n",
       "2725                   bioconductor-rnamodr.ribomethseq              1\n",
       "2726                              bioconductor-rnaprobr            281\n",
       "2727                              bioconductor-rnasense              1\n",
       "2728                            bioconductor-rnaseqcomp            296\n",
       "2729           bioconductor-rnaseqdata.hnrnpc.bam.chr14              5\n",
       "2730                             bioconductor-rnaseqmap            554\n",
       "2731                           bioconductor-rnaseqpower            384\n",
       "2732                               bioconductor-rnaseqr              7\n",
       "2733                           bioconductor-rnaseqrdata              4\n",
       "2734                  bioconductor-rnaseqsamplesizedata             97\n",
       "2735                               bioconductor-rnbeads            204\n",
       "2736                          bioconductor-rnbeads.hg19            199\n",
       "2737                          bioconductor-rnbeads.hg38             10\n",
       "2738                          bioconductor-rnbeads.mm10            198\n",
       "2739                           bioconductor-rnbeads.mm9            189\n",
       "2740                           bioconductor-rnbeads.rn5            189\n",
       "2741                                 bioconductor-rnits            375\n",
       "2742                              bioconductor-rnu34cdf              3\n",
       "2743                              bioconductor-rnu34.db              4\n",
       "2744                            bioconductor-rnu34probe              3\n",
       "2745                                  bioconductor-roar            374\n",
       "2746              bioconductor-roberts2005annotation.db              3\n",
       "2747                                   bioconductor-roc            577\n",
       "2748                            bioconductor-roleswitch            297\n",
       "2749                                  bioconductor-rols            282\n",
       "2750                            bioconductor-rontotools            297\n",
       "2751                                 bioconductor-ropls            814\n",
       "2752                                  bioconductor-rots            751\n",
       "2753                                   bioconductor-rpa            288\n",
       "2754                          bioconductor-rprotobuflib            496\n",
       "2755                               bioconductor-rpsixml            189\n",
       "2756                                   bioconductor-rpx            297\n",
       "2757                                   bioconductor-rqc            287\n",
       "2758                                   bioconductor-rqt              5\n",
       "2759                                bioconductor-rqubic            276\n",
       "2760                              bioconductor-rrbsdata              3\n",
       "2761                                  bioconductor-rrdp            281\n",
       "2762                              bioconductor-rrdpdata              3\n",
       "2763                                  bioconductor-rrho            286\n",
       "2764                             bioconductor-rsamtools          10490\n",
       "2765                                bioconductor-rscudo              2\n",
       "2766                                bioconductor-rseqan            189\n",
       "2767                              bioconductor-rsubread           1437\n",
       "2768                                bioconductor-rsvsim            281\n",
       "2769                      bioconductor-rta10probeset.db              3\n",
       "2770             bioconductor-rta10transcriptcluster.db              3\n",
       "2771                               bioconductor-rtandem            386\n",
       "2772                                  bioconductor-rtca            282\n",
       "2773                                 bioconductor-rtcga            204\n",
       "2774                        bioconductor-rtcga.clinical              8\n",
       "2775                             bioconductor-rtcga.cnv             15\n",
       "2776                     bioconductor-rtcga.methylation              4\n",
       "2777                        bioconductor-rtcga.mirnaseq             10\n",
       "2778                            bioconductor-rtcga.mrna            192\n",
       "2779                       bioconductor-rtcga.mutations              5\n",
       "2780                        bioconductor-rtcga.pancan12             10\n",
       "2781                          bioconductor-rtcga.rnaseq              5\n",
       "2782                            bioconductor-rtcga.rppa              8\n",
       "2783                          bioconductor-rtcgatoolbox            287\n",
       "2784                                   bioconductor-rtn              8\n",
       "2785                              bioconductor-rtnduals              3\n",
       "2786                           bioconductor-rtnsurvival              3\n",
       "2787                               bioconductor-rtopper            295\n",
       "2788                           bioconductor-rtracklayer           8090\n",
       "2789                              bioconductor-rtreemix            566\n",
       "2790                                  bioconductor-rtrm            292\n",
       "2791                                bioconductor-rtrmui            284\n",
       "2792                              bioconductor-rtu34cdf              3\n",
       "2793                              bioconductor-rtu34.db              3\n",
       "2794                            bioconductor-rtu34probe              3\n",
       "2795                               bioconductor-runibic            282\n",
       "2796                               bioconductor-ruvcorr              4\n",
       "2797                          bioconductor-ruvnormalize            196\n",
       "2798                      bioconductor-ruvnormalizedata            196\n",
       "2799                                bioconductor-ruvseq            293\n",
       "2800                                   bioconductor-rvs              3\n",
       "2801                             bioconductor-rwgcod.db              3\n",
       "2802                         bioconductor-rwikipathways              6\n",
       "2803                             bioconductor-s4vectors          13820\n",
       "2804                                  bioconductor-safe            289\n",
       "2805                             bioconductor-sagenhaft            282\n",
       "2806                                  bioconductor-sagx            557\n",
       "2807                              bioconductor-saigegds             92\n",
       "2808                           bioconductor-samexplorer              7\n",
       "2809                      bioconductor-sampleclassifier              3\n",
       "2810                  bioconductor-sampleclassifierdata              7\n",
       "2811                           bioconductor-samspectral            375\n",
       "2812                            bioconductor-sangerseqr            297\n",
       "2813                             bioconductor-sapfinder            385\n",
       "2814                            bioconductor-saureuscdf              3\n",
       "2815                          bioconductor-saureusprobe              3\n",
       "2816                                  bioconductor-savr            292\n",
       "2817                              bioconductor-sbgnview              1\n",
       "2818                         bioconductor-sbgnview.data              1\n",
       "2819                                 bioconductor-sbmlr              6\n",
       "2820                                   bioconductor-sc3            480\n",
       "2821                               bioconductor-scale4c              3\n",
       "2822                               bioconductor-scalign              4\n",
       "2823                              bioconductor-scan.upc            282\n",
       "2824                               bioconductor-scanvis              1\n",
       "2825                                bioconductor-scater           1055\n",
       "2826                                 bioconductor-scbfa              1\n",
       "2827                                  bioconductor-scbn              8\n",
       "2828                           bioconductor-scdblfinder              1\n",
       "2829                                  bioconductor-scdd              6\n",
       "2830                                  bioconductor-scde            480\n",
       "2831                                  bioconductor-scds              2\n",
       "2832                       bioconductor-scfeaturefilter              4\n",
       "2833                                bioconductor-scfind            276\n",
       "2834                                 bioconductor-scgps             92\n",
       "2835                                 bioconductor-schex              1\n",
       "2836                                 bioconductor-scisi            197\n",
       "2837                               bioconductor-sclcbam              4\n",
       "2838                                 bioconductor-scmap            287\n",
       "2839                               bioconductor-scmerge            190\n",
       "2840                                bioconductor-scmeth              3\n",
       "2841                                bioconductor-scnorm            209\n",
       "2842                                 bioconductor-scone              4\n",
       "2843                               bioconductor-sconify              3\n",
       "2844                           bioconductor-scoreinvhap              3\n",
       "2845                                 bioconductor-scpca              1\n",
       "2846                                bioconductor-scpipe            556\n",
       "2847                                 bioconductor-scran            954\n",
       "2848                             bioconductor-screcover              7\n",
       "2849                              bioconductor-scrnaseq            111\n",
       "2850                                bioconductor-scruff             10\n",
       "2851                                  bioconductor-scsr              7\n",
       "2852                              bioconductor-sctensor              2\n",
       "2853                                bioconductor-sctgif             92\n",
       "2854                                 bioconductor-sdams              6\n",
       "2855                            bioconductor-segmentseq            293\n",
       "2856                                 bioconductor-selex              6\n",
       "2857                               bioconductor-semdist            282\n",
       "2858                               bioconductor-semisup            281\n",
       "2859                                bioconductor-sepira              4\n",
       "2860                           bioconductor-seq2pathway            199\n",
       "2861                      bioconductor-seq2pathway.data            194\n",
       "2862                              bioconductor-seqarray            291\n",
       "2863                               bioconductor-seqbias            276\n",
       "2864                                  bioconductor-seqc              3\n",
       "2865                                bioconductor-seqcat              8\n",
       "2866                                bioconductor-seqcna            193\n",
       "2867                          bioconductor-seqcna.annot              4\n",
       "2868                              bioconductor-seqcombo              3\n",
       "2869                               bioconductor-seqgsea            288\n",
       "2870                               bioconductor-seqlogo            588\n",
       "2871                            bioconductor-seqpattern            576\n",
       "2872                              bioconductor-seqplots             10\n",
       "2873                             bioconductor-seqsetvis              3\n",
       "2874                                bioconductor-seqsqc              3\n",
       "2875                              bioconductor-seqtools            583\n",
       "2876                           bioconductor-seqvartools              6\n",
       "2877                      bioconductor-serumstimulation              4\n",
       "2878                                bioconductor-sesame              5\n",
       "2879                            bioconductor-sesamedata              4\n",
       "2880                               bioconductor-setools              1\n",
       "2881                          bioconductor-sevenbridges              9\n",
       "2882                                bioconductor-sevenc              6\n",
       "2883                       bioconductor-seventygenedata              4\n",
       "2884                                 bioconductor-sgseq            281\n",
       "2885                          bioconductor-sharedobject             92\n",
       "2886                               bioconductor-shdz.db              4\n",
       "2887                           bioconductor-shinymethyl            199\n",
       "2888                       bioconductor-shinymethyldata              4\n",
       "2889                           bioconductor-shinytandem             10\n",
       "2890                             bioconductor-shortread           3404\n",
       "2891                               bioconductor-siamcat              3\n",
       "2892                              bioconductor-sictools            273\n",
       "2893                bioconductor-sift.hsapiens.dbsnp132              3\n",
       "2894                bioconductor-sift.hsapiens.dbsnp137              6\n",
       "2895                                 bioconductor-sigar            188\n",
       "2896                              bioconductor-sigcheck              6\n",
       "2897                            bioconductor-sigfeature              7\n",
       "2898                               bioconductor-sigfuge            191\n",
       "2899                              bioconductor-siggenes            567\n",
       "2900                                bioconductor-sights            284\n",
       "2901                       bioconductor-signaturesearch             94\n",
       "2902                   bioconductor-signaturesearchdata              1\n",
       "2903                                bioconductor-signer            651\n",
       "2904                                bioconductor-signet              3\n",
       "2905                            bioconductor-sigpathway            562\n",
       "2906                              bioconductor-sigspack              1\n",
       "2907                            bioconductor-sigsquared              3\n",
       "2908                        bioconductor-silva128.1mgdb              3\n",
       "2909                                   bioconductor-sim            464\n",
       "2910                                 bioconductor-simat            368\n",
       "2911                       bioconductor-simbindprofiles            287\n",
       "2912                                  bioconductor-simd            287\n",
       "2913                           bioconductor-similarpeak            282\n",
       "2914                                 bioconductor-simlr            570\n",
       "2915                          bioconductor-simpintlists            194\n",
       "2916                            bioconductor-simpleaffy            569\n",
       "2917                            bioconductor-simulatorz            276\n",
       "2918                               bioconductor-sincell            573\n",
       "2919                  bioconductor-singlecellexperiment            871\n",
       "2920          bioconductor-singlecellexperiment-scripts             12\n",
       "2921                          bioconductor-singlecelltk              9\n",
       "2922            bioconductor-single.mtec.transcriptomes              9\n",
       "2923                               bioconductor-singler             96\n",
       "2924                             bioconductor-singscore             29\n",
       "2925                                 bioconductor-sispa            281\n",
       "2926                              bioconductor-sitepath            184\n",
       "2927                             bioconductor-sizepower            282\n",
       "2928                                 bioconductor-skewr              7\n",
       "2929                                bioconductor-slalom            377\n",
       "2930                                  bioconductor-slgi            188\n",
       "2931                             bioconductor-slingshot            117\n",
       "2932                                bioconductor-slinky              3\n",
       "2933                                bioconductor-slqpcr            289\n",
       "2934                                  bioconductor-smad            184\n",
       "2935                                  bioconductor-smap            555\n",
       "2936                                 bioconductor-smite            191\n",
       "2937                               bioconductor-snadata              3\n",
       "2938                                bioconductor-snagee            189\n",
       "2939                            bioconductor-snageedata            189\n",
       "2940                               bioconductor-snapcgh            184\n",
       "2941                                   bioconductor-snm            202\n",
       "2942                              bioconductor-snpediar            282\n",
       "2943                               bioconductor-snphood            292\n",
       "2944                           bioconductor-snphooddata              4\n",
       "2945      bioconductor-snplocs.hsapiens.dbsnp141.grch38              4\n",
       "2946      bioconductor-snplocs.hsapiens.dbsnp142.grch37              7\n",
       "2947      bioconductor-snplocs.hsapiens.dbsnp144.grch37              7\n",
       "2948      bioconductor-snplocs.hsapiens.dbsnp144.grch38            110\n",
       "2949      bioconductor-snplocs.hsapiens.dbsnp149.grch38              5\n",
       "2950      bioconductor-snplocs.hsapiens.dbsnp150.grch38              3\n",
       "2951      bioconductor-snplocs.hsapiens.dbsnp151.grch38              8\n",
       "2952       bioconductor-snplocs.hsapiens.dbsnp.20101109              3\n",
       "2953       bioconductor-snplocs.hsapiens.dbsnp.20120608              3\n",
       "2954                             bioconductor-snprelate            584\n",
       "2955                              bioconductor-snpstats            588\n",
       "2956                                 bioconductor-soggi            300\n",
       "2957                             bioconductor-sojourner             92\n",
       "2958                          bioconductor-somaticadata              6\n",
       "2959              bioconductor-somaticcanceralterations              3\n",
       "2960                     bioconductor-somaticsignatures            283\n",
       "2961                            bioconductor-soybeancdf              3\n",
       "2962                          bioconductor-soybeanprobe              3\n",
       "2963                              bioconductor-spacepac            195\n",
       "2964                               bioconductor-spaniel              1\n",
       "2965                           bioconductor-sparsedossa              4\n",
       "2966                          bioconductor-sparsenetgls              4\n",
       "2967                      bioconductor-sparsesignatures              3\n",
       "2968                           bioconductor-spatialcpie              2\n",
       "2969                                 bioconductor-specl            187\n",
       "2970                               bioconductor-specond            281\n",
       "2971                           bioconductor-spectraltad              2\n",
       "2972                                  bioconductor-spem            282\n",
       "2973                                  bioconductor-spia            295\n",
       "2974                             bioconductor-spidermir            198\n",
       "2975                               bioconductor-spikein              6\n",
       "2976                         bioconductor-spikeinsubset              3\n",
       "2977                               bioconductor-spikeli            282\n",
       "2978                              bioconductor-spktools            282\n",
       "2979                              bioconductor-splatter            204\n",
       "2980                        bioconductor-splicinggraphs            188\n",
       "2981                           bioconductor-splinetimer            191\n",
       "2982                              bioconductor-splinter            303\n",
       "2983                                bioconductor-splots            313\n",
       "2984                                bioconductor-sponge              7\n",
       "2985                      bioconductor-spotsegmentation            203\n",
       "2986                          bioconductor-sqldataframe              1\n",
       "2987                                bioconductor-squadd            282\n",
       "2988                               bioconductor-sracipe            184\n",
       "2989                                 bioconductor-sradb            481\n",
       "2990                                  bioconductor-srap            189\n",
       "2991                                bioconductor-srgnet              3\n",
       "2992                              bioconductor-srnadiff            276\n",
       "2993                                bioconductor-sscore            281\n",
       "2994                                  bioconductor-sscu            289\n",
       "2995                                  bioconductor-sseq            283\n",
       "2996                                 bioconductor-ssize            296\n",
       "2997                                  bioconductor-sspa            673\n",
       "2998                               bioconductor-sspaths              1\n",
       "2999                                 bioconductor-ssrch              2\n",
       "3000                                 bioconductor-ssviz            289\n",
       "3001                                bioconductor-stager             42\n",
       "3002                                  bioconductor-stan            482\n",
       "3003                               bioconductor-starank            291\n",
       "3004                           bioconductor-starbiotrek            189\n",
       "3005                                 bioconductor-starr            560\n",
       "3006                              bioconductor-stategra            280\n",
       "3007                            bioconductor-stattarget              5\n",
       "3008                           bioconductor-stemhypoxia            195\n",
       "3009                              bioconductor-stepnorm            282\n",
       "3010                               bioconductor-stjudem              4\n",
       "3011                          bioconductor-strandcheckr              3\n",
       "3012                              bioconductor-streamer            568\n",
       "3013                              bioconductor-stringdb            293\n",
       "3014                               bioconductor-stroma4              5\n",
       "3015                         bioconductor-structstrings            184\n",
       "3016           bioconductor-structuralvariantannotation             47\n",
       "3017                        bioconductor-subcellbarcode              4\n",
       "3018                                bioconductor-subseq            293\n",
       "3019                          bioconductor-sugarcanecdf              3\n",
       "3020                        bioconductor-sugarcaneprobe              4\n",
       "3021                   bioconductor-summarizedbenchmark              3\n",
       "3022                  bioconductor-summarizedexperiment          11621\n",
       "3023                              bioconductor-suprahex            321\n",
       "3024                              bioconductor-survcomp            485\n",
       "3025                              bioconductor-survtype              2\n",
       "3026                                 bioconductor-sushi            311\n",
       "3027                                   bioconductor-sva           1253\n",
       "3028                             bioconductor-svaplsseq            195\n",
       "3029                           bioconductor-svm2crmdata              5\n",
       "3030                           bioconductor-swath2stats            281\n",
       "3031                            bioconductor-swathxtend            282\n",
       "3032                                 bioconductor-swfdr              4\n",
       "3033                                 bioconductor-swimr            200\n",
       "3034                             bioconductor-switchbox            570\n",
       "3035                              bioconductor-switchde            282\n",
       "3036                              bioconductor-synapter             98\n",
       "3037                          bioconductor-synapterdata              3\n",
       "3038                         bioconductor-synergyfinder            375\n",
       "3039                                bioconductor-synlet            208\n",
       "3040                                bioconductor-synmut              2\n",
       "3041                           bioconductor-systempiper            615\n",
       "3042                       bioconductor-systempiperdata              5\n",
       "3043                       bioconductor-tabulamurisdata              6\n",
       "3044                                bioconductor-target              1\n",
       "3045                   bioconductor-targetscan.hs.eg.db            104\n",
       "3046                   bioconductor-targetscan.mm.eg.db              7\n",
       "3047                           bioconductor-targetscore            282\n",
       "3048                       bioconductor-targetscoredata              9\n",
       "3049                          bioconductor-targetsearch            375\n",
       "3050                      bioconductor-targetsearchdata              4\n",
       "3051                              bioconductor-tarseqqc            372\n",
       "3052                               bioconductor-tartare              1\n",
       "3053                        bioconductor-tbx20bamsubset              3\n",
       "3054                                   bioconductor-tcc            190\n",
       "3055                          bioconductor-tcgabiolinks            219\n",
       "3056                       bioconductor-tcgabiolinksgui              7\n",
       "3057                  bioconductor-tcgabiolinksgui.data              9\n",
       "3058                          bioconductor-tcgacrcmirna              3\n",
       "3059                           bioconductor-tcgacrcmrna              9\n",
       "3060                   bioconductor-tcgamethylation450k              4\n",
       "3061                             bioconductor-tcgautils              8\n",
       "3062                     bioconductor-tcgawgbsdata.hg19              8\n",
       "3063                      bioconductor-tcgaworkflowdata              3\n",
       "3064                                 bioconductor-tcseq              5\n",
       "3065                              bioconductor-tdaracne            190\n",
       "3066                         bioconductor-tenxbraindata              3\n",
       "3067                           bioconductor-tenxbusdata              2\n",
       "3068                          bioconductor-tenxpbmcdata              8\n",
       "3069                             bioconductor-tenxplore              3\n",
       "3070                                  bioconductor-teqc            286\n",
       "3071                            bioconductor-ternarynet            559\n",
       "3072                              bioconductor-test1cdf              3\n",
       "3073                              bioconductor-test2cdf              3\n",
       "3074                              bioconductor-test3cdf              3\n",
       "3075                            bioconductor-test3probe              3\n",
       "3076                                 bioconductor-tfarm              9\n",
       "3077                             bioconductor-tfbstools            769\n",
       "3078                             bioconductor-tfea.chip              8\n",
       "3079                                 bioconductor-tfhaz              3\n",
       "3080                               bioconductor-tfutils             11\n",
       "3081                                 bioconductor-tigre            554\n",
       "3082                           bioconductor-tilingarray            276\n",
       "3083                            bioconductor-timecourse            290\n",
       "3084                            bioconductor-timerquant              4\n",
       "3085                             bioconductor-timescape              4\n",
       "3086                  bioconductor-timeseriesexperiment              7\n",
       "3087                                   bioconductor-tin            190\n",
       "3088                          bioconductor-tinesath1cdf              5\n",
       "3089                        bioconductor-tinesath1probe              3\n",
       "3090                          bioconductor-tissueenrich            104\n",
       "3091                            bioconductor-tissuetreg              4\n",
       "3092                              bioconductor-titancna           1076\n",
       "3093                             bioconductor-tkwidgets             18\n",
       "3094                             bioconductor-tmixclust              5\n",
       "3095                              bioconductor-tnbc.cms              2\n",
       "3096                                   bioconductor-tnt              6\n",
       "3097                                 bioconductor-toast              1\n",
       "3098                               bioconductor-tofsims            379\n",
       "3099                           bioconductor-tofsimsdata              4\n",
       "3100                             bioconductor-tomatocdf              9\n",
       "3101                           bioconductor-tomatoprobe              3\n",
       "3102                               bioconductor-topaseq            284\n",
       "3103                           bioconductor-topconfects              2\n",
       "3104                              bioconductor-topdownr              3\n",
       "3105                          bioconductor-topdownrdata              3\n",
       "3106                                 bioconductor-topgo            644\n",
       "3107                                 bioconductor-tpp2d              2\n",
       "3108                                   bioconductor-tpp              7\n",
       "3109                           bioconductor-tracktables            202\n",
       "3110                           bioconductor-trackviewer            199\n",
       "3111                              bioconductor-tradeseq             31\n",
       "3112                     bioconductor-transcriptogramer              3\n",
       "3113                           bioconductor-transcriptr            192\n",
       "3114                              bioconductor-transite            284\n",
       "3115                           bioconductor-translatome            195\n",
       "3116                             bioconductor-transview            558\n",
       "3117                                bioconductor-traser            299\n",
       "3118                                bioconductor-treeio            520\n",
       "3119              bioconductor-treesummarizedexperiment              2\n",
       "3120                                 bioconductor-trena              3\n",
       "3121                                bioconductor-trendy              3\n",
       "3122                               bioconductor-triform            291\n",
       "3123                               bioconductor-trigger            461\n",
       "3124                                  bioconductor-trio            288\n",
       "3125                               bioconductor-triplex            565\n",
       "3126                                  bioconductor-trna              3\n",
       "3127                          bioconductor-trnadbimport              3\n",
       "3128                        bioconductor-trnascanimport              3\n",
       "3129                                bioconductor-tronco              5\n",
       "3130                                 bioconductor-tscan            289\n",
       "3131                                bioconductor-tspair            554\n",
       "3132                            bioconductor-tsrchitect              3\n",
       "3133                                 bioconductor-ttmap              6\n",
       "3134                             bioconductor-turbonorm            461\n",
       "3135                                  bioconductor-tvtb            197\n",
       "3136                             bioconductor-tweedeseq            647\n",
       "3137                    bioconductor-tweedeseqcountdata              3\n",
       "3138                              bioconductor-twilight            571\n",
       "3139                              bioconductor-twoddpcr              3\n",
       "3140   bioconductor-txdb.athaliana.biomart.plantsmart22              3\n",
       "3141   bioconductor-txdb.athaliana.biomart.plantsmart25              3\n",
       "3142   bioconductor-txdb.athaliana.biomart.plantsmart28              7\n",
       "3143     bioconductor-txdb.btaurus.ucsc.bostau8.refgene              7\n",
       "3144     bioconductor-txdb.btaurus.ucsc.bostau9.refgene              1\n",
       "3145       bioconductor-txdb.celegans.ucsc.ce11.ensgene              2\n",
       "3146       bioconductor-txdb.celegans.ucsc.ce11.refgene            384\n",
       "3147        bioconductor-txdb.celegans.ucsc.ce6.ensgene            126\n",
       "3148  bioconductor-txdb.cfamiliaris.ucsc.canfam3.ref...              7\n",
       "3149   bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene            129\n",
       "3150   bioconductor-txdb.dmelanogaster.ucsc.dm6.ensgene            200\n",
       "3151     bioconductor-txdb.drerio.ucsc.danrer10.refgene            282\n",
       "3152     bioconductor-txdb.drerio.ucsc.danrer11.refgene              2\n",
       "3153     bioconductor-txdb.ggallus.ucsc.galgal4.refgene              3\n",
       "3154     bioconductor-txdb.ggallus.ucsc.galgal5.refgene              3\n",
       "3155     bioconductor-txdb.ggallus.ucsc.galgal6.refgene              1\n",
       "3156            bioconductor-txdb.hsapiens.biomart.igis             10\n",
       "3157     bioconductor-txdb.hsapiens.ucsc.hg18.knowngene            131\n",
       "3158     bioconductor-txdb.hsapiens.ucsc.hg19.knowngene            336\n",
       "3159  bioconductor-txdb.hsapiens.ucsc.hg19.lincrnast...             99\n",
       "3160     bioconductor-txdb.hsapiens.ucsc.hg38.knowngene            249\n",
       "3161   bioconductor-txdb.mmulatta.ucsc.rhemac10.refgene              1\n",
       "3162    bioconductor-txdb.mmulatta.ucsc.rhemac3.refgene              6\n",
       "3163    bioconductor-txdb.mmulatta.ucsc.rhemac8.refgene              8\n",
       "3164      bioconductor-txdb.mmusculus.ucsc.mm10.ensgene            208\n",
       "3165    bioconductor-txdb.mmusculus.ucsc.mm10.knowngene            171\n",
       "3166     bioconductor-txdb.mmusculus.ucsc.mm9.knowngene            177\n",
       "3167  bioconductor-txdb.ptroglodytes.ucsc.pantro4.re...              9\n",
       "3168  bioconductor-txdb.ptroglodytes.ucsc.pantro5.re...              7\n",
       "3169  bioconductor-txdb.ptroglodytes.ucsc.pantro6.re...              1\n",
       "3170         bioconductor-txdb.rnorvegicus.biomart.igis              5\n",
       "3171     bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene            125\n",
       "3172     bioconductor-txdb.rnorvegicus.ucsc.rn5.refgene            379\n",
       "3173     bioconductor-txdb.rnorvegicus.ucsc.rn6.refgene            297\n",
       "3174  bioconductor-txdb.scerevisiae.ucsc.saccer2.sgd...            101\n",
       "3175  bioconductor-txdb.scerevisiae.ucsc.saccer3.sgd...            103\n",
       "3176    bioconductor-txdb.sscrofa.ucsc.susscr11.refgene              7\n",
       "3177     bioconductor-txdb.sscrofa.ucsc.susscr3.refgene              3\n",
       "3178                               bioconductor-tximeta             62\n",
       "3179                              bioconductor-tximport           1498\n",
       "3180                          bioconductor-tximportdata             13\n",
       "3181                            bioconductor-txreginfra              3\n",
       "3182                              bioconductor-typeinfo              4\n",
       "3183                        bioconductor-u133aaofav2cdf              3\n",
       "3184                            bioconductor-u133x3pcdf              3\n",
       "3185                            bioconductor-u133x3p.db              3\n",
       "3186                          bioconductor-u133x3pprobe              3\n",
       "3187                              bioconductor-ularcirc              7\n",
       "3188                                  bioconductor-undo            200\n",
       "3189                        bioconductor-unifiedwmwqpcr            202\n",
       "3190                            bioconductor-uniprot.ws            293\n",
       "3191                              bioconductor-uniquorn            195\n",
       "3192                        bioconductor-universalmotif            283\n",
       "3193                                 bioconductor-usort              3\n",
       "3194                            bioconductor-vanillaice            462\n",
       "3195                     bioconductor-variancepartition            225\n",
       "3196                     bioconductor-variantannotation           1845\n",
       "3197                     bioconductor-variantexperiment              1\n",
       "3198                      bioconductor-variantfiltering            461\n",
       "3199                          bioconductor-varianttools            285\n",
       "3200                      bioconductor-varianttoolsdata              3\n",
       "3201                                  bioconductor-vbmp            282\n",
       "3202                              bioconductor-vcfarray              2\n",
       "3203                                  bioconductor-vega            572\n",
       "3204                                bioconductor-vegamc            478\n",
       "3205                            bioconductor-venndetail              3\n",
       "3206                                bioconductor-vidger              4\n",
       "3207                                 bioconductor-viper            291\n",
       "3208                               bioconductor-viseago              1\n",
       "3209                      bioconductor-vitisviniferacdf              4\n",
       "3210                    bioconductor-vitisviniferaprobe              3\n",
       "3211                                   bioconductor-vsn           1188\n",
       "3212                                bioconductor-vtpnet              7\n",
       "3213                                bioconductor-vulcan              6\n",
       "3214                            bioconductor-vulcandata              9\n",
       "3215                                 bioconductor-waddr             92\n",
       "3216                            bioconductor-watermelon            215\n",
       "3217                            bioconductor-wavcluster            190\n",
       "3218                        bioconductor-wavetilingdata              8\n",
       "3219                                bioconductor-weaver            291\n",
       "3220                               bioconductor-webbioc            194\n",
       "3221                         bioconductor-wes.1kg.wugsc              4\n",
       "3222                              bioconductor-wheatcdf              3\n",
       "3223                            bioconductor-wheatprobe              4\n",
       "3224                           bioconductor-widgettools             18\n",
       "3225                           bioconductor-wiggleplotr            280\n",
       "3226                              bioconductor-worm.db0              8\n",
       "3227                                bioconductor-wrench             39\n",
       "3228                                 bioconductor-xbseq            281\n",
       "3229                                  bioconductor-xcir              1\n",
       "3230                                  bioconductor-xcms           1253\n",
       "3231                                   bioconductor-xde            570\n",
       "3232                           bioconductor-xenopus.db0              3\n",
       "3233                      bioconductor-xenopuslaeviscdf              3\n",
       "3234                    bioconductor-xenopuslaevisprobe              3\n",
       "3235                                  bioconductor-xeva              2\n",
       "3236                           bioconductor-xhybcasneuf              3\n",
       "3237                                  bioconductor-xina              4\n",
       "3238                           bioconductor-xlaevis2cdf              3\n",
       "3239                         bioconductor-xlaevis2probe              3\n",
       "3240                            bioconductor-xlaevis.db              3\n",
       "3241                            bioconductor-xmapbridge            282\n",
       "3242  bioconductor-xtrasnplocs.hsapiens.dbsnp141.grch38              3\n",
       "3243  bioconductor-xtrasnplocs.hsapiens.dbsnp144.grch37              7\n",
       "3244  bioconductor-xtrasnplocs.hsapiens.dbsnp144.grch38              3\n",
       "3245                        bioconductor-xtropicaliscdf              3\n",
       "3246                      bioconductor-xtropicalisprobe              3\n",
       "3247                               bioconductor-xvector          12556\n",
       "3248                                 bioconductor-yamss            102\n",
       "3249                                 bioconductor-yapsa            189\n",
       "3250                              bioconductor-yaqcaffy            195\n",
       "3251                                  bioconductor-yarn            195\n",
       "3252                         bioconductor-ye6100subacdf              3\n",
       "3253                         bioconductor-ye6100subbcdf              3\n",
       "3254                         bioconductor-ye6100subccdf              3\n",
       "3255                         bioconductor-ye6100subdcdf              3\n",
       "3256                             bioconductor-yeast2cdf              3\n",
       "3257                             bioconductor-yeast2.db              3\n",
       "3258                           bioconductor-yeast2probe              4\n",
       "3259                               bioconductor-yeastcc            200\n",
       "3260                             bioconductor-yeast.db0              3\n",
       "3261                          bioconductor-yeastexpdata            282\n",
       "3262                           bioconductor-yeastgsdata              4\n",
       "3263                      bioconductor-yeastnagalakshmi              4\n",
       "3264                           bioconductor-yeastrnaseq              4\n",
       "3265                              bioconductor-ygs98cdf              3\n",
       "3266                              bioconductor-ygs98.db              3\n",
       "3267                         bioconductor-ygs98frmavecs              4\n",
       "3268                            bioconductor-ygs98probe              3\n",
       "3269                               bioconductor-yri1kgv              4\n",
       "3270                              bioconductor-yrimulti              3\n",
       "3271                          bioconductor-zebrafishcdf              3\n",
       "3272                         bioconductor-zebrafish.db0              6\n",
       "3273                          bioconductor-zebrafish.db              3\n",
       "3274                        bioconductor-zebrafishprobe              3\n",
       "3275                       bioconductor-zebrafishrnaseq              7\n",
       "3276                                 bioconductor-zfpkm            198\n",
       "3277                              bioconductor-zinbwave            274\n",
       "3278                              bioconductor-zlibbioc          12826\n",
       "3279                                         bioconvert            326\n",
       "3280                                            biodiff             94\n",
       "3281                                     bioexcel_seqqc              4\n",
       "3282                                           biofluff            849\n",
       "3283                                         bioformats            110\n",
       "3284                                          biogridpy            101\n",
       "3285                                         bio_hansel           1252\n",
       "3286                                             biokit            589\n",
       "3287                                             biolib            135\n",
       "3288                                            biolite            185\n",
       "3289                                      biolite-tools            107\n",
       "3290                                             biomaj             95\n",
       "3291                                             biopet            316\n",
       "3292                                    biopet-bamstats              4\n",
       "3293                                 biopet-basecounter              2\n",
       "3294                       biopet-extractadaptersfastqc            212\n",
       "3295                               biopet-fastqsplitter              7\n",
       "3296                                biopet-sampleconfig            107\n",
       "3297                              biopet-scatterregions              4\n",
       "3298                               biopet-seattleseqkit              3\n",
       "3299                                     biopet-seqstat            100\n",
       "3300                          biopet-validateannotation              2\n",
       "3301                               biopet-validatefastq            105\n",
       "3302                                 biopet-validatevcf              2\n",
       "3303                                   biopet-vcffilter             17\n",
       "3304                                    biopet-vcfstats            193\n",
       "3305                                  biopython.convert             11\n",
       "3306                                        bioservices            796\n",
       "3307                                             biotdg             14\n",
       "3308                                   biothings_client            497\n",
       "3309                                          biotradis            228\n",
       "3310                                            bio-vcf            383\n",
       "3311                             biowdl-input-converter              6\n",
       "3312                                             biskit             98\n",
       "3313                                            bismark           1307\n",
       "3314                                            bis-snp            190\n",
       "3315                                      bis-snp-utils            104\n",
       "3316                                        bitmapperbs            463\n",
       "3317                                          bitstring            127\n",
       "3318                                              blasr            707\n",
       "3319                                       blasr_libcpp            584\n",
       "3320                                              blast          10108\n",
       "3321                                         blastalign            109\n",
       "3322                                       blastbesties            104\n",
       "3323                                       blast-legacy            857\n",
       "3324                                           blksheep             11\n",
       "3325                                          blobtools            117\n",
       "3326                                        blockbuster            105\n",
       "3327                                         blockclust            128\n",
       "3328                                            bloocoo             95\n",
       "3329                                    bloomfiltertrie            184\n",
       "3330                                           bmfilter            302\n",
       "3331                                           bmtagger            298\n",
       "3332                                             bmtool            300\n",
       "3333                                              bohra             62\n",
       "3334                                bold-identification              3\n",
       "3335                                               bolt            278\n",
       "3336                                            booster             96\n",
       "3337                                               borf              2\n",
       "3338                                             bowtie          11858\n",
       "3339                                            bowtie2           8426\n",
       "3340                                              bpipe            112\n",
       "3341                                           bpp-core            198\n",
       "3342                                           bpp-phyl            203\n",
       "3343                                         bpp-popgen             94\n",
       "3344                                            bpp-seq            198\n",
       "3345                                            bracken            407\n",
       "3346                                             braker            163\n",
       "3347                                            braker2            165\n",
       "3348                                              brass             93\n",
       "3349                                        breakdancer            134\n",
       "3350                              break-point-inspector            233\n",
       "3351                                          breakseq2             94\n",
       "3352                                             breseq            713\n",
       "3353                                  brockman-pipeline             95\n",
       "3354                                            bs_call            100\n",
       "3355                                              bsmap            132\n",
       "3356                                         bs-seeker2            102\n",
       "3357                                              btrim            189\n",
       "3358                                              bufet            100\n",
       "3359                                        bumbershoot            581\n",
       "3360                                            burrito            254\n",
       "3361                                   burrito-fillings            244\n",
       "3362                                              busco           1475\n",
       "3363                                           bustools            379\n",
       "3364                                                bwa          14995\n",
       "3365                                             bwakit            361\n",
       "3366                                            bwameth            428\n",
       "3367                                              bwapy            104\n",
       "3368                                              bwise             92\n",
       "3369                                          bx-python           4947\n",
       "3370                                            bxtools            187\n",
       "3371                                             cactus            142\n",
       "3372                                               cafe            101\n",
       "3373                                               cage            299\n",
       "3374                                              calib             94\n",
       "3375                                             calour              6\n",
       "3376                                              cameo             16\n",
       "3377                                          cami-opal             13\n",
       "3378                               cancerit-allelecount            332\n",
       "3379                                            cannoli            133\n",
       "3380                                             cansam             94\n",
       "3381                                           cansnper             98\n",
       "3382                                               canu           1254\n",
       "3383                                             canvas            238\n",
       "3384                                               cap3            201\n",
       "3385                                           capc-map            200\n",
       "3386                                              caper            142\n",
       "3387                                         cap-mirseq            187\n",
       "3388                                         card_trick              9\n",
       "3389                                             c-ares            267\n",
       "3390                                              carna            374\n",
       "3391                                          carnac-lr             94\n",
       "3392                                       cas-offinder            179\n",
       "3393                                          cassiopee             95\n",
       "3394                                                cat            336\n",
       "3395                                              catch            103\n",
       "3396                                               ccat             95\n",
       "3397                                           cdbtools            397\n",
       "3398                                             cd-hit           1752\n",
       "3399                                    cd-hit-auxtools            152\n",
       "3400                                       cdna_cupcake            479\n",
       "3401                                               ceas            107\n",
       "3402                                          centreseq              9\n",
       "3403                                         centrifuge            704\n",
       "3404                               centroid_rna_package            185\n",
       "3405                                         centrosome             96\n",
       "3406                                               cesm            202\n",
       "3407                                                cfm             92\n",
       "3408                                          cgat-apps            509\n",
       "3409                                           cgatcore           1072\n",
       "3410                                         cgat-daisy             13\n",
       "3411                                        cgat-report            312\n",
       "3412                                       cgat-scripts            485\n",
       "3413                                        cgcloud-lib            234\n",
       "3414                                            cgecore             21\n",
       "3415                                          cgpbigwig            374\n",
       "3416                                             cgview            110\n",
       "3417                                        chado-tools             20\n",
       "3418                                            changeo             18\n",
       "3419                                             chanjo            190\n",
       "3420                                      checkm-genome            962\n",
       "3421                                   check-sort-order            429\n",
       "3422                                             chemfp            477\n",
       "3423                                          chewbbaca            590\n",
       "3424                                            chexmix             17\n",
       "3425                                       chicagotools            103\n",
       "3426                                   chipseq-greylist            205\n",
       "3427                                              chira             61\n",
       "3428                                             chiron             21\n",
       "3429                                            chorus2            114\n",
       "3430                                      chromatiblock             22\n",
       "3431                                           chromhmm            356\n",
       "3432                                        chromosight             34\n",
       "3433                                        chromosomer            102\n",
       "3434                                    chunked-scatter              1\n",
       "3435                                              cigar             22\n",
       "3436                                      circexplorer2           1883\n",
       "3437                                       circexplorer            566\n",
       "3438                                          circlator            822\n",
       "3439                                         circle-map             16\n",
       "3440                                             circos           1460\n",
       "3441                                       circos-tools            201\n",
       "3442                                     circularmapper            197\n",
       "3443                                      cistrome_beta            104\n",
       "3444                                      cistrome-ceas            101\n",
       "3445                                           cityhash            278\n",
       "3446                                              clair            116\n",
       "3447                                       clairvoyante            302\n",
       "3448                                               clan             83\n",
       "3449                                              clark            560\n",
       "3450                                           clearcut             98\n",
       "3451                                     clever-toolkit            573\n",
       "3452                                             clinod             98\n",
       "3453                                        clinvar-tsv              5\n",
       "3454                                       clipandmerge             96\n",
       "3455                                      clonalframeml            209\n",
       "3456                                              clove            104\n",
       "3457                                             clsify              1\n",
       "3458                                              clust            395\n",
       "3459                                           clustalo            502\n",
       "3460                                           clustalw           3296\n",
       "3461                                     clustergrammer            136\n",
       "3462                                      clusterpicker              1\n",
       "3463                                             cmappy           1246\n",
       "3464                                           cmfinder            384\n",
       "3465                                               cmph             94\n",
       "3466                                              cmsip             44\n",
       "3467                                                cmv            463\n",
       "3468                                            cnvetti            185\n",
       "3469                                         cnv_facets            725\n",
       "3470                                             cnvkit           2842\n",
       "3471                                              cobra            411\n",
       "3472                                          cocoscore             95\n",
       "3473                                       codingquarry            380\n",
       "3474                                             codonw            107\n",
       "3475                                             cofold             96\n",
       "3476                                             cogent            489\n",
       "3477                                         coinfinder            316\n",
       "3478                                    collect-columns              7\n",
       "3479                                           collectl            343\n",
       "3480                                        collect_mgf             92\n",
       "3481                                            colombo            103\n",
       "3482                                        colorbrewer            115\n",
       "3483                                   combined-pvalues            193\n",
       "3484                                           comet-ms            847\n",
       "3485                                             commet            105\n",
       "3486                                         compalignp             97\n",
       "3487                     comparative-annotation-toolkit            105\n",
       "3488                                         comparems2             93\n",
       "3489                                      compare-reads            119\n",
       "3490                                            concoct            785\n",
       "3491                                           confindr            144\n",
       "3492                               connectome-workbench            186\n",
       "3493                                             connor            214\n",
       "3494                                             consan             98\n",
       "3495                                             consel             93\n",
       "3496                                     consensusfixer            196\n",
       "3497                                      conterminator             28\n",
       "3498                                      control-freec            401\n",
       "3499                                              conus             92\n",
       "3500                             convert_zero_one_based              2\n",
       "3501                                             cooler           2887\n",
       "3502                                           coprarna            396\n",
       "3503                                              coral             14\n",
       "3504                                             corset            343\n",
       "3505                                         cortex_con             96\n",
       "3506                                           cortexpy            470\n",
       "3507                                              cosi2            664\n",
       "3508                               count_constant_sites             92\n",
       "3509                                           counterr              7\n",
       "3510                                             coverm            396\n",
       "3511                                           covtobed            529\n",
       "3512                                               cpat            367\n",
       "3513                                            cpinsim             96\n",
       "3514                                          cramtools            308\n",
       "3515                                              crass             95\n",
       "3516                                          crb-blast            112\n",
       "3517                                            crimson            213\n",
       "3518                           crisper_recognition_tool            105\n",
       "3519                                         crisprbact             13\n",
       "3520                                        crispresso2            669\n",
       "3521                                         crispresso            474\n",
       "3522                                          crispritz            762\n",
       "3523                                      crispr_studio             13\n",
       "3524                                          cromshell             26\n",
       "3525                                     cromwell-tools             13\n",
       "3526                                               croo            137\n",
       "3527                                           crossmap            911\n",
       "3528                                            crumble            281\n",
       "3529                                       crux-toolkit            121\n",
       "3530                                              cryfa             92\n",
       "3531                                       crypto_typer              1\n",
       "3532                                                csb             99\n",
       "3533                                              csvtk           2339\n",
       "3534                                        ctat-lncrna            186\n",
       "3535                                  ctat-metagenomics             93\n",
       "3536                                     ctat-mutations            209\n",
       "3537                                          cufflinks            567\n",
       "3538                                            curesim            102\n",
       "3539                                        curlywhirly             99\n",
       "3540                                           cutadapt          14265\n",
       "3541                                             cutesv             14\n",
       "3542                                              cvbio             32\n",
       "3543                                            cwl2wdl            193\n",
       "3544                                            cwltest            446\n",
       "3545                                          cytoscape            124\n",
       "3546                                             cyvcf2           8829\n",
       "3547                                         daisysuite            474\n",
       "3548                                           daligner            293\n",
       "3549                                     damageprofiler            222\n",
       "3550                                           damasker             93\n",
       "3551                                  damidseq_pipeline             97\n",
       "3552                                             dammet             93\n",
       "3553                                             dammit            498\n",
       "3554                                             danpos              3\n",
       "3555                                               dart            431\n",
       "3556                                         dascrubber            185\n",
       "3557                                            dashing             96\n",
       "3558                                           das_tool            172\n",
       "3559                                         data_hacks             99\n",
       "3560                                               dawg             97\n",
       "3561                                            dazz_db            289\n",
       "3562                                            dbg2olc            200\n",
       "3563                                            dbgraph            185\n",
       "3564                                                dca             20\n",
       "3565                                              dcmtk            105\n",
       "3566                                             dcplib             16\n",
       "3567                                            ddocent           1868\n",
       "3568                                             ddrage            845\n",
       "3569                                               deap            128\n",
       "3570                                             deblur           2644\n",
       "3571                                              debwt             94\n",
       "3572                                         decoypyrat              1\n",
       "3573                                              dedup            208\n",
       "3574                                             deepac             10\n",
       "3575                                            deepbgc             47\n",
       "3576                                         deepbinner            216\n",
       "3577                                             deeplc             41\n",
       "3578                                           deeplift             30\n",
       "3579                                          deepmedic            283\n",
       "3580                                          deepstats              6\n",
       "3581                                            deepsvr              1\n",
       "3582                                          deeptools           5174\n",
       "3583                                 deeptoolsintervals           1728\n",
       "3584                                        deepvariant            908\n",
       "3585                                            defiant             92\n",
       "3586                                   dehomopolymerate            187\n",
       "3587                                         delegation              4\n",
       "3588                                              delly           1063\n",
       "3589                                            deltabs             99\n",
       "3590                                           demuxlet            126\n",
       "3591                                           dendropy           4080\n",
       "3592                                         denovogear             93\n",
       "3593                                            deploid             95\n",
       "3594                                             desalt            478\n",
       "3595                                             desman            101\n",
       "3596                                           detonate            100\n",
       "3597                                         dextractor            186\n",
       "3598                                               dfam             99\n",
       "3599                                              dfast            307\n",
       "3600                                           dialign2             95\n",
       "3601                                         dialign-tx            126\n",
       "3602                                            diamond           4416\n",
       "3603                               diamond_add_taxonomy            188\n",
       "3604                                         dia_umpire            195\n",
       "3605                                              dicey             98\n",
       "3606                                               dig2             93\n",
       "3607                                    digestiflow-cli           1305\n",
       "3608                                  digestiflow-demux             52\n",
       "3609                                             dimspy            382\n",
       "3610                                             dinamo            166\n",
       "3611                                             dinopy            625\n",
       "3612                                           dinosaur              2\n",
       "3613                                            discasm            191\n",
       "3614                                              disco            193\n",
       "3615                                           discosnp            309\n",
       "3616                                           discovar            101\n",
       "3617                                              disty             96\n",
       "3618                                            ditasic             93\n",
       "3619                                            divvier             93\n",
       "3620                                   dkfz-bias-filter            165\n",
       "3621                                             dlcpar            172\n",
       "3622                                              dnaio          10175\n",
       "3623                                              dnapi            389\n",
       "3624                                     dnp-binstrings             94\n",
       "3625                                    dnp-corrprofile             94\n",
       "3626                                      dnp-diprofile             98\n",
       "3627                                        dnp-fourier             95\n",
       "3628                                           domclust            203\n",
       "3629                                           downpore            373\n",
       "3630                                               drax             98\n",
       "3631                                           dr-disco            922\n",
       "3632                                         dreamtools            203\n",
       "3633                                               drep            275\n",
       "3634                                              drive            191\n",
       "3635                                            dropest            200\n",
       "3636                               dropletutils-scripts             16\n",
       "3637                                      dropseq_tools            228\n",
       "3638                                            dsh-bio            224\n",
       "3639                                                dsk            121\n",
       "3640                                               dsrc            219\n",
       "3641                                              dudes            289\n",
       "3642                                             dunovo           1113\n",
       "3643                                            duphold           1144\n",
       "3644                                              dupre             96\n",
       "3645                                             dwgsim            125\n",
       "3646                                             dx-cwl           1135\n",
       "3647                                               dxpy            937\n",
       "3648                                               dxua            278\n",
       "3649                                              eagle            195\n",
       "3650                                              easel             99\n",
       "3651                                           ea-utils            233\n",
       "3652                                           ebfilter             96\n",
       "3653                                          ebisearch            186\n",
       "3654                                           ecmwfapi            116\n",
       "3655                                             ecopcr            103\n",
       "3656                                         ecoprimers            103\n",
       "3657                                              ecopy            194\n",
       "3658                                            ectyper            223\n",
       "3659                                                edd            194\n",
       "3660                                               eden            206\n",
       "3661                                              edena             98\n",
       "3662                                            edittag             98\n",
       "3663                                              edlib            503\n",
       "3664                                               edta             18\n",
       "3665                                        effectivet3             94\n",
       "3666                                               ega2              1\n",
       "3667                                             egglib            101\n",
       "3668                                      eggnog-mapper            484\n",
       "3669                                          eigensoft            236\n",
       "3670                                      elasticluster             93\n",
       "3671                                            elector            191\n",
       "3672                                             elprep            662\n",
       "3673                                                ema           4952\n",
       "3674                                     embassy-phylip            112\n",
       "3675                                 embl-api-validator            190\n",
       "3676                                         emblmygff3             21\n",
       "3677                                             emboss           1368\n",
       "3678                                             emirge            135\n",
       "3679                                              emmix            101\n",
       "3680                                           emmtyper              1\n",
       "3681                                            emperor            329\n",
       "3682                                    enabrowsertools             11\n",
       "3683                                             ena-dl              9\n",
       "3684                                          enasearch            471\n",
       "3685                                   encode-blacklist             93\n",
       "3686                                       encyclopedia              1\n",
       "3687                                            enrichm             16\n",
       "3688                                        ensembl-vep           3846\n",
       "3689                                      entrez-direct           6557\n",
       "3690                                           entrezpy             12\n",
       "3691                                            eoulsan            280\n",
       "3692                                             epa-ng            775\n",
       "3693                                              e-pcr             92\n",
       "3694                                          ephemeris           1457\n",
       "3695                                               epic           1561\n",
       "3696                                              epic2           1087\n",
       "3697                                            epicseg            104\n",
       "3698                                             epydoc             94\n",
       "3699                                            eqtlbma            192\n",
       "3700                                               erds             94\n",
       "3701                                         ericscript            175\n",
       "3702                                               erne             99\n",
       "3703                                             escher             17\n",
       "3704                                             esimsa             93\n",
       "3705                                              esmre            101\n",
       "3706                                          estmapper             96\n",
       "3707                                            estscan             96\n",
       "3708                                                eta            134\n",
       "3709                                               ete2            361\n",
       "3710                                               eval             56\n",
       "3711                                    evidencemodeler            502\n",
       "3712                                           evofold2             94\n",
       "3713                                          exonerate            764\n",
       "3714                                    expansionhunter            535\n",
       "3715                                            exparna             97\n",
       "3716                                    export2graphlan            218\n",
       "3717                                            express            208\n",
       "3718                            extract-codon-alignment              1\n",
       "3719                                  extract_fasta_seq              3\n",
       "3720                                    extract_fullseq            298\n",
       "3721                              extract_genome_region             99\n",
       "3722                                   extract-sv-reads            476\n",
       "3723                                         ezclermont              1\n",
       "3724                                             fadapa            382\n",
       "3725                                              falco             95\n",
       "3726                                              famsa            107\n",
       "3727                                             famseq             97\n",
       "3728                                               fann             95\n",
       "3729                                              faqcs            202\n",
       "3730                                            fargene             10\n",
       "3731                                              fast5           1060\n",
       "3732                                     fast5-research            161\n",
       "3733                                          fast5seek            189\n",
       "3734                                             fasta3            401\n",
       "3735                                          fastahack             93\n",
       "3736                                         fastaindex             97\n",
       "3737                                          fastalite              2\n",
       "3738                                            fastani            523\n",
       "3739                                        fastaptamer              1\n",
       "3740                                     fasta-splitter            102\n",
       "3741                                     fasta_ushuffle            122\n",
       "3742                                             fasten             94\n",
       "3743                                          fastindep             93\n",
       "3744                                            fastlmm            208\n",
       "3745                                             fastme            288\n",
       "3746                                             fastml             93\n",
       "3747                                            fastobo            359\n",
       "3748                                            fastool            345\n",
       "3749                                              fastp           7260\n",
       "3750                                          fastphylo             98\n",
       "3751                                  fastq-and-furious            384\n",
       "3752                                    fastq-anonymous            108\n",
       "3753                                             fastqc          14258\n",
       "3754                                        fastq-count             93\n",
       "3755                                           fastq-dl             12\n",
       "3756                                             fastqe            102\n",
       "3757                                         fastq-join            131\n",
       "3758                                       fastqmetrics            101\n",
       "3759                                        fastq-multx            267\n",
       "3760                                         fastq-pair            116\n",
       "3761                                          fastqpuri            395\n",
       "3762                                         fastq-scan            451\n",
       "3763                                       fastq-screen            859\n",
       "3764                                      fastqsplitter            197\n",
       "3765                                        fastq-tools            131\n",
       "3766                                        fastq_utils            684\n",
       "3767                                         fastsimbac             93\n",
       "3768                                           fastspar            310\n",
       "3769                                      faststructure            118\n",
       "3770                                           fasttree           3658\n",
       "3771                                           fastuniq            114\n",
       "3772                                          fastutils            101\n",
       "3773                                      fastx_toolkit            611\n",
       "3774                                                fcp             94\n",
       "3775                                          fcsparser             22\n",
       "3776                                      feature_merge              6\n",
       "3777                                             feelnc            141\n",
       "3778                                               feht            188\n",
       "3779                                              fermi            110\n",
       "3780                                             fermi2            198\n",
       "3781                                           fermikit             97\n",
       "3782                                         fermi-lite             95\n",
       "3783                                               fgap             97\n",
       "3784                                              fgbio           1106\n",
       "3785                                               fgmp            237\n",
       "3786                                              ficus            217\n",
       "3787                                               fido            190\n",
       "3788                                            figtree             62\n",
       "3789                                               fiji            318\n",
       "3790                                           filtlong            494\n",
       "3791                          find_differential_primers            204\n",
       "3792                                   fineradstructure             95\n",
       "3793                                      finestructure             97\n",
       "3794                                              fings              2\n",
       "3795                                          firecloud            379\n",
       "3796                                           fishtaco            144\n",
       "3797                                             fithic             17\n",
       "3798                                         fit_nbinom            914\n",
       "3799                                             fitter            126\n",
       "3800                                         flaimapper            570\n",
       "3801                                              flair             28\n",
       "3802                                           flapjack            297\n",
       "3803                                             flash2            257\n",
       "3804                                              flash            566\n",
       "3805                                           flashlfq            705\n",
       "3806                                            flexbar            317\n",
       "3807                                           flextaxd              3\n",
       "3808                                              flock            187\n",
       "3809                                          flowcraft            695\n",
       "3810                                     flux-simulator            108\n",
       "3811                                               flye           1596\n",
       "3812                                              fmlrc            191\n",
       "3813                                              focus             13\n",
       "3814                                          footprint            109\n",
       "3815                                              forgi            105\n",
       "3816                                        forked-path             98\n",
       "3817                                                fpa            484\n",
       "3818                                            fqtools            133\n",
       "3819                                             fqtrim             96\n",
       "3820                                            fqzcomp             93\n",
       "3821                                       fraggenescan            439\n",
       "3822                                                frc             95\n",
       "3823                                              fred2            211\n",
       "3824                                          freebayes           3075\n",
       "3825                                              frogs            197\n",
       "3826                                                fsa             97\n",
       "3827                                               fseq             99\n",
       "3828                                           fsm-lite            108\n",
       "3829                                             fsnviz            298\n",
       "3830                                               fuma            190\n",
       "3831                                        funannotate            452\n",
       "3832                                          funcannot             98\n",
       "3833                                             funnel            194\n",
       "3834                                      fusioncatcher            152\n",
       "3835                                fusioncatcher-seqtk            118\n",
       "3836                                      fusion-filter            103\n",
       "3837                                   fusion-inspector            314\n",
       "3838                                      fusion-report             10\n",
       "3839                                              fwdpp            387\n",
       "3840                                            fwdpy11           2315\n",
       "3841                                              fwdpy             97\n",
       "3842                                         ga4ghmongo            108\n",
       "3843                                              gadem            206\n",
       "3844                                  galaxy-containers             14\n",
       "3845                                  galaxy-ie-helpers            103\n",
       "3846                                         galaxy-lib           3664\n",
       "3847                                          galaxy-ml           1321\n",
       "3848                                         galaxy-ops            239\n",
       "3849                                      galaxy-parsec            110\n",
       "3850                              galaxy_sequence_utils            567\n",
       "3851                                   galaxy-tool-util             14\n",
       "3852                                        galaxy-util            104\n",
       "3853                                              ganon            489\n",
       "3854                                            gap2seq            320\n",
       "3855                                          gapfiller             93\n",
       "3856                                              gappa            865\n",
       "3857                                             garnet            472\n",
       "3858                                        garnett-cli              5\n",
       "3859                                                gat            443\n",
       "3860                                               gatb            121\n",
       "3861                                               gatk           1703\n",
       "3862                                              gatk4           3636\n",
       "3863                                     gatk-framework            295\n",
       "3864                                           gatktool            126\n",
       "3865                                            gblocks            247\n",
       "3866                                            gbmunge            101\n",
       "3867                                               gbsx              3\n",
       "3868                                  gb_taxonomy_tools            194\n",
       "3869                             gcs-oauth2-boto-plugin            138\n",
       "3870                                                gdc             94\n",
       "3871                                         gdc-client            108\n",
       "3872                                          gdsctools            197\n",
       "3873                                              geco2             33\n",
       "3874                                             gecode            285\n",
       "3875                                        gem3-mapper            214\n",
       "3876                                              gembs            119\n",
       "3877                                             gemini           1688\n",
       "3878                                              gemma            215\n",
       "3879                                          genblasta            186\n",
       "3880                                          genblastg            188\n",
       "3881                                           genclust             94\n",
       "3882                                            gencore             93\n",
       "3883                                           genefuse             88\n",
       "3884                                        geneimpacts           2254\n",
       "3885                                       genenotebook           1519\n",
       "3886                                         genepender             95\n",
       "3887                                            genepop           1635\n",
       "3888                                            generax             97\n",
       "3889                                genericrepeatfinder            110\n",
       "3890                                        genesplicer            328\n",
       "3891                                            genform            107\n",
       "3892                                              genie            470\n",
       "3893                                             genmap            197\n",
       "3894                                         genologics             99\n",
       "3895                                        genomebaser             95\n",
       "3896                                genomeconstellation             93\n",
       "3897                                         genomedata            645\n",
       "3898                                         genomelake            448\n",
       "3899                                           genomepy            863\n",
       "3900                                        genomestrip             97\n",
       "3901                                      genometester4            101\n",
       "3902                                     genomethreader            121\n",
       "3903                                        genometools            129\n",
       "3904                            genometools-genometools            666\n",
       "3905                                     genome_updater              8\n",
       "3906                                  genomicassertions            110\n",
       "3907                                   genomicconsensus            364\n",
       "3908                                           genonets            104\n",
       "3909                                            genrich            241\n",
       "3910                                              geodl            197\n",
       "3911                                          geosketch             13\n",
       "3912                                               gerp             95\n",
       "3913                                              gfapy            317\n",
       "3914                                           gff3sort            159\n",
       "3915                                         gff3toembl            102\n",
       "3916                                         gffcompare            878\n",
       "3917                                          gffmunger             14\n",
       "3918                                            gffread           1034\n",
       "3919                                           gffutils           2548\n",
       "3920                                          gfinisher            109\n",
       "3921                                              gfold            115\n",
       "3922                                                ggd             12\n",
       "3923                                                ghm            196\n",
       "3924                                               ghmm            103\n",
       "3925                                         ghost-tree              5\n",
       "3926                                             ghostx            115\n",
       "3927                                             ghostz            198\n",
       "3928                                        gimmemotifs           1812\n",
       "3929                                            glimmer            212\n",
       "3930                                         glimmerhmm           1076\n",
       "3931                                               gmap           2006\n",
       "3932                                        gmap-fusion            202\n",
       "3933                                           gmcloser              7\n",
       "3934                                               gmtk            137\n",
       "3935                                             gneiss           2516\n",
       "3936                                             gnomic             12\n",
       "3937                                         gnu-getopt            113\n",
       "3938                                         gnuplot-py            107\n",
       "3939                                           gnu-wget            141\n",
       "3940                                            goalign            340\n",
       "3941                                           goatools            711\n",
       "3942                                               gocr             96\n",
       "3943                                       goenrichment             11\n",
       "3944                                             goleft           1781\n",
       "3945                                          goslimmer             10\n",
       "3946                                          gotohscan             97\n",
       "3947                                             gotree            139\n",
       "3948                                            gottcha             99\n",
       "3949                                 gprofiler-official            120\n",
       "3950                                                gqt            100\n",
       "3951                                             grabix            653\n",
       "3952                                             graftm            202\n",
       "3953                                      graph2pro-var             94\n",
       "3954                                       graphaligner            963\n",
       "3955                                graphclust-wrappers           1447\n",
       "3956                                        graph_embed            193\n",
       "3957                                           graphlan            261\n",
       "3958                                          graphlite            117\n",
       "3959                                           graphmap            407\n",
       "3960                                          graphprot            209\n",
       "3961                                               grep            994\n",
       "3962                                             gretel            196\n",
       "3963                                               grid            306\n",
       "3964                                             gridss           1304\n",
       "3965                                            gromacs           1390\n",
       "3966                                             groopm             95\n",
       "3967                                              groot           1230\n",
       "3968              group_humann2_uniref_abundances_to_go            110\n",
       "3969                                            gsalign           1231\n",
       "3970                                             gseapy           1942\n",
       "3971                                              gsort            865\n",
       "3972                                             gtdbtk            934\n",
       "3973                                           gtfparse            282\n",
       "3974                                       gtf_to_genes            100\n",
       "3975                                            gubbins            811\n",
       "3976                                          guessmylt              2\n",
       "3977                                          guidescan            197\n",
       "3978                                             gustaf             97\n",
       "3979                                           gvcf2bed              6\n",
       "3980                                      gvcfgenotyper            314\n",
       "3981                                       gvcf-regions            546\n",
       "3982                                          gvcftools            117\n",
       "3983                                              gwama             94\n",
       "3984                                          gxformat2           1311\n",
       "3985                                              halla            103\n",
       "3986                                            hanselx            101\n",
       "3987                                             hapbin            189\n",
       "3988                                            hapcut2            427\n",
       "3989                                             hapflk            100\n",
       "3990                                        haploclique             97\n",
       "3991                                       haploconduct             95\n",
       "3992                                      haplotype-lso             47\n",
       "3993                                             happer              1\n",
       "3994                                             hap.py            409\n",
       "3995                                           hardklor            206\n",
       "3996                                       harvesttools            115\n",
       "3997                                              haslr             50\n",
       "3998                                       haystack_bio            300\n",
       "3999                                                hca             21\n",
       "4000                              hca-matrix-downloader              3\n",
       "4001                                            hclust2            254\n",
       "4002                                        hcluster_sg            101\n",
       "4003                                              heinz            106\n",
       "4004                                         helperlibs            173\n",
       "4005                                               hera            304\n",
       "4006                                           hg-color            199\n",
       "4007                                            hgtools            124\n",
       "4008                                            hhsuite            357\n",
       "4009                                           hic2cool           1064\n",
       "4010                                              hicap             12\n",
       "4011                                         hicbrowser             96\n",
       "4012                                        hicexplorer           2404\n",
       "4013                                          hichipper            105\n",
       "4014                                          hicmatrix           1371\n",
       "4015                                              hicup            302\n",
       "4016                                      hiddendomains             96\n",
       "4017                                             hifive            753\n",
       "4018                                            hilive2             94\n",
       "4019                                               hint             19\n",
       "4020                                             hisat2           3302\n",
       "4021                                               hits            196\n",
       "4022                                           hivtrace            103\n",
       "4023                                             hla-la            204\n",
       "4024                                              hlama            287\n",
       "4025                                        hlaprofiler            468\n",
       "4026                                     hmftools-amber            149\n",
       "4027                                    hmftools-cobalt            149\n",
       "4028                                      hmftools-linx             52\n",
       "4029                                    hmftools-purple            256\n",
       "4030                                      hmftools-sage              2\n",
       "4031                                            hmmcopy            107\n",
       "4032                                             hmmer2            186\n",
       "4033                                              hmmer           7583\n",
       "4034                                           hmmratac             10\n",
       "4035                                              homer            896\n",
       "4036                                               hops            105\n",
       "4037                                          hotspot3d            190\n",
       "4038                                           howdesbt            283\n",
       "4039                                                hr2             96\n",
       "4040                                          hs-blastn            113\n",
       "4041                                             htsbox            380\n",
       "4042                                              htseq           1945\n",
       "4043                                             htslib          61473\n",
       "4044                                      hts-nim-tools            233\n",
       "4045                                           htstream            221\n",
       "4046                                            hubward            279\n",
       "4047                                        hubward-all            101\n",
       "4048                                               hulk            478\n",
       "4049                                            humann2            807\n",
       "4050                                     hurry.filesize            100\n",
       "4051                                       hypercluster            128\n",
       "4052                                              hyphy            574\n",
       "4053                                               hypo            227\n",
       "4054                                              ibdne            104\n",
       "4055                                             ibdseq            101\n",
       "4056                                               iced            322\n",
       "4057                                            iclipro             96\n",
       "4058                                             icount            102\n",
       "4059                                             icqsol            187\n",
       "4060                                               idba            432\n",
       "4061                                        idba_subasm            203\n",
       "4062                                              ideas             97\n",
       "4063                                                idr            509\n",
       "4064                                             idr-py            312\n",
       "4065                                            igblast            553\n",
       "4066                                        ig-checkfcs             94\n",
       "4067                                  ig-checkflowtypes            103\n",
       "4068                                         igdiscover            886\n",
       "4069                                       ig-flowtools            720\n",
       "4070                                           igor_vdj             93\n",
       "4071                                            igphyml             93\n",
       "4072                                                igv            730\n",
       "4073                                        igv-reports            136\n",
       "4074                                           igvtools            670\n",
       "4075                                   illumina-cleanup             94\n",
       "4076                                   illumina-interop            562\n",
       "4077                                         illuminate            106\n",
       "4078                                     illumina-utils            258\n",
       "4079                                    illumiprocessor            216\n",
       "4080                                           imctools              6\n",
       "4081                                           imfusion            196\n",
       "4082                                            impute2            207\n",
       "4083                                              imseq             98\n",
       "4084                                           imsindel             15\n",
       "4085                                         indelfixer             95\n",
       "4086                                           infernal           2367\n",
       "4087                                          influx_si             43\n",
       "4088                                            inforna             96\n",
       "4089                                        inheritance            948\n",
       "4090                                        insilicoseq             78\n",
       "4091                                            intarna           1767\n",
       "4092                                            intemap             95\n",
       "4093                                         interleafq            113\n",
       "4094                                          intermine            370\n",
       "4095                                      intervalstats             96\n",
       "4096                                   intervaltree_bio            116\n",
       "4097                                          intervene            429\n",
       "4098                                          involucro            855\n",
       "4099                                             ipyrad            393\n",
       "4100                             ipython-cluster-helper           1680\n",
       "4101                                             iqtree           4687\n",
       "4102                                                 ir             93\n",
       "4103                                               irep            103\n",
       "4104                                           irfinder            470\n",
       "4105                                       irida-linker            633\n",
       "4106                                irida-sistr-results            388\n",
       "4107                                     irida-uploader             19\n",
       "4108                                           isatools            424\n",
       "4109                                            isescan            110\n",
       "4110                                         islandpath            196\n",
       "4111                                           ismapper             14\n",
       "4112                                             isocor            104\n",
       "4113                                            isoseq3            655\n",
       "4114                                              ispcr            102\n",
       "4115                                              itero            595\n",
       "4116                                               itsx            159\n",
       "4117                                          itsxpress            624\n",
       "4118                                           iucn_sim             10\n",
       "4119                                                iva            305\n",
       "4120                                               ivar            200\n",
       "4121                                              jaffa            203\n",
       "4122                                               jali             93\n",
       "4123                                            jalview            687\n",
       "4124                                               jamm            299\n",
       "4125                                       jannovar-cli            649\n",
       "4126                                             jarvis             33\n",
       "4127                                            jasmine              4\n",
       "4128                                               jass              5\n",
       "4129                                 jass_preprocessing              8\n",
       "4130                                            jbrowse           1166\n",
       "4131                                               jcvi            936\n",
       "4132                                           je-suite            206\n",
       "4133                                           jmztab-m              1\n",
       "4134                                            jobtree             46\n",
       "4135                                           jolytree              5\n",
       "4136                                           jpredapi              6\n",
       "4137                           json_collect_data_source            200\n",
       "4138                                   jupyterngsplugin             16\n",
       "4139                                        justbackoff            106\n",
       "4140                                    jvarkit-bam2svg             94\n",
       "4141                                    jvarkit-bam2wig            102\n",
       "4142                                    jvarkit-msa2vcf              3\n",
       "4143                                              kaiju           1133\n",
       "4144                                            kalign2            118\n",
       "4145                                           kallisto           2958\n",
       "4146                                            kaptive            330\n",
       "4147                                             karect             99\n",
       "4148                                               kart            465\n",
       "4149                                          karyopype              1\n",
       "4150                                                kat            675\n",
       "4151                                             kfoots            103\n",
       "4152                                             kggseq              2\n",
       "4153                                              khmer           1068\n",
       "4154                                                kid             96\n",
       "4155                                   kinsimriboswitch             96\n",
       "4156                                              kipoi            158\n",
       "4157                                           kipoiseq             38\n",
       "4158                                        kipoi-utils             25\n",
       "4159                                         kipoi_veff             11\n",
       "4160                                          kissplice            152\n",
       "4161                                           kiwidist            291\n",
       "4162                                          kleborate             16\n",
       "4163                                                kma           1834\n",
       "4164                                            kmasker            191\n",
       "4165                                                kmc            835\n",
       "4166                                          kmergenie            130\n",
       "4167                                        kmerinshort            117\n",
       "4168                                     kmer-jellyfish           1306\n",
       "4169                                          kneaddata            286\n",
       "4170                                        knock-knock              5\n",
       "4171                                               knot             93\n",
       "4172                                              kobas            249\n",
       "4173                                             kodoja            230\n",
       "4174                                             koeken             97\n",
       "4175                                          kofamscan             24\n",
       "4176                                               kopt             16\n",
       "4177                                             kounta              2\n",
       "4178                                               kpal            101\n",
       "4179                                             kraken           1017\n",
       "4180                                            kraken2            752\n",
       "4181                                        kraken-biom            137\n",
       "4182                                          kraken-ea            101\n",
       "4183                                         krakenuniq            628\n",
       "4184                                        krbalancing           1208\n",
       "4185                                             krocus            205\n",
       "4186                                              krona           1011\n",
       "4187                                             kronik            103\n",
       "4188                                             kronos              2\n",
       "4189                                             k-slam             93\n",
       "4190                                               kssd             94\n",
       "4191                                                ksw            376\n",
       "4192                                               kwip            106\n",
       "4193                                        kyototycoon            146\n",
       "4194                                               lace            346\n",
       "4195                                             lambda            192\n",
       "4196                                                lap             93\n",
       "4197                                               last           1461\n",
       "4198                                              lastz            416\n",
       "4199                               latentstrainanalysis             99\n",
       "4200                                                lca             94\n",
       "4201                                              lcfit            188\n",
       "4202                                           ldhelmet             97\n",
       "4203                                               ldsc              2\n",
       "4204                                              lefse            368\n",
       "4205                                             legsta              5\n",
       "4206                                            lexmapr            938\n",
       "4207                                         libbambamc            100\n",
       "4208                                          libbigwig            589\n",
       "4209                                      libdivsufsort            113\n",
       "4210                                          libgenome            233\n",
       "4211                                      libgtextutils            537\n",
       "4212                                             libidn            134\n",
       "4213                                              libis            159\n",
       "4214                                            libmems            217\n",
       "4215                                          libmuscle            218\n",
       "4216                                            libsbml             96\n",
       "4217                                        libsequence            958\n",
       "4218                                             libssw            102\n",
       "4219                                         libstatgen            189\n",
       "4220                                     lightassembler             93\n",
       "4221                                            lighter            332\n",
       "4222                                          lightning            101\n",
       "4223                                   lightstringgraph             93\n",
       "4224                                               lima            743\n",
       "4225                                    linkage2allegro            311\n",
       "4226                                              links            344\n",
       "4227                                         livekraken            125\n",
       "4228                                            locarna           1166\n",
       "4229                                   locus_processing             25\n",
       "4230                                             lofreq            521\n",
       "4231                                              logol             95\n",
       "4232                                          logomaker            138\n",
       "4233                                             lohhla            109\n",
       "4234                                           longshot            616\n",
       "4235                                             lordec            215\n",
       "4236                                           lordfast            193\n",
       "4237                                           lorikeet            195\n",
       "4238                                              lorma            104\n",
       "4239                                               lrez            527\n",
       "4240                                              lrzip           1729\n",
       "4241                                              lrzip             98\n",
       "4242                                                lsc           1627\n",
       "4243                                                lsc             95\n",
       "4244                                                lsd            101\n",
       "4245                                                lsd           1911\n",
       "4246                                             ls-gkm            139\n",
       "4247                                             ls-gkm           2985\n",
       "4248                                         ltr_finder           1222\n",
       "4249                                         ltr_finder             50\n",
       "4250                                      ltr_retriever           1522\n",
       "4251                                      ltr_retriever             53\n",
       "4252                                          luciphor2             45\n",
       "4253                                          luciphor2           7580\n",
       "4254                                           lumpy-sv          22513\n",
       "4255                                           lumpy-sv            790\n",
       "4256                                             lusstr             66\n",
       "4257                                              lyner              4\n",
       "4258                                              lyner             94\n",
       "4259                                            lz4-bin            122\n",
       "4260                                            lz4-bin           8631\n",
       "4261                                                 ma           1329\n",
       "4262                                                 ma             96\n",
       "4263                                            maaslin            198\n",
       "4264                                           maaslin2             12\n",
       "4265                                           maaslin2            378\n",
       "4266                                            maaslin           3673\n",
       "4267                                            macaron              8\n",
       "4268                                            macaron             99\n",
       "4269                                               mace           1140\n",
       "4270                                               mace             95\n",
       "4271                                             macrel            184\n",
       "4272                                             macrel           2453\n",
       "4273                                              macs2           2064\n",
       "4274                                              macs2          49852\n",
       "4275                                              macse             37\n",
       "4276                                              mafft         174314\n",
       "4277                                              mafft           5345\n",
       "4278                                             mageck           1513\n",
       "4279                                             mageck          31030\n",
       "4280                                       mageck-vispr          13335\n",
       "4281                                       mageck-vispr            667\n",
       "4282                                         magicblast            205\n",
       "4283                                         magicblast           2219\n",
       "4284                                          magpurify           3211\n",
       "4285                                            makehub           1365\n",
       "4286                                              maker          21204\n",
       "4287                                             malder           1431\n",
       "4288                                               malt           2572\n",
       "4289                                              malva           4573\n",
       "4290                                            mamotif           3323\n",
       "4291                                              mango            244\n",
       "4292                                             manorm           8489\n",
       "4293                                              manta          43748\n",
       "4294                                             mantis           1486\n",
       "4295                                          mapcaller          17608\n",
       "4296                                         mapdamage2           3044\n",
       "4297                                             mapdia           3169\n",
       "4298                                              mapgl            216\n",
       "4299                        mapping-iterative-assembler           1270\n",
       "4300                                              mappy          44044\n",
       "4301                                        mapsembler2           2169\n",
       "4302                                          mapsplice           4582\n",
       "4303                                              marge           2013\n",
       "4304                            mark-nonconverted-reads            168\n",
       "4305                                             marvel            106\n",
       "4306                                               mash          32784\n",
       "4307                                            mashmap           5341\n",
       "4308                                           mashtree          21657\n",
       "4309                                         maskrc-svg            194\n",
       "4310                                              mason           5302\n",
       "4311                                            masurca          13922\n",
       "4312                                              matam            942\n",
       "4313                                          mathstats           4486\n",
       "4314                                            matlock           1491\n",
       "4315                                              mauve           4514\n",
       "4316                                       mauvealigner           3949\n",
       "4317                                              mavis             58\n",
       "4318                                               mawk           6831\n",
       "4319                                            maxbin2          11184\n",
       "4320                                           maxentpy           4593\n",
       "4321                                         maxentscan           7595\n",
       "4322                                           maxquant           3944\n",
       "4323                                                mbg            183\n",
       "4324                                            mbuffer           7388\n",
       "4325                                           mccortex           2617\n",
       "4326                                                mcl          30556\n",
       "4327                                             mdasim           7871\n",
       "4328                                              mdust           2844\n",
       "4329                                                mea           2337\n",
       "4330                                             medaka          29619\n",
       "4331                                              medpy           3118\n",
       "4332                                             medusa            178\n",
       "4333                                 medusa-data-fusion           2185\n",
       "4334                                            megagta           1680\n",
       "4335                                            megahit          43017\n",
       "4336                                          megalodon           2383\n",
       "4337                                              megan           2393\n",
       "4338                                               melt           2832\n",
       "4339                                               meme          46318\n",
       "4340                                         mendelscan           1801\n",
       "4341                                             meneco           1935\n",
       "4342                                          menetools           1955\n",
       "4343                                          mentalist          22971\n",
       "4344                                         meraculous           6600\n",
       "4345                                             mercat           3950\n",
       "4346                                       merfishtools          13680\n",
       "4347                                   mergenotcombined            401\n",
       "4348                                           mergevcf           1835\n",
       "4349                                             merlin           3353\n",
       "4350                                            merqury             94\n",
       "4351                                              meryl           2168\n",
       "4352                                           metabat2          12728\n",
       "4353                                         metabinkit            826\n",
       "4354                                      metacherchant           1854\n",
       "4355                                        metacluster           1777\n",
       "4356                                            metaeuk           1292\n",
       "4357                                 metagene_annotator           3960\n",
       "4358                                   metagenome-atlas           7499\n",
       "4359                                           metalign            124\n",
       "4360                                           metamaps            495\n",
       "4361                                           metameta           7344\n",
       "4362                                      metametamerge           4251\n",
       "4363                                         metaphlan2         119507\n",
       "4364                                          metaphlan           2546\n",
       "4365                                         metaphyler           1802\n",
       "4366                                            metapop             23\n",
       "4367                                           metaprob           1774\n",
       "4368                                         metaprokka             36\n",
       "4369                                          metaquant           1714\n",
       "4370                                       metaquantome           1320\n",
       "4371                                            metaseq           4921\n",
       "4372                                        metaseq-all           2009\n",
       "4373                                            metasnv           4595\n",
       "4374                                        meta-sparse           4904\n",
       "4375                                             metasv           8319\n",
       "4376                                           metatree             21\n",
       "4377                                         metavelvet           4116\n",
       "4378                                  metavelvet-annois           2013\n",
       "4379                                      metavelvet-sl           1871\n",
       "4380                   metavelvet-sl-feature-extraction           2234\n",
       "4381                             metavelvet-sl-pipeline           1558\n",
       "4382                                           metawatt           1974\n",
       "4383                                           metawrap           4248\n",
       "4384                                             metaxa           1826\n",
       "4385                                            metfrag           5507\n",
       "4386                                        methplotlib            427\n",
       "4387                                       methyldackel          12831\n",
       "4388                                      methylextract           1752\n",
       "4389                                           methylpy           2180\n",
       "4390                                           metilene           6151\n",
       "4391                                               mfqe           3152\n",
       "4392                                      mgf-formatter           2790\n",
       "4393                                              mgkit           9904\n",
       "4394                                              mglex           2064\n",
       "4395                                           mgltools           6253\n",
       "4396                                         mgs-canopy           1869\n",
       "4397                                               mhap           3717\n",
       "4398                                          mhcflurry           1161\n",
       "4399                                           mhcnames            769\n",
       "4400                                         mhcnuggets           1843\n",
       "4401                                        micall-lite           4276\n",
       "4402                                      microbecensus           3705\n",
       "4403                                         microhapdb            400\n",
       "4404                                     microhapulator            332\n",
       "4405                                          micropita           1319\n",
       "4406                                       micro_razers           1773\n",
       "4407                                              midas           5962\n",
       "4408                                             migmap           5834\n",
       "4409                                           migraine           3222\n",
       "4410                                          migrate-n           1750\n",
       "4411                                             mikado          10671\n",
       "4412                                              mimeo           2055\n",
       "4413                                             mimodd           6862\n",
       "4414                                             mimseq            120\n",
       "4415                                             minced          35151\n",
       "4416                                         mindthegap           1998\n",
       "4417                                              minia           3591\n",
       "4418                                          minialign          27269\n",
       "4419                                            miniasm          36543\n",
       "4420                                           minimap2         118765\n",
       "4421                                            minimap           4646\n",
       "4422                                           minimock           2242\n",
       "4423                                         minipolish            150\n",
       "4424                                            minirmd             31\n",
       "4425                                             minnow           4042\n",
       "4426                                           minorseq           2440\n",
       "4427                                            mintmap           6147\n",
       "4428                                             minvar          18552\n",
       "4429                                              minys           1402\n",
       "4430                                               mira           9216\n",
       "4431                                            miranda           2898\n",
       "4432                                           mirdeep2          12705\n",
       "4433                                             mirfix            100\n",
       "4434                                              mirge          11618\n",
       "4435                                         mir-prefer           5429\n",
       "4436                                             mirtop          25084\n",
       "4437                                           mirtrace            295\n",
       "4438                                             misopy           2806\n",
       "4439                                              mitos           1272\n",
       "4440                                               mlpy           1275\n",
       "4441                                              mlrho           1754\n",
       "4442                                               mlst          27672\n",
       "4443                                            mmseqs2          23434\n",
       "4444                                               mmvc           1699\n",
       "4445                                              mnnpy            333\n",
       "4446                                              moabs          14268\n",
       "4447                                        mobidic-mpa            125\n",
       "4448                                            mobster           9231\n",
       "4449                                          mob_suite           9622\n",
       "4450                                               moca           7954\n",
       "4451                                         mockinbird           2393\n",
       "4452                                       modeltest-ng           2581\n",
       "4453                                            modlamp             94\n",
       "4454                                             mofapy            538\n",
       "4455                                               moff           9928\n",
       "4456                                 molpopgen-analysis           2019\n",
       "4457                                       monocle3-cli           3251\n",
       "4458                                            monovar           2032\n",
       "4459                                              moods           7790\n",
       "4460                                          moreutils           8400\n",
       "4461                                           morpheus           3571\n",
       "4462                                             mosaik           3603\n",
       "4463                                              mosca             42\n",
       "4464                                           mosdepth          42114\n",
       "4465                                             mothur          27931\n",
       "4466                                        motifraptor           2075\n",
       "4467                                          motifscan           4306\n",
       "4468                                              motus           9593\n",
       "4469                                         mp3treesim             65\n",
       "4470                                       mpa-portable           2479\n",
       "4471                                         mpa-server            173\n",
       "4472                            mpra-data-access-portal             90\n",
       "4473                                                mqc          15800\n",
       "4474                                            mrbayes           5568\n",
       "4475                                              mreps           4007\n",
       "4476                                            mrsfast           1276\n",
       "4477                                                 ms           1634\n",
       "4478                                             ms2pip            202\n",
       "4479                                            msaboot           5937\n",
       "4480                                       msaconverter             74\n",
       "4481                                           msalign2           1541\n",
       "4482                                           msaprobs            445\n",
       "4483                                          msgf_plus          17142\n",
       "4484                                          msisensor          19094\n",
       "4485                                      msisensor-pro           3441\n",
       "4486                                               msms           3456\n",
       "4487                                             msp2db            199\n",
       "4488                                  msproteomicstools           3554\n",
       "4489                                           msstitch          20195\n",
       "4490                                             mtbseq           2076\n",
       "4491                                         mtb-snp-it            441\n",
       "4492                                            mtglink             75\n",
       "4493                                         mtnucratio           3613\n",
       "4494                                               mtsv           5181\n",
       "4495                                         mtsv-tools           1220\n",
       "4496                                              mugsy           2217\n",
       "4497                                           multigps           7714\n",
       "4498                                            multiqc         187074\n",
       "4499                                      multiqc-bcbio          47178\n",
       "4500                                   multivcfanalyzer           2416\n",
       "4501                                             multiz           1432\n",
       "4502                                             mummer          38688\n",
       "4503                                            mummer4          17301\n",
       "4504                                             muscle          73723\n",
       "4505                                               muse           2539\n",
       "4506                                              music           2000\n",
       "4507                                             musicc           4274\n",
       "4508                                music-deconvolution             88\n",
       "4509                                            mustang           1931\n",
       "4510                                             mutmap            819\n",
       "4511                                            mutscan            669\n",
       "4512                                            mvicuna          11906\n",
       "4513                                                mvp           3889\n",
       "4514                                         mwga-utils           2121\n",
       "4515                                             mygene          27902\n",
       "4516                                           mykatlas           2995\n",
       "4517                                            mykrobe           8996\n",
       "4518                                             myriad           3430\n",
       "4519                                           mzml2isa           4650\n",
       "4520                                         mztosqlite           4441\n",
       "4521                                                n50           2051\n",
       "4522                                        nanoblaster           1767\n",
       "4523                                           nanocall          11316\n",
       "4524                                           nanocomp          17974\n",
       "4525                                        nanocompore            396\n",
       "4526                                           nanofilt          19048\n",
       "4527                                            nanoget          29192\n",
       "4528                                           nanolyse           9098\n",
       "4529                                           nanomath          25320\n",
       "4530                                            nanomod             28\n",
       "4531                                            nanonet           1906\n",
       "4532                                         nanoplexer           1787\n",
       "4533                                           nanoplot          30660\n",
       "4534                                        nanoplotter          29508\n",
       "4535                                         nanopolish          36116\n",
       "4536                                     nanopolishcomp            206\n",
       "4537                                nanopore_simulation           1982\n",
       "4538                                             nanoqc          11656\n",
       "4539                                            nanoraw           8755\n",
       "4540                                            nanosim           9324\n",
       "4541                                          nanosim-h           5432\n",
       "4542                                          nanosplit           1724\n",
       "4543                                           nanostat          13733\n",
       "4544                                             nanosv           9471\n",
       "4545                                            nanovar           3297\n",
       "4546                                               nasp           7287\n",
       "4547                                  ncbi-acc-download            718\n",
       "4548                                 ncbi-amrfinderplus          12208\n",
       "4549                               ncbi-genome-download          22755\n",
       "4550                                           ncbimeta            688\n",
       "4551                                       ncbi-ngs-sdk          35929\n",
       "4552                                             ncbitk             59\n",
       "4553                                   ncbi-util-legacy           1297\n",
       "4554                                           ncbi-vdb          11333\n",
       "4555                                               ncfp           2011\n",
       "4556                                               ncls          37462\n",
       "4557                                               ncrf           3938\n",
       "4558                                        ndex-python           1981\n",
       "4559                                             neatms             50\n",
       "4560                                          nebulizer             86\n",
       "4561                                              necat             47\n",
       "4562                          negative_training_sampler            134\n",
       "4563                                               nemo           1475\n",
       "4564                                           nemo-age            165\n",
       "4565                                            neptune           5416\n",
       "4566                               netcdf-metadata-info           1760\n",
       "4567                                       networkxgmml           2077\n",
       "4568                                       newick_utils           4752\n",
       "4569                                           nextflow          89664\n",
       "4570                                         nextgenmap           6928\n",
       "4571                                            nf-core           1511\n",
       "4572                                             ngless          21918\n",
       "4573                                            ngmerge           2104\n",
       "4574                                              ngmlr          13147\n",
       "4575                                           ngs-bits          15231\n",
       "4576                                          ngsderive            139\n",
       "4577                                   ngs-disambiguate          19377\n",
       "4578                                              ngsep             26\n",
       "4579                                         ngseqbasic           1684\n",
       "4580                                        ngshmmalign           1808\n",
       "4581                                           ngsutils           2819\n",
       "4582                                            niemads             28\n",
       "4583                                         nim-falcon          10824\n",
       "4584                                           nimnexus           2837\n",
       "4585                                        nlstradamus           1976\n",
       "4586                                            nmrglue           6164\n",
       "4587                                          nmrml2isa           1903\n",
       "4588                                             nmrpro           2048\n",
       "4589                                          nonpareil          11092\n",
       "4590                                              norns           9751\n",
       "4591                                 nose-capturestderr           3369\n",
       "4592                                         novasplice            188\n",
       "4593                                          novoalign          29615\n",
       "4594                                          novobreak           2222\n",
       "4595                                         novoplasty           2275\n",
       "4596                                              npinv            256\n",
       "4597                                 np-likeness-scorer           2162\n",
       "4598                                          npyscreen           1953\n",
       "4599                                               nseg           2505\n",
       "4600                                              nspdk           3074\n",
       "4601                                             ntcard           3243\n",
       "4602                                             ntedit           1058\n",
       "4603                                             nthits            678\n",
       "4604                                             ntjoin           1488\n",
       "4605                                           nucamino           1694\n",
       "4606                                            nucdiff           4155\n",
       "4607                                         nucleoatac           4002\n",
       "4608                              nucleosome_prediction           1600\n",
       "4609                                              nudup           6931\n",
       "4610                                          nullarbor           2227\n",
       "4611                                                nvc           3591\n",
       "4612                                             nxtrim            883\n",
       "4613                                              oases           4030\n",
       "4614                                           obitools           6945\n",
       "4615                                              ococo           7084\n",
       "4616                                              ocrad           1980\n",
       "4617                                            octopus          15638\n",
       "4618                                               odgi           1673\n",
       "4619                                              odose           1598\n",
       "4620                                               ogdf           1843\n",
       "4621                                      oligoarrayaux           1397\n",
       "4622                                        oligotyping           3276\n",
       "4623                                           omero-py             58\n",
       "4624                                              omssa           7798\n",
       "4625                                           oncofuse           9614\n",
       "4626                                         oncogemini             68\n",
       "4627                                           oncopipe             67\n",
       "4628                                          oncotator           1103\n",
       "4629                                      ont-fast5-api          10166\n",
       "4630                                            onto2nx           2746\n",
       "4631                                          ont-tombo          22094\n",
       "4632                                     opencontactcli           1699\n",
       "4633                                           openduck            135\n",
       "4634                                             openmg           2305\n",
       "4635                                   openslide-python           6361\n",
       "4636                                        opentargets             86\n",
       "4637                             opentargets-urlzsource             25\n",
       "4638                              opentargets-validator             21\n",
       "4639                                              opsin           5885\n",
       "4640                                           optbuild           2658\n",
       "4641                                            optimir             65\n",
       "4642                                           optitype          18313\n",
       "4643                                            optplus           2774\n",
       "4644                                               orfm           3889\n",
       "4645                                           orfquant             52\n",
       "4646                                        orthofinder          20333\n",
       "4647                                           orthomcl           3176\n",
       "4648                                               osra           5639\n",
       "4649                                          outrigger           2215\n",
       "4650                                      pacbio_falcon           2103\n",
       "4651                                              pafpy             63\n",
       "4652                                             pairix          51497\n",
       "4653                                            pairsnp            964\n",
       "4654                                          pairtools           8578\n",
       "4655                                            pal2nal            824\n",
       "4656                                            paladin           5099\n",
       "4657                                         pal_finder           2006\n",
       "4658                                               paml          80195\n",
       "4659                                            panaroo            610\n",
       "4660                                            pancake           2037\n",
       "4661                                           pandaseq           7844\n",
       "4662                                           pangolin            468\n",
       "4663                                             panoct           1781\n",
       "4664                                           panpasco             76\n",
       "4665                                           panphlan           2033\n",
       "4666                                        pan-plaster             21\n",
       "4667                                           pantools           2155\n",
       "4668                                               panx           3375\n",
       "4669                                            paraclu            336\n",
       "4670                                            parafly          13643\n",
       "4671                                          paragraph           3159\n",
       "4672                                parallel-fastq-dump          17800\n",
       "4673                                          paralyzer           3001\n",
       "4674                                    parasail-python          17766\n",
       "4675                                         parent-map            101\n",
       "4676                                             parsnp           3890\n",
       "4677                                               pasa           4919\n",
       "4678                                               pash           1612\n",
       "4679                                              pasta           4476\n",
       "4680                                  pathogen-profiler           1919\n",
       "4681                                          pathogist            519\n",
       "4682                                          patholive             55\n",
       "4683                                         pathoscope           3725\n",
       "4684                                          pathracer            143\n",
       "4685                                     pathwaymatcher           2611\n",
       "4686                                              pauda           1707\n",
       "4687                                             pauvre          14450\n",
       "4688                                            pbalign           6283\n",
       "4689                                        pb-assembly          15176\n",
       "4690                                              pbbam          21989\n",
       "4691                                              pbccs          11619\n",
       "4692                                          pbcommand          10742\n",
       "4693                                           pbcopper          12710\n",
       "4694                                             pbcore          13175\n",
       "4695                                        pbcoretools           4317\n",
       "4696                                           pbdagcon           2221\n",
       "4697                                         pb-dazzler           6361\n",
       "4698                                          pb-falcon          31787\n",
       "4699                                         pb-falconc           1868\n",
       "4700                                    pb-falcon-phase           1837\n",
       "4701                                             pbgcpp           5107\n",
       "4702                                             pbgzip          73784\n",
       "4703                                          pbh5tools           2346\n",
       "4704                                           pbhoover           3318\n",
       "4705                                              pbipa           1453\n",
       "4706                                              pblaa           2148\n",
       "4707                                              pblat            943\n",
       "4708                                          pbmarkdup           1259\n",
       "4709                                              pbmm2          23306\n",
       "4710                                              pbsim           2259\n",
       "4711                                               pbsv          11940\n",
       "4712                                               pbwt           1714\n",
       "4713                                             pdfkit           5298\n",
       "4714                                           peakachu           3045\n",
       "4715                                         peakranger           3476\n",
       "4716                                            peaksql            102\n",
       "4717                                          peakzilla           2358\n",
       "4718                                               pear          12803\n",
       "4719                                              peddy          31278\n",
       "4720                                         ped_parser           3699\n",
       "4721                                          pepgenome            129\n",
       "4722                                            pepnovo           9187\n",
       "4723                                           pepquery            389\n",
       "4724                                               pepr           5949\n",
       "4725                                     peptide-shaker          45590\n",
       "4726                                            perbase            173\n",
       "4727                                         percolator          12501\n",
       "4728                                       perl-aceperl          91047\n",
       "4729                                     perl-acme-damn           5166\n",
       "4730                             perl-algorithm-cluster           9711\n",
       "4731                          perl-algorithm-dependency           3683\n",
       "4732                                perl-algorithm-diff          82117\n",
       "4733                             perl-algorithm-munkres          91185\n",
       "4734                                       perl-aliased           1987\n",
       "4735                                   perl-apache-test          91574\n",
       "4736                                     perl-appconfig          91165\n",
       "4737                                 perl-app-cpanminus         256642\n",
       "4738                               perl-archive-extract          30932\n",
       "4739                                   perl-archive-tar         183935\n",
       "4740                           perl-archive-tar-wrapper           1194\n",
       "4741                                   perl-archive-zip          34402\n",
       "4742                                 perl-array-compare          94236\n",
       "4743                                     perl-array-set           1486\n",
       "4744                                   perl-array-utils          13317\n",
       "4745                            perl-attribute-handlers           4814\n",
       "4746                          perl-authen-sasl-saslprep           3356\n",
       "4747                                       perl-autodie           1685\n",
       "4748                                    perl-autoloader          94532\n",
       "4749                                             perl-b           6746\n",
       "4750                                       perl-b-debug           1130\n",
       "4751                            perl-b-hooks-endofscope           8354\n",
       "4752                                        perl-bignum           1164\n",
       "4753                           perl-bio-asn1-entrezgene          94397\n",
       "4754                       perl-bio-automatedannotation           1623\n",
       "4755                                   perl-bio-bigfile           1680\n",
       "4756                                     perl-bio-cigar           2342\n",
       "4757                                perl-bio-coordinate          75221\n",
       "4758                                       perl-bio-das            171\n",
       "4759                                    perl-bio-db-hts          45855\n",
       "4760                                    perl-bio-db-sam           8449\n",
       "4761                                perl-bio-eutilities           2383\n",
       "4762                                 perl-bio-featureio          89260\n",
       "4763                                      perl-bio-gff3           5044\n",
       "4764                                perl-bio-mlst-check           1998\n",
       "4765                                       perl-bioperl         127487\n",
       "4766                                  perl-bioperl-core         100182\n",
       "4767                                   perl-bioperl-run          92658\n",
       "4768                                     perl-bio-phylo          93600\n",
       "4769                              perl-bio-phylo-cipres             28\n",
       "4770                           perl-bio-rna-rnaalisplit           5756\n",
       "4771                                      perl-biosails           1319\n",
       "4772                                  perl-bio-samtools          88294\n",
       "4773                          perl-bio-tools-phylo-paml          75262\n",
       "4774              perl-bio-tools-run-alignment-clustalw          75208\n",
       "4775               perl-bio-tools-run-alignment-tcoffee          72117\n",
       "4776                                perl-bio-trace-abif            581\n",
       "4777                                 perl-bio-viennangs           6521\n",
       "4778                                      perl-biox-seq           2055\n",
       "4779                         perl-biox-workflow-command           7872\n",
       "4780                                    perl-bit-vector           5553\n",
       "4781                                          perl-blib           1742\n",
       "4782                                  perl-bloom-faster           1884\n",
       "4783                                  perl-border-style           2063\n",
       "4784                                         perl-btlib           1944\n",
       "4785                                perl-bundle-bioperl           1755\n",
       "4786                                 perl-business-isbn         180995\n",
       "4787                            perl-business-isbn-data         180987\n",
       "4788                                   perl-cache-cache          79812\n",
       "4789                                         perl-cairo           1736\n",
       "4790                              perl-canary-stability           5907\n",
       "4791                                  perl-capture-tiny          97832\n",
       "4792                                          perl-carp         316240\n",
       "4793                                     perl-carp-clan           3987\n",
       "4794                                           perl-cgi         121797\n",
       "4795                                   perl-cg-pipeline            116\n",
       "4796                                perl-class-accessor           3266\n",
       "4797                        perl-class-data-inheritable          83971\n",
       "4798                               perl-class-inspector          97153\n",
       "4799                                    perl-class-load          87930\n",
       "4800                                 perl-class-load-xs          89272\n",
       "4801                             perl-class-methodmaker           2061\n",
       "4802                        perl-class-method-modifiers         101876\n",
       "4803                               perl-class-singleton           1544\n",
       "4804                                     perl-class-std           2033\n",
       "4805                                perl-class-std-fast           1905\n",
       "4806                              perl-class-xsaccessor           2293\n",
       "4807                                         perl-clone         114111\n",
       "4808                                  perl-clone-choose           7044\n",
       "4809                                      perl-clone-pp           3401\n",
       "4810                                   perl-color-theme           3517\n",
       "4811                                  perl-common-sense         194862\n",
       "4812                                 perl-compress-bgzf           2049\n",
       "4813                            perl-compress-raw-bzip2         229972\n",
       "4814                             perl-compress-raw-zlib         262613\n",
       "4815                                    perl-config-any           4874\n",
       "4816                               perl-config-autoconf           3362\n",
       "4817                                perl-config-general          30726\n",
       "4818                                 perl-config-simple           3695\n",
       "4819                                   perl-config-tiny           1889\n",
       "4820                                      perl-constant         220149\n",
       "4821                                    perl-const-fast           2926\n",
       "4822                              perl-convert-binary-c          92837\n",
       "4823                                perl-convert-binhex          91670\n",
       "4824                                     perl-cpan-meta          41828\n",
       "4825                               perl-cpan-meta-check           8419\n",
       "4826                        perl-cpan-meta-requirements          37325\n",
       "4827                           perl-cpan-meta-validator           1738\n",
       "4828                                perl-cpan-meta-yaml          35229\n",
       "4829                                    perl-cpan-shell           2601\n",
       "4830                          perl-crypt-openssl-random           1683\n",
       "4831                             perl-crypt-openssl-rsa           1670\n",
       "4832                                     perl-crypt-rc4          91188\n",
       "4833                                  perl-data-compare           2988\n",
       "4834                                     perl-data-dump           2549\n",
       "4835                                   perl-data-dumper         236600\n",
       "4836                                     perl-data-lock             74\n",
       "4837                                    perl-data-match           2814\n",
       "4838                                    perl-data-munge           1569\n",
       "4839                                  perl-data-optlist          92281\n",
       "4840                                perl-data-predicate            218\n",
       "4841                                     perl-data-stag          93048\n",
       "4842                                perl-data-utilities           1744\n",
       "4843                                     perl-data-uuid           8988\n",
       "4844                                  perl-data-visitor           1983\n",
       "4845                                     perl-data-walk           1948\n",
       "4846                                   perl-date-format          79203\n",
       "4847                                    perl-date-manip           7564\n",
       "4848                                      perl-datetime           6498\n",
       "4849                      perl-datetime-format-strptime           3489\n",
       "4850                               perl-datetime-locale           6425\n",
       "4851                             perl-datetime-timezone           6422\n",
       "4852                                     perl-dbd-mysql          46038\n",
       "4853                                        perl-dbd-pg           9457\n",
       "4854                                    perl-dbd-sqlite         104450\n",
       "4855                                       perl-db-file         113622\n",
       "4856                                           perl-dbi         136618\n",
       "4857                                      perl-dbm-deep           2154\n",
       "4858                                  perl-devel-assert           1754\n",
       "4859                                perl-devel-checkbin           2273\n",
       "4860                                 perl-devel-checkos           1230\n",
       "4861                                   perl-devel-cover           1186\n",
       "4862                                   perl-devel-cycle           1631\n",
       "4863                       perl-devel-globaldestruction         103763\n",
       "4864                            perl-devel-overloadinfo          85669\n",
       "4865                                    perl-devel-size           7607\n",
       "4866                              perl-devel-stacktrace          99551\n",
       "4867                                 perl-devel-symdump           9039\n",
       "4868                                  perl-digest-crc32           4974\n",
       "4869                                   perl-digest-hmac         190957\n",
       "4870                                    perl-digest-md5         167946\n",
       "4871                               perl-digest-md5-file           9816\n",
       "4872                               perl-digest-perl-md5         104292\n",
       "4873                                    perl-digest-sha           1754\n",
       "4874                                   perl-digest-sha1          81192\n",
       "4875                           perl-dist-checkconflicts          88864\n",
       "4876                                     perl-dumbbench           1187\n",
       "4877                                    perl-dynaloader          93137\n",
       "4878                             perl-email-date-format          92325\n",
       "4879                                  perl-email-simple            152\n",
       "4880                                        perl-encode         220236\n",
       "4881                                 perl-encode-locale         198252\n",
       "4882                                   perl-ensembl-api            204\n",
       "4883                               perl-ensembl-compara            210\n",
       "4884                                  perl-ensembl-core            253\n",
       "4885                               perl-ensembl-funcgen            217\n",
       "4886                               perl-ensembl-genomes            173\n",
       "4887                                    perl-ensembl-io            211\n",
       "4888                             perl-ensembl-variation            211\n",
       "4889                                           perl-env           1870\n",
       "4890                                      perl-env-path           5236\n",
       "4891                                         perl-error          94005\n",
       "4892                                perl-escape-houdini           1221\n",
       "4893                                      perl-estscan1           1699\n",
       "4894                                      perl-estscan2           1958\n",
       "4895                                  perl-eval-closure          90096\n",
       "4896                             perl-excel-writer-xlsx           5241\n",
       "4897                               perl-exception-class          85919\n",
       "4898                                  perl-experimental           1728\n",
       "4899                                      perl-exporter         311641\n",
       "4900                                 perl-exporter-tidy           1739\n",
       "4901                                 perl-exporter-tiny         259127\n",
       "4902                             perl-extutils-cbuilder          38990\n",
       "4903                               perl-extutils-config           2277\n",
       "4904                             perl-extutils-constant           3020\n",
       "4905                             perl-extutils-cppguess          17689\n",
       "4906                              perl-extutils-depends           3091\n",
       "4907                              perl-extutils-helpers           5095\n",
       "4908                         perl-extutils-installpaths           3597\n",
       "4909                            perl-extutils-makemaker         330596\n",
       "4910                             perl-extutils-manifest          42384\n",
       "4911                              perl-extutils-parsexs          40488\n",
       "4912                            perl-extutils-pkgconfig           1941\n",
       "4913                                     perl-fastx-abi            615\n",
       "4914                                  perl-fastx-reader           2415\n",
       "4915                                    perl-file-chdir           2350\n",
       "4916                                  perl-file-compare           1969\n",
       "4917                                perl-file-copy-link           1804\n",
       "4918                           perl-file-copy-recursive          31912\n",
       "4919                   perl-file-copy-recursive-reduced           1544\n",
       "4920                                  perl-file-details           1947\n",
       "4921                                   perl-filedirutil           1924\n",
       "4922                                    perl-file-fetch          30484\n",
       "4923                                     perl-file-find           1980\n",
       "4924                                perl-file-find-rule          16942\n",
       "4925                                     perl-file-grep          10755\n",
       "4926                                  perl-file-homedir           5864\n",
       "4927                                  perl-file-listing         187767\n",
       "4928                                     perl-file-next           4732\n",
       "4929                                     perl-file-path         220225\n",
       "4930                                    perl-file-pushd           1733\n",
       "4931                            perl-file-readbackwards           4929\n",
       "4932                                   perl-file-remove           1970\n",
       "4933                                    perl-file-share           3327\n",
       "4934                                 perl-file-sharedir          32273\n",
       "4935                         perl-file-sharedir-install          35651\n",
       "4936                                    perl-file-slurp          13981\n",
       "4937                                  perl-file-slurper          12398\n",
       "4938                               perl-file-slurp-tiny          93455\n",
       "4939                                     perl-file-sort          88187\n",
       "4940                                     perl-file-spec          11280\n",
       "4941                                    perl-filesys-df          11290\n",
       "4942                                      perl-file-tee           2300\n",
       "4943                                     perl-file-temp         209533\n",
       "4944                                    perl-file-touch           1770\n",
       "4945                                     perl-file-util           2319\n",
       "4946                                    perl-file-which         114116\n",
       "4947                                 perl-filter-simple           1761\n",
       "4948                                       perl-findbin          11258\n",
       "4949                                  perl-findbin-libs           3560\n",
       "4950                                  perl-findbin-real           1574\n",
       "4951                                      perl-font-afm          91898\n",
       "4952                                      perl-font-ttf          99495\n",
       "4953                                         perl-forks           5647\n",
       "4954                                            perl-gd         147090\n",
       "4955                                       perl-gdgraph          52384\n",
       "4956                             perl-gdgraph-histogram          27932\n",
       "4957                                        perl-gd-svg            187\n",
       "4958                                    perl-gdtextutil          50657\n",
       "4959                               perl-getopt-argvfile           1804\n",
       "4960                                   perl-getopt-long         123163\n",
       "4961                       perl-getopt-long-descriptive           2057\n",
       "4962                                         perl-gfacs             30\n",
       "4963                                   perl-git-wrapper           3798\n",
       "4964                              perl-git-wrapper-plus           3550\n",
       "4965                                       perl-go-perl           3100\n",
       "4966                                         perl-graph          93377\n",
       "4967                           perl-graphics-colornames           4671\n",
       "4968                          perl-graphics-colorobject           4673\n",
       "4969                               perl-graph-readwrite          10414\n",
       "4970                                      perl-graphviz          93671\n",
       "4971                                    perl-hash-merge          10380\n",
       "4972                    perl-hash-util-fieldhash-compat           1949\n",
       "4973                                          perl-heap           1677\n",
       "4974                                   perl-heap-simple           4777\n",
       "4975                              perl-heap-simple-perl           4857\n",
       "4976                                perl-heap-simple-xs           5060\n",
       "4977                                  perl-hook-lexwrap           2162\n",
       "4978                            perl-hpc-runner-command          21989\n",
       "4979       perl-hpc-runner-command-plugin-logger-sqlite           1820\n",
       "4980                         perl-html-element-extended          91000\n",
       "4981                        perl-html-entities-numbered          91765\n",
       "4982                                     perl-html-form           2693\n",
       "4983                                perl-html-formatter          93783\n",
       "4984                                   perl-html-parser         204107\n",
       "4985                             perl-html-tableextract          91010\n",
       "4986                                   perl-html-tagset         201341\n",
       "4987                                 perl-html-template          22164\n",
       "4988                                     perl-html-tidy          93685\n",
       "4989                                     perl-html-tree         187437\n",
       "4990                        perl-html-treebuilder-xpath          91107\n",
       "4991                                  perl-http-cookies         189959\n",
       "4992                                   perl-http-daemon         187757\n",
       "4993                                     perl-http-date         190549\n",
       "4994                                  perl-http-message         190192\n",
       "4995                                perl-http-negotiate         189739\n",
       "4996                            perl-http-server-simple           2065\n",
       "4997                                     perl-http-tiny            214\n",
       "4998                                            perl-if             44\n",
       "4999                                perl-image-exiftool           6708\n",
       "5000                                    perl-image-info          91184\n",
       "5001                                    perl-image-size          91277\n",
       "5002                                      perl-importer          20006\n",
       "5003                                   perl-import-into           1650\n",
       "5004                                      perl-indirect           1778\n",
       "5005                                        perl-inline           8219\n",
       "5006                                      perl-inline-c           7387\n",
       "5007                                       perl-integer           1739\n",
       "5008                                        perl-io-all           7774\n",
       "5009                                   perl-io-compress         229748\n",
       "5010                           perl-io-compress-deflate           1685\n",
       "5011                                       perl-io-gzip          12452\n",
       "5012                                     perl-io-handle           3801\n",
       "5013                                       perl-io-html         190396\n",
       "5014                                perl-io-interactive           4930\n",
       "5015                                       perl-io-null             36\n",
       "5016                                     perl-io-prompt           5348\n",
       "5017                                     perl-io-scalar           1789\n",
       "5018                                perl-io-sessiondata          91486\n",
       "5019                               perl-io-socket-inet6           1936\n",
       "5020                                 perl-io-socket-ssl         194780\n",
       "5021                                     perl-io-string         110358\n",
       "5022                                    perl-io-stringy          91783\n",
       "5023                                        perl-io-tty          93230\n",
       "5024                         perl-io-uncompress-bunzip2           1880\n",
       "5025                          perl-io-uncompress-gunzip           4306\n",
       "5026                      perl-io-uncompress-rawinflate           1718\n",
       "5027                                       perl-io-zlib         177731\n",
       "5028                                       perl-ipc-cmd          63204\n",
       "5029                                      perl-ipc-run3           1582\n",
       "5030                                       perl-ipc-run          95251\n",
       "5031                                 perl-ipc-sharelite          81362\n",
       "5032                             perl-ipc-system-simple          29888\n",
       "5033                                         perl-jcode          92240\n",
       "5034                                          perl-json         192951\n",
       "5035                                   perl-json-create           1793\n",
       "5036                                  perl-json-maybexs           3818\n",
       "5037                                    perl-json-parse           3719\n",
       "5038                                       perl-json-pp          40737\n",
       "5039                                       perl-json-xs         201633\n",
       "5040                                           perl-lib          83391\n",
       "5041                                   perl-libwww-perl         193203\n",
       "5042                                   perl-libxml-perl          93352\n",
       "5043                                  perl-list-compare           2105\n",
       "5044                                perl-list-moreutils         260626\n",
       "5045                             perl-list-moreutils-xs         219721\n",
       "5046                                perl-list-someutils           3780\n",
       "5047                                     perl-list-uniq           2179\n",
       "5048                                        perl-locale           1752\n",
       "5049                        perl-locale-maketext-simple          62273\n",
       "5050                                     perl-local-lib           3468\n",
       "5051                               perl-lockfile-simple            176\n",
       "5052                                       perl-log-any           1757\n",
       "5053                                 perl-logger-simple           3959\n",
       "5054                                  perl-log-log4perl          25395\n",
       "5055                                perl-lwp-mediatypes         193795\n",
       "5056                            perl-lwp-protocol-https         184087\n",
       "5057                                    perl-lwp-simple         104368\n",
       "5058                                      perl-lyve-set            125\n",
       "5059                              perl-mac-propertylist           1596\n",
       "5060                           perl-mac-systemdirectory           1393\n",
       "5061                                     perl-mailtools          94965\n",
       "5062                             perl-math-base-convert           2435\n",
       "5063                                   perl-math-bezier          28922\n",
       "5064                                   perl-math-bigint           3070\n",
       "5065                                   perl-math-bigrat           1137\n",
       "5066                                      perl-math-cdf         103084\n",
       "5067                            perl-math-combinatorics           1833\n",
       "5068                                  perl-math-complex           1705\n",
       "5069                               perl-math-derivative          93593\n",
       "5070                                   perl-math-matrix            113\n",
       "5071                               perl-math-matrixreal             80\n",
       "5072                                   perl-math-random          92981\n",
       "5073                           perl-math-random-mt-auto           3290\n",
       "5074                                    perl-math-round          31721\n",
       "5075                                   perl-math-spline          90994\n",
       "5076                                    perl-math-utils           1615\n",
       "5077                                  perl-math-vecstat          29049\n",
       "5078                                           perl-mce           7989\n",
       "5079                                    perl-mce-shared           6265\n",
       "5080                                       perl-memoize           5252\n",
       "5081              perl-metabolomics-fragment-annotation            150\n",
       "5082                                   perl-mime-base64         194696\n",
       "5083                                     perl-mime-lite          91243\n",
       "5084                              perl-mime-quotedprint           1704\n",
       "5085                                    perl-mime-tools          94390\n",
       "5086                                    perl-mime-types          93251\n",
       "5087                                perl-mixin-linewise           3121\n",
       "5088                                         perl-mldbm          93352\n",
       "5089                                    perl-mldbm-sync           2223\n",
       "5090                                  perl-module-build          56819\n",
       "5091                             perl-module-build-tiny           2731\n",
       "5092                               perl-module-corelist          70935\n",
       "5093                            perl-module-extract-use           1859\n",
       "5094                            perl-module-fromperlver           3567\n",
       "5095                         perl-module-implementation         102398\n",
       "5096                                   perl-module-list           2000\n",
       "5097                                   perl-module-load          62370\n",
       "5098                       perl-module-load-conditional          63428\n",
       "5099                                 perl-module-loaded           4713\n",
       "5100                               perl-module-metadata          65483\n",
       "5101                              perl-module-pluggable          15663\n",
       "5102                                perl-module-runtime         119313\n",
       "5103                      perl-module-runtime-conflicts          85671\n",
       "5104                               perl-module-scandeps           6139\n",
       "5105                                   perl-module-util           1684\n",
       "5106                                           perl-moo         104481\n",
       "5107                                         perl-moose          94442\n",
       "5108                                    perl-moosex-app           6550\n",
       "5109                      perl-moosex-app-role-log4perl           2005\n",
       "5110                                  perl-moosex-clone           1986\n",
       "5111                          perl-moosex-fileattribute           3859\n",
       "5112                                 perl-moosex-getopt           5876\n",
       "5113                               perl-moosex-nonmoose           2071\n",
       "5114                       perl-moosex-object-pluggable           1325\n",
       "5115                     perl-moosex-role-parameterized           1955\n",
       "5116                   perl-moosex-role-withoverloading           1869\n",
       "5117                              perl-moosex-singleton           1710\n",
       "5118                                  perl-moosex-types           5016\n",
       "5119                       perl-moosex-types-path-class           3326\n",
       "5120                        perl-moosex-types-path-tiny           2144\n",
       "5121                       perl-moosex-types-stringlike           1705\n",
       "5122                                    perl-mozilla-ca         170070\n",
       "5123                                    perl-mro-compat          85682\n",
       "5124                                            perl-ms            127\n",
       "5125                           perl-namespace-autoclean           7894\n",
       "5126                               perl-namespace-clean           8345\n",
       "5127                                       perl-net-ftp           1915\n",
       "5128                             perl-net-ftp-recursive           1847\n",
       "5129                                      perl-net-http         191945\n",
       "5130                                     perl-net-netrc           1750\n",
       "5131                                    perl-net-ssleay         203568\n",
       "5132                                          perl-ntlm         189800\n",
       "5133                                perl-number-compare          16392\n",
       "5134                                 perl-number-format          43136\n",
       "5135                                   perl-number-misc           2306\n",
       "5136                                  perl-number-range           1784\n",
       "5137                              perl-number-witherror           1227\n",
       "5138                              perl-object-insideout           6087\n",
       "5139                                 perl-obogaf-parser            177\n",
       "5140                              perl-ole-storage_lite          92678\n",
       "5141                                     perl-onto-perl           2158\n",
       "5142                    perl-package-deprecationmanager          85656\n",
       "5143                                 perl-package-stash          92054\n",
       "5144                              perl-package-stash-xs          87537\n",
       "5145                                     perl-padwalker           2043\n",
       "5146                                           perl-par           1772\n",
       "5147                          perl-parallel-forkmanager          22749\n",
       "5148                             perl-parallel-iterator           1148\n",
       "5149                                perl-parallel-loops           3787\n",
       "5150                                  perl-params-check          62114\n",
       "5151                                 perl-params-coerce           1980\n",
       "5152                                   perl-params-util          94560\n",
       "5153                               perl-params-validate          33495\n",
       "5154                     perl-params-validationcompiler           6550\n",
       "5155                                      perl-par-dist           1775\n",
       "5156                                        perl-parent         230774\n",
       "5157                                    perl-par-packer           1816\n",
       "5158                              perl-parse-recdescent          94668\n",
       "5159                                    perl-parse-yapp          12019\n",
       "5160                                    perl-path-class           6644\n",
       "5161                                     perl-path-tiny          10366\n",
       "5162                                     perl-pathtools         252174\n",
       "5163                                   perl-pbkdf2-tiny           1684\n",
       "5164                                          perl-pcap           8356\n",
       "5165                                      perl-pdf-api2          99799\n",
       "5166                                     perl-pdf-table            743\n",
       "5167                                         perl-pegex           5582\n",
       "5168                                perl-perl4-corelibs           1981\n",
       "5169                                        perl-perlio           2123\n",
       "5170                               perl-perlio-encoding           1716\n",
       "5171                                   perl-perlio-gzip          72763\n",
       "5172                            perl-perlio-utf8_strict          12756\n",
       "5173                                  perl-perl-osnames           2170\n",
       "5174                                   perl-perl-ostype          39683\n",
       "5175                           perl-perl-unsafe-signals           5628\n",
       "5176                                  perl-perl-version           4305\n",
       "5177                                   perl-pod-checker           1765\n",
       "5178                                  perl-pod-coverage           2052\n",
       "5179                         perl-pod-coverage-trustpod           1715\n",
       "5180                                 perl-pod-elemental           3207\n",
       "5181                                   perl-pod-escapes          83011\n",
       "5182                                    perl-pod-parser           1941\n",
       "5183                                 perl-pod-plaintext           1745\n",
       "5184                                    perl-pod-simple           1838\n",
       "5185                               perl-pod-simple-text           1794\n",
       "5186                                     perl-pod-usage          82784\n",
       "5187                                         perl-posix           6118\n",
       "5188                                    perl-postscript          91100\n",
       "5189                                           perl-ppi           2037\n",
       "5190                                       perl-prefork           1224\n",
       "5191                                    perl-probe-perl           1465\n",
       "5192                                     perl-proc-fork           3212\n",
       "5193                                      perl-readonly          31496\n",
       "5194                                 perl-regexp-common          32725\n",
       "5195                                   perl-rest-client            175\n",
       "5196                             perl-return-multilevel           1559\n",
       "5197                                     perl-role-tiny         104105\n",
       "5198                                perl-role-tiny-with           1748\n",
       "5199                                          perl-safe           1918\n",
       "5200                        perl-sanger-cgp-allelecount           9599\n",
       "5201                         perl-sanger-cgp-battenberg           7368\n",
       "5202                            perl-sanger-cgp-vagrent           5470\n",
       "5203                                perl-sanger-cgp-vcf           9384\n",
       "5204                             perl-scalar-list-utils         300579\n",
       "5205                           perl-scalar-util-numeric           7194\n",
       "5206                                   perl-scope-guard          15822\n",
       "5207                                        perl-sereal          38743\n",
       "5208                                perl-sereal-decoder          39884\n",
       "5209                                perl-sereal-encoder          39795\n",
       "5210                              perl-set-intervaltree          54140\n",
       "5211                                   perl-set-intspan          28823\n",
       "5212                                    perl-set-scalar          91067\n",
       "5213                                         perl-slurp           1612\n",
       "5214                                          perl-snap           1544\n",
       "5215                                     perl-soap-lite          91141\n",
       "5216                                        perl-socket         164299\n",
       "5217                                       perl-socket6           3567\n",
       "5218                                perl-sort-naturally          92589\n",
       "5219                                 perl-sort-versions           2450\n",
       "5220                                        perl-specio           3151\n",
       "5221                               perl-specio-exporter           6801\n",
       "5222                                        perl-spiffy           2611\n",
       "5223                        perl-spreadsheet-parseexcel          91046\n",
       "5224                        perl-spreadsheet-writeexcel          92232\n",
       "5225                                 perl-sql-statement           2457\n",
       "5226                              perl-statistics-basic          32796\n",
       "5227                     perl-statistics-caseresampling           1222\n",
       "5228                        perl-statistics-descriptive         101117\n",
       "5229                      perl-statistics-distributions           5892\n",
       "5230                          perl-statistics-frequency           1749\n",
       "5231                               perl-statistics-lite           1912\n",
       "5232                                  perl-statistics-r           4291\n",
       "5233                              perl-statistics-ttest           4469\n",
       "5234                                      perl-storable         173738\n",
       "5235                                 perl-string-approx           2530\n",
       "5236                                   perl-string-diff            558\n",
       "5237                                 perl-string-escape           2869\n",
       "5238                                 perl-string-random           1797\n",
       "5239                          perl-string-rewriteprefix           2136\n",
       "5240                               perl-string-truncate           2147\n",
       "5241                                   perl-string-util           2311\n",
       "5242                                 perl-sub-attribute           2089\n",
       "5243                                  perl-sub-exporter          90883\n",
       "5244                       perl-sub-exporter-formethods           2489\n",
       "5245                      perl-sub-exporter-progressive         104944\n",
       "5246                                  perl-sub-identify          91132\n",
       "5247                                      perl-sub-info          15343\n",
       "5248                                   perl-sub-install          92496\n",
       "5249                                      perl-sub-name          86227\n",
       "5250                                     perl-sub-quote          85871\n",
       "5251                                   perl-sub-uplevel          86539\n",
       "5252                                         perl-super           1819\n",
       "5253                                           perl-svg         107203\n",
       "5254                                     perl-svg-graph          92603\n",
       "5255                                        perl-symbol           1785\n",
       "5256                                   perl-symbol-util           1755\n",
       "5257                                      perl-sys-info           3230\n",
       "5258                                 perl-sys-info-base           3274\n",
       "5259                         perl-sys-info-driver-linux           4505\n",
       "5260                           perl-sys-info-driver-osx           2653\n",
       "5261                                 perl-sys-sigaction           4904\n",
       "5262                               perl-tap-harness-env           1742\n",
       "5263                                   perl-task-weaken          93722\n",
       "5264                              perl-template-toolkit          91274\n",
       "5265                                perl-term-app-roles           3635\n",
       "5266                          perl-term-detect-software           2215\n",
       "5267                                 perl-term-encoding           3000\n",
       "5268                              perl-term-progressbar           3517\n",
       "5269                                   perl-termreadkey          10924\n",
       "5270                                    perl-term-table          17085\n",
       "5271                                         perl-test2           8812\n",
       "5272                                   perl-test2-suite          22953\n",
       "5273                                          perl-test          83033\n",
       "5274                                     perl-test-base           3674\n",
       "5275                           perl-test-builder-tester           5391\n",
       "5276                                 perl-test-classapi           3564\n",
       "5277                              perl-test-class-moose           6605\n",
       "5278                          perl-test-cleannamespaces           3022\n",
       "5279                                perl-test-cpan-meta           2189\n",
       "5280                                     perl-test-deep          70414\n",
       "5281                              perl-test-differences          83164\n",
       "5282                                      perl-test-eol           3661\n",
       "5283                                perl-test-exception          82724\n",
       "5284                                     perl-test-exec           1681\n",
       "5285                                    perl-test-fatal           7465\n",
       "5286                                     perl-test-file           1164\n",
       "5287                            perl-test-file-contents           1962\n",
       "5288                                    perl-test-files           5184\n",
       "5289                                     perl-test-fork           1615\n",
       "5290                                  perl-test-harness          88682\n",
       "5291                                    perl-test-inter           4563\n",
       "5292                                perl-test-leaktrace          95230\n",
       "5293                               perl-test-lectrotest           1193\n",
       "5294                               perl-test-longstring           1931\n",
       "5295                             perl-test-memory-cycle           1767\n",
       "5296                               perl-test-mockmodule           1942\n",
       "5297                                     perl-test-most          83449\n",
       "5298                                    perl-test-needs           2931\n",
       "5299                                   perl-test-notabs           3789\n",
       "5300                               perl-test-nowarnings           3651\n",
       "5301                                   perl-test-object           1990\n",
       "5302                                   perl-test-output           6474\n",
       "5303                                      perl-test-pod          18121\n",
       "5304                             perl-test-pod-coverage           1980\n",
       "5305                                   perl-test-prereq           1831\n",
       "5306                                 perl-test-requires           2870\n",
       "5307                         perl-test-requiresinternet         163206\n",
       "5308                                   perl-test-script           1460\n",
       "5309                                   perl-test-simple          20050\n",
       "5310                                 perl-test-subcalls           1906\n",
       "5311                                 perl-test-sys-info           3065\n",
       "5312                                  perl-test-toolbox           2273\n",
       "5313                                     perl-test-trap           3699\n",
       "5314                                perl-test-unit-lite           1734\n",
       "5315                                     perl-test-utf8           1171\n",
       "5316                                     perl-test-warn          85125\n",
       "5317                                 perl-test-warnings           7174\n",
       "5318                           perl-test-without-module           1552\n",
       "5319                                      perl-test-xml           2195\n",
       "5320                                     perl-test-yaml           3730\n",
       "5321                                   perl-text-abbrev          38557\n",
       "5322                                perl-text-ansitable           1976\n",
       "5323                               perl-text-asciitable           3962\n",
       "5324                                 perl-text-balanced           4597\n",
       "5325                                      perl-text-csv          47770\n",
       "5326                                   perl-text-csv_xs            656\n",
       "5327                                     perl-text-diff          84023\n",
       "5328                                   perl-text-format          29286\n",
       "5329                                     perl-text-glob          19464\n",
       "5330                              perl-text-levenshtein           2159\n",
       "5331                            perl-text-levenshteinxs           1182\n",
       "5332                                      perl-text-nsp           2111\n",
       "5333                               perl-text-parsewords          41822\n",
       "5334                                     perl-text-tabs           1755\n",
       "5335                                perl-text-tabs-wrap           3169\n",
       "5336                                 perl-text-template           1956\n",
       "5337                          perl-text-template-simple           1589\n",
       "5338                                     perl-text-wrap           4108\n",
       "5339                                      perl-threaded         222698\n",
       "5340                                     perl-tie-cache           2305\n",
       "5341                                    perl-tie-cacher           1878\n",
       "5342                                      perl-tie-hash           1788\n",
       "5343                              perl-tie-hash-indexed           1853\n",
       "5344                                    perl-tie-ixhash          92287\n",
       "5345                                  perl-tie-log4perl           2613\n",
       "5346                                   perl-tie-refhash           2214\n",
       "5347                              perl-tie-refhash-weak           2049\n",
       "5348                                  perl-tie-toobject           2005\n",
       "5349                                      perl-timedate          91562\n",
       "5350                                    perl-time-hires         109294\n",
       "5351                                    perl-time-local         166078\n",
       "5352                                    perl-time-piece           8634\n",
       "5353                                 perl-tree-dag_node          93585\n",
       "5354                                      perl-try-tiny         185653\n",
       "5355                                perl-try-tiny-retry           3386\n",
       "5356                              perl-types-serialiser         176717\n",
       "5357                                     perl-type-tiny          97815\n",
       "5358                                   perl-unicode-map          92909\n",
       "5359                             perl-unicode-normalize           4715\n",
       "5360                            perl-unicode-stringprep           1867\n",
       "5361                                  perl-unicode-utf8           1718\n",
       "5362                                           perl-uri         223700\n",
       "5363                               perl-util-properties            171\n",
       "5364                                perl-variable-magic           8456\n",
       "5365                                          perl-vars           1785\n",
       "5366                                  perl-vcftools-vcf          50943\n",
       "5367                               perl-velvetoptimiser           5062\n",
       "5368                                       perl-version          62151\n",
       "5369                                  perl-version-next           4214\n",
       "5370                                          perl-want           5390\n",
       "5371                             perl-warnings-register           6539\n",
       "5372                                 perl-www-mechanize           1591\n",
       "5373                                perl-www-robotrules         188830\n",
       "5374                                       perl-xml-dom          93322\n",
       "5375                                 perl-xml-dom-xpath          91013\n",
       "5376                                  perl-xml-entities           1701\n",
       "5377                         perl-xml-filter-buffertext          91224\n",
       "5378                                    perl-xml-libxml         121576\n",
       "5379                                   perl-xml-libxslt          93334\n",
       "5380                          perl-xml-namespacesupport         205411\n",
       "5381                                    perl-xml-parser         122792\n",
       "5382                               perl-xml-parser-lite           3722\n",
       "5383                                    perl-xml-regexp          91679\n",
       "5384                                       perl-xml-sax         209529\n",
       "5385                                  perl-xml-sax-base         205378\n",
       "5386                                 perl-xml-sax-expat         192576\n",
       "5387                                perl-xml-sax-writer          93440\n",
       "5388                              perl-xml-semanticdiff           3836\n",
       "5389                                    perl-xml-simple         195056\n",
       "5390                                      perl-xml-twig          95408\n",
       "5391                                    perl-xml-writer          91377\n",
       "5392                                     perl-xml-xpath          93518\n",
       "5393                               perl-xml-xpathengine          92050\n",
       "5394                                      perl-xsloader         277632\n",
       "5395                                          perl-yaml         151011\n",
       "5396                                  perl-yaml-libyaml           7989\n",
       "5397                                       perl-yaml-pp             34\n",
       "5398                                     perl-yaml-tiny           4175\n",
       "5399                                          pfam_scan           3965\n",
       "5400                                            pftools            488\n",
       "5401                                             pgcgap           1742\n",
       "5402                                          pgdspider             47\n",
       "5403                                        pgma-simple           3269\n",
       "5404                                               pgsa           1916\n",
       "5405                                    pgx-pipe-helper             56\n",
       "5406                                  pgx-variant-tools            208\n",
       "5407                                              phame           8216\n",
       "5408                               phantompeakqualtools           4860\n",
       "5409                                              phast           1790\n",
       "5410                                            phastaf             55\n",
       "5411                                              phate            376\n",
       "5412                                            pheniqs           4024\n",
       "5413                                             phenix           1642\n",
       "5414                                            phigaro             19\n",
       "5415                                            phipack           3018\n",
       "5416                                             phispy           3247\n",
       "5417                                              phizz           2103\n",
       "5418                                            phylics           2111\n",
       "5419                                             phylip           6999\n",
       "5420                                           phylocsf           1053\n",
       "5421                                            phylodm            872\n",
       "5422                                         phyloflash           5737\n",
       "5423                                      phylogenetics             81\n",
       "5424                                         phylommand           1166\n",
       "5425                                        phylopandas           4469\n",
       "5426                                         phylophlan           1989\n",
       "5427                                          phylorank            195\n",
       "5428                                         phylotoast           4007\n",
       "5429                                          phylovega            236\n",
       "5430                                           phylowgs          10000\n",
       "5431                                            phyluce          11135\n",
       "5432                                              phyml          14348\n",
       "5433                                             phyphy             51\n",
       "5434                                             picard         264985\n",
       "5435                                        picard-slim           2229\n",
       "5436                                           picopore          12268\n",
       "5437                                            picrust          14723\n",
       "5438                                           picrust2           7276\n",
       "5439                                              piler           2154\n",
       "5440                                           pileuppy             58\n",
       "5441                                              pilon          29346\n",
       "5442                                             pindel          48830\n",
       "5443                                           pipelign           1826\n",
       "5444                                              piper            390\n",
       "5445                                             pipits           9567\n",
       "5446                                             pipmir           2551\n",
       "5447                                            piranha           3413\n",
       "5448                                             pirate            489\n",
       "5449                                              piret           1473\n",
       "5450                                               pirs           1960\n",
       "5451                                             pisces           4158\n",
       "5452                                            pispino           4733\n",
       "5453                                             pizzly           9161\n",
       "5454                                              pkiss           1921\n",
       "5455                                            planemo          44322\n",
       "5456               plant_tribes_assembly_post_processor           8095\n",
       "5457                   plant_tribes_gene_family_aligner           5062\n",
       "5458                plant_tribes_gene_family_classifier          10177\n",
       "5459                plant_tribes_gene_family_integrator           5263\n",
       "5460         plant_tribes_gene_family_phylogeny_builder           5403\n",
       "5461                         plant_tribes_kaks_analysis           5122\n",
       "5462                                          plasclass             65\n",
       "5463                                           plascope           3256\n",
       "5464                                           plasflow           2388\n",
       "5465                                      plasmidfinder            369\n",
       "5466                                          plasmidid           1179\n",
       "5467                                        plasmidtron           5270\n",
       "5468                                              plass           2995\n",
       "5469                                            plastid           6996\n",
       "5470                                             platon            461\n",
       "5471                              platypus-conquistador           1790\n",
       "5472                                   platypus-variant          12299\n",
       "5473                                               plek           2407\n",
       "5474                                              plink          14533\n",
       "5475                                             plink2           6890\n",
       "5476                                            plncpro            504\n",
       "5477                                           pmdtools           3107\n",
       "5478                                     pmmrcalculator            118\n",
       "5479                                          pmx_biobb             33\n",
       "5480                                          pneumocat           3673\n",
       "5481                                                poa           2333\n",
       "5482                                           polymutt           1549\n",
       "5483                                          polystest             50\n",
       "5484                                            pomoxis           1317\n",
       "5485                                             popdel          17489\n",
       "5486                                             popera             57\n",
       "5487                                     popgen-entropy            552\n",
       "5488                                            poppunk           4383\n",
       "5489                                            popscle            936\n",
       "5490                                           porechop           8438\n",
       "5491                                          poretools          10546\n",
       "5492                                         portcullis           9791\n",
       "5493                                            potrace           2487\n",
       "5494                                            pourrna           3902\n",
       "5495                                          pout2mzid           1978\n",
       "5496                                                 pp           4246\n",
       "5497                                         ppanggolin           4658\n",
       "5498                                            ppanini           3381\n",
       "5499                                            pplacer          14967\n",
       "5500                                       pp-sketchlib           2206\n",
       "5501                                              prank          14366\n",
       "5502                                          prefersim            964\n",
       "5503                                             preseq          26951\n",
       "5504                                             presto           4860\n",
       "5505                                       pretextgraph            787\n",
       "5506                                         pretextmap           1580\n",
       "5507                                    pretextsnapshot            972\n",
       "5508                                      pretext-suite            107\n",
       "5509                                          primedrpa            200\n",
       "5510                                            primer3          13576\n",
       "5511                                         primer3-py          10369\n",
       "5512                                         primerclip            876\n",
       "5513                                   primerprospector           2001\n",
       "5514                                             prince            894\n",
       "5515                                            prinseq          18170\n",
       "5516                                  prinseq-plus-plus            170\n",
       "5517                                  probabilistic2020            710\n",
       "5518                                      probamconvert           4944\n",
       "5519                                           probcons           1322\n",
       "5520                                              proda           1219\n",
       "5521                                           prodigal          70937\n",
       "5522                                   progressivemauve           3177\n",
       "5523                                         prokaryote            292\n",
       "5524                                             prokka          47219\n",
       "5525                                             pronto           6313\n",
       "5526                                              proot           1802\n",
       "5527                                           prophane            101\n",
       "5528                                           prophasm           1782\n",
       "5529                                            prophex           3448\n",
       "5530                                           prophyle          24637\n",
       "5531                                         prosampler           2740\n",
       "5532                                             prosic           6146\n",
       "5533                                            prosolo           9580\n",
       "5534                                       proteinortho          19577\n",
       "5535                                       proteowizard           2984\n",
       "5536                                              protk           1683\n",
       "5537                                         protmapper            147\n",
       "5538                                            protrac           4642\n",
       "5539                                            psascan           1242\n",
       "5540                                              psass            486\n",
       "5541                                      psm_fragments             16\n",
       "5542                                           ptrimmer           1168\n",
       "5543                                          pubchempy         187185\n",
       "5544                                     pubmlst_client             68\n",
       "5545                                            pullseq           3786\n",
       "5546                                           pureclip          12591\n",
       "5547                                    purge_haplotigs           5784\n",
       "5548                                         purple-bio            143\n",
       "5549                                            pvacseq           3723\n",
       "5550                                             py2bit          67566\n",
       "5551                                             pyaavf            348\n",
       "5552                                              pyani          13422\n",
       "5553                                              pyasp           2319\n",
       "5554                                              pyasr             96\n",
       "5555                                        pybamparser           2007\n",
       "5556                                         pybamtools           3755\n",
       "5557                                              pybda            121\n",
       "5558                                         pybedgraph           5090\n",
       "5559                                         pybedtools         225121\n",
       "5560                                              pybel           6131\n",
       "5561                                           pybigwig         198714\n",
       "5562                                          pybiomart            181\n",
       "5563                                          pychopper           1488\n",
       "5564                                              pycli           5063\n",
       "5565                                            pyclone            119\n",
       "5566                                          pycluster           5109\n",
       "5567                                             pycoqc           3679\n",
       "5568                                              pycsg           2372\n",
       "5569                                           pydemult           1804\n",
       "5570                                              pydna           3870\n",
       "5571                                            pydnase           4104\n",
       "5572                                      pydownsampler             19\n",
       "5573                                               pydp            142\n",
       "5574                                              pydpi             47\n",
       "5575                                             pyega3           3150\n",
       "5576                                          pyensembl           4189\n",
       "5577                                      pyexcelerator           4233\n",
       "5578                                            pyfaidx         146823\n",
       "5579                                            pyfasta           9633\n",
       "5580                                           pyfastaq          12246\n",
       "5581                                            pyfastx           5697\n",
       "5582                                     pygenometracks          25683\n",
       "5583                                             pygmes            215\n",
       "5584                                           pygresql           3331\n",
       "5585                                            pygtftk           9354\n",
       "5586                                    pygvcf2coverage             32\n",
       "5587                                           pyhashxx          13317\n",
       "5588                                     pyimagingmspec           1882\n",
       "5589                                            pyimzml           2355\n",
       "5590                                              pylca            133\n",
       "5591                                           pylibseq           8928\n",
       "5592                                         pyliftover          11520\n",
       "5593                                              pyloh           9095\n",
       "5594                                   pylprotpredictor             99\n",
       "5595                                       pymisc-utils           3464\n",
       "5596                                              pymix           1934\n",
       "5597                                              pymot           1815\n",
       "5598                                           pymummer          16117\n",
       "5599                                             pymvpa           6436\n",
       "5600                                             pymzml          10626\n",
       "5601                                             pynast          30800\n",
       "5602                                         pynnotator            363\n",
       "5603                                     pyomero-upload            343\n",
       "5604                                              pyopt           1663\n",
       "5605                                            pypairs            725\n",
       "5606                                           pypeflow          12069\n",
       "5607                                            pypgatk             50\n",
       "5608                                            pypiper           3383\n",
       "5609                                             pypore           5364\n",
       "5610                                          pyprophet           6276\n",
       "5611                                         pyquant-ms           7514\n",
       "5612                                              pyrad           3416\n",
       "5613                                           pyranges          35568\n",
       "5614                                              pyrle          28693\n",
       "5615                                          pyrodigal            360\n",
       "5616                                            pyrpipe             32\n",
       "5617                                              pysam        1415206\n",
       "5618                                         pysamstats          18500\n",
       "5619                                             pyscaf           1867\n",
       "5620                                             pyseer           6921\n",
       "5621                                         pysnptools           3760\n",
       "5622                                            pysradb           5658\n",
       "5623                                              pysvg           4221\n",
       "5624                                         pysvmlight           1928\n",
       "5625                                            pytabix          71032\n",
       "5626                                         pytaxonkit            126\n",
       "5627                                          pyteomics          23409\n",
       "5628                                       pytest-marks           3065\n",
       "5629                                          pythomics           3987\n",
       "5630                                      python-bioext           5556\n",
       "5631                                  python-bioformats            331\n",
       "5632                                       python-chado          10172\n",
       "5633                                     python-cluster           4297\n",
       "5634                               python-consensuscore          12253\n",
       "5635                              python-consensuscore2           6189\n",
       "5636                                       python-edlib          20187\n",
       "5637                                         python-eel            966\n",
       "5638                                      python-fakemp           2915\n",
       "5639                                python-goenrichment            107\n",
       "5640                               python-hivclustering           7798\n",
       "5641                                        python-hppy           3600\n",
       "5642                                      python-htsget            917\n",
       "5643                                python-hyphy-python           7274\n",
       "5644                                python-intervaltree           4586\n",
       "5645                                     python-libsbml           4972\n",
       "5646                              python-mailund-newick           1971\n",
       "5647                                     python-msgpack           8375\n",
       "5648                                      python-newick           4088\n",
       "5649                                           pythonpy           9922\n",
       "5650                                  python-rnacentral             90\n",
       "5651                            python-sortedcontainers           4690\n",
       "5652                                      python-tripal          14176\n",
       "5653                                         pytriangle           4727\n",
       "5654                                              pywdl           2131\n",
       "5655                                                pyx           4410\n",
       "5656                                               qcat           2261\n",
       "5657                                               qcli          31867\n",
       "5658                                            qcumber           1656\n",
       "5659                                              qfilt           2109\n",
       "5660                                           qgrs-cpp           1554\n",
       "5661                                              qiime          46811\n",
       "5662                            qiime-default-reference          26606\n",
       "5663                                     qiimetomaaslin           1752\n",
       "5664                                              qorts           4406\n",
       "5665                                         qsignature           8331\n",
       "5666                                               qtip           2276\n",
       "5667                                             qtlseq            715\n",
       "5668                                              quake           2075\n",
       "5669                                           qualimap            721\n",
       "5670                                           qualimap          74370\n",
       "5671                                         quantiprot             47\n",
       "5672                                        quantwiz-iq            328\n",
       "5673                                           quasildr             33\n",
       "5674                                        quasirecomb             76\n",
       "5675                                         quasitools          23372\n",
       "5676                                              quast         121985\n",
       "5677                                   query_phenomizer           2203\n",
       "5678                                         quickmerge            685\n",
       "5679                                          quicksect          14408\n",
       "5680                                          quicktree           5615\n",
       "5681                                             quorum           2094\n",
       "5682                                                r2r           3205\n",
       "5683                                                 ra           1792\n",
       "5684                                        rabix-bunny          15977\n",
       "5685                                         r-acidbase           1427\n",
       "5686                                     r-acidgenerics            964\n",
       "5687                                        r-acidplots           1178\n",
       "5688                                              racon          52675\n",
       "5689                                     rad_haplotyper           5095\n",
       "5690                                             radsex           2056\n",
       "5691                                             ragout           8007\n",
       "5692                                             ragtag            236\n",
       "5693                                            rainbow           5337\n",
       "5694                                       ra-integrate           1553\n",
       "5695                                              raiss            103\n",
       "5696                                            rambo-k           1942\n",
       "5697                                           r-ampvis           2012\n",
       "5698                                           randfold           3954\n",
       "5699                                       r-annotables           1862\n",
       "5700                                           rapclust           3949\n",
       "5701                                              rapid          14449\n",
       "5702                                            rapidnj           3258\n",
       "5703                                           rapifilt            444\n",
       "5704                                             rapmap          13737\n",
       "5705                                             rappas            310\n",
       "5706                                          rapsearch           4378\n",
       "5707                                 r-aroma.affymetrix           7435\n",
       "5708                                       r-aroma.core           7659\n",
       "5709                                             rascaf           1887\n",
       "5710                                             rasusa           1505\n",
       "5711                                               ratt            153\n",
       "5712                                    raven-assembler           5280\n",
       "5713                                           rawtools            520\n",
       "5714                                              raxml         112386\n",
       "5715                                           raxml-ng           3117\n",
       "5716                                                ray           3923\n",
       "5717                                            razers3          19593\n",
       "5718                                         r-basejump          16258\n",
       "5719                                        r-bcbiobase          14046\n",
       "5720                                      r-bcbiornaseq          14616\n",
       "5721                                  r-bcbiosinglecell            439\n",
       "5722                                            r-biodb          10998\n",
       "5723                                        r-bisquerna            272\n",
       "5724                                             r-cate           1627\n",
       "5725                                       r-cellassign           4555\n",
       "5726                                         r-chbutils           8897\n",
       "5727                                            r-cimpl           1987\n",
       "5728                                   r-classdiscovery           4173\n",
       "5729                                            r-coloc           9818\n",
       "5730                                         rcorrector           8461\n",
       "5731                                             r-cp4p            381\n",
       "5732                                            r-dartr           3445\n",
       "5733                                             r-ddir             94\n",
       "5734                                  r-deconstructsigs           7099\n",
       "5735                                    r-deseqanalysis           1088\n",
       "5736                                          rdfextras           2036\n",
       "5737                                         r-diffcorr            240\n",
       "5738                                           r-dimsum             93\n",
       "5739                                             r-dnet            366\n",
       "5740                                              rdock           2392\n",
       "5741                                     rdp_classifier           2251\n",
       "5742                                            r-dpeak           2145\n",
       "5743                                        rdp-readseq           1737\n",
       "5744                                           rdptools           5094\n",
       "5745                                            r-eacon             54\n",
       "5746                                             readfq           1598\n",
       "5747                                            readseq           6084\n",
       "5748                                           readucks             93\n",
       "5749                                              reago           2140\n",
       "5750                                               real           1677\n",
       "5751                                            realphy           1789\n",
       "5752                                             reaper           1994\n",
       "5753                                            rebaler           5436\n",
       "5754                                     recalladapters           2102\n",
       "5755                                       recentrifuge            446\n",
       "5756                                         recognizer            192\n",
       "5757                                              recon           5068\n",
       "5758                                           recycler           6630\n",
       "5759                                                red           1648\n",
       "5760                                    referenceseeker            568\n",
       "5761                                         refgenconf            609\n",
       "5762                                           refgenie            592\n",
       "5763                                     refgenieserver            293\n",
       "5764                                            refinem            563\n",
       "5765                                      refseq_masher           4112\n",
       "5766                                           regional            874\n",
       "5767                                           regtools           4896\n",
       "5768                                          relocate2           3550\n",
       "5769                              r-empiricalfdr.deseq2           2485\n",
       "5770                                            remurna           1869\n",
       "5771                                             rename           5196\n",
       "5772                                              repaq            656\n",
       "5773                                   reparation_blast          12179\n",
       "5774                                          repdenovo            873\n",
       "5775                                       repeatmasker          19287\n",
       "5776                                      repeatmodeler           7016\n",
       "5777                                        repeatscout           5561\n",
       "5778                                          repenrich           1777\n",
       "5779                                             r-epic           2064\n",
       "5780                                                req            218\n",
       "5781                                           revbayes            944\n",
       "5782                                             reveal           2130\n",
       "5783                                           revtrans           1619\n",
       "5784                                       r-exomedepth           5068\n",
       "5785                                           r-facets           2676\n",
       "5786                                         r-fastbaps            211\n",
       "5787                                            r-fgwas           3522\n",
       "5788                                      r-firebrowser            109\n",
       "5789                                              rfmix           1703\n",
       "5790                                          r-garnett            328\n",
       "5791                                           rgccacmd             42\n",
       "5792                                    r-genometricorr           1581\n",
       "5793                                     r-genomictools            348\n",
       "5794                         r-genomictools.filehandler             39\n",
       "5795                                           r-george           1705\n",
       "5796                                         r-ggbiplot           2161\n",
       "5797                                                rgi          22986\n",
       "5798                                           r-gkmsvm           4103\n",
       "5799                                             r-gmwt            348\n",
       "5800                                           r-goalie           6719\n",
       "5801                                            r-grain           1598\n",
       "5802                                           r-grbase           1761\n",
       "5803                                                rgt           3920\n",
       "5804                                            r-gwpcr           3777\n",
       "5805                                          r-harmony            210\n",
       "5806                                           r-hemdag          11575\n",
       "5807                                       r-histonehmm           7294\n",
       "5808                                       r-htscluster            173\n",
       "5809                                           r-htssip           2318\n",
       "5810                                           ribocode           9401\n",
       "5811                                           ribodiff           1896\n",
       "5812                                           riboplot           1937\n",
       "5813                                         riboraptor           1887\n",
       "5814                                            riborex           1629\n",
       "5815                                           riboseed          13897\n",
       "5816                                           riboseqc             79\n",
       "5817                                       riboseq-rust           1618\n",
       "5818                                          ribotaper           4598\n",
       "5819                                           ribotish           4647\n",
       "5820                                         ribotricer           1535\n",
       "5821                                             r-ic10           3725\n",
       "5822                                         r-ichorcna           2276\n",
       "5823                                             rilseq            326\n",
       "5824                                     r-immunedeconv            625\n",
       "5825                                       r-imputelcmd            357\n",
       "5826                                           r-intego           1777\n",
       "5827                                           r-intlim             84\n",
       "5828                                             ripser           1153\n",
       "5829                                          r-isogene            194\n",
       "5830                                             r-isva            392\n",
       "5831                                        r-jackstraw            291\n",
       "5832                                                rkp             38\n",
       "5833                                            r-leapp           1619\n",
       "5834                                       r-leidenbase           2075\n",
       "5835                                            r-liger           1205\n",
       "5836                                          r-lme4qtl           1615\n",
       "5837                                  r-lncpipereporter           1897\n",
       "5838                                             r-loom           2004\n",
       "5839                                            r-loomr            155\n",
       "5840                                               rmap           2000\n",
       "5841                                  rmats2sashimiplot           3982\n",
       "5842                                              rmats           9355\n",
       "5843                                            rmblast          16134\n",
       "5844                                       r-mcpcounter           2105\n",
       "5845                                    r-metaboanalyst             93\n",
       "5846                                        r-metacoder           1261\n",
       "5847                                        r-metalonda           2256\n",
       "5848                                           r-metama           2093\n",
       "5849                                              rmetl             56\n",
       "5850                                             r-mgsz             71\n",
       "5851                                          r-minems2           4097\n",
       "5852                                         r-minionqc           1530\n",
       "5853                                            r-misha           3370\n",
       "5854                                        r-mixkernel           1391\n",
       "5855                                           r-mkmisc             58\n",
       "5856                                         r-mmgenome           3675\n",
       "5857                                         r-monocle3           2614\n",
       "5858                                      r-motifbinner            970\n",
       "5859                                           r-msqrob           3106\n",
       "5860                                           r-mutoss           4253\n",
       "5861                                           rnabloom            177\n",
       "5862                                       rnablueprint           5503\n",
       "5863                                             rnabob           2521\n",
       "5864                                             rna-cd            166\n",
       "5865                                  rnachipintegrator           3680\n",
       "5866                                           rnaclust           1627\n",
       "5867                                            rnacode           2932\n",
       "5868                                            rnalien          10660\n",
       "5869                                         r-nanopore           3154\n",
       "5870                                   r-nanostringnorm           4303\n",
       "5871                                           rnaquast           3024\n",
       "5872                                        rnaredprint            498\n",
       "5873                                           rnasamba           4537\n",
       "5874                                           rna-seqc          11028\n",
       "5875                                          rnashapes           5982\n",
       "5876                                          rnasketch           3378\n",
       "5877                                             rnasnp           2427\n",
       "5878                                         r-nastiseq           2537\n",
       "5879                                       rnastructure           6003\n",
       "5880                                               rnaz           6658\n",
       "5881                                           r-nbpseq           1627\n",
       "5882                                           rnftools           9690\n",
       "5883                                          r-ngsplot           2702\n",
       "5884                                              roary          20426\n",
       "5885                                    roary2fripan.py           1649\n",
       "5886                                     r-ontologyplot             97\n",
       "5887                                                rop           1826\n",
       "5888                                           ropebwt2           2669\n",
       "5889                                          roprofile           1592\n",
       "5890                                             r-orqa           1621\n",
       "5891                                        r-pctgcdata           2728\n",
       "5892                                             r-peer           2153\n",
       "5893                                         r-perfmeas           2022\n",
       "5894                                      rpf-count-cds            182\n",
       "5895                                           r-phewas           1609\n",
       "5896                                          r-phyext2           2158\n",
       "5897                                    r-phylomeasures           2043\n",
       "5898                                         r-phytools           4622\n",
       "5899                                          r-pipette            678\n",
       "5900                                       rpkmforgenes           1921\n",
       "5901                                     r-popgenreport           1262\n",
       "5902                                        r-pophelper           1557\n",
       "5903                                             r-pore           3666\n",
       "5904                                          r-prestor            231\n",
       "5905                                        r-probmetab           4004\n",
       "5906                                            r-pscbs          18787\n",
       "5907                                            r-qorts           2154\n",
       "5908                                           r-r4cker            162\n",
       "5909                                           r-rapidr           1985\n",
       "5910                                          r-riborex           1376\n",
       "5911                                           r-rphast           1742\n",
       "5912                                           r-rrbgen           1848\n",
       "5913                                          r-rtassel             67\n",
       "5914                                            r-rtfbs           2339\n",
       "5915                                            r-rubic           4782\n",
       "5916                                             r-samr           5819\n",
       "5917                                         r-sartools          12797\n",
       "5918                                          rsat-core           1998\n",
       "5919                                          r-sbpiper           3521\n",
       "5920                                             rscape          12824\n",
       "5921                                           r-sceasy            494\n",
       "5922                                         r-scimpute           3376\n",
       "5923                                           r-scpred            207\n",
       "5924                                               rseg           2078\n",
       "5925                                               rsem          32054\n",
       "5926                                              rseqc          71908\n",
       "5927                                         r-sequenza           2553\n",
       "5928                                              r-sew           1181\n",
       "5929                                             r-sgtr           1534\n",
       "5930                                           r-shaman           1388\n",
       "5931                                              rsidx            155\n",
       "5932                                            r-sigqc           3365\n",
       "5933                                          r-sigtree           2235\n",
       "5934                                    r-singlecellnet             68\n",
       "5935                                           r-sleuth          20440\n",
       "5936                                         r-smartsva           1534\n",
       "5937                                         r-soap-nmr           2100\n",
       "5938                                            r-speaq           7441\n",
       "5939                                        r-spieceasi           2061\n",
       "5940                                              r-spp          18131\n",
       "5941                                       r-statvisual            151\n",
       "5942                                           r-stitch           6260\n",
       "5943                                        r-structssi           2294\n",
       "5944                                            r-swamp           3818\n",
       "5945                                        r-syntactic           5665\n",
       "5946                                        r-tailfindr            501\n",
       "5947                                       r-taxonomizr           2509\n",
       "5948                                        r-tcga2stat           2303\n",
       "5949                                              r-tcr           3504\n",
       "5950                                        r-tftargets             78\n",
       "5951                                          rtg-tools          30573\n",
       "5952                                     r-tidygenomics           2924\n",
       "5953                                                rtk           1542\n",
       "5954                                             r-tmae            451\n",
       "5955                                         r-txrevise             57\n",
       "5956                                     ruby-dna-tools           2027\n",
       "5957                                               rucs             57\n",
       "5958                                             ruffus          16028\n",
       "5959                                            r-umi4c           1417\n",
       "5960                                     rust-bio-tools          31528\n",
       "5961                                  rust-ncbitaxonomy           6786\n",
       "5962                                      rust-overlaps           2454\n",
       "5963                                       r-velocyto.r            805\n",
       "5964                                        r-virfinder           1261\n",
       "5965                                           r-vision            826\n",
       "5966                                            rvtests           2968\n",
       "5967                                           r-wasabi          17869\n",
       "5968                                            r-wgcna          14044\n",
       "5969                                       r-whopgenome           1918\n",
       "5970                            r-workflowscriptscommon           3306\n",
       "5971                                            r-xcell           1892\n",
       "5972                                             rxdock           1264\n",
       "5973                                              r-xgr            366\n",
       "5974                                           r-xmlrpc           2345\n",
       "5975                                           r-zerone           1758\n",
       "5976                                            s3gof3r           7168\n",
       "5977                                              sabre           1333\n",
       "5978                                        saffrontree           1675\n",
       "5979                                           sailfish          14884\n",
       "5980                                                sak           1904\n",
       "5981                                             salmid            230\n",
       "5982                                             salmon         116310\n",
       "5983                                                sam           2361\n",
       "5984                                           sambamba         202487\n",
       "5985                                         samblaster          32239\n",
       "5986                                            samclip          10145\n",
       "5987                                             sampei            110\n",
       "5988                                       sample-sheet           2696\n",
       "5989                                            samplot          17946\n",
       "5990                                            samsift          12485\n",
       "5991                                          samsifter           2177\n",
       "5992                                           samstats           1212\n",
       "5993                                             samsum           1284\n",
       "5994                                           samtools        1836974\n",
       "5995                                         sanitizeme            146\n",
       "5996                                  sarscov2formatter            205\n",
       "5997                                    sarscov2summary            215\n",
       "5998                                             satrap           1742\n",
       "5999                                           satsuma2           1712\n",
       "6000                                             savage           7836\n",
       "6001                                     sbg-cwl-runner           2720\n",
       "6002                                             sbpipe           2053\n",
       "6003                                        sc3-scripts            275\n",
       "6004                                             scaden            253\n",
       "6005                                   scaffold_builder           1599\n",
       "6006                                           scagaire             33\n",
       "6007                                              scala           3063\n",
       "6008                                            scallop           2709\n",
       "6009                                         scallop-lr           1050\n",
       "6010                                            scalpel          33981\n",
       "6011                                          scanindel           1696\n",
       "6012                                          scanorama            312\n",
       "6013                                             scanpy          25595\n",
       "6014                                     scanpy-scripts           1695\n",
       "6015                                              scapp             64\n",
       "6016                                     scater-scripts            137\n",
       "6017                                              sccaf            604\n",
       "6018                                           sccaller           3313\n",
       "6019                                            scelvis           1067\n",
       "6020                                                scg           1818\n",
       "6021                                           schavott           7769\n",
       "6022                                      schicexplorer            477\n",
       "6023                                 scikits-datasmooth           2384\n",
       "6024                                             sciphi           2296\n",
       "6025                                             scirpy            166\n",
       "6026                                          scmap-cli            387\n",
       "6027                                              scnic            802\n",
       "6028                                             scoary           4367\n",
       "6029                                              scoop           4462\n",
       "6030                                         scpred-cli            333\n",
       "6031                                             scprep           1236\n",
       "6032                                          scran-cli             36\n",
       "6033                                           scrappie           5220\n",
       "6034                                  screamingbackpack           4388\n",
       "6035                                               scrm           6861\n",
       "6036                                           scrublet           1819\n",
       "6037                                             scssim            749\n",
       "6038                                            sctools            624\n",
       "6039                                             scvelo            732\n",
       "6040                                               scvi           3193\n",
       "6041                                              scvis            120\n",
       "6042                                       scvis_galaxy             86\n",
       "6043                                         scvi-tools             87\n",
       "6044                                         scxa-plots             41\n",
       "6045                                                sdm           1037\n",
       "6046                                     sdrf-pipelines            500\n",
       "6047                                              seacr            487\n",
       "6048                                          searchgui          34090\n",
       "6049                                             secapr          17312\n",
       "6050                                         secimtools           2248\n",
       "6051                                             seedme           3410\n",
       "6052                                               seer           1648\n",
       "6053                                           segemehl           6795\n",
       "6054                                  segmentation-fold           4048\n",
       "6055                                           segtools           4075\n",
       "6056                                             segway          10739\n",
       "6057                                        selectfasta            438\n",
       "6058                             selectsequencesfrommsa           4308\n",
       "6059                                         selene-sdk          13595\n",
       "6060                                            selscan           4540\n",
       "6061                                             semeta           1649\n",
       "6062                                            semidbm           2980\n",
       "6063                                              sensv            871\n",
       "6064                                           sentieon          43429\n",
       "6065                                               sepp          34758\n",
       "6066                                      sepp-refgg138            113\n",
       "6067                                   sepp-refsilva128             56\n",
       "6068                                              seq2c          12771\n",
       "6069                                            seq2hla           3403\n",
       "6070                                        seq2science            335\n",
       "6071                                              seqan          26220\n",
       "6072                                             seqan3            242\n",
       "6073                                      seqan_tcoffee           2352\n",
       "6074                                          seqbuster          16181\n",
       "6075                                         seqcluster          40295\n",
       "6076                                              seqfu            698\n",
       "6077                                            seq-gen           3433\n",
       "6078                                             seqhax           1233\n",
       "6079                                             seqkit          80431\n",
       "6080                                            seqlogo           9514\n",
       "6081                                          seqmagick           5049\n",
       "6082                                             seqmap           1655\n",
       "6083                                            seqprep           5744\n",
       "6084                                          seqscreen             80\n",
       "6085                                        seq-seq-pan           4000\n",
       "6086                                           seqsero2            398\n",
       "6087                                           seqstats            469\n",
       "6088                                              seqtk          92153\n",
       "6089                                   seq-to-first-iso            231\n",
       "6090                                            sequana          24584\n",
       "6091                                      sequencetools           4726\n",
       "6092                                     sequenza-utils           3438\n",
       "6093                                        sequnwinder             33\n",
       "6094                                            seqwish           2445\n",
       "6095                                          seqyclean           3277\n",
       "6096                                             seroba           2221\n",
       "6097                                           sesimcmc            993\n",
       "6098                                  setuptools_cython           2709\n",
       "6099                                     seurat-scripts            331\n",
       "6100                                sevenbridges-python          32373\n",
       "6101                                      sexdeterrmine           2601\n",
       "6102                                          sff2fastq           2120\n",
       "6103                                               sfld            417\n",
       "6104                                           sfs_code           1740\n",
       "6105                                                sga           4725\n",
       "6106                                            shannon            159\n",
       "6107                                        shannon_cpp           1361\n",
       "6108                                           shape_it           2347\n",
       "6109                                           shapeit4           2274\n",
       "6110                                        shapemapper           1730\n",
       "6111                                       shapeshifter            215\n",
       "6112                                   shapeshifter-cli            159\n",
       "6113                                          sharedmem           3638\n",
       "6114                                              shark           1313\n",
       "6115                                            sherpas             92\n",
       "6116                                         shigatyper            145\n",
       "6117                                             shiver           7320\n",
       "6118                                            shmlast           6546\n",
       "6119                                             shorah          12621\n",
       "6120                                         shortstack           5118\n",
       "6121                                            shovill          14189\n",
       "6122                                             showit           1081\n",
       "6123                                          shustring           4783\n",
       "6124                                            sibelia           4345\n",
       "6125                                             sicer2           1467\n",
       "6126                                              sicer           4361\n",
       "6127                                             sickle           7857\n",
       "6128                                        sickle-trim          59616\n",
       "6129                                               sidr             18\n",
       "6130                                           sierrapy            607\n",
       "6131                                             sift4g           1840\n",
       "6132                                             sigmut            268\n",
       "6133                                             sim4db           1785\n",
       "6134                                              simka           4016\n",
       "6135                                            simlord           7491\n",
       "6136                                          simplesam            101\n",
       "6137                               simple_sv_annotation          24941\n",
       "6138                                           simwalk2           1660\n",
       "6139                                               sina          67310\n",
       "6140                                singlecell-barcodes           8705\n",
       "6141                                            singlem             79\n",
       "6142                                 sirius-csifingerid           8017\n",
       "6143                                                sis           3594\n",
       "6144                                          sistr_cmd          11307\n",
       "6145                                            sixgill           4433\n",
       "6146                                                ska           2217\n",
       "6147                                              skesa           9840\n",
       "6148                                            sketchy           1490\n",
       "6149                                             skewer          11993\n",
       "6150                                              skmer           4404\n",
       "6151                                           slamdunk           5996\n",
       "6152                                            slclust          15084\n",
       "6153                                              slimm           1655\n",
       "6154                                             slivar          10960\n",
       "6155                                             slncky           1978\n",
       "6156                                             sloika           1624\n",
       "6157                                              smaca            258\n",
       "6158                                              smafa           1015\n",
       "6159                               smallgenomeutilities           4112\n",
       "6160                                              smalt           4399\n",
       "6161                                        smartdenovo           2314\n",
       "6162                                  smashbenchmarking           1604\n",
       "6163                                            smashpp           1098\n",
       "6164                                         smcounter2           3777\n",
       "6165                                               smeg           7705\n",
       "6166                                           smhasher           2745\n",
       "6167                                              smina           3295\n",
       "6168                                               smof            254\n",
       "6169                                             smoove          16139\n",
       "6170                                         smudgeplot           2313\n",
       "6171                                           snakefmt            147\n",
       "6172                                          snakemake         261989\n",
       "6173                                         snakeparse           3411\n",
       "6174                                               snap           8310\n",
       "6175                                       snap-aligner          11222\n",
       "6176                                          snaptools            292\n",
       "6177                                           sniffles          13847\n",
       "6178                                             snippy          30762\n",
       "6179                                               snmf           1894\n",
       "6180                                               snns           2113\n",
       "6181                                            snoscan           6777\n",
       "6182                                           snostrip           1127\n",
       "6183                                          snp-dists          10769\n",
       "6184                                             snpeff          87645\n",
       "6185                                           snpgenie             46\n",
       "6186                                            snpiphy           4905\n",
       "6187                                        snp-mutator            768\n",
       "6188                                          snpomatic           1715\n",
       "6189                                         snp-pileup           4014\n",
       "6190                                       snp-pipeline           6321\n",
       "6191                                            snpsift          26730\n",
       "6192                                          snp-sites          15811\n",
       "6193                                           snpsplit           6181\n",
       "6194                                              snver            175\n",
       "6195                                      snvphyl-tools           3105\n",
       "6196                                        soapaligner           2062\n",
       "6197                                       soapcoverage           1794\n",
       "6198                                        soapdenovo2           4161\n",
       "6199                        soapdenovo2-errorcorrection           2152\n",
       "6200                              soapdenovo2-gapcloser           2395\n",
       "6201                                soapdenovo2-prepare           2077\n",
       "6202                                   soapdenovo-trans           2206\n",
       "6203                                             soapec           2324\n",
       "6204                                         soapsplice           1822\n",
       "6205                                    socksipy-branch           5296\n",
       "6206                                              socru           6240\n",
       "6207                                           solexaqa           1727\n",
       "6208                                           solvebio           8647\n",
       "6209                                         somaticseq           1993\n",
       "6210                                     somatic-sniper           2221\n",
       "6211                                      sonicparanoid           4145\n",
       "6212                                             sonlib           2289\n",
       "6213                                     sorted_nearest          35031\n",
       "6214                                          sortmerna         107451\n",
       "6215                                      sourcetracker            168\n",
       "6216                                           sourmash          45667\n",
       "6217                                        spacepharer            658\n",
       "6218                                             spades          97089\n",
       "6219                                              spaln          10605\n",
       "6220                                             spanki           1643\n",
       "6221                                              sparc           2552\n",
       "6222                                             sparcc            446\n",
       "6223                                    sparseassembler           2497\n",
       "6224                            sparse-neighbors-search           5047\n",
       "6225                                          spectacle           1711\n",
       "6226                                spectra-cluster-cli           2096\n",
       "6227                                        spectral_hk           2056\n",
       "6228                                    spectrassembler           1961\n",
       "6229                                     spectrum_utils           1768\n",
       "6230                                              spine           1821\n",
       "6231                                             spingo           2162\n",
       "6232                                           spliceai            785\n",
       "6233                                   spliceai-wrapper             57\n",
       "6234                                          splicemap           1727\n",
       "6235                                           splitmem           1750\n",
       "6236                                               spoa           3071\n",
       "6237                                          spotyping           2149\n",
       "6238                                         spotyping3            208\n",
       "6239                                              sprai           1700\n",
       "6240                                             spring           1063\n",
       "6241                                          sprinkles           2744\n",
       "6242                                          spydrpick           4545\n",
       "6243                              sqlalchemy-datatables           3925\n",
       "6244                                      sqlitebrowser           1735\n",
       "6245                                                sqt           8981\n",
       "6246                                            squeakr           3027\n",
       "6247                                              squid           3479\n",
       "6248                                             squizz           3718\n",
       "6249                                          sra-tools          94909\n",
       "6250                                               srax            124\n",
       "6251                                           srnapipe           1533\n",
       "6252                                            srprism          17983\n",
       "6253                                              srst2           6801\n",
       "6254                                              ssake           1807\n",
       "6255                                              sshmm           1981\n",
       "6256                                          ssu-align           1989\n",
       "6257                                          ssuissero             25\n",
       "6258                                             stacks          39603\n",
       "6259                                     stacks_summary           3913\n",
       "6260                                      staden_io_lib           2402\n",
       "6261                                              stamp           3535\n",
       "6262                                               star         267257\n",
       "6263                                            staramr          13956\n",
       "6264                                           starcode           8817\n",
       "6265                                           starfish           2241\n",
       "6266                                        star-fusion          30292\n",
       "6267                                              stark            642\n",
       "6268                                           starseqr           7746\n",
       "6269                                         start-asap             33\n",
       "6270                                            stellar           1958\n",
       "6271                                         stereogene           1769\n",
       "6272                                         stoatydive            192\n",
       "6273                                          strainest           4351\n",
       "6274                                       strainseeker            394\n",
       "6275                                        straitrazor           1751\n",
       "6276                                             stream          31573\n",
       "6277                                        stream_atac          12740\n",
       "6278                                            strelka          21613\n",
       "6279                                             stride           2084\n",
       "6280                                             strike           1576\n",
       "6281                                   stringdecomposer            306\n",
       "6282                                         stringmeup            136\n",
       "6283                                         stringmlst          15028\n",
       "6284                                          stringtie          63535\n",
       "6285                                           strip_it           2281\n",
       "6286                                            strling            939\n",
       "6287                                          structure           3214\n",
       "6288                                 structureharvester           2114\n",
       "6289                                            strudel           1693\n",
       "6290                                          sts-smctc           1903\n",
       "6291                                            subread         132027\n",
       "6292                                          sumaclust           2096\n",
       "6293                                       suma_package           1817\n",
       "6294                                     super_distance           2262\n",
       "6295                                        super-focus           5394\n",
       "6296                                              suppa           2749\n",
       "6297                                               sure           2001\n",
       "6298                                           surpyvor            290\n",
       "6299                                           survivor          13287\n",
       "6300                                                sv2           1406\n",
       "6301                                              svaba           1202\n",
       "6302                                         svanalyzer            885\n",
       "6303                                               svdb           9689\n",
       "6304                                              svict           2462\n",
       "6305                                               svim           1805\n",
       "6306                                           svim-asm             63\n",
       "6307                                          svist4get            666\n",
       "6308                                             svjedi             45\n",
       "6309                                           svmlight           1869\n",
       "6310                                               svtk            493\n",
       "6311                                            svtools          17094\n",
       "6312                                            svtyper          19689\n",
       "6313                                              svviz           5631\n",
       "6314                                            swalign           4154\n",
       "6315                                              swarm          13352\n",
       "6316                                       sweepfinder2           1978\n",
       "6317                                               swga           5105\n",
       "6318                                          swiftlink           2143\n",
       "6319                                              swipe           3372\n",
       "6320                                              sword           4027\n",
       "6321                                      synapseclient          14180\n",
       "6322                                              tabix         101027\n",
       "6323                                             tablet           3819\n",
       "6324                                            tabview           2746\n",
       "6325                                               taco           3665\n",
       "6326                                         tadarida-c           4859\n",
       "6327                                         tadarida-d           6461\n",
       "6328                                             tadbit            623\n",
       "6329                                            tadtool           2375\n",
       "6330                                             taeper           1923\n",
       "6331                                                tag            400\n",
       "6332                                             tagger           1710\n",
       "6333                                             tagore            128\n",
       "6334                                         takeabreak           2181\n",
       "6335                                             talloc           1743\n",
       "6336                                   tandem-genotypes            377\n",
       "6337                                             tantan           3273\n",
       "6338                                           tapestry            440\n",
       "6339                                             tardis          12546\n",
       "6340                                           targetdb             74\n",
       "6341                                       targetfinder           2973\n",
       "6342                                             targqc           2688\n",
       "6343                                 tasmanian-mismatch            300\n",
       "6344                                             tassel           5571\n",
       "6345                                         taxator-tk           1761\n",
       "6346                                          taxmapper           5345\n",
       "6347                                           taxonkit          13187\n",
       "6348                                           taxonomy           1143\n",
       "6349                                     taxonomy_ranks            101\n",
       "6350                                             taxopy            187\n",
       "6351                                             taxsbp            402\n",
       "6352                                          taxtastic           5480\n",
       "6353                                            tbl2asn          47855\n",
       "6354                                    tbl2asn-forever           8605\n",
       "6355                                          tbox-scan             78\n",
       "6356                                        tb-profiler           3171\n",
       "6357                                  tb_variant_filter            837\n",
       "6358                                        tbvcfreport            390\n",
       "6359                                           t-coffee           4671\n",
       "6360                                           t_coffee          81184\n",
       "6361                                          tdrmapper           9594\n",
       "6362                                              tedna           1885\n",
       "6363                                          telescope           1732\n",
       "6364                                           teloclip           1717\n",
       "6365                                             telseq           3581\n",
       "6366                                              tendo            131\n",
       "6367                                            tepeaks            795\n",
       "6368                                              tepid           3350\n",
       "6369                                           terminus            604\n",
       "6370                                           tesorter            695\n",
       "6371                                      tetranscripts           2194\n",
       "6372                                            tetyper            353\n",
       "6373                                          textinput           3202\n",
       "6374                                       tgsgapcloser            421\n",
       "6375                                                tgt           4101\n",
       "6376                                        thapbi-pict           2439\n",
       "6377                                thermorawfileparser          12636\n",
       "6378                                            thesias            631\n",
       "6379                                             theta2           6992\n",
       "6380                                             tiddit           6666\n",
       "6381                                         tidehunter           2018\n",
       "6382                                              tidyp          91896\n",
       "6383                                            tigmint            328\n",
       "6384                                            tinscan           1876\n",
       "6385                                          tinyalign            920\n",
       "6386                                          tinyfasta            107\n",
       "6387                                           tinysink           1701\n",
       "6388                                            tiptoft           3755\n",
       "6389                                            tirmite           5298\n",
       "6390                                            tmalign           1804\n",
       "6391                                               tn93          11851\n",
       "6392                                             tobias           8379\n",
       "6393                                               toil          29386\n",
       "6394                                           toolshed          20049\n",
       "6395                                              topas            178\n",
       "6396                                             tophat          33640\n",
       "6397                                 tophat-recondition          17041\n",
       "6398                                          toulligqc           7106\n",
       "6399                                      tpmcalculator           1526\n",
       "6400                                                tpp           3310\n",
       "6401                                             tqdist           1683\n",
       "6402                                             tracer           4023\n",
       "6403                                           trackhub           6884\n",
       "6404                                              tracy          11815\n",
       "6405                                            traitar             75\n",
       "6406                                         transabyss           6478\n",
       "6407                                          transcomb           1701\n",
       "6408                                           transcov            234\n",
       "6409                                       transdecoder          23937\n",
       "6410                                      transgenescan           1608\n",
       "6411                                         transindel             69\n",
       "6412                                            transit           5501\n",
       "6413                                     translate-gard           3232\n",
       "6414                                        translatorx           1916\n",
       "6415                                           translig            801\n",
       "6416                                       transposcope            286\n",
       "6417                                      transposonpsi             82\n",
       "6418                                          transrate            885\n",
       "6419                                    transrate-tools           2339\n",
       "6420                                        transtermhp           2218\n",
       "6421                                            trawler           5288\n",
       "6422                                           treebest           4945\n",
       "6423                                        treecluster             35\n",
       "6424                                            treekin           3465\n",
       "6425                                          treemaker           2064\n",
       "6426                                            treemix           4704\n",
       "6427                                           treerecs            314\n",
       "6428                                           treesapp           2165\n",
       "6429                                          treeswift            445\n",
       "6430                                           treetime           7993\n",
       "6431                                                trf          18658\n",
       "6432                                           triform2           2042\n",
       "6433                                           trimadap           6015\n",
       "6434                                             trimal          18495\n",
       "6435                                        trim-galore          61752\n",
       "6436                                        trimmomatic         108735\n",
       "6437                                           trinculo           1841\n",
       "6438                                            trinity          58666\n",
       "6439                                          trinotate           9766\n",
       "6440                                            triqler            259\n",
       "6441                                        trnanalysis           6120\n",
       "6442                                        trnascan-se          17057\n",
       "6443                                            trtools             92\n",
       "6444                                         trumicount           9512\n",
       "6445                                            truvari           1720\n",
       "6446                                          trycycler             67\n",
       "6447                                              tspex            757\n",
       "6448                                              tssar           1914\n",
       "6449                                       twobitreader           7717\n",
       "6450                                            twopaco           1741\n",
       "6451                                   tximport-scripts             87\n",
       "6452                                       ucsc-addcols           8000\n",
       "6453                                         ucsc-ameme           6451\n",
       "6454                                       ucsc-autodtd           6542\n",
       "6455                                       ucsc-autosql           6474\n",
       "6456                                       ucsc-autoxml           6630\n",
       "6457                                       ucsc-avecols           6390\n",
       "6458                                      ucsc-axtchain           7087\n",
       "6459                                       ucsc-axtsort           6576\n",
       "6460                                       ucsc-axtswap           6570\n",
       "6461                                      ucsc-axttomaf           8043\n",
       "6462                                      ucsc-axttopsl           8008\n",
       "6463                                      ucsc-bamtopsl           5000\n",
       "6464                                       ucsc-bedclip          14210\n",
       "6465                              ucsc-bedcommonregions           8087\n",
       "6466                                   ucsc-bedcoverage           8052\n",
       "6467                               ucsc-bedextendranges           8095\n",
       "6468                                  ucsc-bedgeneparts           7957\n",
       "6469                                  ucsc-bedgraphpack           8130\n",
       "6470                              ucsc-bedgraphtobigwig          72277\n",
       "6471                                  ucsc-bedintersect           8003\n",
       "6472                           ucsc-beditemoverlapcount           8233\n",
       "6473                              ucsc-bedjointaboffset           2935\n",
       "6474                                    ucsc-bedpileups           7980\n",
       "6475                              ucsc-bedremoveoverlap           8024\n",
       "6476                        ucsc-bedrestricttopositions           8152\n",
       "6477                                       ucsc-bedsort          14341\n",
       "6478                                   ucsc-bedtobigbed          35333\n",
       "6479                                 ucsc-bedtogenepred          16092\n",
       "6480                                      ucsc-bedtopsl           7966\n",
       "6481                            ucsc-bedweedoverlapping           7983\n",
       "6482                                    ucsc-bigbedinfo          14280\n",
       "6483                              ucsc-bigbednameditems           8136\n",
       "6484                                 ucsc-bigbedsummary          13821\n",
       "6485                                   ucsc-bigbedtobed          19934\n",
       "6486                                   ucsc-bigmaftomaf           4741\n",
       "6487                                   ucsc-bigpsltopsl           6582\n",
       "6488                          ucsc-bigwigaverageoverbed          16017\n",
       "6489                                     ucsc-bigwigcat           8293\n",
       "6490                                 ucsc-bigwigcluster           8240\n",
       "6491                               ucsc-bigwigcorrelate           8166\n",
       "6492                                    ucsc-bigwiginfo          20321\n",
       "6493                                   ucsc-bigwigmerge          10077\n",
       "6494                                 ucsc-bigwigsummary          27755\n",
       "6495                              ucsc-bigwigtobedgraph          20848\n",
       "6496                                   ucsc-bigwigtowig          16153\n",
       "6497                                    ucsc-blasttopsl           8146\n",
       "6498                                          ucsc-blat           5863\n",
       "6499                                        ucsc-catdir           6473\n",
       "6500                                  ucsc-catuncomment           6406\n",
       "6501                                  ucsc-cell-browser           1739\n",
       "6502                               ucsc-chainantirepeat           7941\n",
       "6503                                   ucsc-chainbridge           1359\n",
       "6504                                   ucsc-chainfilter           8027\n",
       "6505                                ucsc-chainmergesort           8479\n",
       "6506                                      ucsc-chainnet           8325\n",
       "6507                                   ucsc-chainprenet           8130\n",
       "6508                                     ucsc-chainsort           8057\n",
       "6509                                    ucsc-chainsplit           8247\n",
       "6510                                 ucsc-chainstitchid           8154\n",
       "6511                                     ucsc-chainswap           7968\n",
       "6512                                    ucsc-chaintoaxt           8032\n",
       "6513                                    ucsc-chaintopsl           7955\n",
       "6514                               ucsc-chaintopslbasic           8035\n",
       "6515                                 ucsc-checkagpandfa           6496\n",
       "6516                             ucsc-checkcoveragegaps           6402\n",
       "6517                               ucsc-checkhgfindspec           6358\n",
       "6518                              ucsc-checktablecoords           6413\n",
       "6519                                   ucsc-chopfalines           6415\n",
       "6520                             ucsc-chromgraphfrombin           6500\n",
       "6521                               ucsc-chromgraphtobin           6574\n",
       "6522                                  ucsc-clustergenes           4784\n",
       "6523                                  ucsc-coltransform           6432\n",
       "6524                                    ucsc-countchars           6375\n",
       "6525                                ucsc-crtreeindexbed           8058\n",
       "6526                               ucsc-crtreesearchbed           8047\n",
       "6527                                       ucsc-dbsnoop           6539\n",
       "6528                                       ucsc-dbtrash           6519\n",
       "6529                                      ucsc-endsinlf           2937\n",
       "6530                                     ucsc-estorient           7728\n",
       "6531                        ucsc-expmatrixtobarchartbed           4931\n",
       "6532                                       ucsc-faalign           6473\n",
       "6533                                         ucsc-facmp           6381\n",
       "6534                                       ucsc-facount           7163\n",
       "6535                                      ucsc-fafilter           6532\n",
       "6536                                     ucsc-fafiltern           6388\n",
       "6537                                        ucsc-fafrag           6425\n",
       "6538                                       ucsc-fanoise           6396\n",
       "6539                                   ucsc-faonerecord           8231\n",
       "6540                                  ucsc-fapolyasizes           8154\n",
       "6541                                   ucsc-farandomize           7950\n",
       "6542                                          ucsc-farc           7968\n",
       "6543                                        ucsc-fasize           8342\n",
       "6544                                 ucsc-fasomerecords           9000\n",
       "6545                                       ucsc-fasplit           9048\n",
       "6546                        ucsc-fastqstatsandsubsample           4799\n",
       "6547                                     ucsc-fastqtofa           8119\n",
       "6548                                     ucsc-fatofastq           8160\n",
       "6549                                       ucsc-fatotab           7962\n",
       "6550                                    ucsc-fatotwobit          48589\n",
       "6551                                       ucsc-fatrans           8153\n",
       "6552                                   ucsc-featurebits           7992\n",
       "6553                               ucsc-fetchchromsizes          16862\n",
       "6554                                     ucsc-findmotif           7786\n",
       "6555                                     ucsc-gaptolift           7997\n",
       "6556                                 ucsc-genepredcheck           9301\n",
       "6557                                ucsc-genepredfilter           8099\n",
       "6558                                 ucsc-genepredhisto           7931\n",
       "6559                           ucsc-genepredsinglecover           8199\n",
       "6560                                 ucsc-genepredtobed          24900\n",
       "6561                         ucsc-genepredtobiggenepred           9386\n",
       "6562                             ucsc-genepredtofakepsl           9824\n",
       "6563                                 ucsc-genepredtogtf          17857\n",
       "6564                           ucsc-genepredtomafframes           9594\n",
       "6565                                ucsc-genepredtoprot           4698\n",
       "6566                                       ucsc-gensub2           2881\n",
       "6567                                        ucsc-getrna           7995\n",
       "6568                                    ucsc-getrnapred           7985\n",
       "6569                                ucsc-gff3togenepred          18400\n",
       "6570                                     ucsc-gff3topsl           6531\n",
       "6571                                        ucsc-gmtime           4651\n",
       "6572                                 ucsc-gtftogenepred          38615\n",
       "6573                                      ucsc-headrest           4580\n",
       "6574                                   ucsc-hgbbidblink           4618\n",
       "6575                                     ucsc-hgfakeagp           4711\n",
       "6576                                    ucsc-hgfindspec           4600\n",
       "6577                                   ucsc-hggcpercent          10363\n",
       "6578                                   ucsc-hggoldgapgl           1373\n",
       "6579                                     ucsc-hgloadbed           6357\n",
       "6580                                   ucsc-hgloadchain           6453\n",
       "6581                                     ucsc-hgloadmaf           6363\n",
       "6582                                     ucsc-hgloadnet           4735\n",
       "6583                                     ucsc-hgloadout           4755\n",
       "6584                               ucsc-hgloadoutjoined           4673\n",
       "6585                                  ucsc-hgloadsqltab           2896\n",
       "6586                                  ucsc-hgloadwiggle           6480\n",
       "6587                                  ucsc-hgspeciesrna           4686\n",
       "6588                                     ucsc-hgsqldump           4673\n",
       "6589                                     ucsc-hgtrackdb           4595\n",
       "6590                                     ucsc-hgvstovcf           1349\n",
       "6591                                     ucsc-htmlcheck           4608\n",
       "6592                                      ucsc-hubcheck           4792\n",
       "6593                                ucsc-hubpubliccheck           4544\n",
       "6594                                         ucsc-ixixx           6962\n",
       "6595                                      ucsc-lavtoaxt           4726\n",
       "6596                                      ucsc-lavtopsl           6475\n",
       "6597                                      ucsc-ldhggene           4641\n",
       "6598                                      ucsc-liftover          39204\n",
       "6599                                        ucsc-liftup           6543\n",
       "6600                                     ucsc-linestora           4717\n",
       "6601                                     ucsc-localtime           4563\n",
       "6602                                   ucsc-mafaddirows           6317\n",
       "6603                                   ucsc-mafaddqrows           6527\n",
       "6604                                   ucsc-mafcoverage           6434\n",
       "6605                                      ucsc-maffetch           6397\n",
       "6606                                     ucsc-maffilter           6588\n",
       "6607                                       ucsc-maffrag           6373\n",
       "6608                                      ucsc-maffrags           6414\n",
       "6609                                       ucsc-mafgene           6422\n",
       "6610                                    ucsc-mafmefirst           6365\n",
       "6611                                      ucsc-maforder           6347\n",
       "6612                                     ucsc-mafranges           6370\n",
       "6613                                  ucsc-mafsinregion           6654\n",
       "6614                                ucsc-mafspecieslist           6535\n",
       "6615                              ucsc-mafspeciessubset           6529\n",
       "6616                                      ucsc-mafsplit           6377\n",
       "6617                                   ucsc-mafsplitpos           6565\n",
       "6618                                      ucsc-maftoaxt           6485\n",
       "6619                                   ucsc-maftobigmaf           4716\n",
       "6620                                      ucsc-maftopsl           6357\n",
       "6621                                   ucsc-maftosnpbed           6517\n",
       "6622                                 ucsc-maketablelist           4604\n",
       "6623                                     ucsc-maskoutfa           8051\n",
       "6624                                        ucsc-mktime           4587\n",
       "6625                                    ucsc-mrnatogene           4701\n",
       "6626                                ucsc-netchainsubset           6483\n",
       "6627                                      ucsc-netclass           8020\n",
       "6628                                     ucsc-netfilter           6493\n",
       "6629                                      ucsc-netsplit           4693\n",
       "6630                                   ucsc-netsyntenic           7945\n",
       "6631                                      ucsc-nettoaxt           7995\n",
       "6632                                      ucsc-nettobed           9592\n",
       "6633                                       ucsc-newprog           4602\n",
       "6634                                 ucsc-newpythonprog           9586\n",
       "6635                                       ucsc-nibfrag          12491\n",
       "6636                                       ucsc-nibsize           9563\n",
       "6637                                    ucsc-oligomatch          10938\n",
       "6638                                 ucsc-overlapselect           3355\n",
       "6639                                          ucsc-para           2931\n",
       "6640                                     ucsc-parafetch           9613\n",
       "6641                                       ucsc-parahub           2868\n",
       "6642                                   ucsc-parahubstop           2945\n",
       "6643                                      ucsc-paranode           2937\n",
       "6644                                 ucsc-paranodestart           2882\n",
       "6645                                ucsc-paranodestatus           2895\n",
       "6646                                  ucsc-paranodestop           2970\n",
       "6647                                       ucsc-parasol           2893\n",
       "6648                                      ucsc-parasync           9622\n",
       "6649                                   ucsc-paratestjob           2940\n",
       "6650                            ucsc-positionaltblcheck           9844\n",
       "6651                                        ucsc-pslcat           9636\n",
       "6652                                 ucsc-pslcdnafilter          11975\n",
       "6653                                      ucsc-pslcheck          10082\n",
       "6654                                ucsc-psldropoverlap           9623\n",
       "6655                                     ucsc-pslfilter           9681\n",
       "6656                                      ucsc-pslhisto           9485\n",
       "6657                           ucsc-pslliftsubrangeblat           9504\n",
       "6658                                        ucsc-pslmap           8529\n",
       "6659                               ucsc-pslmappostchain           5035\n",
       "6660                                  ucsc-pslmrnacover           9836\n",
       "6661                                      ucsc-pslpairs           9766\n",
       "6662                                  ucsc-pslpartition           9613\n",
       "6663                                  ucsc-pslpostarget           9912\n",
       "6664                                     ucsc-pslpretty           9622\n",
       "6665                                         ucsc-pslrc           4610\n",
       "6666                                ucsc-pslrecalcmatch          10357\n",
       "6667                                       ucsc-pslreps           9673\n",
       "6668                                      ucsc-pslscore           9464\n",
       "6669                                     ucsc-pslselect           9607\n",
       "6670                                ucsc-pslsomerecords           2928\n",
       "6671                                       ucsc-pslsort          10227\n",
       "6672                                      ucsc-pslstats           9520\n",
       "6673                                       ucsc-pslswap           9629\n",
       "6674                                      ucsc-psltobed           9797\n",
       "6675                                   ucsc-psltobigpsl           9857\n",
       "6676                                    ucsc-psltochain           9958\n",
       "6677                                     ucsc-psltopslx           9720\n",
       "6678                                      ucsc-pslxtofa           9489\n",
       "6679                                    ucsc-qacagplift           9493\n",
       "6680                                       ucsc-qactoqa           9634\n",
       "6681                                      ucsc-qactowig           9877\n",
       "6682                                       ucsc-qatoqac           9639\n",
       "6683                                   ucsc-randomlines           9578\n",
       "6684                                    ucsc-rasqlquery           9521\n",
       "6685                                     ucsc-ratolines           9535\n",
       "6686                                       ucsc-ratotab           9481\n",
       "6687                                      ucsc-rmfadups           9651\n",
       "6688                                    ucsc-rowstocols           9560\n",
       "6689                                   ucsc-spacedtotab           9503\n",
       "6690                                     ucsc-splitfile           9695\n",
       "6691                             ucsc-splitfilebycolumn          10028\n",
       "6692                                      ucsc-sqltoxml           9639\n",
       "6693                                     ucsc-stringify           9510\n",
       "6694                                       ucsc-subchar           9700\n",
       "6695                                     ucsc-subcolumn           9545\n",
       "6696                                     ucsc-taillines           9578\n",
       "6697                                      ucsc-tdbquery           9765\n",
       "6698                                 ucsc-texthistogram           9457\n",
       "6699                                    ucsc-ticktodate           9702\n",
       "6700                                       ucsc-tolower           9840\n",
       "6701                                       ucsc-toupper           9624\n",
       "6702                         ucsc-transmappsltogenepred           5086\n",
       "6703                                        ucsc-trfbig           9601\n",
       "6704                                     ucsc-twobitdup           9507\n",
       "6705                                    ucsc-twobitinfo          15618\n",
       "6706                                    ucsc-twobitmask           9676\n",
       "6707                                    ucsc-twobittofa          27327\n",
       "6708                                 ucsc-validatefiles           9592\n",
       "6709                              ucsc-validatemanifest           9514\n",
       "6710                                       ucsc-websync           2923\n",
       "6711                                  ucsc-wigcorrelate           9906\n",
       "6712                                   ucsc-wigtobigwig          53810\n",
       "6713                                      ucsc-wordline           9426\n",
       "6714                                        ucsc-xmlcat           9662\n",
       "6715                                      ucsc-xmltosql           9724\n",
       "6716                                            udocker           3398\n",
       "6717                                       ultraheatmap            218\n",
       "6718                                               umap            125\n",
       "6719                                               umis          34432\n",
       "6720                                           umitools           3829\n",
       "6721                                          umi_tools          38413\n",
       "6722                                            unfazed            126\n",
       "6723                                          unicycler          19892\n",
       "6724                                            unifrac          52573\n",
       "6725                                            uniprot           2186\n",
       "6726                                             unitas           1557\n",
       "6727                                      unitig-caller           3140\n",
       "6728                                     unitig-counter           2976\n",
       "6729                                            unmerge            395\n",
       "6730                                            upimapi            101\n",
       "6731                                              uropa          10470\n",
       "6732                                           ushuffle           4301\n",
       "6733                                                uvp            101\n",
       "6734                                              valet           2134\n",
       "6735                            validate-fasta-database           2218\n",
       "6736                                               vamb           2365\n",
       "6737                                            var-agg           2965\n",
       "6738                                      varda2-client            150\n",
       "6739                                            vardict          52342\n",
       "6740                                       vardict-java          42794\n",
       "6741                              varfish-annotator-cli            883\n",
       "6742                                            vargeno           1621\n",
       "6743                                         variantbam          14183\n",
       "6744                                       variantbreak             32\n",
       "6745                                      variant_tools           4576\n",
       "6746                                           varifier             22\n",
       "6747                                      varlociraptor          15567\n",
       "6748                                            varscan          42424\n",
       "6749                                               vawk           7412\n",
       "6750                                             vcf2db          29462\n",
       "6751                                         vcf2genome           4704\n",
       "6752                                            vcf2maf          29421\n",
       "6753                                       vcf2variants             63\n",
       "6754                                            vcfanno          36457\n",
       "6755                                      vcf-annotator           1890\n",
       "6756                                             vcfkit           3874\n",
       "6757                                          vcflatten           1801\n",
       "6758                                             vcflib          36923\n",
       "6759                                              vcfpy           6497\n",
       "6760                                   vcfsamplecompare           3013\n",
       "6761                                         vcftoolbox           2177\n",
       "6762                                           vcftools          88217\n",
       "6763                                          vcontact2            531\n",
       "6764                                              vdjer           1781\n",
       "6765                                        velocyto.py           3123\n",
       "6766                                             velvet          19866\n",
       "6767                                          velvet-sc           1854\n",
       "6768                                           vembrane            271\n",
       "6769                                       verifybamid2           8156\n",
       "6770                                        verifybamid           7200\n",
       "6771                                              verse           1925\n",
       "6772                                                 vg           2559\n",
       "6773                                            vibrant            642\n",
       "6774                                          viennarna          92753\n",
       "6775                                            virchip             32\n",
       "6776                                             virema           1688\n",
       "6777                                             virmet           6572\n",
       "6778                                          virsorter           2904\n",
       "6779                      visceral-evaluatesegmentation           3559\n",
       "6780                                              vispr          18940\n",
       "6781                                               vkmz            490\n",
       "6782                                             vmatch           1319\n",
       "6783                                                vnl           1563\n",
       "6784                                           vphaser2          12731\n",
       "6785                                              vpolo            682\n",
       "6786                                           vqsr_cnn           4579\n",
       "6787                                            vsearch         162465\n",
       "6788                                               vsnp            233\n",
       "6789                                                 vt          36218\n",
       "6790                                             vtools           2061\n",
       "6791                                     w4mclassfilter          16151\n",
       "6792                                   w4mclstrpeakpics           2062\n",
       "6793                                               wade            460\n",
       "6794                                       watchdog-wms           8702\n",
       "6795                                             wbuild            324\n",
       "6796                                            wdltool           6284\n",
       "6797                                            weblogo          12163\n",
       "6798                                             weeder           5984\n",
       "6799                                 wes-service-client            275\n",
       "6800                                           wg-blimp            706\n",
       "6801                                             wgfast           1853\n",
       "6802                                           wgs2ncbi           2452\n",
       "6803                                      wgs-assembler           1785\n",
       "6804                                              wgsim           3606\n",
       "6805                                               wham          13584\n",
       "6806                                           whatsgnu            136\n",
       "6807                                           whatshap          28093\n",
       "6808                                    wheezy.template           2098\n",
       "6809                                        wiggletools           7983\n",
       "6810                                              wise2           1185\n",
       "6811                                        wisecondorx          36190\n",
       "6812                                          wisestork            101\n",
       "6813                                           wlogdate             61\n",
       "6814                                            womtool           3600\n",
       "6815                                          workspace           2617\n",
       "6816                                              wtdbg           9156\n",
       "6817                                    wtforms-alchemy           5509\n",
       "6818                                 wtforms-components           2454\n",
       "6819                                                wub           4442\n",
       "6820                                             xatlas           3482\n",
       "6821                                              xbioc            190\n",
       "6822                                           xcftools           1702\n",
       "6823                                               xhmm           1743\n",
       "6824                                         xmatchview            309\n",
       "6825                                         xmlbuilder           2402\n",
       "6826                                          xmltramp2          14139\n",
       "6827                                              xpclr            543\n",
       "6828                                         xpressplot            657\n",
       "6829                                                xsd           9896\n",
       "6830                                             xs-sim            884\n",
       "6831                                              xtail           2022\n",
       "6832                                            xtandem          10579\n",
       "6833                                         xtermcolor           2127\n",
       "6834                                      xunit-wrapper           2105\n",
       "6835                                            xxmotif           4852\n",
       "6836                                            xyalign           5143\n",
       "6837                                              yacrd          10903\n",
       "6838                                              yaggo           5213\n",
       "6839                                               yaha           1911\n",
       "6840                                              yahmm           3199\n",
       "6841                                                yak            191\n",
       "6842                                              yamda             38\n",
       "6843                                           yanagiba           3616\n",
       "6844                                               yanc           2299\n",
       "6845                                               yapc             44\n",
       "6846                                               yara           7599\n",
       "6847                                               yass           1954\n",
       "6848                                                ymp            110\n",
       "6849                                             zagros           1804\n",
       "6850                                             zerone           1898\n",
       "6851                                                zga             22\n",
       "6852                                               zifa           2052\n",
       "6853                                              zorro           1305"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "usage=pd.read_csv('clean.txt', names=['module','usage_2019_01'],dtype={'module': 'string','usage_2019_01':'int'},delimiter='\\s+',header=None)\n",
    "time.sleep(2) #don't rush\n",
    "usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>usage_2019_01</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>module</th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>samtools</th>\n",
       "      <td>1836974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pysam</th>\n",
       "      <td>1415206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-makemaker</th>\n",
       "      <td>330596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-carp</th>\n",
       "      <td>316240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-exporter</th>\n",
       "      <td>311641</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-scalar-list-utils</th>\n",
       "      <td>300579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xsloader</th>\n",
       "      <td>277632</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>star</th>\n",
       "      <td>267257</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>picard</th>\n",
       "      <td>264985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-compress-raw-zlib</th>\n",
       "      <td>262613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snakemake</th>\n",
       "      <td>261989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-list-moreutils</th>\n",
       "      <td>260626</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-exporter-tiny</th>\n",
       "      <td>259127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-app-cpanminus</th>\n",
       "      <td>256642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pathtools</th>\n",
       "      <td>252174</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-dumper</th>\n",
       "      <td>236600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-parent</th>\n",
       "      <td>230774</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-compress-raw-bzip2</th>\n",
       "      <td>229972</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-compress</th>\n",
       "      <td>229748</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pybedtools</th>\n",
       "      <td>225121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-uri</th>\n",
       "      <td>223700</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-threaded</th>\n",
       "      <td>222698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-encode</th>\n",
       "      <td>220236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-path</th>\n",
       "      <td>220225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-constant</th>\n",
       "      <td>220149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-list-moreutils-xs</th>\n",
       "      <td>219721</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-temp</th>\n",
       "      <td>209533</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-sax</th>\n",
       "      <td>209529</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-namespacesupport</th>\n",
       "      <td>205411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-sax-base</th>\n",
       "      <td>205378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-parser</th>\n",
       "      <td>204107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-net-ssleay</th>\n",
       "      <td>203568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sambamba</th>\n",
       "      <td>202487</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-json-xs</th>\n",
       "      <td>201633</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-tagset</th>\n",
       "      <td>201341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pybigwig</th>\n",
       "      <td>198714</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-encode-locale</th>\n",
       "      <td>198252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-simple</th>\n",
       "      <td>195056</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-common-sense</th>\n",
       "      <td>194862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-socket-ssl</th>\n",
       "      <td>194780</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mime-base64</th>\n",
       "      <td>194696</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-lwp-mediatypes</th>\n",
       "      <td>193795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-libwww-perl</th>\n",
       "      <td>193203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-json</th>\n",
       "      <td>192951</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-sax-expat</th>\n",
       "      <td>192576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-net-http</th>\n",
       "      <td>191945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-digest-hmac</th>\n",
       "      <td>190957</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-http-date</th>\n",
       "      <td>190549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-html</th>\n",
       "      <td>190396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-http-message</th>\n",
       "      <td>190192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-http-cookies</th>\n",
       "      <td>189959</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ntlm</th>\n",
       "      <td>189800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-http-negotiate</th>\n",
       "      <td>189739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-www-robotrules</th>\n",
       "      <td>188830</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-listing</th>\n",
       "      <td>187767</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-http-daemon</th>\n",
       "      <td>187757</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-tree</th>\n",
       "      <td>187437</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pubchempy</th>\n",
       "      <td>187185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>multiqc</th>\n",
       "      <td>187074</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-try-tiny</th>\n",
       "      <td>185653</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-lwp-protocol-https</th>\n",
       "      <td>184087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-archive-tar</th>\n",
       "      <td>183935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-business-isbn</th>\n",
       "      <td>180995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-business-isbn-data</th>\n",
       "      <td>180987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-zlib</th>\n",
       "      <td>177731</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-types-serialiser</th>\n",
       "      <td>176717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mafft</th>\n",
       "      <td>174314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-storable</th>\n",
       "      <td>173738</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mozilla-ca</th>\n",
       "      <td>170070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-digest-md5</th>\n",
       "      <td>167946</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-time-local</th>\n",
       "      <td>166078</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-socket</th>\n",
       "      <td>164299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-requiresinternet</th>\n",
       "      <td>163206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vsearch</th>\n",
       "      <td>162465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-yaml</th>\n",
       "      <td>151011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-gd</th>\n",
       "      <td>147090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyfaidx</th>\n",
       "      <td>146823</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-dbi</th>\n",
       "      <td>136618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>subread</th>\n",
       "      <td>132027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bioperl</th>\n",
       "      <td>127487</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-getopt-long</th>\n",
       "      <td>123163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-parser</th>\n",
       "      <td>122792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quast</th>\n",
       "      <td>121985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cgi</th>\n",
       "      <td>121797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-libxml</th>\n",
       "      <td>121576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaphlan2</th>\n",
       "      <td>119507</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-runtime</th>\n",
       "      <td>119313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minimap2</th>\n",
       "      <td>118765</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>salmon</th>\n",
       "      <td>116310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-which</th>\n",
       "      <td>114116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-clone</th>\n",
       "      <td>114111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-db-file</th>\n",
       "      <td>113622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>raxml</th>\n",
       "      <td>112386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-string</th>\n",
       "      <td>110358</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-time-hires</th>\n",
       "      <td>109294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trimmomatic</th>\n",
       "      <td>108735</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sortmerna</th>\n",
       "      <td>107451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-svg</th>\n",
       "      <td>107203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-exporter-progressive</th>\n",
       "      <td>104944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moo</th>\n",
       "      <td>104481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-dbd-sqlite</th>\n",
       "      <td>104450</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-lwp-simple</th>\n",
       "      <td>104368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-digest-perl-md5</th>\n",
       "      <td>104292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-role-tiny</th>\n",
       "      <td>104105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-globaldestruction</th>\n",
       "      <td>103763</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-cdf</th>\n",
       "      <td>103084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-implementation</th>\n",
       "      <td>102398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-method-modifiers</th>\n",
       "      <td>101876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-statistics-descriptive</th>\n",
       "      <td>101117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tabix</th>\n",
       "      <td>101027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bioperl-core</th>\n",
       "      <td>100182</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pdf-api2</th>\n",
       "      <td>99799</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-stacktrace</th>\n",
       "      <td>99551</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-font-ttf</th>\n",
       "      <td>99495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-capture-tiny</th>\n",
       "      <td>97832</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-type-tiny</th>\n",
       "      <td>97815</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-inspector</th>\n",
       "      <td>97153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spades</th>\n",
       "      <td>97089</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-twig</th>\n",
       "      <td>95408</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ipc-run</th>\n",
       "      <td>95251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-leaktrace</th>\n",
       "      <td>95230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mailtools</th>\n",
       "      <td>94965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sra-tools</th>\n",
       "      <td>94909</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-parse-recdescent</th>\n",
       "      <td>94668</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-params-util</th>\n",
       "      <td>94560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-autoloader</th>\n",
       "      <td>94532</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moose</th>\n",
       "      <td>94442</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-asn1-entrezgene</th>\n",
       "      <td>94397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mime-tools</th>\n",
       "      <td>94390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-array-compare</th>\n",
       "      <td>94236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-error</th>\n",
       "      <td>94005</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-formatter</th>\n",
       "      <td>93783</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-task-weaken</th>\n",
       "      <td>93722</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-tidy</th>\n",
       "      <td>93685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-graphviz</th>\n",
       "      <td>93671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-phylo</th>\n",
       "      <td>93600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-derivative</th>\n",
       "      <td>93593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tree-dag_node</th>\n",
       "      <td>93585</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-xpath</th>\n",
       "      <td>93518</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-slurp-tiny</th>\n",
       "      <td>93455</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-sax-writer</th>\n",
       "      <td>93440</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-graph</th>\n",
       "      <td>93377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-libxml-perl</th>\n",
       "      <td>93352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mldbm</th>\n",
       "      <td>93352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-libxslt</th>\n",
       "      <td>93334</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-dom</th>\n",
       "      <td>93322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mime-types</th>\n",
       "      <td>93251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-tty</th>\n",
       "      <td>93230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-dynaloader</th>\n",
       "      <td>93137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-stag</th>\n",
       "      <td>93048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-random</th>\n",
       "      <td>92981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-unicode-map</th>\n",
       "      <td>92909</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-convert-binary-c</th>\n",
       "      <td>92837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>viennarna</th>\n",
       "      <td>92753</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ole-storage_lite</th>\n",
       "      <td>92678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bioperl-run</th>\n",
       "      <td>92658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-svg-graph</th>\n",
       "      <td>92603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sort-naturally</th>\n",
       "      <td>92589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-install</th>\n",
       "      <td>92496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-email-date-format</th>\n",
       "      <td>92325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-ixhash</th>\n",
       "      <td>92287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-optlist</th>\n",
       "      <td>92281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-jcode</th>\n",
       "      <td>92240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-spreadsheet-writeexcel</th>\n",
       "      <td>92232</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqtk</th>\n",
       "      <td>92153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-package-stash</th>\n",
       "      <td>92054</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-xpathengine</th>\n",
       "      <td>92050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-font-afm</th>\n",
       "      <td>91898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tidyp</th>\n",
       "      <td>91896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-stringy</th>\n",
       "      <td>91783</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-entities-numbered</th>\n",
       "      <td>91765</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-regexp</th>\n",
       "      <td>91679</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-convert-binhex</th>\n",
       "      <td>91670</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-apache-test</th>\n",
       "      <td>91574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-timedate</th>\n",
       "      <td>91562</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-sessiondata</th>\n",
       "      <td>91486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-writer</th>\n",
       "      <td>91377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-image-size</th>\n",
       "      <td>91277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-template-toolkit</th>\n",
       "      <td>91274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mime-lite</th>\n",
       "      <td>91243</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-filter-buffertext</th>\n",
       "      <td>91224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-crypt-rc4</th>\n",
       "      <td>91188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-algorithm-munkres</th>\n",
       "      <td>91185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-image-info</th>\n",
       "      <td>91184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-appconfig</th>\n",
       "      <td>91165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-soap-lite</th>\n",
       "      <td>91141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-identify</th>\n",
       "      <td>91132</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-treebuilder-xpath</th>\n",
       "      <td>91107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-postscript</th>\n",
       "      <td>91100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-set-scalar</th>\n",
       "      <td>91067</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-aceperl</th>\n",
       "      <td>91047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-spreadsheet-parseexcel</th>\n",
       "      <td>91046</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-dom-xpath</th>\n",
       "      <td>91013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-tableextract</th>\n",
       "      <td>91010</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-element-extended</th>\n",
       "      <td>91000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-spline</th>\n",
       "      <td>90994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-exporter</th>\n",
       "      <td>90883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-eval-closure</th>\n",
       "      <td>90096</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nextflow</th>\n",
       "      <td>89664</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-load-xs</th>\n",
       "      <td>89272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-featureio</th>\n",
       "      <td>89260</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-dist-checkconflicts</th>\n",
       "      <td>88864</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-harness</th>\n",
       "      <td>88682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-samtools</th>\n",
       "      <td>88294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcftools</th>\n",
       "      <td>88217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-sort</th>\n",
       "      <td>88187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-load</th>\n",
       "      <td>87930</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snpeff</th>\n",
       "      <td>87645</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-package-stash-xs</th>\n",
       "      <td>87537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-uplevel</th>\n",
       "      <td>86539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-name</th>\n",
       "      <td>86227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-exception-class</th>\n",
       "      <td>85919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-quote</th>\n",
       "      <td>85871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mro-compat</th>\n",
       "      <td>85682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-runtime-conflicts</th>\n",
       "      <td>85671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-overloadinfo</th>\n",
       "      <td>85669</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-package-deprecationmanager</th>\n",
       "      <td>85656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-warn</th>\n",
       "      <td>85125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-diff</th>\n",
       "      <td>84023</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-data-inheritable</th>\n",
       "      <td>83971</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-most</th>\n",
       "      <td>83449</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-lib</th>\n",
       "      <td>83391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-differences</th>\n",
       "      <td>83164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test</th>\n",
       "      <td>83033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-escapes</th>\n",
       "      <td>83011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-usage</th>\n",
       "      <td>82784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-exception</th>\n",
       "      <td>82724</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-algorithm-diff</th>\n",
       "      <td>82117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ipc-sharelite</th>\n",
       "      <td>81362</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-digest-sha1</th>\n",
       "      <td>81192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>t_coffee</th>\n",
       "      <td>81184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqkit</th>\n",
       "      <td>80431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>paml</th>\n",
       "      <td>80195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cache-cache</th>\n",
       "      <td>79812</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-date-format</th>\n",
       "      <td>79203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-tools-phylo-paml</th>\n",
       "      <td>75262</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-coordinate</th>\n",
       "      <td>75221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-tools-run-alignment-clustalw</th>\n",
       "      <td>75208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qualimap</th>\n",
       "      <td>74370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbgzip</th>\n",
       "      <td>73784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>muscle</th>\n",
       "      <td>73723</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perlio-gzip</th>\n",
       "      <td>72763</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedgraphtobigwig</th>\n",
       "      <td>72277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-tools-run-alignment-tcoffee</th>\n",
       "      <td>72117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rseqc</th>\n",
       "      <td>71908</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pytabix</th>\n",
       "      <td>71032</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prodigal</th>\n",
       "      <td>70937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-corelist</th>\n",
       "      <td>70935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-deep</th>\n",
       "      <td>70414</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>py2bit</th>\n",
       "      <td>67566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sina</th>\n",
       "      <td>67310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-metadata</th>\n",
       "      <td>65483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stringtie</th>\n",
       "      <td>63535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-load-conditional</th>\n",
       "      <td>63428</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ipc-cmd</th>\n",
       "      <td>63204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-load</th>\n",
       "      <td>62370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-locale-maketext-simple</th>\n",
       "      <td>62273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-version</th>\n",
       "      <td>62151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-params-check</th>\n",
       "      <td>62114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trim-galore</th>\n",
       "      <td>61752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>htslib</th>\n",
       "      <td>61473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sickle-trim</th>\n",
       "      <td>59616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trinity</th>\n",
       "      <td>58666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-build</th>\n",
       "      <td>56819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-set-intervaltree</th>\n",
       "      <td>54140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-wigtobigwig</th>\n",
       "      <td>53810</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>racon</th>\n",
       "      <td>52675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unifrac</th>\n",
       "      <td>52573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-gdgraph</th>\n",
       "      <td>52384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vardict</th>\n",
       "      <td>52342</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pairix</th>\n",
       "      <td>51497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-vcftools-vcf</th>\n",
       "      <td>50943</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-gdtextutil</th>\n",
       "      <td>50657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>macs2</th>\n",
       "      <td>49852</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pindel</th>\n",
       "      <td>48830</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fatotwobit</th>\n",
       "      <td>48589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tbl2asn</th>\n",
       "      <td>47855</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-csv</th>\n",
       "      <td>47770</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prokka</th>\n",
       "      <td>47219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>multiqc-bcbio</th>\n",
       "      <td>47178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qiime</th>\n",
       "      <td>46811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>meme</th>\n",
       "      <td>46318</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-dbd-mysql</th>\n",
       "      <td>46038</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-db-hts</th>\n",
       "      <td>45855</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sourmash</th>\n",
       "      <td>45667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>peptide-shaker</th>\n",
       "      <td>45590</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>planemo</th>\n",
       "      <td>44322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcftools</th>\n",
       "      <td>44152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mappy</th>\n",
       "      <td>44044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>manta</th>\n",
       "      <td>43748</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sentieon</th>\n",
       "      <td>43429</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-number-format</th>\n",
       "      <td>43136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>megahit</th>\n",
       "      <td>43017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vardict-java</th>\n",
       "      <td>42794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>varscan</th>\n",
       "      <td>42424</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-manifest</th>\n",
       "      <td>42384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mosdepth</th>\n",
       "      <td>42114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cpan-meta</th>\n",
       "      <td>41828</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-parsewords</th>\n",
       "      <td>41822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-json-pp</th>\n",
       "      <td>40737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-parsexs</th>\n",
       "      <td>40488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqcluster</th>\n",
       "      <td>40295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sereal-decoder</th>\n",
       "      <td>39884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sereal-encoder</th>\n",
       "      <td>39795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perl-ostype</th>\n",
       "      <td>39683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stacks</th>\n",
       "      <td>39603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-liftover</th>\n",
       "      <td>39204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-cbuilder</th>\n",
       "      <td>38990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sereal</th>\n",
       "      <td>38743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mummer</th>\n",
       "      <td>38688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-gtftogenepred</th>\n",
       "      <td>38615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-abbrev</th>\n",
       "      <td>38557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>umi_tools</th>\n",
       "      <td>38413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncls</th>\n",
       "      <td>37462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cpan-meta-requirements</th>\n",
       "      <td>37325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcflib</th>\n",
       "      <td>36923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>miniasm</th>\n",
       "      <td>36543</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcfanno</th>\n",
       "      <td>36457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vt</th>\n",
       "      <td>36218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wisecondorx</th>\n",
       "      <td>36190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanopolish</th>\n",
       "      <td>36116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncbi-ngs-sdk</th>\n",
       "      <td>35929</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-sharedir-install</th>\n",
       "      <td>35651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyranges</th>\n",
       "      <td>35568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedtobigbed</th>\n",
       "      <td>35333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cpan-meta-yaml</th>\n",
       "      <td>35229</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minced</th>\n",
       "      <td>35151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sorted_nearest</th>\n",
       "      <td>35031</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sepp</th>\n",
       "      <td>34758</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>umis</th>\n",
       "      <td>34432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-archive-zip</th>\n",
       "      <td>34402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>searchgui</th>\n",
       "      <td>34090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scalpel</th>\n",
       "      <td>33981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tophat</th>\n",
       "      <td>33640</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-params-validate</th>\n",
       "      <td>33495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-statistics-basic</th>\n",
       "      <td>32796</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mash</th>\n",
       "      <td>32784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-regexp-common</th>\n",
       "      <td>32725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sevenbridges-python</th>\n",
       "      <td>32373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-sharedir</th>\n",
       "      <td>32273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>samblaster</th>\n",
       "      <td>32239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rsem</th>\n",
       "      <td>32054</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-copy-recursive</th>\n",
       "      <td>31912</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qcli</th>\n",
       "      <td>31867</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pb-falcon</th>\n",
       "      <td>31787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-round</th>\n",
       "      <td>31721</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stream</th>\n",
       "      <td>31573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rust-bio-tools</th>\n",
       "      <td>31528</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-readonly</th>\n",
       "      <td>31496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>peddy</th>\n",
       "      <td>31278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mageck</th>\n",
       "      <td>31030</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-archive-extract</th>\n",
       "      <td>30932</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pynast</th>\n",
       "      <td>30800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippy</th>\n",
       "      <td>30762</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-config-general</th>\n",
       "      <td>30726</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanoplot</th>\n",
       "      <td>30660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rtg-tools</th>\n",
       "      <td>30573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mcl</th>\n",
       "      <td>30556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-fetch</th>\n",
       "      <td>30484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>star-fusion</th>\n",
       "      <td>30292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ipc-system-simple</th>\n",
       "      <td>29888</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>medaka</th>\n",
       "      <td>29619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>novoalign</th>\n",
       "      <td>29615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanoplotter</th>\n",
       "      <td>29508</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcf2db</th>\n",
       "      <td>29462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcf2maf</th>\n",
       "      <td>29421</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>toil</th>\n",
       "      <td>29386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pilon</th>\n",
       "      <td>29346</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-format</th>\n",
       "      <td>29286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanoget</th>\n",
       "      <td>29192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-vecstat</th>\n",
       "      <td>29049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-bezier</th>\n",
       "      <td>28922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-set-intspan</th>\n",
       "      <td>28823</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyrle</th>\n",
       "      <td>28693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>whatshap</th>\n",
       "      <td>28093</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-gdgraph-histogram</th>\n",
       "      <td>27932</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mothur</th>\n",
       "      <td>27931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mygene</th>\n",
       "      <td>27902</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwigsummary</th>\n",
       "      <td>27755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mlst</th>\n",
       "      <td>27672</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-twobittofa</th>\n",
       "      <td>27327</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minialign</th>\n",
       "      <td>27269</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>preseq</th>\n",
       "      <td>26951</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snpsift</th>\n",
       "      <td>26730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qiime-default-reference</th>\n",
       "      <td>26606</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqan</th>\n",
       "      <td>26220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pygenometracks</th>\n",
       "      <td>25683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scanpy</th>\n",
       "      <td>25595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-log-log4perl</th>\n",
       "      <td>25395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanomath</th>\n",
       "      <td>25320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mirtop</th>\n",
       "      <td>25084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>simple_sv_annotation</th>\n",
       "      <td>24941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredtobed</th>\n",
       "      <td>24900</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prophyle</th>\n",
       "      <td>24637</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sequana</th>\n",
       "      <td>24584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bedtools</th>\n",
       "      <td>24168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transdecoder</th>\n",
       "      <td>23937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mmseqs2</th>\n",
       "      <td>23434</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyteomics</th>\n",
       "      <td>23409</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quasitools</th>\n",
       "      <td>23372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbmm2</th>\n",
       "      <td>23306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rgi</th>\n",
       "      <td>22986</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mentalist</th>\n",
       "      <td>22971</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test2-suite</th>\n",
       "      <td>22953</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncbi-genome-download</th>\n",
       "      <td>22755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-parallel-forkmanager</th>\n",
       "      <td>22749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lumpy-sv</th>\n",
       "      <td>22513</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-template</th>\n",
       "      <td>22164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ont-tombo</th>\n",
       "      <td>22094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbbam</th>\n",
       "      <td>21989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-hpc-runner-command</th>\n",
       "      <td>21989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngless</th>\n",
       "      <td>21918</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mashtree</th>\n",
       "      <td>21657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>strelka</th>\n",
       "      <td>21613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maker</th>\n",
       "      <td>21204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwigtobedgraph</th>\n",
       "      <td>20848</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sleuth</th>\n",
       "      <td>20440</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>roary</th>\n",
       "      <td>20426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>orthofinder</th>\n",
       "      <td>20333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwiginfo</th>\n",
       "      <td>20321</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msstitch</th>\n",
       "      <td>20195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-edlib</th>\n",
       "      <td>20187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-simple</th>\n",
       "      <td>20050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>toolshed</th>\n",
       "      <td>20049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-importer</th>\n",
       "      <td>20006</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigbedtobed</th>\n",
       "      <td>19934</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unicycler</th>\n",
       "      <td>19892</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>velvet</th>\n",
       "      <td>19866</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svtyper</th>\n",
       "      <td>19689</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>razers3</th>\n",
       "      <td>19593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>proteinortho</th>\n",
       "      <td>19577</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-glob</th>\n",
       "      <td>19464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngs-disambiguate</th>\n",
       "      <td>19377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>repeatmasker</th>\n",
       "      <td>19287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msisensor</th>\n",
       "      <td>19094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanofilt</th>\n",
       "      <td>19048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vispr</th>\n",
       "      <td>18940</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-pscbs</th>\n",
       "      <td>18787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trf</th>\n",
       "      <td>18658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minvar</th>\n",
       "      <td>18552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pysamstats</th>\n",
       "      <td>18500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trimal</th>\n",
       "      <td>18495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-gff3togenepred</th>\n",
       "      <td>18400</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>optitype</th>\n",
       "      <td>18313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prinseq</th>\n",
       "      <td>18170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-spp</th>\n",
       "      <td>18131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-pod</th>\n",
       "      <td>18121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>srprism</th>\n",
       "      <td>17983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanocomp</th>\n",
       "      <td>17974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>samplot</th>\n",
       "      <td>17946</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-wasabi</th>\n",
       "      <td>17869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredtogtf</th>\n",
       "      <td>17857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>parallel-fastq-dump</th>\n",
       "      <td>17800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>parasail-python</th>\n",
       "      <td>17766</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-cppguess</th>\n",
       "      <td>17689</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mapcaller</th>\n",
       "      <td>17608</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>popdel</th>\n",
       "      <td>17489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>secapr</th>\n",
       "      <td>17312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mummer4</th>\n",
       "      <td>17301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msgf_plus</th>\n",
       "      <td>17142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svtools</th>\n",
       "      <td>17094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-term-table</th>\n",
       "      <td>17085</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trnascan-se</th>\n",
       "      <td>17057</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tophat-recondition</th>\n",
       "      <td>17041</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-find-rule</th>\n",
       "      <td>16942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fetchchromsizes</th>\n",
       "      <td>16862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-number-compare</th>\n",
       "      <td>16392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-basejump</th>\n",
       "      <td>16258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqbuster</th>\n",
       "      <td>16181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwigtowig</th>\n",
       "      <td>16153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>w4mclassfilter</th>\n",
       "      <td>16151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smoove</th>\n",
       "      <td>16139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rmblast</th>\n",
       "      <td>16134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pymummer</th>\n",
       "      <td>16117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedtogenepred</th>\n",
       "      <td>16092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ruffus</th>\n",
       "      <td>16028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwigaverageoverbed</th>\n",
       "      <td>16017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rabix-bunny</th>\n",
       "      <td>15977</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-scope-guard</th>\n",
       "      <td>15822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snp-sites</th>\n",
       "      <td>15811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mqc</th>\n",
       "      <td>15800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-pluggable</th>\n",
       "      <td>15663</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>octopus</th>\n",
       "      <td>15638</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-twobitinfo</th>\n",
       "      <td>15618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>varlociraptor</th>\n",
       "      <td>15567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-info</th>\n",
       "      <td>15343</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngs-bits</th>\n",
       "      <td>15231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pb-assembly</th>\n",
       "      <td>15176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>slclust</th>\n",
       "      <td>15084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stringmlst</th>\n",
       "      <td>15028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bwa</th>\n",
       "      <td>14995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pplacer</th>\n",
       "      <td>14967</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sailfish</th>\n",
       "      <td>14884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>picrust</th>\n",
       "      <td>14723</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-bcbiornaseq</th>\n",
       "      <td>14616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plink</th>\n",
       "      <td>14533</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pauvre</th>\n",
       "      <td>14450</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rapid</th>\n",
       "      <td>14449</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quicksect</th>\n",
       "      <td>14408</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prank</th>\n",
       "      <td>14366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phyml</th>\n",
       "      <td>14348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedsort</th>\n",
       "      <td>14341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigbedinfo</th>\n",
       "      <td>14280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>moabs</th>\n",
       "      <td>14268</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cutadapt</th>\n",
       "      <td>14265</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastqc</th>\n",
       "      <td>14258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedclip</th>\n",
       "      <td>14210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shovill</th>\n",
       "      <td>14189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>variantbam</th>\n",
       "      <td>14183</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>synapseclient</th>\n",
       "      <td>14180</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-tripal</th>\n",
       "      <td>14176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xmltramp2</th>\n",
       "      <td>14139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-bcbiobase</th>\n",
       "      <td>14046</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-wgcna</th>\n",
       "      <td>14044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-slurp</th>\n",
       "      <td>13981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iranges</th>\n",
       "      <td>13965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>staramr</th>\n",
       "      <td>13956</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>masurca</th>\n",
       "      <td>13922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>riboseed</th>\n",
       "      <td>13897</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sniffles</th>\n",
       "      <td>13847</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocgenerics</th>\n",
       "      <td>13832</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigbedsummary</th>\n",
       "      <td>13821</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-s4vectors</th>\n",
       "      <td>13820</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rapmap</th>\n",
       "      <td>13737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanostat</th>\n",
       "      <td>13733</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>merfishtools</th>\n",
       "      <td>13680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>parafly</th>\n",
       "      <td>13643</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>selene-sdk</th>\n",
       "      <td>13595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wham</th>\n",
       "      <td>13584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>primer3</th>\n",
       "      <td>13576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyani</th>\n",
       "      <td>13422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>swarm</th>\n",
       "      <td>13352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mageck-vispr</th>\n",
       "      <td>13335</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyhashxx</th>\n",
       "      <td>13317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-array-utils</th>\n",
       "      <td>13317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>survivor</th>\n",
       "      <td>13287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taxonkit</th>\n",
       "      <td>13187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbcore</th>\n",
       "      <td>13175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngmlr</th>\n",
       "      <td>13147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biobase</th>\n",
       "      <td>12941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomicranges</th>\n",
       "      <td>12857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>methyldackel</th>\n",
       "      <td>12831</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-zlibbioc</th>\n",
       "      <td>12826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rscape</th>\n",
       "      <td>12824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pear</th>\n",
       "      <td>12803</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sartools</th>\n",
       "      <td>12797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seq2c</th>\n",
       "      <td>12771</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perlio-utf8_strict</th>\n",
       "      <td>12756</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stream_atac</th>\n",
       "      <td>12740</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vphaser2</th>\n",
       "      <td>12731</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metabat2</th>\n",
       "      <td>12728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbcopper</th>\n",
       "      <td>12710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mirdeep2</th>\n",
       "      <td>12705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>thermorawfileparser</th>\n",
       "      <td>12636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shorah</th>\n",
       "      <td>12621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pureclip</th>\n",
       "      <td>12591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xvector</th>\n",
       "      <td>12556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tardis</th>\n",
       "      <td>12546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>percolator</th>\n",
       "      <td>12501</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-nibfrag</th>\n",
       "      <td>12491</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>samsift</th>\n",
       "      <td>12485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocparallel</th>\n",
       "      <td>12476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-gzip</th>\n",
       "      <td>12452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-slurper</th>\n",
       "      <td>12398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>platypus-variant</th>\n",
       "      <td>12299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>picopore</th>\n",
       "      <td>12268</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-consensuscore</th>\n",
       "      <td>12253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyfastaq</th>\n",
       "      <td>12246</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomeinfodb</th>\n",
       "      <td>12210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncbi-amrfinderplus</th>\n",
       "      <td>12208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>reparation_blast</th>\n",
       "      <td>12179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>weblogo</th>\n",
       "      <td>12163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pypeflow</th>\n",
       "      <td>12069</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-parse-yapp</th>\n",
       "      <td>12019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>skewer</th>\n",
       "      <td>11993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslcdnafilter</th>\n",
       "      <td>11975</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbsv</th>\n",
       "      <td>11940</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mvicuna</th>\n",
       "      <td>11906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bowtie</th>\n",
       "      <td>11858</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tn93</th>\n",
       "      <td>11851</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tracy</th>\n",
       "      <td>11815</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanoqc</th>\n",
       "      <td>11656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-summarizedexperiment</th>\n",
       "      <td>11621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbccs</th>\n",
       "      <td>11619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mirge</th>\n",
       "      <td>11618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-hemdag</th>\n",
       "      <td>11575</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyliftover</th>\n",
       "      <td>11520</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomeinfodbdata</th>\n",
       "      <td>11516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biostrings</th>\n",
       "      <td>11390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncbi-vdb</th>\n",
       "      <td>11333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanocall</th>\n",
       "      <td>11316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sistr_cmd</th>\n",
       "      <td>11307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-filesys-df</th>\n",
       "      <td>11290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-spec</th>\n",
       "      <td>11280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-findbin</th>\n",
       "      <td>11258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snap-aligner</th>\n",
       "      <td>11222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maxbin2</th>\n",
       "      <td>11184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phyluce</th>\n",
       "      <td>11135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-delayedarray</th>\n",
       "      <td>11131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nonpareil</th>\n",
       "      <td>11092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rna-seqc</th>\n",
       "      <td>11028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-biodb</th>\n",
       "      <td>10998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>slivar</th>\n",
       "      <td>10960</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-oligomatch</th>\n",
       "      <td>10938</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-termreadkey</th>\n",
       "      <td>10924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yacrd</th>\n",
       "      <td>10903</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nim-falcon</th>\n",
       "      <td>10824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snp-dists</th>\n",
       "      <td>10769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-grep</th>\n",
       "      <td>10755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbcommand</th>\n",
       "      <td>10742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>segway</th>\n",
       "      <td>10739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomicalignments</th>\n",
       "      <td>10675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mikado</th>\n",
       "      <td>10671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnalien</th>\n",
       "      <td>10660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pymzml</th>\n",
       "      <td>10626</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spaln</th>\n",
       "      <td>10605</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xtandem</th>\n",
       "      <td>10579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>poretools</th>\n",
       "      <td>10546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rsamtools</th>\n",
       "      <td>10490</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>uropa</th>\n",
       "      <td>10470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-graph-readwrite</th>\n",
       "      <td>10414</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-hash-merge</th>\n",
       "      <td>10380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>primer3-py</th>\n",
       "      <td>10369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-path-tiny</th>\n",
       "      <td>10366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hggcpercent</th>\n",
       "      <td>10363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslrecalcmatch</th>\n",
       "      <td>10357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslsort</th>\n",
       "      <td>10227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plant_tribes_gene_family_classifier</th>\n",
       "      <td>10177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dnaio</th>\n",
       "      <td>10175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-chado</th>\n",
       "      <td>10172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ont-fast5-api</th>\n",
       "      <td>10166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>samclip</th>\n",
       "      <td>10145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blast</th>\n",
       "      <td>10108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslcheck</th>\n",
       "      <td>10082</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwigmerge</th>\n",
       "      <td>10077</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-splitfilebycolumn</th>\n",
       "      <td>10028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylowgs</th>\n",
       "      <td>10000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-psltochain</th>\n",
       "      <td>9958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>moff</th>\n",
       "      <td>9928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pythonpy</th>\n",
       "      <td>9922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslpostarget</th>\n",
       "      <td>9912</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-wigcorrelate</th>\n",
       "      <td>9906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mgkit</th>\n",
       "      <td>9904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xsd</th>\n",
       "      <td>9896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-qactowig</th>\n",
       "      <td>9877</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-psltobigpsl</th>\n",
       "      <td>9857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-positionaltblcheck</th>\n",
       "      <td>9844</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-tolower</th>\n",
       "      <td>9840</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>skesa</th>\n",
       "      <td>9840</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslmrnacover</th>\n",
       "      <td>9836</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredtofakepsl</th>\n",
       "      <td>9824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-coloc</th>\n",
       "      <td>9818</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-digest-md5-file</th>\n",
       "      <td>9816</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-psltobed</th>\n",
       "      <td>9797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>portcullis</th>\n",
       "      <td>9791</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslpairs</th>\n",
       "      <td>9766</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trinotate</th>\n",
       "      <td>9766</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-tdbquery</th>\n",
       "      <td>9765</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>norns</th>\n",
       "      <td>9751</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-xmltosql</th>\n",
       "      <td>9724</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-psltopslx</th>\n",
       "      <td>9720</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-algorithm-cluster</th>\n",
       "      <td>9711</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-ticktodate</th>\n",
       "      <td>9702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-subchar</th>\n",
       "      <td>9700</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-splitfile</th>\n",
       "      <td>9695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnftools</th>\n",
       "      <td>9690</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svdb</th>\n",
       "      <td>9689</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslfilter</th>\n",
       "      <td>9681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-twobitmask</th>\n",
       "      <td>9676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslreps</th>\n",
       "      <td>9673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-xmlcat</th>\n",
       "      <td>9662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-rmfadups</th>\n",
       "      <td>9651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-sqltoxml</th>\n",
       "      <td>9639</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-qatoqac</th>\n",
       "      <td>9639</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslcat</th>\n",
       "      <td>9636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-qactoqa</th>\n",
       "      <td>9634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyfasta</th>\n",
       "      <td>9633</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslswap</th>\n",
       "      <td>9629</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-toupper</th>\n",
       "      <td>9624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-psldropoverlap</th>\n",
       "      <td>9623</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-parasync</th>\n",
       "      <td>9622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslpretty</th>\n",
       "      <td>9622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mob_suite</th>\n",
       "      <td>9622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>oncofuse</th>\n",
       "      <td>9614</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslpartition</th>\n",
       "      <td>9613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-parafetch</th>\n",
       "      <td>9613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslselect</th>\n",
       "      <td>9607</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-trfbig</th>\n",
       "      <td>9601</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sanger-cgp-allelecount</th>\n",
       "      <td>9599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredtomafframes</th>\n",
       "      <td>9594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tdrmapper</th>\n",
       "      <td>9594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>motus</th>\n",
       "      <td>9593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-validatefiles</th>\n",
       "      <td>9592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-nettobed</th>\n",
       "      <td>9592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-newpythonprog</th>\n",
       "      <td>9586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prosolo</th>\n",
       "      <td>9580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-taillines</th>\n",
       "      <td>9578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-randomlines</th>\n",
       "      <td>9578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pipits</th>\n",
       "      <td>9567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-nibsize</th>\n",
       "      <td>9563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-rowstocols</th>\n",
       "      <td>9560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-subcolumn</th>\n",
       "      <td>9545</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-ratolines</th>\n",
       "      <td>9535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-rasqlquery</th>\n",
       "      <td>9521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslstats</th>\n",
       "      <td>9520</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqlogo</th>\n",
       "      <td>9514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-validatemanifest</th>\n",
       "      <td>9514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trumicount</th>\n",
       "      <td>9512</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-stringify</th>\n",
       "      <td>9510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-twobitdup</th>\n",
       "      <td>9507</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslliftsubrangeblat</th>\n",
       "      <td>9504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-spacedtotab</th>\n",
       "      <td>9503</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-qacagplift</th>\n",
       "      <td>9493</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslxtofa</th>\n",
       "      <td>9489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslhisto</th>\n",
       "      <td>9485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-ratotab</th>\n",
       "      <td>9481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanosv</th>\n",
       "      <td>9471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslscore</th>\n",
       "      <td>9464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-texthistogram</th>\n",
       "      <td>9457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-dbd-pg</th>\n",
       "      <td>9457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-wordline</th>\n",
       "      <td>9426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ribocode</th>\n",
       "      <td>9401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredtobiggenepred</th>\n",
       "      <td>9386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sanger-cgp-vcf</th>\n",
       "      <td>9384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rmats</th>\n",
       "      <td>9355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pygtftk</th>\n",
       "      <td>9354</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanosim</th>\n",
       "      <td>9324</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredcheck</th>\n",
       "      <td>9301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mobster</th>\n",
       "      <td>9231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mira</th>\n",
       "      <td>9216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pepnovo</th>\n",
       "      <td>9187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pizzly</th>\n",
       "      <td>9161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wtdbg</th>\n",
       "      <td>9156</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanolyse</th>\n",
       "      <td>9098</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyloh</th>\n",
       "      <td>9095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fasplit</th>\n",
       "      <td>9048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-symdump</th>\n",
       "      <td>9039</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fasomerecords</th>\n",
       "      <td>9000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mykrobe</th>\n",
       "      <td>8996</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-uuid</th>\n",
       "      <td>8988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sqt</th>\n",
       "      <td>8981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pylibseq</th>\n",
       "      <td>8928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-chbutils</th>\n",
       "      <td>8897</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cyvcf2</th>\n",
       "      <td>8829</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>starcode</th>\n",
       "      <td>8817</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test2</th>\n",
       "      <td>8812</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanoraw</th>\n",
       "      <td>8755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>singlecell-barcodes</th>\n",
       "      <td>8705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>watchdog-wms</th>\n",
       "      <td>8702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>solvebio</th>\n",
       "      <td>8647</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-time-piece</th>\n",
       "      <td>8634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lz4-bin</th>\n",
       "      <td>8631</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tbl2asn-forever</th>\n",
       "      <td>8605</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pairtools</th>\n",
       "      <td>8578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslmap</th>\n",
       "      <td>8529</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>manorm</th>\n",
       "      <td>8489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainmergesort</th>\n",
       "      <td>8479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rcorrector</th>\n",
       "      <td>8461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-variable-magic</th>\n",
       "      <td>8456</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-db-sam</th>\n",
       "      <td>8449</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>porechop</th>\n",
       "      <td>8438</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bowtie2</th>\n",
       "      <td>8426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cpan-meta-check</th>\n",
       "      <td>8419</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>moreutils</th>\n",
       "      <td>8400</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tobias</th>\n",
       "      <td>8379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-msgpack</th>\n",
       "      <td>8375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pcap</th>\n",
       "      <td>8356</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-b-hooks-endofscope</th>\n",
       "      <td>8354</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-namespace-clean</th>\n",
       "      <td>8345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fasize</th>\n",
       "      <td>8342</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qsignature</th>\n",
       "      <td>8331</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainnet</th>\n",
       "      <td>8325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metasv</th>\n",
       "      <td>8319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snap</th>\n",
       "      <td>8310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwigcat</th>\n",
       "      <td>8293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainsplit</th>\n",
       "      <td>8247</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwigcluster</th>\n",
       "      <td>8240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-beditemoverlapcount</th>\n",
       "      <td>8233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-faonerecord</th>\n",
       "      <td>8231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-inline</th>\n",
       "      <td>8219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phame</th>\n",
       "      <td>8216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredsinglecover</th>\n",
       "      <td>8199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigwigcorrelate</th>\n",
       "      <td>8166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fatofastq</th>\n",
       "      <td>8160</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>verifybamid2</th>\n",
       "      <td>8156</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainstitchid</th>\n",
       "      <td>8154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fapolyasizes</th>\n",
       "      <td>8154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fatrans</th>\n",
       "      <td>8153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedrestricttopositions</th>\n",
       "      <td>8152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-blasttopsl</th>\n",
       "      <td>8146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigbednameditems</th>\n",
       "      <td>8136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedgraphpack</th>\n",
       "      <td>8130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainprenet</th>\n",
       "      <td>8130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fastqtofa</th>\n",
       "      <td>8119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredfilter</th>\n",
       "      <td>8099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plant_tribes_assembly_post_processor</th>\n",
       "      <td>8095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedextendranges</th>\n",
       "      <td>8095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtracklayer</th>\n",
       "      <td>8090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedcommonregions</th>\n",
       "      <td>8087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-crtreeindexbed</th>\n",
       "      <td>8058</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainsort</th>\n",
       "      <td>8057</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedcoverage</th>\n",
       "      <td>8052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maskoutfa</th>\n",
       "      <td>8051</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-crtreesearchbed</th>\n",
       "      <td>8047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-axttomaf</th>\n",
       "      <td>8043</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chaintopslbasic</th>\n",
       "      <td>8035</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chaintoaxt</th>\n",
       "      <td>8032</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainfilter</th>\n",
       "      <td>8027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedremoveoverlap</th>\n",
       "      <td>8024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-netclass</th>\n",
       "      <td>8020</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sirius-csifingerid</th>\n",
       "      <td>8017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-axttopsl</th>\n",
       "      <td>8008</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ragout</th>\n",
       "      <td>8007</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedintersect</th>\n",
       "      <td>8003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-addcols</th>\n",
       "      <td>8000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-gaptolift</th>\n",
       "      <td>7997</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-getrna</th>\n",
       "      <td>7995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-nettoaxt</th>\n",
       "      <td>7995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treetime</th>\n",
       "      <td>7993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-featurebits</th>\n",
       "      <td>7992</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-yaml-libyaml</th>\n",
       "      <td>7989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mce</th>\n",
       "      <td>7989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-getrnapred</th>\n",
       "      <td>7985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedweedoverlapping</th>\n",
       "      <td>7983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wiggletools</th>\n",
       "      <td>7983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedpileups</th>\n",
       "      <td>7980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-farc</th>\n",
       "      <td>7968</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainswap</th>\n",
       "      <td>7968</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedtopsl</th>\n",
       "      <td>7966</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fatotab</th>\n",
       "      <td>7962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedgeneparts</th>\n",
       "      <td>7957</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chaintopsl</th>\n",
       "      <td>7955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>moca</th>\n",
       "      <td>7954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-farandomize</th>\n",
       "      <td>7950</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-netsyntenic</th>\n",
       "      <td>7945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainantirepeat</th>\n",
       "      <td>7941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredhisto</th>\n",
       "      <td>7931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-namespace-autoclean</th>\n",
       "      <td>7894</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-biox-workflow-command</th>\n",
       "      <td>7872</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mdasim</th>\n",
       "      <td>7871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sickle</th>\n",
       "      <td>7857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pandaseq</th>\n",
       "      <td>7844</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>savage</th>\n",
       "      <td>7836</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>omssa</th>\n",
       "      <td>7798</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-hivclustering</th>\n",
       "      <td>7798</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>moods</th>\n",
       "      <td>7790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-findmotif</th>\n",
       "      <td>7786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-all</th>\n",
       "      <td>7774</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>schavott</th>\n",
       "      <td>7769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>starseqr</th>\n",
       "      <td>7746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-estorient</th>\n",
       "      <td>7728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>twobitreader</th>\n",
       "      <td>7717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>multigps</th>\n",
       "      <td>7714</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smeg</th>\n",
       "      <td>7705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-aroma.core</th>\n",
       "      <td>7659</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-size</th>\n",
       "      <td>7607</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yara</th>\n",
       "      <td>7599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maxentscan</th>\n",
       "      <td>7595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmmer</th>\n",
       "      <td>7583</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>luciphor2</th>\n",
       "      <td>7580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bbmap</th>\n",
       "      <td>7568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-date-manip</th>\n",
       "      <td>7564</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyquant-ms</th>\n",
       "      <td>7514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metagenome-atlas</th>\n",
       "      <td>7499</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>simlord</th>\n",
       "      <td>7491</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-fatal</th>\n",
       "      <td>7465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-speaq</th>\n",
       "      <td>7441</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-aroma.affymetrix</th>\n",
       "      <td>7435</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vawk</th>\n",
       "      <td>7412</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mbuffer</th>\n",
       "      <td>7388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-inline-c</th>\n",
       "      <td>7387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sanger-cgp-battenberg</th>\n",
       "      <td>7368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metameta</th>\n",
       "      <td>7344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shiver</th>\n",
       "      <td>7320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-histonehmm</th>\n",
       "      <td>7294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nasp</th>\n",
       "      <td>7287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>picrust2</th>\n",
       "      <td>7276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-hyphy-python</th>\n",
       "      <td>7274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastp</th>\n",
       "      <td>7260</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rhtslib</th>\n",
       "      <td>7232</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>verifybamid</th>\n",
       "      <td>7200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-scalar-util-numeric</th>\n",
       "      <td>7194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-warnings</th>\n",
       "      <td>7174</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>s3gof3r</th>\n",
       "      <td>7168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-facount</th>\n",
       "      <td>7163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>toulligqc</th>\n",
       "      <td>7106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-deconstructsigs</th>\n",
       "      <td>7099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-axtchain</th>\n",
       "      <td>7087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ococo</th>\n",
       "      <td>7084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-clone-choose</th>\n",
       "      <td>7044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>repeatmodeler</th>\n",
       "      <td>7016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylip</th>\n",
       "      <td>6999</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plastid</th>\n",
       "      <td>6996</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>theta2</th>\n",
       "      <td>6992</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-ixixx</th>\n",
       "      <td>6962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>obitools</th>\n",
       "      <td>6945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nudup</th>\n",
       "      <td>6931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nextgenmap</th>\n",
       "      <td>6928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyseer</th>\n",
       "      <td>6921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plink2</th>\n",
       "      <td>6890</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trackhub</th>\n",
       "      <td>6884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mimodd</th>\n",
       "      <td>6862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scrm</th>\n",
       "      <td>6861</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mawk</th>\n",
       "      <td>6831</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>srst2</th>\n",
       "      <td>6801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-specio-exporter</th>\n",
       "      <td>6801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>segemehl</th>\n",
       "      <td>6795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rust-ncbitaxonomy</th>\n",
       "      <td>6786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snoscan</th>\n",
       "      <td>6777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-b</th>\n",
       "      <td>6746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-goalie</th>\n",
       "      <td>6719</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-image-exiftool</th>\n",
       "      <td>6708</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tiddit</th>\n",
       "      <td>6666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnaz</th>\n",
       "      <td>6658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafsinregion</th>\n",
       "      <td>6654</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-path-class</th>\n",
       "      <td>6644</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-autoxml</th>\n",
       "      <td>6630</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>recycler</th>\n",
       "      <td>6630</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-class-moose</th>\n",
       "      <td>6605</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>meraculous</th>\n",
       "      <td>6600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maffilter</th>\n",
       "      <td>6588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigpsltopsl</th>\n",
       "      <td>6582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-axtsort</th>\n",
       "      <td>6576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chromgraphtobin</th>\n",
       "      <td>6574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>virmet</th>\n",
       "      <td>6572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-axtswap</th>\n",
       "      <td>6570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafsplitpos</th>\n",
       "      <td>6565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>entrez-direct</th>\n",
       "      <td>6557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-params-validationcompiler</th>\n",
       "      <td>6550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-app</th>\n",
       "      <td>6550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shmlast</th>\n",
       "      <td>6546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-liftup</th>\n",
       "      <td>6543</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-autodtd</th>\n",
       "      <td>6542</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-dbsnoop</th>\n",
       "      <td>6539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-warnings-register</th>\n",
       "      <td>6539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafspecieslist</th>\n",
       "      <td>6535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fafilter</th>\n",
       "      <td>6532</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-gff3topsl</th>\n",
       "      <td>6531</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafspeciessubset</th>\n",
       "      <td>6529</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafaddqrows</th>\n",
       "      <td>6527</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-viennangs</th>\n",
       "      <td>6521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-dbtrash</th>\n",
       "      <td>6519</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maftosnpbed</th>\n",
       "      <td>6517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chromgraphfrombin</th>\n",
       "      <td>6500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-datetime</th>\n",
       "      <td>6498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcfpy</th>\n",
       "      <td>6497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-checkagpandfa</th>\n",
       "      <td>6496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-netfilter</th>\n",
       "      <td>6493</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maftoaxt</th>\n",
       "      <td>6485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-netchainsubset</th>\n",
       "      <td>6483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgloadwiggle</th>\n",
       "      <td>6480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transabyss</th>\n",
       "      <td>6478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-lavtopsl</th>\n",
       "      <td>6475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-autosql</th>\n",
       "      <td>6474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-output</th>\n",
       "      <td>6474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-faalign</th>\n",
       "      <td>6473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-catdir</th>\n",
       "      <td>6473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tadarida-d</th>\n",
       "      <td>6461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgloadchain</th>\n",
       "      <td>6453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-ameme</th>\n",
       "      <td>6451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pymvpa</th>\n",
       "      <td>6436</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafcoverage</th>\n",
       "      <td>6434</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-coltransform</th>\n",
       "      <td>6432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-datetime-locale</th>\n",
       "      <td>6425</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fafrag</th>\n",
       "      <td>6425</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-datetime-timezone</th>\n",
       "      <td>6422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafgene</th>\n",
       "      <td>6422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chopfalines</th>\n",
       "      <td>6415</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maffrags</th>\n",
       "      <td>6414</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-checktablecoords</th>\n",
       "      <td>6413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-catuncomment</th>\n",
       "      <td>6406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-checkcoveragegaps</th>\n",
       "      <td>6402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maffetch</th>\n",
       "      <td>6397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fanoise</th>\n",
       "      <td>6396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-avecols</th>\n",
       "      <td>6390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fafiltern</th>\n",
       "      <td>6388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-facmp</th>\n",
       "      <td>6381</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafsplit</th>\n",
       "      <td>6377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-countchars</th>\n",
       "      <td>6375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maffrag</th>\n",
       "      <td>6373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafranges</th>\n",
       "      <td>6370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafmefirst</th>\n",
       "      <td>6365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgloadmaf</th>\n",
       "      <td>6363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pb-dazzler</th>\n",
       "      <td>6361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>openslide-python</th>\n",
       "      <td>6361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-checkhgfindspec</th>\n",
       "      <td>6358</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maftopsl</th>\n",
       "      <td>6357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgloadbed</th>\n",
       "      <td>6357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maforder</th>\n",
       "      <td>6347</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snp-pipeline</th>\n",
       "      <td>6321</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mafaddirows</th>\n",
       "      <td>6317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pronto</th>\n",
       "      <td>6313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wdltool</th>\n",
       "      <td>6284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbalign</th>\n",
       "      <td>6283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyprophet</th>\n",
       "      <td>6276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mce-shared</th>\n",
       "      <td>6265</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-stitch</th>\n",
       "      <td>6260</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mgltools</th>\n",
       "      <td>6253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>socru</th>\n",
       "      <td>6240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-consensuscore2</th>\n",
       "      <td>6189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snpsplit</th>\n",
       "      <td>6181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nmrglue</th>\n",
       "      <td>6164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metilene</th>\n",
       "      <td>6151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mintmap</th>\n",
       "      <td>6147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prosic</th>\n",
       "      <td>6146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-scandeps</th>\n",
       "      <td>6139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pybel</th>\n",
       "      <td>6131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trnanalysis</th>\n",
       "      <td>6120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-posix</th>\n",
       "      <td>6118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-object-insideout</th>\n",
       "      <td>6087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>readseq</th>\n",
       "      <td>6084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trimadap</th>\n",
       "      <td>6015</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnastructure</th>\n",
       "      <td>6003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>slamdunk</th>\n",
       "      <td>5996</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>weeder</th>\n",
       "      <td>5984</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnashapes</th>\n",
       "      <td>5982</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>midas</th>\n",
       "      <td>5962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pepr</th>\n",
       "      <td>5949</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msaboot</th>\n",
       "      <td>5937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-canary-stability</th>\n",
       "      <td>5907</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-statistics-distributions</th>\n",
       "      <td>5892</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>opsin</th>\n",
       "      <td>5885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-getopt</th>\n",
       "      <td>5876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-homedir</th>\n",
       "      <td>5864</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-blat</th>\n",
       "      <td>5863</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>migmap</th>\n",
       "      <td>5834</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-samr</th>\n",
       "      <td>5819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>purge_haplotigs</th>\n",
       "      <td>5784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-rna-rnaalisplit</th>\n",
       "      <td>5756</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqprep</th>\n",
       "      <td>5744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phyloflash</th>\n",
       "      <td>5737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyfastx</th>\n",
       "      <td>5697</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-noiseq</th>\n",
       "      <td>5679</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-syntactic</th>\n",
       "      <td>5665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pysradb</th>\n",
       "      <td>5658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-forks</th>\n",
       "      <td>5647</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>osra</th>\n",
       "      <td>5639</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svviz</th>\n",
       "      <td>5631</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perl-unsafe-signals</th>\n",
       "      <td>5628</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quicktree</th>\n",
       "      <td>5615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pegex</th>\n",
       "      <td>5582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tassel</th>\n",
       "      <td>5571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mrbayes</th>\n",
       "      <td>5568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>repeatscout</th>\n",
       "      <td>5561</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-bioext</th>\n",
       "      <td>5556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bit-vector</th>\n",
       "      <td>5553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wtforms-alchemy</th>\n",
       "      <td>5509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metfrag</th>\n",
       "      <td>5507</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnablueprint</th>\n",
       "      <td>5503</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transit</th>\n",
       "      <td>5501</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taxtastic</th>\n",
       "      <td>5480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sanger-cgp-vagrent</th>\n",
       "      <td>5470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rebaler</th>\n",
       "      <td>5436</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanosim-h</th>\n",
       "      <td>5432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mir-prefer</th>\n",
       "      <td>5429</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>neptune</th>\n",
       "      <td>5416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plant_tribes_gene_family_phylogeny_builder</th>\n",
       "      <td>5403</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>super-focus</th>\n",
       "      <td>5394</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-builder-tester</th>\n",
       "      <td>5391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-want</th>\n",
       "      <td>5390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pypore</th>\n",
       "      <td>5364</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-prompt</th>\n",
       "      <td>5348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mafft</th>\n",
       "      <td>5345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taxmapper</th>\n",
       "      <td>5345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mashmap</th>\n",
       "      <td>5341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rainbow</th>\n",
       "      <td>5337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mason</th>\n",
       "      <td>5302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pdfkit</th>\n",
       "      <td>5298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tirmite</th>\n",
       "      <td>5298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>socksipy-branch</th>\n",
       "      <td>5296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trawler</th>\n",
       "      <td>5288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>raven-assembler</th>\n",
       "      <td>5280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plasmidtron</th>\n",
       "      <td>5270</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plant_tribes_gene_family_integrator</th>\n",
       "      <td>5263</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-memoize</th>\n",
       "      <td>5252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-excel-writer-xlsx</th>\n",
       "      <td>5241</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-env-path</th>\n",
       "      <td>5236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scrappie</th>\n",
       "      <td>5220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yaggo</th>\n",
       "      <td>5213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rename</th>\n",
       "      <td>5196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-files</th>\n",
       "      <td>5184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mtsv</th>\n",
       "      <td>5181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deeptools</th>\n",
       "      <td>5174</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-acme-damn</th>\n",
       "      <td>5166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xyalign</th>\n",
       "      <td>5143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plant_tribes_kaks_analysis</th>\n",
       "      <td>5122</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shortstack</th>\n",
       "      <td>5118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pycluster</th>\n",
       "      <td>5109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbgcpp</th>\n",
       "      <td>5107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>swga</th>\n",
       "      <td>5105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>paladin</th>\n",
       "      <td>5099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-helpers</th>\n",
       "      <td>5095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rad_haplotyper</th>\n",
       "      <td>5095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rdptools</th>\n",
       "      <td>5094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pybedgraph</th>\n",
       "      <td>5090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-transmappsltogenepred</th>\n",
       "      <td>5086</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-exomedepth</th>\n",
       "      <td>5068</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>recon</th>\n",
       "      <td>5068</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pycli</th>\n",
       "      <td>5063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-velvetoptimiser</th>\n",
       "      <td>5062</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plant_tribes_gene_family_aligner</th>\n",
       "      <td>5062</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-heap-simple-xs</th>\n",
       "      <td>5060</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqmagick</th>\n",
       "      <td>5049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sparse-neighbors-search</th>\n",
       "      <td>5047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-gff3</th>\n",
       "      <td>5044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslmappostchain</th>\n",
       "      <td>5035</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-types</th>\n",
       "      <td>5016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bamtopsl</th>\n",
       "      <td>5000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-digest-crc32</th>\n",
       "      <td>4974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-libsbml</th>\n",
       "      <td>4972</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ema</th>\n",
       "      <td>4952</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bx-python</th>\n",
       "      <td>4947</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treebest</th>\n",
       "      <td>4945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>probamconvert</th>\n",
       "      <td>4944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-expmatrixtobarchartbed</th>\n",
       "      <td>4931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-interactive</th>\n",
       "      <td>4930</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-readbackwards</th>\n",
       "      <td>4929</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaseq</th>\n",
       "      <td>4921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pasa</th>\n",
       "      <td>4919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>adapterremoval</th>\n",
       "      <td>4908</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snpiphy</th>\n",
       "      <td>4905</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sys-sigaction</th>\n",
       "      <td>4904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>meta-sparse</th>\n",
       "      <td>4904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regtools</th>\n",
       "      <td>4896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-config-any</th>\n",
       "      <td>4874</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phantompeakqualtools</th>\n",
       "      <td>4860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>presto</th>\n",
       "      <td>4860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tadarida-c</th>\n",
       "      <td>4859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-heap-simple-perl</th>\n",
       "      <td>4857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xxmotif</th>\n",
       "      <td>4852</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-limma</th>\n",
       "      <td>4835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-attribute-handlers</th>\n",
       "      <td>4814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-fastqstatsandsubsample</th>\n",
       "      <td>4799</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hubcheck</th>\n",
       "      <td>4792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-clustergenes</th>\n",
       "      <td>4784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shustring</th>\n",
       "      <td>4783</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-rubic</th>\n",
       "      <td>4782</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-heap-simple</th>\n",
       "      <td>4777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgloadout</th>\n",
       "      <td>4755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>newick_utils</th>\n",
       "      <td>4752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bigmaftomaf</th>\n",
       "      <td>4741</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgloadnet</th>\n",
       "      <td>4735</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pispino</th>\n",
       "      <td>4733</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-next</th>\n",
       "      <td>4732</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pytriangle</th>\n",
       "      <td>4727</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sequencetools</th>\n",
       "      <td>4726</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-lavtoaxt</th>\n",
       "      <td>4726</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sga</th>\n",
       "      <td>4725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-linestora</th>\n",
       "      <td>4717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maftobigmaf</th>\n",
       "      <td>4716</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-unicode-normalize</th>\n",
       "      <td>4715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-loaded</th>\n",
       "      <td>4713</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgfakeagp</th>\n",
       "      <td>4711</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treemix</th>\n",
       "      <td>4704</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcf2genome</th>\n",
       "      <td>4704</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mrnatogene</th>\n",
       "      <td>4701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-genepredtoprot</th>\n",
       "      <td>4698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-netsplit</th>\n",
       "      <td>4693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-sortedcontainers</th>\n",
       "      <td>4690</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>iqtree</th>\n",
       "      <td>4687</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgspeciesrna</th>\n",
       "      <td>4686</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-graphics-colorobject</th>\n",
       "      <td>4673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgsqldump</th>\n",
       "      <td>4673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgloadoutjoined</th>\n",
       "      <td>4673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-graphics-colornames</th>\n",
       "      <td>4671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>t-coffee</th>\n",
       "      <td>4671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ppanggolin</th>\n",
       "      <td>4658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-gmtime</th>\n",
       "      <td>4651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mzml2isa</th>\n",
       "      <td>4650</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ribotish</th>\n",
       "      <td>4647</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minimap</th>\n",
       "      <td>4646</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>protrac</th>\n",
       "      <td>4642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-ldhggene</th>\n",
       "      <td>4641</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-phytools</th>\n",
       "      <td>4622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgbbidblink</th>\n",
       "      <td>4618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslrc</th>\n",
       "      <td>4610</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-htmlcheck</th>\n",
       "      <td>4608</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-maketablelist</th>\n",
       "      <td>4604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-newprog</th>\n",
       "      <td>4602</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgfindspec</th>\n",
       "      <td>4600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ribotaper</th>\n",
       "      <td>4598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-balanced</th>\n",
       "      <td>4597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgtrackdb</th>\n",
       "      <td>4595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metasnv</th>\n",
       "      <td>4595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maxentpy</th>\n",
       "      <td>4593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-mktime</th>\n",
       "      <td>4587</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-intervaltree</th>\n",
       "      <td>4586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mapsplice</th>\n",
       "      <td>4582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-headrest</th>\n",
       "      <td>4580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vqsr_cnn</th>\n",
       "      <td>4579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>variant_tools</th>\n",
       "      <td>4576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotationdbi</th>\n",
       "      <td>4575</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>malva</th>\n",
       "      <td>4573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbio-nextgen</th>\n",
       "      <td>4567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-inter</th>\n",
       "      <td>4563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-localtime</th>\n",
       "      <td>4563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-cellassign</th>\n",
       "      <td>4555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spydrpick</th>\n",
       "      <td>4545</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hubpubliccheck</th>\n",
       "      <td>4544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>selscan</th>\n",
       "      <td>4540</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnasamba</th>\n",
       "      <td>4537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mauve</th>\n",
       "      <td>4514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sys-info-driver-linux</th>\n",
       "      <td>4505</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mathstats</th>\n",
       "      <td>4486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pasta</th>\n",
       "      <td>4476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-statistics-ttest</th>\n",
       "      <td>4469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylopandas</th>\n",
       "      <td>4469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scoop</th>\n",
       "      <td>4462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wub</th>\n",
       "      <td>4442</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mztosqlite</th>\n",
       "      <td>4441</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sixgill</th>\n",
       "      <td>4433</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>diamond</th>\n",
       "      <td>4416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyx</th>\n",
       "      <td>4410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qorts</th>\n",
       "      <td>4406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>skmer</th>\n",
       "      <td>4404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smalt</th>\n",
       "      <td>4399</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>screamingbackpack</th>\n",
       "      <td>4388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>poppunk</th>\n",
       "      <td>4383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rapsearch</th>\n",
       "      <td>4378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scoary</th>\n",
       "      <td>4367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sicer</th>\n",
       "      <td>4361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>strainest</th>\n",
       "      <td>4351</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sibelia</th>\n",
       "      <td>4345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbcoretools</th>\n",
       "      <td>4317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>selectsequencesfrommsa</th>\n",
       "      <td>4308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>motifscan</th>\n",
       "      <td>4306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-uncompress-gunzip</th>\n",
       "      <td>4306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perl-version</th>\n",
       "      <td>4305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-nanostringnorm</th>\n",
       "      <td>4303</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ushuffle</th>\n",
       "      <td>4301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-cluster</th>\n",
       "      <td>4297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-statistics-r</th>\n",
       "      <td>4291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>micall-lite</th>\n",
       "      <td>4276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>musicc</th>\n",
       "      <td>4274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-mutoss</th>\n",
       "      <td>4253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metametamerge</th>\n",
       "      <td>4251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metawrap</th>\n",
       "      <td>4248</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pp</th>\n",
       "      <td>4246</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyexcelerator</th>\n",
       "      <td>4233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pysvg</th>\n",
       "      <td>4221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-version-next</th>\n",
       "      <td>4214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyensembl</th>\n",
       "      <td>4189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-yaml-tiny</th>\n",
       "      <td>4175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-classdiscovery</th>\n",
       "      <td>4173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapdenovo2</th>\n",
       "      <td>4161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pisces</th>\n",
       "      <td>4158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nucdiff</th>\n",
       "      <td>4155</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>swalign</th>\n",
       "      <td>4154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sonicparanoid</th>\n",
       "      <td>4145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bamtools</th>\n",
       "      <td>4141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metavelvet</th>\n",
       "      <td>4116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>refseq_masher</th>\n",
       "      <td>4112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smallgenomeutilities</th>\n",
       "      <td>4112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-wrap</th>\n",
       "      <td>4108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pydnase</th>\n",
       "      <td>4104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-gkmsvm</th>\n",
       "      <td>4103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tgt</th>\n",
       "      <td>4101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-minems2</th>\n",
       "      <td>4097</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-newick</th>\n",
       "      <td>4088</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dendropy</th>\n",
       "      <td>4080</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>segtools</th>\n",
       "      <td>4075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>segmentation-fold</th>\n",
       "      <td>4048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minnow</th>\n",
       "      <td>4042</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>oases</th>\n",
       "      <td>4030</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sword</th>\n",
       "      <td>4027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pheniqs</th>\n",
       "      <td>4024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tracer</th>\n",
       "      <td>4023</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>simka</th>\n",
       "      <td>4016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snp-pileup</th>\n",
       "      <td>4014</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mreps</th>\n",
       "      <td>4007</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylotoast</th>\n",
       "      <td>4007</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-probmetab</th>\n",
       "      <td>4004</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nucleoatac</th>\n",
       "      <td>4002</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seq-seq-pan</th>\n",
       "      <td>4000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-carp-clan</th>\n",
       "      <td>3987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pythomics</th>\n",
       "      <td>3987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rmats2sashimiplot</th>\n",
       "      <td>3982</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pfam_scan</th>\n",
       "      <td>3965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-asciitable</th>\n",
       "      <td>3962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metagene_annotator</th>\n",
       "      <td>3960</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-logger-simple</th>\n",
       "      <td>3959</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>randfold</th>\n",
       "      <td>3954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mercat</th>\n",
       "      <td>3950</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rapclust</th>\n",
       "      <td>3949</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mauvealigner</th>\n",
       "      <td>3949</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maxquant</th>\n",
       "      <td>3944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncrf</th>\n",
       "      <td>3938</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sqlalchemy-datatables</th>\n",
       "      <td>3925</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ray</th>\n",
       "      <td>3923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rgt</th>\n",
       "      <td>3920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stacks_summary</th>\n",
       "      <td>3913</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pourrna</th>\n",
       "      <td>3902</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>parsnp</th>\n",
       "      <td>3890</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>orfm</th>\n",
       "      <td>3889</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mvp</th>\n",
       "      <td>3889</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcfkit</th>\n",
       "      <td>3874</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pydna</th>\n",
       "      <td>3870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-fileattribute</th>\n",
       "      <td>3859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ensembl-vep</th>\n",
       "      <td>3846</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-semanticdiff</th>\n",
       "      <td>3836</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>umitools</th>\n",
       "      <td>3829</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tablet</th>\n",
       "      <td>3819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-swamp</th>\n",
       "      <td>3818</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-json-maybexs</th>\n",
       "      <td>3818</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-handle</th>\n",
       "      <td>3801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-git-wrapper</th>\n",
       "      <td>3798</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-notabs</th>\n",
       "      <td>3789</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-parallel-loops</th>\n",
       "      <td>3787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pullseq</th>\n",
       "      <td>3786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-list-someutils</th>\n",
       "      <td>3780</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smcounter2</th>\n",
       "      <td>3777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-gwpcr</th>\n",
       "      <td>3777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pysnptools</th>\n",
       "      <td>3760</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pybamtools</th>\n",
       "      <td>3755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tiptoft</th>\n",
       "      <td>3755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dada2</th>\n",
       "      <td>3751</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-yaml</th>\n",
       "      <td>3730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-ic10</th>\n",
       "      <td>3725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pathoscope</th>\n",
       "      <td>3725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pvacseq</th>\n",
       "      <td>3723</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-parser-lite</th>\n",
       "      <td>3722</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-json-parse</th>\n",
       "      <td>3719</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>squizz</th>\n",
       "      <td>3718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mhap</th>\n",
       "      <td>3717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>microbecensus</th>\n",
       "      <td>3705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ped_parser</th>\n",
       "      <td>3699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-trap</th>\n",
       "      <td>3699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-config-simple</th>\n",
       "      <td>3695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-algorithm-dependency</th>\n",
       "      <td>3683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnachipintegrator</th>\n",
       "      <td>3680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pycoqc</th>\n",
       "      <td>3679</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-mmgenome</th>\n",
       "      <td>3675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-base</th>\n",
       "      <td>3674</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pneumocat</th>\n",
       "      <td>3673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maaslin</th>\n",
       "      <td>3673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-pore</th>\n",
       "      <td>3666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taco</th>\n",
       "      <td>3665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy-lib</th>\n",
       "      <td>3664</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-eol</th>\n",
       "      <td>3661</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fasttree</th>\n",
       "      <td>3658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-nowarnings</th>\n",
       "      <td>3651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genefilter</th>\n",
       "      <td>3648</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sharedmem</th>\n",
       "      <td>3638</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gatk4</th>\n",
       "      <td>3636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-term-app-roles</th>\n",
       "      <td>3635</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yanagiba</th>\n",
       "      <td>3616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mtnucratio</th>\n",
       "      <td>3613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wgsim</th>\n",
       "      <td>3606</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mosaik</th>\n",
       "      <td>3603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>womtool</th>\n",
       "      <td>3600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-hppy</th>\n",
       "      <td>3600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-installpaths</th>\n",
       "      <td>3597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sis</th>\n",
       "      <td>3594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minia</th>\n",
       "      <td>3591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nvc</th>\n",
       "      <td>3591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>telseq</th>\n",
       "      <td>3581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>morpheus</th>\n",
       "      <td>3571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-socket6</th>\n",
       "      <td>3567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-fromperlver</th>\n",
       "      <td>3567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-classapi</th>\n",
       "      <td>3564</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-findbin-libs</th>\n",
       "      <td>3560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>visceral-evaluatesegmentation</th>\n",
       "      <td>3559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msproteomicstools</th>\n",
       "      <td>3554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-git-wrapper-plus</th>\n",
       "      <td>3550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>relocate2</th>\n",
       "      <td>3550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stamp</th>\n",
       "      <td>3535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-fgwas</th>\n",
       "      <td>3522</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sbpiper</th>\n",
       "      <td>3521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-color-theme</th>\n",
       "      <td>3517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-term-progressbar</th>\n",
       "      <td>3517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-tcr</th>\n",
       "      <td>3504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-datetime-format-strptime</th>\n",
       "      <td>3489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xatlas</th>\n",
       "      <td>3482</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>squid</th>\n",
       "      <td>3479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>peakranger</th>\n",
       "      <td>3476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-local-lib</th>\n",
       "      <td>3468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treekin</th>\n",
       "      <td>3465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pymisc-utils</th>\n",
       "      <td>3464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msms</th>\n",
       "      <td>3456</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prophex</th>\n",
       "      <td>3448</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-dartr</th>\n",
       "      <td>3445</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msisensor-pro</th>\n",
       "      <td>3441</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sequenza-utils</th>\n",
       "      <td>3438</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seq-gen</th>\n",
       "      <td>3433</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>myriad</th>\n",
       "      <td>3430</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>abundancebin</th>\n",
       "      <td>3422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyrad</th>\n",
       "      <td>3416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>piranha</th>\n",
       "      <td>3413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snakeparse</th>\n",
       "      <td>3411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seedme</th>\n",
       "      <td>3410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-shortread</th>\n",
       "      <td>3404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seq2hla</th>\n",
       "      <td>3403</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-clone-pp</th>\n",
       "      <td>3401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>udocker</th>\n",
       "      <td>3398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-try-tiny-retry</th>\n",
       "      <td>3386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pypiper</th>\n",
       "      <td>3383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ppanini</th>\n",
       "      <td>3381</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnasketch</th>\n",
       "      <td>3378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-scimpute</th>\n",
       "      <td>3376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>panx</th>\n",
       "      <td>3375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>swipe</th>\n",
       "      <td>3372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-misha</th>\n",
       "      <td>3370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nose-capturestderr</th>\n",
       "      <td>3369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sigqc</th>\n",
       "      <td>3365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-config-autoconf</th>\n",
       "      <td>3362</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-authen-sasl-saslprep</th>\n",
       "      <td>3356</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-overlapselect</th>\n",
       "      <td>3355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>merlin</th>\n",
       "      <td>3353</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tepid</th>\n",
       "      <td>3350</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pygresql</th>\n",
       "      <td>3331</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-share</th>\n",
       "      <td>3327</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-types-path-class</th>\n",
       "      <td>3326</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mamotif</th>\n",
       "      <td>3323</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbhoover</th>\n",
       "      <td>3318</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotate</th>\n",
       "      <td>3317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sccaller</th>\n",
       "      <td>3313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tpp</th>\n",
       "      <td>3310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-workflowscriptscommon</th>\n",
       "      <td>3306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hisat2</th>\n",
       "      <td>3302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanovar</th>\n",
       "      <td>3297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clustalw</th>\n",
       "      <td>3296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smina</th>\n",
       "      <td>3295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-random-mt-auto</th>\n",
       "      <td>3290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqyclean</th>\n",
       "      <td>3277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>oligotyping</th>\n",
       "      <td>3276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sys-info-base</th>\n",
       "      <td>3274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tantan</th>\n",
       "      <td>3273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pgma-simple</th>\n",
       "      <td>3269</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-accessor</th>\n",
       "      <td>3266</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rapidnj</th>\n",
       "      <td>3258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plascope</th>\n",
       "      <td>3256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>monocle3-cli</th>\n",
       "      <td>3251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phispy</th>\n",
       "      <td>3247</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ntcard</th>\n",
       "      <td>3243</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>translate-gard</th>\n",
       "      <td>3232</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sys-info</th>\n",
       "      <td>3230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>migraine</th>\n",
       "      <td>3222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>structure</th>\n",
       "      <td>3214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-proc-fork</th>\n",
       "      <td>3212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>magpurify</th>\n",
       "      <td>3211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-elemental</th>\n",
       "      <td>3207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r2r</th>\n",
       "      <td>3205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>textinput</th>\n",
       "      <td>3202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yahmm</th>\n",
       "      <td>3199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scvi</th>\n",
       "      <td>3193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>progressivemauve</th>\n",
       "      <td>3177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>orthomcl</th>\n",
       "      <td>3176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tb-profiler</th>\n",
       "      <td>3171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mapdia</th>\n",
       "      <td>3169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-tabs-wrap</th>\n",
       "      <td>3169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deseq2</th>\n",
       "      <td>3163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>paragraph</th>\n",
       "      <td>3159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-nanopore</th>\n",
       "      <td>3154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mfqe</th>\n",
       "      <td>3152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-specio</th>\n",
       "      <td>3151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyega3</th>\n",
       "      <td>3150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unitig-caller</th>\n",
       "      <td>3140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>velocyto.py</th>\n",
       "      <td>3123</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mixin-linewise</th>\n",
       "      <td>3121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>medpy</th>\n",
       "      <td>3118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>raxml-ng</th>\n",
       "      <td>3117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pmdtools</th>\n",
       "      <td>3107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-msqrob</th>\n",
       "      <td>3106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snvphyl-tools</th>\n",
       "      <td>3105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-go-perl</th>\n",
       "      <td>3100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-depends</th>\n",
       "      <td>3091</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>freebayes</th>\n",
       "      <td>3075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nspdk</th>\n",
       "      <td>3074</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spoa</th>\n",
       "      <td>3071</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-bigint</th>\n",
       "      <td>3070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-edger</th>\n",
       "      <td>3070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-sys-info</th>\n",
       "      <td>3065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pytest-marks</th>\n",
       "      <td>3065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scala</th>\n",
       "      <td>3063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>peakachu</th>\n",
       "      <td>3045</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mapdamage2</th>\n",
       "      <td>3044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>squeakr</th>\n",
       "      <td>3027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnaquast</th>\n",
       "      <td>3024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-cleannamespaces</th>\n",
       "      <td>3022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-constant</th>\n",
       "      <td>3020</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phipack</th>\n",
       "      <td>3018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcfsamplecompare</th>\n",
       "      <td>3013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>paralyzer</th>\n",
       "      <td>3001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-term-encoding</th>\n",
       "      <td>3000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mykatlas</th>\n",
       "      <td>2995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plass</th>\n",
       "      <td>2995</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-compare</th>\n",
       "      <td>2988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ls-gkm</th>\n",
       "      <td>2985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>proteowizard</th>\n",
       "      <td>2984</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>semidbm</th>\n",
       "      <td>2980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unitig-counter</th>\n",
       "      <td>2976</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>targetfinder</th>\n",
       "      <td>2973</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-paranodestop</th>\n",
       "      <td>2970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rvtests</th>\n",
       "      <td>2968</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>var-agg</th>\n",
       "      <td>2965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kallisto</th>\n",
       "      <td>2958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-parahubstop</th>\n",
       "      <td>2945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-paratestjob</th>\n",
       "      <td>2940</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-endsinlf</th>\n",
       "      <td>2937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-paranode</th>\n",
       "      <td>2937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-bedjointaboffset</th>\n",
       "      <td>2935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnacode</th>\n",
       "      <td>2932</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-para</th>\n",
       "      <td>2931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-needs</th>\n",
       "      <td>2931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-pslsomerecords</th>\n",
       "      <td>2928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-const-fast</th>\n",
       "      <td>2926</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-tidygenomics</th>\n",
       "      <td>2924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-websync</th>\n",
       "      <td>2923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneplotter</th>\n",
       "      <td>2919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-fakemp</th>\n",
       "      <td>2915</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>virsorter</th>\n",
       "      <td>2904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>miranda</th>\n",
       "      <td>2898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgloadsqltab</th>\n",
       "      <td>2896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-paranodestatus</th>\n",
       "      <td>2895</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-parasol</th>\n",
       "      <td>2893</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cooler</th>\n",
       "      <td>2887</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-paranodestart</th>\n",
       "      <td>2882</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-gensub2</th>\n",
       "      <td>2881</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-requires</th>\n",
       "      <td>2870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-string-escape</th>\n",
       "      <td>2869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-parahub</th>\n",
       "      <td>2868</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mdust</th>\n",
       "      <td>2844</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cnvkit</th>\n",
       "      <td>2842</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nimnexus</th>\n",
       "      <td>2837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>melt</th>\n",
       "      <td>2832</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngsutils</th>\n",
       "      <td>2819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-match</th>\n",
       "      <td>2814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>misopy</th>\n",
       "      <td>2806</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mgf-formatter</th>\n",
       "      <td>2790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>optplus</th>\n",
       "      <td>2774</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>suppa</th>\n",
       "      <td>2749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tabview</th>\n",
       "      <td>2746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>onto2nx</th>\n",
       "      <td>2746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smhasher</th>\n",
       "      <td>2745</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sprinkles</th>\n",
       "      <td>2744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prosampler</th>\n",
       "      <td>2740</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-build-tiny</th>\n",
       "      <td>2731</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-pctgcdata</th>\n",
       "      <td>2728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sbg-cwl-runner</th>\n",
       "      <td>2720</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scallop</th>\n",
       "      <td>2709</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>setuptools_cython</th>\n",
       "      <td>2709</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-ngsplot</th>\n",
       "      <td>2702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample-sheet</th>\n",
       "      <td>2696</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-html-form</th>\n",
       "      <td>2693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>targqc</th>\n",
       "      <td>2688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biomart</th>\n",
       "      <td>2686</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-facets</th>\n",
       "      <td>2676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ropebwt2</th>\n",
       "      <td>2669</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rhdf5</th>\n",
       "      <td>2665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>optbuild</th>\n",
       "      <td>2658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sys-info-driver-osx</th>\n",
       "      <td>2653</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deblur</th>\n",
       "      <td>2644</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mccortex</th>\n",
       "      <td>2617</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>workspace</th>\n",
       "      <td>2617</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomicfeatures</th>\n",
       "      <td>2617</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-monocle3</th>\n",
       "      <td>2614</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-log4perl</th>\n",
       "      <td>2613</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-spiffy</th>\n",
       "      <td>2611</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pathwaymatcher</th>\n",
       "      <td>2611</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sexdeterrmine</th>\n",
       "      <td>2601</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cpan-shell</th>\n",
       "      <td>2601</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bam-readcount</th>\n",
       "      <td>2597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>modeltest-ng</th>\n",
       "      <td>2581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>malt</th>\n",
       "      <td>2572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vg</th>\n",
       "      <td>2559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sequenza</th>\n",
       "      <td>2553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sparc</th>\n",
       "      <td>2552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pipmir</th>\n",
       "      <td>2551</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-dump</th>\n",
       "      <td>2549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gffutils</th>\n",
       "      <td>2548</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaphlan</th>\n",
       "      <td>2546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>muse</th>\n",
       "      <td>2539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-nastiseq</th>\n",
       "      <td>2537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>arvados-cwl-runner</th>\n",
       "      <td>2537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-string-approx</th>\n",
       "      <td>2530</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnabob</th>\n",
       "      <td>2521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gneiss</th>\n",
       "      <td>2516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>arvados-python-client</th>\n",
       "      <td>2510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-taxonomizr</th>\n",
       "      <td>2509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nseg</th>\n",
       "      <td>2505</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sparseassembler</th>\n",
       "      <td>2497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-exporter-formethods</th>\n",
       "      <td>2489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>potrace</th>\n",
       "      <td>2487</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-empiricalfdr.deseq2</th>\n",
       "      <td>2485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mpa-portable</th>\n",
       "      <td>2479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svict</th>\n",
       "      <td>2462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sql-statement</th>\n",
       "      <td>2457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wtforms-components</th>\n",
       "      <td>2454</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rust-overlaps</th>\n",
       "      <td>2454</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>macrel</th>\n",
       "      <td>2453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wgs2ncbi</th>\n",
       "      <td>2452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sort-versions</th>\n",
       "      <td>2450</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqwish</th>\n",
       "      <td>2445</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minorseq</th>\n",
       "      <td>2440</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>thapbi-pict</th>\n",
       "      <td>2439</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-base-convert</th>\n",
       "      <td>2435</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnasnp</th>\n",
       "      <td>2427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rhdf5lib</th>\n",
       "      <td>2426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>multivcfanalyzer</th>\n",
       "      <td>2416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-fastx-reader</th>\n",
       "      <td>2415</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plek</th>\n",
       "      <td>2407</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hicexplorer</th>\n",
       "      <td>2404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>staden_io_lib</th>\n",
       "      <td>2402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xmlbuilder</th>\n",
       "      <td>2402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapdenovo2-gapcloser</th>\n",
       "      <td>2395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mockinbird</th>\n",
       "      <td>2393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>megan</th>\n",
       "      <td>2393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rdock</th>\n",
       "      <td>2392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plasflow</th>\n",
       "      <td>2388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scikits-datasmooth</th>\n",
       "      <td>2384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>megalodon</th>\n",
       "      <td>2383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-eutilities</th>\n",
       "      <td>2383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tadtool</th>\n",
       "      <td>2375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pycsg</th>\n",
       "      <td>2372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>infernal</th>\n",
       "      <td>2367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vamb</th>\n",
       "      <td>2365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sam</th>\n",
       "      <td>2361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>peakzilla</th>\n",
       "      <td>2358</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyimzml</th>\n",
       "      <td>2355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqan_tcoffee</th>\n",
       "      <td>2352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-chdir</th>\n",
       "      <td>2350</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shape_it</th>\n",
       "      <td>2347</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbh5tools</th>\n",
       "      <td>2346</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-xmlrpc</th>\n",
       "      <td>2345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-cigar</th>\n",
       "      <td>2342</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>csvtk</th>\n",
       "      <td>2339</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transrate-tools</th>\n",
       "      <td>2339</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-rtfbs</th>\n",
       "      <td>2339</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mea</th>\n",
       "      <td>2337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>poa</th>\n",
       "      <td>2333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapec</th>\n",
       "      <td>2324</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-util</th>\n",
       "      <td>2319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyasp</th>\n",
       "      <td>2319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-htssip</th>\n",
       "      <td>2318</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fwdpy11</th>\n",
       "      <td>2315</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smartdenovo</th>\n",
       "      <td>2314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smudgeplot</th>\n",
       "      <td>2313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-string-util</th>\n",
       "      <td>2311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-number-misc</th>\n",
       "      <td>2306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-cache</th>\n",
       "      <td>2305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>openmg</th>\n",
       "      <td>2305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-tcga2stat</th>\n",
       "      <td>2303</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-tee</th>\n",
       "      <td>2300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yanc</th>\n",
       "      <td>2299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sciphi</th>\n",
       "      <td>2296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-structssi</th>\n",
       "      <td>2294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-xsaccessor</th>\n",
       "      <td>2293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sonlib</th>\n",
       "      <td>2289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>strip_it</th>\n",
       "      <td>2281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-config</th>\n",
       "      <td>2277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-ichorcna</th>\n",
       "      <td>2276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qtip</th>\n",
       "      <td>2276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>novoplasty</th>\n",
       "      <td>2275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shapeit4</th>\n",
       "      <td>2274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-toolbox</th>\n",
       "      <td>2273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-checkbin</th>\n",
       "      <td>2273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>super_distance</th>\n",
       "      <td>2262</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qcat</th>\n",
       "      <td>2261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbsim</th>\n",
       "      <td>2259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bedops</th>\n",
       "      <td>2257</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-metalonda</th>\n",
       "      <td>2256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>geneimpacts</th>\n",
       "      <td>2254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rdp_classifier</th>\n",
       "      <td>2251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>secimtools</th>\n",
       "      <td>2248</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minimock</th>\n",
       "      <td>2242</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>starfish</th>\n",
       "      <td>2241</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sigtree</th>\n",
       "      <td>2235</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metavelvet-sl-feature-extraction</th>\n",
       "      <td>2234</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>picard-slim</th>\n",
       "      <td>2229</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nullarbor</th>\n",
       "      <td>2227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mldbm-sync</th>\n",
       "      <td>2223</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>novobreak</th>\n",
       "      <td>2222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seroba</th>\n",
       "      <td>2221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbdagcon</th>\n",
       "      <td>2221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>somatic-sniper</th>\n",
       "      <td>2221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>magicblast</th>\n",
       "      <td>2219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transtermhp</th>\n",
       "      <td>2218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>validate-fasta-database</th>\n",
       "      <td>2218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mugsy</th>\n",
       "      <td>2217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ska</th>\n",
       "      <td>2217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-term-detect-software</th>\n",
       "      <td>2215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>outrigger</th>\n",
       "      <td>2215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-refhash</th>\n",
       "      <td>2214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pp-sketchlib</th>\n",
       "      <td>2206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapdenovo-trans</th>\n",
       "      <td>2206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>query_phenomizer</th>\n",
       "      <td>2203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-xml</th>\n",
       "      <td>2195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tetranscripts</th>\n",
       "      <td>2194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-cpan-meta</th>\n",
       "      <td>2189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>uniprot</th>\n",
       "      <td>2186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>medusa-data-fusion</th>\n",
       "      <td>2185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>takeabreak</th>\n",
       "      <td>2181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>methylpy</th>\n",
       "      <td>2180</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-list-uniq</th>\n",
       "      <td>2179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>samsifter</th>\n",
       "      <td>2177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcftoolbox</th>\n",
       "      <td>2177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perl-osnames</th>\n",
       "      <td>2170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mapsembler2</th>\n",
       "      <td>2169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>meryl</th>\n",
       "      <td>2168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treesapp</th>\n",
       "      <td>2165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spingo</th>\n",
       "      <td>2162</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>np-likeness-scorer</th>\n",
       "      <td>2162</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-hook-lexwrap</th>\n",
       "      <td>2162</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-ggbiplot</th>\n",
       "      <td>2161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-levenshtein</th>\n",
       "      <td>2159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-phyext2</th>\n",
       "      <td>2158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-onto-perl</th>\n",
       "      <td>2158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pantools</th>\n",
       "      <td>2155</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>piler</th>\n",
       "      <td>2154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-dbm-deep</th>\n",
       "      <td>2154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-qorts</th>\n",
       "      <td>2154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-peer</th>\n",
       "      <td>2153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapdenovo2-errorcorrection</th>\n",
       "      <td>2152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spotyping</th>\n",
       "      <td>2149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pblaa</th>\n",
       "      <td>2148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-string-truncate</th>\n",
       "      <td>2147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-dpeak</th>\n",
       "      <td>2145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-types-path-tiny</th>\n",
       "      <td>2144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>swiftlink</th>\n",
       "      <td>2143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>reago</th>\n",
       "      <td>2140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-string-rewriteprefix</th>\n",
       "      <td>2136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>valet</th>\n",
       "      <td>2134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pywdl</th>\n",
       "      <td>2131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>reveal</th>\n",
       "      <td>2130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xtermcolor</th>\n",
       "      <td>2127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perlio</th>\n",
       "      <td>2123</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mwga-utils</th>\n",
       "      <td>2121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sff2fastq</th>\n",
       "      <td>2120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>structureharvester</th>\n",
       "      <td>2114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snns</th>\n",
       "      <td>2113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylics</th>\n",
       "      <td>2111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-nsp</th>\n",
       "      <td>2111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qfilt</th>\n",
       "      <td>2109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-list-compare</th>\n",
       "      <td>2105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xunit-wrapper</th>\n",
       "      <td>2105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-mcpcounter</th>\n",
       "      <td>2105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngmerge</th>\n",
       "      <td>2104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pacbio_falcon</th>\n",
       "      <td>2103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phizz</th>\n",
       "      <td>2103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>recalladapters</th>\n",
       "      <td>2102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-soap-nmr</th>\n",
       "      <td>2100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wheezy.template</th>\n",
       "      <td>2098</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spectra-cluster-cli</th>\n",
       "      <td>2096</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sumaclust</th>\n",
       "      <td>2096</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quorum</th>\n",
       "      <td>2094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-metama</th>\n",
       "      <td>2093</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-sub-attribute</th>\n",
       "      <td>2089</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stride</th>\n",
       "      <td>2084</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rseg</th>\n",
       "      <td>2078</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapdenovo2-prepare</th>\n",
       "      <td>2077</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>networkxgmml</th>\n",
       "      <td>2077</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mtbseq</th>\n",
       "      <td>2076</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quake</th>\n",
       "      <td>2075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>motifraptor</th>\n",
       "      <td>2075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-leidenbase</th>\n",
       "      <td>2075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-nonmoose</th>\n",
       "      <td>2071</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-http-server-simple</th>\n",
       "      <td>2065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>macs2</th>\n",
       "      <td>2064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mglex</th>\n",
       "      <td>2064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treemaker</th>\n",
       "      <td>2064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-epic</th>\n",
       "      <td>2064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-border-style</th>\n",
       "      <td>2063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>w4mclstrpeakpics</th>\n",
       "      <td>2062</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapaligner</th>\n",
       "      <td>2062</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-methodmaker</th>\n",
       "      <td>2061</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-spieceasi</th>\n",
       "      <td>2061</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vtools</th>\n",
       "      <td>2061</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-getopt-long-descriptive</th>\n",
       "      <td>2057</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spectral_hk</th>\n",
       "      <td>2056</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>radsex</th>\n",
       "      <td>2056</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mimeo</th>\n",
       "      <td>2055</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-biox-seq</th>\n",
       "      <td>2055</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sbpipe</th>\n",
       "      <td>2053</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>zifa</th>\n",
       "      <td>2052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-coverage</th>\n",
       "      <td>2052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>n50</th>\n",
       "      <td>2051</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-refhash-weak</th>\n",
       "      <td>2049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-compress-bgzf</th>\n",
       "      <td>2049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nmrpro</th>\n",
       "      <td>2048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-phylomeasures</th>\n",
       "      <td>2043</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-padwalker</th>\n",
       "      <td>2043</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>triform2</th>\n",
       "      <td>2042</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ppi</th>\n",
       "      <td>2037</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pancake</th>\n",
       "      <td>2037</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rdfextras</th>\n",
       "      <td>2036</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-std</th>\n",
       "      <td>2033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>panphlan</th>\n",
       "      <td>2033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>monovar</th>\n",
       "      <td>2032</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ruby-dna-tools</th>\n",
       "      <td>2027</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-perfmeas</th>\n",
       "      <td>2022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xtail</th>\n",
       "      <td>2022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>molpopgen-analysis</th>\n",
       "      <td>2019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tidehunter</th>\n",
       "      <td>2018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>marge</th>\n",
       "      <td>2013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metavelvet-annois</th>\n",
       "      <td>2013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-ampvis</th>\n",
       "      <td>2012</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncfp</th>\n",
       "      <td>2011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaseq-all</th>\n",
       "      <td>2009</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pybamparser</th>\n",
       "      <td>2007</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gmap</th>\n",
       "      <td>2006</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pal_finder</th>\n",
       "      <td>2006</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-toobject</th>\n",
       "      <td>2005</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-app-role-log4perl</th>\n",
       "      <td>2005</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-loom</th>\n",
       "      <td>2004</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sure</th>\n",
       "      <td>2001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>primerprospector</th>\n",
       "      <td>2001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rmap</th>\n",
       "      <td>2000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-list</th>\n",
       "      <td>2000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>music</th>\n",
       "      <td>2000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-mlst-check</th>\n",
       "      <td>1998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mindthegap</th>\n",
       "      <td>1998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rsat-core</th>\n",
       "      <td>1998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>reaper</th>\n",
       "      <td>1994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>augustus</th>\n",
       "      <td>1994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>somaticseq</th>\n",
       "      <td>1993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>barrnap</th>\n",
       "      <td>1993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-object</th>\n",
       "      <td>1990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylophlan</th>\n",
       "      <td>1989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ssu-align</th>\n",
       "      <td>1989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-cimpl</th>\n",
       "      <td>1987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-aliased</th>\n",
       "      <td>1987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-clone</th>\n",
       "      <td>1986</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-rapidr</th>\n",
       "      <td>1985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-visitor</th>\n",
       "      <td>1983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanopore_simulation</th>\n",
       "      <td>1982</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perl4-corelibs</th>\n",
       "      <td>1981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sshmm</th>\n",
       "      <td>1981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ndex-python</th>\n",
       "      <td>1981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-find</th>\n",
       "      <td>1980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-params-coerce</th>\n",
       "      <td>1980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-pod-coverage</th>\n",
       "      <td>1980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ocrad</th>\n",
       "      <td>1980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sweepfinder2</th>\n",
       "      <td>1978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pout2mzid</th>\n",
       "      <td>1978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>slncky</th>\n",
       "      <td>1978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-ansitable</th>\n",
       "      <td>1976</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nlstradamus</th>\n",
       "      <td>1976</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metawatt</th>\n",
       "      <td>1974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-mailund-newick</th>\n",
       "      <td>1971</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-remove</th>\n",
       "      <td>1970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-compare</th>\n",
       "      <td>1969</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioblend</th>\n",
       "      <td>1965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-file-contents</th>\n",
       "      <td>1962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spectrassembler</th>\n",
       "      <td>1961</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pirs</th>\n",
       "      <td>1960</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-estscan2</th>\n",
       "      <td>1958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stellar</th>\n",
       "      <td>1958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-template</th>\n",
       "      <td>1956</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>menetools</th>\n",
       "      <td>1955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-role-parameterized</th>\n",
       "      <td>1955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yass</th>\n",
       "      <td>1954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>npyscreen</th>\n",
       "      <td>1953</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-hash-util-fieldhash-compat</th>\n",
       "      <td>1949</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-walk</th>\n",
       "      <td>1948</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-details</th>\n",
       "      <td>1947</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>htseq</th>\n",
       "      <td>1945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-btlib</th>\n",
       "      <td>1944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-mockmodule</th>\n",
       "      <td>1942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gseapy</th>\n",
       "      <td>1942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rambo-k</th>\n",
       "      <td>1942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-parser</th>\n",
       "      <td>1941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-extutils-pkgconfig</th>\n",
       "      <td>1941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>riboplot</th>\n",
       "      <td>1937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-socket-inet6</th>\n",
       "      <td>1936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>meneco</th>\n",
       "      <td>1935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pymix</th>\n",
       "      <td>1934</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mustang</th>\n",
       "      <td>1931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-longstring</th>\n",
       "      <td>1931</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pysvmlight</th>\n",
       "      <td>1928</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>verse</th>\n",
       "      <td>1925</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-filedirutil</th>\n",
       "      <td>1924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taeper</th>\n",
       "      <td>1923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pkiss</th>\n",
       "      <td>1921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rpkmforgenes</th>\n",
       "      <td>1921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pathogen-profiler</th>\n",
       "      <td>1919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-whopgenome</th>\n",
       "      <td>1918</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-safe</th>\n",
       "      <td>1918</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pgsa</th>\n",
       "      <td>1916</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>translatorx</th>\n",
       "      <td>1916</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-net-ftp</th>\n",
       "      <td>1915</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tssar</th>\n",
       "      <td>1914</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-statistics-lite</th>\n",
       "      <td>1912</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yaha</th>\n",
       "      <td>1911</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lsd</th>\n",
       "      <td>1911</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-subcalls</th>\n",
       "      <td>1906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanonet</th>\n",
       "      <td>1906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-std-fast</th>\n",
       "      <td>1905</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sak</th>\n",
       "      <td>1904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sts-smctc</th>\n",
       "      <td>1903</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nmrml2isa</th>\n",
       "      <td>1903</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>zerone</th>\n",
       "      <td>1898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-lncpipereporter</th>\n",
       "      <td>1897</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ribodiff</th>\n",
       "      <td>1896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snmf</th>\n",
       "      <td>1894</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-xcell</th>\n",
       "      <td>1892</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcf-annotator</th>\n",
       "      <td>1890</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-config-tiny</th>\n",
       "      <td>1889</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>riboraptor</th>\n",
       "      <td>1887</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rascaf</th>\n",
       "      <td>1887</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tedna</th>\n",
       "      <td>1885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bloom-faster</th>\n",
       "      <td>1884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>circexplorer2</th>\n",
       "      <td>1883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyimagingmspec</th>\n",
       "      <td>1882</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-uncompress-bunzip2</th>\n",
       "      <td>1880</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-cacher</th>\n",
       "      <td>1878</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tinscan</th>\n",
       "      <td>1876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metavelvet-sl</th>\n",
       "      <td>1871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-env</th>\n",
       "      <td>1870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>remurna</th>\n",
       "      <td>1869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-role-withoverloading</th>\n",
       "      <td>1869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mgs-canopy</th>\n",
       "      <td>1869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svmlight</th>\n",
       "      <td>1869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pb-falconc</th>\n",
       "      <td>1868</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ddocent</th>\n",
       "      <td>1868</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-unicode-stringprep</th>\n",
       "      <td>1867</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyscaf</th>\n",
       "      <td>1867</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-annotables</th>\n",
       "      <td>1862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-extract-use</th>\n",
       "      <td>1859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metacherchant</th>\n",
       "      <td>1854</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>velvet-sc</th>\n",
       "      <td>1854</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wgfast</th>\n",
       "      <td>1853</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-hash-indexed</th>\n",
       "      <td>1853</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-rrbgen</th>\n",
       "      <td>1848</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-net-ftp-recursive</th>\n",
       "      <td>1847</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-variantannotation</th>\n",
       "      <td>1845</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mhcnuggets</th>\n",
       "      <td>1843</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ogdf</th>\n",
       "      <td>1843</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trinculo</th>\n",
       "      <td>1841</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sift4g</th>\n",
       "      <td>1840</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-simple</th>\n",
       "      <td>1838</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pb-falcon-phase</th>\n",
       "      <td>1837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mergevcf</th>\n",
       "      <td>1835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kma</th>\n",
       "      <td>1834</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-combinatorics</th>\n",
       "      <td>1833</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-prereq</th>\n",
       "      <td>1831</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pipelign</th>\n",
       "      <td>1826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaxa</th>\n",
       "      <td>1826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rop</th>\n",
       "      <td>1826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapsplice</th>\n",
       "      <td>1822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spine</th>\n",
       "      <td>1821</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-hpc-runner-command-plugin-logger-sqlite</th>\n",
       "      <td>1820</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-super</th>\n",
       "      <td>1819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scrublet</th>\n",
       "      <td>1819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scg</th>\n",
       "      <td>1818</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>suma_package</th>\n",
       "      <td>1817</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-par-packer</th>\n",
       "      <td>1816</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pymot</th>\n",
       "      <td>1815</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gimmemotifs</th>\n",
       "      <td>1812</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngshmmalign</th>\n",
       "      <td>1808</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ssake</th>\n",
       "      <td>1807</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svim</th>\n",
       "      <td>1805</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-getopt-argvfile</th>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pydemult</th>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-copy-link</th>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tmalign</th>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>zagros</th>\n",
       "      <td>1804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaphyler</th>\n",
       "      <td>1802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>proot</th>\n",
       "      <td>1802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mendelscan</th>\n",
       "      <td>1801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcflatten</th>\n",
       "      <td>1801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-string-random</th>\n",
       "      <td>1797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>soapcoverage</th>\n",
       "      <td>1794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-simple-text</th>\n",
       "      <td>1794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-json-create</th>\n",
       "      <td>1793</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ra</th>\n",
       "      <td>1792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>platypus-conquistador</th>\n",
       "      <td>1790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phast</th>\n",
       "      <td>1790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>realphy</th>\n",
       "      <td>1789</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-scalar</th>\n",
       "      <td>1789</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tie-hash</th>\n",
       "      <td>1788</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-graph</th>\n",
       "      <td>1787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanoplexer</th>\n",
       "      <td>1787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wgs-assembler</th>\n",
       "      <td>1785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sim4db</th>\n",
       "      <td>1785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-vars</th>\n",
       "      <td>1785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-symbol</th>\n",
       "      <td>1785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-number-range</th>\n",
       "      <td>1784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prophasm</th>\n",
       "      <td>1782</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vdjer</th>\n",
       "      <td>1781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>panoct</th>\n",
       "      <td>1781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>goleft</th>\n",
       "      <td>1781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-indirect</th>\n",
       "      <td>1778</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metacluster</th>\n",
       "      <td>1777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-intego</th>\n",
       "      <td>1777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>repenrich</th>\n",
       "      <td>1777</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-par-dist</th>\n",
       "      <td>1775</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaprob</th>\n",
       "      <td>1774</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>micro_razers</th>\n",
       "      <td>1773</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-par</th>\n",
       "      <td>1772</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-touch</th>\n",
       "      <td>1770</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stereogene</th>\n",
       "      <td>1769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spectrum_utils</th>\n",
       "      <td>1768</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanoblaster</th>\n",
       "      <td>1767</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>intarna</th>\n",
       "      <td>1767</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-memory-cycle</th>\n",
       "      <td>1767</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-checker</th>\n",
       "      <td>1765</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-grbase</th>\n",
       "      <td>1761</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taxator-tk</th>\n",
       "      <td>1761</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-filter-simple</th>\n",
       "      <td>1761</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>netcdf-metadata-info</th>\n",
       "      <td>1760</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-zerone</th>\n",
       "      <td>1758</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-log-any</th>\n",
       "      <td>1757</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bundle-bioperl</th>\n",
       "      <td>1755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-tabs</th>\n",
       "      <td>1755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-symbol-util</th>\n",
       "      <td>1755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-assert</th>\n",
       "      <td>1754</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mlrho</th>\n",
       "      <td>1754</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-digest-sha</th>\n",
       "      <td>1754</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>methylextract</th>\n",
       "      <td>1752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-locale</th>\n",
       "      <td>1752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cd-hit</th>\n",
       "      <td>1752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qiimetomaaslin</th>\n",
       "      <td>1752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>straitrazor</th>\n",
       "      <td>1751</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>migrate-n</th>\n",
       "      <td>1750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-net-netrc</th>\n",
       "      <td>1750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>splitmem</th>\n",
       "      <td>1750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-statistics-frequency</th>\n",
       "      <td>1749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-role-tiny-with</th>\n",
       "      <td>1748</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-plaintext</th>\n",
       "      <td>1745</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-utilities</th>\n",
       "      <td>1744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xhmm</th>\n",
       "      <td>1743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>talloc</th>\n",
       "      <td>1743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pgcgap</th>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-tap-harness-env</th>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>satrap</th>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-blib</th>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-rphast</th>\n",
       "      <td>1742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>twopaco</th>\n",
       "      <td>1741</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sfs_code</th>\n",
       "      <td>1740</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-integer</th>\n",
       "      <td>1739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-exporter-tidy</th>\n",
       "      <td>1739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-cell-browser</th>\n",
       "      <td>1739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cpan-meta-validator</th>\n",
       "      <td>1738</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rdp-readseq</th>\n",
       "      <td>1737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cairo</th>\n",
       "      <td>1736</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sqlitebrowser</th>\n",
       "      <td>1735</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-unit-lite</th>\n",
       "      <td>1734</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-pushd</th>\n",
       "      <td>1733</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>telescope</th>\n",
       "      <td>1732</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shapemapper</th>\n",
       "      <td>1730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lrzip</th>\n",
       "      <td>1729</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-experimental</th>\n",
       "      <td>1728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deeptoolsintervals</th>\n",
       "      <td>1728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>splicemap</th>\n",
       "      <td>1727</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>solexaqa</th>\n",
       "      <td>1727</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanosplit</th>\n",
       "      <td>1724</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>truvari</th>\n",
       "      <td>1720</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-uncompress-rawinflate</th>\n",
       "      <td>1718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-unicode-utf8</th>\n",
       "      <td>1718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>teloclip</th>\n",
       "      <td>1717</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-perlio-encoding</th>\n",
       "      <td>1716</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pod-coverage-trustpod</th>\n",
       "      <td>1715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snpomatic</th>\n",
       "      <td>1715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaquant</th>\n",
       "      <td>1714</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbwt</th>\n",
       "      <td>1714</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>satsuma2</th>\n",
       "      <td>1712</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spectacle</th>\n",
       "      <td>1711</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tagger</th>\n",
       "      <td>1710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-singleton</th>\n",
       "      <td>1710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pauda</th>\n",
       "      <td>1707</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-complex</th>\n",
       "      <td>1705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-types-stringlike</th>\n",
       "      <td>1705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-george</th>\n",
       "      <td>1705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mime-quotedprint</th>\n",
       "      <td>1704</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rfmix</th>\n",
       "      <td>1703</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gatk</th>\n",
       "      <td>1703</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xcftools</th>\n",
       "      <td>1702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transcomb</th>\n",
       "      <td>1701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-xml-entities</th>\n",
       "      <td>1701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tinysink</th>\n",
       "      <td>1701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sprai</th>\n",
       "      <td>1700</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>opencontactcli</th>\n",
       "      <td>1699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-estscan1</th>\n",
       "      <td>1699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mmvc</th>\n",
       "      <td>1699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scanindel</th>\n",
       "      <td>1696</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scanpy-scripts</th>\n",
       "      <td>1695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nucamino</th>\n",
       "      <td>1694</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>strudel</th>\n",
       "      <td>1693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>virema</th>\n",
       "      <td>1688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gemini</th>\n",
       "      <td>1688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-autodie</th>\n",
       "      <td>1685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-compress-deflate</th>\n",
       "      <td>1685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pbkdf2-tiny</th>\n",
       "      <td>1684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-module-util</th>\n",
       "      <td>1684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngseqbasic</th>\n",
       "      <td>1684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>protk</th>\n",
       "      <td>1683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tqdist</th>\n",
       "      <td>1683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-crypt-openssl-random</th>\n",
       "      <td>1683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-exec</th>\n",
       "      <td>1681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>megagta</th>\n",
       "      <td>1680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-bigfile</th>\n",
       "      <td>1680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ipython-cluster-helper</th>\n",
       "      <td>1680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>real</th>\n",
       "      <td>1677</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-heap</th>\n",
       "      <td>1677</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>saffrontree</th>\n",
       "      <td>1675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>odgi</th>\n",
       "      <td>1673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-crypt-openssl-rsa</th>\n",
       "      <td>1670</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyopt</th>\n",
       "      <td>1663</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>simwalk2</th>\n",
       "      <td>1660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qcumber</th>\n",
       "      <td>1656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqmap</th>\n",
       "      <td>1655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>slimm</th>\n",
       "      <td>1655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-import-into</th>\n",
       "      <td>1650</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>semeta</th>\n",
       "      <td>1649</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>roary2fripan.py</th>\n",
       "      <td>1649</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>red</th>\n",
       "      <td>1648</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seer</th>\n",
       "      <td>1648</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spanki</th>\n",
       "      <td>1643</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phenix</th>\n",
       "      <td>1642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genepop</th>\n",
       "      <td>1635</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ms</th>\n",
       "      <td>1634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>abyss</th>\n",
       "      <td>1633</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-cycle</th>\n",
       "      <td>1631</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>riborex</th>\n",
       "      <td>1629</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnaclust</th>\n",
       "      <td>1627</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-cate</th>\n",
       "      <td>1627</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lsc</th>\n",
       "      <td>1627</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-nbpseq</th>\n",
       "      <td>1627</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sloika</th>\n",
       "      <td>1624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-automatedannotation</th>\n",
       "      <td>1623</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome</th>\n",
       "      <td>1622</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vargeno</th>\n",
       "      <td>1621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-orqa</th>\n",
       "      <td>1621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-leapp</th>\n",
       "      <td>1619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>revtrans</th>\n",
       "      <td>1619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>abeona</th>\n",
       "      <td>1618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>riboseq-rust</th>\n",
       "      <td>1618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-preprocesscore</th>\n",
       "      <td>1617</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-utils</th>\n",
       "      <td>1615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-lme4qtl</th>\n",
       "      <td>1615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-fork</th>\n",
       "      <td>1615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pash</th>\n",
       "      <td>1612</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-slurp</th>\n",
       "      <td>1612</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-phewas</th>\n",
       "      <td>1609</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transgenescan</th>\n",
       "      <td>1608</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smashbenchmarking</th>\n",
       "      <td>1604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nucleosome_prediction</th>\n",
       "      <td>1600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scaffold_builder</th>\n",
       "      <td>1599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>odose</th>\n",
       "      <td>1598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-grain</th>\n",
       "      <td>1598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>readfq</th>\n",
       "      <td>1598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobambam</th>\n",
       "      <td>1597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flye</th>\n",
       "      <td>1596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mac-propertylist</th>\n",
       "      <td>1596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>roprofile</th>\n",
       "      <td>1592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-www-mechanize</th>\n",
       "      <td>1591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-template-simple</th>\n",
       "      <td>1589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ipc-run3</th>\n",
       "      <td>1582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-genometricorr</th>\n",
       "      <td>1581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pretextmap</th>\n",
       "      <td>1580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>strike</th>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-findbin-real</th>\n",
       "      <td>1574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-munge</th>\n",
       "      <td>1569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vnl</th>\n",
       "      <td>1563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>epic</th>\n",
       "      <td>1561</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-return-multilevel</th>\n",
       "      <td>1559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metavelvet-sl-pipeline</th>\n",
       "      <td>1558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unitas</th>\n",
       "      <td>1557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-pophelper</th>\n",
       "      <td>1557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qgrs-cpp</th>\n",
       "      <td>1554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ra-integrate</th>\n",
       "      <td>1553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-without-module</th>\n",
       "      <td>1552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>polymutt</th>\n",
       "      <td>1549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-file-copy-recursive-reduced</th>\n",
       "      <td>1544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-snap</th>\n",
       "      <td>1544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-class-singleton</th>\n",
       "      <td>1544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rtk</th>\n",
       "      <td>1542</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msalign2</th>\n",
       "      <td>1541</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ribotricer</th>\n",
       "      <td>1535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-smartsva</th>\n",
       "      <td>1534</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sgtr</th>\n",
       "      <td>1534</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>srnapipe</th>\n",
       "      <td>1533</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-minionqc</th>\n",
       "      <td>1530</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tpmcalculator</th>\n",
       "      <td>1526</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ltr_retriever</th>\n",
       "      <td>1522</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genenotebook</th>\n",
       "      <td>1519</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mageck</th>\n",
       "      <td>1513</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nf-core</th>\n",
       "      <td>1511</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rasusa</th>\n",
       "      <td>1505</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>aragorn</th>\n",
       "      <td>1500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tximport</th>\n",
       "      <td>1498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>matlock</th>\n",
       "      <td>1491</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sketchy</th>\n",
       "      <td>1490</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ntjoin</th>\n",
       "      <td>1488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pychopper</th>\n",
       "      <td>1488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mantis</th>\n",
       "      <td>1486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-array-set</th>\n",
       "      <td>1486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>busco</th>\n",
       "      <td>1475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nemo</th>\n",
       "      <td>1475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>piret</th>\n",
       "      <td>1473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rbgl</th>\n",
       "      <td>1470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sicer2</th>\n",
       "      <td>1467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-probe-perl</th>\n",
       "      <td>1465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>last</th>\n",
       "      <td>1461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>circos</th>\n",
       "      <td>1460</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-script</th>\n",
       "      <td>1460</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ephemeris</th>\n",
       "      <td>1457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbipa</th>\n",
       "      <td>1453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graphclust-wrappers</th>\n",
       "      <td>1447</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rsubread</th>\n",
       "      <td>1437</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>multiz</th>\n",
       "      <td>1432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>malder</th>\n",
       "      <td>1431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-acidbase</th>\n",
       "      <td>1427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>atropos</th>\n",
       "      <td>1420</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-umi4c</th>\n",
       "      <td>1417</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sv2</th>\n",
       "      <td>1406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minys</th>\n",
       "      <td>1402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>oligoarrayaux</th>\n",
       "      <td>1397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-mac-systemdirectory</th>\n",
       "      <td>1393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-mixkernel</th>\n",
       "      <td>1391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dnacopy</th>\n",
       "      <td>1390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gromacs</th>\n",
       "      <td>1390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-shaman</th>\n",
       "      <td>1388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-riborex</th>\n",
       "      <td>1376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ariba</th>\n",
       "      <td>1376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hggoldgapgl</th>\n",
       "      <td>1373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hicmatrix</th>\n",
       "      <td>1371</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>emboss</th>\n",
       "      <td>1368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyio</th>\n",
       "      <td>1368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>makehub</th>\n",
       "      <td>1365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shannon_cpp</th>\n",
       "      <td>1361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-chainbridge</th>\n",
       "      <td>1359</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affy</th>\n",
       "      <td>1354</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ucsc-hgvstovcf</th>\n",
       "      <td>1349</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sabre</th>\n",
       "      <td>1333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ma</th>\n",
       "      <td>1329</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>anvio</th>\n",
       "      <td>1327</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-moosex-object-pluggable</th>\n",
       "      <td>1325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>probcons</th>\n",
       "      <td>1322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy-ml</th>\n",
       "      <td>1321</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaquantome</th>\n",
       "      <td>1320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-biosails</th>\n",
       "      <td>1319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>micropita</th>\n",
       "      <td>1319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vmatch</th>\n",
       "      <td>1319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pomoxis</th>\n",
       "      <td>1317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shark</th>\n",
       "      <td>1313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gxformat2</th>\n",
       "      <td>1311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bismark</th>\n",
       "      <td>1307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kmer-jellyfish</th>\n",
       "      <td>1306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>digestiflow-cli</th>\n",
       "      <td>1305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>zorro</th>\n",
       "      <td>1305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gridss</th>\n",
       "      <td>1304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-protgenerics</th>\n",
       "      <td>1300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncbi-util-legacy</th>\n",
       "      <td>1297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaeuk</th>\n",
       "      <td>1292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>samsum</th>\n",
       "      <td>1284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mrsfast</th>\n",
       "      <td>1276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mlpy</th>\n",
       "      <td>1275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multtest</th>\n",
       "      <td>1275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mitos</th>\n",
       "      <td>1272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mapping-iterative-assembler</th>\n",
       "      <td>1270</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-go.db</th>\n",
       "      <td>1270</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rxdock</th>\n",
       "      <td>1264</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-popgenreport</th>\n",
       "      <td>1262</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-metacoder</th>\n",
       "      <td>1261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-virfinder</th>\n",
       "      <td>1261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pbmarkdup</th>\n",
       "      <td>1259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>canu</th>\n",
       "      <td>1254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sva</th>\n",
       "      <td>1253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xcms</th>\n",
       "      <td>1253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bio_hansel</th>\n",
       "      <td>1252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cmappy</th>\n",
       "      <td>1246</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>alfred</th>\n",
       "      <td>1244</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>psascan</th>\n",
       "      <td>1242</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scprep</th>\n",
       "      <td>1236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqhax</th>\n",
       "      <td>1233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gsalign</th>\n",
       "      <td>1231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-checkos</th>\n",
       "      <td>1230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>groot</th>\n",
       "      <td>1230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-number-witherror</th>\n",
       "      <td>1227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocfilecache</th>\n",
       "      <td>1225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-prefork</th>\n",
       "      <td>1224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-statistics-caseresampling</th>\n",
       "      <td>1222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ltr_finder</th>\n",
       "      <td>1222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-escape-houdini</th>\n",
       "      <td>1221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mtsv-tools</th>\n",
       "      <td>1220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>proda</th>\n",
       "      <td>1219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>samstats</th>\n",
       "      <td>1212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>krbalancing</th>\n",
       "      <td>1208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-liger</th>\n",
       "      <td>1205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biovizbase</th>\n",
       "      <td>1204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svaba</th>\n",
       "      <td>1202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qvalue</th>\n",
       "      <td>1201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-archive-tar-wrapper</th>\n",
       "      <td>1194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-lectrotest</th>\n",
       "      <td>1193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-impute</th>\n",
       "      <td>1192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.hs.eg.db</th>\n",
       "      <td>1190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vsn</th>\n",
       "      <td>1188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-dumbbench</th>\n",
       "      <td>1187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>abricate</th>\n",
       "      <td>1187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-devel-cover</th>\n",
       "      <td>1186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wise2</th>\n",
       "      <td>1185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-levenshteinxs</th>\n",
       "      <td>1182</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sew</th>\n",
       "      <td>1181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plasmidid</th>\n",
       "      <td>1179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-acidplots</th>\n",
       "      <td>1178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-utf8</th>\n",
       "      <td>1171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hdf5array</th>\n",
       "      <td>1169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ptrimmer</th>\n",
       "      <td>1168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>locarna</th>\n",
       "      <td>1166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jbrowse</th>\n",
       "      <td>1166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylommand</th>\n",
       "      <td>1166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bignum</th>\n",
       "      <td>1164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-test-file</th>\n",
       "      <td>1164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgraphviz</th>\n",
       "      <td>1163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mhcflurry</th>\n",
       "      <td>1161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beachmat</th>\n",
       "      <td>1158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ripser</th>\n",
       "      <td>1153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gosemsim</th>\n",
       "      <td>1148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-parallel-iterator</th>\n",
       "      <td>1148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensembldb</th>\n",
       "      <td>1146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>duphold</th>\n",
       "      <td>1144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taxonomy</th>\n",
       "      <td>1143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mace</th>\n",
       "      <td>1140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-bigrat</th>\n",
       "      <td>1137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dx-cwl</th>\n",
       "      <td>1135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kaiju</th>\n",
       "      <td>1133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-b-debug</th>\n",
       "      <td>1130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snostrip</th>\n",
       "      <td>1127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>behst</th>\n",
       "      <td>1116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dunovo</th>\n",
       "      <td>1113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fgbio</th>\n",
       "      <td>1106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>oncotator</th>\n",
       "      <td>1103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smashpp</th>\n",
       "      <td>1098</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-csaw</th>\n",
       "      <td>1092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-deseqanalysis</th>\n",
       "      <td>1088</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>epic2</th>\n",
       "      <td>1087</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diffbind</th>\n",
       "      <td>1086</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deseq</th>\n",
       "      <td>1082</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>showit</th>\n",
       "      <td>1081</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>glimmerhmm</th>\n",
       "      <td>1076</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-titancna</th>\n",
       "      <td>1076</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgatcore</th>\n",
       "      <td>1072</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>khmer</th>\n",
       "      <td>1068</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scelvis</th>\n",
       "      <td>1067</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hic2cool</th>\n",
       "      <td>1064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>delly</th>\n",
       "      <td>1063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spring</th>\n",
       "      <td>1063</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fast5</th>\n",
       "      <td>1060</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ntedit</th>\n",
       "      <td>1058</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scater</th>\n",
       "      <td>1055</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylocsf</th>\n",
       "      <td>1053</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scallop-lr</th>\n",
       "      <td>1050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sdm</th>\n",
       "      <td>1037</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gffread</th>\n",
       "      <td>1034</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dose</th>\n",
       "      <td>1033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.mm.eg.db</th>\n",
       "      <td>1033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconda-utils</th>\n",
       "      <td>1024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kraken</th>\n",
       "      <td>1017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smafa</th>\n",
       "      <td>1015</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>krona</th>\n",
       "      <td>1011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>grep</th>\n",
       "      <td>994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sesimcmc</th>\n",
       "      <td>993</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mzr</th>\n",
       "      <td>984</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcamethods</th>\n",
       "      <td>981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-massspecwavelet</th>\n",
       "      <td>978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pretextsnapshot</th>\n",
       "      <td>972</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-motifbinner</th>\n",
       "      <td>970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-eel</th>\n",
       "      <td>966</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prefersim</th>\n",
       "      <td>964</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pairsnp</th>\n",
       "      <td>964</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-acidgenerics</th>\n",
       "      <td>964</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graphaligner</th>\n",
       "      <td>963</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>checkm-genome</th>\n",
       "      <td>962</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbreport</th>\n",
       "      <td>961</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libsequence</th>\n",
       "      <td>958</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scran</th>\n",
       "      <td>954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>inheritance</th>\n",
       "      <td>948</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>revbayes</th>\n",
       "      <td>944</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pblat</th>\n",
       "      <td>943</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>matam</th>\n",
       "      <td>942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>strling</th>\n",
       "      <td>939</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lexmapr</th>\n",
       "      <td>938</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dxpy</th>\n",
       "      <td>937</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jcvi</th>\n",
       "      <td>936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>popscle</th>\n",
       "      <td>936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gtdbtk</th>\n",
       "      <td>934</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dr-disco</th>\n",
       "      <td>922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tinyalign</th>\n",
       "      <td>920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-camera</th>\n",
       "      <td>920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocstyle</th>\n",
       "      <td>919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-htsget</th>\n",
       "      <td>917</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fit_nbinom</th>\n",
       "      <td>914</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crossmap</th>\n",
       "      <td>911</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deepvariant</th>\n",
       "      <td>908</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clusterprofiler</th>\n",
       "      <td>905</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotationhub</th>\n",
       "      <td>904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fgsea</th>\n",
       "      <td>899</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>homer</th>\n",
       "      <td>896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prince</th>\n",
       "      <td>894</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>igdiscover</th>\n",
       "      <td>886</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transrate</th>\n",
       "      <td>885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svanalyzer</th>\n",
       "      <td>885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xs-sim</th>\n",
       "      <td>884</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>varfish-annotator-cli</th>\n",
       "      <td>883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nxtrim</th>\n",
       "      <td>883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowcore</th>\n",
       "      <td>879</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsva</th>\n",
       "      <td>879</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gffcompare</th>\n",
       "      <td>878</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>primerclip</th>\n",
       "      <td>876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regional</th>\n",
       "      <td>874</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>repdenovo</th>\n",
       "      <td>873</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotationfilter</th>\n",
       "      <td>873</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylodm</th>\n",
       "      <td>872</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-monocle</th>\n",
       "      <td>872</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sensv</th>\n",
       "      <td>871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-singlecellexperiment</th>\n",
       "      <td>871</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylkit</th>\n",
       "      <td>866</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gappa</th>\n",
       "      <td>865</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gsort</th>\n",
       "      <td>865</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomepy</th>\n",
       "      <td>863</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ebimage</th>\n",
       "      <td>862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cn.mops</th>\n",
       "      <td>862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomation</th>\n",
       "      <td>860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-screen</th>\n",
       "      <td>859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blast-legacy</th>\n",
       "      <td>857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>involucro</th>\n",
       "      <td>855</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biofluff</th>\n",
       "      <td>849</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminaio</th>\n",
       "      <td>848</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>comet-ms</th>\n",
       "      <td>847</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gseabase</th>\n",
       "      <td>845</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ddrage</th>\n",
       "      <td>845</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>asciigenome</th>\n",
       "      <td>838</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tb_variant_filter</th>\n",
       "      <td>837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kmc</th>\n",
       "      <td>835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbio-nextgen-vm</th>\n",
       "      <td>835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metabinkit</th>\n",
       "      <td>826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-vision</th>\n",
       "      <td>826</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pal2nal</th>\n",
       "      <td>824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>circlator</th>\n",
       "      <td>822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mutmap</th>\n",
       "      <td>819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ropls</th>\n",
       "      <td>814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gubbins</th>\n",
       "      <td>811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bumphunter</th>\n",
       "      <td>811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>humann2</th>\n",
       "      <td>807</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-velocyto.r</th>\n",
       "      <td>805</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scnic</th>\n",
       "      <td>802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ggbio</th>\n",
       "      <td>801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>translig</th>\n",
       "      <td>801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioservices</th>\n",
       "      <td>796</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tepeaks</th>\n",
       "      <td>795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-interactivedisplaybase</th>\n",
       "      <td>795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msnbase</th>\n",
       "      <td>794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>advntr</th>\n",
       "      <td>794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ebseq</th>\n",
       "      <td>792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lumpy-sv</th>\n",
       "      <td>790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pretextgraph</th>\n",
       "      <td>787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dirichletmultinomial</th>\n",
       "      <td>786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>concoct</th>\n",
       "      <td>785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spliceai</th>\n",
       "      <td>785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>epa-ng</th>\n",
       "      <td>775</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ncdfflow</th>\n",
       "      <td>775</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-keggrest</th>\n",
       "      <td>771</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tfbstools</th>\n",
       "      <td>769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mhcnames</th>\n",
       "      <td>769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snp-mutator</th>\n",
       "      <td>768</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-regioner</th>\n",
       "      <td>764</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>exonerate</th>\n",
       "      <td>764</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cardinal</th>\n",
       "      <td>763</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dexseq</th>\n",
       "      <td>763</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-delayedmatrixstats</th>\n",
       "      <td>762</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crispritz</th>\n",
       "      <td>762</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-complexheatmap</th>\n",
       "      <td>759</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tspex</th>\n",
       "      <td>757</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hifive</th>\n",
       "      <td>753</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kraken2</th>\n",
       "      <td>752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rots</th>\n",
       "      <td>751</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scssim</th>\n",
       "      <td>749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mosaics</th>\n",
       "      <td>744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lima</th>\n",
       "      <td>743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-pdf-table</th>\n",
       "      <td>743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geoquery</th>\n",
       "      <td>739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gviz</th>\n",
       "      <td>738</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scvelo</th>\n",
       "      <td>732</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>igv</th>\n",
       "      <td>730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pypairs</th>\n",
       "      <td>725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lpsymphony</th>\n",
       "      <td>725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cnv_facets</th>\n",
       "      <td>725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>beast</th>\n",
       "      <td>722</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qualimap</th>\n",
       "      <td>721</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ig-flowtools</th>\n",
       "      <td>720</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncbi-acc-download</th>\n",
       "      <td>718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.dm.eg.db</th>\n",
       "      <td>718</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>qtlseq</th>\n",
       "      <td>715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>breseq</th>\n",
       "      <td>713</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>goatools</th>\n",
       "      <td>711</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>probabilistic2020</th>\n",
       "      <td>710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bd2k-python-lib</th>\n",
       "      <td>709</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blasr</th>\n",
       "      <td>707</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wg-blimp</th>\n",
       "      <td>706</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flashlfq</th>\n",
       "      <td>705</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>centrifuge</th>\n",
       "      <td>704</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phyloseq</th>\n",
       "      <td>699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqfu</th>\n",
       "      <td>698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipseq</th>\n",
       "      <td>697</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flowcraft</th>\n",
       "      <td>695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tesorter</th>\n",
       "      <td>695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncbimeta</th>\n",
       "      <td>688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jalview</th>\n",
       "      <td>687</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowsom</th>\n",
       "      <td>687</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotationforge</th>\n",
       "      <td>686</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-apeglm</th>\n",
       "      <td>685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fastseg</th>\n",
       "      <td>685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quickmerge</th>\n",
       "      <td>685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq_utils</th>\n",
       "      <td>684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>anvio-minimal</th>\n",
       "      <td>683</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oligo</th>\n",
       "      <td>682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vpolo</th>\n",
       "      <td>682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prada</th>\n",
       "      <td>681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cner</th>\n",
       "      <td>681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hilbertvis</th>\n",
       "      <td>678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nthits</th>\n",
       "      <td>678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-pipette</th>\n",
       "      <td>678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-organismdbi</th>\n",
       "      <td>678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bacon</th>\n",
       "      <td>676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ringo</th>\n",
       "      <td>676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowworkspace</th>\n",
       "      <td>676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kat</th>\n",
       "      <td>675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bayespeak</th>\n",
       "      <td>675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-compass</th>\n",
       "      <td>674</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sspa</th>\n",
       "      <td>673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-repitools</th>\n",
       "      <td>673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-goseq</th>\n",
       "      <td>672</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-exomecopy</th>\n",
       "      <td>671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>igvtools</th>\n",
       "      <td>670</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mutscan</th>\n",
       "      <td>669</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crispresso2</th>\n",
       "      <td>669</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mageck-vispr</th>\n",
       "      <td>667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svist4get</th>\n",
       "      <td>666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genometools-genometools</th>\n",
       "      <td>666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genelendatabase</th>\n",
       "      <td>666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbio-prioritize</th>\n",
       "      <td>665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cosi2</th>\n",
       "      <td>664</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>elprep</th>\n",
       "      <td>662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gaga</th>\n",
       "      <td>662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-masigpro</th>\n",
       "      <td>662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mspurity</th>\n",
       "      <td>661</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-motiv</th>\n",
       "      <td>660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-matter</th>\n",
       "      <td>660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bader</th>\n",
       "      <td>660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowfp</th>\n",
       "      <td>658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spacepharer</th>\n",
       "      <td>658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcas</th>\n",
       "      <td>658</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xpressplot</th>\n",
       "      <td>657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgadem</th>\n",
       "      <td>657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-text-csv_xs</th>\n",
       "      <td>656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>repaq</th>\n",
       "      <td>656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biomformat</th>\n",
       "      <td>655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>isoseq3</th>\n",
       "      <td>655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>grabix</th>\n",
       "      <td>653</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deepsnv</th>\n",
       "      <td>652</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipcomp</th>\n",
       "      <td>652</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-signer</th>\n",
       "      <td>651</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jannovar-cli</th>\n",
       "      <td>649</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tweedeseq</th>\n",
       "      <td>647</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellnoptr</th>\n",
       "      <td>646</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomedata</th>\n",
       "      <td>645</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-topgo</th>\n",
       "      <td>644</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocneighbors</th>\n",
       "      <td>643</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vibrant</th>\n",
       "      <td>642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stark</th>\n",
       "      <td>642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsseq</th>\n",
       "      <td>641</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kegggraph</th>\n",
       "      <td>634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>irida-linker</th>\n",
       "      <td>633</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipseeker</th>\n",
       "      <td>632</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-degreport</th>\n",
       "      <td>632</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>thesias</th>\n",
       "      <td>631</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>krakenuniq</th>\n",
       "      <td>628</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affxparser</th>\n",
       "      <td>626</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dinopy</th>\n",
       "      <td>625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>beagle-lib</th>\n",
       "      <td>625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-immunedeconv</th>\n",
       "      <td>625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>itsxpress</th>\n",
       "      <td>624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sctools</th>\n",
       "      <td>624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tadbit</th>\n",
       "      <td>623</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cummerbund</th>\n",
       "      <td>621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>longshot</th>\n",
       "      <td>616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-systempiper</th>\n",
       "      <td>615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-fastx-abi</th>\n",
       "      <td>615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastx_toolkit</th>\n",
       "      <td>611</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>panaroo</th>\n",
       "      <td>610</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>refgenconf</th>\n",
       "      <td>609</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-purecn</th>\n",
       "      <td>609</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sierrapy</th>\n",
       "      <td>607</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sccaf</th>\n",
       "      <td>604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>terminus</th>\n",
       "      <td>604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ggtree</th>\n",
       "      <td>603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocviews</th>\n",
       "      <td>601</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gdsfmt</th>\n",
       "      <td>596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>itero</th>\n",
       "      <td>595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-category</th>\n",
       "      <td>594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hsmmsinglecell</th>\n",
       "      <td>594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>refgenie</th>\n",
       "      <td>592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbio-variation-recall</th>\n",
       "      <td>592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowpeaks</th>\n",
       "      <td>591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chewbbaca</th>\n",
       "      <td>590</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biokit</th>\n",
       "      <td>589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libbigwig</th>\n",
       "      <td>589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqlogo</th>\n",
       "      <td>588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snpstats</th>\n",
       "      <td>588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gcrma</th>\n",
       "      <td>588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bubbletree</th>\n",
       "      <td>587</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beadarray</th>\n",
       "      <td>585</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hmmcopy</th>\n",
       "      <td>585</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aseb</th>\n",
       "      <td>584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snprelate</th>\n",
       "      <td>584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rbowtie</th>\n",
       "      <td>584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blasr_libcpp</th>\n",
       "      <td>584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqtools</th>\n",
       "      <td>583</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bumbershoot</th>\n",
       "      <td>581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-trace-abif</th>\n",
       "      <td>581</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fithic</th>\n",
       "      <td>580</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-roc</th>\n",
       "      <td>577</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-acgh</th>\n",
       "      <td>576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqpattern</th>\n",
       "      <td>576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metahdep</th>\n",
       "      <td>575</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gostats</th>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cntools</th>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hyphy</th>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pics</th>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dexus</th>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-acme</th>\n",
       "      <td>574</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sincell</th>\n",
       "      <td>573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clever-toolkit</th>\n",
       "      <td>573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chemminer</th>\n",
       "      <td>573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-amountain</th>\n",
       "      <td>573</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beaddatapackr</th>\n",
       "      <td>572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vega</th>\n",
       "      <td>572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plier</th>\n",
       "      <td>572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bicare</th>\n",
       "      <td>572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-twilight</th>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-adsplit</th>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plw</th>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-degseq</th>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-blima</th>\n",
       "      <td>571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flaimapper</th>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-simlr</th>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-switchbox</th>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xde</th>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylpipe</th>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aroma.light</th>\n",
       "      <td>570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metaarray</th>\n",
       "      <td>569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-simpleaffy</th>\n",
       "      <td>569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-messina</th>\n",
       "      <td>569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-streamer</th>\n",
       "      <td>568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>referenceseeker</th>\n",
       "      <td>568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bac</th>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cufflinks</th>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bhc</th>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-siggenes</th>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qrqc</th>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy_sequence_utils</th>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chromstar</th>\n",
       "      <td>567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flash</th>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multiscan</th>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtreemix</th>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>circexplorer</th>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netpathminer</th>\n",
       "      <td>565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bcrank</th>\n",
       "      <td>565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-triplex</th>\n",
       "      <td>565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rbioinf</th>\n",
       "      <td>563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>refinem</th>\n",
       "      <td>563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dnashaper</th>\n",
       "      <td>563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>illumina-interop</th>\n",
       "      <td>562</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sigpathway</th>\n",
       "      <td>562</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-minfi</th>\n",
       "      <td>562</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-interactionset</th>\n",
       "      <td>561</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ping</th>\n",
       "      <td>561</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bitseq</th>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-starr</th>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bgmix</th>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clark</th>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowai</th>\n",
       "      <td>560</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ternarynet</th>\n",
       "      <td>559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cancerinsilico</th>\n",
       "      <td>558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-transview</th>\n",
       "      <td>558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-profia</th>\n",
       "      <td>558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-string-diff</th>\n",
       "      <td>558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maanova</th>\n",
       "      <td>557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sagx</th>\n",
       "      <td>557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-birta</th>\n",
       "      <td>557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-isolde</th>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chromdraw</th>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scpipe</th>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgsa</th>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rama</th>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cancermutationanalysis</th>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-edda</th>\n",
       "      <td>556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oppar</th>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-smap</th>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-microrna</th>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylmnm</th>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-podkat</th>\n",
       "      <td>555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tigre</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-makecdfenv</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ebarrays</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rfpred</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-casper</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaseqmap</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tspair</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnanorm</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metaseq</th>\n",
       "      <td>554</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>igblast</th>\n",
       "      <td>553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomictuples</th>\n",
       "      <td>553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biosigner</th>\n",
       "      <td>552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>popgen-entropy</th>\n",
       "      <td>552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gvcf-regions</th>\n",
       "      <td>546</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xpclr</th>\n",
       "      <td>543</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mofapy</th>\n",
       "      <td>538</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbiogff</th>\n",
       "      <td>538</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libgtextutils</th>\n",
       "      <td>537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>expansionhunter</th>\n",
       "      <td>535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcontact2</th>\n",
       "      <td>531</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>covtobed</th>\n",
       "      <td>529</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lrez</th>\n",
       "      <td>527</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qdnaseq</th>\n",
       "      <td>523</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastani</th>\n",
       "      <td>523</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lofreq</th>\n",
       "      <td>521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rawtools</th>\n",
       "      <td>520</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-treeio</th>\n",
       "      <td>520</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pathogist</th>\n",
       "      <td>519</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dropletutils</th>\n",
       "      <td>517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>angsd</th>\n",
       "      <td>516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathview</th>\n",
       "      <td>516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cghcall</th>\n",
       "      <td>511</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mzid</th>\n",
       "      <td>510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-decipher</th>\n",
       "      <td>510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgat-apps</th>\n",
       "      <td>509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>idr</th>\n",
       "      <td>509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chromvar</th>\n",
       "      <td>506</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-all</th>\n",
       "      <td>504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plncpro</th>\n",
       "      <td>504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>edlib</th>\n",
       "      <td>503</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bamsignals</th>\n",
       "      <td>503</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>evidencemodeler</th>\n",
       "      <td>502</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clustalo</th>\n",
       "      <td>502</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-tailfindr</th>\n",
       "      <td>501</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sdrf-pipelines</th>\n",
       "      <td>500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioawk</th>\n",
       "      <td>500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyplm</th>\n",
       "      <td>499</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dammit</th>\n",
       "      <td>498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnaredprint</th>\n",
       "      <td>498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hpar</th>\n",
       "      <td>497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biothings_client</th>\n",
       "      <td>497</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rprotobuflib</th>\n",
       "      <td>496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metamaps</th>\n",
       "      <td>495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-sceasy</th>\n",
       "      <td>494</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>filtlong</th>\n",
       "      <td>494</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svtk</th>\n",
       "      <td>493</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-copynumber</th>\n",
       "      <td>491</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vkmz</th>\n",
       "      <td>490</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gagedata</th>\n",
       "      <td>489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ganon</th>\n",
       "      <td>489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pirate</th>\n",
       "      <td>489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cogent</th>\n",
       "      <td>489</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-birewire</th>\n",
       "      <td>488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pftools</th>\n",
       "      <td>488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-consensusclusterplus</th>\n",
       "      <td>488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beclear</th>\n",
       "      <td>488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seacr</th>\n",
       "      <td>487</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>psass</th>\n",
       "      <td>486</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgat-scripts</th>\n",
       "      <td>485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-survcomp</th>\n",
       "      <td>485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dss</th>\n",
       "      <td>485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dupradar</th>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fpa</th>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>askocli</th>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>albatradis</th>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>eggnog-mapper</th>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lfa</th>\n",
       "      <td>484</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-minet</th>\n",
       "      <td>483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gcatest</th>\n",
       "      <td>483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kebabs</th>\n",
       "      <td>483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stan</th>\n",
       "      <td>482</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-quaternaryprod</th>\n",
       "      <td>482</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sradb</th>\n",
       "      <td>481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ichip</th>\n",
       "      <td>481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-isoformswitchanalyzer</th>\n",
       "      <td>481</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scde</th>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-junctionseq</th>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-harman</th>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hapfabia</th>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sc3</th>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-motifcounter</th>\n",
       "      <td>480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-risa</th>\n",
       "      <td>479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cdna_cupcake</th>\n",
       "      <td>479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>alignstats</th>\n",
       "      <td>479</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-abaenrichment</th>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vegamc</th>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hulk</th>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pqsfinder</th>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>desalt</th>\n",
       "      <td>478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>schicexplorer</th>\n",
       "      <td>477</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chemfp</th>\n",
       "      <td>477</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>extract-sv-reads</th>\n",
       "      <td>476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-isomirs</th>\n",
       "      <td>476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-findmyfriends</th>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chimera</th>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowcl</th>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>alignlib-lite</th>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cohcap</th>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crispresso</th>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-globalancova</th>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qpgraph</th>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-linc</th>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>daisysuite</th>\n",
       "      <td>474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oncosimulr</th>\n",
       "      <td>473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowmatch</th>\n",
       "      <td>473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-motifrg</th>\n",
       "      <td>473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cn.farms</th>\n",
       "      <td>472</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-funchip</th>\n",
       "      <td>472</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>garnet</th>\n",
       "      <td>472</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maigespack</th>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-contibait</th>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-crlmm</th>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qsea</th>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-proloc</th>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>enasearch</th>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bridge</th>\n",
       "      <td>471</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genie</th>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>irfinder</th>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cortexpy</th>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cosmiq</th>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fabia</th>\n",
       "      <td>470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-puma</th>\n",
       "      <td>469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnorode</th>\n",
       "      <td>469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqstats</th>\n",
       "      <td>469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-graphite</th>\n",
       "      <td>469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chopsticks</th>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bus</th>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hlaprofiler</th>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iseq</th>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pangolin</th>\n",
       "      <td>468</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-harshlight</th>\n",
       "      <td>467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bioqc</th>\n",
       "      <td>467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ippd</th>\n",
       "      <td>467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-quasr</th>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-m3d</th>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genogam</th>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-girafe</th>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pdinfobuilder</th>\n",
       "      <td>466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flagme</th>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnorfuzzy</th>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kart</th>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbio-rnaseq</th>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hem</th>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bufferedmatrix</th>\n",
       "      <td>465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sim</th>\n",
       "      <td>464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bufferedmatrixmethods</th>\n",
       "      <td>464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bitmapperbs</th>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dnabarcodes</th>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cmv</th>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diffustats</th>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-inversion</th>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cispath</th>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hibag</th>\n",
       "      <td>463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netreg</th>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-networkbma</th>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnvtools</th>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hopach</th>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-generxcluster</th>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mulcom</th>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vanillaice</th>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsreg</th>\n",
       "      <td>462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-medme</th>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genoset</th>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-trigger</th>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>platon</th>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocsingular</th>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-turbonorm</th>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-variantfiltering</th>\n",
       "      <td>461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wade</th>\n",
       "      <td>460</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-logitt</th>\n",
       "      <td>460</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.dr.eg.db</th>\n",
       "      <td>459</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-marray</th>\n",
       "      <td>453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>funannotate</th>\n",
       "      <td>452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>antismash</th>\n",
       "      <td>452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-scan</th>\n",
       "      <td>451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-tmae</th>\n",
       "      <td>451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomelake</th>\n",
       "      <td>448</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>recentrifuge</th>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cwltest</th>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metagenomeseq</th>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sparcc</th>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treeswift</th>\n",
       "      <td>445</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msaprobs</th>\n",
       "      <td>445</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rapifilt</th>\n",
       "      <td>444</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ihw</th>\n",
       "      <td>444</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gat</th>\n",
       "      <td>443</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mtb-snp-it</th>\n",
       "      <td>441</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tapestry</th>\n",
       "      <td>440</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fraggenescan</th>\n",
       "      <td>439</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-bcbiosinglecell</th>\n",
       "      <td>439</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>selectfasta</th>\n",
       "      <td>438</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>idba</th>\n",
       "      <td>432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ballgown</th>\n",
       "      <td>432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>arriba</th>\n",
       "      <td>432</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dart</th>\n",
       "      <td>431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>avro-python2</th>\n",
       "      <td>431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>intervene</th>\n",
       "      <td>429</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>check-sort-order</th>\n",
       "      <td>429</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bwameth</th>\n",
       "      <td>428</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>methplotlib</th>\n",
       "      <td>427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hapcut2</th>\n",
       "      <td>427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>isatools</th>\n",
       "      <td>424</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-edaseq</th>\n",
       "      <td>422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cghbase</th>\n",
       "      <td>422</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tgsgapcloser</th>\n",
       "      <td>421</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ataqv</th>\n",
       "      <td>420</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.rn.eg.db</th>\n",
       "      <td>420</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oligoclasses</th>\n",
       "      <td>417</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sfld</th>\n",
       "      <td>417</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lastz</th>\n",
       "      <td>416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>avro-python3</th>\n",
       "      <td>416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.ce.eg.db</th>\n",
       "      <td>415</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowviz</th>\n",
       "      <td>411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cobra</th>\n",
       "      <td>411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ggcyto</th>\n",
       "      <td>410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hap.py</th>\n",
       "      <td>409</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bracken</th>\n",
       "      <td>407</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graphmap</th>\n",
       "      <td>407</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genenetworkbuilder</th>\n",
       "      <td>406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-do.db</th>\n",
       "      <td>405</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.gg.eg.db</th>\n",
       "      <td>404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>amptk</th>\n",
       "      <td>402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>art</th>\n",
       "      <td>402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taxsbp</th>\n",
       "      <td>402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mergenotcombined</th>\n",
       "      <td>401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fasta3</th>\n",
       "      <td>401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gage</th>\n",
       "      <td>401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>control-freec</th>\n",
       "      <td>401</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tag</th>\n",
       "      <td>400</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>microhapdb</th>\n",
       "      <td>400</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipqc</th>\n",
       "      <td>398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqsero2</th>\n",
       "      <td>398</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cdbtools</th>\n",
       "      <td>397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.ss.eg.db</th>\n",
       "      <td>397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanocompore</th>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcaexplorer</th>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genie3</th>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>coprarna</th>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>coverm</th>\n",
       "      <td>396</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unmerge</th>\n",
       "      <td>395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clust</th>\n",
       "      <td>395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastqpuri</th>\n",
       "      <td>395</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>strainseeker</th>\n",
       "      <td>394</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ipyrad</th>\n",
       "      <td>393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nucpos</th>\n",
       "      <td>393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.bt.eg.db</th>\n",
       "      <td>392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-isva</th>\n",
       "      <td>392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bamutil</th>\n",
       "      <td>391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bionet</th>\n",
       "      <td>391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-motifdb</th>\n",
       "      <td>391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-basic4cseq</th>\n",
       "      <td>391</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tbvcfreport</th>\n",
       "      <td>390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>piper</th>\n",
       "      <td>390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dnapi</th>\n",
       "      <td>389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pepquery</th>\n",
       "      <td>389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qdnaseq.mm10</th>\n",
       "      <td>389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chicago</th>\n",
       "      <td>389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>irida-sistr-results</th>\n",
       "      <td>388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-heatplus</th>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-motifstack</th>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fwdpp</th>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scmap-cli</th>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-linnorm</th>\n",
       "      <td>387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fourcseq</th>\n",
       "      <td>386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtandem</th>\n",
       "      <td>386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.cf.eg.db</th>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>any2fasta</th>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cexor</th>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sapfinder</th>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mitoode</th>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pchicdata</th>\n",
       "      <td>385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-alpine</th>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgalaxy</th>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cmfinder</th>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-and-furious</th>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaseqpower</th>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gothic</th>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.celegans.ucsc.ce11.refgene</th>\n",
       "      <td>384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cager</th>\n",
       "      <td>383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bio-vcf</th>\n",
       "      <td>383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-medips</th>\n",
       "      <td>383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowdensity</th>\n",
       "      <td>383</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-piano</th>\n",
       "      <td>382</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcftools-snvphyl-plugin</th>\n",
       "      <td>382</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dimspy</th>\n",
       "      <td>382</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fadapa</th>\n",
       "      <td>382</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-cp4p</th>\n",
       "      <td>381</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-icare</th>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>codingquarry</th>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>htsbox</th>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-esatac</th>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arrayexpresshts</th>\n",
       "      <td>380</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bustools</th>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.rnorvegicus.ucsc.rn5.refgene</th>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-made4</th>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>firecloud</th>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tofsims</th>\n",
       "      <td>379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genebreak</th>\n",
       "      <td>378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-outrider</th>\n",
       "      <td>378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maaslin2</th>\n",
       "      <td>378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-derfinderhelper</th>\n",
       "      <td>378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mimosa</th>\n",
       "      <td>377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tandem-genotypes</th>\n",
       "      <td>377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kcsmart</th>\n",
       "      <td>377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-slalom</th>\n",
       "      <td>377</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ksw</th>\n",
       "      <td>376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phate</th>\n",
       "      <td>376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-samspectral</th>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-targetsearch</th>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnits</th>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-quantsmooth</th>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-synergyfinder</th>\n",
       "      <td>375</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metams</th>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>carna</th>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgpbigwig</th>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-probamr</th>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mutationalpatterns</th>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mfa</th>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-motifbreakr</th>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-roar</th>\n",
       "      <td>374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>astalavista</th>\n",
       "      <td>373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ucsc.hg38</th>\n",
       "      <td>373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>downpore</th>\n",
       "      <td>373</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tarseqqc</th>\n",
       "      <td>372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pgsea</th>\n",
       "      <td>372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>intermine</th>\n",
       "      <td>370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnadecay</th>\n",
       "      <td>369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plasmidfinder</th>\n",
       "      <td>369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methimpute</th>\n",
       "      <td>369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnordt</th>\n",
       "      <td>369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geecc</th>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mait</th>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rhdf5client</th>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-simat</th>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msnid</th>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lefse</th>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-discordant</th>\n",
       "      <td>368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rdisop</th>\n",
       "      <td>367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genextender</th>\n",
       "      <td>367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cpat</th>\n",
       "      <td>367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-enrichplot</th>\n",
       "      <td>367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-xgr</th>\n",
       "      <td>366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-dnet</th>\n",
       "      <td>366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>artemis</th>\n",
       "      <td>364</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomicconsensus</th>\n",
       "      <td>364</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clusterexperiment</th>\n",
       "      <td>363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pynnotator</th>\n",
       "      <td>363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ete2</th>\n",
       "      <td>361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bwakit</th>\n",
       "      <td>361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyrodigal</th>\n",
       "      <td>360</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastobo</th>\n",
       "      <td>359</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-imputelcmd</th>\n",
       "      <td>357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hhsuite</th>\n",
       "      <td>357</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chromhmm</th>\n",
       "      <td>356</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ucsc.hg19</th>\n",
       "      <td>355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tetyper</th>\n",
       "      <td>353</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-gmwt</th>\n",
       "      <td>348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-genomictools</th>\n",
       "      <td>348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>beagle</th>\n",
       "      <td>348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyaavf</th>\n",
       "      <td>348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lace</th>\n",
       "      <td>346</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastool</th>\n",
       "      <td>345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>links</th>\n",
       "      <td>344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>collectl</th>\n",
       "      <td>343</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyomero-upload</th>\n",
       "      <td>343</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>corset</th>\n",
       "      <td>343</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotationfuncs</th>\n",
       "      <td>341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>goalign</th>\n",
       "      <td>340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-psicquic</th>\n",
       "      <td>337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cat</th>\n",
       "      <td>336</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.hsapiens.ucsc.hg19.knowngene</th>\n",
       "      <td>336</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>paraclu</th>\n",
       "      <td>336</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seq2science</th>\n",
       "      <td>335</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bamread</th>\n",
       "      <td>334</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mnnpy</th>\n",
       "      <td>333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scpred-cli</th>\n",
       "      <td>333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>microhapulator</th>\n",
       "      <td>332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lighter</th>\n",
       "      <td>332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cancerit-allelecount</th>\n",
       "      <td>332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-bioformats</th>\n",
       "      <td>331</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seurat-scripts</th>\n",
       "      <td>331</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kaptive</th>\n",
       "      <td>330</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-reactome.db</th>\n",
       "      <td>330</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>emperor</th>\n",
       "      <td>329</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quantwiz-iq</th>\n",
       "      <td>328</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tigmint</th>\n",
       "      <td>328</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genesplicer</th>\n",
       "      <td>328</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-garnett</th>\n",
       "      <td>328</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconvert</th>\n",
       "      <td>326</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rilseq</th>\n",
       "      <td>326</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bayseq</th>\n",
       "      <td>325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>aria2</th>\n",
       "      <td>324</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wbuild</th>\n",
       "      <td>324</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylumi</th>\n",
       "      <td>322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>iced</th>\n",
       "      <td>322</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-suprahex</th>\n",
       "      <td>321</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gap2seq</th>\n",
       "      <td>320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chippeakanno</th>\n",
       "      <td>320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ascat</th>\n",
       "      <td>319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fiji</th>\n",
       "      <td>318</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flexbar</th>\n",
       "      <td>317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gfapy</th>\n",
       "      <td>317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-recount</th>\n",
       "      <td>317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet</th>\n",
       "      <td>316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>coinfinder</th>\n",
       "      <td>316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gvcfgenotyper</th>\n",
       "      <td>314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treerecs</th>\n",
       "      <td>314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fusion-inspector</th>\n",
       "      <td>314</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-splots</th>\n",
       "      <td>313</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>idr-py</th>\n",
       "      <td>312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aldex2</th>\n",
       "      <td>312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgat-report</th>\n",
       "      <td>312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scanorama</th>\n",
       "      <td>312</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sushi</th>\n",
       "      <td>311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>linkage2allegro</th>\n",
       "      <td>311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rappas</th>\n",
       "      <td>310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastspar</th>\n",
       "      <td>310</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xmatchview</th>\n",
       "      <td>309</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>discosnp</th>\n",
       "      <td>309</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msa</th>\n",
       "      <td>309</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cramtools</th>\n",
       "      <td>308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bbknn</th>\n",
       "      <td>308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-opencyto</th>\n",
       "      <td>308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-destiny</th>\n",
       "      <td>307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dfast</th>\n",
       "      <td>307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genvisr</th>\n",
       "      <td>307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgreat</th>\n",
       "      <td>306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multidataset</th>\n",
       "      <td>306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stringdecomposer</th>\n",
       "      <td>306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>grid</th>\n",
       "      <td>306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>adam</th>\n",
       "      <td>305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>iva</th>\n",
       "      <td>305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-banocc</th>\n",
       "      <td>305</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hera</th>\n",
       "      <td>304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biodist</th>\n",
       "      <td>304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mast</th>\n",
       "      <td>304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-globaltest</th>\n",
       "      <td>303</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-splinter</th>\n",
       "      <td>303</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hicup</th>\n",
       "      <td>302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clairvoyante</th>\n",
       "      <td>302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bmfilter</th>\n",
       "      <td>302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rlmm</th>\n",
       "      <td>301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-asgsca</th>\n",
       "      <td>301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowclust</th>\n",
       "      <td>301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellhts2</th>\n",
       "      <td>301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-codex</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annaffy</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cqn</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qusage</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-soggi</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fishalyser</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tasmanian-mismatch</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>haystack_bio</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bmtool</th>\n",
       "      <td>300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-traser</th>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aims</th>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jamm</th>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-agilp</th>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arrayquality</th>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cage</th>\n",
       "      <td>299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-acde</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bmtagger</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>extract_fullseq</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgsepd</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fsnviz</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gcspikelite</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>besst</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomicscores</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-a4preproc</th>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rpx</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.rnorvegicus.ucsc.rn6.refgene</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sangerseqr</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-roleswitch</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rontotools</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-m3drop</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flapjack</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-quantro</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beat</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-antiprofiles</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metcirc</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bgafun</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chromplot</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-proper</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomicdatacommons</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cleaver</th>\n",
       "      <td>297</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-atacseqqc</th>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocworkflowtools</th>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ssize</th>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affycomp</th>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaseqcomp</th>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affydata</th>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rbm</th>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-asafe</th>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gatk-framework</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qubic</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-psea</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arraytools</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oscope</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mirtrace</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spia</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bbcanalyzer</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtopper</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-basespacer</th>\n",
       "      <td>295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-osat</th>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bgeedb</th>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-anota</th>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-absseq</th>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affycompatible</th>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-heatmaps</th>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beadarraysnp</th>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iwtomics</th>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ruvseq</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cghregions</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirnatap</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annmap</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clomial</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-a4core</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>daligner</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stringdb</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>refgenieserver</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rbsurv</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cand</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-causalr</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-segmentseq</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cfassay</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-subseq</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-uniprot.ws</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methped</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotationtools</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mantelcorr</th>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-badregionfinder</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snaptools</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qpcrnorm</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prokaryote</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snphood</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-savr</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-occugene</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clonality</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomicfiles</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-michip</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowutils</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-jaspar2016</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtrm</th>\n",
       "      <td>292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-compepitools</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-starank</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-randpack</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>admixtools</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-parody</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mdqc</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-myvariant</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomicinteractions</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-weaver</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mergeomics</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-viper</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqarray</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kiwidist</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-jackstraw</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcan</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-triform</th>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-timecourse</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-agdex</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netbiov</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>surpyvor</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-panp</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mbased</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowrepositoryr</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-macat</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cogena</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aspli</th>\n",
       "      <td>290</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-globalseq</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tscan</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sscu</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-safe</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ssviz</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyilm</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyrnadegradation</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-doppelgangr</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genocn</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyexpress</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dazz_db</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ipo</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-garfield</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lea</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bags</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ntw</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ppinfer</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-farms</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-slqpcr</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dudes</th>\n",
       "      <td>289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rpa</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nupop</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affycontam</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pvac</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-curatedovariandata</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-trio</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqgsea</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rebet</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cghmcr</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-outlierd</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastme</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgfm</th>\n",
       "      <td>288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aneufinder</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rbowtie2</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-simbindprofiles</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-derfinder</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-riboprofiling</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>andi</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rqc</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gosim</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hlama</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pepxmltab</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biseq</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chronos</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scmap</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcgatoolbox</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-simd</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmrcaller</th>\n",
       "      <td>287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bnbc</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kneaddata</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transposcope</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-path2ppi</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-npgsea</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-teqc</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneselectmmd</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pviz</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-grohmm</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clipper</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-matrixrider</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cosnet</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rrho</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bdmmacorrect</th>\n",
       "      <td>286</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-varianttools</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netresponse</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-analysispageserver</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mlp</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirnapath</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diffhic</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cssp</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylseekr</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gecode</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>barriers</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mbttest</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multiomicsviz</th>\n",
       "      <td>285</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>assemblerflow</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-normr</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-topaseq</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gmapr</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fmcsr</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hdtd</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtrmui</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-transite</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>addrg</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-opossom</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-r3cseq</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-motifmatchr</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-copa</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rain</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rbiopaxparser</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellmapper</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sights</th>\n",
       "      <td>284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-universalmotif</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgin</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multimed</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mvcclass</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phenopath</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgfr</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mbamethyl</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metacca</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-psichomics</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-macorrplot</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sseq</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omiccircos</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>howdesbt</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgsea</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omicade4</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathnet</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arrayexpress</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deepmedic</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-somaticsignatures</th>\n",
       "      <td>283</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-massarray</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mpfe</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rimmport</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pickgene</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methtargetedngs</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcaspar</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirsynergy</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xmapbridge</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scan.upc</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spktools</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-runibic</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spem</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snpediar</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gtfparse</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sizepower</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-squadd</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-polyfit</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spikeli</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ibmq</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-orfik</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-similarpeak</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mdgsa</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nnnorm</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bcseq</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stepnorm</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-swathxtend</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ncigraph</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-muscle</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-basics</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plethy</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-switchde</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rols</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plgem</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vbmp</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-targetscore</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.drerio.ucsc.danrer10.refgene</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-semdist</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeastexpdata</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-baynorm</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-r3cpet</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtca</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pepstat</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sagenhaft</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirage</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-olin</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rdgidb</th>\n",
       "      <td>282</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-recoup</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mops</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pkgdeptools</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgraph2js</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-massir</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sscore</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-promise</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-semisup</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-riboseqr</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hicrep</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-swath2stats</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mipp</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ogsa</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xbseq</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaprobr</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mergemaid</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nucler</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crumble</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rrdp</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sgseq</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sispa</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-peca</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-specond</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rbcbook1</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-meshdbi</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-paircompviz</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rsvsim</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-otubase</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-jaspar2014</th>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-readqpcr</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowtype</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>eoulsan</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-r4rna</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-graphalignment</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nethet</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qualifier</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-wiggleplotr</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rmmquant</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stategra</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-narrowpeaks</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nucleosim</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nanostringdiff</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcot2</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-isogenegui</th>\n",
       "      <td>280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-edge</th>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-expressionview</th>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-refnet</th>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bprmeth</th>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-enrichedheatmap</th>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hubward</th>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biomvrcns</th>\n",
       "      <td>279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dxua</th>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ihwpaper</th>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ibbig</th>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bolt</th>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gosummaries</th>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-paa</th>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cityhash</th>\n",
       "      <td>278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-csar</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hireewas</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-martini</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netbenchmark</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fdrame</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-immunoclust</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cgen</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-coregnet</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipseqr</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fastqcleaner</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ramwas</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rmassbank</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneticsped</th>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-eir</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-r453plus1toolbox</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ebcoexpress</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqbias</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tilingarray</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-grenits</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cogaps</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-buscorrect</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rjmcmcnucleosomes</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gpart</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-srnadiff</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-profilescoredist</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rqubic</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-levi</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-h5vc</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-reb</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gopro</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-predictionet</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rchyoptimyx</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cydar</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-simulatorz</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-macpet</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scfind</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gofuncr</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-descan2</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cancerclass</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ccfindr</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-decomplexdisease</th>\n",
       "      <td>276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sc3-scripts</th>\n",
       "      <td>275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wes-service-client</th>\n",
       "      <td>275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>drep</th>\n",
       "      <td>275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-zinbwave</th>\n",
       "      <td>274</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sictools</th>\n",
       "      <td>273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-bisquerna</th>\n",
       "      <td>272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vembrane</th>\n",
       "      <td>271</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sigmut</th>\n",
       "      <td>268</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-multx</th>\n",
       "      <td>267</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>c-ares</th>\n",
       "      <td>267</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-reactomepa</th>\n",
       "      <td>266</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graphlan</th>\n",
       "      <td>261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-batchelor</th>\n",
       "      <td>259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>triqler</th>\n",
       "      <td>259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deformats</th>\n",
       "      <td>259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>illumina-utils</th>\n",
       "      <td>258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smaca</th>\n",
       "      <td>258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flash2</th>\n",
       "      <td>257</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmftools-purple</th>\n",
       "      <td>256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>npinv</th>\n",
       "      <td>256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>burrito</th>\n",
       "      <td>254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>smof</th>\n",
       "      <td>254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hclust2</th>\n",
       "      <td>254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ensembl-core</th>\n",
       "      <td>253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scaden</th>\n",
       "      <td>253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bam2fastx</th>\n",
       "      <td>250</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bio_assembly_refinement</th>\n",
       "      <td>250</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>assembly-stats</th>\n",
       "      <td>250</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.hsapiens.ucsc.hg38.knowngene</th>\n",
       "      <td>249</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kobas</th>\n",
       "      <td>249</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gblocks</th>\n",
       "      <td>247</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drimseq</th>\n",
       "      <td>245</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>burrito-fillings</th>\n",
       "      <td>244</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mango</th>\n",
       "      <td>244</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maftools</th>\n",
       "      <td>243</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqan3</th>\n",
       "      <td>242</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genrich</th>\n",
       "      <td>241</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-diffcorr</th>\n",
       "      <td>240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-crisprseek</th>\n",
       "      <td>240</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-glimma</th>\n",
       "      <td>239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy-ops</th>\n",
       "      <td>239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cytolib</th>\n",
       "      <td>239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>canvas</th>\n",
       "      <td>238</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fgmp</th>\n",
       "      <td>237</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylovega</th>\n",
       "      <td>236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>eigensoft</th>\n",
       "      <td>236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ragtag</th>\n",
       "      <td>236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgcloud-lib</th>\n",
       "      <td>234</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transcov</th>\n",
       "      <td>234</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ea-utils</th>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>break-point-inspector</th>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vsnp</th>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hts-nim-tools</th>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libgenome</th>\n",
       "      <td>233</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seq-to-first-iso</th>\n",
       "      <td>231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-prestor</th>\n",
       "      <td>231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>align_it</th>\n",
       "      <td>231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kegg.db</th>\n",
       "      <td>231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kodoja</th>\n",
       "      <td>230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>salmid</th>\n",
       "      <td>230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cma</th>\n",
       "      <td>228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biotradis</th>\n",
       "      <td>228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dropseq_tools</th>\n",
       "      <td>228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hypo</th>\n",
       "      <td>227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcalm</th>\n",
       "      <td>226</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcl2fastq-nextseq</th>\n",
       "      <td>225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-variancepartition</th>\n",
       "      <td>225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bandage</th>\n",
       "      <td>224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dsh-bio</th>\n",
       "      <td>224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>beast2</th>\n",
       "      <td>224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ectyper</th>\n",
       "      <td>223</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ctc</th>\n",
       "      <td>223</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>damageprofiler</th>\n",
       "      <td>222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensdb.hsapiens.v75</th>\n",
       "      <td>222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>htstream</th>\n",
       "      <td>221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgabiolinks</th>\n",
       "      <td>219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dsrc</th>\n",
       "      <td>219</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>req</th>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ultraheatmap</th>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libmuscle</th>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>export2graphlan</th>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-predicate</th>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensemblvep</th>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ensembl-funcgen</th>\n",
       "      <td>217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libmems</th>\n",
       "      <td>217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ficus</th>\n",
       "      <td>217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>illumiprocessor</th>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cytoml</th>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deepbinner</th>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mapgl</th>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-watermelon</th>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sarscov2summary</th>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lumi</th>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pygmes</th>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shapeshifter</th>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>abra2</th>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lordec</th>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gemma</th>\n",
       "      <td>215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gem3-mapper</th>\n",
       "      <td>214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>connor</th>\n",
       "      <td>214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-http-tiny</th>\n",
       "      <td>214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dyndoc</th>\n",
       "      <td>213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crimson</th>\n",
       "      <td>213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-microbiome</th>\n",
       "      <td>213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ant</th>\n",
       "      <td>213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-extractadaptersfastqc</th>\n",
       "      <td>212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneoverlap</th>\n",
       "      <td>212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multiassayexperiment</th>\n",
       "      <td>212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>glimmer</th>\n",
       "      <td>212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>basic</th>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-fastbaps</th>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ensembl-variation</th>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fred2</th>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>agfusion</th>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ensembl-io</th>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ensembl-compara</th>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bamscale</th>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-harmony</th>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-copyhelper</th>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mygene</th>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-padog</th>\n",
       "      <td>210</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scnorm</th>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geometadb</th>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bladderbatch</th>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clonalframeml</th>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graphprot</th>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ctat-mutations</th>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pgx-variant-tools</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-experimenthub</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastlmm</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.mmusculus.ucsc.mm10.ensgene</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.sc.sgd.db</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>express</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-synlet</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spotyping3</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dedup</th>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diffloop</th>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-impulsede</th>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-abadata</th>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-missmethyl</th>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>impute2</th>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-scpred</th>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bctools</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gadem</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanopolishcomp</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-egseadata</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>eden</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ctsge</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>je-suite</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clst</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hardklor</th>\n",
       "      <td>206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chipseq-greylist</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>magicblast</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmrcate</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biosvd</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sarscov2formatter</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-a4reporting</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clustcomp</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-brain</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arrmnormalization</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>krocus</th>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-basecallqc</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-eudysbiome</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htsfilter</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-splatter</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnbeads</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clstutils</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>afterqc</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chromstardata</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hla-la</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-golubesets</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ensembl-api</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hiccompare</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>find_differential_primers</th>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spotsegmentation</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirbaseversions.db</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>domclust</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arrayqualitymetrics</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>autodock-vina</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bpp-phyl</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rankprod</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cogps</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nondetects</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-consensusseeker</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arrmdata</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dama</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-copywriter</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dreamtools</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jaffa</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gewist</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-brainstars</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pfam.db</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clustersignificance</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cindex</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>idba_subasm</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcy3</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>berokka</th>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-unifiedwmwqpcr</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-correp</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hiannotator</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snm</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>faqcs</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tracktables</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deqms</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-a4base</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-help</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ms2pip</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lungcanceracvssccgeo</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cesm</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cohcapanno</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gmap-fusion</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lpe</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsca</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graftm</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-idiogram</th>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-batchqc</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bamhash</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipsim</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-inpower</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genega</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipxpressdata</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cap3</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lmdme</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gotools</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>circos-tools</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genbankr</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-epivizrserver</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-generecommender</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clusterstab</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genefu</th>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-inspect</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>primedrpa</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ipac</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ivar</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chromheatmap</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dbchip</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-consensusov</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-undo</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-generegionscan</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeastcc</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-reportingtools</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-encodexplorer</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clonotyper</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>capc-map</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dropest</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-codelink</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dupchecker</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mapredictdsc</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-swimr</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.dmelanogaster.ucsc.dm6.ensgene</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotationhubdata</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmrcatedata</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dbg2olc</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>json_collect_data_source</th>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-convert</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-director</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ebsea</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drivernet</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msp2db</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-philr</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-covrna</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kegglincs</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-panr</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cancersubtypes</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hg-color</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kimod</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-imas</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>age-metasv</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-browserviz</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-egsea</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneattribution</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>adapterremovalfixprefix</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-demand</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-shinymethyl</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-procoil</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genrank</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ggbase</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-classifyr</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seq2pathway</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomeintervals</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnbeads.hg19</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-trackviewer</th>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-keggdzpathwaysgeo</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathifier</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chimeraviz</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lola</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bpp-core</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fermi2</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-zfpkm</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneplast</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowbeads</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affycoretools</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ghostz</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-grmetrics</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnbeads.mm10</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drugvsdiseasedata</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bpp-seq</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pandar</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-easyrnaseq</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diffgeneanalysis</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spidermir</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maaslin</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-crisprvariants</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plpe</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneticsdesign</th>\n",
       "      <td>198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tvtb</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>circularmapper</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ndexr</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastqsplitter</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-apcomplex</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-refplus</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-empiricalbrownsmethod</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pwmenrich</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>geodl</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ggtools</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cmap2data</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genmap</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>frogs</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scisi</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowploidy</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-jaspar2018</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genemeta</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dta</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dyebias</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pvca</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>guidescan</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-helloranges</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cleanupdtseq</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gdsctools</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-eximir</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnvpanelizer</th>\n",
       "      <td>197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ppistats</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ghm</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gretel</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ruvnormalizedata</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hits</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>imfusion</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ruvnormalize</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mlinterfaces</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lpnet</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnvgsa</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>islandpath</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ffpe</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-interminer</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-amplican</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-focalcall</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dfp</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-epivizrdata</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowclean</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biomvcclass</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-guideseq</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cpvsnp</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cormotif</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gcrisprtools</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cghnormaliter</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>consensusfixer</th>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-translatome</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yaqcaffy</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-crimage</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stemhypoxia</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lorikeet</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-psygenet2r</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-crisprseekplus</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathostat</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genetclassifier</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-madseq</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yarn</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-reder</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>arcs</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-raggedexperiment</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dia_umpire</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gofunction</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-meshes</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-svaplsseq</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-epivizrstandalone</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deepbluer</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-orderedlist</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spacepac</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hitc</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-baalchip</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-icens</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-keggorthology</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htqpcr</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylorank</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gmrp</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mlseq</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>eagle</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drugvsdisease</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-uniquorn</th>\n",
       "      <td>195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocgraph</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gb_taxonomy_tools</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multiclust</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ideal</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-altcdfenvs</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>funnel</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fccac</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-curatedmetagenomicdata</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seq2pathway.data</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maskrc-svg</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ddct</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-simpintlists</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rarevariantvis</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbio-variation</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ale</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dcgsa</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ecopy</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-isogene</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>edd</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-webbioc</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iterativebma</th>\n",
       "      <td>194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gaprediction</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lordfast</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pwomics</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>athena_meta</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqcna</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-frma</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-epivizr</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mwastools</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipxpress</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-vcfstats</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-frmatools</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgug4122a.db</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>combined-pvalues</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gtrellis</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcgh</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-degraph</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graph_embed</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>disco</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cwl2wdl</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methyanalysis</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowsorted.blood.450k</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-regsplice</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prebs</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>akt</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-epigenomix</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-preda</th>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-transcriptr</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>eqtlbma</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stoatydive</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-esetvis</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-graphpac</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hyperdraw</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dchiprep</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>recognizer</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsalightning</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbio_monitor</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.mrna</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-les</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phastcons100way.ucsc.hg19</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phosphonormalizer</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fis</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lambda</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lbe</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pmm</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathprint</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geosubmission</th>\n",
       "      <td>192</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-graphat</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kodata</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yak</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fmlrc</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-countclust</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnainteract</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>drive</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-smite</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>discasm</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>elector</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sigfuge</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-splinetimer</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iaseq</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-derfinderplot</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kmasker</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-amaretto</th>\n",
       "      <td>191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-wavcluster</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chanjo</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>xbioc</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>embl-api-validator</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metabosignal</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-icobra</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ibh</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fido</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tdaracne</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iterativebmasurv</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hotspot3d</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fuma</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bis-snp</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scmerge</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tin</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nadfinder</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsealm</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcc</th>\n",
       "      <td>190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-starbiotrek</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hybridmtest</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snagee</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snageedata</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prize</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-process</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-srap</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libstatgen</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mbkmeans</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qcmetrics</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yapsa</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gpls</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-interest</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genphen</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ioniser</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ideoviz</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hypergraph</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathvar</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fast5seek</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hapbin</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rpsixml</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowmap</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lpeadj</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>asqcan</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>allegro</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-liquidassociation</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnbeads.rn5</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnbeads.mm9</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rmir</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mmdiff2</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-minimumdistance</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>btrim</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rseqan</th>\n",
       "      <td>189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ppidata</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-splicinggraphs</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcpheno</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>feht</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-meigor</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-adacgh2</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>novasplice</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bandits</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genblastg</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bgt</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-measurementerror.cor</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.db</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-quartpac</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lapmix</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirintegrator</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ivas</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-depecher</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-atsnp</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirsponger</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sigar</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathrender</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>diamond_add_taxonomy</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-slgi</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirnameconverter</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaither</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsri</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lcfit</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lmgene</th>\n",
       "      <td>188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flock</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taxopy</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>icqsol</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-specl</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dehomopolymerate</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maskbad</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genegeneinter</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-gd-svg</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bxtools</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cap-mirseq</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rariant</th>\n",
       "      <td>187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-adam</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dextractor</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phenstat</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ebisearch</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirsm</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ctat-lncrna</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genblasta</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>connectome-workbench</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmmer2</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-imagehts</th>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cnvetti</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dascrubber</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dbgraph</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cola</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-glad</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-interactivedisplay</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-graper</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>agg</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-manor</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biolite</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metaneighbor</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>centroid_rna_package</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qckitfastq</th>\n",
       "      <td>185</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gnet2</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-demixt</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>macrel</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oveseg</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sracipe</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sitepath</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snapcgh</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-celda</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bloomfiltertrie</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rhisat2</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-smad</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bgx</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-structstrings</th>\n",
       "      <td>184</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mbg</th>\n",
       "      <td>183</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netboost</th>\n",
       "      <td>183</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rpf-count-cds</th>\n",
       "      <td>182</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pybiomart</th>\n",
       "      <td>181</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cas-offinder</th>\n",
       "      <td>179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>topas</th>\n",
       "      <td>178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>medusa</th>\n",
       "      <td>178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-obogaf-parser</th>\n",
       "      <td>177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rnabloom</th>\n",
       "      <td>177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.mmusculus.ucsc.mm9.knowngene</th>\n",
       "      <td>177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-lockfile-simple</th>\n",
       "      <td>176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-annotatr</th>\n",
       "      <td>175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snver</th>\n",
       "      <td>175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ericscript</th>\n",
       "      <td>175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-rest-client</th>\n",
       "      <td>175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-htscluster</th>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmusculus.ucsc.mm10</th>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mpa-server</th>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>helperlibs</th>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ensembl-genomes</th>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perbase</th>\n",
       "      <td>173</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmrseq</th>\n",
       "      <td>172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>das_tool</th>\n",
       "      <td>172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dlcpar</th>\n",
       "      <td>172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.mmusculus.ucsc.mm10.knowngene</th>\n",
       "      <td>171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-das</th>\n",
       "      <td>171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-util-properties</th>\n",
       "      <td>171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prinseq-plus-plus</th>\n",
       "      <td>170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bax2bam</th>\n",
       "      <td>169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mark-nonconverted-reads</th>\n",
       "      <td>168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sourcetracker</th>\n",
       "      <td>168</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rna-cd</th>\n",
       "      <td>166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scirpy</th>\n",
       "      <td>166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcbio-gff</th>\n",
       "      <td>166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dinamo</th>\n",
       "      <td>166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>braker2</th>\n",
       "      <td>165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nemo-age</th>\n",
       "      <td>165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dkfz-bias-filter</th>\n",
       "      <td>165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylation450kanno.ilmn12.hg19</th>\n",
       "      <td>164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>braker</th>\n",
       "      <td>163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bambamc</th>\n",
       "      <td>163</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-r4cker</th>\n",
       "      <td>162</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fast5-research</th>\n",
       "      <td>161</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocversion</th>\n",
       "      <td>160</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>itsx</th>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gff3sort</th>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libis</th>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shannon</th>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shapeshifter-cli</th>\n",
       "      <td>159</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kipoi</th>\n",
       "      <td>158</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>admixture</th>\n",
       "      <td>156</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-loomr</th>\n",
       "      <td>155</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rsidx</th>\n",
       "      <td>155</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ratt</th>\n",
       "      <td>153</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-email-simple</th>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cd-hit-auxtools</th>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kissplice</th>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fusioncatcher</th>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmusculus.ucsc.mm9</th>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-statvisual</th>\n",
       "      <td>151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minipolish</th>\n",
       "      <td>150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-metabolomics-fragment-annotation</th>\n",
       "      <td>150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>varda2-client</th>\n",
       "      <td>150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>binsanity</th>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmftools-amber</th>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hom.hs.inp.db</th>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmftools-cobalt</th>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snakefmt</th>\n",
       "      <td>147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>protmapper</th>\n",
       "      <td>147</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sanitizeme</th>\n",
       "      <td>146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kyototycoon</th>\n",
       "      <td>146</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>shigatyper</th>\n",
       "      <td>145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fishtaco</th>\n",
       "      <td>144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>confindr</th>\n",
       "      <td>144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pathracer</th>\n",
       "      <td>143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>purple-bio</th>\n",
       "      <td>143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>caper</th>\n",
       "      <td>142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cactus</th>\n",
       "      <td>142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pydp</th>\n",
       "      <td>142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gnu-wget</th>\n",
       "      <td>141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-homo.sapiens</th>\n",
       "      <td>141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>feelnc</th>\n",
       "      <td>141</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylation450kmanifest</th>\n",
       "      <td>140</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gotree</th>\n",
       "      <td>139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngsderive</th>\n",
       "      <td>139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ls-gkm</th>\n",
       "      <td>139</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>logomaker</th>\n",
       "      <td>138</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gcs-oauth2-boto-plugin</th>\n",
       "      <td>138</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kraken-biom</th>\n",
       "      <td>137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>croo</th>\n",
       "      <td>137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gmtk</th>\n",
       "      <td>137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scater-scripts</th>\n",
       "      <td>137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clustergrammer</th>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>whatsgnu</th>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stringmeup</th>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>igv-reports</th>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fdb.infiniummethylation.hg19</th>\n",
       "      <td>136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>openduck</th>\n",
       "      <td>135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>emirge</th>\n",
       "      <td>135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biolib</th>\n",
       "      <td>135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libidn</th>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>eta</th>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>negative_training_sampler</th>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>breakdancer</th>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>arb-bio</th>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fqtools</th>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pylca</th>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cannoli</th>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bsmap</th>\n",
       "      <td>132</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-tools</th>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>amos</th>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-join</th>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tendo</th>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.hsapiens.ucsc.hg18.knowngene</th>\n",
       "      <td>131</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133plus2.db</th>\n",
       "      <td>130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kmergenie</th>\n",
       "      <td>130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene</th>\n",
       "      <td>129</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pepgenome</th>\n",
       "      <td>129</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genometools</th>\n",
       "      <td>129</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hypercluster</th>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deap</th>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blockclust</th>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylationepicanno.ilm10b4.hg19</th>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tagore</th>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-ms</th>\n",
       "      <td>127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bitstring</th>\n",
       "      <td>127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gatktool</th>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pytaxonkit</th>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fitter</th>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.celegans.ucsc.ce6.ensgene</th>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dialign-tx</th>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133a.db</th>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>demuxlet</th>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unfazed</th>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>umap</th>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mobidic-mpa</th>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene</th>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-lyve-set</th>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dwgsim</th>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>livekraken</th>\n",
       "      <td>125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metab</th>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hgtools</th>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metalign</th>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cytoscape</th>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>srax</th>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylationepicmanifest</th>\n",
       "      <td>123</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fasta_ushuffle</th>\n",
       "      <td>122</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lz4-bin</th>\n",
       "      <td>122</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gatb</th>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pybda</th>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crux-toolkit</th>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dsk</th>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomethreader</th>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mimseq</th>\n",
       "      <td>120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scvis</th>\n",
       "      <td>120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gprofiler-official</th>\n",
       "      <td>120</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyclone</th>\n",
       "      <td>119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>compare-reads</th>\n",
       "      <td>119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gembs</th>\n",
       "      <td>119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-customprodb</th>\n",
       "      <td>119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kalign2</th>\n",
       "      <td>118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fusioncatcher-seqtk</th>\n",
       "      <td>118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>faststructure</th>\n",
       "      <td>118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pmmrcalculator</th>\n",
       "      <td>118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blobtools</th>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-slingshot</th>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kmerinshort</th>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graphlite</th>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gvcftools</th>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clair</th>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gwastools</th>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>intervaltree_bio</th>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ecmwfapi</th>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-pair</th>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-cg-pipeline</th>\n",
       "      <td>116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>harvesttools</th>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ghostx</th>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ecoli.ncbi.20080805</th>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>colorbrewer</th>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_analysis</th>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gfold</th>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>autolog</th>\n",
       "      <td>115</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chorus2</th>\n",
       "      <td>114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastuniq</th>\n",
       "      <td>114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.dmelanogaster.ucsc.dm3</th>\n",
       "      <td>114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hs-blastn</th>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sepp-refgg138</th>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libdivsufsort</th>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-matrix</th>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gnu-getopt</th>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>interleafq</th>\n",
       "      <td>113</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bpipe</th>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>embassy-phylip</th>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mspuritydata</th>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_common</th>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_md</th>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crb-blast</th>\n",
       "      <td>112</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylationepicanno.ilm10b2.hg19</th>\n",
       "      <td>111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scrnaseq</th>\n",
       "      <td>111</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sampei</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgview</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ucsc.hg18</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ymp</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy-parsec</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp144.grch38</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminamousev2.db</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioformats</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomicassertions</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>group_humann2_uniref_abundances_to_go</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genericrepeatfinder</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>isescan</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fermi</th>\n",
       "      <td>110</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-firebrowser</th>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lohhla</th>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blastalign</th>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>footprint</th>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>assemblyutility</th>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gfinisher</th>\n",
       "      <td>109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-elmer</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-anonymous</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-panther.db</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flux-simulator</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fsm-lite</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gdc-client</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.celegans.ucsc.ce10</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>atactk</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ga4ghmongo</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>a5-miseq</th>\n",
       "      <td>108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-isee</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>avro-cwl</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmmcopy</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pretext-suite</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tinyfasta</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gnuplot-py</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>codonw</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95av2.db</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biolite-tools</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-sampleconfig</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-goenrichment</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>famsa</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ceas</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133a2.db</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genform</th>\n",
       "      <td>107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>illuminate</th>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>heinz</th>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.celegans.ucsc.ce6</th>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fishpond</th>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>justbackoff</th>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>marvel</th>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kwip</th>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>comparative-annotation-toolkit</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-validatefastq</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dcmtk</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crisper_recognition_tool</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcatools</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blockbuster</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>commet</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hichipper</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hops</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.drerio.ucsc.danrer7</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>forgi</th>\n",
       "      <td>105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ibdne</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blastbesties</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>anise_basil</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>isocor</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bfc</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy-util</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cistrome_beta</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genonets</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-targetscan.hs.eg.db</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>afplot</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lorma</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tissueenrich</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lumihumanall.db</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-busparse</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bamm</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>azure-cli</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-elmer.data</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clove</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bwapy</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>epicseg</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>assembly-scan</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bis-snp-utils</th>\n",
       "      <td>104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hivtrace</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>catch</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ecoprimers</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ecopcr</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fusion-filter</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chicagotools</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy-ie-helpers</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ghmm</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>halla</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>colombo</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.scerevisiae.ucsc.saccer3.sgdgene</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kronik</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kfoots</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ig-checkflowtypes</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>raiss</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>irep</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylationepicanno.ilm10b3.hg19</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lobstahs</th>\n",
       "      <td>103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libssw</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>curesim</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>barrnap-python</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_io</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jvarkit-bam2wig</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gff3toembl</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ispcr</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fasta-splitter</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>icount</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metavizr</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chromosomer</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bs-seeker2</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yamss</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prolocgui</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-faahko</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>peaksql</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastqe</th>\n",
       "      <td>102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rmir.hs.mirna</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>discovar</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>emmix</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cmap</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>esmre</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>taxonomy_ranks</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bayescan</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cafe</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>egglib</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wisestork</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gbmunge</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>upimapi</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastqmetrics</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastutils</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>desman</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>parent-map</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>uvp</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cistrome-ceas</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hanselx</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>simplesam</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.scerevisiae.ucsc.saccer2.sgdgene</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lsd</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kraken-ea</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genometester4</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lightning</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biogridpy</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kpal</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hcluster_sg</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mineica</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hubward-all</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ibdseq</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>prophane</th>\n",
       "      <td>101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>apoc</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-seqstat</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mirfix</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcool</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_model</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gqt</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gtf_to_genes</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genesis</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bs_call</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hapflk</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hurry.filesize</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bufet</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libbambamc</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>backspinpy</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>detonate</th>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>arem</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-enrichmentbrowser</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bamclipper</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gottcha</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>curlywhirly</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>csb</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>erne</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>extract_genome_region</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>macaron</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.hsapiens.ucsc.hg19.lincrnastranscripts</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>easel</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>karect</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2pg_cartesian</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genologics</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>data_hacks</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pylprotpredictor</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fseq</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dfam</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deltabs</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>latentstrainanalysis</th>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastphylo</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>funcannot</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>axiome</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>consan</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dicey</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msmseda</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>alder</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iclusterplus</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lrzip</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clinod</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biskit</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>edena</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clearcut</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dnp-diprofile</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>forked-path</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>drax</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>edittag</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-synapter</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cansnper</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>imseq</th>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ldhelmet</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>exparna</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gustaf</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>koeken</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dawg</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>damidseq_pipeline</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcftools-gtc2vcf-plugin</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>compalignp</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bgreat</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>haploclique</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>generax</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fwdpy</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fgap</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ideas</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastaindex</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fsa</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gotohscan</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>famseq</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirnatap.db</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>finestructure</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-ontologyplot</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-proteomicsannotationhubdata</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fermikit</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaseqsamplesizedata</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomestrip</th>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>centrosome</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>estscan</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>estmapper</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kinsimriboswitch</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dupre</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fqtrim</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ebfilter</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>booster</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bicseq2-seg</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kid</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-singler</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyasr</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>intervalstats</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>amused</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dashing</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clipandmerge</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>inforna</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hicbrowser</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hiddendomains</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cpinsim</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>disty</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>libsbml</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ma</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cofold</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cortex_con</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>iclipro</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gocr</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hr2</th>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>asn2gb</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crass</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>baitfisher</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>indelfixer</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mace</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lsc</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bayestyper</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dialign2</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>logol</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>atac</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bellmans-gapc</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bicseq2-norm</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>intemap</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>haploconduct</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dnp-fourier</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cassiopee</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deploid</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomebaser</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cocoscore</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>groopm</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>falco</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>brockman-pipeline</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bloocoo</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>frc</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fann</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gerp</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ccat</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biomaj</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fermi-lite</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fineradstructure</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genepender</th>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>merqury</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>graph2pro-var</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bpp-popgen</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>modlamp</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-ddir</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mapkl</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ac-diamond</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ig-checkfcs</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>breakseq2</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>antarna</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fcp</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>debwt</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biodiff</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>evofold2</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cmph</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jvarkit-bam2svg</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-signaturesearch</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>abawaca</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lyner</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hilive2</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dnp-corrprofile</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>carnac-lr</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>illumina-cleanup</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genclust</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gdc</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bindash</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>erds</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fasten</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dnp-binstrings</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>batvi</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lca</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>epydoc</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kssd</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>calib</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cansam</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>effectivet3</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-awfisher</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gwama</th>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ir</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>elasticluster</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>readucks</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastahack</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastindep</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gencore</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jali</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genomeconstellation</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gapfiller</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastsimbac</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>igor_vdj</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-count</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fqzcomp</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-metaboanalyst</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>igphyml</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastml</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-dimsum</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>k-slam</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bcov</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dig2</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lap</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>damasker</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-brendadb</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>brass</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>esimsa</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>encode-blacklist</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ditasic</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bibliospec</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>denovogear</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ctat-metagenomics</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>divvier</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dammet</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>consel</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>apt-probeset-summarize</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>comparems2</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>knot</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lightstringgraph</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lightassembler</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>amap</th>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>conus</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>arvados-cli</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>count_constant_sites</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>collect_mgf</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scgps</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-waddr</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ncgtw</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trtools</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowspy</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sctgif</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cfm</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sojourner</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sharedobject</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>e-pcr</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-saigegds</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>defiant</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sherpas</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cryfa</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bwise</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hilda</th>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iggeneusage</th>\n",
       "      <td>91</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>python-rnacentral</th>\n",
       "      <td>90</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mpra-data-access-portal</th>\n",
       "      <td>90</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>music-deconvolution</th>\n",
       "      <td>88</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genefuse</th>\n",
       "      <td>88</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scvi-tools</th>\n",
       "      <td>87</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tximport-scripts</th>\n",
       "      <td>87</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scvis_galaxy</th>\n",
       "      <td>86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nebulizer</th>\n",
       "      <td>86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>opentargets</th>\n",
       "      <td>86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-intlim</th>\n",
       "      <td>84</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clan</th>\n",
       "      <td>83</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transposonpsi</th>\n",
       "      <td>82</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phylogenetics</th>\n",
       "      <td>81</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>seqscreen</th>\n",
       "      <td>80</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-math-matrixreal</th>\n",
       "      <td>80</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>riboseqc</th>\n",
       "      <td>79</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>singlem</th>\n",
       "      <td>79</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plyranges</th>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>insilicoseq</th>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>tbox-scan</th>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-tftargets</th>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>panpasco</th>\n",
       "      <td>76</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quasirecomb</th>\n",
       "      <td>76</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-enhancedvolcano</th>\n",
       "      <td>75</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>traitar</th>\n",
       "      <td>75</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mtglink</th>\n",
       "      <td>75</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>msaconverter</th>\n",
       "      <td>74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-data-lock</th>\n",
       "      <td>74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>targetdb</th>\n",
       "      <td>74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-mgsz</th>\n",
       "      <td>71</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>transindel</th>\n",
       "      <td>69</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-singlecellnet</th>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>oncogemini</th>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pubmlst_client</th>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>oncopipe</th>\n",
       "      <td>67</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>trycycler</th>\n",
       "      <td>67</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-rtassel</th>\n",
       "      <td>67</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lusstr</th>\n",
       "      <td>66</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mp3treesim</th>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>optimir</th>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_structure_checking</th>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>plasclass</th>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scapp</th>\n",
       "      <td>64</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svim-asm</th>\n",
       "      <td>63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pafpy</th>\n",
       "      <td>63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vcf2variants</th>\n",
       "      <td>63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bohra</th>\n",
       "      <td>62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tximeta</th>\n",
       "      <td>62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>figtree</th>\n",
       "      <td>62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>wlogdate</th>\n",
       "      <td>61</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chira</th>\n",
       "      <td>61</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-regionreport</th>\n",
       "      <td>60</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>anndata2ri</th>\n",
       "      <td>60</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_structure_manager</th>\n",
       "      <td>60</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ncbitk</th>\n",
       "      <td>59</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>omero-py</th>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-mkmisc</th>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mavis</th>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pileuppy</th>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>popera</th>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>spliceai-wrapper</th>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rucs</th>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-txrevise</th>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>eval</th>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rmetl</th>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pgx-pipe-helper</th>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sepp-refsilva128</th>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phastaf</th>\n",
       "      <td>55</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>patholive</th>\n",
       "      <td>55</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-eacon</th>\n",
       "      <td>54</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>augur</th>\n",
       "      <td>53</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ltr_retriever</th>\n",
       "      <td>53</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>orfquant</th>\n",
       "      <td>52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmftools-linx</th>\n",
       "      <td>52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>digestiflow-demux</th>\n",
       "      <td>52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>arboreto</th>\n",
       "      <td>51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.mmu.eg.db</th>\n",
       "      <td>51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phyphy</th>\n",
       "      <td>51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ltr_finder</th>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pypgatk</th>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>polystest</th>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>haslr</th>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>neatms</th>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>haplotype-lso</th>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-structuralvariantannotation</th>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quantiprot</th>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deepbgc</th>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pydpi</th>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>necat</th>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pgdspider</th>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snpgenie</th>\n",
       "      <td>46</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jobtree</th>\n",
       "      <td>46</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>svjedi</th>\n",
       "      <td>45</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>luciphor2</th>\n",
       "      <td>45</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-if</th>\n",
       "      <td>44</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cmsip</th>\n",
       "      <td>44</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yapc</th>\n",
       "      <td>44</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>influx_si</th>\n",
       "      <td>43</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stager</th>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rgccacmd</th>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mosca</th>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scxa-plots</th>\n",
       "      <td>41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deeplc</th>\n",
       "      <td>41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bam2fasta</th>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-wrench</th>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>r-genomictools.filehandler</th>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kipoiseq</th>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensdb.hsapiens.v86</th>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.at.tair.db</th>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>agat</th>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>yamda</th>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>rkp</th>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>macse</th>\n",
       "      <td>37</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-loomexperiment</th>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-io-null</th>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metaprokka</th>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scran-cli</th>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>treecluster</th>\n",
       "      <td>35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mixomics</th>\n",
       "      <td>35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chromosight</th>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-yaml-pp</th>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowstats</th>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>quasildr</th>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>geco2</th>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jarvis</th>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sequnwinder</th>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>start-asap</th>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pmx_biobb</th>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>scagaire</th>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cvbio</th>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>virchip</th>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pygvcf2coverage</th>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pyrpipe</th>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>variantbreak</th>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tradeseq</th>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>minirmd</th>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowworkspacedata</th>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deeplift</th>\n",
       "      <td>30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-gfacs</th>\n",
       "      <td>30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-singscore</th>\n",
       "      <td>29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>niemads</th>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>perl-bio-phylo-cipres</th>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bel-resources</th>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nanomod</th>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flair</th>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>conterminator</th>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_chemistry</th>\n",
       "      <td>27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ngsep</th>\n",
       "      <td>26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.xl.eg.db</th>\n",
       "      <td>26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cromshell</th>\n",
       "      <td>26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>opentargets-urlzsource</th>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ssuissero</th>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_pmx</th>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>aquila_stlfr</th>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>locus_processing</th>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kipoi-utils</th>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kofamscan</th>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bactopia</th>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metapop</th>\n",
       "      <td>23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>apollo</th>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cigar</th>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fcsparser</th>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chromatiblock</th>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bigmelon</th>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>varifier</th>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>zga</th>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>opentargets-validator</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hca</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>metatree</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pan-plaster</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgecore</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>emblmygff3</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-decontam</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chiron</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mfuzz</th>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dca</th>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cll</th>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>altamisa</th>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>assemblytics</th>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chado-tools</th>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aucell</th>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-polyester</th>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pydownsampler</th>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phigaro</th>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rdavidwebservice</th>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hint</th>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-infercnv</th>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cicero</th>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.eck12.eg.db</th>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>irida-uploader</th>\n",
       "      <td>19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msstats</th>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>edta</th>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-widgettools</th>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sidr</th>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tkwidgets</th>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>changeo</th>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-vcffilter</th>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-erccdashboard</th>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-karyoploter</th>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>escher</th>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chexmix</th>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fithic</th>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dcplib</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cameo</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>enrichm</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jupyterngsplugin</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kleborate</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>psm_fragments</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>circle-map</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ac</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-champdata</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dropletutils-scripts</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobasehttptools</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.pf.plasmo.db</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kopt</th>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.cnv</th>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminadatatestfiles</th>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aracne.networks</th>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>imsindel</th>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>airr</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biotdg</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ismapper</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bcellviper</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gffmunger</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-compcoder</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beta7</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>coral</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mus.musculus</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy-tool-util</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>galaxy-containers</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cutesv</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ccmap</th>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-airway</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crisprbact</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cgat-daisy</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crispr_studio</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tximportdata</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cami-opal</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cromwell-tools</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qdnaseq.hg19</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>barcode_splitter</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>geosketch</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-champ</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>focus</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chicken.db</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcistarget</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bridgedbr</th>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gnomic</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.ecsakai.eg.db</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-anopheles.db0</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cytodx</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-abseqr</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hicap</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>maaslin2</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cormut</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-goprofiles</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-breastcancernki</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affymetrixdatatestfiles</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellscape</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aneufinderdata</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ccdata</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyhgu133a2expr</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>entrezpy</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dart</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-asset</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.pt.eg.db</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-singlecellexperiment-scripts</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanv4.db</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq-dl</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-desingle</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-caomicsv</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.ag.eg.db</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ggd</th>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-consensus</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-connectivitymap</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcagopromoter.rn.rn4</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cemitool</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blksheep</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-panelcn.mops</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chic.data</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ahcytobands</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ag.db</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-alsace</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chic</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-alternativesplicingevents.hg38</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tfutils</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene10sttranscriptcluster.db</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopython.convert</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bioassayr</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-catalyst</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chickencdf</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bigmemoryextras</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-elbow</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pasillabamsubset</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>goenrichment</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kipoi_veff</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-crcl18</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arabidopsis.db0</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-coveb</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>enabrowsertools</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ashkenazimsonchr21</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hellorangesdata</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dapar</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsar</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-basicstarrseq</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ccrepe</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.athaliana.tair.tair9</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clusterjudge</th>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.vvinifera.urgi.iggp12xv2</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133b.db</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fabiadata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chromswitch</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fargene</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bronchialil13</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rforproteomics</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fci</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.celegans.ucsc.ce11</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mageckflute</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-minfidata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-breastcancerunt</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fusion-report</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.mirnaseq</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmmratac</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>iucn_sim</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hugene.1.1.st.v1</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-assessorf</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133plus2cdf</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-assessorfdata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyhgu133aexpr</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-shinytandem</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cobindr</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-davidtiling</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-artms</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene11sttranscriptcluster.db</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dapardata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-anaquin</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylation27kanno.ilmn12.hg19</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipexoqualexample</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chickenprobe</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-abarray</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-emdomics</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-breastcancertransbig</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocancer</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ebseqhmm</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqplots</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genarise</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnbeads.hg38</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-impulsede2</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scruff</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-breakpointrdata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-brainimagerdata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>goslimmer</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deepac</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse4302barcodevecs</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.pancan12</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-a4</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-allelicimbalance</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gskb</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-confessdata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogsa</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affymoe4302expr</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ptroglodytes.ucsc.pantro3.masked</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-copdsexualdimorphism.data</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-epidish</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-frgepistasis</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.1000genomes.hs37d5</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.hsapiens.biomart.igis</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-breastcancerupp</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biotmledata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cardinalworkflows</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-curatedtcgadata</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>amas</th>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-blma</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylaid</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metamsdata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipseqspike</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgacrcmrna</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illumina450probevariants.db</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-blimatestingdata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.ptroglodytes.ucsc.pantro4.refgene</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-immunespacer</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biobroom</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-canine2.db</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-minfidataepic</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drosophila2probe</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-guitar</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyhgu133plus2expr</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bioccheck</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-celaref</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vulcandata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipanalyser</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mcseadata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.pto.eg.db</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcistarget.hg19.motifdbs.cisbponly.500bp</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mcsurvdata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgabiolinksgui.data</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133abarcodevecs</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-barley1probe</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biotmle</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsvadata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-allmll</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocor</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-celarefdata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ena-dl</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mogene.1.1.st.v1</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-branchpointer</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipenrich</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-attract</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bazam</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-breakpointr</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-singlecelltk</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-single.mtec.transcriptomes</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hicdatalymphoblast</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-targetscoredata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omicsprint</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hugene.1.0.st.v1</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bioccasestudies</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dep</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sevenbridges</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oncoscore</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-intramirexplorer</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-canine.db0</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.dmelanogaster.ucsc.dm3.masked</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneplast.data.string.v91</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomewidesnp6crlmm</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcxndata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.alyrata.jgi.v1</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.amellifera.beebase.assembly4</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bumhmm</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fcbf</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>centreseq</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsseqdata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drosophila2.db</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tfarm</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fem</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phantasus</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-agcdf</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ritan</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ath1.121501</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affyqcreport</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.syn.eg.db</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-asicsdata</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.tgondii.toxodb.7.0</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.vvinifera.urgi.iggp12xv0</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellity</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gep2pep</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylmix</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cocitestats</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-breastcancervdx</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-crossmeta</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95av2</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensdb.hsapiens.v79</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsubtilisprobe</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.rnorvegicus.ucsc.rn5</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.rnorvegicus.ucsc.rn4.masked</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-coseq</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arraymvout</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>card_trick</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tomatocdf</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.rnorvegicus.ucsc.rn4</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cocoa</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deconrnaseq</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowsorted.blood.epic</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-colonca</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hom.dm.inp.db</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diggit</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gdsarray</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drosophila2cdf</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-coverageview</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensdb.mmusculus.v79</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensdb.rnorvegicus.v75</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multimir</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-celltrails</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-copyneutralima</th>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133afrmavecs</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ampaffyexample</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ahensdbs</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgawgbsdata.hg19</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgautils</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-derfinderdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-alpinedata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-alternativesplicingevents.hg19</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drawproteins</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmrscan</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-anf</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-antiprofilesdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dominoeffect</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dream4</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-asics</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-agprobe</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp151.grch38</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tenxpbmcdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lumibarnes</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-agimicrorna</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tfea.chip</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-assign</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.mmulatta.ucsc.rhemac8.refgene</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.rppa</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.clinical</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bovine.db0</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bovineprobe</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-breastcancermainz</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cyp450cdf</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cycle</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnagilentdesign028282.db</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-curatedcrcdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beadarrayexampledata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse430a2frmavecs</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bayesknockdown</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affixcan</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-odseq</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qubicdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ath1121501cdf</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.aofficinalis.ncbi.v1</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jass_preprocessing</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.clariom.d.human</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ctrap</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-brgedata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gatingmldata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gigsea</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.clariom.s.mouse</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcxn</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.clariom.s.mouse.ht</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.athaliana.tair.04232008</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gigseadata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmulatta.ucsc.rhemac2</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-curatedbreastdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gispa</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.amellifera.ucsc.apimel2</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genestructuretools</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomationdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomes</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomewidesnp5crlmm</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathprintgeodata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fantom3and4cage</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rattus.norvegicus</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cottonprobe</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-curatedbladderdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moe430bprobe</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ucsc.hg19.masked</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mapscape</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.cfamiliaris.ucsc.canfam3</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>macaron</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rdrtoolbox</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ucsc.hg17.masked</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.clariom.s.human.ht</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ace</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ggallus.ucsc.galgal4.masked</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-wavetilingdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-igc</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocsklearn</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fission</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowvs</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rta.1.0</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ccl4</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.sscrofa.ucsc.susscr11</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chimphumanbraindata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnvrd2</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.vvinifera.urgi.iggp8x</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowcybar</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rice</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u133a</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clariomsmousehttranscriptcluster.db</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-canine2cdf</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cancerdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowchic</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cctutorial</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pga</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-celeganscdf</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-celltree</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clariomsrathttranscriptcluster.db</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ptroglodytes.ucsc.pantro6</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scbn</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowplots</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fibroeset</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-categorycompare</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cbaf</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fgnet</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fitcons.ucsc.hg19</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-compgo</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hugene.2.0.st</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmusculus.ucsc.mm10.masked</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gaia</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ceu1kg</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmusculus.ucsc.mm8</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-confess</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.moex.1.0.st.v1</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ceu1kgv</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-camthc</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ecoli.asv2</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cagefightr</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bearscc</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.sco.a32.eg.db</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pgca</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-difflogo</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtn</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biocpkgtools</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-damirseq</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dvddata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>genome_updater</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ceuhm3</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bifet</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-harmandata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqcat</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-worm.db0</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-onlinefdr</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fella</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-openprimer</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-adaptest</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylation27k.db</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-celegansprobe</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-citruscdf</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-canine2probe</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chemminedrugs</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.tguttata.ucsc.taegut2</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-estrogen</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellmapperdata</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-celegans.db</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-meshr</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-charge</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>atlas-metadata-validator</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fastliquidassociation</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipexoqual</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clariomdhumantranscriptcluster.db</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-canine.db</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.huex.1.0.st.v2</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp144.grch37</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp142.grch37</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-skewr</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-icheck</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.ssc.eg.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-barley1cdf</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tpp</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>counterr</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-fastqsplitter</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gaschyhs</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sigfeature</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylation27kmanifest</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-beadarrayusecases</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-harmonizedtcgadata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scsr</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-screcover</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human370v1ccrlmm</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.oni.eg.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-arraytv</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gwascat</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133acdf</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sponge</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowploidydata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mfuro.ucsc.musfur1</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gatefinder</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bioconcotk</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>collect-columns</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iterclust</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hmp16sdata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-timeseriesexperiment</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hipathia</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xtrasnplocs.hsapiens.dbsnp144.grch37</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affypara</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-alevinqc</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgabiolinksgui</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hicdatahumanimr90</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-targetscan.mm.eg.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133afrmavecs</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-appreci8r</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methyvim</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowtrans</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diffcyt</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-caninecdf</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cafe</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowsorted.dlpfc.450k</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowmeans</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.poplar</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mogene.1.0.st.v1</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methyvimdata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ptroglodytes.ucsc.pantro2.masked</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ptroglodytes.ucsc.pantro3</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mta.1.0</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-frmaexampledata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metnet</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mogene.2.1.st</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellscore</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fdb.ucsc.trnas</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ga4ghclient</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gaggle</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ucsc.hg38.masked</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metagene</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phastcons7way.ucsc.hg38</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellbaser</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsubtiliscdf</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowmerge</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bftools</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ccpromise</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-zebrafishrnaseq</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.scerevisiae.ucsc.saccer1</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.scerevisiae.ucsc.saccer2</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.sugar.cane</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowfitexampledata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.sscrofa.ucsc.susscr3</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methinheritsim</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biasaway</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-polyphen.hsapiens.dbsnp131</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-canineprobe</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.sscrofa.ucsc.susscr3.masked</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metaseqr</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmulatta.ucsc.rhemac8</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowfit</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mfascicularis.ncbi.5.0</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gmcloser</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.cfa.eg.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.cytogenetics.array</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecolileucine</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.chicken</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oligodata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clippda</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-eventpointer</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chimp.db0</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-eatonetalchipseq</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.clariom.s.rat.ht</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.cfamiliaris.ucsc.canfam3.refgene</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u133a.2</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.ptroglodytes.ucsc.pantro5.refgene</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clariomshumanhttranscriptcluster.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-organism.dplyr</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clariomdhumanprobeset.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-citrusprobe</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.focus</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecolitk</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.sscrofa.ucsc.susscr11.refgene</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-envisionquery</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oncomix</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dks</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.cal.sc5314.eg.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-coexnet</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.btaurus.ucsc.bostau8.refgene</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affycompdata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mmdiffbamsubset</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deltagseg</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mvoutdata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-factdesign</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.athaliana.biomart.plantsmart28</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-adme16cod.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mapping250k.sty</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-desubs</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ularcirc</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmelsgi</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipenrich.data</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-compartmap</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-enmix</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-onesense</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-desousa2013</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-onassisjavalibs</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmulatta.ucsc.rhemac3.masked</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lumimouseidmapping</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaseqr</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lymphoseq</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-goexpress</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-a4classif</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ggdata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geuvpack</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gesper</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maqcsubset</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bovinecdf</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-affylmgui</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcpi</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.dmelanogaster.ucsc.dm2</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vtpnet</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-randomwalkrestartmh</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.aor.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cllmethylation</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.ath.eg.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bovine.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.btaurus.ucsc.bostau4</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-eupathdb</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-countsimqc</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirbase.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.dre.eg.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnorfeeder</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnvgsadata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-epivizrchart</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cottoncdf</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dualks</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dsqtl</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cosmic.67</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-consensusde</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-daglogo</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-famagg</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geuvadistranscriptexpr</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sampleclassifierdata</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-samexplorer</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.ath.eg.db</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ncbi.grch38</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcyjs</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ggallus.ucsc.galgal5</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.cfamiliaris.ucsc.canfam3.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cordon</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.tomato</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ttmap</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-reqon</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clusterseq</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ucsc.hg18.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-copynumberplots</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmulatta.ucsc.rhemac2.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pth2o2lipids</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.gaculeatus.ucsc.gasacu1</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.drerio.ucsc.danrer7.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.drerio.ucsc.danrer6.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.clariom.s.rat</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.clariom.s.human</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>banner</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.drerio.ucsc.danrer5.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.osativa.msu.msu7</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathwaysplice</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ontoproc</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-openprimerui</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.drerio.ucsc.danrer10</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mulder2012</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.btaurus.ucsc.bostau6.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.cfamiliaris.ucsc.canfam2.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-restfulsedata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmusculus.ucsc.mm9.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgmql</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clariomsrattranscriptcluster.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.celegans.ucsc.ce2</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.carietinum.ncbi.v1</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clariomsmousetranscriptcluster.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pasilla</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.btaurus.ucsc.bostau6</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>calour</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.btaurus.ucsc.bostau4.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.mmulatta.ucsc.rhemac3.refgene</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.btaurus.ucsc.bostau3.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clariomshumantranscriptcluster.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.amellifera.ucsc.apimel2.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.charm.hg18.example</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deepstats</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.dmelanogaster.ucsc.dm6</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cgdv17</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-primeviewprobe</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-selex</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sdams</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bioseqclass</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.x.tropicalis</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prodata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.medgene.1.1.st</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omicrexposome</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rae230b</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-zebrafish.db0</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rwikipathways</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chicken.db0</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bionetstat</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jpredapi</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sbmlr</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scdd</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bigpint</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.tguttata.ucsc.taegut1</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-eegc</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.tguttata.ucsc.taegut1.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-anota2seq</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcade</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spikein</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prostatecancervarambally</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.dmelanogaster.ucsc.dm2.masked</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-curatedadipochip</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ptroglodytes.ucsc.pantro2</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ht.hg.u133a</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.hsapiens.ucsc.hg17</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ath1121501probe</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqvartools</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ddpcrclust</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pi</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ht.hg.u133.plus.pm</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sevenc</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sift.hsapiens.dbsnp137</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sigcheck</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ath1121501.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ptroglodytes.ucsc.pantro5</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tabulamurisdata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-somaticadata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plrs</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mcsea</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mtseekerdata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecolisakai.db0</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ipddb</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human610quadv1bcrlmm</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95a.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human.db0</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-adductomicsr</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>imctools</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dlbcl</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-expressionatlas</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95dprobe</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgfocus.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-icetea</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dyebiasexamples</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gars</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowbin</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fly.db0</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fdb.infiniummethylation.hg18</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-eisa</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maqcsubsetilm</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maser</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genotypeeval</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylgsa</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gvcf2bed</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.gma.eg.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133plus2barcodevecs</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.exac.r1.0.nontcga.hs37d5</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-duoclustering2018</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msd16s</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diffloopdata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-experimenthubdata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-igvr</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biowdl-input-converter</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hom.dr.inp.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metid</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metagxovarian</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mpra</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanv3.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-facsdorit</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ffpeexampledata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74cv2.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133atagprobe</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-adductdata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-doscheda</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diggitdata</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mcbiclust</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133bcdf</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-epihet</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kissde</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gseabenchmarker</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>feature_merge</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.sil.eg.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gqtlstats</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.xtr.eg.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-erssa</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene20sttranscriptcluster.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecoli2cdf</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecoli2.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gostag</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirbaseconverter</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecoli2probe</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene.1.0.st.v1frmavecs</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mofa</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tnt</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-glmsparsenet</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecoliprobe</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mdts</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.esp6500si.v2.ssa137.grch38</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moonlightr</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.pfa.3d7.eg.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-keggprofile</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gcmapweb</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vulcan</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lumihumanidmapping</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneexpressionsignature</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.topmed.freeze5.hg19</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.ani.fgsc.eg.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-foldgo</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gem</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gdcrnatools</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.eco.k12.mg1655.eg.db</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-meal</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensdb.mmusculus.v75</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-funcisnp.data</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gcapc</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-furrowseg</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.topmed.freeze5.hg38</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geuvstore2</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gcmap</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ivygapse</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-comet</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mapping250k.nsp</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chipseqdbdata</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jass</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-excluster</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-donapllp2013</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-etec16s</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-micrornaome</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-minionsummarydata</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omicplotr</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-erma</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.btaurus.ucsc.bostau3</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mugaexampledata</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>knock-knock</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lumiratidmapping</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaseqdata.hnrnpc.bam.chr14</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clinvar-tsv</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lymphoseqdb</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-m3c</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ritandata</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.exac.r1.0.grch38</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.btaurus.ucsc.bostau8</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ensdb.rnorvegicus.v79</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgmqllib</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-debrowser</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ragene.1.0.st.v1</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-conumee</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ghost-tree</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tronco</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mogene.2.0.st</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmchmm</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jolytree</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rae230a</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-multihiccompare</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nbsplice</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-onassis</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmrforpairs</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>legsta</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omicsmarker</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-opweight</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drosgenome1cdf</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.rnorvegicus.biomart.igis</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omadb</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecoliasv2cdf</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecoliasv2probe</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecolicdf</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ecolik12.db0</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mofadata</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msmb</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-egad</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msmstests</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drosgenome1.db</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.cfamiliaris.ucsc.canfam2</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fdb.ucsc.snp137common.hg19</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mdp</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.rnorvegicus.ucsc.rn5.masked</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>garnett-cli</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bloodcancermultiomics2017</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stroma4</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ggallus.ucsc.galgal4</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mbcb</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-funcisnp</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-projectr</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-progeny</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowtime</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.eco.iai39.eg.db</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowsorted.cordblood.450k</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.rnorvegicus.ucsc.rn6</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-imman</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-funtoonorm</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-post</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmusculus.ucsc.mm8.masked</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.scerevisiae.ucsc.saccer3</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prostar</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowcatchr</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fletcher2013a</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mmulatta.ucsc.rhemac3</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fletcher2013b</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ga4ghshiny</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.drerio.ucsc.danrer5</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-indeed</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-primirtss</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-imetagene</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hiergwas</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.drerio.ucsc.danrer6</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_structure_utils</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneclassifiers</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneaccord</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.gaculeatus.ucsc.gasacu1.masked</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ggallus.ucsc.galgal3</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rqt</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.rnaseq</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bodymaprat</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tmixclust</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tinesath1cdf</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.mutations</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcseq</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-svm2crmdata</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sesame</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stattarget</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133plus2frmavecs</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.ggallus.ucsc.galgal3.masked</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp149.grch38</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubc.db</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-systempiperdata</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgug4112a.db</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hguqiagenv3.db</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.zebrafish</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metagenomefeatures</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pepsnmrdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metacyto</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metabomxtr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-perturbatr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-chetah</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rat2302cdf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.zebgene.1.1.st</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowsorted.cordbloodnorway.450k</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vitisviniferacdf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeastgsdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pepsnmr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plasmodiumanophelesprobe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mirna.3.0</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ragene.1.1.st.v1</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.wheat</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ragene.2.0.st</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.u133.x3p</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rcngene.1.1.st</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.s.aureus</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellbench</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.vvi.eg.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pharmacogx</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cellmixs</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plotgrouper</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.drogene.1.0.st</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mitoodedata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>aquila</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-geneanswers</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yri1kgv</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lydata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vidger</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.genomewidesnp.6</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.spu.eg.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clumsiddata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-migsadata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-proteinprofiles</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-clumsid</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74cprobe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.nle.eg.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-profileplyr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.lma.eg.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.mdo.eg.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.medicago</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-adamgui</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mg.u74bv2</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeast2probe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.mdomestica.ucsc.mondom5</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>array-as-vcf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rae230acdf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.genomewidesnp.5</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnvranger</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rae230bprobe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mircompdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.e.coli.2</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.drosophila.2</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene11sttranscriptcluster.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.nugo.hs1a520180</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.gnomad.r2.1.hs37d5</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-porcine.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rattoxfxprobe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gladiatox</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse4302.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioexcel_seqqc</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rmassbankdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-m3dexampledata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>aquila_umap</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse4302frmavecs</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maxcontrastprojection</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cytofast</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bigsi</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-bsgenome.cjacchus.ucsc.caljac3</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.bovgene.1.0.st</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-meebodata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.1kgenomes.phase3.grch38</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moe430acdf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.atdschip.tiling</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcellminerdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ag</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.canine</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-curatedadiporna</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcagopromoter.mm.mm9</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ratchrloc</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-medipsdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-coregflux</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moex10sttranscriptcluster.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-regparallel</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rheumaticconditionwollbold</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mosaicsexample</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.ptr.eg.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-remp</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-powerexplorer</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-powertcr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jasmine</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.pab.eg.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prebsdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.moe430a</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fdb.fantom4.promoters.hg19</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeastnagalakshmi</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>lyner</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rat2302.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pcagopromoter.hs.hg19</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-matchbox</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-paxtoolsr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgug4131a.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maqcsubsetafx</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathwaypca</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maqcexpression4plex</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pathnetdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mapkldata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mammaprintdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seventygenedata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pasillatranscriptexpr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-panvizgenerator</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-paeg1aprobe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rguatlas4k.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.exac.r1.0.nontcga.grch38</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rjpgene.1.0.st</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-malaria.db0</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-neighbornet</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hapmapsnp5</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse430a2.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnaseqrdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnu34.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tofsimsdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tissuetreg</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtcga.methylation</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gprege</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.rno.eg.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mousechrloc</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse.db0</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mpedbarray.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-grndata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methvisual</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ruvcorr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hapmapsnp6</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hpaanalyze</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hivcdnavantwout03</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsbenchmark</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-logicfs</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-loci2path</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-listeretalbsseq</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-timescape</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-olingui</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-limmagui</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-timerquant</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-leukemiaseset</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hierinf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lowmacaannotation</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ygs98frmavecs</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-typeinfo</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-h5vcdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ngscopydata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-jctseqdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-wheatprobe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dcanr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gwasurvivr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kinswingr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scalign</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-wes.1kg.wugsc</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hapmap100khind</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lapointe.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msgfplus</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ledpred</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sugarcaneprobe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgamethylation450k</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scfeaturefilter</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biomm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hapmap100kxba</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sclcbam</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-targetsearchdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-italicsdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-italics</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-isocorrector</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hapmap500knsp</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scone</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133bprobe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hapmap500ksty</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-diffcoexp</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hapmap370k</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msstatsbiodata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humanomni5quadv1bcrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-drosgenome1probe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133a2cdf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-subcellbarcode</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-impcdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sepira</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqcna.annot</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nanostringqcpro</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-bamstats</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dresscheck</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hcg110cdf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msstatstmt</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgug4100a.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-serumstimulation</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sesamedata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htrat230pmcdf</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mtbls2</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-shdz.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-shinymethyldata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-heebodata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mtseeker</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-decomptumor2sig</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-icnv</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hguatlas13k.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ncigraphdata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snphooddata</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nugohs1a520180.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp141.grch38</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133a2frmavecs</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>delegation</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu15v1.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human650v3acrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humanomniexpress12v1bcrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-swfdr</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene.1.0.st.v1frmavecs</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humanomni25quadv1bcrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeastrnaseq</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humanomni1quadv1bcrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu2beta7</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humancytosnp12v2p1hcrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humanchrloc</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-huex10stprobeset.db</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133a2probe</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-huex.1.0.st.v2frmavecs</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human660quadv1acrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netsmooth</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sparsedossa</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nestlink</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sparsenetgls</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-scatterregions</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human370quadv3ccrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human1mv1ccrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xina</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human1mduov3bcrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-human550v3bcrlmm</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netprior</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-animalcules</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-stjudem</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dda.3937.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.cre.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dan.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.der.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.cpo.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.cja.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.ddi.ax4.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.aml.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dgr.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dme.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.ana.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.ame.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bold-identification</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.1kgenomes.phase1.grch38</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.1kgenomes.phase3.hs37d5</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.cel.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.1kgenomes.phase1.hs37d5</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.cin.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iyer517</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.cbr.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.bta.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-indac.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.exac.r1.0.hs37d5</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-inpas</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-intad</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-intansv</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-isobar</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iteremoval</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-jazaerimetadata.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-joda</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-keggandmetacoredzpathwaysgeo</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-kidpack</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lungexpression</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-m10kcod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-m20kcod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-macrophage</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.bfl.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.bsu.168.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.esp6500si.v2.ssa137.hs37d5</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubbcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.gnomadex.r2.1.hs37d5</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanv1.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htratfocusprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htseqgenie</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubacdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksuba.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubaprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133pluspmprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133b.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xlaevis.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htrat230pmprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hom.mm.inp.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hom.rn.inp.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hom.sc.inp.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hpannot</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hs25kresogen.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hs6ug171.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminamousev1p1.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htratfocuscdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htmg430pmprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maizecdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanwgdaslv3.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vitisviniferaprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maizeprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hca-matrix-downloader</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mbpcr</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-medicagoprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.aca.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.aga.pest.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanv2beadid.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htmg430pmcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dpe.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133pluspmcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htmg430acdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htmg430aprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htmg430bcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-htmg430bprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xlaevis2probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminaratv1.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ribosomaldatabaseproject11.5mgdb</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminamousev1.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanwgdaslv4.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hspec</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-raexexonprobesetlocation</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34acdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34aprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34bcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34bprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34ccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34c.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34cprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgug4105a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgug4130a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rhesuscdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rhesus.db0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rhesusprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ri16cod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rexposome</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-raex10sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rae230b.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prolocdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pocrcannotation.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prostatecancerstockholm</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prostatecancergrasso</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prostatecancercamcap</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-poplarcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plasfia</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-plasmodiumanophelescdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.athaliana.biomart.plantsmart22</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pogos</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-primeviewcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-twoddpcr</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-poplarprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-props</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-porcinecdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-porcineprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-potra</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-predasampledata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-restfulse</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-trnascanimport</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene20sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.pab.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.bta.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.cel.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbasedbi</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.dme.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.gga.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.hsa.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.mmu.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lrbase.ssc.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-logolas</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lumimouseall.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lumiratall.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lungcancerlines</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-liebermanaidenhic2009</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ldblock</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanv2.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-wheatcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lowmaca</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lineagepulse</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene10stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene21sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene10sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene10stv1cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene10stv1probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene11stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene20stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ragene21stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-redseq</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rat2302probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-leebamviews</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rat.db0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rattoxfxcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcellminer</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tsrchitect</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu19ksubccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nugomm1a520177probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu19ksubbcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hsagilentdesign026652.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moe430a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hspeccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moexexonprobesetlocation</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse430a2probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse430a2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse4302probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mouse4302cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene21sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene21stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene20stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene11sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene11stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene10stv1probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene10stv1cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene10sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74bcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirna10probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.eqc.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.hsa.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.zma.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mousegastrulationdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mpranalyze</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msgbsr</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu11ksubaprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu11ksuba.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msstatsqc</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msstatsqcgui</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mta10probeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mta10transcriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu11ksubacdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-u133x3pcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mogene10stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msqc1</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu11ksubbcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu11ksubb.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu11ksubbprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu19ksubacdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu19ksuba.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-u133aaofav2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-u133x3p.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.xtr.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.xla.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.tgu.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.eco.o157.h7.sakai.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-venndetail</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.eco.55989.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dya.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74aprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74av2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dvi.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74av2.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.oan.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74acdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dsi.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-usort</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>extract_fasta_seq</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylinheritance</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dse.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylaiddata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.eco.ed1a.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metagxbreast</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.tgo.me49.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.mes.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.gga.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.tbr.9274.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.sce.s288c.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.rno.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.pcr.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.laf.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metagxpancreas</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.pae.pao1.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.ocu.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.mga.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.miy.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.osa.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.mml.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.mmu.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-varianttoolsdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.mtr.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-u133x3pprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moex10stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hta20probeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mi16cod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-iasva</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xenopuslaeviscdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xenopus.db0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-illuminahumanmethylation450kprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene10stv1probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene10stv1cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene10stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu6800probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xlaevis2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubdcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubd.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubdprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu6800cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu6800.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu6800subacdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-huexexonprobesetlocationhg19</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu6800subbcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hwgcod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-huo22.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humanstemcell</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hta20transcriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133acdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133aprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hthgu133bcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubb.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubbprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu35ksubcprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humanaffydata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene11stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xhybcasneuf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene20stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene20sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene21stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hugene21sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xenopuslaevisprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu6800subccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hu6800subdcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-huex10sttranscriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74bv2probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mimager</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74av2probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74c.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74b.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74bprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74bv2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74bv2.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74ccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moe430b.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74cv2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-migsa</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgu74cv2probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mguatlas5k.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgug4104a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgug4120a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mgug4121a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mircomp</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moe430bcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-huexexonprobesetlocation</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirna10cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-huexexonprobesetlocationhg18</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dmo.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-medicagocdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.dre.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mm24kresogen.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirmine</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirna102xgaincdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirna20cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moe430aprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mirnatarget</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-missrows</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mmagilentdesign026655.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mira</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mmpalatemirna</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-moda</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rae230aprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rae230bcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rgu34b.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rae230a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mu11ksubb</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mapping50k.xba240</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hc.g110</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mapping50k.hind240</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u133a.tag</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u133b</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u133.plus.2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u219</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u95a</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u95av2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u95b</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u95c</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u95d</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg.u95e</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hta.2.0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ht.mg.430a</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hu6800</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hugene.2.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.athaliana.biomart.plantsmart25</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mu11ksuba</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mouse430a.2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mg.u74av2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hmyrib36</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mesh.eco.umn026.eg.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hom.at.inp.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.margene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.margene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.medgene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mg.u74a</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mg.u74b</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mouse430.2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mg.u74c</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mg.u74cv2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mirna.1.0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mirna.2.0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mirna.3.1</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.mirna.4.0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.moe430b</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.maize</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.nugo.mm1a520177</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nugohs1a520180probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ovigene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.pae.g1a</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.plasmodium.anopheles</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.porcine</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.porgene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.porgene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rabgene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.raex.1.0.st.v1</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rhegene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ragene.2.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rat230.2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rcngene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rg.u34a</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rg.u34b</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rg.u34c</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.hg18.60mer.expr</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.guigene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu19ksubb.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rabgene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rhegene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rhesus</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.vitis.vinifera</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ovigene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rn.u34</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rusgene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rusgene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.soybean</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.soygene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.soygene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.xenopus.laevis</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.rjpgene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.x.laevis.2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.yeast.2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.yg.s98</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.zebgene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pedbarrayv10.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pedbarrayv9.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pepdat</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ygs98probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133plus2cellscore</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hdcytodata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133plus2probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gp53cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xtropicalisprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ygs98cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hmdbquery</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ye6100subacdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hivprtplus2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jvarkit-msa2vcf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hilbertcurve</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hi16cod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hiiragi2013</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hcadata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gqtlbase</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xtrasnplocs.hsapiens.dbsnp144.grch38</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95ccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-grasp2db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hicbricks</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95c.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hireadsprocessor</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-greengenes13.5mgdb</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-h20kcod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biscuiteer</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-grridge</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-greylistchip</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgug4111a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgug4110b.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95cprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgug4101a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95dcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95eprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgudkfz31.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hcg110.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgubeta7.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95d.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hcg110probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgug4845a.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95ecdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95e.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95av2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xtrasnplocs.hsapiens.dbsnp141.grch38</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95bprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95aprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_wf_mutations</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-zebrafish.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95b.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95bcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95av2probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xtropicaliscdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gghumanmethcancerpanelv1.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-zebrafishprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133bprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu95acdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu219probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-harbchip</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-r10kcod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeast2.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu219cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hd2013sgi</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gse62944</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgfocusprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fdb.ucsc.snp135common.hg19</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeast2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yeast.db0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gsean</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133atagcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu133aprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-seattleseqkit</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-healthyflowdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ye6100subbcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ye6100subdcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-h10kcod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ygs98.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ye6100subccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-zebrafishcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gwasdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-explorase</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-yrimulti</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgfocuscdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-norway981.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hgu219.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nugomm1a520177cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sconify</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rta10transcriptcluster.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-trnadbimport</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtnduals</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtnsurvival</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-siamcat</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sift.hsapiens.dbsnp132</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-signet</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>danpos</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sugarcanecdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-summarizedbenchmark</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tomatoprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-synapterdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tbx20bamsubset</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgacrcmirna</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tcgaworkflowdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tenxbraindata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tenxplore</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rta10probeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rrdpdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-roberts2005annotation.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtu34.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-trendy</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqc</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqcombo</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqsetvis</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-seqsqc</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-trena</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rvs</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pigengene</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnu34probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtu34cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rmagpie</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rmelting</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rmir.hsa</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnainteractmapk</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gbsx</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnu34cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-test1cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-test2cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-test3cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pumadata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-river</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-riceprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ricecdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-raex10stprobeset.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prostatecancertaylor</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tweedeseqcountdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-proteomm</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pwmenrich.dmelanogaster.background</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sparsesignatures</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pwmenrich.hsapiens.background</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pwmenrich.mmusculus.background</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qplexanalyzer</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qplexdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qsmooth</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qsutils</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nugomm1a520177.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spikeinsubset</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-soybeanprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-test3probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-silva128.1mgdb</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tfhaz</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tinesath1probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-topdownr</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-strandcheckr</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-topdownrdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp150.grch38</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sigsquared</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>flextaxd</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-soybeancdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-transcriptogramer</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-slinky</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snadata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp.20101109</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-srgnet</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-snplocs.hsapiens.dbsnp.20120608</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-somaticcanceralterations</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scoreinvhap</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rrbsdata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scmeth</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.ecoli</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu6500subccdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu6500subdcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-muscat</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mwgcod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nanotator</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txreginfra</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-netsam</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-neve2006</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.sscrofa.ucsc.susscr3.refgene</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ngsreports</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-normalize450k</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.2006.07.18.mm8.refseq.promoter</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.2006.10.31.rn34.refseq.promoter</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.aragene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.aragene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.cyrgene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.drogene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu6500subbcdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-trna</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu22v3.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-parathyroidse</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hom.ce.inp.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ocplus</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oct4</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omicspcadata</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-operonhumanv3.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-paeg1acdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pairadise</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-parglms</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu19ksubc.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-partheenmetadata.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.081229.hg18.promoter.medip.hx1</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.2006.07.18.hg18.refseq.promoter</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nugohs1a520180cdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-normqpcr</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.guigene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-normalyzerde</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.drosgenome1</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mu6500subacdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.elegene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phenotest</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.ggallus.ucsc.galgal5.refgene</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.chigene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.chigene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.chogene.2.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.chogene.2.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.citrus</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phastcons100way.ucsc.hg38</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pig.db0</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.cangene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rtu34probe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rwgcod.db</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sampleclassifier</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-saureuscdf</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-saureusprobe</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.elegene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scale4c</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.celegans</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.cotton</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.felgene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.cyrgene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.equgene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.cangene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.feinberg.hg18.me.hx1</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.feinberg.mm8.me.hx1</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.felgene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.fingene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.ggallus.ucsc.galgal4.refgene</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.fingene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.equgene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.cyngene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.cyngene.1.0.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.canine.2</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.barley1</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.bovgene.1.1.st</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.bovine</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pd.bsubtilis</th>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-knowseq</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-circrnaprofiler</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.drerio.ucsc.danrer11.refgene</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rcm</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gapgom</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rat2302frmavecs</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>hmftools-sage</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.celegans.ucsc.ce11.ensgene</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fcoex</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-repviz</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metagene2</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>guessmylt</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-isocorrectorgui</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-cnvfilter</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-evaluomer</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hcabrowser</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>borf</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pwrewas</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastalite</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-validatevcf</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rscudo</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fings</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xeva</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-crossicc</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dinosaur</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biscuiteerdata</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-biotip</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-probatch</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-blacksheepr</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-modstrings</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-encodexplorerdata</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kronos</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scds</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-prince</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-precisiontrialdrawer</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lipidr</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mcagui</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sctensor</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-divergence</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-past</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-melissa</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-benchmarkfdrdata2019</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hyper</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pipeframe</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-humantranscriptomecompendium</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kggseq</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nbamseq</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-survtype</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-alphabeta</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-topconfects</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-ssrch</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tenxbusdata</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-nanotubes</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dnabarcodecompatibility</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-easyqpcr</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-autotuner</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omicslonda</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>convert_zero_one_based</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phyloprofiledata</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-treesummarizedexperiment</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowsorted.cordbloodcombined.450k</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-aspediafi</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phemd</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tnbc.cms</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.gnomadex.r2.1.grch38</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-synmut</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deco</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-depmap</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ldsc</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-doser</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-vcfarray</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>kounta</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-enrichtf</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pram</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biobb_adapters</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-alps</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spectraltad</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spatialcpie</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-apalyzer</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-validateannotation</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mmappr2</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>abritamr</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-calm</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>biopet-basecounter</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tpp2d</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-delayeddataframe</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-muscdata</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gramm4r</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dmcfb</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>jmztab-m</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-fcscan</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msstatssamplesize</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-flowspecs</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gemini</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-genomicozone</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gmicr</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scanvis</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gscreend</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hcaexplorer</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-hmp2data</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-gcsscore</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>karyopype</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omicspca</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.ptroglodytes.ucsc.pantro6.refgene</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sbgnview</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sbgnview.data</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-dewseq</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-discorhythm</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-macsquantifyr</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-deltacapturec</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-debcam</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scdblfinder</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mafdb.gnomad.r2.1.grch38</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-meb</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-viseago</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>happer</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-metavolcanor</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methcp</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methrix</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-methylcc</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-variantexperiment</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-microbiomedasim</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mmappr2data</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mmuphin</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-mosim</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-msgfgui</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-lionessr</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-linkhd</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-xcir</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-idr2d</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scbfa</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-perfect</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-schex</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-scpca</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-qtlizer</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-reactomegsa</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-reactomegsa.data</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnamodr</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnamodr.alkanilineseq</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-proda</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastaptamer</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-toast</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>decoypyrat</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>emmtyper</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>crypto_typer</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clsify</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>clusterpicker</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deepsvr</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chunked-scatter</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ega2</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>encyclopedia</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ezclermont</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>extract-codon-alignment</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pwrewas.data</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-pulsedsilac</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-phyloprofile</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnasense</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-setools</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-signaturesearchdata</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sigspack</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-spaniel</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sqldataframe</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-sspaths</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-target</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-tartare</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnamodr.ribomethseq</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-omnipathr</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnamodr.ml</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-rnamodr.data</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-oppti</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-org.mxanthus.db</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.mmulatta.ucsc.rhemac10.refgene</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.ggallus.ucsc.galgal6.refgene</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-txdb.btaurus.ucsc.bostau9.refgene</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-peakpanther</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>bioconductor-maaslin2</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                    usage_2019_01\n",
       "module                                                           \n",
       "samtools                                                  1836974\n",
       "pysam                                                     1415206\n",
       "perl-extutils-makemaker                                    330596\n",
       "perl-carp                                                  316240\n",
       "perl-exporter                                              311641\n",
       "perl-scalar-list-utils                                     300579\n",
       "perl-xsloader                                              277632\n",
       "star                                                       267257\n",
       "picard                                                     264985\n",
       "perl-compress-raw-zlib                                     262613\n",
       "snakemake                                                  261989\n",
       "perl-list-moreutils                                        260626\n",
       "perl-exporter-tiny                                         259127\n",
       "perl-app-cpanminus                                         256642\n",
       "perl-pathtools                                             252174\n",
       "perl-data-dumper                                           236600\n",
       "perl-parent                                                230774\n",
       "perl-compress-raw-bzip2                                    229972\n",
       "perl-io-compress                                           229748\n",
       "pybedtools                                                 225121\n",
       "perl-uri                                                   223700\n",
       "perl-threaded                                              222698\n",
       "perl-encode                                                220236\n",
       "perl-file-path                                             220225\n",
       "perl-constant                                              220149\n",
       "perl-list-moreutils-xs                                     219721\n",
       "perl-file-temp                                             209533\n",
       "perl-xml-sax                                               209529\n",
       "perl-xml-namespacesupport                                  205411\n",
       "perl-xml-sax-base                                          205378\n",
       "perl-html-parser                                           204107\n",
       "perl-net-ssleay                                            203568\n",
       "sambamba                                                   202487\n",
       "perl-json-xs                                               201633\n",
       "perl-html-tagset                                           201341\n",
       "pybigwig                                                   198714\n",
       "perl-encode-locale                                         198252\n",
       "perl-xml-simple                                            195056\n",
       "perl-common-sense                                          194862\n",
       "perl-io-socket-ssl                                         194780\n",
       "perl-mime-base64                                           194696\n",
       "perl-lwp-mediatypes                                        193795\n",
       "perl-libwww-perl                                           193203\n",
       "perl-json                                                  192951\n",
       "perl-xml-sax-expat                                         192576\n",
       "perl-net-http                                              191945\n",
       "perl-digest-hmac                                           190957\n",
       "perl-http-date                                             190549\n",
       "perl-io-html                                               190396\n",
       "perl-http-message                                          190192\n",
       "perl-http-cookies                                          189959\n",
       "perl-ntlm                                                  189800\n",
       "perl-http-negotiate                                        189739\n",
       "perl-www-robotrules                                        188830\n",
       "perl-file-listing                                          187767\n",
       "perl-http-daemon                                           187757\n",
       "perl-html-tree                                             187437\n",
       "pubchempy                                                  187185\n",
       "multiqc                                                    187074\n",
       "perl-try-tiny                                              185653\n",
       "perl-lwp-protocol-https                                    184087\n",
       "perl-archive-tar                                           183935\n",
       "perl-business-isbn                                         180995\n",
       "perl-business-isbn-data                                    180987\n",
       "perl-io-zlib                                               177731\n",
       "perl-types-serialiser                                      176717\n",
       "mafft                                                      174314\n",
       "perl-storable                                              173738\n",
       "perl-mozilla-ca                                            170070\n",
       "perl-digest-md5                                            167946\n",
       "perl-time-local                                            166078\n",
       "perl-socket                                                164299\n",
       "perl-test-requiresinternet                                 163206\n",
       "vsearch                                                    162465\n",
       "perl-yaml                                                  151011\n",
       "perl-gd                                                    147090\n",
       "pyfaidx                                                    146823\n",
       "perl-dbi                                                   136618\n",
       "subread                                                    132027\n",
       "perl-bioperl                                               127487\n",
       "perl-getopt-long                                           123163\n",
       "perl-xml-parser                                            122792\n",
       "quast                                                      121985\n",
       "perl-cgi                                                   121797\n",
       "perl-xml-libxml                                            121576\n",
       "metaphlan2                                                 119507\n",
       "perl-module-runtime                                        119313\n",
       "minimap2                                                   118765\n",
       "salmon                                                     116310\n",
       "perl-file-which                                            114116\n",
       "perl-clone                                                 114111\n",
       "perl-db-file                                               113622\n",
       "raxml                                                      112386\n",
       "perl-io-string                                             110358\n",
       "perl-time-hires                                            109294\n",
       "trimmomatic                                                108735\n",
       "sortmerna                                                  107451\n",
       "perl-svg                                                   107203\n",
       "perl-sub-exporter-progressive                              104944\n",
       "perl-moo                                                   104481\n",
       "perl-dbd-sqlite                                            104450\n",
       "perl-lwp-simple                                            104368\n",
       "perl-digest-perl-md5                                       104292\n",
       "perl-role-tiny                                             104105\n",
       "perl-devel-globaldestruction                               103763\n",
       "perl-math-cdf                                              103084\n",
       "perl-module-implementation                                 102398\n",
       "perl-class-method-modifiers                                101876\n",
       "perl-statistics-descriptive                                101117\n",
       "tabix                                                      101027\n",
       "perl-bioperl-core                                          100182\n",
       "perl-pdf-api2                                               99799\n",
       "perl-devel-stacktrace                                       99551\n",
       "perl-font-ttf                                               99495\n",
       "perl-capture-tiny                                           97832\n",
       "perl-type-tiny                                              97815\n",
       "perl-class-inspector                                        97153\n",
       "spades                                                      97089\n",
       "perl-xml-twig                                               95408\n",
       "perl-ipc-run                                                95251\n",
       "perl-test-leaktrace                                         95230\n",
       "perl-mailtools                                              94965\n",
       "sra-tools                                                   94909\n",
       "perl-parse-recdescent                                       94668\n",
       "perl-params-util                                            94560\n",
       "perl-autoloader                                             94532\n",
       "perl-moose                                                  94442\n",
       "perl-bio-asn1-entrezgene                                    94397\n",
       "perl-mime-tools                                             94390\n",
       "perl-array-compare                                          94236\n",
       "perl-error                                                  94005\n",
       "perl-html-formatter                                         93783\n",
       "perl-task-weaken                                            93722\n",
       "perl-html-tidy                                              93685\n",
       "perl-graphviz                                               93671\n",
       "perl-bio-phylo                                              93600\n",
       "perl-math-derivative                                        93593\n",
       "perl-tree-dag_node                                          93585\n",
       "perl-xml-xpath                                              93518\n",
       "perl-file-slurp-tiny                                        93455\n",
       "perl-xml-sax-writer                                         93440\n",
       "perl-graph                                                  93377\n",
       "perl-libxml-perl                                            93352\n",
       "perl-mldbm                                                  93352\n",
       "perl-xml-libxslt                                            93334\n",
       "perl-xml-dom                                                93322\n",
       "perl-mime-types                                             93251\n",
       "perl-io-tty                                                 93230\n",
       "perl-dynaloader                                             93137\n",
       "perl-data-stag                                              93048\n",
       "perl-math-random                                            92981\n",
       "perl-unicode-map                                            92909\n",
       "perl-convert-binary-c                                       92837\n",
       "viennarna                                                   92753\n",
       "perl-ole-storage_lite                                       92678\n",
       "perl-bioperl-run                                            92658\n",
       "perl-svg-graph                                              92603\n",
       "perl-sort-naturally                                         92589\n",
       "perl-sub-install                                            92496\n",
       "perl-email-date-format                                      92325\n",
       "perl-tie-ixhash                                             92287\n",
       "perl-data-optlist                                           92281\n",
       "perl-jcode                                                  92240\n",
       "perl-spreadsheet-writeexcel                                 92232\n",
       "seqtk                                                       92153\n",
       "perl-package-stash                                          92054\n",
       "perl-xml-xpathengine                                        92050\n",
       "perl-font-afm                                               91898\n",
       "tidyp                                                       91896\n",
       "perl-io-stringy                                             91783\n",
       "perl-html-entities-numbered                                 91765\n",
       "perl-xml-regexp                                             91679\n",
       "perl-convert-binhex                                         91670\n",
       "perl-apache-test                                            91574\n",
       "perl-timedate                                               91562\n",
       "perl-io-sessiondata                                         91486\n",
       "perl-xml-writer                                             91377\n",
       "perl-image-size                                             91277\n",
       "perl-template-toolkit                                       91274\n",
       "perl-mime-lite                                              91243\n",
       "perl-xml-filter-buffertext                                  91224\n",
       "perl-crypt-rc4                                              91188\n",
       "perl-algorithm-munkres                                      91185\n",
       "perl-image-info                                             91184\n",
       "perl-appconfig                                              91165\n",
       "perl-soap-lite                                              91141\n",
       "perl-sub-identify                                           91132\n",
       "perl-html-treebuilder-xpath                                 91107\n",
       "perl-postscript                                             91100\n",
       "perl-set-scalar                                             91067\n",
       "perl-aceperl                                                91047\n",
       "perl-spreadsheet-parseexcel                                 91046\n",
       "perl-xml-dom-xpath                                          91013\n",
       "perl-html-tableextract                                      91010\n",
       "perl-html-element-extended                                  91000\n",
       "perl-math-spline                                            90994\n",
       "perl-sub-exporter                                           90883\n",
       "perl-eval-closure                                           90096\n",
       "nextflow                                                    89664\n",
       "perl-class-load-xs                                          89272\n",
       "perl-bio-featureio                                          89260\n",
       "perl-dist-checkconflicts                                    88864\n",
       "perl-test-harness                                           88682\n",
       "perl-bio-samtools                                           88294\n",
       "vcftools                                                    88217\n",
       "perl-file-sort                                              88187\n",
       "perl-class-load                                             87930\n",
       "snpeff                                                      87645\n",
       "perl-package-stash-xs                                       87537\n",
       "perl-sub-uplevel                                            86539\n",
       "perl-sub-name                                               86227\n",
       "perl-exception-class                                        85919\n",
       "perl-sub-quote                                              85871\n",
       "perl-mro-compat                                             85682\n",
       "perl-module-runtime-conflicts                               85671\n",
       "perl-devel-overloadinfo                                     85669\n",
       "perl-package-deprecationmanager                             85656\n",
       "perl-test-warn                                              85125\n",
       "perl-text-diff                                              84023\n",
       "perl-class-data-inheritable                                 83971\n",
       "perl-test-most                                              83449\n",
       "perl-lib                                                    83391\n",
       "perl-test-differences                                       83164\n",
       "perl-test                                                   83033\n",
       "perl-pod-escapes                                            83011\n",
       "perl-pod-usage                                              82784\n",
       "perl-test-exception                                         82724\n",
       "perl-algorithm-diff                                         82117\n",
       "perl-ipc-sharelite                                          81362\n",
       "perl-digest-sha1                                            81192\n",
       "t_coffee                                                    81184\n",
       "seqkit                                                      80431\n",
       "paml                                                        80195\n",
       "perl-cache-cache                                            79812\n",
       "perl-date-format                                            79203\n",
       "perl-bio-tools-phylo-paml                                   75262\n",
       "perl-bio-coordinate                                         75221\n",
       "perl-bio-tools-run-alignment-clustalw                       75208\n",
       "qualimap                                                    74370\n",
       "pbgzip                                                      73784\n",
       "muscle                                                      73723\n",
       "perl-perlio-gzip                                            72763\n",
       "ucsc-bedgraphtobigwig                                       72277\n",
       "perl-bio-tools-run-alignment-tcoffee                        72117\n",
       "rseqc                                                       71908\n",
       "pytabix                                                     71032\n",
       "prodigal                                                    70937\n",
       "perl-module-corelist                                        70935\n",
       "perl-test-deep                                              70414\n",
       "py2bit                                                      67566\n",
       "sina                                                        67310\n",
       "perl-module-metadata                                        65483\n",
       "stringtie                                                   63535\n",
       "perl-module-load-conditional                                63428\n",
       "perl-ipc-cmd                                                63204\n",
       "perl-module-load                                            62370\n",
       "perl-locale-maketext-simple                                 62273\n",
       "perl-version                                                62151\n",
       "perl-params-check                                           62114\n",
       "trim-galore                                                 61752\n",
       "htslib                                                      61473\n",
       "sickle-trim                                                 59616\n",
       "trinity                                                     58666\n",
       "perl-module-build                                           56819\n",
       "perl-set-intervaltree                                       54140\n",
       "ucsc-wigtobigwig                                            53810\n",
       "racon                                                       52675\n",
       "unifrac                                                     52573\n",
       "perl-gdgraph                                                52384\n",
       "vardict                                                     52342\n",
       "pairix                                                      51497\n",
       "perl-vcftools-vcf                                           50943\n",
       "perl-gdtextutil                                             50657\n",
       "macs2                                                       49852\n",
       "pindel                                                      48830\n",
       "ucsc-fatotwobit                                             48589\n",
       "tbl2asn                                                     47855\n",
       "perl-text-csv                                               47770\n",
       "prokka                                                      47219\n",
       "multiqc-bcbio                                               47178\n",
       "qiime                                                       46811\n",
       "meme                                                        46318\n",
       "perl-dbd-mysql                                              46038\n",
       "perl-bio-db-hts                                             45855\n",
       "sourmash                                                    45667\n",
       "peptide-shaker                                              45590\n",
       "planemo                                                     44322\n",
       "bcftools                                                    44152\n",
       "mappy                                                       44044\n",
       "manta                                                       43748\n",
       "sentieon                                                    43429\n",
       "perl-number-format                                          43136\n",
       "megahit                                                     43017\n",
       "vardict-java                                                42794\n",
       "varscan                                                     42424\n",
       "perl-extutils-manifest                                      42384\n",
       "mosdepth                                                    42114\n",
       "perl-cpan-meta                                              41828\n",
       "perl-text-parsewords                                        41822\n",
       "perl-json-pp                                                40737\n",
       "perl-extutils-parsexs                                       40488\n",
       "seqcluster                                                  40295\n",
       "perl-sereal-decoder                                         39884\n",
       "perl-sereal-encoder                                         39795\n",
       "perl-perl-ostype                                            39683\n",
       "stacks                                                      39603\n",
       "ucsc-liftover                                               39204\n",
       "perl-extutils-cbuilder                                      38990\n",
       "perl-sereal                                                 38743\n",
       "mummer                                                      38688\n",
       "ucsc-gtftogenepred                                          38615\n",
       "perl-text-abbrev                                            38557\n",
       "umi_tools                                                   38413\n",
       "ncls                                                        37462\n",
       "perl-cpan-meta-requirements                                 37325\n",
       "vcflib                                                      36923\n",
       "miniasm                                                     36543\n",
       "vcfanno                                                     36457\n",
       "vt                                                          36218\n",
       "wisecondorx                                                 36190\n",
       "nanopolish                                                  36116\n",
       "ncbi-ngs-sdk                                                35929\n",
       "perl-file-sharedir-install                                  35651\n",
       "pyranges                                                    35568\n",
       "ucsc-bedtobigbed                                            35333\n",
       "perl-cpan-meta-yaml                                         35229\n",
       "minced                                                      35151\n",
       "sorted_nearest                                              35031\n",
       "sepp                                                        34758\n",
       "umis                                                        34432\n",
       "perl-archive-zip                                            34402\n",
       "searchgui                                                   34090\n",
       "scalpel                                                     33981\n",
       "tophat                                                      33640\n",
       "perl-params-validate                                        33495\n",
       "perl-statistics-basic                                       32796\n",
       "mash                                                        32784\n",
       "perl-regexp-common                                          32725\n",
       "sevenbridges-python                                         32373\n",
       "perl-file-sharedir                                          32273\n",
       "samblaster                                                  32239\n",
       "rsem                                                        32054\n",
       "perl-file-copy-recursive                                    31912\n",
       "qcli                                                        31867\n",
       "pb-falcon                                                   31787\n",
       "perl-math-round                                             31721\n",
       "stream                                                      31573\n",
       "rust-bio-tools                                              31528\n",
       "perl-readonly                                               31496\n",
       "peddy                                                       31278\n",
       "mageck                                                      31030\n",
       "perl-archive-extract                                        30932\n",
       "pynast                                                      30800\n",
       "snippy                                                      30762\n",
       "perl-config-general                                         30726\n",
       "nanoplot                                                    30660\n",
       "rtg-tools                                                   30573\n",
       "mcl                                                         30556\n",
       "perl-file-fetch                                             30484\n",
       "star-fusion                                                 30292\n",
       "perl-ipc-system-simple                                      29888\n",
       "medaka                                                      29619\n",
       "novoalign                                                   29615\n",
       "nanoplotter                                                 29508\n",
       "vcf2db                                                      29462\n",
       "vcf2maf                                                     29421\n",
       "toil                                                        29386\n",
       "pilon                                                       29346\n",
       "perl-text-format                                            29286\n",
       "nanoget                                                     29192\n",
       "perl-math-vecstat                                           29049\n",
       "perl-math-bezier                                            28922\n",
       "perl-set-intspan                                            28823\n",
       "pyrle                                                       28693\n",
       "whatshap                                                    28093\n",
       "perl-gdgraph-histogram                                      27932\n",
       "mothur                                                      27931\n",
       "mygene                                                      27902\n",
       "ucsc-bigwigsummary                                          27755\n",
       "mlst                                                        27672\n",
       "ucsc-twobittofa                                             27327\n",
       "minialign                                                   27269\n",
       "preseq                                                      26951\n",
       "snpsift                                                     26730\n",
       "qiime-default-reference                                     26606\n",
       "seqan                                                       26220\n",
       "pygenometracks                                              25683\n",
       "scanpy                                                      25595\n",
       "perl-log-log4perl                                           25395\n",
       "nanomath                                                    25320\n",
       "mirtop                                                      25084\n",
       "simple_sv_annotation                                        24941\n",
       "ucsc-genepredtobed                                          24900\n",
       "prophyle                                                    24637\n",
       "sequana                                                     24584\n",
       "bedtools                                                    24168\n",
       "transdecoder                                                23937\n",
       "mmseqs2                                                     23434\n",
       "pyteomics                                                   23409\n",
       "quasitools                                                  23372\n",
       "pbmm2                                                       23306\n",
       "rgi                                                         22986\n",
       "mentalist                                                   22971\n",
       "perl-test2-suite                                            22953\n",
       "ncbi-genome-download                                        22755\n",
       "perl-parallel-forkmanager                                   22749\n",
       "lumpy-sv                                                    22513\n",
       "perl-html-template                                          22164\n",
       "ont-tombo                                                   22094\n",
       "pbbam                                                       21989\n",
       "perl-hpc-runner-command                                     21989\n",
       "ngless                                                      21918\n",
       "mashtree                                                    21657\n",
       "strelka                                                     21613\n",
       "maker                                                       21204\n",
       "ucsc-bigwigtobedgraph                                       20848\n",
       "r-sleuth                                                    20440\n",
       "roary                                                       20426\n",
       "orthofinder                                                 20333\n",
       "ucsc-bigwiginfo                                             20321\n",
       "msstitch                                                    20195\n",
       "python-edlib                                                20187\n",
       "perl-test-simple                                            20050\n",
       "toolshed                                                    20049\n",
       "perl-importer                                               20006\n",
       "ucsc-bigbedtobed                                            19934\n",
       "unicycler                                                   19892\n",
       "velvet                                                      19866\n",
       "svtyper                                                     19689\n",
       "razers3                                                     19593\n",
       "proteinortho                                                19577\n",
       "perl-text-glob                                              19464\n",
       "ngs-disambiguate                                            19377\n",
       "repeatmasker                                                19287\n",
       "msisensor                                                   19094\n",
       "nanofilt                                                    19048\n",
       "vispr                                                       18940\n",
       "r-pscbs                                                     18787\n",
       "trf                                                         18658\n",
       "minvar                                                      18552\n",
       "pysamstats                                                  18500\n",
       "trimal                                                      18495\n",
       "ucsc-gff3togenepred                                         18400\n",
       "optitype                                                    18313\n",
       "prinseq                                                     18170\n",
       "r-spp                                                       18131\n",
       "perl-test-pod                                               18121\n",
       "srprism                                                     17983\n",
       "nanocomp                                                    17974\n",
       "samplot                                                     17946\n",
       "r-wasabi                                                    17869\n",
       "ucsc-genepredtogtf                                          17857\n",
       "parallel-fastq-dump                                         17800\n",
       "parasail-python                                             17766\n",
       "perl-extutils-cppguess                                      17689\n",
       "mapcaller                                                   17608\n",
       "popdel                                                      17489\n",
       "secapr                                                      17312\n",
       "mummer4                                                     17301\n",
       "msgf_plus                                                   17142\n",
       "svtools                                                     17094\n",
       "perl-term-table                                             17085\n",
       "trnascan-se                                                 17057\n",
       "tophat-recondition                                          17041\n",
       "perl-file-find-rule                                         16942\n",
       "ucsc-fetchchromsizes                                        16862\n",
       "perl-number-compare                                         16392\n",
       "r-basejump                                                  16258\n",
       "seqbuster                                                   16181\n",
       "ucsc-bigwigtowig                                            16153\n",
       "w4mclassfilter                                              16151\n",
       "smoove                                                      16139\n",
       "rmblast                                                     16134\n",
       "pymummer                                                    16117\n",
       "ucsc-bedtogenepred                                          16092\n",
       "ruffus                                                      16028\n",
       "ucsc-bigwigaverageoverbed                                   16017\n",
       "rabix-bunny                                                 15977\n",
       "perl-scope-guard                                            15822\n",
       "snp-sites                                                   15811\n",
       "mqc                                                         15800\n",
       "perl-module-pluggable                                       15663\n",
       "octopus                                                     15638\n",
       "ucsc-twobitinfo                                             15618\n",
       "varlociraptor                                               15567\n",
       "perl-sub-info                                               15343\n",
       "ngs-bits                                                    15231\n",
       "pb-assembly                                                 15176\n",
       "slclust                                                     15084\n",
       "stringmlst                                                  15028\n",
       "bwa                                                         14995\n",
       "pplacer                                                     14967\n",
       "sailfish                                                    14884\n",
       "picrust                                                     14723\n",
       "r-bcbiornaseq                                               14616\n",
       "plink                                                       14533\n",
       "pauvre                                                      14450\n",
       "rapid                                                       14449\n",
       "quicksect                                                   14408\n",
       "prank                                                       14366\n",
       "phyml                                                       14348\n",
       "ucsc-bedsort                                                14341\n",
       "ucsc-bigbedinfo                                             14280\n",
       "moabs                                                       14268\n",
       "cutadapt                                                    14265\n",
       "fastqc                                                      14258\n",
       "ucsc-bedclip                                                14210\n",
       "shovill                                                     14189\n",
       "variantbam                                                  14183\n",
       "synapseclient                                               14180\n",
       "python-tripal                                               14176\n",
       "xmltramp2                                                   14139\n",
       "r-bcbiobase                                                 14046\n",
       "r-wgcna                                                     14044\n",
       "perl-file-slurp                                             13981\n",
       "bioconductor-iranges                                        13965\n",
       "staramr                                                     13956\n",
       "masurca                                                     13922\n",
       "riboseed                                                    13897\n",
       "sniffles                                                    13847\n",
       "bioconductor-biocgenerics                                   13832\n",
       "ucsc-bigbedsummary                                          13821\n",
       "bioconductor-s4vectors                                      13820\n",
       "rapmap                                                      13737\n",
       "nanostat                                                    13733\n",
       "merfishtools                                                13680\n",
       "parafly                                                     13643\n",
       "selene-sdk                                                  13595\n",
       "wham                                                        13584\n",
       "primer3                                                     13576\n",
       "pyani                                                       13422\n",
       "swarm                                                       13352\n",
       "mageck-vispr                                                13335\n",
       "pyhashxx                                                    13317\n",
       "perl-array-utils                                            13317\n",
       "survivor                                                    13287\n",
       "taxonkit                                                    13187\n",
       "pbcore                                                      13175\n",
       "ngmlr                                                       13147\n",
       "bioconductor-biobase                                        12941\n",
       "bioconductor-genomicranges                                  12857\n",
       "methyldackel                                                12831\n",
       "bioconductor-zlibbioc                                       12826\n",
       "rscape                                                      12824\n",
       "pear                                                        12803\n",
       "r-sartools                                                  12797\n",
       "seq2c                                                       12771\n",
       "perl-perlio-utf8_strict                                     12756\n",
       "stream_atac                                                 12740\n",
       "vphaser2                                                    12731\n",
       "metabat2                                                    12728\n",
       "pbcopper                                                    12710\n",
       "mirdeep2                                                    12705\n",
       "thermorawfileparser                                         12636\n",
       "shorah                                                      12621\n",
       "pureclip                                                    12591\n",
       "bioconductor-xvector                                        12556\n",
       "tardis                                                      12546\n",
       "percolator                                                  12501\n",
       "ucsc-nibfrag                                                12491\n",
       "samsift                                                     12485\n",
       "bioconductor-biocparallel                                   12476\n",
       "perl-io-gzip                                                12452\n",
       "perl-file-slurper                                           12398\n",
       "platypus-variant                                            12299\n",
       "picopore                                                    12268\n",
       "python-consensuscore                                        12253\n",
       "pyfastaq                                                    12246\n",
       "bioconductor-genomeinfodb                                   12210\n",
       "ncbi-amrfinderplus                                          12208\n",
       "reparation_blast                                            12179\n",
       "weblogo                                                     12163\n",
       "pypeflow                                                    12069\n",
       "perl-parse-yapp                                             12019\n",
       "skewer                                                      11993\n",
       "ucsc-pslcdnafilter                                          11975\n",
       "pbsv                                                        11940\n",
       "mvicuna                                                     11906\n",
       "bowtie                                                      11858\n",
       "tn93                                                        11851\n",
       "tracy                                                       11815\n",
       "nanoqc                                                      11656\n",
       "bioconductor-summarizedexperiment                           11621\n",
       "pbccs                                                       11619\n",
       "mirge                                                       11618\n",
       "r-hemdag                                                    11575\n",
       "pyliftover                                                  11520\n",
       "bioconductor-genomeinfodbdata                               11516\n",
       "bioconductor-biostrings                                     11390\n",
       "ncbi-vdb                                                    11333\n",
       "nanocall                                                    11316\n",
       "sistr_cmd                                                   11307\n",
       "perl-filesys-df                                             11290\n",
       "perl-file-spec                                              11280\n",
       "perl-findbin                                                11258\n",
       "snap-aligner                                                11222\n",
       "maxbin2                                                     11184\n",
       "phyluce                                                     11135\n",
       "bioconductor-delayedarray                                   11131\n",
       "nonpareil                                                   11092\n",
       "rna-seqc                                                    11028\n",
       "r-biodb                                                     10998\n",
       "slivar                                                      10960\n",
       "ucsc-oligomatch                                             10938\n",
       "perl-termreadkey                                            10924\n",
       "yacrd                                                       10903\n",
       "nim-falcon                                                  10824\n",
       "snp-dists                                                   10769\n",
       "perl-file-grep                                              10755\n",
       "pbcommand                                                   10742\n",
       "segway                                                      10739\n",
       "bioconductor-genomicalignments                              10675\n",
       "mikado                                                      10671\n",
       "rnalien                                                     10660\n",
       "pymzml                                                      10626\n",
       "spaln                                                       10605\n",
       "xtandem                                                     10579\n",
       "poretools                                                   10546\n",
       "bioconductor-rsamtools                                      10490\n",
       "uropa                                                       10470\n",
       "perl-graph-readwrite                                        10414\n",
       "perl-hash-merge                                             10380\n",
       "primer3-py                                                  10369\n",
       "perl-path-tiny                                              10366\n",
       "ucsc-hggcpercent                                            10363\n",
       "ucsc-pslrecalcmatch                                         10357\n",
       "ucsc-pslsort                                                10227\n",
       "plant_tribes_gene_family_classifier                         10177\n",
       "dnaio                                                       10175\n",
       "python-chado                                                10172\n",
       "ont-fast5-api                                               10166\n",
       "samclip                                                     10145\n",
       "blast                                                       10108\n",
       "ucsc-pslcheck                                               10082\n",
       "ucsc-bigwigmerge                                            10077\n",
       "ucsc-splitfilebycolumn                                      10028\n",
       "phylowgs                                                    10000\n",
       "ucsc-psltochain                                              9958\n",
       "moff                                                         9928\n",
       "pythonpy                                                     9922\n",
       "ucsc-pslpostarget                                            9912\n",
       "ucsc-wigcorrelate                                            9906\n",
       "mgkit                                                        9904\n",
       "xsd                                                          9896\n",
       "ucsc-qactowig                                                9877\n",
       "ucsc-psltobigpsl                                             9857\n",
       "ucsc-positionaltblcheck                                      9844\n",
       "ucsc-tolower                                                 9840\n",
       "skesa                                                        9840\n",
       "ucsc-pslmrnacover                                            9836\n",
       "ucsc-genepredtofakepsl                                       9824\n",
       "r-coloc                                                      9818\n",
       "perl-digest-md5-file                                         9816\n",
       "ucsc-psltobed                                                9797\n",
       "portcullis                                                   9791\n",
       "ucsc-pslpairs                                                9766\n",
       "trinotate                                                    9766\n",
       "ucsc-tdbquery                                                9765\n",
       "norns                                                        9751\n",
       "ucsc-xmltosql                                                9724\n",
       "ucsc-psltopslx                                               9720\n",
       "perl-algorithm-cluster                                       9711\n",
       "ucsc-ticktodate                                              9702\n",
       "ucsc-subchar                                                 9700\n",
       "ucsc-splitfile                                               9695\n",
       "rnftools                                                     9690\n",
       "svdb                                                         9689\n",
       "ucsc-pslfilter                                               9681\n",
       "ucsc-twobitmask                                              9676\n",
       "ucsc-pslreps                                                 9673\n",
       "ucsc-xmlcat                                                  9662\n",
       "ucsc-rmfadups                                                9651\n",
       "ucsc-sqltoxml                                                9639\n",
       "ucsc-qatoqac                                                 9639\n",
       "ucsc-pslcat                                                  9636\n",
       "ucsc-qactoqa                                                 9634\n",
       "pyfasta                                                      9633\n",
       "ucsc-pslswap                                                 9629\n",
       "ucsc-toupper                                                 9624\n",
       "ucsc-psldropoverlap                                          9623\n",
       "ucsc-parasync                                                9622\n",
       "ucsc-pslpretty                                               9622\n",
       "mob_suite                                                    9622\n",
       "oncofuse                                                     9614\n",
       "ucsc-pslpartition                                            9613\n",
       "ucsc-parafetch                                               9613\n",
       "ucsc-pslselect                                               9607\n",
       "ucsc-trfbig                                                  9601\n",
       "perl-sanger-cgp-allelecount                                  9599\n",
       "ucsc-genepredtomafframes                                     9594\n",
       "tdrmapper                                                    9594\n",
       "motus                                                        9593\n",
       "ucsc-validatefiles                                           9592\n",
       "ucsc-nettobed                                                9592\n",
       "ucsc-newpythonprog                                           9586\n",
       "prosolo                                                      9580\n",
       "ucsc-taillines                                               9578\n",
       "ucsc-randomlines                                             9578\n",
       "pipits                                                       9567\n",
       "ucsc-nibsize                                                 9563\n",
       "ucsc-rowstocols                                              9560\n",
       "ucsc-subcolumn                                               9545\n",
       "ucsc-ratolines                                               9535\n",
       "ucsc-rasqlquery                                              9521\n",
       "ucsc-pslstats                                                9520\n",
       "seqlogo                                                      9514\n",
       "ucsc-validatemanifest                                        9514\n",
       "trumicount                                                   9512\n",
       "ucsc-stringify                                               9510\n",
       "ucsc-twobitdup                                               9507\n",
       "ucsc-pslliftsubrangeblat                                     9504\n",
       "ucsc-spacedtotab                                             9503\n",
       "ucsc-qacagplift                                              9493\n",
       "ucsc-pslxtofa                                                9489\n",
       "ucsc-pslhisto                                                9485\n",
       "ucsc-ratotab                                                 9481\n",
       "nanosv                                                       9471\n",
       "ucsc-pslscore                                                9464\n",
       "ucsc-texthistogram                                           9457\n",
       "perl-dbd-pg                                                  9457\n",
       "ucsc-wordline                                                9426\n",
       "ribocode                                                     9401\n",
       "ucsc-genepredtobiggenepred                                   9386\n",
       "perl-sanger-cgp-vcf                                          9384\n",
       "rmats                                                        9355\n",
       "pygtftk                                                      9354\n",
       "nanosim                                                      9324\n",
       "ucsc-genepredcheck                                           9301\n",
       "mobster                                                      9231\n",
       "mira                                                         9216\n",
       "pepnovo                                                      9187\n",
       "pizzly                                                       9161\n",
       "wtdbg                                                        9156\n",
       "nanolyse                                                     9098\n",
       "pyloh                                                        9095\n",
       "ucsc-fasplit                                                 9048\n",
       "perl-devel-symdump                                           9039\n",
       "ucsc-fasomerecords                                           9000\n",
       "mykrobe                                                      8996\n",
       "perl-data-uuid                                               8988\n",
       "sqt                                                          8981\n",
       "pylibseq                                                     8928\n",
       "r-chbutils                                                   8897\n",
       "cyvcf2                                                       8829\n",
       "starcode                                                     8817\n",
       "perl-test2                                                   8812\n",
       "nanoraw                                                      8755\n",
       "singlecell-barcodes                                          8705\n",
       "watchdog-wms                                                 8702\n",
       "solvebio                                                     8647\n",
       "perl-time-piece                                              8634\n",
       "lz4-bin                                                      8631\n",
       "tbl2asn-forever                                              8605\n",
       "pairtools                                                    8578\n",
       "ucsc-pslmap                                                  8529\n",
       "manorm                                                       8489\n",
       "ucsc-chainmergesort                                          8479\n",
       "rcorrector                                                   8461\n",
       "perl-variable-magic                                          8456\n",
       "perl-bio-db-sam                                              8449\n",
       "porechop                                                     8438\n",
       "bowtie2                                                      8426\n",
       "perl-cpan-meta-check                                         8419\n",
       "moreutils                                                    8400\n",
       "tobias                                                       8379\n",
       "python-msgpack                                               8375\n",
       "perl-pcap                                                    8356\n",
       "perl-b-hooks-endofscope                                      8354\n",
       "perl-namespace-clean                                         8345\n",
       "ucsc-fasize                                                  8342\n",
       "qsignature                                                   8331\n",
       "ucsc-chainnet                                                8325\n",
       "metasv                                                       8319\n",
       "snap                                                         8310\n",
       "ucsc-bigwigcat                                               8293\n",
       "ucsc-chainsplit                                              8247\n",
       "ucsc-bigwigcluster                                           8240\n",
       "ucsc-beditemoverlapcount                                     8233\n",
       "ucsc-faonerecord                                             8231\n",
       "perl-inline                                                  8219\n",
       "phame                                                        8216\n",
       "ucsc-genepredsinglecover                                     8199\n",
       "ucsc-bigwigcorrelate                                         8166\n",
       "ucsc-fatofastq                                               8160\n",
       "verifybamid2                                                 8156\n",
       "ucsc-chainstitchid                                           8154\n",
       "ucsc-fapolyasizes                                            8154\n",
       "ucsc-fatrans                                                 8153\n",
       "ucsc-bedrestricttopositions                                  8152\n",
       "ucsc-blasttopsl                                              8146\n",
       "ucsc-bigbednameditems                                        8136\n",
       "ucsc-bedgraphpack                                            8130\n",
       "ucsc-chainprenet                                             8130\n",
       "ucsc-fastqtofa                                               8119\n",
       "ucsc-genepredfilter                                          8099\n",
       "plant_tribes_assembly_post_processor                         8095\n",
       "ucsc-bedextendranges                                         8095\n",
       "bioconductor-rtracklayer                                     8090\n",
       "ucsc-bedcommonregions                                        8087\n",
       "ucsc-crtreeindexbed                                          8058\n",
       "ucsc-chainsort                                               8057\n",
       "ucsc-bedcoverage                                             8052\n",
       "ucsc-maskoutfa                                               8051\n",
       "ucsc-crtreesearchbed                                         8047\n",
       "ucsc-axttomaf                                                8043\n",
       "ucsc-chaintopslbasic                                         8035\n",
       "ucsc-chaintoaxt                                              8032\n",
       "ucsc-chainfilter                                             8027\n",
       "ucsc-bedremoveoverlap                                        8024\n",
       "ucsc-netclass                                                8020\n",
       "sirius-csifingerid                                           8017\n",
       "ucsc-axttopsl                                                8008\n",
       "ragout                                                       8007\n",
       "ucsc-bedintersect                                            8003\n",
       "ucsc-addcols                                                 8000\n",
       "ucsc-gaptolift                                               7997\n",
       "ucsc-getrna                                                  7995\n",
       "ucsc-nettoaxt                                                7995\n",
       "treetime                                                     7993\n",
       "ucsc-featurebits                                             7992\n",
       "perl-yaml-libyaml                                            7989\n",
       "perl-mce                                                     7989\n",
       "ucsc-getrnapred                                              7985\n",
       "ucsc-bedweedoverlapping                                      7983\n",
       "wiggletools                                                  7983\n",
       "ucsc-bedpileups                                              7980\n",
       "ucsc-farc                                                    7968\n",
       "ucsc-chainswap                                               7968\n",
       "ucsc-bedtopsl                                                7966\n",
       "ucsc-fatotab                                                 7962\n",
       "ucsc-bedgeneparts                                            7957\n",
       "ucsc-chaintopsl                                              7955\n",
       "moca                                                         7954\n",
       "ucsc-farandomize                                             7950\n",
       "ucsc-netsyntenic                                             7945\n",
       "ucsc-chainantirepeat                                         7941\n",
       "ucsc-genepredhisto                                           7931\n",
       "perl-namespace-autoclean                                     7894\n",
       "perl-biox-workflow-command                                   7872\n",
       "mdasim                                                       7871\n",
       "sickle                                                       7857\n",
       "pandaseq                                                     7844\n",
       "savage                                                       7836\n",
       "omssa                                                        7798\n",
       "python-hivclustering                                         7798\n",
       "moods                                                        7790\n",
       "ucsc-findmotif                                               7786\n",
       "perl-io-all                                                  7774\n",
       "schavott                                                     7769\n",
       "starseqr                                                     7746\n",
       "ucsc-estorient                                               7728\n",
       "twobitreader                                                 7717\n",
       "multigps                                                     7714\n",
       "smeg                                                         7705\n",
       "r-aroma.core                                                 7659\n",
       "perl-devel-size                                              7607\n",
       "yara                                                         7599\n",
       "maxentscan                                                   7595\n",
       "hmmer                                                        7583\n",
       "luciphor2                                                    7580\n",
       "bbmap                                                        7568\n",
       "perl-date-manip                                              7564\n",
       "pyquant-ms                                                   7514\n",
       "metagenome-atlas                                             7499\n",
       "simlord                                                      7491\n",
       "perl-test-fatal                                              7465\n",
       "r-speaq                                                      7441\n",
       "r-aroma.affymetrix                                           7435\n",
       "vawk                                                         7412\n",
       "mbuffer                                                      7388\n",
       "perl-inline-c                                                7387\n",
       "perl-sanger-cgp-battenberg                                   7368\n",
       "metameta                                                     7344\n",
       "shiver                                                       7320\n",
       "r-histonehmm                                                 7294\n",
       "nasp                                                         7287\n",
       "picrust2                                                     7276\n",
       "python-hyphy-python                                          7274\n",
       "fastp                                                        7260\n",
       "bioconductor-rhtslib                                         7232\n",
       "verifybamid                                                  7200\n",
       "perl-scalar-util-numeric                                     7194\n",
       "perl-test-warnings                                           7174\n",
       "s3gof3r                                                      7168\n",
       "ucsc-facount                                                 7163\n",
       "toulligqc                                                    7106\n",
       "r-deconstructsigs                                            7099\n",
       "ucsc-axtchain                                                7087\n",
       "ococo                                                        7084\n",
       "perl-clone-choose                                            7044\n",
       "repeatmodeler                                                7016\n",
       "phylip                                                       6999\n",
       "plastid                                                      6996\n",
       "theta2                                                       6992\n",
       "ucsc-ixixx                                                   6962\n",
       "obitools                                                     6945\n",
       "nudup                                                        6931\n",
       "nextgenmap                                                   6928\n",
       "pyseer                                                       6921\n",
       "plink2                                                       6890\n",
       "trackhub                                                     6884\n",
       "mimodd                                                       6862\n",
       "scrm                                                         6861\n",
       "mawk                                                         6831\n",
       "srst2                                                        6801\n",
       "perl-specio-exporter                                         6801\n",
       "segemehl                                                     6795\n",
       "rust-ncbitaxonomy                                            6786\n",
       "snoscan                                                      6777\n",
       "perl-b                                                       6746\n",
       "r-goalie                                                     6719\n",
       "perl-image-exiftool                                          6708\n",
       "tiddit                                                       6666\n",
       "rnaz                                                         6658\n",
       "ucsc-mafsinregion                                            6654\n",
       "perl-path-class                                              6644\n",
       "ucsc-autoxml                                                 6630\n",
       "recycler                                                     6630\n",
       "perl-test-class-moose                                        6605\n",
       "meraculous                                                   6600\n",
       "ucsc-maffilter                                               6588\n",
       "ucsc-bigpsltopsl                                             6582\n",
       "ucsc-axtsort                                                 6576\n",
       "ucsc-chromgraphtobin                                         6574\n",
       "virmet                                                       6572\n",
       "ucsc-axtswap                                                 6570\n",
       "ucsc-mafsplitpos                                             6565\n",
       "entrez-direct                                                6557\n",
       "perl-params-validationcompiler                               6550\n",
       "perl-moosex-app                                              6550\n",
       "shmlast                                                      6546\n",
       "ucsc-liftup                                                  6543\n",
       "ucsc-autodtd                                                 6542\n",
       "ucsc-dbsnoop                                                 6539\n",
       "perl-warnings-register                                       6539\n",
       "ucsc-mafspecieslist                                          6535\n",
       "ucsc-fafilter                                                6532\n",
       "ucsc-gff3topsl                                               6531\n",
       "ucsc-mafspeciessubset                                        6529\n",
       "ucsc-mafaddqrows                                             6527\n",
       "perl-bio-viennangs                                           6521\n",
       "ucsc-dbtrash                                                 6519\n",
       "ucsc-maftosnpbed                                             6517\n",
       "ucsc-chromgraphfrombin                                       6500\n",
       "perl-datetime                                                6498\n",
       "vcfpy                                                        6497\n",
       "ucsc-checkagpandfa                                           6496\n",
       "ucsc-netfilter                                               6493\n",
       "ucsc-maftoaxt                                                6485\n",
       "ucsc-netchainsubset                                          6483\n",
       "ucsc-hgloadwiggle                                            6480\n",
       "transabyss                                                   6478\n",
       "ucsc-lavtopsl                                                6475\n",
       "ucsc-autosql                                                 6474\n",
       "perl-test-output                                             6474\n",
       "ucsc-faalign                                                 6473\n",
       "ucsc-catdir                                                  6473\n",
       "tadarida-d                                                   6461\n",
       "ucsc-hgloadchain                                             6453\n",
       "ucsc-ameme                                                   6451\n",
       "pymvpa                                                       6436\n",
       "ucsc-mafcoverage                                             6434\n",
       "ucsc-coltransform                                            6432\n",
       "perl-datetime-locale                                         6425\n",
       "ucsc-fafrag                                                  6425\n",
       "perl-datetime-timezone                                       6422\n",
       "ucsc-mafgene                                                 6422\n",
       "ucsc-chopfalines                                             6415\n",
       "ucsc-maffrags                                                6414\n",
       "ucsc-checktablecoords                                        6413\n",
       "ucsc-catuncomment                                            6406\n",
       "ucsc-checkcoveragegaps                                       6402\n",
       "ucsc-maffetch                                                6397\n",
       "ucsc-fanoise                                                 6396\n",
       "ucsc-avecols                                                 6390\n",
       "ucsc-fafiltern                                               6388\n",
       "ucsc-facmp                                                   6381\n",
       "ucsc-mafsplit                                                6377\n",
       "ucsc-countchars                                              6375\n",
       "ucsc-maffrag                                                 6373\n",
       "ucsc-mafranges                                               6370\n",
       "ucsc-mafmefirst                                              6365\n",
       "ucsc-hgloadmaf                                               6363\n",
       "pb-dazzler                                                   6361\n",
       "openslide-python                                             6361\n",
       "ucsc-checkhgfindspec                                         6358\n",
       "ucsc-maftopsl                                                6357\n",
       "ucsc-hgloadbed                                               6357\n",
       "ucsc-maforder                                                6347\n",
       "snp-pipeline                                                 6321\n",
       "ucsc-mafaddirows                                             6317\n",
       "pronto                                                       6313\n",
       "wdltool                                                      6284\n",
       "pbalign                                                      6283\n",
       "pyprophet                                                    6276\n",
       "perl-mce-shared                                              6265\n",
       "r-stitch                                                     6260\n",
       "mgltools                                                     6253\n",
       "socru                                                        6240\n",
       "python-consensuscore2                                        6189\n",
       "snpsplit                                                     6181\n",
       "nmrglue                                                      6164\n",
       "metilene                                                     6151\n",
       "mintmap                                                      6147\n",
       "prosic                                                       6146\n",
       "perl-module-scandeps                                         6139\n",
       "pybel                                                        6131\n",
       "trnanalysis                                                  6120\n",
       "perl-posix                                                   6118\n",
       "perl-object-insideout                                        6087\n",
       "readseq                                                      6084\n",
       "trimadap                                                     6015\n",
       "rnastructure                                                 6003\n",
       "slamdunk                                                     5996\n",
       "weeder                                                       5984\n",
       "rnashapes                                                    5982\n",
       "midas                                                        5962\n",
       "pepr                                                         5949\n",
       "msaboot                                                      5937\n",
       "perl-canary-stability                                        5907\n",
       "perl-statistics-distributions                                5892\n",
       "opsin                                                        5885\n",
       "perl-moosex-getopt                                           5876\n",
       "perl-file-homedir                                            5864\n",
       "ucsc-blat                                                    5863\n",
       "migmap                                                       5834\n",
       "r-samr                                                       5819\n",
       "purge_haplotigs                                              5784\n",
       "perl-bio-rna-rnaalisplit                                     5756\n",
       "seqprep                                                      5744\n",
       "phyloflash                                                   5737\n",
       "pyfastx                                                      5697\n",
       "bioconductor-noiseq                                          5679\n",
       "r-syntactic                                                  5665\n",
       "pysradb                                                      5658\n",
       "perl-forks                                                   5647\n",
       "osra                                                         5639\n",
       "svviz                                                        5631\n",
       "perl-perl-unsafe-signals                                     5628\n",
       "quicktree                                                    5615\n",
       "perl-pegex                                                   5582\n",
       "tassel                                                       5571\n",
       "mrbayes                                                      5568\n",
       "repeatscout                                                  5561\n",
       "python-bioext                                                5556\n",
       "perl-bit-vector                                              5553\n",
       "wtforms-alchemy                                              5509\n",
       "metfrag                                                      5507\n",
       "rnablueprint                                                 5503\n",
       "transit                                                      5501\n",
       "taxtastic                                                    5480\n",
       "perl-sanger-cgp-vagrent                                      5470\n",
       "rebaler                                                      5436\n",
       "nanosim-h                                                    5432\n",
       "mir-prefer                                                   5429\n",
       "neptune                                                      5416\n",
       "plant_tribes_gene_family_phylogeny_builder                   5403\n",
       "super-focus                                                  5394\n",
       "perl-test-builder-tester                                     5391\n",
       "perl-want                                                    5390\n",
       "pypore                                                       5364\n",
       "perl-io-prompt                                               5348\n",
       "mafft                                                        5345\n",
       "taxmapper                                                    5345\n",
       "mashmap                                                      5341\n",
       "rainbow                                                      5337\n",
       "mason                                                        5302\n",
       "pdfkit                                                       5298\n",
       "tirmite                                                      5298\n",
       "socksipy-branch                                              5296\n",
       "trawler                                                      5288\n",
       "raven-assembler                                              5280\n",
       "plasmidtron                                                  5270\n",
       "plant_tribes_gene_family_integrator                          5263\n",
       "perl-memoize                                                 5252\n",
       "perl-excel-writer-xlsx                                       5241\n",
       "perl-env-path                                                5236\n",
       "scrappie                                                     5220\n",
       "yaggo                                                        5213\n",
       "rename                                                       5196\n",
       "perl-test-files                                              5184\n",
       "mtsv                                                         5181\n",
       "deeptools                                                    5174\n",
       "perl-acme-damn                                               5166\n",
       "xyalign                                                      5143\n",
       "plant_tribes_kaks_analysis                                   5122\n",
       "shortstack                                                   5118\n",
       "pycluster                                                    5109\n",
       "pbgcpp                                                       5107\n",
       "swga                                                         5105\n",
       "paladin                                                      5099\n",
       "perl-extutils-helpers                                        5095\n",
       "rad_haplotyper                                               5095\n",
       "rdptools                                                     5094\n",
       "pybedgraph                                                   5090\n",
       "ucsc-transmappsltogenepred                                   5086\n",
       "r-exomedepth                                                 5068\n",
       "recon                                                        5068\n",
       "pycli                                                        5063\n",
       "perl-velvetoptimiser                                         5062\n",
       "plant_tribes_gene_family_aligner                             5062\n",
       "perl-heap-simple-xs                                          5060\n",
       "seqmagick                                                    5049\n",
       "sparse-neighbors-search                                      5047\n",
       "perl-bio-gff3                                                5044\n",
       "ucsc-pslmappostchain                                         5035\n",
       "perl-moosex-types                                            5016\n",
       "ucsc-bamtopsl                                                5000\n",
       "perl-digest-crc32                                            4974\n",
       "python-libsbml                                               4972\n",
       "ema                                                          4952\n",
       "bx-python                                                    4947\n",
       "treebest                                                     4945\n",
       "probamconvert                                                4944\n",
       "ucsc-expmatrixtobarchartbed                                  4931\n",
       "perl-io-interactive                                          4930\n",
       "perl-file-readbackwards                                      4929\n",
       "metaseq                                                      4921\n",
       "pasa                                                         4919\n",
       "adapterremoval                                               4908\n",
       "snpiphy                                                      4905\n",
       "perl-sys-sigaction                                           4904\n",
       "meta-sparse                                                  4904\n",
       "regtools                                                     4896\n",
       "perl-config-any                                              4874\n",
       "phantompeakqualtools                                         4860\n",
       "presto                                                       4860\n",
       "tadarida-c                                                   4859\n",
       "perl-heap-simple-perl                                        4857\n",
       "xxmotif                                                      4852\n",
       "bioconductor-limma                                           4835\n",
       "perl-attribute-handlers                                      4814\n",
       "ucsc-fastqstatsandsubsample                                  4799\n",
       "ucsc-hubcheck                                                4792\n",
       "ucsc-clustergenes                                            4784\n",
       "shustring                                                    4783\n",
       "r-rubic                                                      4782\n",
       "perl-heap-simple                                             4777\n",
       "ucsc-hgloadout                                               4755\n",
       "newick_utils                                                 4752\n",
       "ucsc-bigmaftomaf                                             4741\n",
       "ucsc-hgloadnet                                               4735\n",
       "pispino                                                      4733\n",
       "perl-file-next                                               4732\n",
       "pytriangle                                                   4727\n",
       "sequencetools                                                4726\n",
       "ucsc-lavtoaxt                                                4726\n",
       "sga                                                          4725\n",
       "ucsc-linestora                                               4717\n",
       "ucsc-maftobigmaf                                             4716\n",
       "perl-unicode-normalize                                       4715\n",
       "perl-module-loaded                                           4713\n",
       "ucsc-hgfakeagp                                               4711\n",
       "treemix                                                      4704\n",
       "vcf2genome                                                   4704\n",
       "ucsc-mrnatogene                                              4701\n",
       "ucsc-genepredtoprot                                          4698\n",
       "ucsc-netsplit                                                4693\n",
       "python-sortedcontainers                                      4690\n",
       "iqtree                                                       4687\n",
       "ucsc-hgspeciesrna                                            4686\n",
       "perl-graphics-colorobject                                    4673\n",
       "ucsc-hgsqldump                                               4673\n",
       "ucsc-hgloadoutjoined                                         4673\n",
       "perl-graphics-colornames                                     4671\n",
       "t-coffee                                                     4671\n",
       "ppanggolin                                                   4658\n",
       "ucsc-gmtime                                                  4651\n",
       "mzml2isa                                                     4650\n",
       "ribotish                                                     4647\n",
       "minimap                                                      4646\n",
       "protrac                                                      4642\n",
       "ucsc-ldhggene                                                4641\n",
       "r-phytools                                                   4622\n",
       "ucsc-hgbbidblink                                             4618\n",
       "ucsc-pslrc                                                   4610\n",
       "ucsc-htmlcheck                                               4608\n",
       "ucsc-maketablelist                                           4604\n",
       "ucsc-newprog                                                 4602\n",
       "ucsc-hgfindspec                                              4600\n",
       "ribotaper                                                    4598\n",
       "perl-text-balanced                                           4597\n",
       "ucsc-hgtrackdb                                               4595\n",
       "metasnv                                                      4595\n",
       "maxentpy                                                     4593\n",
       "ucsc-mktime                                                  4587\n",
       "python-intervaltree                                          4586\n",
       "mapsplice                                                    4582\n",
       "ucsc-headrest                                                4580\n",
       "vqsr_cnn                                                     4579\n",
       "variant_tools                                                4576\n",
       "bioconductor-annotationdbi                                   4575\n",
       "malva                                                        4573\n",
       "bcbio-nextgen                                                4567\n",
       "perl-test-inter                                              4563\n",
       "ucsc-localtime                                               4563\n",
       "r-cellassign                                                 4555\n",
       "spydrpick                                                    4545\n",
       "ucsc-hubpubliccheck                                          4544\n",
       "selscan                                                      4540\n",
       "rnasamba                                                     4537\n",
       "mauve                                                        4514\n",
       "perl-sys-info-driver-linux                                   4505\n",
       "mathstats                                                    4486\n",
       "pasta                                                        4476\n",
       "perl-statistics-ttest                                        4469\n",
       "phylopandas                                                  4469\n",
       "scoop                                                        4462\n",
       "wub                                                          4442\n",
       "mztosqlite                                                   4441\n",
       "sixgill                                                      4433\n",
       "diamond                                                      4416\n",
       "pyx                                                          4410\n",
       "qorts                                                        4406\n",
       "skmer                                                        4404\n",
       "smalt                                                        4399\n",
       "screamingbackpack                                            4388\n",
       "poppunk                                                      4383\n",
       "rapsearch                                                    4378\n",
       "scoary                                                       4367\n",
       "sicer                                                        4361\n",
       "strainest                                                    4351\n",
       "sibelia                                                      4345\n",
       "pbcoretools                                                  4317\n",
       "selectsequencesfrommsa                                       4308\n",
       "motifscan                                                    4306\n",
       "perl-io-uncompress-gunzip                                    4306\n",
       "perl-perl-version                                            4305\n",
       "r-nanostringnorm                                             4303\n",
       "ushuffle                                                     4301\n",
       "python-cluster                                               4297\n",
       "perl-statistics-r                                            4291\n",
       "micall-lite                                                  4276\n",
       "musicc                                                       4274\n",
       "r-mutoss                                                     4253\n",
       "metametamerge                                                4251\n",
       "metawrap                                                     4248\n",
       "pp                                                           4246\n",
       "pyexcelerator                                                4233\n",
       "pysvg                                                        4221\n",
       "perl-version-next                                            4214\n",
       "pyensembl                                                    4189\n",
       "perl-yaml-tiny                                               4175\n",
       "r-classdiscovery                                             4173\n",
       "soapdenovo2                                                  4161\n",
       "pisces                                                       4158\n",
       "nucdiff                                                      4155\n",
       "swalign                                                      4154\n",
       "sonicparanoid                                                4145\n",
       "bamtools                                                     4141\n",
       "metavelvet                                                   4116\n",
       "refseq_masher                                                4112\n",
       "smallgenomeutilities                                         4112\n",
       "perl-text-wrap                                               4108\n",
       "pydnase                                                      4104\n",
       "r-gkmsvm                                                     4103\n",
       "tgt                                                          4101\n",
       "r-minems2                                                    4097\n",
       "python-newick                                                4088\n",
       "dendropy                                                     4080\n",
       "segtools                                                     4075\n",
       "segmentation-fold                                            4048\n",
       "minnow                                                       4042\n",
       "oases                                                        4030\n",
       "sword                                                        4027\n",
       "pheniqs                                                      4024\n",
       "tracer                                                       4023\n",
       "simka                                                        4016\n",
       "snp-pileup                                                   4014\n",
       "mreps                                                        4007\n",
       "phylotoast                                                   4007\n",
       "r-probmetab                                                  4004\n",
       "nucleoatac                                                   4002\n",
       "seq-seq-pan                                                  4000\n",
       "perl-carp-clan                                               3987\n",
       "pythomics                                                    3987\n",
       "rmats2sashimiplot                                            3982\n",
       "pfam_scan                                                    3965\n",
       "perl-text-asciitable                                         3962\n",
       "metagene_annotator                                           3960\n",
       "perl-logger-simple                                           3959\n",
       "randfold                                                     3954\n",
       "mercat                                                       3950\n",
       "rapclust                                                     3949\n",
       "mauvealigner                                                 3949\n",
       "maxquant                                                     3944\n",
       "ncrf                                                         3938\n",
       "sqlalchemy-datatables                                        3925\n",
       "ray                                                          3923\n",
       "rgt                                                          3920\n",
       "stacks_summary                                               3913\n",
       "pourrna                                                      3902\n",
       "parsnp                                                       3890\n",
       "orfm                                                         3889\n",
       "mvp                                                          3889\n",
       "vcfkit                                                       3874\n",
       "pydna                                                        3870\n",
       "perl-moosex-fileattribute                                    3859\n",
       "ensembl-vep                                                  3846\n",
       "perl-xml-semanticdiff                                        3836\n",
       "umitools                                                     3829\n",
       "tablet                                                       3819\n",
       "r-swamp                                                      3818\n",
       "perl-json-maybexs                                            3818\n",
       "perl-io-handle                                               3801\n",
       "perl-git-wrapper                                             3798\n",
       "perl-test-notabs                                             3789\n",
       "perl-parallel-loops                                          3787\n",
       "pullseq                                                      3786\n",
       "perl-list-someutils                                          3780\n",
       "smcounter2                                                   3777\n",
       "r-gwpcr                                                      3777\n",
       "pysnptools                                                   3760\n",
       "pybamtools                                                   3755\n",
       "tiptoft                                                      3755\n",
       "bioconductor-dada2                                           3751\n",
       "perl-test-yaml                                               3730\n",
       "r-ic10                                                       3725\n",
       "pathoscope                                                   3725\n",
       "pvacseq                                                      3723\n",
       "perl-xml-parser-lite                                         3722\n",
       "perl-json-parse                                              3719\n",
       "squizz                                                       3718\n",
       "mhap                                                         3717\n",
       "microbecensus                                                3705\n",
       "ped_parser                                                   3699\n",
       "perl-test-trap                                               3699\n",
       "perl-config-simple                                           3695\n",
       "perl-algorithm-dependency                                    3683\n",
       "rnachipintegrator                                            3680\n",
       "pycoqc                                                       3679\n",
       "r-mmgenome                                                   3675\n",
       "perl-test-base                                               3674\n",
       "pneumocat                                                    3673\n",
       "maaslin                                                      3673\n",
       "r-pore                                                       3666\n",
       "taco                                                         3665\n",
       "galaxy-lib                                                   3664\n",
       "perl-test-eol                                                3661\n",
       "fasttree                                                     3658\n",
       "perl-test-nowarnings                                         3651\n",
       "bioconductor-genefilter                                      3648\n",
       "sharedmem                                                    3638\n",
       "gatk4                                                        3636\n",
       "perl-term-app-roles                                          3635\n",
       "yanagiba                                                     3616\n",
       "mtnucratio                                                   3613\n",
       "wgsim                                                        3606\n",
       "mosaik                                                       3603\n",
       "womtool                                                      3600\n",
       "python-hppy                                                  3600\n",
       "perl-extutils-installpaths                                   3597\n",
       "sis                                                          3594\n",
       "minia                                                        3591\n",
       "nvc                                                          3591\n",
       "telseq                                                       3581\n",
       "morpheus                                                     3571\n",
       "perl-socket6                                                 3567\n",
       "perl-module-fromperlver                                      3567\n",
       "perl-test-classapi                                           3564\n",
       "perl-findbin-libs                                            3560\n",
       "visceral-evaluatesegmentation                                3559\n",
       "msproteomicstools                                            3554\n",
       "perl-git-wrapper-plus                                        3550\n",
       "relocate2                                                    3550\n",
       "stamp                                                        3535\n",
       "r-fgwas                                                      3522\n",
       "r-sbpiper                                                    3521\n",
       "perl-color-theme                                             3517\n",
       "perl-term-progressbar                                        3517\n",
       "r-tcr                                                        3504\n",
       "perl-datetime-format-strptime                                3489\n",
       "xatlas                                                       3482\n",
       "squid                                                        3479\n",
       "peakranger                                                   3476\n",
       "perl-local-lib                                               3468\n",
       "treekin                                                      3465\n",
       "pymisc-utils                                                 3464\n",
       "msms                                                         3456\n",
       "prophex                                                      3448\n",
       "r-dartr                                                      3445\n",
       "msisensor-pro                                                3441\n",
       "sequenza-utils                                               3438\n",
       "seq-gen                                                      3433\n",
       "myriad                                                       3430\n",
       "abundancebin                                                 3422\n",
       "pyrad                                                        3416\n",
       "piranha                                                      3413\n",
       "snakeparse                                                   3411\n",
       "seedme                                                       3410\n",
       "bioconductor-shortread                                       3404\n",
       "seq2hla                                                      3403\n",
       "perl-clone-pp                                                3401\n",
       "udocker                                                      3398\n",
       "perl-try-tiny-retry                                          3386\n",
       "pypiper                                                      3383\n",
       "ppanini                                                      3381\n",
       "rnasketch                                                    3378\n",
       "r-scimpute                                                   3376\n",
       "panx                                                         3375\n",
       "swipe                                                        3372\n",
       "r-misha                                                      3370\n",
       "nose-capturestderr                                           3369\n",
       "r-sigqc                                                      3365\n",
       "perl-config-autoconf                                         3362\n",
       "perl-authen-sasl-saslprep                                    3356\n",
       "ucsc-overlapselect                                           3355\n",
       "merlin                                                       3353\n",
       "tepid                                                        3350\n",
       "pygresql                                                     3331\n",
       "perl-file-share                                              3327\n",
       "perl-moosex-types-path-class                                 3326\n",
       "mamotif                                                      3323\n",
       "pbhoover                                                     3318\n",
       "bioconductor-annotate                                        3317\n",
       "sccaller                                                     3313\n",
       "tpp                                                          3310\n",
       "r-workflowscriptscommon                                      3306\n",
       "hisat2                                                       3302\n",
       "nanovar                                                      3297\n",
       "clustalw                                                     3296\n",
       "smina                                                        3295\n",
       "perl-math-random-mt-auto                                     3290\n",
       "seqyclean                                                    3277\n",
       "oligotyping                                                  3276\n",
       "perl-sys-info-base                                           3274\n",
       "tantan                                                       3273\n",
       "pgma-simple                                                  3269\n",
       "perl-class-accessor                                          3266\n",
       "rapidnj                                                      3258\n",
       "plascope                                                     3256\n",
       "monocle3-cli                                                 3251\n",
       "phispy                                                       3247\n",
       "ntcard                                                       3243\n",
       "translate-gard                                               3232\n",
       "perl-sys-info                                                3230\n",
       "migraine                                                     3222\n",
       "structure                                                    3214\n",
       "perl-proc-fork                                               3212\n",
       "magpurify                                                    3211\n",
       "perl-pod-elemental                                           3207\n",
       "r2r                                                          3205\n",
       "textinput                                                    3202\n",
       "yahmm                                                        3199\n",
       "scvi                                                         3193\n",
       "progressivemauve                                             3177\n",
       "orthomcl                                                     3176\n",
       "tb-profiler                                                  3171\n",
       "mapdia                                                       3169\n",
       "perl-text-tabs-wrap                                          3169\n",
       "bioconductor-deseq2                                          3163\n",
       "paragraph                                                    3159\n",
       "r-nanopore                                                   3154\n",
       "mfqe                                                         3152\n",
       "perl-specio                                                  3151\n",
       "pyega3                                                       3150\n",
       "unitig-caller                                                3140\n",
       "velocyto.py                                                  3123\n",
       "perl-mixin-linewise                                          3121\n",
       "medpy                                                        3118\n",
       "raxml-ng                                                     3117\n",
       "pmdtools                                                     3107\n",
       "r-msqrob                                                     3106\n",
       "snvphyl-tools                                                3105\n",
       "perl-go-perl                                                 3100\n",
       "perl-extutils-depends                                        3091\n",
       "freebayes                                                    3075\n",
       "nspdk                                                        3074\n",
       "spoa                                                         3071\n",
       "perl-math-bigint                                             3070\n",
       "bioconductor-edger                                           3070\n",
       "perl-test-sys-info                                           3065\n",
       "pytest-marks                                                 3065\n",
       "scala                                                        3063\n",
       "peakachu                                                     3045\n",
       "mapdamage2                                                   3044\n",
       "squeakr                                                      3027\n",
       "rnaquast                                                     3024\n",
       "perl-test-cleannamespaces                                    3022\n",
       "perl-extutils-constant                                       3020\n",
       "phipack                                                      3018\n",
       "vcfsamplecompare                                             3013\n",
       "paralyzer                                                    3001\n",
       "perl-term-encoding                                           3000\n",
       "mykatlas                                                     2995\n",
       "plass                                                        2995\n",
       "perl-data-compare                                            2988\n",
       "ls-gkm                                                       2985\n",
       "proteowizard                                                 2984\n",
       "semidbm                                                      2980\n",
       "unitig-counter                                               2976\n",
       "targetfinder                                                 2973\n",
       "ucsc-paranodestop                                            2970\n",
       "rvtests                                                      2968\n",
       "var-agg                                                      2965\n",
       "kallisto                                                     2958\n",
       "ucsc-parahubstop                                             2945\n",
       "ucsc-paratestjob                                             2940\n",
       "ucsc-endsinlf                                                2937\n",
       "ucsc-paranode                                                2937\n",
       "ucsc-bedjointaboffset                                        2935\n",
       "rnacode                                                      2932\n",
       "ucsc-para                                                    2931\n",
       "perl-test-needs                                              2931\n",
       "ucsc-pslsomerecords                                          2928\n",
       "perl-const-fast                                              2926\n",
       "r-tidygenomics                                               2924\n",
       "ucsc-websync                                                 2923\n",
       "bioconductor-geneplotter                                     2919\n",
       "python-fakemp                                                2915\n",
       "virsorter                                                    2904\n",
       "miranda                                                      2898\n",
       "ucsc-hgloadsqltab                                            2896\n",
       "ucsc-paranodestatus                                          2895\n",
       "ucsc-parasol                                                 2893\n",
       "cooler                                                       2887\n",
       "ucsc-paranodestart                                           2882\n",
       "ucsc-gensub2                                                 2881\n",
       "perl-test-requires                                           2870\n",
       "perl-string-escape                                           2869\n",
       "ucsc-parahub                                                 2868\n",
       "mdust                                                        2844\n",
       "cnvkit                                                       2842\n",
       "nimnexus                                                     2837\n",
       "melt                                                         2832\n",
       "ngsutils                                                     2819\n",
       "perl-data-match                                              2814\n",
       "misopy                                                       2806\n",
       "mgf-formatter                                                2790\n",
       "optplus                                                      2774\n",
       "suppa                                                        2749\n",
       "tabview                                                      2746\n",
       "onto2nx                                                      2746\n",
       "smhasher                                                     2745\n",
       "sprinkles                                                    2744\n",
       "prosampler                                                   2740\n",
       "perl-module-build-tiny                                       2731\n",
       "r-pctgcdata                                                  2728\n",
       "sbg-cwl-runner                                               2720\n",
       "scallop                                                      2709\n",
       "setuptools_cython                                            2709\n",
       "r-ngsplot                                                    2702\n",
       "sample-sheet                                                 2696\n",
       "perl-html-form                                               2693\n",
       "targqc                                                       2688\n",
       "bioconductor-biomart                                         2686\n",
       "r-facets                                                     2676\n",
       "ropebwt2                                                     2669\n",
       "bioconductor-rhdf5                                           2665\n",
       "optbuild                                                     2658\n",
       "perl-sys-info-driver-osx                                     2653\n",
       "deblur                                                       2644\n",
       "mccortex                                                     2617\n",
       "workspace                                                    2617\n",
       "bioconductor-genomicfeatures                                 2617\n",
       "r-monocle3                                                   2614\n",
       "perl-tie-log4perl                                            2613\n",
       "perl-spiffy                                                  2611\n",
       "pathwaymatcher                                               2611\n",
       "sexdeterrmine                                                2601\n",
       "perl-cpan-shell                                              2601\n",
       "bam-readcount                                                2597\n",
       "modeltest-ng                                                 2581\n",
       "malt                                                         2572\n",
       "vg                                                           2559\n",
       "r-sequenza                                                   2553\n",
       "sparc                                                        2552\n",
       "pipmir                                                       2551\n",
       "perl-data-dump                                               2549\n",
       "gffutils                                                     2548\n",
       "metaphlan                                                    2546\n",
       "muse                                                         2539\n",
       "r-nastiseq                                                   2537\n",
       "arvados-cwl-runner                                           2537\n",
       "perl-string-approx                                           2530\n",
       "rnabob                                                       2521\n",
       "gneiss                                                       2516\n",
       "arvados-python-client                                        2510\n",
       "r-taxonomizr                                                 2509\n",
       "nseg                                                         2505\n",
       "sparseassembler                                              2497\n",
       "perl-sub-exporter-formethods                                 2489\n",
       "potrace                                                      2487\n",
       "r-empiricalfdr.deseq2                                        2485\n",
       "mpa-portable                                                 2479\n",
       "svict                                                        2462\n",
       "perl-sql-statement                                           2457\n",
       "wtforms-components                                           2454\n",
       "rust-overlaps                                                2454\n",
       "macrel                                                       2453\n",
       "wgs2ncbi                                                     2452\n",
       "perl-sort-versions                                           2450\n",
       "seqwish                                                      2445\n",
       "minorseq                                                     2440\n",
       "thapbi-pict                                                  2439\n",
       "perl-math-base-convert                                       2435\n",
       "rnasnp                                                       2427\n",
       "bioconductor-rhdf5lib                                        2426\n",
       "multivcfanalyzer                                             2416\n",
       "perl-fastx-reader                                            2415\n",
       "plek                                                         2407\n",
       "hicexplorer                                                  2404\n",
       "staden_io_lib                                                2402\n",
       "xmlbuilder                                                   2402\n",
       "soapdenovo2-gapcloser                                        2395\n",
       "mockinbird                                                   2393\n",
       "megan                                                        2393\n",
       "rdock                                                        2392\n",
       "plasflow                                                     2388\n",
       "scikits-datasmooth                                           2384\n",
       "megalodon                                                    2383\n",
       "perl-bio-eutilities                                          2383\n",
       "tadtool                                                      2375\n",
       "pycsg                                                        2372\n",
       "infernal                                                     2367\n",
       "vamb                                                         2365\n",
       "sam                                                          2361\n",
       "peakzilla                                                    2358\n",
       "pyimzml                                                      2355\n",
       "seqan_tcoffee                                                2352\n",
       "perl-file-chdir                                              2350\n",
       "shape_it                                                     2347\n",
       "pbh5tools                                                    2346\n",
       "r-xmlrpc                                                     2345\n",
       "perl-bio-cigar                                               2342\n",
       "csvtk                                                        2339\n",
       "transrate-tools                                              2339\n",
       "r-rtfbs                                                      2339\n",
       "mea                                                          2337\n",
       "poa                                                          2333\n",
       "soapec                                                       2324\n",
       "perl-file-util                                               2319\n",
       "pyasp                                                        2319\n",
       "r-htssip                                                     2318\n",
       "fwdpy11                                                      2315\n",
       "smartdenovo                                                  2314\n",
       "smudgeplot                                                   2313\n",
       "perl-string-util                                             2311\n",
       "perl-number-misc                                             2306\n",
       "perl-tie-cache                                               2305\n",
       "openmg                                                       2305\n",
       "r-tcga2stat                                                  2303\n",
       "perl-file-tee                                                2300\n",
       "yanc                                                         2299\n",
       "sciphi                                                       2296\n",
       "r-structssi                                                  2294\n",
       "perl-class-xsaccessor                                        2293\n",
       "sonlib                                                       2289\n",
       "strip_it                                                     2281\n",
       "perl-extutils-config                                         2277\n",
       "r-ichorcna                                                   2276\n",
       "qtip                                                         2276\n",
       "novoplasty                                                   2275\n",
       "shapeit4                                                     2274\n",
       "perl-test-toolbox                                            2273\n",
       "perl-devel-checkbin                                          2273\n",
       "super_distance                                               2262\n",
       "qcat                                                         2261\n",
       "pbsim                                                        2259\n",
       "bedops                                                       2257\n",
       "r-metalonda                                                  2256\n",
       "geneimpacts                                                  2254\n",
       "rdp_classifier                                               2251\n",
       "secimtools                                                   2248\n",
       "minimock                                                     2242\n",
       "starfish                                                     2241\n",
       "r-sigtree                                                    2235\n",
       "metavelvet-sl-feature-extraction                             2234\n",
       "picard-slim                                                  2229\n",
       "nullarbor                                                    2227\n",
       "perl-mldbm-sync                                              2223\n",
       "novobreak                                                    2222\n",
       "seroba                                                       2221\n",
       "pbdagcon                                                     2221\n",
       "somatic-sniper                                               2221\n",
       "magicblast                                                   2219\n",
       "transtermhp                                                  2218\n",
       "validate-fasta-database                                      2218\n",
       "mugsy                                                        2217\n",
       "ska                                                          2217\n",
       "perl-term-detect-software                                    2215\n",
       "outrigger                                                    2215\n",
       "perl-tie-refhash                                             2214\n",
       "pp-sketchlib                                                 2206\n",
       "soapdenovo-trans                                             2206\n",
       "query_phenomizer                                             2203\n",
       "perl-test-xml                                                2195\n",
       "tetranscripts                                                2194\n",
       "perl-test-cpan-meta                                          2189\n",
       "uniprot                                                      2186\n",
       "medusa-data-fusion                                           2185\n",
       "takeabreak                                                   2181\n",
       "methylpy                                                     2180\n",
       "perl-list-uniq                                               2179\n",
       "samsifter                                                    2177\n",
       "vcftoolbox                                                   2177\n",
       "perl-perl-osnames                                            2170\n",
       "mapsembler2                                                  2169\n",
       "meryl                                                        2168\n",
       "treesapp                                                     2165\n",
       "spingo                                                       2162\n",
       "np-likeness-scorer                                           2162\n",
       "perl-hook-lexwrap                                            2162\n",
       "r-ggbiplot                                                   2161\n",
       "perl-text-levenshtein                                        2159\n",
       "r-phyext2                                                    2158\n",
       "perl-onto-perl                                               2158\n",
       "pantools                                                     2155\n",
       "piler                                                        2154\n",
       "perl-dbm-deep                                                2154\n",
       "r-qorts                                                      2154\n",
       "r-peer                                                       2153\n",
       "soapdenovo2-errorcorrection                                  2152\n",
       "spotyping                                                    2149\n",
       "pblaa                                                        2148\n",
       "perl-string-truncate                                         2147\n",
       "r-dpeak                                                      2145\n",
       "perl-moosex-types-path-tiny                                  2144\n",
       "swiftlink                                                    2143\n",
       "reago                                                        2140\n",
       "perl-string-rewriteprefix                                    2136\n",
       "valet                                                        2134\n",
       "pywdl                                                        2131\n",
       "reveal                                                       2130\n",
       "xtermcolor                                                   2127\n",
       "perl-perlio                                                  2123\n",
       "mwga-utils                                                   2121\n",
       "sff2fastq                                                    2120\n",
       "structureharvester                                           2114\n",
       "snns                                                         2113\n",
       "phylics                                                      2111\n",
       "perl-text-nsp                                                2111\n",
       "qfilt                                                        2109\n",
       "perl-list-compare                                            2105\n",
       "xunit-wrapper                                                2105\n",
       "r-mcpcounter                                                 2105\n",
       "ngmerge                                                      2104\n",
       "pacbio_falcon                                                2103\n",
       "phizz                                                        2103\n",
       "recalladapters                                               2102\n",
       "r-soap-nmr                                                   2100\n",
       "wheezy.template                                              2098\n",
       "spectra-cluster-cli                                          2096\n",
       "sumaclust                                                    2096\n",
       "quorum                                                       2094\n",
       "r-metama                                                     2093\n",
       "perl-sub-attribute                                           2089\n",
       "stride                                                       2084\n",
       "rseg                                                         2078\n",
       "soapdenovo2-prepare                                          2077\n",
       "networkxgmml                                                 2077\n",
       "mtbseq                                                       2076\n",
       "quake                                                        2075\n",
       "motifraptor                                                  2075\n",
       "r-leidenbase                                                 2075\n",
       "perl-moosex-nonmoose                                         2071\n",
       "perl-http-server-simple                                      2065\n",
       "macs2                                                        2064\n",
       "mglex                                                        2064\n",
       "treemaker                                                    2064\n",
       "r-epic                                                       2064\n",
       "perl-border-style                                            2063\n",
       "w4mclstrpeakpics                                             2062\n",
       "soapaligner                                                  2062\n",
       "perl-class-methodmaker                                       2061\n",
       "r-spieceasi                                                  2061\n",
       "vtools                                                       2061\n",
       "perl-getopt-long-descriptive                                 2057\n",
       "spectral_hk                                                  2056\n",
       "radsex                                                       2056\n",
       "mimeo                                                        2055\n",
       "perl-biox-seq                                                2055\n",
       "sbpipe                                                       2053\n",
       "zifa                                                         2052\n",
       "perl-pod-coverage                                            2052\n",
       "n50                                                          2051\n",
       "perl-tie-refhash-weak                                        2049\n",
       "perl-compress-bgzf                                           2049\n",
       "nmrpro                                                       2048\n",
       "r-phylomeasures                                              2043\n",
       "perl-padwalker                                               2043\n",
       "triform2                                                     2042\n",
       "perl-ppi                                                     2037\n",
       "pancake                                                      2037\n",
       "rdfextras                                                    2036\n",
       "perl-class-std                                               2033\n",
       "panphlan                                                     2033\n",
       "monovar                                                      2032\n",
       "ruby-dna-tools                                               2027\n",
       "r-perfmeas                                                   2022\n",
       "xtail                                                        2022\n",
       "molpopgen-analysis                                           2019\n",
       "tidehunter                                                   2018\n",
       "marge                                                        2013\n",
       "metavelvet-annois                                            2013\n",
       "r-ampvis                                                     2012\n",
       "ncfp                                                         2011\n",
       "metaseq-all                                                  2009\n",
       "pybamparser                                                  2007\n",
       "gmap                                                         2006\n",
       "pal_finder                                                   2006\n",
       "perl-tie-toobject                                            2005\n",
       "perl-moosex-app-role-log4perl                                2005\n",
       "r-loom                                                       2004\n",
       "sure                                                         2001\n",
       "primerprospector                                             2001\n",
       "rmap                                                         2000\n",
       "perl-module-list                                             2000\n",
       "music                                                        2000\n",
       "perl-bio-mlst-check                                          1998\n",
       "mindthegap                                                   1998\n",
       "rsat-core                                                    1998\n",
       "reaper                                                       1994\n",
       "augustus                                                     1994\n",
       "somaticseq                                                   1993\n",
       "barrnap                                                      1993\n",
       "perl-test-object                                             1990\n",
       "phylophlan                                                   1989\n",
       "ssu-align                                                    1989\n",
       "r-cimpl                                                      1987\n",
       "perl-aliased                                                 1987\n",
       "perl-moosex-clone                                            1986\n",
       "r-rapidr                                                     1985\n",
       "perl-data-visitor                                            1983\n",
       "nanopore_simulation                                          1982\n",
       "perl-perl4-corelibs                                          1981\n",
       "sshmm                                                        1981\n",
       "ndex-python                                                  1981\n",
       "perl-file-find                                               1980\n",
       "perl-params-coerce                                           1980\n",
       "perl-test-pod-coverage                                       1980\n",
       "ocrad                                                        1980\n",
       "sweepfinder2                                                 1978\n",
       "pout2mzid                                                    1978\n",
       "slncky                                                       1978\n",
       "perl-text-ansitable                                          1976\n",
       "nlstradamus                                                  1976\n",
       "metawatt                                                     1974\n",
       "python-mailund-newick                                        1971\n",
       "perl-file-remove                                             1970\n",
       "perl-file-compare                                            1969\n",
       "bioblend                                                     1965\n",
       "perl-test-file-contents                                      1962\n",
       "spectrassembler                                              1961\n",
       "pirs                                                         1960\n",
       "perl-estscan2                                                1958\n",
       "stellar                                                      1958\n",
       "perl-text-template                                           1956\n",
       "menetools                                                    1955\n",
       "perl-moosex-role-parameterized                               1955\n",
       "yass                                                         1954\n",
       "npyscreen                                                    1953\n",
       "perl-hash-util-fieldhash-compat                              1949\n",
       "perl-data-walk                                               1948\n",
       "perl-file-details                                            1947\n",
       "htseq                                                        1945\n",
       "perl-btlib                                                   1944\n",
       "perl-test-mockmodule                                         1942\n",
       "gseapy                                                       1942\n",
       "rambo-k                                                      1942\n",
       "perl-pod-parser                                              1941\n",
       "perl-extutils-pkgconfig                                      1941\n",
       "riboplot                                                     1937\n",
       "perl-io-socket-inet6                                         1936\n",
       "meneco                                                       1935\n",
       "pymix                                                        1934\n",
       "mustang                                                      1931\n",
       "perl-test-longstring                                         1931\n",
       "pysvmlight                                                   1928\n",
       "verse                                                        1925\n",
       "perl-filedirutil                                             1924\n",
       "taeper                                                       1923\n",
       "pkiss                                                        1921\n",
       "rpkmforgenes                                                 1921\n",
       "pathogen-profiler                                            1919\n",
       "r-whopgenome                                                 1918\n",
       "perl-safe                                                    1918\n",
       "pgsa                                                         1916\n",
       "translatorx                                                  1916\n",
       "perl-net-ftp                                                 1915\n",
       "tssar                                                        1914\n",
       "perl-statistics-lite                                         1912\n",
       "yaha                                                         1911\n",
       "lsd                                                          1911\n",
       "perl-test-subcalls                                           1906\n",
       "nanonet                                                      1906\n",
       "perl-class-std-fast                                          1905\n",
       "sak                                                          1904\n",
       "sts-smctc                                                    1903\n",
       "nmrml2isa                                                    1903\n",
       "zerone                                                       1898\n",
       "r-lncpipereporter                                            1897\n",
       "ribodiff                                                     1896\n",
       "snmf                                                         1894\n",
       "r-xcell                                                      1892\n",
       "vcf-annotator                                                1890\n",
       "perl-config-tiny                                             1889\n",
       "riboraptor                                                   1887\n",
       "rascaf                                                       1887\n",
       "tedna                                                        1885\n",
       "perl-bloom-faster                                            1884\n",
       "circexplorer2                                                1883\n",
       "pyimagingmspec                                               1882\n",
       "perl-io-uncompress-bunzip2                                   1880\n",
       "perl-tie-cacher                                              1878\n",
       "tinscan                                                      1876\n",
       "metavelvet-sl                                                1871\n",
       "perl-env                                                     1870\n",
       "remurna                                                      1869\n",
       "perl-moosex-role-withoverloading                             1869\n",
       "mgs-canopy                                                   1869\n",
       "svmlight                                                     1869\n",
       "pb-falconc                                                   1868\n",
       "ddocent                                                      1868\n",
       "perl-unicode-stringprep                                      1867\n",
       "pyscaf                                                       1867\n",
       "r-annotables                                                 1862\n",
       "perl-module-extract-use                                      1859\n",
       "metacherchant                                                1854\n",
       "velvet-sc                                                    1854\n",
       "wgfast                                                       1853\n",
       "perl-tie-hash-indexed                                        1853\n",
       "r-rrbgen                                                     1848\n",
       "perl-net-ftp-recursive                                       1847\n",
       "bioconductor-variantannotation                               1845\n",
       "mhcnuggets                                                   1843\n",
       "ogdf                                                         1843\n",
       "trinculo                                                     1841\n",
       "sift4g                                                       1840\n",
       "perl-pod-simple                                              1838\n",
       "pb-falcon-phase                                              1837\n",
       "mergevcf                                                     1835\n",
       "kma                                                          1834\n",
       "perl-math-combinatorics                                      1833\n",
       "perl-test-prereq                                             1831\n",
       "pipelign                                                     1826\n",
       "metaxa                                                       1826\n",
       "rop                                                          1826\n",
       "soapsplice                                                   1822\n",
       "spine                                                        1821\n",
       "perl-hpc-runner-command-plugin-logger-sqlite                 1820\n",
       "perl-super                                                   1819\n",
       "scrublet                                                     1819\n",
       "scg                                                          1818\n",
       "suma_package                                                 1817\n",
       "perl-par-packer                                              1816\n",
       "pymot                                                        1815\n",
       "gimmemotifs                                                  1812\n",
       "ngshmmalign                                                  1808\n",
       "ssake                                                        1807\n",
       "svim                                                         1805\n",
       "perl-getopt-argvfile                                         1804\n",
       "pydemult                                                     1804\n",
       "perl-file-copy-link                                          1804\n",
       "tmalign                                                      1804\n",
       "zagros                                                       1804\n",
       "metaphyler                                                   1802\n",
       "proot                                                        1802\n",
       "mendelscan                                                   1801\n",
       "vcflatten                                                    1801\n",
       "perl-string-random                                           1797\n",
       "soapcoverage                                                 1794\n",
       "perl-pod-simple-text                                         1794\n",
       "perl-json-create                                             1793\n",
       "ra                                                           1792\n",
       "platypus-conquistador                                        1790\n",
       "phast                                                        1790\n",
       "realphy                                                      1789\n",
       "perl-io-scalar                                               1789\n",
       "perl-tie-hash                                                1788\n",
       "bioconductor-graph                                           1787\n",
       "nanoplexer                                                   1787\n",
       "wgs-assembler                                                1785\n",
       "sim4db                                                       1785\n",
       "perl-vars                                                    1785\n",
       "perl-symbol                                                  1785\n",
       "perl-number-range                                            1784\n",
       "prophasm                                                     1782\n",
       "vdjer                                                        1781\n",
       "panoct                                                       1781\n",
       "goleft                                                       1781\n",
       "perl-indirect                                                1778\n",
       "metacluster                                                  1777\n",
       "r-intego                                                     1777\n",
       "repenrich                                                    1777\n",
       "perl-par-dist                                                1775\n",
       "metaprob                                                     1774\n",
       "micro_razers                                                 1773\n",
       "perl-par                                                     1772\n",
       "perl-file-touch                                              1770\n",
       "stereogene                                                   1769\n",
       "spectrum_utils                                               1768\n",
       "nanoblaster                                                  1767\n",
       "intarna                                                      1767\n",
       "perl-test-memory-cycle                                       1767\n",
       "perl-pod-checker                                             1765\n",
       "r-grbase                                                     1761\n",
       "taxator-tk                                                   1761\n",
       "perl-filter-simple                                           1761\n",
       "netcdf-metadata-info                                         1760\n",
       "r-zerone                                                     1758\n",
       "perl-log-any                                                 1757\n",
       "perl-bundle-bioperl                                          1755\n",
       "perl-text-tabs                                               1755\n",
       "perl-symbol-util                                             1755\n",
       "perl-devel-assert                                            1754\n",
       "mlrho                                                        1754\n",
       "perl-digest-sha                                              1754\n",
       "methylextract                                                1752\n",
       "perl-locale                                                  1752\n",
       "cd-hit                                                       1752\n",
       "qiimetomaaslin                                               1752\n",
       "straitrazor                                                  1751\n",
       "migrate-n                                                    1750\n",
       "perl-net-netrc                                               1750\n",
       "splitmem                                                     1750\n",
       "perl-statistics-frequency                                    1749\n",
       "perl-role-tiny-with                                          1748\n",
       "perl-pod-plaintext                                           1745\n",
       "perl-data-utilities                                          1744\n",
       "xhmm                                                         1743\n",
       "talloc                                                       1743\n",
       "pgcgap                                                       1742\n",
       "perl-tap-harness-env                                         1742\n",
       "satrap                                                       1742\n",
       "perl-blib                                                    1742\n",
       "r-rphast                                                     1742\n",
       "twopaco                                                      1741\n",
       "sfs_code                                                     1740\n",
       "perl-integer                                                 1739\n",
       "perl-exporter-tidy                                           1739\n",
       "ucsc-cell-browser                                            1739\n",
       "perl-cpan-meta-validator                                     1738\n",
       "rdp-readseq                                                  1737\n",
       "perl-cairo                                                   1736\n",
       "sqlitebrowser                                                1735\n",
       "perl-test-unit-lite                                          1734\n",
       "perl-file-pushd                                              1733\n",
       "telescope                                                    1732\n",
       "shapemapper                                                  1730\n",
       "lrzip                                                        1729\n",
       "perl-experimental                                            1728\n",
       "deeptoolsintervals                                           1728\n",
       "splicemap                                                    1727\n",
       "solexaqa                                                     1727\n",
       "nanosplit                                                    1724\n",
       "truvari                                                      1720\n",
       "perl-io-uncompress-rawinflate                                1718\n",
       "perl-unicode-utf8                                            1718\n",
       "teloclip                                                     1717\n",
       "perl-perlio-encoding                                         1716\n",
       "perl-pod-coverage-trustpod                                   1715\n",
       "snpomatic                                                    1715\n",
       "metaquant                                                    1714\n",
       "pbwt                                                         1714\n",
       "satsuma2                                                     1712\n",
       "spectacle                                                    1711\n",
       "tagger                                                       1710\n",
       "perl-moosex-singleton                                        1710\n",
       "pauda                                                        1707\n",
       "perl-math-complex                                            1705\n",
       "perl-moosex-types-stringlike                                 1705\n",
       "r-george                                                     1705\n",
       "perl-mime-quotedprint                                        1704\n",
       "rfmix                                                        1703\n",
       "gatk                                                         1703\n",
       "xcftools                                                     1702\n",
       "transcomb                                                    1701\n",
       "perl-xml-entities                                            1701\n",
       "tinysink                                                     1701\n",
       "sprai                                                        1700\n",
       "opencontactcli                                               1699\n",
       "perl-estscan1                                                1699\n",
       "mmvc                                                         1699\n",
       "scanindel                                                    1696\n",
       "scanpy-scripts                                               1695\n",
       "nucamino                                                     1694\n",
       "strudel                                                      1693\n",
       "virema                                                       1688\n",
       "gemini                                                       1688\n",
       "perl-autodie                                                 1685\n",
       "perl-io-compress-deflate                                     1685\n",
       "perl-pbkdf2-tiny                                             1684\n",
       "perl-module-util                                             1684\n",
       "ngseqbasic                                                   1684\n",
       "protk                                                        1683\n",
       "tqdist                                                       1683\n",
       "perl-crypt-openssl-random                                    1683\n",
       "perl-test-exec                                               1681\n",
       "megagta                                                      1680\n",
       "perl-bio-bigfile                                             1680\n",
       "ipython-cluster-helper                                       1680\n",
       "real                                                         1677\n",
       "perl-heap                                                    1677\n",
       "saffrontree                                                  1675\n",
       "odgi                                                         1673\n",
       "perl-crypt-openssl-rsa                                       1670\n",
       "pyopt                                                        1663\n",
       "simwalk2                                                     1660\n",
       "qcumber                                                      1656\n",
       "seqmap                                                       1655\n",
       "slimm                                                        1655\n",
       "perl-import-into                                             1650\n",
       "semeta                                                       1649\n",
       "roary2fripan.py                                              1649\n",
       "red                                                          1648\n",
       "seer                                                         1648\n",
       "spanki                                                       1643\n",
       "phenix                                                       1642\n",
       "genepop                                                      1635\n",
       "ms                                                           1634\n",
       "abyss                                                        1633\n",
       "perl-devel-cycle                                             1631\n",
       "riborex                                                      1629\n",
       "rnaclust                                                     1627\n",
       "r-cate                                                       1627\n",
       "lsc                                                          1627\n",
       "r-nbpseq                                                     1627\n",
       "sloika                                                       1624\n",
       "perl-bio-automatedannotation                                 1623\n",
       "bioconductor-bsgenome                                        1622\n",
       "vargeno                                                      1621\n",
       "r-orqa                                                       1621\n",
       "r-leapp                                                      1619\n",
       "revtrans                                                     1619\n",
       "abeona                                                       1618\n",
       "riboseq-rust                                                 1618\n",
       "bioconductor-preprocesscore                                  1617\n",
       "perl-math-utils                                              1615\n",
       "r-lme4qtl                                                    1615\n",
       "perl-test-fork                                               1615\n",
       "pash                                                         1612\n",
       "perl-slurp                                                   1612\n",
       "r-phewas                                                     1609\n",
       "transgenescan                                                1608\n",
       "smashbenchmarking                                            1604\n",
       "nucleosome_prediction                                        1600\n",
       "scaffold_builder                                             1599\n",
       "odose                                                        1598\n",
       "r-grain                                                      1598\n",
       "readfq                                                       1598\n",
       "biobambam                                                    1597\n",
       "flye                                                         1596\n",
       "perl-mac-propertylist                                        1596\n",
       "roprofile                                                    1592\n",
       "perl-www-mechanize                                           1591\n",
       "perl-text-template-simple                                    1589\n",
       "perl-ipc-run3                                                1582\n",
       "r-genometricorr                                              1581\n",
       "pretextmap                                                   1580\n",
       "strike                                                       1576\n",
       "perl-findbin-real                                            1574\n",
       "perl-data-munge                                              1569\n",
       "vnl                                                          1563\n",
       "epic                                                         1561\n",
       "perl-return-multilevel                                       1559\n",
       "metavelvet-sl-pipeline                                       1558\n",
       "unitas                                                       1557\n",
       "r-pophelper                                                  1557\n",
       "qgrs-cpp                                                     1554\n",
       "ra-integrate                                                 1553\n",
       "perl-test-without-module                                     1552\n",
       "polymutt                                                     1549\n",
       "perl-file-copy-recursive-reduced                             1544\n",
       "perl-snap                                                    1544\n",
       "perl-class-singleton                                         1544\n",
       "rtk                                                          1542\n",
       "msalign2                                                     1541\n",
       "ribotricer                                                   1535\n",
       "r-smartsva                                                   1534\n",
       "r-sgtr                                                       1534\n",
       "srnapipe                                                     1533\n",
       "r-minionqc                                                   1530\n",
       "tpmcalculator                                                1526\n",
       "ltr_retriever                                                1522\n",
       "genenotebook                                                 1519\n",
       "mageck                                                       1513\n",
       "nf-core                                                      1511\n",
       "rasusa                                                       1505\n",
       "aragorn                                                      1500\n",
       "bioconductor-tximport                                        1498\n",
       "matlock                                                      1491\n",
       "sketchy                                                      1490\n",
       "ntjoin                                                       1488\n",
       "pychopper                                                    1488\n",
       "mantis                                                       1486\n",
       "perl-array-set                                               1486\n",
       "busco                                                        1475\n",
       "nemo                                                         1475\n",
       "piret                                                        1473\n",
       "bioconductor-rbgl                                            1470\n",
       "sicer2                                                       1467\n",
       "perl-probe-perl                                              1465\n",
       "last                                                         1461\n",
       "circos                                                       1460\n",
       "perl-test-script                                             1460\n",
       "ephemeris                                                    1457\n",
       "pbipa                                                        1453\n",
       "graphclust-wrappers                                          1447\n",
       "bioconductor-rsubread                                        1437\n",
       "multiz                                                       1432\n",
       "malder                                                       1431\n",
       "r-acidbase                                                   1427\n",
       "atropos                                                      1420\n",
       "r-umi4c                                                      1417\n",
       "sv2                                                          1406\n",
       "minys                                                        1402\n",
       "oligoarrayaux                                                1397\n",
       "perl-mac-systemdirectory                                     1393\n",
       "r-mixkernel                                                  1391\n",
       "bioconductor-dnacopy                                         1390\n",
       "gromacs                                                      1390\n",
       "r-shaman                                                     1388\n",
       "r-riborex                                                    1376\n",
       "ariba                                                        1376\n",
       "ucsc-hggoldgapgl                                             1373\n",
       "hicmatrix                                                    1371\n",
       "emboss                                                       1368\n",
       "bioconductor-affyio                                          1368\n",
       "makehub                                                      1365\n",
       "shannon_cpp                                                  1361\n",
       "ucsc-chainbridge                                             1359\n",
       "bioconductor-affy                                            1354\n",
       "ucsc-hgvstovcf                                               1349\n",
       "sabre                                                        1333\n",
       "ma                                                           1329\n",
       "anvio                                                        1327\n",
       "perl-moosex-object-pluggable                                 1325\n",
       "probcons                                                     1322\n",
       "galaxy-ml                                                    1321\n",
       "metaquantome                                                 1320\n",
       "perl-biosails                                                1319\n",
       "micropita                                                    1319\n",
       "vmatch                                                       1319\n",
       "pomoxis                                                      1317\n",
       "shark                                                        1313\n",
       "gxformat2                                                    1311\n",
       "bismark                                                      1307\n",
       "kmer-jellyfish                                               1306\n",
       "digestiflow-cli                                              1305\n",
       "zorro                                                        1305\n",
       "gridss                                                       1304\n",
       "bioconductor-protgenerics                                    1300\n",
       "ncbi-util-legacy                                             1297\n",
       "metaeuk                                                      1292\n",
       "samsum                                                       1284\n",
       "mrsfast                                                      1276\n",
       "mlpy                                                         1275\n",
       "bioconductor-multtest                                        1275\n",
       "mitos                                                        1272\n",
       "mapping-iterative-assembler                                  1270\n",
       "bioconductor-go.db                                           1270\n",
       "rxdock                                                       1264\n",
       "r-popgenreport                                               1262\n",
       "r-metacoder                                                  1261\n",
       "r-virfinder                                                  1261\n",
       "pbmarkdup                                                    1259\n",
       "canu                                                         1254\n",
       "bioconductor-sva                                             1253\n",
       "bioconductor-xcms                                            1253\n",
       "bio_hansel                                                   1252\n",
       "cmappy                                                       1246\n",
       "alfred                                                       1244\n",
       "psascan                                                      1242\n",
       "scprep                                                       1236\n",
       "seqhax                                                       1233\n",
       "gsalign                                                      1231\n",
       "perl-devel-checkos                                           1230\n",
       "groot                                                        1230\n",
       "perl-number-witherror                                        1227\n",
       "bioconductor-biocfilecache                                   1225\n",
       "perl-prefork                                                 1224\n",
       "perl-statistics-caseresampling                               1222\n",
       "ltr_finder                                                   1222\n",
       "perl-escape-houdini                                          1221\n",
       "mtsv-tools                                                   1220\n",
       "proda                                                        1219\n",
       "samstats                                                     1212\n",
       "krbalancing                                                  1208\n",
       "r-liger                                                      1205\n",
       "bioconductor-biovizbase                                      1204\n",
       "svaba                                                        1202\n",
       "bioconductor-qvalue                                          1201\n",
       "perl-archive-tar-wrapper                                     1194\n",
       "perl-test-lectrotest                                         1193\n",
       "bioconductor-impute                                          1192\n",
       "bioconductor-org.hs.eg.db                                    1190\n",
       "bioconductor-vsn                                             1188\n",
       "perl-dumbbench                                               1187\n",
       "abricate                                                     1187\n",
       "perl-devel-cover                                             1186\n",
       "wise2                                                        1185\n",
       "perl-text-levenshteinxs                                      1182\n",
       "r-sew                                                        1181\n",
       "plasmidid                                                    1179\n",
       "r-acidplots                                                  1178\n",
       "perl-test-utf8                                               1171\n",
       "bioconductor-hdf5array                                       1169\n",
       "ptrimmer                                                     1168\n",
       "locarna                                                      1166\n",
       "jbrowse                                                      1166\n",
       "phylommand                                                   1166\n",
       "perl-bignum                                                  1164\n",
       "perl-test-file                                               1164\n",
       "bioconductor-rgraphviz                                       1163\n",
       "mhcflurry                                                    1161\n",
       "bioconductor-beachmat                                        1158\n",
       "ripser                                                       1153\n",
       "bioconductor-gosemsim                                        1148\n",
       "perl-parallel-iterator                                       1148\n",
       "bioconductor-ensembldb                                       1146\n",
       "duphold                                                      1144\n",
       "taxonomy                                                     1143\n",
       "mace                                                         1140\n",
       "perl-math-bigrat                                             1137\n",
       "dx-cwl                                                       1135\n",
       "kaiju                                                        1133\n",
       "perl-b-debug                                                 1130\n",
       "snostrip                                                     1127\n",
       "behst                                                        1116\n",
       "dunovo                                                       1113\n",
       "fgbio                                                        1106\n",
       "oncotator                                                    1103\n",
       "smashpp                                                      1098\n",
       "bioconductor-csaw                                            1092\n",
       "r-deseqanalysis                                              1088\n",
       "epic2                                                        1087\n",
       "bioconductor-diffbind                                        1086\n",
       "bioconductor-deseq                                           1082\n",
       "showit                                                       1081\n",
       "glimmerhmm                                                   1076\n",
       "bioconductor-titancna                                        1076\n",
       "cgatcore                                                     1072\n",
       "khmer                                                        1068\n",
       "scelvis                                                      1067\n",
       "hic2cool                                                     1064\n",
       "delly                                                        1063\n",
       "spring                                                       1063\n",
       "fast5                                                        1060\n",
       "ntedit                                                       1058\n",
       "bioconductor-scater                                          1055\n",
       "phylocsf                                                     1053\n",
       "scallop-lr                                                   1050\n",
       "sdm                                                          1037\n",
       "gffread                                                      1034\n",
       "bioconductor-dose                                            1033\n",
       "bioconductor-org.mm.eg.db                                    1033\n",
       "bioconda-utils                                               1024\n",
       "kraken                                                       1017\n",
       "smafa                                                        1015\n",
       "krona                                                        1011\n",
       "grep                                                          994\n",
       "sesimcmc                                                      993\n",
       "bioconductor-mzr                                              984\n",
       "bioconductor-pcamethods                                       981\n",
       "bioconductor-massspecwavelet                                  978\n",
       "pretextsnapshot                                               972\n",
       "r-motifbinner                                                 970\n",
       "python-eel                                                    966\n",
       "prefersim                                                     964\n",
       "pairsnp                                                       964\n",
       "r-acidgenerics                                                964\n",
       "graphaligner                                                  963\n",
       "checkm-genome                                                 962\n",
       "bcbreport                                                     961\n",
       "libsequence                                                   958\n",
       "bioconductor-scran                                            954\n",
       "inheritance                                                   948\n",
       "revbayes                                                      944\n",
       "pblat                                                         943\n",
       "matam                                                         942\n",
       "strling                                                       939\n",
       "lexmapr                                                       938\n",
       "dxpy                                                          937\n",
       "jcvi                                                          936\n",
       "popscle                                                       936\n",
       "gtdbtk                                                        934\n",
       "dr-disco                                                      922\n",
       "tinyalign                                                     920\n",
       "bioconductor-camera                                           920\n",
       "bioconductor-biocstyle                                        919\n",
       "python-htsget                                                 917\n",
       "fit_nbinom                                                    914\n",
       "crossmap                                                      911\n",
       "deepvariant                                                   908\n",
       "bioconductor-clusterprofiler                                  905\n",
       "bioconductor-annotationhub                                    904\n",
       "bioconductor-fgsea                                            899\n",
       "homer                                                         896\n",
       "prince                                                        894\n",
       "igdiscover                                                    886\n",
       "transrate                                                     885\n",
       "svanalyzer                                                    885\n",
       "xs-sim                                                        884\n",
       "varfish-annotator-cli                                         883\n",
       "nxtrim                                                        883\n",
       "bioconductor-flowcore                                         879\n",
       "bioconductor-gsva                                             879\n",
       "gffcompare                                                    878\n",
       "primerclip                                                    876\n",
       "regional                                                      874\n",
       "repdenovo                                                     873\n",
       "bioconductor-annotationfilter                                 873\n",
       "phylodm                                                       872\n",
       "bioconductor-monocle                                          872\n",
       "sensv                                                         871\n",
       "bioconductor-singlecellexperiment                             871\n",
       "bioconductor-methylkit                                        866\n",
       "gappa                                                         865\n",
       "gsort                                                         865\n",
       "genomepy                                                      863\n",
       "bioconductor-ebimage                                          862\n",
       "bioconductor-cn.mops                                          862\n",
       "bioconductor-genomation                                       860\n",
       "fastq-screen                                                  859\n",
       "blast-legacy                                                  857\n",
       "involucro                                                     855\n",
       "biofluff                                                      849\n",
       "bioconductor-illuminaio                                       848\n",
       "comet-ms                                                      847\n",
       "bioconductor-gseabase                                         845\n",
       "ddrage                                                        845\n",
       "asciigenome                                                   838\n",
       "tb_variant_filter                                             837\n",
       "kmc                                                           835\n",
       "bcbio-nextgen-vm                                              835\n",
       "metabinkit                                                    826\n",
       "r-vision                                                      826\n",
       "pal2nal                                                       824\n",
       "circlator                                                     822\n",
       "mutmap                                                        819\n",
       "bioconductor-ropls                                            814\n",
       "gubbins                                                       811\n",
       "bioconductor-bumphunter                                       811\n",
       "humann2                                                       807\n",
       "r-velocyto.r                                                  805\n",
       "scnic                                                         802\n",
       "bioconductor-ggbio                                            801\n",
       "translig                                                      801\n",
       "bioservices                                                   796\n",
       "tepeaks                                                       795\n",
       "bioconductor-interactivedisplaybase                           795\n",
       "bioconductor-msnbase                                          794\n",
       "advntr                                                        794\n",
       "bioconductor-ebseq                                            792\n",
       "lumpy-sv                                                      790\n",
       "pretextgraph                                                  787\n",
       "bioconductor-dirichletmultinomial                             786\n",
       "concoct                                                       785\n",
       "spliceai                                                      785\n",
       "epa-ng                                                        775\n",
       "bioconductor-ncdfflow                                         775\n",
       "bioconductor-keggrest                                         771\n",
       "bioconductor-tfbstools                                        769\n",
       "mhcnames                                                      769\n",
       "snp-mutator                                                   768\n",
       "bioconductor-regioner                                         764\n",
       "exonerate                                                     764\n",
       "bioconductor-cardinal                                         763\n",
       "bioconductor-dexseq                                           763\n",
       "bioconductor-delayedmatrixstats                               762\n",
       "crispritz                                                     762\n",
       "bioconductor-complexheatmap                                   759\n",
       "tspex                                                         757\n",
       "hifive                                                        753\n",
       "kraken2                                                       752\n",
       "bioconductor-rots                                             751\n",
       "scssim                                                        749\n",
       "bioconductor-mosaics                                          744\n",
       "lima                                                          743\n",
       "perl-pdf-table                                                743\n",
       "bioconductor-geoquery                                         739\n",
       "bioconductor-gviz                                             738\n",
       "scvelo                                                        732\n",
       "igv                                                           730\n",
       "pypairs                                                       725\n",
       "bioconductor-lpsymphony                                       725\n",
       "cnv_facets                                                    725\n",
       "beast                                                         722\n",
       "qualimap                                                      721\n",
       "ig-flowtools                                                  720\n",
       "ncbi-acc-download                                             718\n",
       "bioconductor-org.dm.eg.db                                     718\n",
       "qtlseq                                                        715\n",
       "breseq                                                        713\n",
       "goatools                                                      711\n",
       "probabilistic2020                                             710\n",
       "bd2k-python-lib                                               709\n",
       "blasr                                                         707\n",
       "wg-blimp                                                      706\n",
       "flashlfq                                                      705\n",
       "centrifuge                                                    704\n",
       "bioconductor-phyloseq                                         699\n",
       "seqfu                                                         698\n",
       "bioconductor-chipseq                                          697\n",
       "flowcraft                                                     695\n",
       "tesorter                                                      695\n",
       "ncbimeta                                                      688\n",
       "jalview                                                       687\n",
       "bioconductor-flowsom                                          687\n",
       "bioconductor-annotationforge                                  686\n",
       "bioconductor-apeglm                                           685\n",
       "bioconductor-fastseg                                          685\n",
       "quickmerge                                                    685\n",
       "fastq_utils                                                   684\n",
       "anvio-minimal                                                 683\n",
       "bioconductor-oligo                                            682\n",
       "vpolo                                                         682\n",
       "bioconductor-prada                                            681\n",
       "bioconductor-cner                                             681\n",
       "bioconductor-hilbertvis                                       678\n",
       "nthits                                                        678\n",
       "r-pipette                                                     678\n",
       "bioconductor-organismdbi                                      678\n",
       "bioconductor-bacon                                            676\n",
       "bioconductor-ringo                                            676\n",
       "bioconductor-flowworkspace                                    676\n",
       "kat                                                           675\n",
       "bioconductor-bayespeak                                        675\n",
       "bioconductor-compass                                          674\n",
       "bioconductor-sspa                                             673\n",
       "bioconductor-repitools                                        673\n",
       "bioconductor-goseq                                            672\n",
       "bioconductor-exomecopy                                        671\n",
       "igvtools                                                      670\n",
       "mutscan                                                       669\n",
       "crispresso2                                                   669\n",
       "mageck-vispr                                                  667\n",
       "svist4get                                                     666\n",
       "genometools-genometools                                       666\n",
       "bioconductor-genelendatabase                                  666\n",
       "bcbio-prioritize                                              665\n",
       "cosi2                                                         664\n",
       "elprep                                                        662\n",
       "bioconductor-gaga                                             662\n",
       "bioconductor-masigpro                                         662\n",
       "bioconductor-mspurity                                         661\n",
       "bioconductor-motiv                                            660\n",
       "bioconductor-matter                                           660\n",
       "bioconductor-bader                                            660\n",
       "bioconductor-flowfp                                           658\n",
       "spacepharer                                                   658\n",
       "bioconductor-rcas                                             658\n",
       "xpressplot                                                    657\n",
       "bioconductor-rgadem                                           657\n",
       "perl-text-csv_xs                                              656\n",
       "repaq                                                         656\n",
       "bioconductor-biomformat                                       655\n",
       "isoseq3                                                       655\n",
       "grabix                                                        653\n",
       "bioconductor-deepsnv                                          652\n",
       "bioconductor-chipcomp                                         652\n",
       "bioconductor-signer                                           651\n",
       "jannovar-cli                                                  649\n",
       "bioconductor-tweedeseq                                        647\n",
       "bioconductor-cellnoptr                                        646\n",
       "genomedata                                                    645\n",
       "bioconductor-topgo                                            644\n",
       "bioconductor-biocneighbors                                    643\n",
       "vibrant                                                       642\n",
       "stark                                                         642\n",
       "bioconductor-bsseq                                            641\n",
       "bioconductor-kegggraph                                        634\n",
       "irida-linker                                                  633\n",
       "bioconductor-chipseeker                                       632\n",
       "bioconductor-degreport                                        632\n",
       "thesias                                                       631\n",
       "krakenuniq                                                    628\n",
       "bioconductor-affxparser                                       626\n",
       "dinopy                                                        625\n",
       "beagle-lib                                                    625\n",
       "r-immunedeconv                                                625\n",
       "itsxpress                                                     624\n",
       "sctools                                                       624\n",
       "tadbit                                                        623\n",
       "bioconductor-cummerbund                                       621\n",
       "longshot                                                      616\n",
       "bioconductor-systempiper                                      615\n",
       "perl-fastx-abi                                                615\n",
       "fastx_toolkit                                                 611\n",
       "panaroo                                                       610\n",
       "refgenconf                                                    609\n",
       "bioconductor-purecn                                           609\n",
       "sierrapy                                                      607\n",
       "sccaf                                                         604\n",
       "terminus                                                      604\n",
       "bioconductor-ggtree                                           603\n",
       "bioconductor-biocviews                                        601\n",
       "bioconductor-gdsfmt                                           596\n",
       "itero                                                         595\n",
       "bioconductor-category                                         594\n",
       "bioconductor-hsmmsinglecell                                   594\n",
       "refgenie                                                      592\n",
       "bcbio-variation-recall                                        592\n",
       "bioconductor-flowpeaks                                        591\n",
       "chewbbaca                                                     590\n",
       "biokit                                                        589\n",
       "libbigwig                                                     589\n",
       "bioconductor-seqlogo                                          588\n",
       "bioconductor-snpstats                                         588\n",
       "bioconductor-gcrma                                            588\n",
       "bioconductor-bubbletree                                       587\n",
       "bioconductor-beadarray                                        585\n",
       "bioconductor-hmmcopy                                          585\n",
       "bioconductor-aseb                                             584\n",
       "bioconductor-snprelate                                        584\n",
       "bioconductor-rbowtie                                          584\n",
       "blasr_libcpp                                                  584\n",
       "bioconductor-seqtools                                         583\n",
       "bumbershoot                                                   581\n",
       "perl-bio-trace-abif                                           581\n",
       "bioconductor-fithic                                           580\n",
       "bioconductor-roc                                              577\n",
       "bioconductor-acgh                                             576\n",
       "bioconductor-seqpattern                                       576\n",
       "bioconductor-metahdep                                         575\n",
       "bioconductor-gostats                                          574\n",
       "bioconductor-cntools                                          574\n",
       "hyphy                                                         574\n",
       "bioconductor-pics                                             574\n",
       "bioconductor-dexus                                            574\n",
       "bioconductor-acme                                             574\n",
       "bioconductor-sincell                                          573\n",
       "clever-toolkit                                                573\n",
       "bioconductor-chemminer                                        573\n",
       "bioconductor-amountain                                        573\n",
       "bioconductor-beaddatapackr                                    572\n",
       "bioconductor-vega                                             572\n",
       "bioconductor-plier                                            572\n",
       "bioconductor-bicare                                           572\n",
       "bioconductor-twilight                                         571\n",
       "bioconductor-adsplit                                          571\n",
       "bioconductor-plw                                              571\n",
       "bioconductor-degseq                                           571\n",
       "bioconductor-blima                                            571\n",
       "flaimapper                                                    570\n",
       "bioconductor-simlr                                            570\n",
       "bioconductor-switchbox                                        570\n",
       "bioconductor-xde                                              570\n",
       "bioconductor-methylpipe                                       570\n",
       "bioconductor-aroma.light                                      570\n",
       "bioconductor-metaarray                                        569\n",
       "bioconductor-simpleaffy                                       569\n",
       "bioconductor-messina                                          569\n",
       "bioconductor-streamer                                         568\n",
       "referenceseeker                                               568\n",
       "bioconductor-bac                                              567\n",
       "cufflinks                                                     567\n",
       "bioconductor-bhc                                              567\n",
       "bioconductor-siggenes                                         567\n",
       "bioconductor-qrqc                                             567\n",
       "galaxy_sequence_utils                                         567\n",
       "bioconductor-chromstar                                        567\n",
       "flash                                                         566\n",
       "bioconductor-multiscan                                        566\n",
       "bioconductor-rtreemix                                         566\n",
       "circexplorer                                                  566\n",
       "bioconductor-netpathminer                                     565\n",
       "bioconductor-bcrank                                           565\n",
       "bioconductor-triplex                                          565\n",
       "bioconductor-rbioinf                                          563\n",
       "refinem                                                       563\n",
       "bioconductor-dnashaper                                        563\n",
       "illumina-interop                                              562\n",
       "bioconductor-sigpathway                                       562\n",
       "bioconductor-minfi                                            562\n",
       "bioconductor-interactionset                                   561\n",
       "bioconductor-ping                                             561\n",
       "bioconductor-bitseq                                           560\n",
       "bioconductor-starr                                            560\n",
       "bioconductor-bgmix                                            560\n",
       "clark                                                         560\n",
       "bioconductor-flowai                                           560\n",
       "bioconductor-ternarynet                                       559\n",
       "bioconductor-cancerinsilico                                   558\n",
       "bioconductor-transview                                        558\n",
       "bioconductor-profia                                           558\n",
       "perl-string-diff                                              558\n",
       "bioconductor-maanova                                          557\n",
       "bioconductor-sagx                                             557\n",
       "bioconductor-birta                                            557\n",
       "bioconductor-isolde                                           556\n",
       "bioconductor-chromdraw                                        556\n",
       "bioconductor-scpipe                                           556\n",
       "bioconductor-mgsa                                             556\n",
       "bioconductor-rama                                             556\n",
       "bioconductor-cancermutationanalysis                           556\n",
       "bioconductor-edda                                             556\n",
       "bioconductor-oppar                                            555\n",
       "bioconductor-smap                                             555\n",
       "bioconductor-microrna                                         555\n",
       "bioconductor-methylmnm                                        555\n",
       "bioconductor-podkat                                           555\n",
       "bioconductor-tigre                                            554\n",
       "bioconductor-makecdfenv                                       554\n",
       "bioconductor-ebarrays                                         554\n",
       "bioconductor-rfpred                                           554\n",
       "bioconductor-casper                                           554\n",
       "bioconductor-rnaseqmap                                        554\n",
       "bioconductor-tspair                                           554\n",
       "bioconductor-cnanorm                                          554\n",
       "bioconductor-metaseq                                          554\n",
       "igblast                                                       553\n",
       "bioconductor-genomictuples                                    553\n",
       "bioconductor-biosigner                                        552\n",
       "popgen-entropy                                                552\n",
       "gvcf-regions                                                  546\n",
       "xpclr                                                         543\n",
       "mofapy                                                        538\n",
       "bcbiogff                                                      538\n",
       "libgtextutils                                                 537\n",
       "expansionhunter                                               535\n",
       "vcontact2                                                     531\n",
       "covtobed                                                      529\n",
       "lrez                                                          527\n",
       "bioconductor-qdnaseq                                          523\n",
       "fastani                                                       523\n",
       "lofreq                                                        521\n",
       "rawtools                                                      520\n",
       "bioconductor-treeio                                           520\n",
       "pathogist                                                     519\n",
       "bioconductor-dropletutils                                     517\n",
       "angsd                                                         516\n",
       "bioconductor-pathview                                         516\n",
       "bioconductor-cghcall                                          511\n",
       "bioconductor-mzid                                             510\n",
       "bioconductor-decipher                                         510\n",
       "cgat-apps                                                     509\n",
       "idr                                                           509\n",
       "bioconductor-chromvar                                         506\n",
       "bioconductor-all                                              504\n",
       "plncpro                                                       504\n",
       "edlib                                                         503\n",
       "bioconductor-bamsignals                                       503\n",
       "evidencemodeler                                               502\n",
       "clustalo                                                      502\n",
       "r-tailfindr                                                   501\n",
       "sdrf-pipelines                                                500\n",
       "bioawk                                                        500\n",
       "bioconductor-affyplm                                          499\n",
       "dammit                                                        498\n",
       "rnaredprint                                                   498\n",
       "bioconductor-hpar                                             497\n",
       "biothings_client                                              497\n",
       "bioconductor-rprotobuflib                                     496\n",
       "metamaps                                                      495\n",
       "r-sceasy                                                      494\n",
       "filtlong                                                      494\n",
       "svtk                                                          493\n",
       "bioconductor-copynumber                                       491\n",
       "vkmz                                                          490\n",
       "bioconductor-gagedata                                         489\n",
       "ganon                                                         489\n",
       "pirate                                                        489\n",
       "cogent                                                        489\n",
       "bioconductor-birewire                                         488\n",
       "pftools                                                       488\n",
       "bioconductor-consensusclusterplus                             488\n",
       "bioconductor-beclear                                          488\n",
       "seacr                                                         487\n",
       "psass                                                         486\n",
       "cgat-scripts                                                  485\n",
       "bioconductor-survcomp                                         485\n",
       "bioconductor-dss                                              485\n",
       "bioconductor-dupradar                                         484\n",
       "fpa                                                           484\n",
       "askocli                                                       484\n",
       "albatradis                                                    484\n",
       "eggnog-mapper                                                 484\n",
       "bioconductor-lfa                                              484\n",
       "bioconductor-minet                                            483\n",
       "bioconductor-gcatest                                          483\n",
       "bioconductor-kebabs                                           483\n",
       "bioconductor-stan                                             482\n",
       "bioconductor-quaternaryprod                                   482\n",
       "bioconductor-sradb                                            481\n",
       "bioconductor-ichip                                            481\n",
       "bioconductor-isoformswitchanalyzer                            481\n",
       "bioconductor-scde                                             480\n",
       "bioconductor-junctionseq                                      480\n",
       "bioconductor-harman                                           480\n",
       "bioconductor-hapfabia                                         480\n",
       "bioconductor-sc3                                              480\n",
       "bioconductor-motifcounter                                     480\n",
       "bioconductor-risa                                             479\n",
       "cdna_cupcake                                                  479\n",
       "alignstats                                                    479\n",
       "bioconductor-abaenrichment                                    478\n",
       "bioconductor-vegamc                                           478\n",
       "hulk                                                          478\n",
       "bioconductor-pqsfinder                                        478\n",
       "desalt                                                        478\n",
       "schicexplorer                                                 477\n",
       "chemfp                                                        477\n",
       "extract-sv-reads                                              476\n",
       "bioconductor-isomirs                                          476\n",
       "bioconductor-findmyfriends                                    475\n",
       "bioconductor-chimera                                          475\n",
       "bioconductor-flowcl                                           475\n",
       "alignlib-lite                                                 475\n",
       "bioconductor-cohcap                                           474\n",
       "crispresso                                                    474\n",
       "bioconductor-globalancova                                     474\n",
       "bioconductor-qpgraph                                          474\n",
       "bioconductor-linc                                             474\n",
       "daisysuite                                                    474\n",
       "bioconductor-oncosimulr                                       473\n",
       "bioconductor-flowmatch                                        473\n",
       "bioconductor-motifrg                                          473\n",
       "bioconductor-cn.farms                                         472\n",
       "bioconductor-funchip                                          472\n",
       "garnet                                                        472\n",
       "bioconductor-maigespack                                       471\n",
       "bioconductor-contibait                                        471\n",
       "bioconductor-crlmm                                            471\n",
       "bioconductor-qsea                                             471\n",
       "bioconductor-proloc                                           471\n",
       "enasearch                                                     471\n",
       "bioconductor-bridge                                           471\n",
       "genie                                                         470\n",
       "irfinder                                                      470\n",
       "cortexpy                                                      470\n",
       "bioconductor-cosmiq                                           470\n",
       "bioconductor-fabia                                            470\n",
       "bioconductor-puma                                             469\n",
       "bioconductor-cnorode                                          469\n",
       "seqstats                                                      469\n",
       "bioconductor-graphite                                         469\n",
       "bioconductor-chopsticks                                       468\n",
       "bioconductor-bus                                              468\n",
       "hlaprofiler                                                   468\n",
       "bioconductor-iseq                                             468\n",
       "pangolin                                                      468\n",
       "bioconductor-harshlight                                       467\n",
       "bioconductor-bioqc                                            467\n",
       "bioconductor-ippd                                             467\n",
       "bioconductor-quasr                                            466\n",
       "bioconductor-m3d                                              466\n",
       "bioconductor-genogam                                          466\n",
       "bioconductor-girafe                                           466\n",
       "bioconductor-pdinfobuilder                                    466\n",
       "bioconductor-flagme                                           465\n",
       "bioconductor-cnorfuzzy                                        465\n",
       "kart                                                          465\n",
       "bcbio-rnaseq                                                  465\n",
       "bioconductor-hem                                              465\n",
       "bioconductor-bufferedmatrix                                   465\n",
       "bioconductor-sim                                              464\n",
       "bioconductor-bufferedmatrixmethods                            464\n",
       "bitmapperbs                                                   463\n",
       "bioconductor-dnabarcodes                                      463\n",
       "cmv                                                           463\n",
       "bioconductor-diffustats                                       463\n",
       "bioconductor-inversion                                        463\n",
       "bioconductor-cispath                                          463\n",
       "bioconductor-hibag                                            463\n",
       "bioconductor-netreg                                           462\n",
       "bioconductor-networkbma                                       462\n",
       "bioconductor-cnvtools                                         462\n",
       "bioconductor-hopach                                           462\n",
       "bioconductor-generxcluster                                    462\n",
       "bioconductor-mulcom                                           462\n",
       "bioconductor-vanillaice                                       462\n",
       "bioconductor-gsreg                                            462\n",
       "bioconductor-medme                                            461\n",
       "bioconductor-genoset                                          461\n",
       "bioconductor-trigger                                          461\n",
       "platon                                                        461\n",
       "bioconductor-biocsingular                                     461\n",
       "bioconductor-turbonorm                                        461\n",
       "bioconductor-variantfiltering                                 461\n",
       "wade                                                          460\n",
       "bioconductor-logitt                                           460\n",
       "bioconductor-org.dr.eg.db                                     459\n",
       "bioconductor-marray                                           453\n",
       "funannotate                                                   452\n",
       "antismash                                                     452\n",
       "fastq-scan                                                    451\n",
       "r-tmae                                                        451\n",
       "genomelake                                                    448\n",
       "recentrifuge                                                  446\n",
       "cwltest                                                       446\n",
       "bioconductor-metagenomeseq                                    446\n",
       "sparcc                                                        446\n",
       "treeswift                                                     445\n",
       "msaprobs                                                      445\n",
       "rapifilt                                                      444\n",
       "bioconductor-ihw                                              444\n",
       "gat                                                           443\n",
       "mtb-snp-it                                                    441\n",
       "tapestry                                                      440\n",
       "fraggenescan                                                  439\n",
       "r-bcbiosinglecell                                             439\n",
       "selectfasta                                                   438\n",
       "idba                                                          432\n",
       "bioconductor-ballgown                                         432\n",
       "arriba                                                        432\n",
       "dart                                                          431\n",
       "avro-python2                                                  431\n",
       "intervene                                                     429\n",
       "check-sort-order                                              429\n",
       "bwameth                                                       428\n",
       "methplotlib                                                   427\n",
       "hapcut2                                                       427\n",
       "isatools                                                      424\n",
       "bioconductor-edaseq                                           422\n",
       "bioconductor-cghbase                                          422\n",
       "tgsgapcloser                                                  421\n",
       "ataqv                                                         420\n",
       "bioconductor-org.rn.eg.db                                     420\n",
       "bioconductor-oligoclasses                                     417\n",
       "sfld                                                          417\n",
       "lastz                                                         416\n",
       "avro-python3                                                  416\n",
       "bioconductor-org.ce.eg.db                                     415\n",
       "bioconductor-flowviz                                          411\n",
       "cobra                                                         411\n",
       "bioconductor-ggcyto                                           410\n",
       "hap.py                                                        409\n",
       "bracken                                                       407\n",
       "graphmap                                                      407\n",
       "bioconductor-genenetworkbuilder                               406\n",
       "bioconductor-do.db                                            405\n",
       "bioconductor-org.gg.eg.db                                     404\n",
       "amptk                                                         402\n",
       "art                                                           402\n",
       "taxsbp                                                        402\n",
       "mergenotcombined                                              401\n",
       "fasta3                                                        401\n",
       "bioconductor-gage                                             401\n",
       "control-freec                                                 401\n",
       "tag                                                           400\n",
       "microhapdb                                                    400\n",
       "bioconductor-chipqc                                           398\n",
       "seqsero2                                                      398\n",
       "cdbtools                                                      397\n",
       "bioconductor-org.ss.eg.db                                     397\n",
       "nanocompore                                                   396\n",
       "bioconductor-pcaexplorer                                      396\n",
       "bioconductor-genie3                                           396\n",
       "coprarna                                                      396\n",
       "coverm                                                        396\n",
       "unmerge                                                       395\n",
       "clust                                                         395\n",
       "fastqpuri                                                     395\n",
       "strainseeker                                                  394\n",
       "ipyrad                                                        393\n",
       "bioconductor-nucpos                                           393\n",
       "bioconductor-org.bt.eg.db                                     392\n",
       "r-isva                                                        392\n",
       "bamutil                                                       391\n",
       "bioconductor-bionet                                           391\n",
       "bioconductor-motifdb                                          391\n",
       "bioconductor-basic4cseq                                       391\n",
       "tbvcfreport                                                   390\n",
       "piper                                                         390\n",
       "dnapi                                                         389\n",
       "pepquery                                                      389\n",
       "bioconductor-qdnaseq.mm10                                     389\n",
       "bioconductor-chicago                                          389\n",
       "irida-sistr-results                                           388\n",
       "bioconductor-heatplus                                         387\n",
       "bioconductor-motifstack                                       387\n",
       "fwdpp                                                         387\n",
       "scmap-cli                                                     387\n",
       "bioconductor-linnorm                                          387\n",
       "bioconductor-fourcseq                                         386\n",
       "bioconductor-rtandem                                          386\n",
       "bioconductor-org.cf.eg.db                                     385\n",
       "any2fasta                                                     385\n",
       "bioconductor-cexor                                            385\n",
       "bioconductor-sapfinder                                        385\n",
       "bioconductor-mitoode                                          385\n",
       "bioconductor-pchicdata                                        385\n",
       "bioconductor-alpine                                           384\n",
       "bioconductor-rgalaxy                                          384\n",
       "cmfinder                                                      384\n",
       "fastq-and-furious                                             384\n",
       "bioconductor-rnaseqpower                                      384\n",
       "bioconductor-gothic                                           384\n",
       "bioconductor-txdb.celegans.ucsc.ce11.refgene                  384\n",
       "bioconductor-cager                                            383\n",
       "bio-vcf                                                       383\n",
       "bioconductor-medips                                           383\n",
       "bioconductor-flowdensity                                      383\n",
       "bioconductor-piano                                            382\n",
       "bcftools-snvphyl-plugin                                       382\n",
       "dimspy                                                        382\n",
       "fadapa                                                        382\n",
       "r-cp4p                                                        381\n",
       "bioconductor-icare                                            380\n",
       "codingquarry                                                  380\n",
       "htsbox                                                        380\n",
       "bioconductor-esatac                                           380\n",
       "bioconductor-arrayexpresshts                                  380\n",
       "bustools                                                      379\n",
       "bioconductor-txdb.rnorvegicus.ucsc.rn5.refgene                379\n",
       "bioconductor-made4                                            379\n",
       "firecloud                                                     379\n",
       "bioconductor-tofsims                                          379\n",
       "bioconductor-genebreak                                        378\n",
       "bioconductor-outrider                                         378\n",
       "maaslin2                                                      378\n",
       "bioconductor-derfinderhelper                                  378\n",
       "bioconductor-mimosa                                           377\n",
       "tandem-genotypes                                              377\n",
       "bioconductor-kcsmart                                          377\n",
       "bioconductor-slalom                                           377\n",
       "ksw                                                           376\n",
       "phate                                                         376\n",
       "bioconductor-samspectral                                      375\n",
       "bioconductor-targetsearch                                     375\n",
       "bioconductor-rnits                                            375\n",
       "bioconductor-quantsmooth                                      375\n",
       "bioconductor-synergyfinder                                    375\n",
       "bioconductor-metams                                           374\n",
       "carna                                                         374\n",
       "cgpbigwig                                                     374\n",
       "bioconductor-probamr                                          374\n",
       "bioconductor-mutationalpatterns                               374\n",
       "bioconductor-mfa                                              374\n",
       "bioconductor-motifbreakr                                      374\n",
       "bioconductor-roar                                             374\n",
       "astalavista                                                   373\n",
       "bioconductor-bsgenome.hsapiens.ucsc.hg38                      373\n",
       "downpore                                                      373\n",
       "bioconductor-tarseqqc                                         372\n",
       "bioconductor-pgsea                                            372\n",
       "intermine                                                     370\n",
       "bioconductor-rnadecay                                         369\n",
       "plasmidfinder                                                 369\n",
       "bioconductor-methimpute                                       369\n",
       "bioconductor-cnordt                                           369\n",
       "bioconductor-geecc                                            368\n",
       "bioconductor-mait                                             368\n",
       "bioconductor-rhdf5client                                      368\n",
       "bioconductor-simat                                            368\n",
       "bioconductor-msnid                                            368\n",
       "lefse                                                         368\n",
       "bioconductor-discordant                                       368\n",
       "bioconductor-rdisop                                           367\n",
       "bioconductor-genextender                                      367\n",
       "cpat                                                          367\n",
       "bioconductor-enrichplot                                       367\n",
       "r-xgr                                                         366\n",
       "r-dnet                                                        366\n",
       "artemis                                                       364\n",
       "genomicconsensus                                              364\n",
       "bioconductor-clusterexperiment                                363\n",
       "pynnotator                                                    363\n",
       "ete2                                                          361\n",
       "bwakit                                                        361\n",
       "pyrodigal                                                     360\n",
       "fastobo                                                       359\n",
       "r-imputelcmd                                                  357\n",
       "hhsuite                                                       357\n",
       "chromhmm                                                      356\n",
       "bioconductor-bsgenome.hsapiens.ucsc.hg19                      355\n",
       "tetyper                                                       353\n",
       "r-gmwt                                                        348\n",
       "r-genomictools                                                348\n",
       "beagle                                                        348\n",
       "pyaavf                                                        348\n",
       "lace                                                          346\n",
       "fastool                                                       345\n",
       "links                                                         344\n",
       "collectl                                                      343\n",
       "pyomero-upload                                                343\n",
       "corset                                                        343\n",
       "bioconductor-annotationfuncs                                  341\n",
       "goalign                                                       340\n",
       "bioconductor-psicquic                                         337\n",
       "cat                                                           336\n",
       "bioconductor-txdb.hsapiens.ucsc.hg19.knowngene                336\n",
       "paraclu                                                       336\n",
       "seq2science                                                   335\n",
       "bamread                                                       334\n",
       "mnnpy                                                         333\n",
       "scpred-cli                                                    333\n",
       "microhapulator                                                332\n",
       "lighter                                                       332\n",
       "cancerit-allelecount                                          332\n",
       "python-bioformats                                             331\n",
       "seurat-scripts                                                331\n",
       "kaptive                                                       330\n",
       "bioconductor-reactome.db                                      330\n",
       "emperor                                                       329\n",
       "quantwiz-iq                                                   328\n",
       "tigmint                                                       328\n",
       "genesplicer                                                   328\n",
       "r-garnett                                                     328\n",
       "bioconvert                                                    326\n",
       "rilseq                                                        326\n",
       "bioconductor-bayseq                                           325\n",
       "aria2                                                         324\n",
       "wbuild                                                        324\n",
       "bioconductor-methylumi                                        322\n",
       "iced                                                          322\n",
       "bioconductor-suprahex                                         321\n",
       "gap2seq                                                       320\n",
       "bioconductor-chippeakanno                                     320\n",
       "ascat                                                         319\n",
       "fiji                                                          318\n",
       "flexbar                                                       317\n",
       "gfapy                                                         317\n",
       "bioconductor-recount                                          317\n",
       "biopet                                                        316\n",
       "coinfinder                                                    316\n",
       "gvcfgenotyper                                                 314\n",
       "treerecs                                                      314\n",
       "fusion-inspector                                              314\n",
       "bioconductor-splots                                           313\n",
       "idr-py                                                        312\n",
       "bioconductor-aldex2                                           312\n",
       "cgat-report                                                   312\n",
       "scanorama                                                     312\n",
       "bioconductor-sushi                                            311\n",
       "linkage2allegro                                               311\n",
       "rappas                                                        310\n",
       "fastspar                                                      310\n",
       "xmatchview                                                    309\n",
       "discosnp                                                      309\n",
       "bioconductor-msa                                              309\n",
       "cramtools                                                     308\n",
       "bbknn                                                         308\n",
       "bioconductor-opencyto                                         308\n",
       "bioconductor-destiny                                          307\n",
       "dfast                                                         307\n",
       "bioconductor-genvisr                                          307\n",
       "bioconductor-rgreat                                           306\n",
       "bioconductor-multidataset                                     306\n",
       "stringdecomposer                                              306\n",
       "grid                                                          306\n",
       "adam                                                          305\n",
       "iva                                                           305\n",
       "bioconductor-banocc                                           305\n",
       "hera                                                          304\n",
       "bioconductor-biodist                                          304\n",
       "bioconductor-mast                                             304\n",
       "bioconductor-globaltest                                       303\n",
       "bioconductor-splinter                                         303\n",
       "hicup                                                         302\n",
       "clairvoyante                                                  302\n",
       "bmfilter                                                      302\n",
       "bioconductor-rlmm                                             301\n",
       "bioconductor-asgsca                                           301\n",
       "bioconductor-flowclust                                        301\n",
       "bioconductor-cellhts2                                         301\n",
       "bioconductor-codex                                            300\n",
       "bioconductor-annaffy                                          300\n",
       "bioconductor-cqn                                              300\n",
       "bioconductor-qusage                                           300\n",
       "bioconductor-soggi                                            300\n",
       "bioconductor-fishalyser                                       300\n",
       "tasmanian-mismatch                                            300\n",
       "haystack_bio                                                  300\n",
       "bmtool                                                        300\n",
       "bioconductor-traser                                           299\n",
       "bioconductor-aims                                             299\n",
       "jamm                                                          299\n",
       "bioconductor-agilp                                            299\n",
       "bioconductor-arrayquality                                     299\n",
       "cage                                                          299\n",
       "bioconductor-acde                                             298\n",
       "bmtagger                                                      298\n",
       "extract_fullseq                                               298\n",
       "bioconductor-rgsepd                                           298\n",
       "fsnviz                                                        298\n",
       "bioconductor-gcspikelite                                      298\n",
       "besst                                                         298\n",
       "bioconductor-genomicscores                                    298\n",
       "bioconductor-a4preproc                                        298\n",
       "bioconductor-rpx                                              297\n",
       "bioconductor-txdb.rnorvegicus.ucsc.rn6.refgene                297\n",
       "bioconductor-sangerseqr                                       297\n",
       "bioconductor-roleswitch                                       297\n",
       "bioconductor-rontotools                                       297\n",
       "bioconductor-m3drop                                           297\n",
       "flapjack                                                      297\n",
       "bioconductor-quantro                                          297\n",
       "bioconductor-beat                                             297\n",
       "bioconductor-antiprofiles                                     297\n",
       "bioconductor-metcirc                                          297\n",
       "bioconductor-bgafun                                           297\n",
       "bioconductor-chromplot                                        297\n",
       "bioconductor-proper                                           297\n",
       "bioconductor-genomicdatacommons                               297\n",
       "bioconductor-cleaver                                          297\n",
       "bioconductor-atacseqqc                                        296\n",
       "bioconductor-biocworkflowtools                                296\n",
       "bioconductor-ssize                                            296\n",
       "bioconductor-affycomp                                         296\n",
       "bioconductor-rnaseqcomp                                       296\n",
       "bioconductor-affydata                                         296\n",
       "bioconductor-rbm                                              296\n",
       "bioconductor-asafe                                            296\n",
       "gatk-framework                                                295\n",
       "bioconductor-qubic                                            295\n",
       "bioconductor-psea                                             295\n",
       "bioconductor-arraytools                                       295\n",
       "bioconductor-oscope                                           295\n",
       "mirtrace                                                      295\n",
       "bioconductor-spia                                             295\n",
       "bioconductor-bbcanalyzer                                      295\n",
       "bioconductor-rtopper                                          295\n",
       "bioconductor-basespacer                                       295\n",
       "bioconductor-osat                                             294\n",
       "bioconductor-bgeedb                                           294\n",
       "bioconductor-anota                                            294\n",
       "bioconductor-absseq                                           294\n",
       "bioconductor-affycompatible                                   294\n",
       "bioconductor-heatmaps                                         294\n",
       "bioconductor-beadarraysnp                                     294\n",
       "bioconductor-iwtomics                                         294\n",
       "bioconductor-ruvseq                                           293\n",
       "bioconductor-cghregions                                       293\n",
       "bioconductor-mirnatap                                         293\n",
       "bioconductor-annmap                                           293\n",
       "bioconductor-clomial                                          293\n",
       "bioconductor-a4core                                           293\n",
       "daligner                                                      293\n",
       "bioconductor-stringdb                                         293\n",
       "refgenieserver                                                293\n",
       "bioconductor-rbsurv                                           293\n",
       "bioconductor-cand                                             293\n",
       "bioconductor-causalr                                          293\n",
       "bioconductor-segmentseq                                       293\n",
       "bioconductor-cfassay                                          293\n",
       "bioconductor-subseq                                           293\n",
       "bioconductor-uniprot.ws                                       293\n",
       "bioconductor-methped                                          293\n",
       "bioconductor-annotationtools                                  293\n",
       "bioconductor-mantelcorr                                       293\n",
       "bioconductor-badregionfinder                                  292\n",
       "snaptools                                                     292\n",
       "bioconductor-qpcrnorm                                         292\n",
       "prokaryote                                                    292\n",
       "bioconductor-snphood                                          292\n",
       "bioconductor-savr                                             292\n",
       "bioconductor-occugene                                         292\n",
       "bioconductor-clonality                                        292\n",
       "bioconductor-genomicfiles                                     292\n",
       "bioconductor-michip                                           292\n",
       "bioconductor-flowutils                                        292\n",
       "bioconductor-jaspar2016                                       292\n",
       "bioconductor-rtrm                                             292\n",
       "bioconductor-compepitools                                     291\n",
       "bioconductor-starank                                          291\n",
       "bioconductor-randpack                                         291\n",
       "admixtools                                                    291\n",
       "bioconductor-parody                                           291\n",
       "bioconductor-mdqc                                             291\n",
       "bioconductor-myvariant                                        291\n",
       "bioconductor-genomicinteractions                              291\n",
       "bioconductor-weaver                                           291\n",
       "bioconductor-mergeomics                                       291\n",
       "bioconductor-viper                                            291\n",
       "bioconductor-seqarray                                         291\n",
       "kiwidist                                                      291\n",
       "r-jackstraw                                                   291\n",
       "bioconductor-pcan                                             291\n",
       "bioconductor-triform                                          291\n",
       "bioconductor-timecourse                                       290\n",
       "bioconductor-agdex                                            290\n",
       "bioconductor-netbiov                                          290\n",
       "surpyvor                                                      290\n",
       "bioconductor-panp                                             290\n",
       "bioconductor-mbased                                           290\n",
       "bioconductor-flowrepositoryr                                  290\n",
       "bioconductor-macat                                            290\n",
       "bioconductor-cogena                                           290\n",
       "bioconductor-aspli                                            290\n",
       "bioconductor-globalseq                                        289\n",
       "bioconductor-tscan                                            289\n",
       "bioconductor-sscu                                             289\n",
       "bioconductor-safe                                             289\n",
       "bioconductor-ssviz                                            289\n",
       "bioconductor-affyilm                                          289\n",
       "bioconductor-affyrnadegradation                               289\n",
       "bioconductor-doppelgangr                                      289\n",
       "bioconductor-genocn                                           289\n",
       "bioconductor-affyexpress                                      289\n",
       "dazz_db                                                       289\n",
       "bioconductor-ipo                                              289\n",
       "bioconductor-garfield                                         289\n",
       "bioconductor-lea                                              289\n",
       "bioconductor-bags                                             289\n",
       "bioconductor-ntw                                              289\n",
       "bioconductor-ppinfer                                          289\n",
       "bioconductor-farms                                            289\n",
       "bioconductor-slqpcr                                           289\n",
       "dudes                                                         289\n",
       "bioconductor-rpa                                              288\n",
       "bioconductor-nupop                                            288\n",
       "bioconductor-affycontam                                       288\n",
       "bioconductor-pvac                                             288\n",
       "bioconductor-curatedovariandata                               288\n",
       "bioconductor-trio                                             288\n",
       "bioconductor-seqgsea                                          288\n",
       "bioconductor-rebet                                            288\n",
       "bioconductor-cghmcr                                           288\n",
       "bioconductor-outlierd                                         288\n",
       "fastme                                                        288\n",
       "bioconductor-mgfm                                             288\n",
       "bioconductor-aneufinder                                       287\n",
       "bioconductor-rbowtie2                                         287\n",
       "bioconductor-simbindprofiles                                  287\n",
       "bioconductor-derfinder                                        287\n",
       "bioconductor-riboprofiling                                    287\n",
       "andi                                                          287\n",
       "bioconductor-rqc                                              287\n",
       "bioconductor-gosim                                            287\n",
       "hlama                                                         287\n",
       "bioconductor-pepxmltab                                        287\n",
       "bioconductor-biseq                                            287\n",
       "bioconductor-chronos                                          287\n",
       "bioconductor-scmap                                            287\n",
       "bioconductor-rtcgatoolbox                                     287\n",
       "bioconductor-simd                                             287\n",
       "bioconductor-dmrcaller                                        287\n",
       "bioconductor-bnbc                                             286\n",
       "kneaddata                                                     286\n",
       "transposcope                                                  286\n",
       "bioconductor-path2ppi                                         286\n",
       "bioconductor-npgsea                                           286\n",
       "bioconductor-teqc                                             286\n",
       "bioconductor-geneselectmmd                                    286\n",
       "bioconductor-pviz                                             286\n",
       "bioconductor-grohmm                                           286\n",
       "bioconductor-clipper                                          286\n",
       "bioconductor-matrixrider                                      286\n",
       "bioconductor-cosnet                                           286\n",
       "bioconductor-rrho                                             286\n",
       "bioconductor-bdmmacorrect                                     286\n",
       "bioconductor-varianttools                                     285\n",
       "bioconductor-netresponse                                      285\n",
       "bioconductor-analysispageserver                               285\n",
       "bioconductor-mlp                                              285\n",
       "bioconductor-mirnapath                                        285\n",
       "bioconductor-diffhic                                          285\n",
       "bioconductor-cssp                                             285\n",
       "bioconductor-methylseekr                                      285\n",
       "gecode                                                        285\n",
       "barriers                                                      285\n",
       "bioconductor-mbttest                                          285\n",
       "bioconductor-multiomicsviz                                    285\n",
       "assemblerflow                                                 284\n",
       "bioconductor-normr                                            284\n",
       "bioconductor-topaseq                                          284\n",
       "bioconductor-gmapr                                            284\n",
       "bioconductor-fmcsr                                            284\n",
       "bioconductor-hdtd                                             284\n",
       "bioconductor-rtrmui                                           284\n",
       "bioconductor-transite                                         284\n",
       "addrg                                                         284\n",
       "bioconductor-opossom                                          284\n",
       "bioconductor-r3cseq                                           284\n",
       "bioconductor-motifmatchr                                      284\n",
       "bioconductor-copa                                             284\n",
       "bioconductor-rain                                             284\n",
       "bioconductor-rbiopaxparser                                    284\n",
       "bioconductor-cellmapper                                       284\n",
       "bioconductor-sights                                           284\n",
       "bioconductor-universalmotif                                   283\n",
       "bioconductor-rgin                                             283\n",
       "bioconductor-multimed                                         283\n",
       "bioconductor-mvcclass                                         283\n",
       "bioconductor-phenopath                                        283\n",
       "bioconductor-mgfr                                             283\n",
       "bioconductor-mbamethyl                                        283\n",
       "bioconductor-metacca                                          283\n",
       "bioconductor-psichomics                                       283\n",
       "bioconductor-macorrplot                                       283\n",
       "bioconductor-sseq                                             283\n",
       "bioconductor-omiccircos                                       283\n",
       "howdesbt                                                      283\n",
       "bioconductor-rgsea                                            283\n",
       "bioconductor-omicade4                                         283\n",
       "bioconductor-pathnet                                          283\n",
       "bioconductor-arrayexpress                                     283\n",
       "deepmedic                                                     283\n",
       "bioconductor-somaticsignatures                                283\n",
       "bioconductor-massarray                                        282\n",
       "bioconductor-mpfe                                             282\n",
       "bioconductor-rimmport                                         282\n",
       "bioconductor-pickgene                                         282\n",
       "bioconductor-methtargetedngs                                  282\n",
       "bioconductor-rcaspar                                          282\n",
       "bioconductor-mirsynergy                                       282\n",
       "bioconductor-xmapbridge                                       282\n",
       "bioconductor-scan.upc                                         282\n",
       "bioconductor-spktools                                         282\n",
       "bioconductor-runibic                                          282\n",
       "bioconductor-spem                                             282\n",
       "bioconductor-snpediar                                         282\n",
       "gtfparse                                                      282\n",
       "bioconductor-sizepower                                        282\n",
       "bioconductor-squadd                                           282\n",
       "bioconductor-polyfit                                          282\n",
       "bioconductor-spikeli                                          282\n",
       "bioconductor-ibmq                                             282\n",
       "bioconductor-orfik                                            282\n",
       "bioconductor-similarpeak                                      282\n",
       "bioconductor-mdgsa                                            282\n",
       "bioconductor-nnnorm                                           282\n",
       "bioconductor-bcseq                                            282\n",
       "bioconductor-stepnorm                                         282\n",
       "bioconductor-swathxtend                                       282\n",
       "bioconductor-ncigraph                                         282\n",
       "bioconductor-muscle                                           282\n",
       "bioconductor-basics                                           282\n",
       "bioconductor-plethy                                           282\n",
       "bioconductor-switchde                                         282\n",
       "bioconductor-rols                                             282\n",
       "bioconductor-plgem                                            282\n",
       "bioconductor-vbmp                                             282\n",
       "bioconductor-targetscore                                      282\n",
       "bioconductor-txdb.drerio.ucsc.danrer10.refgene                282\n",
       "bioconductor-semdist                                          282\n",
       "bioconductor-yeastexpdata                                     282\n",
       "bioconductor-baynorm                                          282\n",
       "bioconductor-r3cpet                                           282\n",
       "bioconductor-rtca                                             282\n",
       "bioconductor-pepstat                                          282\n",
       "bioconductor-sagenhaft                                        282\n",
       "bioconductor-mirage                                           282\n",
       "bioconductor-olin                                             282\n",
       "bioconductor-rdgidb                                           282\n",
       "bioconductor-recoup                                           281\n",
       "bioconductor-mops                                             281\n",
       "bioconductor-pkgdeptools                                      281\n",
       "bioconductor-rgraph2js                                        281\n",
       "bioconductor-massir                                           281\n",
       "bioconductor-sscore                                           281\n",
       "bioconductor-promise                                          281\n",
       "bioconductor-semisup                                          281\n",
       "bioconductor-riboseqr                                         281\n",
       "bioconductor-hicrep                                           281\n",
       "bioconductor-swath2stats                                      281\n",
       "bioconductor-mipp                                             281\n",
       "bioconductor-ogsa                                             281\n",
       "bioconductor-xbseq                                            281\n",
       "bioconductor-rnaprobr                                         281\n",
       "bioconductor-mergemaid                                        281\n",
       "bioconductor-nucler                                           281\n",
       "crumble                                                       281\n",
       "bioconductor-rrdp                                             281\n",
       "bioconductor-sgseq                                            281\n",
       "bioconductor-sispa                                            281\n",
       "bioconductor-peca                                             281\n",
       "bioconductor-specond                                          281\n",
       "bioconductor-rbcbook1                                         281\n",
       "bioconductor-meshdbi                                          281\n",
       "bioconductor-paircompviz                                      281\n",
       "bioconductor-rsvsim                                           281\n",
       "bioconductor-otubase                                          281\n",
       "bioconductor-jaspar2014                                       281\n",
       "bioconductor-readqpcr                                         280\n",
       "bioconductor-flowtype                                         280\n",
       "eoulsan                                                       280\n",
       "bioconductor-r4rna                                            280\n",
       "bioconductor-graphalignment                                   280\n",
       "bioconductor-nethet                                           280\n",
       "bioconductor-qualifier                                        280\n",
       "bioconductor-wiggleplotr                                      280\n",
       "bioconductor-rmmquant                                         280\n",
       "bioconductor-stategra                                         280\n",
       "bioconductor-narrowpeaks                                      280\n",
       "bioconductor-nucleosim                                        280\n",
       "bioconductor-nanostringdiff                                   280\n",
       "bioconductor-pcot2                                            280\n",
       "bioconductor-isogenegui                                       280\n",
       "bioconductor-edge                                             279\n",
       "bioconductor-expressionview                                   279\n",
       "bioconductor-refnet                                           279\n",
       "bioconductor-bprmeth                                          279\n",
       "bioconductor-enrichedheatmap                                  279\n",
       "hubward                                                       279\n",
       "bioconductor-biomvrcns                                        279\n",
       "dxua                                                          278\n",
       "bioconductor-ihwpaper                                         278\n",
       "bioconductor-ibbig                                            278\n",
       "bolt                                                          278\n",
       "bioconductor-gosummaries                                      278\n",
       "bioconductor-paa                                              278\n",
       "cityhash                                                      278\n",
       "bioconductor-csar                                             277\n",
       "bioconductor-hireewas                                         277\n",
       "bioconductor-martini                                          277\n",
       "bioconductor-netbenchmark                                     277\n",
       "bioconductor-fdrame                                           277\n",
       "bioconductor-immunoclust                                      277\n",
       "bioconductor-cgen                                             277\n",
       "bioconductor-coregnet                                         277\n",
       "bioconductor-chipseqr                                         277\n",
       "bioconductor-fastqcleaner                                     277\n",
       "bioconductor-ramwas                                           277\n",
       "bioconductor-rmassbank                                        277\n",
       "bioconductor-geneticsped                                      277\n",
       "bioconductor-eir                                              276\n",
       "bioconductor-r453plus1toolbox                                 276\n",
       "bioconductor-ebcoexpress                                      276\n",
       "bioconductor-seqbias                                          276\n",
       "bioconductor-tilingarray                                      276\n",
       "bioconductor-grenits                                          276\n",
       "bioconductor-cogaps                                           276\n",
       "bioconductor-buscorrect                                       276\n",
       "bioconductor-rjmcmcnucleosomes                                276\n",
       "bioconductor-gpart                                            276\n",
       "bioconductor-srnadiff                                         276\n",
       "bioconductor-profilescoredist                                 276\n",
       "bioconductor-rqubic                                           276\n",
       "bioconductor-levi                                             276\n",
       "bioconductor-h5vc                                             276\n",
       "bioconductor-reb                                              276\n",
       "bioconductor-gopro                                            276\n",
       "bioconductor-predictionet                                     276\n",
       "bioconductor-rchyoptimyx                                      276\n",
       "bioconductor-cydar                                            276\n",
       "bioconductor-simulatorz                                       276\n",
       "bioconductor-macpet                                           276\n",
       "bioconductor-scfind                                           276\n",
       "bioconductor-gofuncr                                          276\n",
       "bioconductor-descan2                                          276\n",
       "bioconductor-cancerclass                                      276\n",
       "bioconductor-ccfindr                                          276\n",
       "bioconductor-decomplexdisease                                 276\n",
       "sc3-scripts                                                   275\n",
       "wes-service-client                                            275\n",
       "drep                                                          275\n",
       "bioconductor-zinbwave                                         274\n",
       "bioconductor-sictools                                         273\n",
       "r-bisquerna                                                   272\n",
       "vembrane                                                      271\n",
       "sigmut                                                        268\n",
       "fastq-multx                                                   267\n",
       "c-ares                                                        267\n",
       "bioconductor-reactomepa                                       266\n",
       "graphlan                                                      261\n",
       "bioconductor-batchelor                                        259\n",
       "triqler                                                       259\n",
       "bioconductor-deformats                                        259\n",
       "illumina-utils                                                258\n",
       "smaca                                                         258\n",
       "flash2                                                        257\n",
       "hmftools-purple                                               256\n",
       "npinv                                                         256\n",
       "burrito                                                       254\n",
       "smof                                                          254\n",
       "hclust2                                                       254\n",
       "perl-ensembl-core                                             253\n",
       "scaden                                                        253\n",
       "bam2fastx                                                     250\n",
       "bio_assembly_refinement                                       250\n",
       "assembly-stats                                                250\n",
       "bioconductor-txdb.hsapiens.ucsc.hg38.knowngene                249\n",
       "kobas                                                         249\n",
       "gblocks                                                       247\n",
       "bioconductor-drimseq                                          245\n",
       "burrito-fillings                                              244\n",
       "mango                                                         244\n",
       "bioconductor-maftools                                         243\n",
       "seqan3                                                        242\n",
       "genrich                                                       241\n",
       "r-diffcorr                                                    240\n",
       "bioconductor-crisprseek                                       240\n",
       "bioconductor-glimma                                           239\n",
       "galaxy-ops                                                    239\n",
       "bioconductor-cytolib                                          239\n",
       "canvas                                                        238\n",
       "fgmp                                                          237\n",
       "phylovega                                                     236\n",
       "eigensoft                                                     236\n",
       "ragtag                                                        236\n",
       "cgcloud-lib                                                   234\n",
       "transcov                                                      234\n",
       "ea-utils                                                      233\n",
       "break-point-inspector                                         233\n",
       "vsnp                                                          233\n",
       "hts-nim-tools                                                 233\n",
       "libgenome                                                     233\n",
       "seq-to-first-iso                                              231\n",
       "r-prestor                                                     231\n",
       "align_it                                                      231\n",
       "bioconductor-kegg.db                                          231\n",
       "kodoja                                                        230\n",
       "salmid                                                        230\n",
       "bioconductor-cma                                              228\n",
       "biotradis                                                     228\n",
       "dropseq_tools                                                 228\n",
       "hypo                                                          227\n",
       "bcalm                                                         226\n",
       "bcl2fastq-nextseq                                             225\n",
       "bioconductor-variancepartition                                225\n",
       "bandage                                                       224\n",
       "dsh-bio                                                       224\n",
       "beast2                                                        224\n",
       "ectyper                                                       223\n",
       "bioconductor-ctc                                              223\n",
       "damageprofiler                                                222\n",
       "bioconductor-ensdb.hsapiens.v75                               222\n",
       "htstream                                                      221\n",
       "bioconductor-tcgabiolinks                                     219\n",
       "dsrc                                                          219\n",
       "req                                                           218\n",
       "ultraheatmap                                                  218\n",
       "libmuscle                                                     218\n",
       "export2graphlan                                               218\n",
       "perl-data-predicate                                           218\n",
       "bioconductor-ensemblvep                                       218\n",
       "perl-ensembl-funcgen                                          217\n",
       "libmems                                                       217\n",
       "ficus                                                         217\n",
       "illumiprocessor                                               216\n",
       "bioconductor-cytoml                                           216\n",
       "deepbinner                                                    216\n",
       "mapgl                                                         216\n",
       "bioconductor-watermelon                                       215\n",
       "sarscov2summary                                               215\n",
       "bioconductor-lumi                                             215\n",
       "pygmes                                                        215\n",
       "shapeshifter                                                  215\n",
       "abra2                                                         215\n",
       "lordec                                                        215\n",
       "gemma                                                         215\n",
       "gem3-mapper                                                   214\n",
       "connor                                                        214\n",
       "perl-http-tiny                                                214\n",
       "bioconductor-dyndoc                                           213\n",
       "crimson                                                       213\n",
       "bioconductor-microbiome                                       213\n",
       "ant                                                           213\n",
       "biopet-extractadaptersfastqc                                  212\n",
       "bioconductor-geneoverlap                                      212\n",
       "bioconductor-multiassayexperiment                             212\n",
       "glimmer                                                       212\n",
       "basic                                                         211\n",
       "r-fastbaps                                                    211\n",
       "perl-ensembl-variation                                        211\n",
       "fred2                                                         211\n",
       "agfusion                                                      211\n",
       "perl-ensembl-io                                               211\n",
       "perl-ensembl-compara                                          210\n",
       "bamscale                                                      210\n",
       "r-harmony                                                     210\n",
       "bioconductor-copyhelper                                       210\n",
       "bioconductor-mygene                                           210\n",
       "bioconductor-padog                                            210\n",
       "bioconductor-scnorm                                           209\n",
       "bioconductor-geometadb                                        209\n",
       "bioconductor-bladderbatch                                     209\n",
       "clonalframeml                                                 209\n",
       "graphprot                                                     209\n",
       "ctat-mutations                                                209\n",
       "pgx-variant-tools                                             208\n",
       "bioconductor-experimenthub                                    208\n",
       "fastlmm                                                       208\n",
       "bioconductor-txdb.mmusculus.ucsc.mm10.ensgene                 208\n",
       "bioconductor-org.sc.sgd.db                                    208\n",
       "express                                                       208\n",
       "bioconductor-synlet                                           208\n",
       "spotyping3                                                    208\n",
       "dedup                                                         208\n",
       "bioconductor-diffloop                                         207\n",
       "bioconductor-impulsede                                        207\n",
       "bioconductor-abadata                                          207\n",
       "bioconductor-missmethyl                                       207\n",
       "impute2                                                       207\n",
       "r-scpred                                                      207\n",
       "bctools                                                       206\n",
       "gadem                                                         206\n",
       "nanopolishcomp                                                206\n",
       "bioconductor-egseadata                                        206\n",
       "eden                                                          206\n",
       "bioconductor-ctsge                                            206\n",
       "je-suite                                                      206\n",
       "bioconductor-clst                                             206\n",
       "hardklor                                                      206\n",
       "chipseq-greylist                                              205\n",
       "magicblast                                                    205\n",
       "bioconductor-dmrcate                                          205\n",
       "bioconductor-biosvd                                           205\n",
       "sarscov2formatter                                             205\n",
       "bioconductor-a4reporting                                      205\n",
       "bioconductor-clustcomp                                        205\n",
       "bioconductor-brain                                            205\n",
       "bioconductor-arrmnormalization                                205\n",
       "krocus                                                        205\n",
       "bioconductor-basecallqc                                       204\n",
       "bioconductor-eudysbiome                                       204\n",
       "bioconductor-htsfilter                                        204\n",
       "bioconductor-splatter                                         204\n",
       "bioconductor-rnbeads                                          204\n",
       "bioconductor-clstutils                                        204\n",
       "afterqc                                                       204\n",
       "bioconductor-chromstardata                                    204\n",
       "bioconductor-rtcga                                            204\n",
       "hla-la                                                        204\n",
       "bioconductor-golubesets                                       204\n",
       "perl-ensembl-api                                              204\n",
       "bioconductor-hiccompare                                       204\n",
       "find_differential_primers                                     204\n",
       "bioconductor-spotsegmentation                                 203\n",
       "bioconductor-mirbaseversions.db                               203\n",
       "domclust                                                      203\n",
       "bioconductor-arrayqualitymetrics                              203\n",
       "autodock-vina                                                 203\n",
       "bpp-phyl                                                      203\n",
       "bioconductor-rankprod                                         203\n",
       "bioconductor-cogps                                            203\n",
       "bioconductor-nondetects                                       203\n",
       "bioconductor-consensusseeker                                  203\n",
       "bioconductor-arrmdata                                         203\n",
       "bioconductor-dama                                             203\n",
       "bioconductor-copywriter                                       203\n",
       "dreamtools                                                    203\n",
       "jaffa                                                         203\n",
       "bioconductor-gewist                                           203\n",
       "bioconductor-brainstars                                       203\n",
       "bioconductor-pfam.db                                          203\n",
       "bioconductor-clustersignificance                              203\n",
       "bioconductor-cindex                                           203\n",
       "idba_subasm                                                   203\n",
       "bioconductor-rcy3                                             203\n",
       "berokka                                                       203\n",
       "bioconductor-unifiedwmwqpcr                                   202\n",
       "bioconductor-correp                                           202\n",
       "bioconductor-hiannotator                                      202\n",
       "bioconductor-snm                                              202\n",
       "faqcs                                                         202\n",
       "bioconductor-tracktables                                      202\n",
       "bioconductor-deqms                                            202\n",
       "bioconductor-a4base                                           202\n",
       "bioconductor-help                                             202\n",
       "ms2pip                                                        202\n",
       "bioconductor-lungcanceracvssccgeo                             202\n",
       "cesm                                                          202\n",
       "bioconductor-cohcapanno                                       202\n",
       "gmap-fusion                                                   202\n",
       "bioconductor-lpe                                              202\n",
       "bioconductor-gsca                                             202\n",
       "graftm                                                        202\n",
       "bioconductor-idiogram                                         202\n",
       "bioconductor-batchqc                                          201\n",
       "bamhash                                                       201\n",
       "bioconductor-chipsim                                          201\n",
       "bioconductor-inpower                                          201\n",
       "bioconductor-genega                                           201\n",
       "bioconductor-chipxpressdata                                   201\n",
       "cap3                                                          201\n",
       "bioconductor-lmdme                                            201\n",
       "bioconductor-gotools                                          201\n",
       "circos-tools                                                  201\n",
       "bioconductor-genbankr                                         201\n",
       "bioconductor-epivizrserver                                    201\n",
       "bioconductor-generecommender                                  201\n",
       "bioconductor-clusterstab                                      201\n",
       "bioconductor-genefu                                           201\n",
       "bioconductor-inspect                                          200\n",
       "primedrpa                                                     200\n",
       "bioconductor-ipac                                             200\n",
       "ivar                                                          200\n",
       "bioconductor-chromheatmap                                     200\n",
       "bioconductor-dbchip                                           200\n",
       "bioconductor-consensusov                                      200\n",
       "bioconductor-undo                                             200\n",
       "bioconductor-generegionscan                                   200\n",
       "bioconductor-yeastcc                                          200\n",
       "bioconductor-reportingtools                                   200\n",
       "bioconductor-encodexplorer                                    200\n",
       "bioconductor-clonotyper                                       200\n",
       "capc-map                                                      200\n",
       "dropest                                                       200\n",
       "bioconductor-codelink                                         200\n",
       "bioconductor-dupchecker                                       200\n",
       "bioconductor-mapredictdsc                                     200\n",
       "bioconductor-swimr                                            200\n",
       "bioconductor-txdb.dmelanogaster.ucsc.dm6.ensgene              200\n",
       "bioconductor-annotationhubdata                                200\n",
       "bioconductor-dmrcatedata                                      200\n",
       "dbg2olc                                                       200\n",
       "json_collect_data_source                                      200\n",
       "bioconductor-convert                                          199\n",
       "bioconductor-director                                         199\n",
       "bioconductor-ebsea                                            199\n",
       "bioconductor-drivernet                                        199\n",
       "msp2db                                                        199\n",
       "bioconductor-philr                                            199\n",
       "bioconductor-covrna                                           199\n",
       "bioconductor-kegglincs                                        199\n",
       "bioconductor-panr                                             199\n",
       "bioconductor-cancersubtypes                                   199\n",
       "hg-color                                                      199\n",
       "bioconductor-kimod                                            199\n",
       "bioconductor-imas                                             199\n",
       "age-metasv                                                    199\n",
       "bioconductor-browserviz                                       199\n",
       "bioconductor-egsea                                            199\n",
       "bioconductor-geneattribution                                  199\n",
       "adapterremovalfixprefix                                       199\n",
       "bioconductor-demand                                           199\n",
       "bioconductor-shinymethyl                                      199\n",
       "bioconductor-procoil                                          199\n",
       "bioconductor-genrank                                          199\n",
       "bioconductor-ggbase                                           199\n",
       "bioconductor-classifyr                                        199\n",
       "bioconductor-seq2pathway                                      199\n",
       "bioconductor-genomeintervals                                  199\n",
       "bioconductor-rnbeads.hg19                                     199\n",
       "bioconductor-trackviewer                                      199\n",
       "bioconductor-keggdzpathwaysgeo                                198\n",
       "bioconductor-pathifier                                        198\n",
       "bioconductor-chimeraviz                                       198\n",
       "bioconductor-lola                                             198\n",
       "bpp-core                                                      198\n",
       "fermi2                                                        198\n",
       "bioconductor-zfpkm                                            198\n",
       "bioconductor-geneplast                                        198\n",
       "bioconductor-flowbeads                                        198\n",
       "bioconductor-affycoretools                                    198\n",
       "ghostz                                                        198\n",
       "bioconductor-grmetrics                                        198\n",
       "bioconductor-rnbeads.mm10                                     198\n",
       "bioconductor-drugvsdiseasedata                                198\n",
       "bpp-seq                                                       198\n",
       "bioconductor-pandar                                           198\n",
       "bioconductor-easyrnaseq                                       198\n",
       "bioconductor-diffgeneanalysis                                 198\n",
       "bioconductor-spidermir                                        198\n",
       "maaslin                                                       198\n",
       "bioconductor-crisprvariants                                   198\n",
       "bioconductor-plpe                                             198\n",
       "bioconductor-geneticsdesign                                   198\n",
       "bioconductor-tvtb                                             197\n",
       "circularmapper                                                197\n",
       "bioconductor-ndexr                                            197\n",
       "fastqsplitter                                                 197\n",
       "bioconductor-apcomplex                                        197\n",
       "bioconductor-refplus                                          197\n",
       "bioconductor-empiricalbrownsmethod                            197\n",
       "bioconductor-pwmenrich                                        197\n",
       "geodl                                                         197\n",
       "bioconductor-ggtools                                          197\n",
       "bioconductor-cmap2data                                        197\n",
       "genmap                                                        197\n",
       "frogs                                                         197\n",
       "bioconductor-scisi                                            197\n",
       "bioconductor-flowploidy                                       197\n",
       "bioconductor-jaspar2018                                       197\n",
       "bioconductor-genemeta                                         197\n",
       "bioconductor-dta                                              197\n",
       "bioconductor-dyebias                                          197\n",
       "bioconductor-pvca                                             197\n",
       "guidescan                                                     197\n",
       "bioconductor-helloranges                                      197\n",
       "bioconductor-cleanupdtseq                                     197\n",
       "gdsctools                                                     197\n",
       "bioconductor-eximir                                           197\n",
       "bioconductor-cnvpanelizer                                     197\n",
       "bioconductor-ppistats                                         196\n",
       "ghm                                                           196\n",
       "gretel                                                        196\n",
       "bioconductor-ruvnormalizedata                                 196\n",
       "hits                                                          196\n",
       "imfusion                                                      196\n",
       "bioconductor-ruvnormalize                                     196\n",
       "bioconductor-mlinterfaces                                     196\n",
       "bioconductor-lpnet                                            196\n",
       "bioconductor-cnvgsa                                           196\n",
       "islandpath                                                    196\n",
       "bioconductor-ffpe                                             196\n",
       "bioconductor-interminer                                       196\n",
       "bioconductor-amplican                                         196\n",
       "bioconductor-focalcall                                        196\n",
       "bioconductor-dfp                                              196\n",
       "bioconductor-epivizrdata                                      196\n",
       "bioconductor-flowclean                                        196\n",
       "bioconductor-biomvcclass                                      196\n",
       "bioconductor-guideseq                                         196\n",
       "bioconductor-cpvsnp                                           196\n",
       "bioconductor-cormotif                                         196\n",
       "bioconductor-gcrisprtools                                     196\n",
       "bioconductor-cghnormaliter                                    196\n",
       "consensusfixer                                                196\n",
       "bioconductor-translatome                                      195\n",
       "bioconductor-yaqcaffy                                         195\n",
       "bioconductor-crimage                                          195\n",
       "bioconductor-stemhypoxia                                      195\n",
       "lorikeet                                                      195\n",
       "bioconductor-psygenet2r                                       195\n",
       "bioconductor-crisprseekplus                                   195\n",
       "bioconductor-pathostat                                        195\n",
       "bioconductor-genetclassifier                                  195\n",
       "bioconductor-madseq                                           195\n",
       "bioconductor-yarn                                             195\n",
       "bioconductor-reder                                            195\n",
       "arcs                                                          195\n",
       "bioconductor-raggedexperiment                                 195\n",
       "dia_umpire                                                    195\n",
       "bioconductor-gofunction                                       195\n",
       "bioconductor-meshes                                           195\n",
       "bioconductor-svaplsseq                                        195\n",
       "bioconductor-epivizrstandalone                                195\n",
       "bioconductor-deepbluer                                        195\n",
       "bioconductor-orderedlist                                      195\n",
       "bioconductor-spacepac                                         195\n",
       "bioconductor-hitc                                             195\n",
       "bioconductor-baalchip                                         195\n",
       "bioconductor-icens                                            195\n",
       "bioconductor-keggorthology                                    195\n",
       "bioconductor-htqpcr                                           195\n",
       "phylorank                                                     195\n",
       "bioconductor-gmrp                                             195\n",
       "bioconductor-mlseq                                            195\n",
       "eagle                                                         195\n",
       "bioconductor-drugvsdisease                                    195\n",
       "bioconductor-uniquorn                                         195\n",
       "bioconductor-biocgraph                                        194\n",
       "gb_taxonomy_tools                                             194\n",
       "bioconductor-multiclust                                       194\n",
       "bioconductor-ideal                                            194\n",
       "bioconductor-altcdfenvs                                       194\n",
       "funnel                                                        194\n",
       "bioconductor-fccac                                            194\n",
       "bioconductor-curatedmetagenomicdata                           194\n",
       "bioconductor-seq2pathway.data                                 194\n",
       "maskrc-svg                                                    194\n",
       "bioconductor-ddct                                             194\n",
       "bioconductor-simpintlists                                     194\n",
       "bioconductor-rarevariantvis                                   194\n",
       "bcbio-variation                                               194\n",
       "ale                                                           194\n",
       "bioconductor-dcgsa                                            194\n",
       "ecopy                                                         194\n",
       "r-isogene                                                     194\n",
       "edd                                                           194\n",
       "bioconductor-webbioc                                          194\n",
       "bioconductor-iterativebma                                     194\n",
       "bioconductor-gaprediction                                     193\n",
       "lordfast                                                      193\n",
       "bioconductor-pwomics                                          193\n",
       "athena_meta                                                   193\n",
       "bioconductor-seqcna                                           193\n",
       "bioconductor-frma                                             193\n",
       "bioconductor-epivizr                                          193\n",
       "bioconductor-mwastools                                        193\n",
       "bioconductor-chipxpress                                       193\n",
       "biopet-vcfstats                                               193\n",
       "bioconductor-frmatools                                        193\n",
       "bioconductor-mgug4122a.db                                     193\n",
       "combined-pvalues                                              193\n",
       "bioconductor-gtrellis                                         193\n",
       "bioconductor-rcgh                                             193\n",
       "bioconductor-degraph                                          193\n",
       "graph_embed                                                   193\n",
       "disco                                                         193\n",
       "cwl2wdl                                                       193\n",
       "bioconductor-methyanalysis                                    193\n",
       "bioconductor-flowsorted.blood.450k                            193\n",
       "bioconductor-regsplice                                        193\n",
       "bioconductor-prebs                                            193\n",
       "akt                                                           193\n",
       "bioconductor-epigenomix                                       193\n",
       "bioconductor-preda                                            193\n",
       "bioconductor-transcriptr                                      192\n",
       "eqtlbma                                                       192\n",
       "stoatydive                                                    192\n",
       "bioconductor-esetvis                                          192\n",
       "bioconductor-graphpac                                         192\n",
       "bioconductor-hyperdraw                                        192\n",
       "bioconductor-dchiprep                                         192\n",
       "recognizer                                                    192\n",
       "bioconductor-gsalightning                                     192\n",
       "bcbio_monitor                                                 192\n",
       "bioconductor-rtcga.mrna                                       192\n",
       "bioconductor-les                                              192\n",
       "bioconductor-phastcons100way.ucsc.hg19                        192\n",
       "bioconductor-phosphonormalizer                                192\n",
       "bioconductor-fis                                              192\n",
       "lambda                                                        192\n",
       "bioconductor-lbe                                              192\n",
       "bioconductor-pmm                                              192\n",
       "bioconductor-pathprint                                        192\n",
       "bioconductor-geosubmission                                    192\n",
       "bioconductor-graphat                                          191\n",
       "bioconductor-kodata                                           191\n",
       "yak                                                           191\n",
       "fmlrc                                                         191\n",
       "bioconductor-countclust                                       191\n",
       "bioconductor-rnainteract                                      191\n",
       "drive                                                         191\n",
       "bioconductor-smite                                            191\n",
       "discasm                                                       191\n",
       "elector                                                       191\n",
       "bioconductor-sigfuge                                          191\n",
       "bioconductor-splinetimer                                      191\n",
       "bioconductor-iaseq                                            191\n",
       "bioconductor-derfinderplot                                    191\n",
       "kmasker                                                       191\n",
       "bioconductor-amaretto                                         191\n",
       "bioconductor-wavcluster                                       190\n",
       "chanjo                                                        190\n",
       "xbioc                                                         190\n",
       "embl-api-validator                                            190\n",
       "bioconductor-metabosignal                                     190\n",
       "bioconductor-icobra                                           190\n",
       "bioconductor-ibh                                              190\n",
       "fido                                                          190\n",
       "bioconductor-tdaracne                                         190\n",
       "bioconductor-iterativebmasurv                                 190\n",
       "hotspot3d                                                     190\n",
       "fuma                                                          190\n",
       "bis-snp                                                       190\n",
       "bioconductor-scmerge                                          190\n",
       "bioconductor-tin                                              190\n",
       "bioconductor-nadfinder                                        190\n",
       "bioconductor-gsealm                                           190\n",
       "bioconductor-tcc                                              190\n",
       "bioconductor-starbiotrek                                      189\n",
       "bioconductor-hybridmtest                                      189\n",
       "bioconductor-snagee                                           189\n",
       "bioconductor-snageedata                                       189\n",
       "bioconductor-prize                                            189\n",
       "bioconductor-process                                          189\n",
       "bioconductor-srap                                             189\n",
       "libstatgen                                                    189\n",
       "bioconductor-mbkmeans                                         189\n",
       "bioconductor-qcmetrics                                        189\n",
       "bioconductor-yapsa                                            189\n",
       "bioconductor-gpls                                             189\n",
       "bioconductor-interest                                         189\n",
       "bioconductor-genphen                                          189\n",
       "bioconductor-ioniser                                          189\n",
       "bioconductor-ideoviz                                          189\n",
       "bioconductor-hypergraph                                       189\n",
       "bioconductor-pathvar                                          189\n",
       "fast5seek                                                     189\n",
       "hapbin                                                        189\n",
       "bioconductor-rpsixml                                          189\n",
       "bioconductor-flowmap                                          189\n",
       "bioconductor-lpeadj                                           189\n",
       "asqcan                                                        189\n",
       "allegro                                                       189\n",
       "bioconductor-liquidassociation                                189\n",
       "bioconductor-rnbeads.rn5                                      189\n",
       "bioconductor-rnbeads.mm9                                      189\n",
       "bioconductor-rmir                                             189\n",
       "bioconductor-mmdiff2                                          189\n",
       "bioconductor-minimumdistance                                  189\n",
       "btrim                                                         189\n",
       "bioconductor-rseqan                                           189\n",
       "bioconductor-ppidata                                          188\n",
       "bioconductor-splicinggraphs                                   188\n",
       "bioconductor-pcpheno                                          188\n",
       "feht                                                          188\n",
       "bioconductor-meigor                                           188\n",
       "bioconductor-adacgh2                                          188\n",
       "novasplice                                                    188\n",
       "bioconductor-bandits                                          188\n",
       "genblastg                                                     188\n",
       "bgt                                                           188\n",
       "bioconductor-measurementerror.cor                             188\n",
       "bioconductor-mesh.db                                          188\n",
       "bioconductor-quartpac                                         188\n",
       "bioconductor-lapmix                                           188\n",
       "bioconductor-mirintegrator                                    188\n",
       "bioconductor-ivas                                             188\n",
       "bioconductor-depecher                                         188\n",
       "bioconductor-atsnp                                            188\n",
       "bioconductor-mirsponger                                       188\n",
       "bioconductor-sigar                                            188\n",
       "bioconductor-pathrender                                       188\n",
       "diamond_add_taxonomy                                          188\n",
       "bioconductor-slgi                                             188\n",
       "bioconductor-mirnameconverter                                 188\n",
       "bioconductor-rnaither                                         188\n",
       "bioconductor-gsri                                             188\n",
       "lcfit                                                         188\n",
       "bioconductor-lmgene                                           188\n",
       "flock                                                         187\n",
       "taxopy                                                        187\n",
       "icqsol                                                        187\n",
       "bioconductor-specl                                            187\n",
       "dehomopolymerate                                              187\n",
       "bioconductor-maskbad                                          187\n",
       "bioconductor-genegeneinter                                    187\n",
       "perl-gd-svg                                                   187\n",
       "bxtools                                                       187\n",
       "cap-mirseq                                                    187\n",
       "bioconductor-rariant                                          187\n",
       "bioconductor-adam                                             186\n",
       "dextractor                                                    186\n",
       "bioconductor-phenstat                                         186\n",
       "ebisearch                                                     186\n",
       "bioconductor-mirsm                                            186\n",
       "ctat-lncrna                                                   186\n",
       "genblasta                                                     186\n",
       "connectome-workbench                                          186\n",
       "hmmer2                                                        186\n",
       "bioconductor-imagehts                                         186\n",
       "cnvetti                                                       185\n",
       "dascrubber                                                    185\n",
       "dbgraph                                                       185\n",
       "bioconductor-cola                                             185\n",
       "bioconductor-glad                                             185\n",
       "bioconductor-interactivedisplay                               185\n",
       "bioconductor-graper                                           185\n",
       "agg                                                           185\n",
       "bioconductor-manor                                            185\n",
       "biolite                                                       185\n",
       "bioconductor-metaneighbor                                     185\n",
       "centroid_rna_package                                          185\n",
       "bioconductor-qckitfastq                                       185\n",
       "bioconductor-gnet2                                            184\n",
       "bioconductor-demixt                                           184\n",
       "macrel                                                        184\n",
       "bioconductor-oveseg                                           184\n",
       "bioconductor-sracipe                                          184\n",
       "bioconductor-sitepath                                         184\n",
       "bioconductor-snapcgh                                          184\n",
       "bioconductor-celda                                            184\n",
       "bloomfiltertrie                                               184\n",
       "bioconductor-rhisat2                                          184\n",
       "bioconductor-smad                                             184\n",
       "bioconductor-bgx                                              184\n",
       "bioconductor-structstrings                                    184\n",
       "mbg                                                           183\n",
       "bioconductor-netboost                                         183\n",
       "rpf-count-cds                                                 182\n",
       "pybiomart                                                     181\n",
       "cas-offinder                                                  179\n",
       "topas                                                         178\n",
       "medusa                                                        178\n",
       "perl-obogaf-parser                                            177\n",
       "rnabloom                                                      177\n",
       "bioconductor-txdb.mmusculus.ucsc.mm9.knowngene                177\n",
       "perl-lockfile-simple                                          176\n",
       "bioconductor-annotatr                                         175\n",
       "snver                                                         175\n",
       "ericscript                                                    175\n",
       "perl-rest-client                                              175\n",
       "r-htscluster                                                  173\n",
       "bioconductor-bsgenome.mmusculus.ucsc.mm10                     173\n",
       "mpa-server                                                    173\n",
       "helperlibs                                                    173\n",
       "perl-ensembl-genomes                                          173\n",
       "perbase                                                       173\n",
       "bioconductor-dmrseq                                           172\n",
       "das_tool                                                      172\n",
       "dlcpar                                                        172\n",
       "bioconductor-txdb.mmusculus.ucsc.mm10.knowngene               171\n",
       "perl-bio-das                                                  171\n",
       "perl-util-properties                                          171\n",
       "prinseq-plus-plus                                             170\n",
       "bax2bam                                                       169\n",
       "mark-nonconverted-reads                                       168\n",
       "sourcetracker                                                 168\n",
       "rna-cd                                                        166\n",
       "scirpy                                                        166\n",
       "bcbio-gff                                                     166\n",
       "dinamo                                                        166\n",
       "braker2                                                       165\n",
       "nemo-age                                                      165\n",
       "dkfz-bias-filter                                              165\n",
       "bioconductor-illuminahumanmethylation450kanno.i...            164\n",
       "braker                                                        163\n",
       "bambamc                                                       163\n",
       "r-r4cker                                                      162\n",
       "fast5-research                                                161\n",
       "bioconductor-biocversion                                      160\n",
       "itsx                                                          159\n",
       "gff3sort                                                      159\n",
       "libis                                                         159\n",
       "shannon                                                       159\n",
       "shapeshifter-cli                                              159\n",
       "kipoi                                                         158\n",
       "admixture                                                     156\n",
       "r-loomr                                                       155\n",
       "rsidx                                                         155\n",
       "ratt                                                          153\n",
       "perl-email-simple                                             152\n",
       "cd-hit-auxtools                                               152\n",
       "kissplice                                                     152\n",
       "fusioncatcher                                                 152\n",
       "bioconductor-bsgenome.mmusculus.ucsc.mm9                      152\n",
       "r-statvisual                                                  151\n",
       "minipolish                                                    150\n",
       "perl-metabolomics-fragment-annotation                         150\n",
       "varda2-client                                                 150\n",
       "binsanity                                                     149\n",
       "hmftools-amber                                                149\n",
       "bioconductor-hom.hs.inp.db                                    149\n",
       "hmftools-cobalt                                               149\n",
       "snakefmt                                                      147\n",
       "protmapper                                                    147\n",
       "sanitizeme                                                    146\n",
       "kyototycoon                                                   146\n",
       "shigatyper                                                    145\n",
       "fishtaco                                                      144\n",
       "confindr                                                      144\n",
       "pathracer                                                     143\n",
       "purple-bio                                                    143\n",
       "caper                                                         142\n",
       "cactus                                                        142\n",
       "pydp                                                          142\n",
       "gnu-wget                                                      141\n",
       "bioconductor-homo.sapiens                                     141\n",
       "feelnc                                                        141\n",
       "bioconductor-illuminahumanmethylation450kmanifest             140\n",
       "gotree                                                        139\n",
       "ngsderive                                                     139\n",
       "ls-gkm                                                        139\n",
       "logomaker                                                     138\n",
       "gcs-oauth2-boto-plugin                                        138\n",
       "kraken-biom                                                   137\n",
       "croo                                                          137\n",
       "gmtk                                                          137\n",
       "scater-scripts                                                137\n",
       "clustergrammer                                                136\n",
       "whatsgnu                                                      136\n",
       "stringmeup                                                    136\n",
       "igv-reports                                                   136\n",
       "bioconductor-fdb.infiniummethylation.hg19                     136\n",
       "openduck                                                      135\n",
       "emirge                                                        135\n",
       "biolib                                                        135\n",
       "libidn                                                        134\n",
       "eta                                                           134\n",
       "negative_training_sampler                                     134\n",
       "breakdancer                                                   134\n",
       "arb-bio                                                       133\n",
       "fqtools                                                       133\n",
       "pylca                                                         133\n",
       "cannoli                                                       133\n",
       "bsmap                                                         132\n",
       "fastq-tools                                                   131\n",
       "amos                                                          131\n",
       "fastq-join                                                    131\n",
       "tendo                                                         131\n",
       "bioconductor-txdb.hsapiens.ucsc.hg18.knowngene                131\n",
       "bioconductor-hgu133plus2.db                                   130\n",
       "kmergenie                                                     130\n",
       "bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene              129\n",
       "pepgenome                                                     129\n",
       "genometools                                                   129\n",
       "hypercluster                                                  128\n",
       "deap                                                          128\n",
       "blockclust                                                    128\n",
       "bioconductor-illuminahumanmethylationepicanno.i...            128\n",
       "tagore                                                        128\n",
       "perl-ms                                                       127\n",
       "bitstring                                                     127\n",
       "gatktool                                                      126\n",
       "pytaxonkit                                                    126\n",
       "fitter                                                        126\n",
       "bioconductor-txdb.celegans.ucsc.ce6.ensgene                   126\n",
       "dialign-tx                                                    126\n",
       "bioconductor-hgu133a.db                                       126\n",
       "demuxlet                                                      126\n",
       "unfazed                                                       126\n",
       "umap                                                          125\n",
       "mobidic-mpa                                                   125\n",
       "bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene                125\n",
       "perl-lyve-set                                                 125\n",
       "dwgsim                                                        125\n",
       "livekraken                                                    125\n",
       "bioconductor-metab                                            124\n",
       "hgtools                                                       124\n",
       "metalign                                                      124\n",
       "cytoscape                                                     124\n",
       "srax                                                          124\n",
       "bioconductor-illuminahumanmethylationepicmanifest             123\n",
       "fasta_ushuffle                                                122\n",
       "lz4-bin                                                       122\n",
       "gatb                                                          121\n",
       "pybda                                                         121\n",
       "crux-toolkit                                                  121\n",
       "dsk                                                           121\n",
       "genomethreader                                                121\n",
       "mimseq                                                        120\n",
       "scvis                                                         120\n",
       "gprofiler-official                                            120\n",
       "pyclone                                                       119\n",
       "compare-reads                                                 119\n",
       "gembs                                                         119\n",
       "bioconductor-customprodb                                      119\n",
       "kalign2                                                       118\n",
       "fusioncatcher-seqtk                                           118\n",
       "faststructure                                                 118\n",
       "pmmrcalculator                                                118\n",
       "blobtools                                                     117\n",
       "bioconductor-slingshot                                        117\n",
       "kmerinshort                                                   117\n",
       "graphlite                                                     117\n",
       "gvcftools                                                     117\n",
       "clair                                                         116\n",
       "bioconductor-gwastools                                        116\n",
       "intervaltree_bio                                              116\n",
       "ecmwfapi                                                      116\n",
       "fastq-pair                                                    116\n",
       "perl-cg-pipeline                                              116\n",
       "harvesttools                                                  115\n",
       "ghostx                                                        115\n",
       "bioconductor-bsgenome.ecoli.ncbi.20080805                     115\n",
       "colorbrewer                                                   115\n",
       "biobb_analysis                                                115\n",
       "gfold                                                         115\n",
       "autolog                                                       115\n",
       "chorus2                                                       114\n",
       "fastuniq                                                      114\n",
       "bioconductor-bsgenome.dmelanogaster.ucsc.dm3                  114\n",
       "hs-blastn                                                     113\n",
       "sepp-refgg138                                                 113\n",
       "libdivsufsort                                                 113\n",
       "perl-math-matrix                                              113\n",
       "gnu-getopt                                                    113\n",
       "interleafq                                                    113\n",
       "bpipe                                                         112\n",
       "embassy-phylip                                                112\n",
       "bioconductor-mspuritydata                                     112\n",
       "biobb_common                                                  112\n",
       "biobb_md                                                      112\n",
       "crb-blast                                                     112\n",
       "bioconductor-illuminahumanmethylationepicanno.i...            111\n",
       "bioconductor-scrnaseq                                         111\n",
       "sampei                                                        110\n",
       "cgview                                                        110\n",
       "bioconductor-bsgenome.hsapiens.ucsc.hg18                      110\n",
       "ymp                                                           110\n",
       "galaxy-parsec                                                 110\n",
       "bioconductor-snplocs.hsapiens.dbsnp144.grch38                 110\n",
       "bioconductor-illuminamousev2.db                               110\n",
       "bioformats                                                    110\n",
       "genomicassertions                                             110\n",
       "group_humann2_uniref_abundances_to_go                         110\n",
       "genericrepeatfinder                                           110\n",
       "isescan                                                       110\n",
       "fermi                                                         110\n",
       "r-firebrowser                                                 109\n",
       "lohhla                                                        109\n",
       "blastalign                                                    109\n",
       "footprint                                                     109\n",
       "assemblyutility                                               109\n",
       "gfinisher                                                     109\n",
       "bioconductor-elmer                                            108\n",
       "fastq-anonymous                                               108\n",
       "bioconductor-panther.db                                       108\n",
       "flux-simulator                                                108\n",
       "fsm-lite                                                      108\n",
       "gdc-client                                                    108\n",
       "bioconductor-bsgenome.celegans.ucsc.ce10                      108\n",
       "atactk                                                        108\n",
       "ga4ghmongo                                                    108\n",
       "a5-miseq                                                      108\n",
       "bioconductor-isee                                             107\n",
       "avro-cwl                                                      107\n",
       "hmmcopy                                                       107\n",
       "pretext-suite                                                 107\n",
       "tinyfasta                                                     107\n",
       "gnuplot-py                                                    107\n",
       "codonw                                                        107\n",
       "bioconductor-hgu95av2.db                                      107\n",
       "biolite-tools                                                 107\n",
       "biopet-sampleconfig                                           107\n",
       "python-goenrichment                                           107\n",
       "famsa                                                         107\n",
       "ceas                                                          107\n",
       "bioconductor-hgu133a2.db                                      107\n",
       "genform                                                       107\n",
       "illuminate                                                    106\n",
       "heinz                                                         106\n",
       "bioconductor-bsgenome.celegans.ucsc.ce6                       106\n",
       "bioconductor-fishpond                                         106\n",
       "justbackoff                                                   106\n",
       "marvel                                                        106\n",
       "kwip                                                          106\n",
       "comparative-annotation-toolkit                                105\n",
       "biopet-validatefastq                                          105\n",
       "dcmtk                                                         105\n",
       "crisper_recognition_tool                                      105\n",
       "bioconductor-pcatools                                         105\n",
       "blockbuster                                                   105\n",
       "commet                                                        105\n",
       "hichipper                                                     105\n",
       "hops                                                          105\n",
       "bioconductor-bsgenome.drerio.ucsc.danrer7                     105\n",
       "forgi                                                         105\n",
       "ibdne                                                         104\n",
       "blastbesties                                                  104\n",
       "anise_basil                                                   104\n",
       "isocor                                                        104\n",
       "bfc                                                           104\n",
       "galaxy-util                                                   104\n",
       "cistrome_beta                                                 104\n",
       "genonets                                                      104\n",
       "bioconductor-targetscan.hs.eg.db                              104\n",
       "afplot                                                        104\n",
       "lorma                                                         104\n",
       "bioconductor-tissueenrich                                     104\n",
       "bioconductor-lumihumanall.db                                  104\n",
       "bioconductor-busparse                                         104\n",
       "bamm                                                          104\n",
       "azure-cli                                                     104\n",
       "bioconductor-elmer.data                                       104\n",
       "clove                                                         104\n",
       "bwapy                                                         104\n",
       "epicseg                                                       104\n",
       "assembly-scan                                                 104\n",
       "bis-snp-utils                                                 104\n",
       "hivtrace                                                      103\n",
       "catch                                                         103\n",
       "ecoprimers                                                    103\n",
       "ecopcr                                                        103\n",
       "fusion-filter                                                 103\n",
       "chicagotools                                                  103\n",
       "galaxy-ie-helpers                                             103\n",
       "ghmm                                                          103\n",
       "halla                                                         103\n",
       "colombo                                                       103\n",
       "bioconductor-txdb.scerevisiae.ucsc.saccer3.sgdgene            103\n",
       "kronik                                                        103\n",
       "kfoots                                                        103\n",
       "ig-checkflowtypes                                             103\n",
       "raiss                                                         103\n",
       "irep                                                          103\n",
       "bioconductor-illuminahumanmethylationepicanno.i...            103\n",
       "bioconductor-lobstahs                                         103\n",
       "libssw                                                        102\n",
       "curesim                                                       102\n",
       "barrnap-python                                                102\n",
       "biobb_io                                                      102\n",
       "jvarkit-bam2wig                                               102\n",
       "gff3toembl                                                    102\n",
       "ispcr                                                         102\n",
       "fasta-splitter                                                102\n",
       "icount                                                        102\n",
       "bioconductor-metavizr                                         102\n",
       "chromosomer                                                   102\n",
       "bs-seeker2                                                    102\n",
       "bioconductor-yamss                                            102\n",
       "bioconductor-prolocgui                                        102\n",
       "bioconductor-faahko                                           102\n",
       "peaksql                                                       102\n",
       "fastqe                                                        102\n",
       "bioconductor-rmir.hs.mirna                                    101\n",
       "discovar                                                      101\n",
       "emmix                                                         101\n",
       "bioconductor-cmap                                             101\n",
       "esmre                                                         101\n",
       "taxonomy_ranks                                                101\n",
       "bayescan                                                      101\n",
       "cafe                                                          101\n",
       "egglib                                                        101\n",
       "wisestork                                                     101\n",
       "gbmunge                                                       101\n",
       "upimapi                                                       101\n",
       "fastqmetrics                                                  101\n",
       "fastutils                                                     101\n",
       "desman                                                        101\n",
       "parent-map                                                    101\n",
       "uvp                                                           101\n",
       "cistrome-ceas                                                 101\n",
       "hanselx                                                       101\n",
       "simplesam                                                     101\n",
       "bioconductor-txdb.scerevisiae.ucsc.saccer2.sgdgene            101\n",
       "lsd                                                           101\n",
       "kraken-ea                                                     101\n",
       "genometester4                                                 101\n",
       "lightning                                                     101\n",
       "biogridpy                                                     101\n",
       "kpal                                                          101\n",
       "hcluster_sg                                                   101\n",
       "bioconductor-mineica                                          101\n",
       "hubward-all                                                   101\n",
       "ibdseq                                                        101\n",
       "prophane                                                      101\n",
       "apoc                                                          100\n",
       "biopet-seqstat                                                100\n",
       "mirfix                                                        100\n",
       "bcool                                                         100\n",
       "biobb_model                                                   100\n",
       "gqt                                                           100\n",
       "gtf_to_genes                                                  100\n",
       "bioconductor-genesis                                          100\n",
       "bs_call                                                       100\n",
       "hapflk                                                        100\n",
       "hurry.filesize                                                100\n",
       "bufet                                                         100\n",
       "libbambamc                                                    100\n",
       "backspinpy                                                    100\n",
       "detonate                                                      100\n",
       "arem                                                           99\n",
       "bioconductor-enrichmentbrowser                                 99\n",
       "bamclipper                                                     99\n",
       "gottcha                                                        99\n",
       "curlywhirly                                                    99\n",
       "csb                                                            99\n",
       "erne                                                           99\n",
       "extract_genome_region                                          99\n",
       "macaron                                                        99\n",
       "bioconductor-txdb.hsapiens.ucsc.hg19.lincrnastr...             99\n",
       "easel                                                          99\n",
       "karect                                                         99\n",
       "2pg_cartesian                                                  99\n",
       "genologics                                                     99\n",
       "data_hacks                                                     99\n",
       "pylprotpredictor                                               99\n",
       "fseq                                                           99\n",
       "dfam                                                           99\n",
       "deltabs                                                        99\n",
       "latentstrainanalysis                                           99\n",
       "fastphylo                                                      98\n",
       "funcannot                                                      98\n",
       "axiome                                                         98\n",
       "consan                                                         98\n",
       "dicey                                                          98\n",
       "bioconductor-msmseda                                           98\n",
       "alder                                                          98\n",
       "bioconductor-iclusterplus                                      98\n",
       "lrzip                                                          98\n",
       "clinod                                                         98\n",
       "biskit                                                         98\n",
       "edena                                                          98\n",
       "clearcut                                                       98\n",
       "dnp-diprofile                                                  98\n",
       "forked-path                                                    98\n",
       "drax                                                           98\n",
       "edittag                                                        98\n",
       "bioconductor-synapter                                          98\n",
       "cansnper                                                       98\n",
       "imseq                                                          98\n",
       "ldhelmet                                                       97\n",
       "exparna                                                        97\n",
       "gustaf                                                         97\n",
       "koeken                                                         97\n",
       "dawg                                                           97\n",
       "damidseq_pipeline                                              97\n",
       "bcftools-gtc2vcf-plugin                                        97\n",
       "compalignp                                                     97\n",
       "bgreat                                                         97\n",
       "haploclique                                                    97\n",
       "generax                                                        97\n",
       "fwdpy                                                          97\n",
       "fgap                                                           97\n",
       "ideas                                                          97\n",
       "fastaindex                                                     97\n",
       "fsa                                                            97\n",
       "gotohscan                                                      97\n",
       "famseq                                                         97\n",
       "bioconductor-mirnatap.db                                       97\n",
       "finestructure                                                  97\n",
       "r-ontologyplot                                                 97\n",
       "bioconductor-proteomicsannotationhubdata                       97\n",
       "fermikit                                                       97\n",
       "bioconductor-rnaseqsamplesizedata                              97\n",
       "genomestrip                                                    97\n",
       "centrosome                                                     96\n",
       "estscan                                                        96\n",
       "estmapper                                                      96\n",
       "kinsimriboswitch                                               96\n",
       "dupre                                                          96\n",
       "fqtrim                                                         96\n",
       "ebfilter                                                       96\n",
       "booster                                                        96\n",
       "bicseq2-seg                                                    96\n",
       "kid                                                            96\n",
       "bioconductor-singler                                           96\n",
       "pyasr                                                          96\n",
       "intervalstats                                                  96\n",
       "amused                                                         96\n",
       "dashing                                                        96\n",
       "clipandmerge                                                   96\n",
       "inforna                                                        96\n",
       "hicbrowser                                                     96\n",
       "hiddendomains                                                  96\n",
       "cpinsim                                                        96\n",
       "disty                                                          96\n",
       "libsbml                                                        96\n",
       "ma                                                             96\n",
       "cofold                                                         96\n",
       "cortex_con                                                     96\n",
       "iclipro                                                        96\n",
       "gocr                                                           96\n",
       "hr2                                                            96\n",
       "asn2gb                                                         95\n",
       "crass                                                          95\n",
       "baitfisher                                                     95\n",
       "indelfixer                                                     95\n",
       "mace                                                           95\n",
       "lsc                                                            95\n",
       "bayestyper                                                     95\n",
       "dialign2                                                       95\n",
       "logol                                                          95\n",
       "atac                                                           95\n",
       "bellmans-gapc                                                  95\n",
       "bicseq2-norm                                                   95\n",
       "intemap                                                        95\n",
       "haploconduct                                                   95\n",
       "dnp-fourier                                                    95\n",
       "cassiopee                                                      95\n",
       "deploid                                                        95\n",
       "genomebaser                                                    95\n",
       "cocoscore                                                      95\n",
       "groopm                                                         95\n",
       "falco                                                          95\n",
       "brockman-pipeline                                              95\n",
       "bloocoo                                                        95\n",
       "frc                                                            95\n",
       "fann                                                           95\n",
       "gerp                                                           95\n",
       "ccat                                                           95\n",
       "biomaj                                                         95\n",
       "fermi-lite                                                     95\n",
       "fineradstructure                                               95\n",
       "genepender                                                     95\n",
       "merqury                                                        94\n",
       "graph2pro-var                                                  94\n",
       "bpp-popgen                                                     94\n",
       "modlamp                                                        94\n",
       "r-ddir                                                         94\n",
       "bioconductor-mapkl                                             94\n",
       "ac-diamond                                                     94\n",
       "ig-checkfcs                                                    94\n",
       "breakseq2                                                      94\n",
       "antarna                                                        94\n",
       "fcp                                                            94\n",
       "debwt                                                          94\n",
       "biodiff                                                        94\n",
       "evofold2                                                       94\n",
       "cmph                                                           94\n",
       "jvarkit-bam2svg                                                94\n",
       "bioconductor-signaturesearch                                   94\n",
       "abawaca                                                        94\n",
       "lyner                                                          94\n",
       "hilive2                                                        94\n",
       "dnp-corrprofile                                                94\n",
       "carnac-lr                                                      94\n",
       "illumina-cleanup                                               94\n",
       "genclust                                                       94\n",
       "gdc                                                            94\n",
       "bindash                                                        94\n",
       "erds                                                           94\n",
       "fasten                                                         94\n",
       "dnp-binstrings                                                 94\n",
       "batvi                                                          94\n",
       "lca                                                            94\n",
       "epydoc                                                         94\n",
       "kssd                                                           94\n",
       "calib                                                          94\n",
       "cansam                                                         94\n",
       "effectivet3                                                    94\n",
       "bioconductor-awfisher                                          94\n",
       "gwama                                                          94\n",
       "ir                                                             93\n",
       "elasticluster                                                  93\n",
       "readucks                                                       93\n",
       "fastahack                                                      93\n",
       "fastindep                                                      93\n",
       "gencore                                                        93\n",
       "jali                                                           93\n",
       "genomeconstellation                                            93\n",
       "gapfiller                                                      93\n",
       "fastsimbac                                                     93\n",
       "igor_vdj                                                       93\n",
       "fastq-count                                                    93\n",
       "fqzcomp                                                        93\n",
       "r-metaboanalyst                                                93\n",
       "igphyml                                                        93\n",
       "fastml                                                         93\n",
       "r-dimsum                                                       93\n",
       "k-slam                                                         93\n",
       "bcov                                                           93\n",
       "dig2                                                           93\n",
       "lap                                                            93\n",
       "damasker                                                       93\n",
       "bioconductor-brendadb                                          93\n",
       "brass                                                          93\n",
       "esimsa                                                         93\n",
       "encode-blacklist                                               93\n",
       "ditasic                                                        93\n",
       "bibliospec                                                     93\n",
       "denovogear                                                     93\n",
       "ctat-metagenomics                                              93\n",
       "divvier                                                        93\n",
       "dammet                                                         93\n",
       "consel                                                         93\n",
       "apt-probeset-summarize                                         93\n",
       "comparems2                                                     93\n",
       "knot                                                           93\n",
       "lightstringgraph                                               93\n",
       "lightassembler                                                 93\n",
       "amap                                                           93\n",
       "conus                                                          92\n",
       "arvados-cli                                                    92\n",
       "count_constant_sites                                           92\n",
       "collect_mgf                                                    92\n",
       "bioconductor-scgps                                             92\n",
       "bioconductor-waddr                                             92\n",
       "bioconductor-ncgtw                                             92\n",
       "trtools                                                        92\n",
       "bioconductor-flowspy                                           92\n",
       "bioconductor-sctgif                                            92\n",
       "cfm                                                            92\n",
       "bioconductor-sojourner                                         92\n",
       "bioconductor-sharedobject                                      92\n",
       "e-pcr                                                          92\n",
       "bioconductor-saigegds                                          92\n",
       "defiant                                                        92\n",
       "sherpas                                                        92\n",
       "cryfa                                                          92\n",
       "bwise                                                          92\n",
       "bioconductor-hilda                                             92\n",
       "bioconductor-iggeneusage                                       91\n",
       "python-rnacentral                                              90\n",
       "mpra-data-access-portal                                        90\n",
       "music-deconvolution                                            88\n",
       "genefuse                                                       88\n",
       "scvi-tools                                                     87\n",
       "tximport-scripts                                               87\n",
       "scvis_galaxy                                                   86\n",
       "nebulizer                                                      86\n",
       "opentargets                                                    86\n",
       "r-intlim                                                       84\n",
       "clan                                                           83\n",
       "transposonpsi                                                  82\n",
       "phylogenetics                                                  81\n",
       "seqscreen                                                      80\n",
       "perl-math-matrixreal                                           80\n",
       "riboseqc                                                       79\n",
       "singlem                                                        79\n",
       "bioconductor-plyranges                                         78\n",
       "insilicoseq                                                    78\n",
       "tbox-scan                                                      78\n",
       "r-tftargets                                                    78\n",
       "panpasco                                                       76\n",
       "quasirecomb                                                    76\n",
       "bioconductor-enhancedvolcano                                   75\n",
       "traitar                                                        75\n",
       "mtglink                                                        75\n",
       "msaconverter                                                   74\n",
       "perl-data-lock                                                 74\n",
       "targetdb                                                       74\n",
       "r-mgsz                                                         71\n",
       "transindel                                                     69\n",
       "r-singlecellnet                                                68\n",
       "oncogemini                                                     68\n",
       "pubmlst_client                                                 68\n",
       "oncopipe                                                       67\n",
       "trycycler                                                      67\n",
       "r-rtassel                                                      67\n",
       "lusstr                                                         66\n",
       "mp3treesim                                                     65\n",
       "optimir                                                        65\n",
       "biobb_structure_checking                                       65\n",
       "plasclass                                                      65\n",
       "scapp                                                          64\n",
       "svim-asm                                                       63\n",
       "pafpy                                                          63\n",
       "vcf2variants                                                   63\n",
       "bohra                                                          62\n",
       "bioconductor-tximeta                                           62\n",
       "figtree                                                        62\n",
       "wlogdate                                                       61\n",
       "chira                                                          61\n",
       "bioconductor-regionreport                                      60\n",
       "anndata2ri                                                     60\n",
       "biobb_structure_manager                                        60\n",
       "ncbitk                                                         59\n",
       "omero-py                                                       58\n",
       "r-mkmisc                                                       58\n",
       "mavis                                                          58\n",
       "pileuppy                                                       58\n",
       "popera                                                         57\n",
       "spliceai-wrapper                                               57\n",
       "rucs                                                           57\n",
       "r-txrevise                                                     57\n",
       "eval                                                           56\n",
       "rmetl                                                          56\n",
       "pgx-pipe-helper                                                56\n",
       "sepp-refsilva128                                               56\n",
       "phastaf                                                        55\n",
       "patholive                                                      55\n",
       "r-eacon                                                        54\n",
       "augur                                                          53\n",
       "ltr_retriever                                                  53\n",
       "orfquant                                                       52\n",
       "hmftools-linx                                                  52\n",
       "digestiflow-demux                                              52\n",
       "arboreto                                                       51\n",
       "bioconductor-org.mmu.eg.db                                     51\n",
       "phyphy                                                         51\n",
       "ltr_finder                                                     50\n",
       "pypgatk                                                        50\n",
       "polystest                                                      50\n",
       "haslr                                                          50\n",
       "neatms                                                         50\n",
       "haplotype-lso                                                  47\n",
       "bioconductor-structuralvariantannotation                       47\n",
       "quantiprot                                                     47\n",
       "deepbgc                                                        47\n",
       "pydpi                                                          47\n",
       "necat                                                          47\n",
       "pgdspider                                                      47\n",
       "snpgenie                                                       46\n",
       "jobtree                                                        46\n",
       "svjedi                                                         45\n",
       "luciphor2                                                      45\n",
       "perl-if                                                        44\n",
       "cmsip                                                          44\n",
       "yapc                                                           44\n",
       "influx_si                                                      43\n",
       "bioconductor-stager                                            42\n",
       "rgccacmd                                                       42\n",
       "mosca                                                          42\n",
       "scxa-plots                                                     41\n",
       "deeplc                                                         41\n",
       "bam2fasta                                                      39\n",
       "bioconductor-wrench                                            39\n",
       "r-genomictools.filehandler                                     39\n",
       "kipoiseq                                                       38\n",
       "bioconductor-ensdb.hsapiens.v86                                38\n",
       "bioconductor-org.at.tair.db                                    38\n",
       "agat                                                           38\n",
       "yamda                                                          38\n",
       "rkp                                                            38\n",
       "macse                                                          37\n",
       "bioconductor-loomexperiment                                    36\n",
       "perl-io-null                                                   36\n",
       "metaprokka                                                     36\n",
       "scran-cli                                                      36\n",
       "treecluster                                                    35\n",
       "bioconductor-mixomics                                          35\n",
       "chromosight                                                    34\n",
       "perl-yaml-pp                                                   34\n",
       "bioconductor-flowstats                                         33\n",
       "quasildr                                                       33\n",
       "geco2                                                          33\n",
       "jarvis                                                         33\n",
       "sequnwinder                                                    33\n",
       "start-asap                                                     33\n",
       "pmx_biobb                                                      33\n",
       "scagaire                                                       33\n",
       "cvbio                                                          32\n",
       "virchip                                                        32\n",
       "pygvcf2coverage                                                32\n",
       "pyrpipe                                                        32\n",
       "variantbreak                                                   32\n",
       "bioconductor-tradeseq                                          31\n",
       "minirmd                                                        31\n",
       "bioconductor-flowworkspacedata                                 31\n",
       "deeplift                                                       30\n",
       "perl-gfacs                                                     30\n",
       "bioconductor-singscore                                         29\n",
       "niemads                                                        28\n",
       "perl-bio-phylo-cipres                                          28\n",
       "bel-resources                                                  28\n",
       "nanomod                                                        28\n",
       "flair                                                          28\n",
       "conterminator                                                  28\n",
       "biobb_chemistry                                                27\n",
       "ngsep                                                          26\n",
       "bioconductor-org.xl.eg.db                                      26\n",
       "cromshell                                                      26\n",
       "opentargets-urlzsource                                         25\n",
       "ssuissero                                                      25\n",
       "biobb_pmx                                                      25\n",
       "aquila_stlfr                                                   25\n",
       "locus_processing                                               25\n",
       "kipoi-utils                                                    25\n",
       "kofamscan                                                      24\n",
       "bactopia                                                       24\n",
       "metapop                                                        23\n",
       "apollo                                                         22\n",
       "cigar                                                          22\n",
       "fcsparser                                                      22\n",
       "chromatiblock                                                  22\n",
       "bioconductor-bigmelon                                          22\n",
       "varifier                                                       22\n",
       "zga                                                            22\n",
       "opentargets-validator                                          21\n",
       "hca                                                            21\n",
       "metatree                                                       21\n",
       "pan-plaster                                                    21\n",
       "cgecore                                                        21\n",
       "emblmygff3                                                     21\n",
       "bioconductor-decontam                                          21\n",
       "chiron                                                         21\n",
       "bioconductor-mfuzz                                             21\n",
       "dca                                                            20\n",
       "bioconductor-cll                                               20\n",
       "altamisa                                                       20\n",
       "assemblytics                                                   20\n",
       "chado-tools                                                    20\n",
       "bioconductor-aucell                                            20\n",
       "bioconductor-polyester                                         20\n",
       "pydownsampler                                                  19\n",
       "phigaro                                                        19\n",
       "bioconductor-rdavidwebservice                                  19\n",
       "hint                                                           19\n",
       "bioconductor-infercnv                                          19\n",
       "bioconductor-cicero                                            19\n",
       "bioconductor-org.eck12.eg.db                                   19\n",
       "irida-uploader                                                 19\n",
       "bioconductor-msstats                                           18\n",
       "edta                                                           18\n",
       "bioconductor-widgettools                                       18\n",
       "sidr                                                           18\n",
       "bioconductor-tkwidgets                                         18\n",
       "changeo                                                        18\n",
       "biopet-vcffilter                                               17\n",
       "bioconductor-erccdashboard                                     17\n",
       "bioconductor-karyoploter                                       17\n",
       "escher                                                         17\n",
       "chexmix                                                        17\n",
       "fithic                                                         17\n",
       "dcplib                                                         16\n",
       "cameo                                                          16\n",
       "enrichm                                                        16\n",
       "jupyterngsplugin                                               16\n",
       "kleborate                                                      16\n",
       "psm_fragments                                                  16\n",
       "circle-map                                                     16\n",
       "ac                                                             16\n",
       "bioconductor-champdata                                         16\n",
       "dropletutils-scripts                                           16\n",
       "biobasehttptools                                               16\n",
       "bioconductor-org.pf.plasmo.db                                  16\n",
       "kopt                                                           16\n",
       "bioconductor-rtcga.cnv                                         15\n",
       "bioconductor-illuminadatatestfiles                             15\n",
       "bioconductor-aracne.networks                                   15\n",
       "imsindel                                                       15\n",
       "airr                                                           14\n",
       "biotdg                                                         14\n",
       "ismapper                                                       14\n",
       "bioconductor-bcellviper                                        14\n",
       "gffmunger                                                      14\n",
       "bioconductor-compcoder                                         14\n",
       "bioconductor-beta7                                             14\n",
       "coral                                                          14\n",
       "bioconductor-mus.musculus                                      14\n",
       "galaxy-tool-util                                               14\n",
       "galaxy-containers                                              14\n",
       "cutesv                                                         14\n",
       "bioconductor-ccmap                                             14\n",
       "bioconductor-airway                                            13\n",
       "crisprbact                                                     13\n",
       "cgat-daisy                                                     13\n",
       "crispr_studio                                                  13\n",
       "bioconductor-tximportdata                                      13\n",
       "cami-opal                                                      13\n",
       "cromwell-tools                                                 13\n",
       "bioconductor-qdnaseq.hg19                                      13\n",
       "barcode_splitter                                               13\n",
       "geosketch                                                      13\n",
       "bioconductor-champ                                             13\n",
       "focus                                                          13\n",
       "bioconductor-chicken.db                                        13\n",
       "bioconductor-rcistarget                                        13\n",
       "bioconductor-bridgedbr                                         13\n",
       "gnomic                                                         12\n",
       "bioconductor-org.ecsakai.eg.db                                 12\n",
       "bioconductor-anopheles.db0                                     12\n",
       "bioconductor-cytodx                                            12\n",
       "bioconductor-abseqr                                            12\n",
       "hicap                                                          12\n",
       "maaslin2                                                       12\n",
       "bioconductor-cormut                                            12\n",
       "bioconductor-goprofiles                                        12\n",
       "bioconductor-breastcancernki                                   12\n",
       "bioconductor-affymetrixdatatestfiles                           12\n",
       "bioconductor-cellscape                                         12\n",
       "bioconductor-aneufinderdata                                    12\n",
       "bioconductor-ccdata                                            12\n",
       "bioconductor-affyhgu133a2expr                                  12\n",
       "entrezpy                                                       12\n",
       "bioconductor-dart                                              12\n",
       "bioconductor-asset                                             12\n",
       "bioconductor-org.pt.eg.db                                      12\n",
       "bioconductor-singlecellexperiment-scripts                      12\n",
       "bioconductor-illuminahumanv4.db                                12\n",
       "fastq-dl                                                       12\n",
       "bioconductor-desingle                                          12\n",
       "bioconductor-caomicsv                                          12\n",
       "bioconductor-org.ag.eg.db                                      12\n",
       "ggd                                                            12\n",
       "bioconductor-consensus                                         11\n",
       "bioconductor-connectivitymap                                   11\n",
       "bioconductor-pcagopromoter.rn.rn4                              11\n",
       "bioconductor-cemitool                                          11\n",
       "blksheep                                                       11\n",
       "bioconductor-panelcn.mops                                      11\n",
       "bioconductor-chic.data                                         11\n",
       "bioconductor-ahcytobands                                       11\n",
       "bioconductor-ag.db                                             11\n",
       "bioconductor-alsace                                            11\n",
       "bioconductor-chic                                              11\n",
       "bioconductor-alternativesplicingevents.hg38                    11\n",
       "bioconductor-tfutils                                           11\n",
       "bioconductor-hugene10sttranscriptcluster.db                    11\n",
       "biopython.convert                                              11\n",
       "bioconductor-bioassayr                                         11\n",
       "bioconductor-catalyst                                          11\n",
       "bioconductor-chickencdf                                        11\n",
       "bioconductor-bigmemoryextras                                   11\n",
       "bioconductor-elbow                                             11\n",
       "bioconductor-pasillabamsubset                                  11\n",
       "goenrichment                                                   11\n",
       "kipoi_veff                                                     11\n",
       "bioconductor-crcl18                                            11\n",
       "bioconductor-arabidopsis.db0                                   11\n",
       "bioconductor-coveb                                             11\n",
       "enabrowsertools                                                11\n",
       "bioconductor-ashkenazimsonchr21                                11\n",
       "bioconductor-hellorangesdata                                   11\n",
       "bioconductor-dapar                                             11\n",
       "bioconductor-gsar                                              11\n",
       "bioconductor-basicstarrseq                                     11\n",
       "bioconductor-ccrepe                                            11\n",
       "bioconductor-bsgenome.athaliana.tair.tair9                     11\n",
       "bioconductor-clusterjudge                                      11\n",
       "bioconductor-bsgenome.vvinifera.urgi.iggp12xv2                 10\n",
       "bioconductor-hgu133b.db                                        10\n",
       "bioconductor-fabiadata                                         10\n",
       "bioconductor-chromswitch                                       10\n",
       "fargene                                                        10\n",
       "bioconductor-bronchialil13                                     10\n",
       "bioconductor-rforproteomics                                    10\n",
       "bioconductor-fci                                               10\n",
       "bioconductor-bsgenome.celegans.ucsc.ce11                       10\n",
       "bioconductor-mageckflute                                       10\n",
       "bioconductor-minfidata                                         10\n",
       "bioconductor-breastcancerunt                                   10\n",
       "fusion-report                                                  10\n",
       "bioconductor-rtcga.mirnaseq                                    10\n",
       "hmmratac                                                       10\n",
       "iucn_sim                                                       10\n",
       "bioconductor-pd.hugene.1.1.st.v1                               10\n",
       "bioconductor-assessorf                                         10\n",
       "bioconductor-hgu133plus2cdf                                    10\n",
       "bioconductor-assessorfdata                                     10\n",
       "bioconductor-affyhgu133aexpr                                   10\n",
       "bioconductor-shinytandem                                       10\n",
       "bioconductor-cobindr                                           10\n",
       "bioconductor-davidtiling                                       10\n",
       "bioconductor-artms                                             10\n",
       "bioconductor-hugene11sttranscriptcluster.db                    10\n",
       "bioconductor-dapardata                                         10\n",
       "bioconductor-anaquin                                           10\n",
       "bioconductor-illuminahumanmethylation27kanno.il...             10\n",
       "bioconductor-chipexoqualexample                                10\n",
       "bioconductor-chickenprobe                                      10\n",
       "bioconductor-abarray                                           10\n",
       "bioconductor-emdomics                                          10\n",
       "bioconductor-breastcancertransbig                              10\n",
       "bioconductor-biocancer                                         10\n",
       "bioconductor-ebseqhmm                                          10\n",
       "bioconductor-seqplots                                          10\n",
       "bioconductor-genarise                                          10\n",
       "bioconductor-rnbeads.hg38                                      10\n",
       "bioconductor-impulsede2                                        10\n",
       "bioconductor-scruff                                            10\n",
       "bioconductor-breakpointrdata                                   10\n",
       "bioconductor-brainimagerdata                                   10\n",
       "goslimmer                                                      10\n",
       "deepac                                                         10\n",
       "bioconductor-mouse4302barcodevecs                              10\n",
       "bioconductor-rtcga.pancan12                                    10\n",
       "bioconductor-a4                                                10\n",
       "bioconductor-allelicimbalance                                  10\n",
       "bioconductor-gskb                                              10\n",
       "bioconductor-confessdata                                       10\n",
       "bioconductor-mogsa                                             10\n",
       "bioconductor-affymoe4302expr                                   10\n",
       "bioconductor-bsgenome.ptroglodytes.ucsc.pantro3...             10\n",
       "bioconductor-copdsexualdimorphism.data                         10\n",
       "bioconductor-epidish                                           10\n",
       "bioconductor-frgepistasis                                      10\n",
       "bioconductor-bsgenome.hsapiens.1000genomes.hs37d5              10\n",
       "bioconductor-txdb.hsapiens.biomart.igis                        10\n",
       "bioconductor-breastcancerupp                                   10\n",
       "bioconductor-biotmledata                                       10\n",
       "bioconductor-cardinalworkflows                                 10\n",
       "bioconductor-curatedtcgadata                                   10\n",
       "amas                                                           10\n",
       "bioconductor-blma                                               9\n",
       "bioconductor-methylaid                                          9\n",
       "bioconductor-metamsdata                                         9\n",
       "bioconductor-chipseqspike                                       9\n",
       "bioconductor-tcgacrcmrna                                        9\n",
       "bioconductor-illumina450probevariants.db                        9\n",
       "bioconductor-blimatestingdata                                   9\n",
       "bioconductor-txdb.ptroglodytes.ucsc.pantro4.ref...              9\n",
       "bioconductor-immunespacer                                       9\n",
       "bioconductor-biobroom                                           9\n",
       "bioconductor-canine2.db                                         9\n",
       "bioconductor-minfidataepic                                      9\n",
       "bioconductor-drosophila2probe                                   9\n",
       "bioconductor-guitar                                             9\n",
       "bioconductor-affyhgu133plus2expr                                9\n",
       "bioconductor-bioccheck                                          9\n",
       "bioconductor-celaref                                            9\n",
       "bioconductor-vulcandata                                         9\n",
       "bioconductor-chipanalyser                                       9\n",
       "bioconductor-mcseadata                                          9\n",
       "bioconductor-mesh.pto.eg.db                                     9\n",
       "bioconductor-rcistarget.hg19.motifdbs.cisbponly...              9\n",
       "bioconductor-mcsurvdata                                         9\n",
       "bioconductor-tcgabiolinksgui.data                               9\n",
       "bioconductor-hgu133abarcodevecs                                 9\n",
       "bioconductor-barley1probe                                       9\n",
       "bioconductor-biotmle                                            9\n",
       "bioconductor-gsvadata                                           9\n",
       "bioconductor-allmll                                             9\n",
       "bioconductor-biocor                                             9\n",
       "bioconductor-celarefdata                                        9\n",
       "ena-dl                                                          9\n",
       "bioconductor-pd.mogene.1.1.st.v1                                9\n",
       "bioconductor-branchpointer                                      9\n",
       "bioconductor-chipenrich                                         9\n",
       "bioconductor-attract                                            9\n",
       "bazam                                                           9\n",
       "bioconductor-breakpointr                                        9\n",
       "bioconductor-singlecelltk                                       9\n",
       "bioconductor-single.mtec.transcriptomes                         9\n",
       "bioconductor-hicdatalymphoblast                                 9\n",
       "bioconductor-targetscoredata                                    9\n",
       "bioconductor-omicsprint                                         9\n",
       "bioconductor-pd.hugene.1.0.st.v1                                9\n",
       "bioconductor-bioccasestudies                                    9\n",
       "bioconductor-dep                                                9\n",
       "bioconductor-sevenbridges                                       9\n",
       "bioconductor-oncoscore                                          9\n",
       "bioconductor-intramirexplorer                                   9\n",
       "bioconductor-canine.db0                                         9\n",
       "bioconductor-bsgenome.dmelanogaster.ucsc.dm3.ma...              9\n",
       "bioconductor-geneplast.data.string.v91                          9\n",
       "bioconductor-genomewidesnp6crlmm                                9\n",
       "bioconductor-pcxndata                                           9\n",
       "bioconductor-bsgenome.alyrata.jgi.v1                            9\n",
       "bioconductor-bsgenome.amellifera.beebase.assembly4              9\n",
       "bioconductor-bumhmm                                             9\n",
       "bioconductor-fcbf                                               9\n",
       "centreseq                                                       9\n",
       "bioconductor-bsseqdata                                          9\n",
       "bioconductor-drosophila2.db                                     9\n",
       "bioconductor-tfarm                                              9\n",
       "bioconductor-fem                                                9\n",
       "bioconductor-phantasus                                          9\n",
       "bioconductor-agcdf                                              9\n",
       "bioconductor-ritan                                              9\n",
       "bioconductor-pd.ath1.121501                                     9\n",
       "bioconductor-affyqcreport                                       9\n",
       "bioconductor-mesh.syn.eg.db                                     9\n",
       "bioconductor-asicsdata                                          9\n",
       "bioconductor-bsgenome.tgondii.toxodb.7.0                        9\n",
       "bioconductor-bsgenome.vvinifera.urgi.iggp12xv0                  9\n",
       "bioconductor-cellity                                            9\n",
       "bioconductor-gep2pep                                            9\n",
       "bioconductor-methylmix                                          9\n",
       "bioconductor-cocitestats                                        9\n",
       "bioconductor-breastcancervdx                                    9\n",
       "bioconductor-crossmeta                                          9\n",
       "bioconductor-hgu95av2                                           9\n",
       "bioconductor-ensdb.hsapiens.v79                                 9\n",
       "bioconductor-bsubtilisprobe                                     9\n",
       "bioconductor-bsgenome.rnorvegicus.ucsc.rn5                      9\n",
       "bioconductor-bsgenome.rnorvegicus.ucsc.rn4.masked               9\n",
       "bioconductor-coseq                                              9\n",
       "bioconductor-arraymvout                                         9\n",
       "card_trick                                                      9\n",
       "bioconductor-tomatocdf                                          9\n",
       "bioconductor-bsgenome.rnorvegicus.ucsc.rn4                      9\n",
       "bioconductor-cocoa                                              9\n",
       "bioconductor-deconrnaseq                                        9\n",
       "bioconductor-flowsorted.blood.epic                              9\n",
       "bioconductor-colonca                                            9\n",
       "bioconductor-hom.dm.inp.db                                      9\n",
       "bioconductor-diggit                                             9\n",
       "bioconductor-gdsarray                                           9\n",
       "bioconductor-drosophila2cdf                                     9\n",
       "bioconductor-coverageview                                       9\n",
       "bioconductor-ensdb.mmusculus.v79                                9\n",
       "bioconductor-ensdb.rnorvegicus.v75                              9\n",
       "bioconductor-multimir                                           9\n",
       "bioconductor-celltrails                                         9\n",
       "bioconductor-copyneutralima                                     9\n",
       "bioconductor-hthgu133afrmavecs                                  8\n",
       "bioconductor-ampaffyexample                                     8\n",
       "bioconductor-ahensdbs                                           8\n",
       "bioconductor-tcgawgbsdata.hg19                                  8\n",
       "bioconductor-tcgautils                                          8\n",
       "bioconductor-derfinderdata                                      8\n",
       "bioconductor-alpinedata                                         8\n",
       "bioconductor-alternativesplicingevents.hg19                     8\n",
       "bioconductor-drawproteins                                       8\n",
       "bioconductor-dmrscan                                            8\n",
       "bioconductor-anf                                                8\n",
       "bioconductor-antiprofilesdata                                   8\n",
       "bioconductor-dominoeffect                                       8\n",
       "bioconductor-dream4                                             8\n",
       "bioconductor-asics                                              8\n",
       "bioconductor-agprobe                                            8\n",
       "bioconductor-snplocs.hsapiens.dbsnp151.grch38                   8\n",
       "bioconductor-tenxpbmcdata                                       8\n",
       "bioconductor-lumibarnes                                         8\n",
       "bioconductor-agimicrorna                                        8\n",
       "bioconductor-tfea.chip                                          8\n",
       "bioconductor-assign                                             8\n",
       "bioconductor-txdb.mmulatta.ucsc.rhemac8.refgene                 8\n",
       "bioconductor-rtcga.rppa                                         8\n",
       "bioconductor-rtcga.clinical                                     8\n",
       "bioconductor-bovine.db0                                         8\n",
       "bioconductor-bovineprobe                                        8\n",
       "bioconductor-breastcancermainz                                  8\n",
       "bioconductor-cyp450cdf                                          8\n",
       "bioconductor-cycle                                              8\n",
       "bioconductor-rnagilentdesign028282.db                           8\n",
       "bioconductor-curatedcrcdata                                     8\n",
       "bioconductor-beadarrayexampledata                               8\n",
       "bioconductor-mouse430a2frmavecs                                 8\n",
       "bioconductor-bayesknockdown                                     8\n",
       "bioconductor-affixcan                                           8\n",
       "bioconductor-odseq                                              8\n",
       "bioconductor-qubicdata                                          8\n",
       "bioconductor-ath1121501cdf                                      8\n",
       "bioconductor-bsgenome.aofficinalis.ncbi.v1                      8\n",
       "jass_preprocessing                                              8\n",
       "bioconductor-pd.clariom.d.human                                 8\n",
       "bioconductor-ctrap                                              8\n",
       "bioconductor-brgedata                                           8\n",
       "bioconductor-gatingmldata                                       8\n",
       "bioconductor-gigsea                                             8\n",
       "bioconductor-pd.clariom.s.mouse                                 8\n",
       "bioconductor-pcxn                                               8\n",
       "bioconductor-msdata                                             8\n",
       "bioconductor-pd.clariom.s.mouse.ht                              8\n",
       "bioconductor-bsgenome.athaliana.tair.04232008                   8\n",
       "bioconductor-gigseadata                                         8\n",
       "bioconductor-bsgenome.mmulatta.ucsc.rhemac2                     8\n",
       "bioconductor-curatedbreastdata                                  8\n",
       "bioconductor-gispa                                              8\n",
       "bioconductor-bsgenome.amellifera.ucsc.apimel2                   8\n",
       "bioconductor-genestructuretools                                 8\n",
       "bioconductor-genomationdata                                     8\n",
       "bioconductor-genomes                                            8\n",
       "bioconductor-genomewidesnp5crlmm                                8\n",
       "bioconductor-pathprintgeodata                                   8\n",
       "bioconductor-fantom3and4cage                                    8\n",
       "bioconductor-rattus.norvegicus                                  8\n",
       "bioconductor-cottonprobe                                        8\n",
       "bioconductor-curatedbladderdata                                 8\n",
       "bioconductor-moe430bprobe                                       8\n",
       "bioconductor-bsgenome.hsapiens.ucsc.hg19.masked                 8\n",
       "bioconductor-mapscape                                           8\n",
       "bioconductor-bsgenome.cfamiliaris.ucsc.canfam3                  8\n",
       "macaron                                                         8\n",
       "bioconductor-rdrtoolbox                                         8\n",
       "bioconductor-bsgenome.hsapiens.ucsc.hg17.masked                 8\n",
       "bioconductor-pd.clariom.s.human.ht                              8\n",
       "bioconductor-ace                                                8\n",
       "bioconductor-bsgenome.ggallus.ucsc.galgal4.masked               8\n",
       "bioconductor-wavetilingdata                                     8\n",
       "bioconductor-igc                                                8\n",
       "bioconductor-biocsklearn                                        8\n",
       "bioconductor-fission                                            8\n",
       "bioconductor-flowvs                                             8\n",
       "bioconductor-pd.rta.1.0                                         8\n",
       "bioconductor-ccl4                                               8\n",
       "bioconductor-bsgenome.sscrofa.ucsc.susscr11                     8\n",
       "bioconductor-chimphumanbraindata                                8\n",
       "bioconductor-cnvrd2                                             8\n",
       "bioconductor-bsgenome.vvinifera.urgi.iggp8x                     8\n",
       "bioconductor-flowcybar                                          8\n",
       "bioconductor-pd.rice                                            8\n",
       "bioconductor-pd.hg.u133a                                        8\n",
       "bioconductor-clariomsmousehttranscriptcluster.db                8\n",
       "bioconductor-canine2cdf                                         8\n",
       "bioconductor-cancerdata                                         8\n",
       "bioconductor-flowchic                                           8\n",
       "bioconductor-cctutorial                                         8\n",
       "bioconductor-pga                                                8\n",
       "bioconductor-celeganscdf                                        8\n",
       "bioconductor-celltree                                           8\n",
       "bioconductor-clariomsrathttranscriptcluster.db                  8\n",
       "bioconductor-bsgenome.ptroglodytes.ucsc.pantro6                 8\n",
       "bioconductor-scbn                                               8\n",
       "bioconductor-flowplots                                          8\n",
       "bioconductor-fibroeset                                          8\n",
       "bioconductor-categorycompare                                    8\n",
       "bioconductor-cbaf                                               8\n",
       "bioconductor-fgnet                                              8\n",
       "bioconductor-fitcons.ucsc.hg19                                  8\n",
       "bioconductor-compgo                                             8\n",
       "bioconductor-pd.hugene.2.0.st                                   8\n",
       "bioconductor-bsgenome.mmusculus.ucsc.mm10.masked                8\n",
       "bioconductor-gaia                                               8\n",
       "bioconductor-ceu1kg                                             8\n",
       "bioconductor-bsgenome.mmusculus.ucsc.mm8                        8\n",
       "bioconductor-confess                                            8\n",
       "bioconductor-pd.moex.1.0.st.v1                                  8\n",
       "bioconductor-ceu1kgv                                            8\n",
       "bioconductor-camthc                                             8\n",
       "bioconductor-pd.ecoli.asv2                                      8\n",
       "bioconductor-cagefightr                                         8\n",
       "bioconductor-bearscc                                            8\n",
       "bioconductor-mesh.sco.a32.eg.db                                 8\n",
       "bioconductor-pgca                                               8\n",
       "bioconductor-difflogo                                           8\n",
       "bioconductor-rtn                                                8\n",
       "bioconductor-biocpkgtools                                       8\n",
       "bioconductor-damirseq                                           8\n",
       "bioconductor-dvddata                                            8\n",
       "genome_updater                                                  8\n",
       "bioconductor-ceuhm3                                             8\n",
       "bioconductor-bifet                                              8\n",
       "bioconductor-harmandata                                         8\n",
       "bioconductor-seqcat                                             8\n",
       "bioconductor-worm.db0                                           8\n",
       "bioconductor-onlinefdr                                          8\n",
       "bioconductor-fella                                              8\n",
       "bioconductor-openprimer                                         8\n",
       "bioconductor-adaptest                                           8\n",
       "bioconductor-illuminahumanmethylation27k.db                     8\n",
       "bioconductor-celegansprobe                                      8\n",
       "bioconductor-citruscdf                                          8\n",
       "bioconductor-canine2probe                                       8\n",
       "bioconductor-chemminedrugs                                      8\n",
       "bioconductor-bsgenome.tguttata.ucsc.taegut2                     8\n",
       "bioconductor-estrogen                                           8\n",
       "bioconductor-cellmapperdata                                     8\n",
       "bioconductor-celegans.db                                        8\n",
       "bioconductor-meshr                                              8\n",
       "bioconductor-charge                                             8\n",
       "atlas-metadata-validator                                        8\n",
       "bioconductor-fastliquidassociation                              8\n",
       "bioconductor-chipexoqual                                        8\n",
       "bioconductor-clariomdhumantranscriptcluster.db                  8\n",
       "bioconductor-canine.db                                          8\n",
       "bioconductor-pd.huex.1.0.st.v2                                  8\n",
       "bioconductor-snplocs.hsapiens.dbsnp144.grch37                   7\n",
       "bioconductor-snplocs.hsapiens.dbsnp142.grch37                   7\n",
       "bioconductor-skewr                                              7\n",
       "bioconductor-icheck                                             7\n",
       "bioconductor-mesh.ssc.eg.db                                     7\n",
       "bioconductor-barley1cdf                                         7\n",
       "bioconductor-tpp                                                7\n",
       "counterr                                                        7\n",
       "biopet-fastqsplitter                                            7\n",
       "bioconductor-gaschyhs                                           7\n",
       "bioconductor-sigfeature                                         7\n",
       "bioconductor-illuminahumanmethylation27kmanifest                7\n",
       "bioconductor-beadarrayusecases                                  7\n",
       "bioconductor-harmonizedtcgadata                                 7\n",
       "bioconductor-scsr                                               7\n",
       "bioconductor-screcover                                          7\n",
       "bioconductor-human370v1ccrlmm                                   7\n",
       "bioconductor-mesh.oni.eg.db                                     7\n",
       "bioconductor-arraytv                                            7\n",
       "bioconductor-gwascat                                            7\n",
       "bioconductor-hgu133acdf                                         7\n",
       "bioconductor-sponge                                             7\n",
       "bioconductor-flowploidydata                                     7\n",
       "bioconductor-bsgenome.mfuro.ucsc.musfur1                        7\n",
       "bioconductor-gatefinder                                         7\n",
       "bioconductor-bioconcotk                                         7\n",
       "collect-columns                                                 7\n",
       "bioconductor-iterclust                                          7\n",
       "bioconductor-hmp16sdata                                         7\n",
       "bioconductor-timeseriesexperiment                               7\n",
       "bioconductor-hipathia                                           7\n",
       "bioconductor-xtrasnplocs.hsapiens.dbsnp144.grch37               7\n",
       "bioconductor-affypara                                           7\n",
       "bioconductor-alevinqc                                           7\n",
       "bioconductor-tcgabiolinksgui                                    7\n",
       "bioconductor-hicdatahumanimr90                                  7\n",
       "bioconductor-targetscan.mm.eg.db                                7\n",
       "bioconductor-hgu133afrmavecs                                    7\n",
       "bioconductor-appreci8r                                          7\n",
       "bioconductor-methyvim                                           7\n",
       "bioconductor-flowtrans                                          7\n",
       "bioconductor-diffcyt                                            7\n",
       "bioconductor-caninecdf                                          7\n",
       "bioconductor-cafe                                               7\n",
       "bioconductor-flowsorted.dlpfc.450k                              7\n",
       "bioconductor-flowmeans                                          7\n",
       "bioconductor-pd.poplar                                          7\n",
       "bioconductor-pd.mogene.1.0.st.v1                                7\n",
       "bioconductor-methyvimdata                                       7\n",
       "bioconductor-bsgenome.ptroglodytes.ucsc.pantro2...              7\n",
       "bioconductor-bsgenome.ptroglodytes.ucsc.pantro3                 7\n",
       "bioconductor-pd.mta.1.0                                         7\n",
       "bioconductor-frmaexampledata                                    7\n",
       "bioconductor-metnet                                             7\n",
       "bioconductor-pd.mogene.2.1.st                                   7\n",
       "bioconductor-cellscore                                          7\n",
       "bioconductor-fdb.ucsc.trnas                                     7\n",
       "bioconductor-ga4ghclient                                        7\n",
       "bioconductor-gaggle                                             7\n",
       "bioconductor-bsgenome.hsapiens.ucsc.hg38.masked                 7\n",
       "bioconductor-metagene                                           7\n",
       "bioconductor-phastcons7way.ucsc.hg38                            7\n",
       "bioconductor-cellbaser                                          7\n",
       "bioconductor-bsubtiliscdf                                       7\n",
       "bioconductor-flowmerge                                          7\n",
       "bftools                                                         7\n",
       "bioconductor-ccpromise                                          7\n",
       "bioconductor-zebrafishrnaseq                                    7\n",
       "bioconductor-bsgenome.scerevisiae.ucsc.saccer1                  7\n",
       "bioconductor-bsgenome.scerevisiae.ucsc.saccer2                  7\n",
       "bioconductor-pd.sugar.cane                                      7\n",
       "bioconductor-flowfitexampledata                                 7\n",
       "bioconductor-bsgenome.sscrofa.ucsc.susscr3                      7\n",
       "bioconductor-methinheritsim                                     7\n",
       "biasaway                                                        7\n",
       "bioconductor-polyphen.hsapiens.dbsnp131                         7\n",
       "bioconductor-canineprobe                                        7\n",
       "bioconductor-bsgenome.sscrofa.ucsc.susscr3.masked               7\n",
       "bioconductor-metaseqr                                           7\n",
       "bioconductor-bsgenome.mmulatta.ucsc.rhemac8                     7\n",
       "bioconductor-flowfit                                            7\n",
       "bioconductor-bsgenome.mfascicularis.ncbi.5.0                    7\n",
       "gmcloser                                                        7\n",
       "bioconductor-mesh.cfa.eg.db                                     7\n",
       "bioconductor-pd.cytogenetics.array                              7\n",
       "bioconductor-ecolileucine                                       7\n",
       "bioconductor-pd.chicken                                         7\n",
       "bioconductor-oligodata                                          7\n",
       "bioconductor-clippda                                            7\n",
       "bioconductor-eventpointer                                       7\n",
       "bioconductor-chimp.db0                                          7\n",
       "bioconductor-eatonetalchipseq                                   7\n",
       "bioconductor-pd.clariom.s.rat.ht                                7\n",
       "bioconductor-txdb.cfamiliaris.ucsc.canfam3.refgene              7\n",
       "bioconductor-pd.hg.u133a.2                                      7\n",
       "bioconductor-txdb.ptroglodytes.ucsc.pantro5.ref...              7\n",
       "bioconductor-clariomshumanhttranscriptcluster.db                7\n",
       "bioconductor-organism.dplyr                                     7\n",
       "bioconductor-clariomdhumanprobeset.db                           7\n",
       "bioconductor-citrusprobe                                        7\n",
       "bioconductor-pd.hg.focus                                        7\n",
       "bioconductor-ecolitk                                            7\n",
       "bioconductor-txdb.sscrofa.ucsc.susscr11.refgene                 7\n",
       "bioconductor-envisionquery                                      7\n",
       "bioconductor-oncomix                                            7\n",
       "bioconductor-dks                                                7\n",
       "bioconductor-mesh.cal.sc5314.eg.db                              7\n",
       "bioconductor-coexnet                                            7\n",
       "bioconductor-txdb.btaurus.ucsc.bostau8.refgene                  7\n",
       "bioconductor-affycompdata                                       7\n",
       "bioconductor-mmdiffbamsubset                                    7\n",
       "bioconductor-deltagseg                                          7\n",
       "bioconductor-mvoutdata                                          7\n",
       "bioconductor-factdesign                                         7\n",
       "bioconductor-txdb.athaliana.biomart.plantsmart28                7\n",
       "bioconductor-adme16cod.db                                       7\n",
       "bioconductor-pd.mapping250k.sty                                 7\n",
       "bioconductor-desubs                                             7\n",
       "bioconductor-ularcirc                                           7\n",
       "bioconductor-dmelsgi                                            7\n",
       "bioconductor-chipenrich.data                                    7\n",
       "bioconductor-compartmap                                         7\n",
       "bioconductor-enmix                                              7\n",
       "bioconductor-onesense                                           7\n",
       "bioconductor-desousa2013                                        7\n",
       "bioconductor-onassisjavalibs                                    7\n",
       "bioconductor-bsgenome.mmulatta.ucsc.rhemac3.masked              7\n",
       "bioconductor-lumimouseidmapping                                 7\n",
       "bioconductor-rnaseqr                                            7\n",
       "bioconductor-lymphoseq                                          7\n",
       "bioconductor-goexpress                                          7\n",
       "bioconductor-a4classif                                          7\n",
       "bioconductor-ggdata                                             7\n",
       "bioconductor-geuvpack                                           7\n",
       "bioconductor-gesper                                             7\n",
       "bioconductor-maqcsubset                                         7\n",
       "bioconductor-bovinecdf                                          7\n",
       "bioconductor-affylmgui                                          7\n",
       "bioconductor-rcpi                                               7\n",
       "bioconductor-bsgenome.dmelanogaster.ucsc.dm2                    7\n",
       "bioconductor-vtpnet                                             7\n",
       "bioconductor-randomwalkrestartmh                                7\n",
       "bioconductor-mesh.aor.db                                        7\n",
       "bioconductor-cllmethylation                                     7\n",
       "bioconductor-mesh.ath.eg.db                                     7\n",
       "bioconductor-bovine.db                                          7\n",
       "bioconductor-bsgenome.btaurus.ucsc.bostau4                      7\n",
       "bioconductor-eupathdb                                           7\n",
       "bioconductor-countsimqc                                         7\n",
       "bioconductor-mirbase.db                                         7\n",
       "bioconductor-lrbase.dre.eg.db                                   7\n",
       "bioconductor-cnorfeeder                                         7\n",
       "bioconductor-cnvgsadata                                         7\n",
       "bioconductor-epivizrchart                                       7\n",
       "bioconductor-cottoncdf                                          7\n",
       "bioconductor-dualks                                             7\n",
       "bioconductor-dsqtl                                              7\n",
       "bioconductor-cosmic.67                                          7\n",
       "bioconductor-consensusde                                        7\n",
       "bioconductor-daglogo                                            7\n",
       "bioconductor-famagg                                             7\n",
       "bioconductor-geuvadistranscriptexpr                             7\n",
       "bioconductor-sampleclassifierdata                               7\n",
       "bioconductor-samexplorer                                        7\n",
       "bioconductor-lrbase.ath.eg.db                                   7\n",
       "bioconductor-bsgenome.hsapiens.ncbi.grch38                      6\n",
       "bioconductor-rcyjs                                              6\n",
       "bioconductor-bsgenome.ggallus.ucsc.galgal5                      6\n",
       "bioconductor-bsgenome.cfamiliaris.ucsc.canfam3....              6\n",
       "bioconductor-cordon                                             6\n",
       "bioconductor-pd.tomato                                          6\n",
       "bioconductor-ttmap                                              6\n",
       "bioconductor-reqon                                              6\n",
       "bioconductor-clusterseq                                         6\n",
       "bioconductor-bsgenome.hsapiens.ucsc.hg18.masked                 6\n",
       "bioconductor-copynumberplots                                    6\n",
       "bioconductor-bsgenome.mmulatta.ucsc.rhemac2.masked              6\n",
       "bioconductor-pth2o2lipids                                       6\n",
       "bioconductor-bsgenome.gaculeatus.ucsc.gasacu1                   6\n",
       "bioconductor-bsgenome.drerio.ucsc.danrer7.masked                6\n",
       "bioconductor-bsgenome.drerio.ucsc.danrer6.masked                6\n",
       "bioconductor-pd.clariom.s.rat                                   6\n",
       "bioconductor-pd.clariom.s.human                                 6\n",
       "banner                                                          6\n",
       "bioconductor-bsgenome.drerio.ucsc.danrer5.masked                6\n",
       "bioconductor-bsgenome.osativa.msu.msu7                          6\n",
       "bioconductor-pathwaysplice                                      6\n",
       "bioconductor-ontoproc                                           6\n",
       "bioconductor-openprimerui                                       6\n",
       "bioconductor-bsgenome.drerio.ucsc.danrer10                      6\n",
       "bioconductor-mulder2012                                         6\n",
       "bioconductor-bsgenome.btaurus.ucsc.bostau6.masked               6\n",
       "bioconductor-bsgenome.cfamiliaris.ucsc.canfam2....              6\n",
       "bioconductor-restfulsedata                                      6\n",
       "bioconductor-bsgenome.mmusculus.ucsc.mm9.masked                 6\n",
       "bioconductor-rgmql                                              6\n",
       "bioconductor-clariomsrattranscriptcluster.db                    6\n",
       "bioconductor-bsgenome.celegans.ucsc.ce2                         6\n",
       "bioconductor-bsgenome.carietinum.ncbi.v1                        6\n",
       "bioconductor-clariomsmousetranscriptcluster.db                  6\n",
       "bioconductor-pasilla                                            6\n",
       "bioconductor-bsgenome.btaurus.ucsc.bostau6                      6\n",
       "calour                                                          6\n",
       "bioconductor-bsgenome.btaurus.ucsc.bostau4.masked               6\n",
       "bioconductor-txdb.mmulatta.ucsc.rhemac3.refgene                 6\n",
       "bioconductor-bsgenome.btaurus.ucsc.bostau3.masked               6\n",
       "bioconductor-clariomshumantranscriptcluster.db                  6\n",
       "bioconductor-bsgenome.amellifera.ucsc.apimel2.m...              6\n",
       "bioconductor-pd.charm.hg18.example                              6\n",
       "deepstats                                                       6\n",
       "bioconductor-bsgenome.dmelanogaster.ucsc.dm6                    6\n",
       "bioconductor-cgdv17                                             6\n",
       "bioconductor-primeviewprobe                                     6\n",
       "bioconductor-selex                                              6\n",
       "bioconductor-sdams                                              6\n",
       "bioconductor-bioseqclass                                        6\n",
       "bioconductor-pd.x.tropicalis                                    6\n",
       "bioconductor-prodata                                            6\n",
       "bioconductor-pd.medgene.1.1.st                                  6\n",
       "bioconductor-omicrexposome                                      6\n",
       "bioconductor-pd.rae230b                                         6\n",
       "bioconductor-zebrafish.db0                                      6\n",
       "bioconductor-rwikipathways                                      6\n",
       "bioconductor-chicken.db0                                        6\n",
       "bioconductor-bionetstat                                         6\n",
       "jpredapi                                                        6\n",
       "bioconductor-sbmlr                                              6\n",
       "bioconductor-scdd                                               6\n",
       "bioconductor-bigpint                                            6\n",
       "bioconductor-bsgenome.tguttata.ucsc.taegut1                     6\n",
       "bioconductor-eegc                                               6\n",
       "bioconductor-bsgenome.tguttata.ucsc.taegut1.masked              6\n",
       "bioconductor-anota2seq                                          6\n",
       "bioconductor-rcade                                              6\n",
       "bioconductor-spikein                                            6\n",
       "bioconductor-prostatecancervarambally                           6\n",
       "bioconductor-bsgenome.dmelanogaster.ucsc.dm2.ma...              6\n",
       "bioconductor-curatedadipochip                                   6\n",
       "bioconductor-bsgenome.ptroglodytes.ucsc.pantro2                 6\n",
       "bioconductor-pd.ht.hg.u133a                                     6\n",
       "bioconductor-bsgenome.hsapiens.ucsc.hg17                        6\n",
       "bioconductor-ath1121501probe                                    6\n",
       "bioconductor-seqvartools                                        6\n",
       "bioconductor-ddpcrclust                                         6\n",
       "bioconductor-pi                                                 6\n",
       "bioconductor-pd.ht.hg.u133.plus.pm                              6\n",
       "bioconductor-sevenc                                             6\n",
       "bioconductor-sift.hsapiens.dbsnp137                             6\n",
       "bioconductor-sigcheck                                           6\n",
       "bioconductor-ath1121501.db                                      6\n",
       "bioconductor-bsgenome.ptroglodytes.ucsc.pantro5                 6\n",
       "bioconductor-tabulamurisdata                                    6\n",
       "bioconductor-somaticadata                                       6\n",
       "bioconductor-plrs                                               6\n",
       "bioconductor-mcsea                                              6\n",
       "bioconductor-mtseekerdata                                       6\n",
       "bioconductor-ecolisakai.db0                                     6\n",
       "bioconductor-ipddb                                              6\n",
       "bioconductor-human610quadv1bcrlmm                               6\n",
       "bioconductor-hgu95a.db                                          6\n",
       "bioconductor-human.db0                                          6\n",
       "bioconductor-adductomicsr                                       6\n",
       "imctools                                                        6\n",
       "bioconductor-dlbcl                                              6\n",
       "bioconductor-expressionatlas                                    6\n",
       "bioconductor-hgu95dprobe                                        6\n",
       "bioconductor-hgfocus.db                                         6\n",
       "bioconductor-icetea                                             6\n",
       "bioconductor-dyebiasexamples                                    6\n",
       "bioconductor-gars                                               6\n",
       "bioconductor-flowbin                                            6\n",
       "bioconductor-fly.db0                                            6\n",
       "bioconductor-fdb.infiniummethylation.hg18                       6\n",
       "bioconductor-eisa                                               6\n",
       "bioconductor-maqcsubsetilm                                      6\n",
       "bioconductor-maser                                              6\n",
       "bioconductor-genotypeeval                                       6\n",
       "bioconductor-methylgsa                                          6\n",
       "gvcf2bed                                                        6\n",
       "bioconductor-mesh.gma.eg.db                                     6\n",
       "bioconductor-hgu133plus2barcodevecs                             6\n",
       "bioconductor-mafdb.exac.r1.0.nontcga.hs37d5                     6\n",
       "bioconductor-duoclustering2018                                  6\n",
       "bioconductor-msd16s                                             6\n",
       "bioconductor-diffloopdata                                       6\n",
       "bioconductor-experimenthubdata                                  6\n",
       "bioconductor-igvr                                               6\n",
       "biowdl-input-converter                                          6\n",
       "bioconductor-hom.dr.inp.db                                      6\n",
       "bioconductor-metid                                              6\n",
       "bioconductor-metagxovarian                                      6\n",
       "bioconductor-mpra                                               6\n",
       "bioconductor-illuminahumanv3.db                                 6\n",
       "bioconductor-facsdorit                                          6\n",
       "bioconductor-ffpeexampledata                                    6\n",
       "bioconductor-mgu74cv2.db                                        6\n",
       "bioconductor-hgu133atagprobe                                    6\n",
       "bioconductor-adductdata                                         6\n",
       "bioconductor-doscheda                                           6\n",
       "bioconductor-diggitdata                                         6\n",
       "bioconductor-mcbiclust                                          6\n",
       "bioconductor-hgu133bcdf                                         6\n",
       "bioconductor-epihet                                             6\n",
       "bioconductor-kissde                                             6\n",
       "bioconductor-gseabenchmarker                                    6\n",
       "feature_merge                                                   6\n",
       "bioconductor-mesh.sil.eg.db                                     6\n",
       "bioconductor-gqtlstats                                          6\n",
       "bioconductor-lrbase.xtr.eg.db                                   6\n",
       "bioconductor-erssa                                              6\n",
       "bioconductor-mogene20sttranscriptcluster.db                     6\n",
       "bioconductor-ecoli2cdf                                          6\n",
       "bioconductor-ecoli2.db                                          6\n",
       "bioconductor-gostag                                             6\n",
       "bioconductor-mirbaseconverter                                   6\n",
       "bioconductor-ecoli2probe                                        6\n",
       "bioconductor-mogene.1.0.st.v1frmavecs                           6\n",
       "bioconductor-mofa                                               6\n",
       "bioconductor-tnt                                                6\n",
       "bioconductor-glmsparsenet                                       6\n",
       "bioconductor-ecoliprobe                                         6\n",
       "bioconductor-mdts                                               6\n",
       "bioconductor-mafdb.esp6500si.v2.ssa137.grch38                   6\n",
       "bioconductor-moonlightr                                         6\n",
       "bioconductor-mesh.pfa.3d7.eg.db                                 6\n",
       "bioconductor-keggprofile                                        6\n",
       "bioconductor-gcmapweb                                           6\n",
       "bioconductor-vulcan                                             6\n",
       "bioconductor-lumihumanidmapping                                 6\n",
       "bioconductor-geneexpressionsignature                            6\n",
       "bioconductor-mafdb.topmed.freeze5.hg19                          6\n",
       "bioconductor-mesh.ani.fgsc.eg.db                                6\n",
       "bioconductor-foldgo                                             6\n",
       "bioconductor-gem                                                6\n",
       "bioconductor-gdcrnatools                                        6\n",
       "bioconductor-mesh.eco.k12.mg1655.eg.db                          6\n",
       "bioconductor-meal                                               6\n",
       "bioconductor-ensdb.mmusculus.v75                                6\n",
       "bioconductor-funcisnp.data                                      6\n",
       "bioconductor-gcapc                                              6\n",
       "bioconductor-furrowseg                                          6\n",
       "bioconductor-mafdb.topmed.freeze5.hg38                          6\n",
       "bioconductor-geuvstore2                                         6\n",
       "bioconductor-gcmap                                              6\n",
       "bioconductor-ivygapse                                           6\n",
       "bioconductor-comet                                              5\n",
       "bioconductor-pd.mapping250k.nsp                                 5\n",
       "bioconductor-chipseqdbdata                                      5\n",
       "jass                                                            5\n",
       "bioconductor-excluster                                          5\n",
       "bioconductor-donapllp2013                                       5\n",
       "bioconductor-etec16s                                            5\n",
       "bioconductor-micrornaome                                        5\n",
       "bioconductor-minionsummarydata                                  5\n",
       "bioconductor-omicplotr                                          5\n",
       "bioconductor-erma                                               5\n",
       "bioconductor-bsgenome.btaurus.ucsc.bostau3                      5\n",
       "bioconductor-mugaexampledata                                    5\n",
       "knock-knock                                                     5\n",
       "bioconductor-lumiratidmapping                                   5\n",
       "bioconductor-rnaseqdata.hnrnpc.bam.chr14                        5\n",
       "clinvar-tsv                                                     5\n",
       "bioconductor-lymphoseqdb                                        5\n",
       "bioconductor-m3c                                                5\n",
       "bioconductor-ritandata                                          5\n",
       "bioconductor-mafdb.exac.r1.0.grch38                             5\n",
       "bioconductor-bsgenome.btaurus.ucsc.bostau8                      5\n",
       "bioconductor-ensdb.rnorvegicus.v79                              5\n",
       "bioconductor-rgmqllib                                           5\n",
       "bioconductor-debrowser                                          5\n",
       "bioconductor-pd.ragene.1.0.st.v1                                5\n",
       "bioconductor-conumee                                            5\n",
       "ghost-tree                                                      5\n",
       "bioconductor-tronco                                             5\n",
       "bioconductor-pd.mogene.2.0.st                                   5\n",
       "bioconductor-dmchmm                                             5\n",
       "jolytree                                                        5\n",
       "bioconductor-pd.rae230a                                         5\n",
       "bioconductor-multihiccompare                                    5\n",
       "bioconductor-nbsplice                                           5\n",
       "bioconductor-onassis                                            5\n",
       "bioconductor-dmrforpairs                                        5\n",
       "legsta                                                          5\n",
       "bioconductor-omicsmarker                                        5\n",
       "bioconductor-opweight                                           5\n",
       "bioconductor-drosgenome1cdf                                     5\n",
       "bioconductor-txdb.rnorvegicus.biomart.igis                      5\n",
       "bioconductor-omadb                                              5\n",
       "bioconductor-ecoliasv2cdf                                       5\n",
       "bioconductor-ecoliasv2probe                                     5\n",
       "bioconductor-ecolicdf                                           5\n",
       "bioconductor-ecolik12.db0                                       5\n",
       "bioconductor-mofadata                                           5\n",
       "bioconductor-msmb                                               5\n",
       "bioconductor-egad                                               5\n",
       "bioconductor-msmstests                                          5\n",
       "bioconductor-drosgenome1.db                                     5\n",
       "bioconductor-bsgenome.cfamiliaris.ucsc.canfam2                  5\n",
       "bioconductor-fdb.ucsc.snp137common.hg19                         5\n",
       "bioconductor-mdp                                                5\n",
       "bioconductor-bsgenome.rnorvegicus.ucsc.rn5.masked               5\n",
       "garnett-cli                                                     5\n",
       "bioconductor-bloodcancermultiomics2017                          5\n",
       "bioconductor-stroma4                                            5\n",
       "bioconductor-bsgenome.ggallus.ucsc.galgal4                      5\n",
       "bioconductor-mbcb                                               5\n",
       "bioconductor-funcisnp                                           5\n",
       "bioconductor-projectr                                           5\n",
       "bioconductor-progeny                                            5\n",
       "bioconductor-flowtime                                           5\n",
       "bioconductor-mesh.eco.iai39.eg.db                               5\n",
       "bioconductor-flowsorted.cordblood.450k                          5\n",
       "bioconductor-bsgenome.rnorvegicus.ucsc.rn6                      5\n",
       "bioconductor-imman                                              5\n",
       "bioconductor-funtoonorm                                         5\n",
       "bioconductor-post                                               5\n",
       "bioconductor-bsgenome.mmusculus.ucsc.mm8.masked                 5\n",
       "bioconductor-bsgenome.scerevisiae.ucsc.saccer3                  5\n",
       "bioconductor-prostar                                            5\n",
       "bioconductor-flowcatchr                                         5\n",
       "bioconductor-fletcher2013a                                      5\n",
       "bioconductor-bsgenome.mmulatta.ucsc.rhemac3                     5\n",
       "bioconductor-fletcher2013b                                      5\n",
       "bioconductor-ga4ghshiny                                         5\n",
       "bioconductor-bsgenome.drerio.ucsc.danrer5                       5\n",
       "bioconductor-indeed                                             5\n",
       "bioconductor-primirtss                                          5\n",
       "bioconductor-imetagene                                          5\n",
       "bioconductor-hiergwas                                           5\n",
       "bioconductor-bsgenome.drerio.ucsc.danrer6                       5\n",
       "biobb_structure_utils                                           5\n",
       "bioconductor-geneclassifiers                                    5\n",
       "bioconductor-geneaccord                                         5\n",
       "bioconductor-bsgenome.gaculeatus.ucsc.gasacu1.m...              5\n",
       "bioconductor-bsgenome.ggallus.ucsc.galgal3                      5\n",
       "bioconductor-rqt                                                5\n",
       "bioconductor-rtcga.rnaseq                                       5\n",
       "bioconductor-bodymaprat                                         5\n",
       "bioconductor-tmixclust                                          5\n",
       "bioconductor-tinesath1cdf                                       5\n",
       "bioconductor-rtcga.mutations                                    5\n",
       "bioconductor-tcseq                                              5\n",
       "bioconductor-svm2crmdata                                        5\n",
       "bioconductor-sesame                                             5\n",
       "bioconductor-stattarget                                         5\n",
       "bioconductor-hgu133plus2frmavecs                                5\n",
       "bioconductor-bsgenome.ggallus.ucsc.galgal3.masked               5\n",
       "bioconductor-snplocs.hsapiens.dbsnp149.grch38                   5\n",
       "bioconductor-hu35ksubc.db                                       5\n",
       "bioconductor-systempiperdata                                    5\n",
       "bioconductor-hgug4112a.db                                       5\n",
       "bioconductor-hguqiagenv3.db                                     5\n",
       "bioconductor-pd.zebrafish                                       4\n",
       "bioconductor-metagenomefeatures                                 4\n",
       "bioconductor-pepsnmrdata                                        4\n",
       "bioconductor-metacyto                                           4\n",
       "bioconductor-metabomxtr                                         4\n",
       "bioconductor-perturbatr                                         4\n",
       "bioconductor-chetah                                             4\n",
       "bioconductor-rat2302cdf                                         4\n",
       "bioconductor-pd.zebgene.1.1.st                                  4\n",
       "bioconductor-flowsorted.cordbloodnorway.450k                    4\n",
       "bioconductor-vitisviniferacdf                                   4\n",
       "bioconductor-yeastgsdata                                        4\n",
       "bioconductor-pepsnmr                                            4\n",
       "bioconductor-plasmodiumanophelesprobe                           4\n",
       "bioconductor-pd.mirna.3.0                                       4\n",
       "bioconductor-pd.ragene.1.1.st.v1                                4\n",
       "bioconductor-pd.wheat                                           4\n",
       "bioconductor-pd.ragene.2.0.st                                   4\n",
       "bioconductor-pd.u133.x3p                                        4\n",
       "bioconductor-pd.rcngene.1.1.st                                  4\n",
       "bioconductor-pd.s.aureus                                        4\n",
       "bioconductor-cellbench                                          4\n",
       "bioconductor-mesh.vvi.eg.db                                     4\n",
       "bioconductor-pharmacogx                                         4\n",
       "bioconductor-cellmixs                                           4\n",
       "bioconductor-plotgrouper                                        4\n",
       "bioconductor-pd.drogene.1.0.st                                  4\n",
       "bioconductor-mitoodedata                                        4\n",
       "aquila                                                          4\n",
       "bioconductor-geneanswers                                        4\n",
       "bioconductor-yri1kgv                                            4\n",
       "bioconductor-lydata                                             4\n",
       "bioconductor-vidger                                             4\n",
       "bioconductor-pd.genomewidesnp.6                                 4\n",
       "bioconductor-mesh.spu.eg.db                                     4\n",
       "bioconductor-clumsiddata                                        4\n",
       "bioconductor-migsadata                                          4\n",
       "bioconductor-proteinprofiles                                    4\n",
       "bioconductor-clumsid                                            4\n",
       "bioconductor-mgu74cprobe                                        4\n",
       "bioconductor-mesh.nle.eg.db                                     4\n",
       "bioconductor-profileplyr                                        4\n",
       "bioconductor-mesh.lma.eg.db                                     4\n",
       "bioconductor-mesh.mdo.eg.db                                     4\n",
       "bioconductor-pd.medicago                                        4\n",
       "bioconductor-adamgui                                            4\n",
       "bioconductor-pd.mg.u74bv2                                       4\n",
       "bioconductor-yeast2probe                                        4\n",
       "bioconductor-bsgenome.mdomestica.ucsc.mondom5                   4\n",
       "array-as-vcf                                                    4\n",
       "bioconductor-rae230acdf                                         4\n",
       "bioconductor-pd.genomewidesnp.5                                 4\n",
       "bioconductor-cnvranger                                          4\n",
       "bioconductor-rae230bprobe                                       4\n",
       "bioconductor-mircompdata                                        4\n",
       "bioconductor-pd.e.coli.2                                        4\n",
       "bioconductor-pd.drosophila.2                                    4\n",
       "bioconductor-ragene11sttranscriptcluster.db                     4\n",
       "bioconductor-pd.nugo.hs1a520180                                 4\n",
       "bioconductor-mafdb.gnomad.r2.1.hs37d5                           4\n",
       "bioconductor-porcine.db                                         4\n",
       "bioconductor-rattoxfxprobe                                      4\n",
       "bioconductor-gladiatox                                          4\n",
       "bioconductor-mouse4302.db                                       4\n",
       "bioexcel_seqqc                                                  4\n",
       "bioconductor-rmassbankdata                                      4\n",
       "bioconductor-m3dexampledata                                     4\n",
       "aquila_umap                                                     4\n",
       "bioconductor-mouse4302frmavecs                                  4\n",
       "bioconductor-maxcontrastprojection                              4\n",
       "bioconductor-cytofast                                           4\n",
       "bigsi                                                           4\n",
       "bioconductor-bsgenome.cjacchus.ucsc.caljac3                     4\n",
       "bioconductor-pd.bovgene.1.0.st                                  4\n",
       "bioconductor-meebodata                                          4\n",
       "bioconductor-mafdb.1kgenomes.phase3.grch38                      4\n",
       "bioconductor-moe430acdf                                         4\n",
       "bioconductor-pd.atdschip.tiling                                 4\n",
       "bioconductor-rcellminerdata                                     4\n",
       "bioconductor-pd.ag                                              4\n",
       "bioconductor-pd.canine                                          4\n",
       "bioconductor-curatedadiporna                                    4\n",
       "bioconductor-pcagopromoter.mm.mm9                               4\n",
       "bioconductor-ratchrloc                                          4\n",
       "bioconductor-medipsdata                                         4\n",
       "bioconductor-coregflux                                          4\n",
       "bioconductor-moex10sttranscriptcluster.db                       4\n",
       "bioconductor-regparallel                                        4\n",
       "bioconductor-rheumaticconditionwollbold                         4\n",
       "bioconductor-mosaicsexample                                     4\n",
       "bioconductor-mesh.ptr.eg.db                                     4\n",
       "bioconductor-remp                                               4\n",
       "bioconductor-powerexplorer                                      4\n",
       "bioconductor-powertcr                                           4\n",
       "jasmine                                                         4\n",
       "bioconductor-mesh.pab.eg.db                                     4\n",
       "bioconductor-prebsdata                                          4\n",
       "bioconductor-pd.moe430a                                         4\n",
       "bioconductor-fdb.fantom4.promoters.hg19                         4\n",
       "bioconductor-yeastnagalakshmi                                   4\n",
       "lyner                                                           4\n",
       "bioconductor-rat2302.db                                         4\n",
       "bioconductor-pcagopromoter.hs.hg19                              4\n",
       "bioconductor-matchbox                                           4\n",
       "bioconductor-paxtoolsr                                          4\n",
       "bioconductor-rgug4131a.db                                       4\n",
       "bioconductor-maqcsubsetafx                                      4\n",
       "bioconductor-pathwaypca                                         4\n",
       "bioconductor-maqcexpression4plex                                4\n",
       "bioconductor-pathnetdata                                        4\n",
       "bioconductor-mapkldata                                          4\n",
       "bioconductor-mammaprintdata                                     4\n",
       "bioconductor-seventygenedata                                    4\n",
       "bioconductor-pasillatranscriptexpr                              4\n",
       "bioconductor-panvizgenerator                                    4\n",
       "bioconductor-paeg1aprobe                                        4\n",
       "bioconductor-rguatlas4k.db                                      4\n",
       "bioconductor-mafdb.exac.r1.0.nontcga.grch38                     4\n",
       "bioconductor-pd.rjpgene.1.0.st                                  4\n",
       "bioconductor-malaria.db0                                        4\n",
       "bioconductor-neighbornet                                        4\n",
       "bioconductor-hapmapsnp5                                         4\n",
       "bioconductor-mouse430a2.db                                      4\n",
       "bioconductor-rnaseqrdata                                        4\n",
       "bioconductor-rnu34.db                                           4\n",
       "bioconductor-tofsimsdata                                        4\n",
       "bioconductor-tissuetreg                                         4\n",
       "bioconductor-rtcga.methylation                                  4\n",
       "bioconductor-gprege                                             4\n",
       "bioconductor-lrbase.rno.eg.db                                   4\n",
       "bioconductor-mousechrloc                                        4\n",
       "bioconductor-mouse.db0                                          4\n",
       "bioconductor-mpedbarray.db                                      4\n",
       "bioconductor-grndata                                            4\n",
       "bioconductor-methvisual                                         4\n",
       "bioconductor-ruvcorr                                            4\n",
       "bioconductor-hapmapsnp6                                         4\n",
       "bioconductor-hpaanalyze                                         4\n",
       "bioconductor-hivcdnavantwout03                                  4\n",
       "bioconductor-gsbenchmark                                        4\n",
       "bioconductor-logicfs                                            4\n",
       "bioconductor-loci2path                                          4\n",
       "bioconductor-listeretalbsseq                                    4\n",
       "bioconductor-timescape                                          4\n",
       "bioconductor-olingui                                            4\n",
       "bioconductor-limmagui                                           4\n",
       "bioconductor-timerquant                                         4\n",
       "bioconductor-leukemiaseset                                      4\n",
       "bioconductor-hierinf                                            4\n",
       "bioconductor-lowmacaannotation                                  4\n",
       "bioconductor-ygs98frmavecs                                      4\n",
       "bioconductor-typeinfo                                           4\n",
       "bioconductor-h5vcdata                                           4\n",
       "bioconductor-ngscopydata                                        4\n",
       "bioconductor-jctseqdata                                         4\n",
       "bioconductor-wheatprobe                                         4\n",
       "bioconductor-dcanr                                              4\n",
       "bioconductor-gwasurvivr                                         4\n",
       "bioconductor-kinswingr                                          4\n",
       "bioconductor-scalign                                            4\n",
       "bioconductor-wes.1kg.wugsc                                      4\n",
       "bioconductor-hapmap100khind                                     4\n",
       "bioconductor-lapointe.db                                        4\n",
       "bioconductor-msgfplus                                           4\n",
       "bioconductor-ledpred                                            4\n",
       "bioconductor-sugarcaneprobe                                     4\n",
       "bioconductor-tcgamethylation450k                                4\n",
       "bioconductor-scfeaturefilter                                    4\n",
       "bioconductor-biomm                                              4\n",
       "bioconductor-hapmap100kxba                                      4\n",
       "bioconductor-sclcbam                                            4\n",
       "bioconductor-targetsearchdata                                   4\n",
       "bioconductor-italicsdata                                        4\n",
       "bioconductor-italics                                            4\n",
       "bioconductor-isocorrector                                       4\n",
       "bioconductor-hapmap500knsp                                      4\n",
       "bioconductor-scone                                              4\n",
       "bioconductor-hthgu133bprobe                                     4\n",
       "bioconductor-hapmap500ksty                                      4\n",
       "bioconductor-diffcoexp                                          4\n",
       "bioconductor-hapmap370k                                         4\n",
       "bioconductor-msstatsbiodata                                     4\n",
       "bioconductor-humanomni5quadv1bcrlmm                             4\n",
       "bioconductor-drosgenome1probe                                   4\n",
       "bioconductor-hgu133a2cdf                                        4\n",
       "bioconductor-subcellbarcode                                     4\n",
       "bioconductor-impcdata                                           4\n",
       "bioconductor-sepira                                             4\n",
       "bioconductor-seqcna.annot                                       4\n",
       "bioconductor-nanostringqcpro                                    4\n",
       "biopet-bamstats                                                 4\n",
       "bioconductor-dresscheck                                         4\n",
       "bioconductor-hcg110cdf                                          4\n",
       "bioconductor-msstatstmt                                         4\n",
       "bioconductor-hgug4100a.db                                       4\n",
       "bioconductor-serumstimulation                                   4\n",
       "bioconductor-sesamedata                                         4\n",
       "bioconductor-htrat230pmcdf                                      4\n",
       "bioconductor-mtbls2                                             4\n",
       "bioconductor-shdz.db                                            4\n",
       "bioconductor-shinymethyldata                                    4\n",
       "bioconductor-heebodata                                          4\n",
       "bioconductor-mtseeker                                           4\n",
       "bioconductor-decomptumor2sig                                    4\n",
       "bioconductor-icnv                                               4\n",
       "bioconductor-hguatlas13k.db                                     4\n",
       "bioconductor-ncigraphdata                                       4\n",
       "bioconductor-snphooddata                                        4\n",
       "bioconductor-nugohs1a520180.db                                  4\n",
       "bioconductor-snplocs.hsapiens.dbsnp141.grch38                   4\n",
       "bioconductor-hgu133a2frmavecs                                   4\n",
       "delegation                                                      4\n",
       "bioconductor-mu15v1.db                                          4\n",
       "bioconductor-human650v3acrlmm                                   4\n",
       "bioconductor-humanomniexpress12v1bcrlmm                         4\n",
       "bioconductor-swfdr                                              4\n",
       "bioconductor-hugene.1.0.st.v1frmavecs                           4\n",
       "bioconductor-humanomni25quadv1bcrlmm                            4\n",
       "bioconductor-yeastrnaseq                                        4\n",
       "bioconductor-humanomni1quadv1bcrlmm                             4\n",
       "bioconductor-hgu2beta7                                          4\n",
       "bioconductor-humancytosnp12v2p1hcrlmm                           4\n",
       "bioconductor-humanchrloc                                        4\n",
       "bioconductor-huex10stprobeset.db                                4\n",
       "bioconductor-hgu133a2probe                                      4\n",
       "bioconductor-huex.1.0.st.v2frmavecs                             4\n",
       "bioconductor-human660quadv1acrlmm                               4\n",
       "bioconductor-netsmooth                                          4\n",
       "bioconductor-sparsedossa                                        4\n",
       "bioconductor-nestlink                                           4\n",
       "bioconductor-sparsenetgls                                       4\n",
       "biopet-scatterregions                                           4\n",
       "bioconductor-human370quadv3ccrlmm                               4\n",
       "bioconductor-human1mv1ccrlmm                                    4\n",
       "bioconductor-xina                                               4\n",
       "bioconductor-human1mduov3bcrlmm                                 4\n",
       "bioconductor-human550v3bcrlmm                                   4\n",
       "bioconductor-netprior                                           4\n",
       "bioconductor-animalcules                                        4\n",
       "bioconductor-stjudem                                            4\n",
       "bioconductor-mesh.dda.3937.eg.db                                3\n",
       "bioconductor-mesh.cre.eg.db                                     3\n",
       "bioconductor-mesh.dan.eg.db                                     3\n",
       "bioconductor-mesh.der.eg.db                                     3\n",
       "bioconductor-mesh.cpo.eg.db                                     3\n",
       "bioconductor-mesh.cja.eg.db                                     3\n",
       "bioconductor-mesh.ddi.ax4.eg.db                                 3\n",
       "bioconductor-mesh.aml.eg.db                                     3\n",
       "bioconductor-mesh.dgr.eg.db                                     3\n",
       "bioconductor-mesh.dme.eg.db                                     3\n",
       "bioconductor-mesh.ana.eg.db                                     3\n",
       "bioconductor-mesh.ame.eg.db                                     3\n",
       "bold-identification                                             3\n",
       "bioconductor-mafdb.1kgenomes.phase1.grch38                      3\n",
       "bioconductor-mafdb.1kgenomes.phase3.hs37d5                      3\n",
       "bioconductor-mesh.cel.eg.db                                     3\n",
       "bioconductor-mafdb.1kgenomes.phase1.hs37d5                      3\n",
       "bioconductor-mesh.cin.eg.db                                     3\n",
       "bioconductor-iyer517                                            3\n",
       "bioconductor-mesh.cbr.eg.db                                     3\n",
       "bioconductor-mesh.bta.eg.db                                     3\n",
       "bioconductor-indac.db                                           3\n",
       "bioconductor-mafdb.exac.r1.0.hs37d5                             3\n",
       "bioconductor-inpas                                              3\n",
       "bioconductor-intad                                              3\n",
       "bioconductor-intansv                                            3\n",
       "bioconductor-isobar                                             3\n",
       "bioconductor-iteremoval                                         3\n",
       "bioconductor-jazaerimetadata.db                                 3\n",
       "bioconductor-joda                                               3\n",
       "bioconductor-keggandmetacoredzpathwaysgeo                       3\n",
       "bioconductor-kidpack                                            3\n",
       "bioconductor-lungexpression                                     3\n",
       "bioconductor-m10kcod.db                                         3\n",
       "bioconductor-m20kcod.db                                         3\n",
       "bioconductor-macrophage                                         3\n",
       "bioconductor-mesh.bfl.eg.db                                     3\n",
       "bioconductor-mesh.bsu.168.eg.db                                 3\n",
       "bioconductor-mafdb.esp6500si.v2.ssa137.hs37d5                   3\n",
       "bioconductor-hu35ksubbcdf                                       3\n",
       "bioconductor-mafdb.gnomadex.r2.1.hs37d5                         3\n",
       "bioconductor-illuminahumanv1.db                                 3\n",
       "bioconductor-htratfocusprobe                                    3\n",
       "bioconductor-htseqgenie                                         3\n",
       "bioconductor-hu35ksubacdf                                       3\n",
       "bioconductor-hu35ksuba.db                                       3\n",
       "bioconductor-hu35ksubaprobe                                     3\n",
       "bioconductor-hthgu133pluspmprobe                                3\n",
       "bioconductor-hthgu133b.db                                       3\n",
       "bioconductor-xlaevis.db                                         3\n",
       "bioconductor-htrat230pmprobe                                    3\n",
       "bioconductor-hom.mm.inp.db                                      3\n",
       "bioconductor-hom.rn.inp.db                                      3\n",
       "bioconductor-hom.sc.inp.db                                      3\n",
       "bioconductor-hpannot                                            3\n",
       "bioconductor-hs25kresogen.db                                    3\n",
       "bioconductor-hs6ug171.db                                        3\n",
       "bioconductor-illuminamousev1p1.db                               3\n",
       "bioconductor-htratfocuscdf                                      3\n",
       "bioconductor-htmg430pmprobe                                     3\n",
       "bioconductor-maizecdf                                           3\n",
       "bioconductor-illuminahumanwgdaslv3.db                           3\n",
       "bioconductor-vitisviniferaprobe                                 3\n",
       "bioconductor-maizeprobe                                         3\n",
       "hca-matrix-downloader                                           3\n",
       "bioconductor-mbpcr                                              3\n",
       "bioconductor-medicagoprobe                                      3\n",
       "bioconductor-mesh.aca.eg.db                                     3\n",
       "bioconductor-mesh.aga.pest.eg.db                                3\n",
       "bioconductor-illuminahumanv2beadid.db                           3\n",
       "bioconductor-htmg430pmcdf                                       3\n",
       "bioconductor-mesh.dpe.eg.db                                     3\n",
       "bioconductor-hthgu133pluspmcdf                                  3\n",
       "bioconductor-htmg430acdf                                        3\n",
       "bioconductor-htmg430aprobe                                      3\n",
       "bioconductor-htmg430bcdf                                        3\n",
       "bioconductor-htmg430bprobe                                      3\n",
       "bioconductor-xlaevis2probe                                      3\n",
       "bioconductor-illuminaratv1.db                                   3\n",
       "bioconductor-ribosomaldatabaseproject11.5mgdb                   3\n",
       "bioconductor-illuminamousev1.db                                 3\n",
       "bioconductor-illuminahumanwgdaslv4.db                           3\n",
       "bioconductor-hspec                                              3\n",
       "bioconductor-raexexonprobesetlocation                           3\n",
       "bioconductor-rgu34acdf                                          3\n",
       "bioconductor-rgu34a.db                                          3\n",
       "bioconductor-rgu34aprobe                                        3\n",
       "bioconductor-rgu34bcdf                                          3\n",
       "bioconductor-rgu34bprobe                                        3\n",
       "bioconductor-rgu34ccdf                                          3\n",
       "bioconductor-rgu34c.db                                          3\n",
       "bioconductor-rgu34cprobe                                        3\n",
       "bioconductor-rgug4105a.db                                       3\n",
       "bioconductor-rgug4130a.db                                       3\n",
       "bioconductor-rhesuscdf                                          3\n",
       "bioconductor-rhesus.db0                                         3\n",
       "bioconductor-rhesusprobe                                        3\n",
       "bioconductor-ri16cod.db                                         3\n",
       "bioconductor-rexposome                                          3\n",
       "bioconductor-raex10sttranscriptcluster.db                       3\n",
       "bioconductor-rae230b.db                                         3\n",
       "bioconductor-prolocdata                                         3\n",
       "bioconductor-pocrcannotation.db                                 3\n",
       "bioconductor-prostatecancerstockholm                            3\n",
       "bioconductor-prostatecancergrasso                               3\n",
       "bioconductor-prostatecancercamcap                               3\n",
       "bioconductor-poplarcdf                                          3\n",
       "bioconductor-plasfia                                            3\n",
       "bioconductor-plasmodiumanophelescdf                             3\n",
       "bioconductor-txdb.athaliana.biomart.plantsmart22                3\n",
       "bioconductor-pogos                                              3\n",
       "bioconductor-primeviewcdf                                       3\n",
       "bioconductor-twoddpcr                                           3\n",
       "bioconductor-poplarprobe                                        3\n",
       "bioconductor-props                                              3\n",
       "bioconductor-porcinecdf                                         3\n",
       "bioconductor-porcineprobe                                       3\n",
       "bioconductor-potra                                              3\n",
       "bioconductor-predasampledata                                    3\n",
       "bioconductor-restfulse                                          3\n",
       "bioconductor-trnascanimport                                     3\n",
       "bioconductor-ragene20sttranscriptcluster.db                     3\n",
       "bioconductor-lrbase.pab.eg.db                                   3\n",
       "bioconductor-lrbase.bta.eg.db                                   3\n",
       "bioconductor-lrbase.cel.eg.db                                   3\n",
       "bioconductor-lrbasedbi                                          3\n",
       "bioconductor-lrbase.dme.eg.db                                   3\n",
       "bioconductor-lrbase.gga.eg.db                                   3\n",
       "bioconductor-lrbase.hsa.eg.db                                   3\n",
       "bioconductor-lrbase.mmu.eg.db                                   3\n",
       "bioconductor-lrbase.ssc.eg.db                                   3\n",
       "bioconductor-logolas                                            3\n",
       "bioconductor-lumimouseall.db                                    3\n",
       "bioconductor-lumiratall.db                                      3\n",
       "bioconductor-lungcancerlines                                    3\n",
       "bioconductor-liebermanaidenhic2009                              3\n",
       "bioconductor-ldblock                                            3\n",
       "bioconductor-illuminahumanv2.db                                 3\n",
       "bioconductor-wheatcdf                                           3\n",
       "bioconductor-lowmaca                                            3\n",
       "bioconductor-lineagepulse                                       3\n",
       "bioconductor-ragene10stprobeset.db                              3\n",
       "bioconductor-ragene21sttranscriptcluster.db                     3\n",
       "bioconductor-ragene10sttranscriptcluster.db                     3\n",
       "bioconductor-ragene10stv1cdf                                    3\n",
       "bioconductor-ragene10stv1probe                                  3\n",
       "bioconductor-ragene11stprobeset.db                              3\n",
       "bioconductor-ragene20stprobeset.db                              3\n",
       "bioconductor-ragene21stprobeset.db                              3\n",
       "bioconductor-redseq                                             3\n",
       "bioconductor-rat2302probe                                       3\n",
       "bioconductor-leebamviews                                        3\n",
       "bioconductor-rat.db0                                            3\n",
       "bioconductor-rattoxfxcdf                                        3\n",
       "bioconductor-rcellminer                                         3\n",
       "bioconductor-tsrchitect                                         3\n",
       "bioconductor-mu19ksubccdf                                       3\n",
       "bioconductor-nugomm1a520177probe                                3\n",
       "bioconductor-mu19ksubbcdf                                       3\n",
       "bioconductor-hsagilentdesign026652.db                           3\n",
       "bioconductor-moe430a.db                                         3\n",
       "bioconductor-hspeccdf                                           3\n",
       "bioconductor-moexexonprobesetlocation                           3\n",
       "bioconductor-mouse430a2probe                                    3\n",
       "bioconductor-mouse430a2cdf                                      3\n",
       "bioconductor-mouse4302probe                                     3\n",
       "bioconductor-mouse4302cdf                                       3\n",
       "bioconductor-mogene21sttranscriptcluster.db                     3\n",
       "bioconductor-mogene21stprobeset.db                              3\n",
       "bioconductor-mogene20stprobeset.db                              3\n",
       "bioconductor-mogene11sttranscriptcluster.db                     3\n",
       "bioconductor-mogene11stprobeset.db                              3\n",
       "bioconductor-mogene10stv1probe                                  3\n",
       "bioconductor-mogene10stv1cdf                                    3\n",
       "bioconductor-mogene10sttranscriptcluster.db                     3\n",
       "bioconductor-mgu74bcdf                                          3\n",
       "bioconductor-mirna10probe                                       3\n",
       "bioconductor-mesh.eqc.eg.db                                     3\n",
       "bioconductor-mesh.hsa.eg.db                                     3\n",
       "bioconductor-mesh.zma.eg.db                                     3\n",
       "bioconductor-mousegastrulationdata                              3\n",
       "bioconductor-mpranalyze                                         3\n",
       "bioconductor-msgbsr                                             3\n",
       "bioconductor-mu11ksubaprobe                                     3\n",
       "bioconductor-mu11ksuba.db                                       3\n",
       "bioconductor-msstatsqc                                          3\n",
       "bioconductor-msstatsqcgui                                       3\n",
       "bioconductor-mta10probeset.db                                   3\n",
       "bioconductor-mta10transcriptcluster.db                          3\n",
       "bioconductor-mu11ksubacdf                                       3\n",
       "bioconductor-u133x3pcdf                                         3\n",
       "bioconductor-mogene10stprobeset.db                              3\n",
       "bioconductor-msqc1                                              3\n",
       "bioconductor-mu11ksubbcdf                                       3\n",
       "bioconductor-mu11ksubb.db                                       3\n",
       "bioconductor-mu11ksubbprobe                                     3\n",
       "bioconductor-mu19ksubacdf                                       3\n",
       "bioconductor-mu19ksuba.db                                       3\n",
       "bioconductor-u133aaofav2cdf                                     3\n",
       "bioconductor-u133x3p.db                                         3\n",
       "bioconductor-mesh.xtr.eg.db                                     3\n",
       "bioconductor-mesh.xla.eg.db                                     3\n",
       "bioconductor-mesh.tgu.eg.db                                     3\n",
       "bioconductor-mesh.eco.o157.h7.sakai.eg.db                       3\n",
       "bioconductor-venndetail                                         3\n",
       "bioconductor-mesh.eco.55989.eg.db                               3\n",
       "bioconductor-mesh.dya.eg.db                                     3\n",
       "bioconductor-mgu74a.db                                          3\n",
       "bioconductor-mgu74aprobe                                        3\n",
       "bioconductor-mgu74av2cdf                                        3\n",
       "bioconductor-mesh.dvi.eg.db                                     3\n",
       "bioconductor-mgu74av2.db                                        3\n",
       "bioconductor-mesh.oan.eg.db                                     3\n",
       "bioconductor-mgu74acdf                                          3\n",
       "bioconductor-mesh.dsi.eg.db                                     3\n",
       "bioconductor-usort                                              3\n",
       "extract_fasta_seq                                               3\n",
       "bioconductor-methylinheritance                                  3\n",
       "bioconductor-mesh.dse.eg.db                                     3\n",
       "bioconductor-methylaiddata                                      3\n",
       "bioconductor-mesh.eco.ed1a.eg.db                                3\n",
       "bioconductor-metagxbreast                                       3\n",
       "bioconductor-mesh.tgo.me49.eg.db                                3\n",
       "bioconductor-mesh.mes.eg.db                                     3\n",
       "bioconductor-mesh.gga.eg.db                                     3\n",
       "bioconductor-mesh.tbr.9274.eg.db                                3\n",
       "bioconductor-mesh.sce.s288c.eg.db                               3\n",
       "bioconductor-mesh.rno.eg.db                                     3\n",
       "bioconductor-mesh.pcr.db                                        3\n",
       "bioconductor-mesh.laf.eg.db                                     3\n",
       "bioconductor-metagxpancreas                                     3\n",
       "bioconductor-mesh.pae.pao1.eg.db                                3\n",
       "bioconductor-mesh.ocu.eg.db                                     3\n",
       "bioconductor-mesh.mga.eg.db                                     3\n",
       "bioconductor-mesh.miy.eg.db                                     3\n",
       "bioconductor-mesh.osa.eg.db                                     3\n",
       "bioconductor-mesh.mml.eg.db                                     3\n",
       "bioconductor-mesh.mmu.eg.db                                     3\n",
       "bioconductor-varianttoolsdata                                   3\n",
       "bioconductor-mesh.mtr.eg.db                                     3\n",
       "bioconductor-u133x3pprobe                                       3\n",
       "bioconductor-moex10stprobeset.db                                3\n",
       "bioconductor-hta20probeset.db                                   3\n",
       "bioconductor-mi16cod.db                                         3\n",
       "bioconductor-iasva                                              3\n",
       "bioconductor-xenopuslaeviscdf                                   3\n",
       "bioconductor-xenopus.db0                                        3\n",
       "bioconductor-illuminahumanmethylation450kprobe                  3\n",
       "bioconductor-hugene10stv1probe                                  3\n",
       "bioconductor-hugene10stv1cdf                                    3\n",
       "bioconductor-hugene10stprobeset.db                              3\n",
       "bioconductor-hu6800probe                                        3\n",
       "bioconductor-xlaevis2cdf                                        3\n",
       "bioconductor-hu35ksubdcdf                                       3\n",
       "bioconductor-hu35ksubd.db                                       3\n",
       "bioconductor-hu35ksubdprobe                                     3\n",
       "bioconductor-hu6800cdf                                          3\n",
       "bioconductor-hu6800.db                                          3\n",
       "bioconductor-hu6800subacdf                                      3\n",
       "bioconductor-huexexonprobesetlocationhg19                       3\n",
       "bioconductor-hu6800subbcdf                                      3\n",
       "bioconductor-hwgcod.db                                          3\n",
       "bioconductor-huo22.db                                           3\n",
       "bioconductor-humanstemcell                                      3\n",
       "bioconductor-hu35ksubccdf                                       3\n",
       "bioconductor-hta20transcriptcluster.db                          3\n",
       "bioconductor-hthgu133acdf                                       3\n",
       "bioconductor-hthgu133a.db                                       3\n",
       "bioconductor-hthgu133aprobe                                     3\n",
       "bioconductor-hthgu133bcdf                                       3\n",
       "bioconductor-hu35ksubb.db                                       3\n",
       "bioconductor-hu35ksubbprobe                                     3\n",
       "bioconductor-hu35ksubcprobe                                     3\n",
       "bioconductor-humanaffydata                                      3\n",
       "bioconductor-hugene11stprobeset.db                              3\n",
       "bioconductor-xhybcasneuf                                        3\n",
       "bioconductor-hugene20stprobeset.db                              3\n",
       "bioconductor-hugene20sttranscriptcluster.db                     3\n",
       "bioconductor-hugene21stprobeset.db                              3\n",
       "bioconductor-hugene21sttranscriptcluster.db                     3\n",
       "bioconductor-xenopuslaevisprobe                                 3\n",
       "bioconductor-hu6800subccdf                                      3\n",
       "bioconductor-hu6800subdcdf                                      3\n",
       "bioconductor-huex10sttranscriptcluster.db                       3\n",
       "bioconductor-mgu74bv2probe                                      3\n",
       "bioconductor-mimager                                            3\n",
       "bioconductor-mgu74av2probe                                      3\n",
       "bioconductor-mgu74c.db                                          3\n",
       "bioconductor-mgu74b.db                                          3\n",
       "bioconductor-mgu74bprobe                                        3\n",
       "bioconductor-mgu74bv2cdf                                        3\n",
       "bioconductor-mgu74bv2.db                                        3\n",
       "bioconductor-mgu74ccdf                                          3\n",
       "bioconductor-moe430b.db                                         3\n",
       "bioconductor-mgu74cv2cdf                                        3\n",
       "bioconductor-migsa                                              3\n",
       "bioconductor-mgu74cv2probe                                      3\n",
       "bioconductor-mguatlas5k.db                                      3\n",
       "bioconductor-mgug4104a.db                                       3\n",
       "bioconductor-mgug4120a.db                                       3\n",
       "bioconductor-mgug4121a.db                                       3\n",
       "bioconductor-mircomp                                            3\n",
       "bioconductor-moe430bcdf                                         3\n",
       "bioconductor-huexexonprobesetlocation                           3\n",
       "bioconductor-mirna10cdf                                         3\n",
       "bioconductor-huexexonprobesetlocationhg18                       3\n",
       "bioconductor-mesh.dmo.eg.db                                     3\n",
       "bioconductor-medicagocdf                                        3\n",
       "bioconductor-mesh.dre.eg.db                                     3\n",
       "bioconductor-mm24kresogen.db                                    3\n",
       "bioconductor-mirmine                                            3\n",
       "bioconductor-mirna102xgaincdf                                   3\n",
       "bioconductor-mirna20cdf                                         3\n",
       "bioconductor-moe430aprobe                                       3\n",
       "bioconductor-mirnatarget                                        3\n",
       "bioconductor-missrows                                           3\n",
       "bioconductor-mmagilentdesign026655.db                           3\n",
       "bioconductor-mira                                               3\n",
       "bioconductor-mmpalatemirna                                      3\n",
       "bioconductor-moda                                               3\n",
       "bioconductor-rae230aprobe                                       3\n",
       "bioconductor-rae230bcdf                                         3\n",
       "bioconductor-rgu34b.db                                          3\n",
       "bioconductor-rae230a.db                                         3\n",
       "bioconductor-pd.mu11ksubb                                       3\n",
       "bioconductor-pd.mapping50k.xba240                               3\n",
       "bioconductor-pd.hc.g110                                         3\n",
       "bioconductor-pd.mapping50k.hind240                              3\n",
       "bioconductor-pd.hg.u133a.tag                                    3\n",
       "bioconductor-pd.hg.u133b                                        3\n",
       "bioconductor-pd.hg.u133.plus.2                                  3\n",
       "bioconductor-pd.hg.u219                                         3\n",
       "bioconductor-pd.hg.u95a                                         3\n",
       "bioconductor-pd.hg.u95av2                                       3\n",
       "bioconductor-pd.hg.u95b                                         3\n",
       "bioconductor-pd.hg.u95c                                         3\n",
       "bioconductor-pd.hg.u95d                                         3\n",
       "bioconductor-pd.hg.u95e                                         3\n",
       "bioconductor-pd.hta.2.0                                         3\n",
       "bioconductor-pd.ht.mg.430a                                      3\n",
       "bioconductor-pd.hu6800                                          3\n",
       "bioconductor-pd.hugene.2.1.st                                   3\n",
       "bioconductor-txdb.athaliana.biomart.plantsmart25                3\n",
       "bioconductor-pd.mu11ksuba                                       3\n",
       "bioconductor-pd.mouse430a.2                                     3\n",
       "bioconductor-pd.mg.u74av2                                       3\n",
       "bioconductor-hmyrib36                                           3\n",
       "bioconductor-mesh.eco.umn026.eg.db                              3\n",
       "bioconductor-hom.at.inp.db                                      3\n",
       "bioconductor-pd.margene.1.0.st                                  3\n",
       "bioconductor-pd.margene.1.1.st                                  3\n",
       "bioconductor-pd.medgene.1.0.st                                  3\n",
       "bioconductor-pd.mg.u74a                                         3\n",
       "bioconductor-pd.mg.u74b                                         3\n",
       "bioconductor-pd.mouse430.2                                      3\n",
       "bioconductor-pd.mg.u74c                                         3\n",
       "bioconductor-pd.mg.u74cv2                                       3\n",
       "bioconductor-pd.mirna.1.0                                       3\n",
       "bioconductor-pd.mirna.2.0                                       3\n",
       "bioconductor-pd.mirna.3.1                                       3\n",
       "bioconductor-pd.mirna.4.0                                       3\n",
       "bioconductor-pd.moe430b                                         3\n",
       "bioconductor-pd.maize                                           3\n",
       "bioconductor-pd.nugo.mm1a520177                                 3\n",
       "bioconductor-nugohs1a520180probe                                3\n",
       "bioconductor-pd.ovigene.1.0.st                                  3\n",
       "bioconductor-pd.pae.g1a                                         3\n",
       "bioconductor-pd.plasmodium.anopheles                            3\n",
       "bioconductor-pd.porcine                                         3\n",
       "bioconductor-pd.porgene.1.0.st                                  3\n",
       "bioconductor-pd.porgene.1.1.st                                  3\n",
       "bioconductor-pd.rabgene.1.0.st                                  3\n",
       "bioconductor-pd.raex.1.0.st.v1                                  3\n",
       "bioconductor-pd.rhegene.1.0.st                                  3\n",
       "bioconductor-pd.ragene.2.1.st                                   3\n",
       "bioconductor-pd.rat230.2                                        3\n",
       "bioconductor-pd.rcngene.1.0.st                                  3\n",
       "bioconductor-pd.rg.u34a                                         3\n",
       "bioconductor-pd.rg.u34b                                         3\n",
       "bioconductor-pd.rg.u34c                                         3\n",
       "bioconductor-pd.hg18.60mer.expr                                 3\n",
       "bioconductor-pd.guigene.1.1.st                                  3\n",
       "bioconductor-mu19ksubb.db                                       3\n",
       "bioconductor-pd.rabgene.1.1.st                                  3\n",
       "bioconductor-pd.rhegene.1.1.st                                  3\n",
       "bioconductor-pd.rhesus                                          3\n",
       "bioconductor-pd.vitis.vinifera                                  3\n",
       "bioconductor-pd.ovigene.1.1.st                                  3\n",
       "bioconductor-pd.rn.u34                                          3\n",
       "bioconductor-pd.rusgene.1.0.st                                  3\n",
       "bioconductor-pd.rusgene.1.1.st                                  3\n",
       "bioconductor-pd.soybean                                         3\n",
       "bioconductor-pd.soygene.1.0.st                                  3\n",
       "bioconductor-pd.soygene.1.1.st                                  3\n",
       "bioconductor-pd.xenopus.laevis                                  3\n",
       "bioconductor-pd.rjpgene.1.1.st                                  3\n",
       "bioconductor-pd.x.laevis.2                                      3\n",
       "bioconductor-pd.yeast.2                                         3\n",
       "bioconductor-pd.yg.s98                                          3\n",
       "bioconductor-pd.zebgene.1.0.st                                  3\n",
       "bioconductor-pedbarrayv10.db                                    3\n",
       "bioconductor-pedbarrayv9.db                                     3\n",
       "bioconductor-pepdat                                             3\n",
       "bioconductor-ygs98probe                                         3\n",
       "bioconductor-hgu133plus2cellscore                               3\n",
       "bioconductor-hdcytodata                                         3\n",
       "bioconductor-hgu133plus2probe                                   3\n",
       "bioconductor-gp53cdf                                            3\n",
       "bioconductor-xtropicalisprobe                                   3\n",
       "bioconductor-ygs98cdf                                           3\n",
       "bioconductor-hmdbquery                                          3\n",
       "bioconductor-ye6100subacdf                                      3\n",
       "bioconductor-hivprtplus2cdf                                     3\n",
       "jvarkit-msa2vcf                                                 3\n",
       "bioconductor-hilbertcurve                                       3\n",
       "bioconductor-hi16cod.db                                         3\n",
       "bioconductor-hiiragi2013                                        3\n",
       "bioconductor-hcadata                                            3\n",
       "bioconductor-gqtlbase                                           3\n",
       "bioconductor-xtrasnplocs.hsapiens.dbsnp144.grch38               3\n",
       "bioconductor-hgu95ccdf                                          3\n",
       "bioconductor-grasp2db                                           3\n",
       "bioconductor-hicbricks                                          3\n",
       "bioconductor-hgu95c.db                                          3\n",
       "bioconductor-hireadsprocessor                                   3\n",
       "bioconductor-greengenes13.5mgdb                                 3\n",
       "bioconductor-h20kcod.db                                         3\n",
       "bioconductor-biscuiteer                                         3\n",
       "bioconductor-grridge                                            3\n",
       "bioconductor-greylistchip                                       3\n",
       "bioconductor-hgug4111a.db                                       3\n",
       "bioconductor-hgug4110b.db                                       3\n",
       "bioconductor-hgu95cprobe                                        3\n",
       "bioconductor-hgug4101a.db                                       3\n",
       "bioconductor-hgu95dcdf                                          3\n",
       "bioconductor-hgu95eprobe                                        3\n",
       "bioconductor-hgudkfz31.db                                       3\n",
       "bioconductor-hcg110.db                                          3\n",
       "bioconductor-hgubeta7.db                                        3\n",
       "bioconductor-hgu95d.db                                          3\n",
       "bioconductor-hcg110probe                                        3\n",
       "bioconductor-hgug4845a.db                                       3\n",
       "bioconductor-hgu95ecdf                                          3\n",
       "bioconductor-hgu95e.db                                          3\n",
       "bioconductor-hgu95av2cdf                                        3\n",
       "bioconductor-xtrasnplocs.hsapiens.dbsnp141.grch38               3\n",
       "bioconductor-hgu95bprobe                                        3\n",
       "bioconductor-hgu95aprobe                                        3\n",
       "biobb_wf_mutations                                              3\n",
       "bioconductor-zebrafish.db                                       3\n",
       "bioconductor-hgu95b.db                                          3\n",
       "bioconductor-hgu95bcdf                                          3\n",
       "bioconductor-hgu95av2probe                                      3\n",
       "bioconductor-xtropicaliscdf                                     3\n",
       "bioconductor-gghumanmethcancerpanelv1.db                        3\n",
       "bioconductor-zebrafishprobe                                     3\n",
       "bioconductor-hgu133bprobe                                       3\n",
       "bioconductor-hgu95acdf                                          3\n",
       "bioconductor-hgu219probe                                        3\n",
       "bioconductor-harbchip                                           3\n",
       "bioconductor-r10kcod.db                                         3\n",
       "bioconductor-yeast2.db                                          3\n",
       "bioconductor-hgu219cdf                                          3\n",
       "bioconductor-hd2013sgi                                          3\n",
       "bioconductor-gse62944                                           3\n",
       "bioconductor-hgfocusprobe                                       3\n",
       "bioconductor-fdb.ucsc.snp135common.hg19                         3\n",
       "bioconductor-yeast2cdf                                          3\n",
       "bioconductor-yeast.db0                                          3\n",
       "bioconductor-gsean                                              3\n",
       "bioconductor-hgu133atagcdf                                      3\n",
       "bioconductor-hgu133aprobe                                       3\n",
       "biopet-seattleseqkit                                            3\n",
       "bioconductor-healthyflowdata                                    3\n",
       "bioconductor-ye6100subbcdf                                      3\n",
       "bioconductor-ye6100subdcdf                                      3\n",
       "bioconductor-h10kcod.db                                         3\n",
       "bioconductor-ygs98.db                                           3\n",
       "bioconductor-ye6100subccdf                                      3\n",
       "bioconductor-zebrafishcdf                                       3\n",
       "bioconductor-gwasdata                                           3\n",
       "bioconductor-explorase                                          3\n",
       "bioconductor-yrimulti                                           3\n",
       "bioconductor-hgfocuscdf                                         3\n",
       "bioconductor-norway981.db                                       3\n",
       "bioconductor-hgu219.db                                          3\n",
       "bioconductor-nugomm1a520177cdf                                  3\n",
       "bioconductor-sconify                                            3\n",
       "bioconductor-rta10transcriptcluster.db                          3\n",
       "bioconductor-trnadbimport                                       3\n",
       "bioconductor-rtnduals                                           3\n",
       "bioconductor-rtnsurvival                                        3\n",
       "bioconductor-siamcat                                            3\n",
       "bioconductor-sift.hsapiens.dbsnp132                             3\n",
       "bioconductor-signet                                             3\n",
       "danpos                                                          3\n",
       "bioconductor-sugarcanecdf                                       3\n",
       "bioconductor-summarizedbenchmark                                3\n",
       "bioconductor-tomatoprobe                                        3\n",
       "bioconductor-synapterdata                                       3\n",
       "bioconductor-tbx20bamsubset                                     3\n",
       "bioconductor-tcgacrcmirna                                       3\n",
       "bioconductor-tcgaworkflowdata                                   3\n",
       "bioconductor-tenxbraindata                                      3\n",
       "bioconductor-tenxplore                                          3\n",
       "bioconductor-rta10probeset.db                                   3\n",
       "bioconductor-rrdpdata                                           3\n",
       "bioconductor-roberts2005annotation.db                           3\n",
       "bioconductor-rtu34.db                                           3\n",
       "bioconductor-trendy                                             3\n",
       "bioconductor-seqc                                               3\n",
       "bioconductor-seqcombo                                           3\n",
       "bioconductor-seqsetvis                                          3\n",
       "bioconductor-seqsqc                                             3\n",
       "bioconductor-trena                                              3\n",
       "bioconductor-rvs                                                3\n",
       "bioconductor-pigengene                                          3\n",
       "bioconductor-rnu34probe                                         3\n",
       "bioconductor-rtu34cdf                                           3\n",
       "bioconductor-rmagpie                                            3\n",
       "bioconductor-rmelting                                           3\n",
       "bioconductor-rmir.hsa                                           3\n",
       "bioconductor-rnainteractmapk                                    3\n",
       "gbsx                                                            3\n",
       "bioconductor-rnu34cdf                                           3\n",
       "bioconductor-test1cdf                                           3\n",
       "bioconductor-test2cdf                                           3\n",
       "bioconductor-test3cdf                                           3\n",
       "bioconductor-pumadata                                           3\n",
       "bioconductor-river                                              3\n",
       "bioconductor-riceprobe                                          3\n",
       "bioconductor-ricecdf                                            3\n",
       "bioconductor-raex10stprobeset.db                                3\n",
       "bioconductor-prostatecancertaylor                               3\n",
       "bioconductor-tweedeseqcountdata                                 3\n",
       "bioconductor-proteomm                                           3\n",
       "bioconductor-pwmenrich.dmelanogaster.background                 3\n",
       "bioconductor-sparsesignatures                                   3\n",
       "bioconductor-pwmenrich.hsapiens.background                      3\n",
       "bioconductor-pwmenrich.mmusculus.background                     3\n",
       "bioconductor-qplexanalyzer                                      3\n",
       "bioconductor-qplexdata                                          3\n",
       "bioconductor-qsmooth                                            3\n",
       "bioconductor-qsutils                                            3\n",
       "bioconductor-nugomm1a520177.db                                  3\n",
       "bioconductor-spikeinsubset                                      3\n",
       "bioconductor-soybeanprobe                                       3\n",
       "bioconductor-test3probe                                         3\n",
       "bioconductor-silva128.1mgdb                                     3\n",
       "bioconductor-tfhaz                                              3\n",
       "bioconductor-tinesath1probe                                     3\n",
       "bioconductor-topdownr                                           3\n",
       "bioconductor-strandcheckr                                       3\n",
       "bioconductor-topdownrdata                                       3\n",
       "bioconductor-snplocs.hsapiens.dbsnp150.grch38                   3\n",
       "bioconductor-sigsquared                                         3\n",
       "flextaxd                                                        3\n",
       "bioconductor-soybeancdf                                         3\n",
       "bioconductor-transcriptogramer                                  3\n",
       "bioconductor-slinky                                             3\n",
       "bioconductor-snadata                                            3\n",
       "bioconductor-snplocs.hsapiens.dbsnp.20101109                    3\n",
       "bioconductor-srgnet                                             3\n",
       "bioconductor-snplocs.hsapiens.dbsnp.20120608                    3\n",
       "bioconductor-somaticcanceralterations                           3\n",
       "bioconductor-scoreinvhap                                        3\n",
       "bioconductor-rrbsdata                                           3\n",
       "bioconductor-scmeth                                             3\n",
       "bioconductor-pd.ecoli                                           3\n",
       "bioconductor-mu6500subccdf                                      3\n",
       "bioconductor-mu6500subdcdf                                      3\n",
       "bioconductor-muscat                                             3\n",
       "bioconductor-mwgcod.db                                          3\n",
       "bioconductor-nanotator                                          3\n",
       "bioconductor-txreginfra                                         3\n",
       "bioconductor-netsam                                             3\n",
       "bioconductor-neve2006                                           3\n",
       "bioconductor-txdb.sscrofa.ucsc.susscr3.refgene                  3\n",
       "bioconductor-ngsreports                                         3\n",
       "bioconductor-normalize450k                                      3\n",
       "bioconductor-pd.2006.07.18.mm8.refseq.promoter                  3\n",
       "bioconductor-pd.2006.10.31.rn34.refseq.promoter                 3\n",
       "bioconductor-pd.aragene.1.0.st                                  3\n",
       "bioconductor-pd.aragene.1.1.st                                  3\n",
       "bioconductor-pd.cyrgene.1.1.st                                  3\n",
       "bioconductor-pd.drogene.1.1.st                                  3\n",
       "bioconductor-mu6500subbcdf                                      3\n",
       "bioconductor-trna                                               3\n",
       "bioconductor-mu22v3.db                                          3\n",
       "bioconductor-parathyroidse                                      3\n",
       "bioconductor-hom.ce.inp.db                                      3\n",
       "bioconductor-ocplus                                             3\n",
       "bioconductor-oct4                                               3\n",
       "bioconductor-omicspcadata                                       3\n",
       "bioconductor-operonhumanv3.db                                   3\n",
       "bioconductor-paeg1acdf                                          3\n",
       "bioconductor-pairadise                                          3\n",
       "bioconductor-parglms                                            3\n",
       "bioconductor-mu19ksubc.db                                       3\n",
       "bioconductor-partheenmetadata.db                                3\n",
       "bioconductor-pd.081229.hg18.promoter.medip.hx1                  3\n",
       "bioconductor-pd.2006.07.18.hg18.refseq.promoter                 3\n",
       "bioconductor-nugohs1a520180cdf                                  3\n",
       "bioconductor-normqpcr                                           3\n",
       "bioconductor-pd.guigene.1.0.st                                  3\n",
       "bioconductor-normalyzerde                                       3\n",
       "bioconductor-pd.drosgenome1                                     3\n",
       "bioconductor-mu6500subacdf                                      3\n",
       "bioconductor-pd.elegene.1.0.st                                  3\n",
       "bioconductor-phenotest                                          3\n",
       "bioconductor-txdb.ggallus.ucsc.galgal5.refgene                  3\n",
       "bioconductor-pd.chigene.1.0.st                                  3\n",
       "bioconductor-pd.chigene.1.1.st                                  3\n",
       "bioconductor-pd.chogene.2.0.st                                  3\n",
       "bioconductor-pd.chogene.2.1.st                                  3\n",
       "bioconductor-pd.citrus                                          3\n",
       "bioconductor-phastcons100way.ucsc.hg38                          3\n",
       "bioconductor-pig.db0                                            3\n",
       "bioconductor-pd.cangene.1.1.st                                  3\n",
       "bioconductor-rtu34probe                                         3\n",
       "bioconductor-rwgcod.db                                          3\n",
       "bioconductor-sampleclassifier                                   3\n",
       "bioconductor-saureuscdf                                         3\n",
       "bioconductor-saureusprobe                                       3\n",
       "bioconductor-pd.elegene.1.1.st                                  3\n",
       "bioconductor-scale4c                                            3\n",
       "bioconductor-pd.celegans                                        3\n",
       "bioconductor-pd.cotton                                          3\n",
       "bioconductor-pd.felgene.1.1.st                                  3\n",
       "bioconductor-pd.cyrgene.1.0.st                                  3\n",
       "bioconductor-pd.equgene.1.1.st                                  3\n",
       "bioconductor-pd.cangene.1.0.st                                  3\n",
       "bioconductor-pd.feinberg.hg18.me.hx1                            3\n",
       "bioconductor-pd.feinberg.mm8.me.hx1                             3\n",
       "bioconductor-pd.felgene.1.0.st                                  3\n",
       "bioconductor-pd.fingene.1.0.st                                  3\n",
       "bioconductor-txdb.ggallus.ucsc.galgal4.refgene                  3\n",
       "bioconductor-pd.fingene.1.1.st                                  3\n",
       "bioconductor-pd.equgene.1.0.st                                  3\n",
       "bioconductor-pd.cyngene.1.1.st                                  3\n",
       "bioconductor-pd.cyngene.1.0.st                                  3\n",
       "bioconductor-pd.canine.2                                        3\n",
       "bioconductor-pd.barley1                                         3\n",
       "bioconductor-pd.bovgene.1.1.st                                  3\n",
       "bioconductor-pd.bovine                                          3\n",
       "bioconductor-pd.bsubtilis                                       3\n",
       "bioconductor-knowseq                                            2\n",
       "bioconductor-circrnaprofiler                                    2\n",
       "bioconductor-txdb.drerio.ucsc.danrer11.refgene                  2\n",
       "bioconductor-rcm                                                2\n",
       "bioconductor-gapgom                                             2\n",
       "bioconductor-rat2302frmavecs                                    2\n",
       "hmftools-sage                                                   2\n",
       "bioconductor-txdb.celegans.ucsc.ce11.ensgene                    2\n",
       "bioconductor-fcoex                                              2\n",
       "bioconductor-repviz                                             2\n",
       "bioconductor-metagene2                                          2\n",
       "guessmylt                                                       2\n",
       "bioconductor-isocorrectorgui                                    2\n",
       "bioconductor-cnvfilter                                          2\n",
       "bioconductor-evaluomer                                          2\n",
       "bioconductor-hcabrowser                                         2\n",
       "borf                                                            2\n",
       "bioconductor-pwrewas                                            2\n",
       "fastalite                                                       2\n",
       "biopet-validatevcf                                              2\n",
       "bioconductor-rscudo                                             2\n",
       "fings                                                           2\n",
       "bioconductor-xeva                                               2\n",
       "bioconductor-crossicc                                           2\n",
       "dinosaur                                                        2\n",
       "bioconductor-biscuiteerdata                                     2\n",
       "bioconductor-biotip                                             2\n",
       "bioconductor-probatch                                           2\n",
       "bioconductor-blacksheepr                                        2\n",
       "bioconductor-modstrings                                         2\n",
       "bioconductor-encodexplorerdata                                  2\n",
       "kronos                                                          2\n",
       "bioconductor-scds                                               2\n",
       "bioconductor-prince                                             2\n",
       "bioconductor-precisiontrialdrawer                               2\n",
       "bioconductor-lipidr                                             2\n",
       "bioconductor-mcagui                                             2\n",
       "bioconductor-sctensor                                           2\n",
       "bioconductor-divergence                                         2\n",
       "bioconductor-past                                               2\n",
       "bioconductor-melissa                                            2\n",
       "bioconductor-benchmarkfdrdata2019                               2\n",
       "bioconductor-hyper                                              2\n",
       "bioconductor-pipeframe                                          2\n",
       "bioconductor-humantranscriptomecompendium                       2\n",
       "kggseq                                                          2\n",
       "bioconductor-nbamseq                                            2\n",
       "bioconductor-survtype                                           2\n",
       "bioconductor-alphabeta                                          2\n",
       "bioconductor-topconfects                                        2\n",
       "bioconductor-ssrch                                              2\n",
       "bioconductor-tenxbusdata                                        2\n",
       "bioconductor-nanotubes                                          2\n",
       "bioconductor-dnabarcodecompatibility                            2\n",
       "bioconductor-easyqpcr                                           2\n",
       "bioconductor-autotuner                                          2\n",
       "bioconductor-omicslonda                                         2\n",
       "convert_zero_one_based                                          2\n",
       "bioconductor-phyloprofiledata                                   2\n",
       "bioconductor-treesummarizedexperiment                           2\n",
       "bioconductor-flowsorted.cordbloodcombined.450k                  2\n",
       "bioconductor-aspediafi                                          2\n",
       "bioconductor-phemd                                              2\n",
       "bioconductor-tnbc.cms                                           2\n",
       "bioconductor-mafdb.gnomadex.r2.1.grch38                         2\n",
       "bioconductor-synmut                                             2\n",
       "bioconductor-deco                                               2\n",
       "bioconductor-depmap                                             2\n",
       "ldsc                                                            2\n",
       "bioconductor-doser                                              2\n",
       "bioconductor-vcfarray                                           2\n",
       "kounta                                                          2\n",
       "bioconductor-enrichtf                                           2\n",
       "bioconductor-pram                                               2\n",
       "biobb_adapters                                                  2\n",
       "bioconductor-alps                                               2\n",
       "bioconductor-spectraltad                                        2\n",
       "bioconductor-spatialcpie                                        2\n",
       "bioconductor-apalyzer                                           2\n",
       "biopet-validateannotation                                       2\n",
       "bioconductor-mmappr2                                            2\n",
       "abritamr                                                        2\n",
       "bioconductor-calm                                               2\n",
       "biopet-basecounter                                              2\n",
       "bioconductor-tpp2d                                              2\n",
       "bioconductor-delayeddataframe                                   2\n",
       "bioconductor-muscdata                                           2\n",
       "bioconductor-gramm4r                                            1\n",
       "bioconductor-dmcfb                                              1\n",
       "jmztab-m                                                        1\n",
       "bioconductor-fcscan                                             1\n",
       "bioconductor-msstatssamplesize                                  1\n",
       "bioconductor-flowspecs                                          1\n",
       "bioconductor-gemini                                             1\n",
       "bioconductor-genomicozone                                       1\n",
       "bioconductor-gmicr                                              1\n",
       "bioconductor-scanvis                                            1\n",
       "bioconductor-gscreend                                           1\n",
       "bioconductor-hcaexplorer                                        1\n",
       "bioconductor-hmp2data                                           1\n",
       "bioconductor-gcsscore                                           1\n",
       "karyopype                                                       1\n",
       "bioconductor-omicspca                                           1\n",
       "bioconductor-txdb.ptroglodytes.ucsc.pantro6.ref...              1\n",
       "bioconductor-sbgnview                                           1\n",
       "bioconductor-sbgnview.data                                      1\n",
       "bioconductor-dewseq                                             1\n",
       "bioconductor-discorhythm                                        1\n",
       "bioconductor-macsquantifyr                                      1\n",
       "bioconductor-deltacapturec                                      1\n",
       "bioconductor-debcam                                             1\n",
       "bioconductor-scdblfinder                                        1\n",
       "bioconductor-mafdb.gnomad.r2.1.grch38                           1\n",
       "bioconductor-meb                                                1\n",
       "bioconductor-viseago                                            1\n",
       "happer                                                          1\n",
       "bioconductor-metavolcanor                                       1\n",
       "bioconductor-methcp                                             1\n",
       "bioconductor-methrix                                            1\n",
       "bioconductor-methylcc                                           1\n",
       "bioconductor-variantexperiment                                  1\n",
       "bioconductor-microbiomedasim                                    1\n",
       "bioconductor-mmappr2data                                        1\n",
       "bioconductor-mmuphin                                            1\n",
       "bioconductor-mosim                                              1\n",
       "bioconductor-msgfgui                                            1\n",
       "bioconductor-lionessr                                           1\n",
       "bioconductor-linkhd                                             1\n",
       "bioconductor-xcir                                               1\n",
       "bioconductor-idr2d                                              1\n",
       "bioconductor-scbfa                                              1\n",
       "bioconductor-perfect                                            1\n",
       "bioconductor-schex                                              1\n",
       "bioconductor-scpca                                              1\n",
       "bioconductor-qtlizer                                            1\n",
       "bioconductor-reactomegsa                                        1\n",
       "bioconductor-reactomegsa.data                                   1\n",
       "bioconductor-rnamodr                                            1\n",
       "bioconductor-rnamodr.alkanilineseq                              1\n",
       "bioconductor-proda                                              1\n",
       "fastaptamer                                                     1\n",
       "bioconductor-toast                                              1\n",
       "decoypyrat                                                      1\n",
       "emmtyper                                                        1\n",
       "crypto_typer                                                    1\n",
       "clsify                                                          1\n",
       "clusterpicker                                                   1\n",
       "deepsvr                                                         1\n",
       "chunked-scatter                                                 1\n",
       "ega2                                                            1\n",
       "encyclopedia                                                    1\n",
       "ezclermont                                                      1\n",
       "extract-codon-alignment                                         1\n",
       "bioconductor-pwrewas.data                                       1\n",
       "bioconductor-pulsedsilac                                        1\n",
       "bioconductor-phyloprofile                                       1\n",
       "bioconductor-rnasense                                           1\n",
       "bioconductor-setools                                            1\n",
       "bioconductor-signaturesearchdata                                1\n",
       "bioconductor-sigspack                                           1\n",
       "bioconductor-spaniel                                            1\n",
       "bioconductor-sqldataframe                                       1\n",
       "bioconductor-sspaths                                            1\n",
       "bioconductor-target                                             1\n",
       "bioconductor-tartare                                            1\n",
       "bioconductor-rnamodr.ribomethseq                                1\n",
       "bioconductor-omnipathr                                          1\n",
       "bioconductor-rnamodr.ml                                         1\n",
       "bioconductor-rnamodr.data                                       1\n",
       "bioconductor-oppti                                              1\n",
       "bioconductor-org.mxanthus.db                                    1\n",
       "bioconductor-txdb.mmulatta.ucsc.rhemac10.refgene                1\n",
       "bioconductor-txdb.ggallus.ucsc.galgal6.refgene                  1\n",
       "bioconductor-txdb.btaurus.ucsc.bostau9.refgene                  1\n",
       "bioconductor-peakpanther                                        1\n",
       "bioconductor-maaslin2                                           1"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "usage=usage.sort_values('usage_2019_01',ascending=False)\n",
    "usage=usage.set_index('module')\n",
    "usage"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Qgrid is a "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "import qgrid"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "myqgrid=qgrid.show_grid(usage,show_toolbar=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "1decd998c7dd43b19c09d35854e11dd7",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "QgridWidget(grid_options={'fullWidthRows': True, 'syncColumnCellResize': True, 'forceFitColumns': True, 'defau…"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "myqgrid"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
